# **Available software**

----

<!-- Reference: https://docs.alliancecan.ca/wiki/Available_software -->

A current list of the software available on our SDRE environment is below. This list changes frequently as new software is added. You can request the installation or updating of a particular program or library by contacting technical support. If you wish to use our software environment on your own system, please see Accessing CVMFS.

## **Notes**

----

Except for basic system programs, you access most software by loading a **module**. See [Using modules](/pages/high_perf_computing/using_modules/) for more on how to use the Lmod module system. Note that some prerequisite modules are loaded by default.

Here are a few things to know about the available software:

- Most Python modules are not installed as (Lmod) modules. They are instead provided as binary wheels, stored on our systems under `/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/`. One such package is TensorFlow. For instructions on how to install or list Python packages, see the [Virtual environment](/pages/software_catalog/virtual_environment/) and [Available Python wheels](/pages/software_catalog/available_python_wheels/) pages.

- Similarly, most R or Perl packages are not installed either. We recommend installing them in your personal or group file space. See the [R](/pages/related_links/r/) and [Perl](/pages/related_links/perl/) pages for instructions on how to do so.

- A page discusses [symbolic algebra software](/pages/related_links/symbolic_algebra_software/) like Mathematica and Sage.

- Note that [Docker](https://www.docker.com/) is not available on our clusters but [Apptainer](/pages/software_catalog/apptainer/) is available by loading the module apptainer. Docker containers can be converted to Apptainer as discussed [here](https://apptainer.org/documentation/).

- Some of the software packages listed below are not immediately usable because they require you to have a license. You may need to be granted access to them by us. Attempting to load the module for one of these will give you instructions on what to do to obtain access.

- The packages listed below are available in one or more [standard software environments](/pages/high_perf_computing/standard_software_environment/). In rare circumstances it may be necessary to load a different standard environment (StdEnv) to access a particular version of a particular package. For more on this please read [Standard software environments](/pages/high_perf_computing/standard_software_environment/).

- Many packages related to the operating system, such as Autotools, Make, Git, and others, are not installed as modules but are part of the default environment. These are not listed below.

## **List of globally-installed modules**

----

The table below lists software for which an environment module has been installed on our systems. 

In simple cases, the module name listed in the Module column can be used with the `module load` command to configure your environment. In more complicated cases, some prerequisite modules may also need to be loaded. Click on the corresponding [Expand] link in the Description column for a list of prerequisites and brief notes about the software. If more extensive documentation about a package is available, there will be a link in the Documentation column. 

Click the double arrows in the column heading to sort in ascending or descending order for a given column. In particular, sorting by software type might be of interest. The Type column shows the software tagged as: ai (artificial intelligence), bio (biology, bioinformatics), chem (chemistry), geo (earth sciences), io (input/output), math (mathematics), mpi (MPI), phys (physics and engineering), tools (languages and libraries), vis (visualization).

=== "AVX 512"

	|**Module**|**Type**|**Versions**|**Description**|
	|:----:|:----:|:----:|:----:|
	|[4ti2](https://4ti2.github.io/)|-|1.6.9|<details><summary>Expand</summary>Description: A software package for algebraic, geometric and combinatorial problems on linear spaces Homepage: https://4ti2.github.io/ URL: https://4ti2.github.io/</details>|
	|[abaqus](http://www.simulia.com/products/abaqus_fea.html)|phys|2021|<details><summary>Expand</summary>Description: Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA. Homepage: http://www.simulia.com/products/abaqus_fea.html URL: http://www.simulia.com/products/abaqus_fea.html</details>|
	|[abinit](https://www.abinit.org/)|chem|9.2.2, 9.6.2, 10.0.3, 10.2.3|<details><summary>Expand</summary>Description: ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. Homepage: https://www.abinit.org/ URL: https://www.abinit.org/</details>|
	|[abricate](https://github.com/tseemann/abricate)|-|1.0.0|<details><summary>Expand</summary>Description: Mass screening of contigs for antimicrobial and virulence genes Homepage: https://github.com/tseemann/abricate URL: https://github.com/tseemann/abricate</details>|
	|[abseil](https://abseil.io/)|-|20230125.3|<details><summary>Expand</summary>Description: Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives. Homepage: https://abseil.io/ URL: https://abseil.io/</details>|
	|[abyss](https://www.bcgsc.ca/platform/bioinfo/software/abyss)|bio|2.2.5, 2.3.7|<details><summary>Expand</summary>Description: Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler Homepage: https://www.bcgsc.ca/platform/bioinfo/software/abyss URL: https://www.bcgsc.ca/platform/bioinfo/software/abyss</details>|
	|[actc](https://sourceforge.net/projects/actc)|-|1.1|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[admixture](https://dalexander.github.io/admixture/)|bio|1.3.0|<details><summary>Expand</summary>Description: ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. Homepage: https://dalexander.github.io/admixture/ URL: https://dalexander.github.io/admixture/</details>|
	|[adol-c](https://projects.coin-or.org/ADOL-C)|-|2.7.2|<details><summary>Expand</summary>Description: The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C. Homepage: https://projects.coin-or.org/ADOL-C URL: https://projects.coin-or.org/ADOL-C</details>|
	|[adtree](https://github.com/tudelft3d/AdTree)|-|1.1.2|<details><summary>Expand</summary>Description: AdTree: Accurate, Detailed, and Automatic Modelling of Laser-Scanned Trees Homepage: https://github.com/tudelft3d/AdTree URL: https://github.com/tudelft3d/AdTree</details>|
	|[advisor](https://software.intel.com/intel-advisor-xe)|tools|2020.3|<details><summary>Expand</summary>Description: Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain Homepage: https://software.intel.com/intel-advisor-xe URL: https://software.intel.com/intel-advisor-xe</details>|
	|[afni](http://afni.nimh.nih.gov/)|bio|20.3.05, 21.2.10, 22.1.12, 23.1.00, 23.1.08, 24.1.03|<details><summary>Expand</summary>Description: AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data - a technique for mapping human brain activity. Homepage: http://afni.nimh.nih.gov/ URL: http://afni.nimh.nih.gov/</details>|
	|[alevin-fry](https://github.com/COMBINE-lab/alevin-fry)|-|0.8.2|<details><summary>Expand</summary>Description: alevin-fry is an efficient and flexible tool for processing single-cell sequencing data, currently focused on single-cell transcriptomics and feature barcoding. Homepage: https://github.com/COMBINE-lab/alevin-fry URL: https://github.com/COMBINE-lab/alevin-fry</details>|
	|[alm](https://github.com/ttadano/ALM)|-|2.0.0_dev.2|<details><summary>Expand</summary>Description: Spglib is a software for calculating harmonic and anharmonic interatomic force constants in solids and molecules Homepage: https://github.com/ttadano/ALM URL: https://github.com/ttadano/ALM</details>|
	|[almosthere](https://github.com/horta/almosthere)|-|1.0.10|<details><summary>Expand</summary>Description: Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows, Linux, and macOS. It is non-blocking as the progress update is done via a dedicated, lightweight thread, as to not impair the performance of the calling program. Homepage: https://github.com/horta/almosthere URL: https://github.com/horta/almosthere</details>|
	|[alpscore](https://github.com/ALPSCore/ALPSCore)|phys|2.2.0|<details><summary>Expand</summary>Description: The ALPSCore project, based on the ALPS (Algorithms and Libraries for Physics Simulations) project, provides generic algorithms and utilities for physics problems. It strives to increase software reuse in the physics community. For copyright see COPYRIGHT.TXT For licensing see LICENSE.TXT For acknowledgment in scientific publications see ACKNOWLEDGE.TXT Homepage: https://github.com/ALPSCore/ALPSCore URL: https://github.com/ALPSCore/ALPSCore</details>|
	|[amber](https://ambermd.org/)|chem|18.14-18.17, 20.9-20.15, 20.12-20.15, 22.5-23.5|<details><summary>Expand</summary>Description: Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction. Homepage: https://ambermd.org/ URL: https://ambermd.org/</details>|
	|[ambertools](https://ambermd.org/AmberTools.php)|chem|20, 21, 22, 23, 23.5|<details><summary>Expand</summary>Description: AmberTools consists of several independently developed packages that work well by themselves, and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations, with either explicit water or generalized Born solvent models. Homepage: https://ambermd.org/AmberTools.php URL: https://ambermd.org/AmberTools.php</details>|
	|[amos](http://amos.sourceforge.net)|-|3.1.0|<details><summary>Expand</summary>Description: The AMOS consortium is committed to the development of open-source whole genome assembly software Homepage: http://amos.sourceforge.net URL: http://amos.sourceforge.net</details>|
	|[ampl-mp](https://github.com/ampl/mp)|-|3.1.0|<details><summary>Expand</summary>Description: An open-source library for mathematical programming. Homepage: https://github.com/ampl/mp URL: https://github.com/ampl/mp</details>|
	|[amrfinderplus](https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus)|-|3.11.18, 3.11.26, 3.12.8|<details><summary>Expand</summary>Description: This software and the accompanying database identify acquired antimicrobial resistance genes in bacterial protein and/or assembled nucleotide sequences as well as known resistance-associated point mutations for several taxa. With AMRFinderPlus we added select members of additional classes of genes such as virulence factors, biocide, heat, acid, and metal resistance genes. Homepage: https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus URL: https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus</details>|
	|[amrplusplus](https://megares.meglab.org/amrplusplus/latest/html/v2)|-|2.0-20200114|<details><summary>Expand</summary>Description: AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. Homepage: https://megares.meglab.org/amrplusplus/latest/html/v2 URL: https://megares.meglab.org/amrplusplus/latest/html/v2</details>|
	|[andi](https://github.com/evolbioinf/andi/)|-|0.14|<details><summary>Expand</summary>Description: This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes. Homepage: https://github.com/evolbioinf/andi/ URL: https://github.com/evolbioinf/andi/</details>|
	|[angsd](http://www.popgen.dk/angsd)|bio|0.933, 0.935, 0.936, 0.939, 0.940|<details><summary>Expand</summary>Description: Program for analysing NGS data. Homepage: http://www.popgen.dk/angsd URL: http://www.popgen.dk/angsd</details>|
	|[annovar](http://annovar.openbioinformatics.org/en/latest/)|bio|20191024|<details><summary>Expand</summary>Description: ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). Homepage: http://annovar.openbioinformatics.org/en/latest/ URL: http://annovar.openbioinformatics.org/en/latest/</details>|
	|[anserini](https://github.com/castorini/anserini)|-|0.9.4|<details><summary>Expand</summary>Description: An open-source information retreival toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h Homepage: https://github.com/castorini/anserini URL: https://github.com/castorini/anserini</details>|
	|[ansys](http://www.ansys.com)|phys|2024R1.03, 2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ansysedt](http://www.ansys.com)|-|2021R2, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ant](https://ant.apache.org/)|tools|1.9.15, 1.10.8, 1.10.14|<details><summary>Expand</summary>Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. Homepage: https://ant.apache.org/ URL: https://ant.apache.org/</details>|
	|[antlr](https://www.antlr2.org/)|-|2.7.7|<details><summary>Expand</summary>Description: ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. Homepage: https://www.antlr2.org/ URL: https://www.antlr2.org/</details>|
	|[ants](https://stnava.github.io/ANTs/)|vis|2.3.2, 2.3.5, 2.4.4, 2.5.0|<details><summary>Expand</summary>Description: ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing, interpreting and visualizing multidimensional data. Homepage: https://stnava.github.io/ANTs/ URL: https://stnava.github.io/ANTs/</details>|
	|[any2fasta](https://github.com/tseemann/any2fasta)|-|0.4.2|<details><summary>Expand</summary>Description: Convert various sequence formats to FASTA Homepage: https://github.com/tseemann/any2fasta URL: https://github.com/tseemann/any2fasta</details>|
	|[apbs](http://www.poissonboltzmann.org/apbs)|chem|1.3|<details><summary>Expand</summary>Description: APBS is a software package for modeling biomolecular solvation through solution of the Poisson-Boltzmann equation (PBE), one of the most popular continuum models for describing electrostatic interactions between molecular solutes in salty, aqueous media. Homepage: http://www.poissonboltzmann.org/apbs URL: http://www.poissonboltzmann.org/apbs</details>|
	|[apptainer](https://apptainer.org)|-|1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4, 1.3.4, 1.3.5|<details><summary>Expand</summary>Description: Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. Homepage: https://apptainer.org URL: https://apptainer.org</details>|
	|[aragorn](http://www.ansikte.se/ARAGORN/)|-|1.2.38, 1.2.41|<details><summary>Expand</summary>Description: a program to detect tRNA genes and tmRNA genes in nucleotide sequences Homepage: http://www.ansikte.se/ARAGORN/ URL: http://www.ansikte.se/ARAGORN/</details>|
	|[arcs](https://github.com/bcgsc/arcs)|bio|1.2.1, 1.2.5, 1.2.7|<details><summary>Expand</summary>Description: Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. Homepage: https://github.com/bcgsc/arcs URL: https://github.com/bcgsc/arcs</details>|
	|[argtable](http://argtable.sourceforge.net/)|tools|2.13|<details><summary>Expand</summary>Description: Argtable is an ANSI C library for parsing GNU style command line options with a minimum of fuss. Homepage: http://argtable.sourceforge.net/ URL: http://argtable.sourceforge.net/</details>|
	|[arioc](https://sourceforge.net/projects/actc)|-|1.43|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[arks](https://github.com/bcgsc/arcs)|-|1.0.4|<details><summary>Expand</summary>Description: Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. Homepage: https://github.com/bcgsc/arcs URL: https://github.com/bcgsc/arcs</details>|
	|[armadillo](https://arma.sourceforge.net/)|math|9.900.2, 12.6.4|<details><summary>Expand</summary>Description: Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions. Homepage: https://arma.sourceforge.net/ URL: https://arma.sourceforge.net/</details>|
	|[arpack-ng](https://github.com/opencollab/arpack-ng)|math|3.7.0, 3.8.0, 3.9.0, 3.9.1|<details><summary>Expand</summary>Description: ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems. Homepage: https://github.com/opencollab/arpack-ng URL: https://github.com/opencollab/arpack-ng</details>|
	|[arrayfire](https://arrayfire.com/)|-|3.7.3, 3.8.2, 3.9.0|<details><summary>Expand</summary>Description: ArrayFire is a general-purpose library that simplifies the process of developing software that targets parallel and massively-parallel architectures including CPUs, GPUs, and other hardware acceleration devices. Homepage: https://arrayfire.com/ URL: https://arrayfire.com/</details>|
	|[arrow](https://arrow.apache.org)|tools|0.16.0, 0.17.1, 2.0.0, 5.0.0, 8.0.0, 9.0.0, 10.0.1, 11.0.0, 12.0.1, 13.0.0, 14.0.0, 14.0.1, 15.0.1, 16.1.0, 17.0.0, 18.1.0, 19.0.1|<details><summary>Expand</summary>Description: Apache Arrow is a cross-language development platform for in-memory data. Homepage: https://arrow.apache.org URL: https://arrow.apache.org</details>|
	|[ascp](http://downloads.asperasoft.com/)|tools|3.5.4|<details><summary>Expand</summary>Description: Aspera ASCP is a high speed file transfer software Homepage: http://downloads.asperasoft.com/ URL: http://downloads.asperasoft.com/</details>|
	|[aspect](https://aspect.geodynamics.org/)|-|2.4.0|<details><summary>Expand</summary>Description: ASPECT: Advanced Solver for Problems in Earth's ConvecTion. An extensible code written in C++ to support research in simulating convection in the Earth's mantle and elsewhere to provide the geosciences with a well-documented and extensible code base for their research needs and to create an open, inclusive, participatory community providing users and developers with a state-of-the-art, comprehensive software that performs well while being simple to extend. Homepage: https://aspect.geodynamics.org/ URL: https://aspect.geodynamics.org/</details>|
	|[assimp](https://github.com/assimp/assimp)|-|5.0.0, 5.2.5|<details><summary>Expand</summary>Description: Open Asset Import Library (assimp) is a library to import and export various 3d-model-formats including scene-post-processing to generate missing render data. Homepage: https://github.com/assimp/assimp URL: https://github.com/assimp/assimp</details>|
	|[astrid](https://github.com/pranjalv123/ASTRID)|-|2.2.1|<details><summary>Expand</summary>Description: ASTRID-2 is a method for estimating species trees from gene trees. Homepage: https://github.com/pranjalv123/ASTRID URL: https://github.com/pranjalv123/ASTRID</details>|
	|[atat](https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/)|-|3.36|<details><summary>Expand</summary>Description: ATAT is a generic name that refers to a collection of alloy theory tools developped by Axel van de Walle, in collaboration with various research groups and with various sources of financial support. Homepage: https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/ URL: https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/</details>|
	|[atom](https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html)|chem|4.2.7_2|<details><summary>Expand</summary>Description: ATOM is the name of a program originally written (circa 1982) by Sverre Froyen at the University of California at Berkeley, modified starting in 1990 by Norman Troullier and Jose Luis Martins at the University of Minnesota, and currently maintained by Alberto Garcia, who added some features and made substantial structural changes to the April 1990 (5.0) Minnesota version while at Berkeley and elsewhere. Homepage: https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html URL: https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html</details>|
	|[atomicrex](http://atomicrex.gitlab.io/atomicrex/)|chem|1.0.20181114|<details><summary>Expand</summary>Description: atomicrex is a versatile tool for the construction of advanced atomistic models. It is written in C++ and Python. It was primarily developed to fit interatomic potential models. Thanks to its flexible generic structure its application range, however, is much larger. In a general sense, it allows one to develop models that describe a given property as a function of an atomic (or atom-like) configuration. The property in question can be scalar or vectorial in nature, and could represent e.g., total energies and forces, or eventually electronic eigen energies. It thus provides already the basic framework for constructing in the future for example tight binding models. Homepage: http://atomicrex.gitlab.io/atomicrex/ URL: http://atomicrex.gitlab.io/atomicrex/</details>|
	|[atompaw](http://users.wfu.edu/natalie/papers/pwpaw/man.html)|chem|4.1.0.6, 4.2.0.3|<details><summary>Expand</summary>Description: AtomPAW is a Projector-Augmented Wave Dataset Generator that can be used both as a standalone program and a library. Homepage: http://users.wfu.edu/natalie/papers/pwpaw/man.html URL: http://users.wfu.edu/natalie/papers/pwpaw/man.html</details>|
	|[augustus](https://bioinf.uni-greifswald.de/augustus/)|bio|3.3.3, 3.4.0, 3.5.0|<details><summary>Expand</summary>Description: AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences Homepage: https://bioinf.uni-greifswald.de/augustus/ URL: https://bioinf.uni-greifswald.de/augustus/</details>|
	|[autodiff](https://github.com/autodiff/autodiff)|-|0.6.0, 0.6.12|<details><summary>Expand</summary>Description: autodiff is a C++17 library that uses modern and advanced programming techniques to enable automatic computation of derivatives in an efficient, easy, and intuitive way. Homepage: https://github.com/autodiff/autodiff URL: https://github.com/autodiff/autodiff</details>|
	|[autodock-gpu](https://github.com/ccsb-scripps/AutoDock-GPU)|-|1.5.3, 1.6|<details><summary>Expand</summary>Description: OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure. Homepage: https://github.com/ccsb-scripps/AutoDock-GPU URL: https://github.com/ccsb-scripps/AutoDock-GPU</details>|
	|[autodock_vina](http://vina.scripps.edu/index.html)|chem|1.1.2|<details><summary>Expand</summary>Description: AutoDock Vina is an open-source program for doing molecular docking. Homepage: http://vina.scripps.edu/index.html URL: http://vina.scripps.edu/index.html</details>|
	|[bacio](https://github.com/NOAA-EMC/NCEPLIBS-bacio/)|-|2.6.0|<details><summary>Expand</summary>Description: This library performs binary I/O for the NCEP models, processing unformatted byte-addressable data records, and transforming the little endian files and big endian files. This is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-bacio/ URL: https://github.com/NOAA-EMC/NCEPLIBS-bacio/</details>|
	|[bamm](http://bamm-project.org/)|bio|2.5.0|<details><summary>Expand</summary>Description: BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates. It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of lineage diversification and trait evolution. Homepage: http://bamm-project.org/ URL: http://bamm-project.org/</details>|
	|[bamtools](https://github.com/pezmaster31/bamtools)|bio|2.5.1, 2.5.2|<details><summary>Expand</summary>Description: BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files. Homepage: https://github.com/pezmaster31/bamtools URL: https://github.com/pezmaster31/bamtools</details>|
	|[bamutil](http://genome.sph.umich.edu/wiki/BamUtil)|bio|1.0.14|<details><summary>Expand</summary>Description: BamUtil is a repository that contains several programs that perform operations on SAM/BAM files. All of these programs are built into a single executable, bam. Homepage: http://genome.sph.umich.edu/wiki/BamUtil URL: http://genome.sph.umich.edu/wiki/BamUtil</details>|
	|[barracuda](https://sourceforge.net/projects/seqbarracuda/)|-|0.7.107h|<details><summary>Expand</summary>Description: Barracuda is a high-speed sequence aligner based on Sanger's BWA and utilizes the latest Nvidia CUDA architecture for accelerating alignments of sequence reads generated by next-generation sequencers. Homepage: https://sourceforge.net/projects/seqbarracuda/ URL: https://sourceforge.net/projects/seqbarracuda/</details>|
	|[barrnap](https://github.com/tseemann/barrnap)|-|0.9|<details><summary>Expand</summary>Description: Barrnap predicts the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), metazoan mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S). It takes FASTA DNA sequence as input, and write GFF3 as output. It uses the new nhmmer tool that comes with HMMER 3.1 for HMM searching in RNA:DNA style. Multithreading is supported and one can expect roughly linear speed-ups with more CPUs. Homepage: https://github.com/tseemann/barrnap URL: https://github.com/tseemann/barrnap</details>|
	|[bayesass](http://www.rannala.org/?page_id=245)|bio|3.0.4|<details><summary>Expand</summary>Description: BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes Homepage: http://www.rannala.org/?page_id=245 URL: http://www.rannala.org/?page_id=245</details>|
	|[bayesass3-snps](http://www.rannala.org/inference-of-recent-migration)|-|1.1|<details><summary>Expand</summary>Description: Modification of BayesAss 3.0.4 to allow handling of large SNP datasets generated via methods such as RADseq protocols. This is an enhancement of the code produced by the Rannala Research Group: http://www.rannala.org/inference-of-recent-migration Most changes will not be obvious to the average user. However, the most important changes will now allow the user to input large SNP datasets that previously would have caused BayesAss to crash with a "segmentation fault" error. The upper limit of 420 loci has been removed. Homepage: https://github.com/stevemussmann/BayesAss3-SNPs URL: https://github.com/stevemussmann/BayesAss3-SNPs</details>|
	|[bayescan](http://cmpg.unibe.ch/software/BayeScan/)|bio|2.1|<details><summary>Expand</summary>Description: BayeScan aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. Homepage: http://cmpg.unibe.ch/software/BayeScan/ URL: http://cmpg.unibe.ch/software/BayeScan/</details>|
	|[baypass](http://www1.montpellier.inra.fr/CBGP/software/baypass/)|bio|2.1, 2.2|<details><summary>Expand</summary>Description: The package BayPass is a population genomics software which is primarily aimed at identifying genetic markers subjected to selection and/or associated to population-specific covariates (e.g., environmental variables, quantitative or categorical phenotypic characteristics). Homepage: http://www1.montpellier.inra.fr/CBGP/software/baypass/ URL: http://www1.montpellier.inra.fr/CBGP/software/baypass/</details>|
	|[bazel](https://bazel.io/)|tools|3.6.0|<details><summary>Expand</summary>Description: Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software. Homepage: https://bazel.io/ URL: https://bazel.io/</details>|
	|[bbmap](https://sourceforge.net/projects/bbmap/)|bio|37.78, 38.86, 39.06|<details><summary>Expand</summary>Description: BBMap short read aligner, and other bioinformatic tools. Homepage: https://sourceforge.net/projects/bbmap/ URL: https://sourceforge.net/projects/bbmap/</details>|
	|[bcftools](https://www.htslib.org/)|bio|1.9, 1.10.2, 1.11, 1.13, 1.16, 1.18, 1.19|<details><summary>Expand</summary>Description: Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[bcl-convert](https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html)|-|4.2.4, 4.2.7|<details><summary>Expand</summary>Description: The Illumina BCL Convert v4.0 is a standalone local software app that converts the Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files. Homepage: https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html URL: https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html</details>|
	|[bcl2fastq2](https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html)|bio|2.20.0|<details><summary>Expand</summary>Description: bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by Illumina sequencing systems to standard FASTQ file formats for downstream analysis. Homepage: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html URL: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html</details>|
	|[beagle](https://faculty.washington.edu/browning/beagle/beagle.html)|-|5.4, 5.4-240301|<details><summary>Expand</summary>Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers. Homepage: https://faculty.washington.edu/browning/beagle/beagle.html URL: https://faculty.washington.edu/browning/beagle/beagle.html</details>|
	|[beagle-lib](https://github.com/beagle-dev/beagle-lib)|bio|3.1.2, 4.0.0, 4.0.1|<details><summary>Expand</summary>Description: beagle-lib is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. Homepage: https://github.com/beagle-dev/beagle-lib URL: https://github.com/beagle-dev/beagle-lib</details>|
	|[beast](http://beast2.org)|bio|2.6.3, 2.7.5|<details><summary>Expand</summary>Description: BEAST is a cross-platform program for Bayesian MCMC analysis of molecular sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted proportional to its posterior probability. Homepage: http://beast2.org URL: http://beast2.org</details>|
	|[bedops](http://bedops.readthedocs.io/en/latest/index.html)|-|2.4.39, 2.4.41|<details><summary>Expand</summary>Description: BEDOPS is an open-source command-line toolkit that performs highly efficient and scalable Boolean and other set operations, statistical calculations, archiving, conversion and other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for distributing whole-genome analyses across a computational cluster. Homepage: http://bedops.readthedocs.io/en/latest/index.html URL: http://bedops.readthedocs.io/en/latest/index.html</details>|
	|[bedtools](https://github.com/arq5x/bedtools2)|bio|2.29.2, 2.30.0, 2.31.0|<details><summary>Expand</summary>Description: The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM. Homepage: https://github.com/arq5x/bedtools2 URL: https://github.com/arq5x/bedtools2</details>|
	|[beef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|chem|0.1.1|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: http://suncat.stanford.edu/facility/software/functional URL: http://suncat.stanford.edu/facility/software/functional</details>|
	|[berkeleygw](https://arxiv.org/abs/1111.4429).)|phys|2.1.0, 3.0.1, 4.0|<details><summary>Expand</summary>Description: The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle properties and the optical responses of a large variety of materials from bulk periodic crystals to nanostructures such as slabs, wires and molecules. We ask that you please cite the following papers when using results obtained from the BerkeleyGW package: - Mark S. Hybertsen and Steven G. Louie, Electron correlation in semiconductors and insulators: Band gaps and quasiparticle energies, Phys. Rev. B 34, 5390 (1986). - Michael Rohlfing and Steven G. Louie, Electron-hole excitations and optical spectra from first principles, Phys. Rev. B 62, 4927 (2000). - Jack Deslippe, Georgy Samsonidze, David A. Strubbe, Manish Jain, Marvin L. Cohen, and Steven G. Louie, BerkeleyGW: A Massively Parallel Computer Package for the Calculation of the Quasiparticle and Optical Properties of Materials and Nanostructures, Comput. Phys. Commun. 183, 1269 (2012) (https://arxiv.org/abs/1111.4429). Homepage: http://www.berkeleygw.org URL: http://www.berkeleygw.org</details>|
	|[bgcdist](https://sites.google.com/site/bgcsoftware/)|-|1.03|<details><summary>Expand</summary>Description: bgc implements Bayesian estimation of genomic clines to quantify introgression at many loci. Models are available for linked loci, genotype uncertainty and sequence errors. Models for genotype uncertainty and sequence error can be used to estimate genomic cline parameters from next-generation sequence data. Homepage: https://sites.google.com/site/bgcsoftware/ URL: https://sites.google.com/site/bgcsoftware/</details>|
	|[bgen-lib](https://enkre.net/cgi-bin/code/bgen)|-|1.1.7|<details><summary>Expand</summary>Description: This repository contains a reference implementation of the BGEN format, written in C++. The library can be used as the basis for BGEN support in other software, or as a reference for developers writing their own implementations of the BGEN format. What's included? This repository contains the library itself, a set of example data files, and a number of example programs (e.g. bgen_to_vcf) that demonstrate the use of the library API. In addition, a number of utilities built using the library are also included in this repository: bgenix - a tool to index and efficiently retrieve subsets of a BGEN file. cat-bgen - a tool to efficiently concatenate BGEN files. edit-bgen - a tool to edit BGEN file metadata. An R package called rbgen is also constructed in the build directory. See the rbgen wiki page for more information on using this package. Homepage: https://enkre.net/cgi-bin/code/bgen URL: https://enkre.net/cgi-bin/code/bgen</details>|
	|[bigdft](http://bigdft.org/Wiki/index.php?title=BigDFT_website)|chem|1.8.3, 1.9.5|<details><summary>Expand</summary>Description: BigDFT is a DFT massively parallel electronic structure code using a wavelet basis set with the capability to use a linear scaling method. Wavelets form a real space basis set distributed on an adaptive mesh (two levels of resolution in our implementation). GTH or HGH pseudopotentials are used to remove the core electrons. The code BigDFT is available in ABINIT v5.5 and higher but can also be downloaded in a standalone version from the website. Thanks to our Poisson solver based on a Green function formalism, periodic systems, surfaces and isolated systems can be simulated with explicit boundary conditions. The Poisson solver can also be downloaded and used independently and is integrated in ABINIT, Octopus and CP2K. The code is free software, available under GNU-GPL license and the BigDFT developer community encourages anyone willing to contribute to join the team. Homepage: http://bigdft.org/Wiki/index.php?title=BigDFT_website URL: http://bigdft.org/Wiki/index.php?title=BigDFT_website</details>|
	|[bio-searchio-hmmer](http://hmmer.org.)|-|1.7.3|<details><summary>Expand</summary>Description: Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org. Homepage: https://metacpan.org/pod/Bio::SearchIO::hmmer3 URL: https://metacpan.org/pod/Bio::SearchIO::hmmer3</details>|
	|[bioawk](https://github.com/lh3/bioawk)|-|1.0|<details><summary>Expand</summary>Description: Bioawk is an extension to Brian Kernighan's awk, adding the support of several common biological data formats, including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. Homepage: https://github.com/lh3/bioawk URL: https://github.com/lh3/bioawk</details>|
	|[biobloomtools](https://github.com/bcgsc/biobloom)|-|2.3.2-20200731|<details><summary>Expand</summary>Description: BioBloom Tools (BBT) provides the means to create filters for a given reference and then to categorize sequences. This methodology is faster than alignment but does not provide mapping locations. BBT was initially intended to be used for pre-processing and QC applications like contamination detection, but is flexible to accommodate other purposes. This tool is intended to be a pipeline component to replace costly alignment steps. Homepage: https://github.com/bcgsc/biobloom URL: https://github.com/bcgsc/biobloom</details>|
	|[bioperl](https://bioperl.org/)|bio|1.7.7, 1.7.8|<details><summary>Expand</summary>Description: Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects. Homepage: https://bioperl.org/ URL: https://bioperl.org/</details>|
	|[biopp](https://github.com/BioPP/bpp-core)|-|2.4.1|<details><summary>Expand</summary>Description: Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics. Bio++ is Object Oriented and is designed to be both easy to use and computer efficient. Bio++ intends to help programmers to write computer expensive programs, by providing them a set of re-usable tools. Homepage: https://github.com/BioPP/bpp-core URL: https://github.com/BioPP/bpp-core</details>|
	|[bismark](http://www.bioinformatics.babraham.ac.uk/projects/bismark/)|-|0.22.3, 0.24.1|<details><summary>Expand</summary>Description: A tool to map bisulfite converted sequence reads and determine cytosine methylation states Homepage: http://www.bioinformatics.babraham.ac.uk/projects/bismark/ URL: http://www.bioinformatics.babraham.ac.uk/projects/bismark/</details>|
	|[bison](https://www.gnu.org/software/bison)|-|3.7.1, 3.8.2|<details><summary>Expand</summary>Description: Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables. Homepage: https://www.gnu.org/software/bison URL: https://www.gnu.org/software/bison</details>|
	|[blacs](http://www.netlib.org/blacs/)|math|1.1|<details><summary>Expand</summary>Description: The BLACS (Basic Linear Algebra Communication Subprograms) project is an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface that may be implemented efficiently and uniformly across a large range of distributed memory platforms. Homepage: http://www.netlib.org/blacs/ URL: http://www.netlib.org/blacs/</details>|
	|[blasr](https://github.com/PacificBiosciences/blasr)|bio|5.3.3|<details><summary>Expand</summary>Description: The PacBio long read aligner Homepage: https://github.com/PacificBiosciences/blasr URL: https://github.com/PacificBiosciences/blasr</details>|
	|[blasr_libcpp](https://github.com/PacificBiosciences/blasr_libcpp)|bio|5.3.4|<details><summary>Expand</summary>Description: Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for analyzing PacBio sequences Homepage: https://github.com/PacificBiosciences/blasr_libcpp URL: https://github.com/PacificBiosciences/blasr_libcpp</details>|
	|[blast](http://blast.ncbi.nlm.nih.gov/)|bio|2.2.26|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: http://blast.ncbi.nlm.nih.gov/ URL: http://blast.ncbi.nlm.nih.gov/</details>|
	|[blast+](https://blast.ncbi.nlm.nih.gov/)|bio|2.10.1, 2.11.0, 2.12.0, 2.13.0, 2.14.1|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: https://blast.ncbi.nlm.nih.gov/ URL: https://blast.ncbi.nlm.nih.gov/</details>|
	|[blat](https://genome.ucsc.edu/goldenPath/help/blatSpec.html)|bio|3.5, 3.7|<details><summary>Expand</summary>Description: BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more. Homepage: https://genome.ucsc.edu/goldenPath/help/blatSpec.html URL: https://genome.ucsc.edu/goldenPath/help/blatSpec.html</details>|
	|[blender](https://www.blender.org/)|vis|2.92.0, 3.6.0, 4.0.2, 4.2.8|<details><summary>Expand</summary>Description: Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. Homepage: https://www.blender.org/ URL: https://www.blender.org/</details>|
	|[blis](https://github.com/flame/blis/)|-|0.8.1, 0.9.0, 1.0|<details><summary>Expand</summary>Description: BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries. Homepage: https://github.com/flame/blis/ URL: https://github.com/flame/blis/</details>|
	|[blitz++](https://github.com/blitzpp/blitz)|tools|1.0.2|<details><summary>Expand</summary>Description: Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++ with a speed comparable to Fortran implementations, while preserving an object-oriented interface Homepage: https://github.com/blitzpp/blitz URL: https://github.com/blitzpp/blitz</details>|
	|[blosc](https://www.blosc.org/)|-|1.17.1, 1.21.3|<details><summary>Expand</summary>Description: Blosc, an extremely fast, multi-threaded, meta-compressor library Homepage: https://www.blosc.org/ URL: https://www.blosc.org/</details>|
	|[blosc2](https://www.blosc.org/)|-|2.10.3|<details><summary>Expand</summary>Description: Blosc, an extremely fast, multi-threaded, meta-compressor library Homepage: https://www.blosc.org/ URL: https://www.blosc.org/</details>|
	|[bmtagger](https://www.google.ca/search?q=bmtagger)|-|3.101|<details><summary>Expand</summary>Description: Best Match Tagger for removing human reads from metagenomics datasets Homepage: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/ URL: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/</details>|
	|[bolt-lmm](https://data.broadinstitute.org/alkesgroup/BOLT-LMM/)|bio|2.3.4, 2.4|<details><summary>Expand</summary>Description: The BOLT-LMM software package currently consists of two main algorithms, the BOLT-LMM algorithm for mixed model association testing, and the BOLT-REML algorithm for variance components analysis (i.e., partitioning of SNP-heritability and estimation of genetic correlations). The BOLT-LMM algorithm computes statistics for testing association between phenotype and genotypes using a linear mixed model (LMM). The BOLT-REML algorithm estimates heritability explained by genotyped SNPs and genetic correlations among multiple traits measured on the same set of individuals. Like the GCTA software. Homepage: https://data.broadinstitute.org/alkesgroup/BOLT-LMM/ URL: https://data.broadinstitute.org/alkesgroup/BOLT-LMM/</details>|
	|[boltztrap](http://www.icams.de/content/departments/cmat/boltztrap/)|phys|1.2.5|<details><summary>Expand</summary>Description: Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport coefficients. Homepage: http://www.icams.de/content/departments/cmat/boltztrap/ URL: http://www.icams.de/content/departments/cmat/boltztrap/</details>|
	|[boost](https://www.boost.org/)|tools|1.72.0, 1.76.0, 1.80.0, 1.82.0, 1.85.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-build](https://www.boost.org/)|-|1.80.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-mpi](https://www.boost.org/)|tools|1.72.0, 1.80.0, 1.82.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[bowtie](http://bowtie-bio.sourceforge.net/index.shtml)|bio|1.3.0|<details><summary>Expand</summary>Description: Bowtie is an ultrafast, memory-efficient short read aligner. It aligns short DNA sequences (reads) to the human genome. Homepage: http://bowtie-bio.sourceforge.net/index.shtml URL: http://bowtie-bio.sourceforge.net/index.shtml</details>|
	|[bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)|bio|2.4.1, 2.4.2, 2.4.4, 2.5.1, 2.5.2, 2.5.4|<details><summary>Expand</summary>Description: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. Homepage: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml URL: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</details>|
	|[bpp](https://github.com/bpp/bpp)|-|4.3.8, 4.6.1|<details><summary>Expand</summary>Description: The aim of this project is to implement a versatile high-performance version of the BPP software. BPP implements four methods: * Estimation of the parameters of species divergence times and population sizes under the multi-species coalescent (MSC) model when the species phylogeny is given * Inference of the species tree when the assignments are given by the user * Species delimitation using a user-specified guide treed * Joint species delimitation and species tree estimation Homepage: https://github.com/bpp/bpp URL: https://github.com/bpp/bpp</details>|
	|[bracken](https://ccb.jhu.edu/software/bracken/)|-|2.6.0, 2.7, 3.0|<details><summary>Expand</summary>Description: Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample. Braken uses the taxonomy labels assigned by Kraken, a highly accurate metagenomics classification algorithm, to estimate the number of reads originating from each species present in a sample. Kraken classifies reads to the best matching location in the taxonomic tree, but does not estimate abundances of species. We use the Kraken database itself to derive probabilities that describe how much sequence from each genome is identical to other genomes in the database, and combine this information with the assignments for a particular sample to estimate abundance at the species level, the genus level, or above. Combined with the Kraken classifier, Bracken produces accurate species- and genus-level abundance estimates even when a sample contains two or more near-identical species. NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the default kmer length is different depending on the version of Kraken used. If you use Kraken 1 defaults, specify 31 as the kmer length. If you use Kraken 2 defaults, specify 35 as the kmer length. Homepage: https://ccb.jhu.edu/software/bracken/ URL: https://ccb.jhu.edu/software/bracken/</details>|
	|[breseq](https://barricklab.org/breseq)|-|0.35.2, 0.36.1, 0.38.2|<details><summary>Expand</summary>Description: breseq is a computational pipeline for the analysis of short-read re-sequencing data Homepage: https://barricklab.org/breseq URL: https://barricklab.org/breseq</details>|
	|[brotli](https://github.com/google/brotli)|-|1.0.9|<details><summary>Expand</summary>Description: Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932. Homepage: https://github.com/google/brotli URL: https://github.com/google/brotli</details>|
	|[brunsli](https://github.com/google/brunsli/)|-|0.1|<details><summary>Expand</summary>Description: Brunsli is a lossless JPEG repacking library. Homepage: https://github.com/google/brunsli/ URL: https://github.com/google/brunsli/</details>|
	|[btllib](https://github.com/bcgsc/btllib)|-|1.6.2|<details><summary>Expand</summary>Description: Bioinformatics Technology Lab common code library Homepage: https://github.com/bcgsc/btllib URL: https://github.com/bcgsc/btllib</details>|
	|[bufr](https://github.com/NOAA-EMC/NCEPLIBS-bufr/)|-|12.0.1|<details><summary>Expand</summary>Description: The NCEPLIBS-bufr library contains routines and utilites for working with the WMO BUFR format. It is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-bufr/ URL: https://github.com/NOAA-EMC/NCEPLIBS-bufr/</details>|
	|[bufrlib](https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB)|geo|11.3.0.2|<details><summary>Expand</summary>Description: NCEP BUFRLIB software to encode or decode BUFR messages. It is not intended to be a primer on the BUFR code form itself. Homepage: https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB URL: https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB</details>|
	|[bullet](http://bulletphysics.org/)|-|2.89, 3.24|<details><summary>Expand</summary>Description: Bullet professional 3D Game Multiphysics Library provides state of the art collision detection, soft body and rigid body dynamics. Homepage: http://bulletphysics.org/ URL: http://bulletphysics.org/</details>|
	|[bustools](https://github.com/BUStools/bustools)|-|0.40.0|<details><summary>Expand</summary>Description: bustools is a program for manipulating BUS files for single cell RNA-Seq datasets. It can be used to error correct barcodes, collapse UMIs, produce gene count or transcript compatibility count matrices, and is useful for many other tasks. See the kallisto bustools website for examples and instructions on how to use bustools as part of a single-cell RNA-seq workflow. Homepage: https://github.com/BUStools/bustools URL: https://github.com/BUStools/bustools</details>|
	|[bwa](http://bio-bwa.sourceforge.net/)|bio|0.7.17, 0.7.18|<details><summary>Expand</summary>Description: Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. Homepage: http://bio-bwa.sourceforge.net/ URL: http://bio-bwa.sourceforge.net/</details>|
	|[bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)|-|2.2.1|<details><summary>Expand</summary>Description: The tool bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case, dataset and the running machine. The original bwa was developed by Heng Li (@lh3). Performance enhancement in bwa-mem2 was primarily done by Vasimuddin Md (@yuk12) and Sanchit Misra (@sanchit-misra) from Parallel Computing Lab, Intel. bwa-mem2 is distributed under the MIT license. Homepage: https://github.com/bwa-mem2/bwa-mem2 URL: https://github.com/bwa-mem2/bwa-mem2</details>|
	|[bwidget](https://core.tcl-lang.org/bwidget/home)|-|1.9.14|<details><summary>Expand</summary>Description: The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces. Homepage: https://core.tcl-lang.org/bwidget/home URL: https://core.tcl-lang.org/bwidget/home</details>|
	|[cafe5](https://github.com/hahnlab/CAFE5)|-|5.1.0|<details><summary>Expand</summary>Description: Software for Computational Analysis of gene Family Evolution The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa. Homepage: https://github.com/hahnlab/CAFE5 URL: https://github.com/hahnlab/CAFE5</details>|
	|[cantera](https://github.com/Cantera/cantera)|chem|2.5.1, 2.6.0|<details><summary>Expand</summary>Description: Chemical kinetics, thermodynamics, and transport tool suite Homepage: https://github.com/Cantera/cantera URL: https://github.com/Cantera/cantera</details>|
	|[canu](https://github.com/marbl/canu)|bio|2.0, 2.1.1, 2.2|<details><summary>Expand</summary>Description: Canu is a fork of the Celera Assembler, designed for high-noise single-molecule sequencing (such as the PacBio RS II or Oxford Nanopore MinION). Canu is a hierarchical assembly pipeline which runs in four steps: Detect overlaps in high-noise sequences using MHAP Generate corrected sequence consensus Trim corrected sequences Assemble trimmed corrected sequences Homepage: https://github.com/marbl/canu URL: https://github.com/marbl/canu</details>|
	|[cap3](http://seq.cs.iastate.edu/)|bio|20151002|<details><summary>Expand</summary>Description: CAP3 assembly program Homepage: http://seq.cs.iastate.edu/ URL: http://seq.cs.iastate.edu/</details>|
	|[capnproto](https://capnproto.org)|-|0.7.0, 1.0.2|<details><summary>Expand</summary>Description: Cap'n Proto is an insanely fast data interchange format and capability-based RPC system. Homepage: https://capnproto.org URL: https://capnproto.org</details>|
	|[casacore](https://github.com/casacore/casacore)|-|3.6.1|<details><summary>Expand</summary>Description: A suite of C++ libraries for radio astronomy data processing. The ephemerides data needs to be in DATA_DIR and the location must be specified at runtime. Thus user's can update them. Homepage: https://github.com/casacore/casacore URL: https://github.com/casacore/casacore</details>|
	|[casadi](https://github.com/casadi/casadi)|-|3.6.3, 3.6.7|<details><summary>Expand</summary>Description: CasADi is an open-source tool for nonlinear optimization and algorithmic differentiation. Homepage: https://github.com/casadi/casadi URL: https://github.com/casadi/casadi</details>|
	|[casper](http://best.snu.ac.kr/casper/)|bio|0.8.2|<details><summary>Expand</summary>Description: CASPER (Context-Aware Scheme for Paired-End Read) is state-of-the art merging tool in terms of accuracy and robustness. Using this sophisticated merging method, we could get elongated reads from the forward and reverse reads. Homepage: http://best.snu.ac.kr/casper/ URL: http://best.snu.ac.kr/casper/</details>|
	|[catch2](https://github.com/catchorg/Catch2)|-|2.11.0, 3.2.1|<details><summary>Expand</summary>Description: A modern, C++-native, header-only, test framework for unit-tests, TDD and BDD - using C++11, C++14, C++17 and later (or C++03 on the Catch1.x branch) Homepage: https://github.com/catchorg/Catch2 URL: https://github.com/catchorg/Catch2</details>|
	|[ccfits](http://heasarc.gsfc.nasa.gov/fitsio/CCfits/)|vis|2.5|<details><summary>Expand</summary>Description: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. Homepage: http://heasarc.gsfc.nasa.gov/fitsio/CCfits/ URL: http://heasarc.gsfc.nasa.gov/fitsio/CCfits/</details>|
	|[ccsm](http://www.cesm.ucar.edu/models/ccsm4.0/)|geo|4_0_a02|<details><summary>Expand</summary>Description: The Community Climate System Model (CCSM) is a coupled climate model for simulating the earth's climate system. Composed of four separate models simultaneously simulating the earth's atmosphere, ocean, land surface and sea-ice, and one central coupler component, the CCSM allows researchers to conduct fundamental research into the earth's past, present and future climate states. Homepage: http://www.cesm.ucar.edu/models/ccsm4.0/ URL: http://www.cesm.ucar.edu/models/ccsm4.0/</details>|
	|[cd-hit](http://weizhongli-lab.org/cd-hit/)|bio|4.8.1|<details><summary>Expand</summary>Description: CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences. Homepage: http://weizhongli-lab.org/cd-hit/ URL: http://weizhongli-lab.org/cd-hit/</details>|
	|[cdbfasta](https://sourceforge.net/projects/cdbfasta)|-|0.99|<details><summary>Expand</summary>Description: Fasta file indexing and retrival tool Homepage: https://sourceforge.net/projects/cdbfasta URL: https://sourceforge.net/projects/cdbfasta</details>|
	|[cdo](https://code.zmaw.de/projects/cdo)|geo|1.9.8, 1.9.10, 2.0.4, 2.0.5, 2.2.2|<details><summary>Expand</summary>Description: CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data. Homepage: https://code.zmaw.de/projects/cdo URL: https://code.zmaw.de/projects/cdo</details>|
	|[cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)|bio|2.1.0|<details><summary>Expand</summary>Description: Cell Ranger is a set of analysis pipelines that process Chromium single cell 3-RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis. Cell Ranger 1.2 and later support libraries generated by the Chromium Single Cell 3i-v1 and v2 reagent kits, whereas Cell Ranger 1.1 and earlier do not support v2 libraries. Cell Ranger includes four main gene expression pipelines: - cellranger mkfastq wraps Illumina's bcl2fastq to correctly demultiplex Chromium-prepared sequencing samples and to convert barcode and read data to FASTQ files. - cellranger count takes FASTQ files from cellranger mkfastq and performs alignment, filtering, and UMI counting. - cellranger aggr aggregates outputs from multiple runs of cellranger count, normalizing those runs to the same sequencing depth and then recomputing the gene-barcode matrices and analysis on the combined data. - cellranger reanalyze takes gene-barcode matrices produced by cellranger count or cellranger aggr and reruns the dimensionality reduction, clustering, and gene expression algorithms using tunable parameter settings. These pipelines combine Chromium-specific algorithms with the widely used RNA-seq aligner STAR. Output is delivered in standard BAM, MEX, CSV, HDF5 and HTML formats that are augmented with cellular information. Homepage: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome URL: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</details>|
	|[cellsnp-lite](https://github.com/single-cell-genetics/cellsnp-lite)|-|1.2.2|<details><summary>Expand</summary>Description: cellsnp-lite was initially designed to pileup the expressed alleles in single-cell or bulk RNA-seq data, which can be directly used for donor deconvolution in multiplexed single-cell RNA-seq data, particularly with vireo, which assigns cells to donors and detects doublets, even without genotyping reference. Now besides RNA-seq data, cellsnp-lite could also be applied on DNA-seq and ATAC-seq data, either in bulk or single-cell. Homepage: https://github.com/single-cell-genetics/cellsnp-lite URL: https://github.com/single-cell-genetics/cellsnp-lite</details>|
	|[centrifuge](https://github.com/infphilo/centrifuge)|bio|1.0.4-beta, 1.0.4.2|<details><summary>Expand</summary>Description: [Centrifuge] is a novel microbial classification engine that enables rapid, accurate and sensitive labeling of reads and quantification of species on desktop computers. The system uses a novel indexing scheme based on the Burrows-Wheeler transform (BWT) and the Ferragina-Manzini (FM) index, optimized specifically for the metagenomic classification problem. Centrifuge requires a relatively small index (4.7 GB for all complete bacterial and viral genomes plus the human genome) and classifies sequences at very high speed, allowing it to process the millions of reads from a typical high-throughput DNA sequencing run within a few minutes. Together these advances enable timely and accurate analysis of large metagenomics data sets on conventional desktop computers Homepage: https://github.com/infphilo/centrifuge URL: https://github.com/infphilo/centrifuge</details>|
	|[cereal](https://uscilab.github.io/cereal/)|-|1.3.0, 1.3.2|<details><summary>Expand</summary>Description: cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or used standalone. Homepage: https://uscilab.github.io/cereal/ URL: https://uscilab.github.io/cereal/</details>|
	|[ceres-solver](https://ceres-solver.googlesource.com/ceres-solver/)|-|1.14.0|<details><summary>Expand</summary>Description: Ceres Solver is an open source C++ library for modeling and solving large, complicated optimization problems. It is a feature rich, mature and performant library which has been used in production at Google since 2010. Ceres Solver can solve two kinds of problems: - Non-linear Least Squares problems with bounds constraints. - General unconstrained optimization problems. Homepage: https://ceres-solver.googlesource.com/ceres-solver/ URL: https://ceres-solver.googlesource.com/ceres-solver/</details>|
	|[cesm](http://www.cesm.ucar.edu/models/cesm2/)|geo|2.1.3|<details><summary>Expand</summary>Description: The Community Earth System Model (CESM) is a fully-coupled, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states. Homepage: http://www.cesm.ucar.edu/models/cesm2/ URL: http://www.cesm.ucar.edu/models/cesm2/</details>|
	|[cfitsio](https://heasarc.gsfc.nasa.gov/fitsio/)|vis|3.41, 3.48, 3.49, 4.1.0, 4.3.0, 4.5.0|<details><summary>Expand</summary>Description: CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format. Homepage: https://heasarc.gsfc.nasa.gov/fitsio/ URL: https://heasarc.gsfc.nasa.gov/fitsio/</details>|
	|[cfour](http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download)|chem|2.1|<details><summary>Expand</summary>Description: Serial version of CFOUR (Coupled-Cluster techniques for Computational Chemistry) is a program package for performing high-level quantum chemical calculations on atoms and molecules. The major strength of the program suite is its rather sophisticated arsenal of high-level ab initio methods for the calculation of atomic and molecular properties. Virtually all approaches based on Muller-Plesset (MP) perturbation theory and the coupled-cluster approximation (CC) are available; most of these have complementary analytic derivative approaches within the package as well. CFOUR is not a commercial code. It is rather a program that is undergoing development; new techniques and improvements are constantly being made (mainly in the groups of John F. Stanton (until the end of 2016 at the University of Texas at Austin, USA, now at the Quantum Theory Project of the University of Florida at Gainesville, USA) and Juergen Gauss (Universitat Mainz, Germany)). For non-commercial purposes there is no charge to obtain CFOUR; one must simply sign a license agreement. http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download Homepage: http://www.cfour.de/ URL: http://www.cfour.de/</details>|
	|[cfour-mpi](http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download)|chem|2.1|<details><summary>Expand</summary>Description: CFOUR (Coupled-Cluster techniques for Computational Chemistry) is a program package for performing high-level quantum chemical calculations on atoms and molecules. The major strength of the program suite is its rather sophisticated arsenal of high-level ab initio methods for the calculation of atomic and molecular properties. Virtually all approaches based on Muller-Plesset (MP) perturbation theory and the coupled-cluster approximation (CC) are available; most of these have complementary analytic derivative approaches within the package as well. CFOUR is not a commercial code. It is rather a program that is undergoing development; new techniques and improvements are constantly being made (mainly in the groups of John F. Stanton (until the end of 2016 at the University of Texas at Austin, USA, now at the Quantum Theory Project of the University of Florida at Gainesville, USA) and Juergen Gauss (Universitat Mainz, Germany)). For non-commercial purposes there is no charge to obtain CFOUR; one must simply sign a license agreement. http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download Homepage: http://www.cfour.de/ URL: http://www.cfour.de/</details>|
	|[cgal](https://www.cgal.org/)|math|4.14.3, 5.2.4, 5.5.2|<details><summary>Expand</summary>Description: The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library. Homepage: https://www.cgal.org/ URL: https://www.cgal.org/</details>|
	|[cgns](https://cgns.github.io/)|phys|3.4.1, 4.1.0, 4.1.2, 4.5.0|<details><summary>Expand</summary>Description: The CGNS system is designed to facilitate the exchange of data between sites and applications, and to help stabilize the archiving of aerodynamic data. Homepage: https://cgns.github.io/ URL: https://cgns.github.io/</details>|
	|[chapel-multicore](https://chapel-lang.org)|-|2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ofi](https://chapel-lang.org)|-|1.31.0|<details><summary>Expand</summary>Description: Chapel is an emerging parallel programming language whose design and development is being led by Cray Inc. Chapel is designed to improve the productivity of high-end computer users while also serving as a portable parallel programming model that can be used on commodity clusters or desktop multicore systems. Chapel strives to vastly improve the programmability of large-scale parallel computers while matching or beating the performance and portability of current programming models like MPI. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ucx](https://chapel-lang.org)|-|2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ucx-cuda](https://chapel-lang.org)|-|2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[charm-gems](https://github.com/simnibs/charm-gems)|-|1.3.3|<details><summary>Expand</summary>Description: This repository contains the gems C++ code and python bindings used in Freesurfer's Sequence-Adaptive Multimodal SEGmentation (SAMSEG) and in SimNIBS 4.0 Complete Head Anatomy Reconstruction Method (CHARM) to create individualized head models for electric field simulations. Homepage: https://github.com/simnibs/charm-gems URL: https://github.com/simnibs/charm-gems</details>|
	|[chemps2](https://github.com/SebWouters/CheMPS2)|chem|1.8.9|<details><summary>Expand</summary>Description: CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry. Homepage: https://github.com/SebWouters/CheMPS2 URL: https://github.com/SebWouters/CheMPS2</details>|
	|[circos](http://www.circos.ca/)|vis|0.69-9|<details><summary>Expand</summary>Description: Circos is a software package for visualizing data and information. It visualizes data in a circular layout - this makes Circos ideal for exploring relationships between objects or positions. Homepage: http://www.circos.ca/ URL: http://www.circos.ca/</details>|
	|[clang](https://clang.llvm.org/)|tools|9.0.1, 11.0.0, 13.0.1, 17.0.6, 18.1.8|<details><summary>Expand</summary>Description: C, C++, Objective-C compiler, based on LLVM. Does not include C++ standard library -- use libstdc++ from GCC. Homepage: https://clang.llvm.org/ URL: https://clang.llvm.org/</details>|
	|[clhep](https://proj-clhep.web.cern.ch/proj-clhep/)|math|2.4.1.3, 2.4.4.0, 2.4.6.2, 2.4.7.1|<details><summary>Expand</summary>Description: The CLHEP project is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra. CLHEP is structured in a set of packages independent of any external package. Homepage: https://proj-clhep.web.cern.ch/proj-clhep/ URL: https://proj-clhep.web.cern.ch/proj-clhep/</details>|
	|[clustal-omega](http://www.clustal.org/omega/)|bio|1.2.4|<details><summary>Expand</summary>Description: Clustal Omega is a multiple sequence alignment program for proteins. It produces biologically meaningful multiple sequence alignments of divergent sequences. Evolutionary relationships can be seen via viewing Cladograms or Phylograms Homepage: http://www.clustal.org/omega/ URL: http://www.clustal.org/omega/</details>|
	|[cmake](https://www.cmake.org)|tools|3.18.4, 3.20.1, 3.21.4, 3.22.1, 3.23.1, 3.27.7, 3.31.0|<details><summary>Expand</summary>Description: CMake, the cross-platform, open-source build system. CMake is a family of tools designed to build, test and package software. Homepage: https://www.cmake.org URL: https://www.cmake.org</details>|
	|[cnvnator](https://github.com/abyzovlab/CNVnator)|bio|0.4.1|<details><summary>Expand</summary>Description: a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads Homepage: https://github.com/abyzovlab/CNVnator URL: https://github.com/abyzovlab/CNVnator</details>|
	|[code-server](https://github.com/cdr/code-server)|-|3.5.0, 3.12.0, 4.92.2|<details><summary>Expand</summary>Description: Run VS Code on any machine anywhere and access it in the browser. Homepage: https://github.com/cdr/code-server URL: https://github.com/cdr/code-server</details>|
	|[coinmp](https://projects.coin-or.org/CoinMP)|-|1.8.4|<details><summary>Expand</summary>Description: CoinMP is a C-API library that supports most of the functionality of CLP (Coin LP), CBC (Coin Branch-and-Cut), and CGL (Cut Generation Library) projects. Homepage: https://projects.coin-or.org/CoinMP URL: https://projects.coin-or.org/CoinMP</details>|
	|[colmap](https://github.com/colmap/colmap)|-|3.6|<details><summary>Expand</summary>Description: COLMAP is a general-purpose Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline with a graphical and command-line interface. It offers a wide range of features for reconstruction of ordered and unordered image collections. The software is licensed under the new BSD license. Homepage: https://github.com/colmap/colmap URL: https://github.com/colmap/colmap</details>|
	|[combblas](https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/)|-|1.6.2|<details><summary>Expand</summary>Description: The Combinatorial BLAS (CombBLAS) is an extensible distributed-memory parallel graph library offering a small but powerful set of linear algebra primitives specifically targeting graph analytics. Homepage: https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/ URL: https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/</details>|
	|[comsol](https://www.comsol.com)|phys|5.6, 6.0.0.405, 6.1.0.357, 6.2.0.415, 6.3|<details><summary>Expand</summary>Description: COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems. Homepage: https://www.comsol.com URL: https://www.comsol.com</details>|
	|[coordgenlibs](https://github.com/schrodinger/coordgenlibs)|chem|1.4.2|<details><summary>Expand</summary>Description: Schrodinger-developed 2D Coordinate Generation Homepage: https://github.com/schrodinger/coordgenlibs URL: https://github.com/schrodinger/coordgenlibs</details>|
	|[coretran](https://github.com/leonfoks/coretran#compile)|-|1.0.1|<details><summary>Expand</summary>Description: An easy to follow library to make Fortran easier in general with wrapped interfaces, sorting routines, kD-Trees, and other algorithms to handle scientific data and concepts. The library contains core fortran routines and object-oriented classes. Homepage: https://github.com/leonfoks/coretran#compile URL: https://github.com/leonfoks/coretran#compile</details>|
	|[corset](https://github.com/Oshlack/Corset/wiki)|bio|1.09|<details><summary>Expand</summary>Description: Corset is a command-line software program to go from a de novo transcriptome assembly to gene-level counts. Our software takes a set of reads that have been multi-mapped to the transcriptome (where multiple alignments per read were reported) and hierarchically clusters the transcripts based on the proportion of shared reads and expression patterns. It will report the clusters and gene-level counts for each sample, which are easily tested for differential expression with count based tools such as edgeR and DESeq. Homepage: https://github.com/Oshlack/Corset/wiki URL: https://github.com/Oshlack/Corset/wiki</details>|
	|[cp2k](https://www.cp2k.org/)|chem|7.1, 8.2, 9.1, 2023.1|<details><summary>Expand</summary>Description: CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials. Homepage: https://www.cp2k.org/ URL: https://www.cp2k.org/</details>|
	|[cpmd](http://cpmd.org)|chem|4.3|<details><summary>Expand</summary>Description: The CPMD code is a parallelized plane wave / pseudopotential implementation of Density Functional Theory, particularly designed for ab-initio molecular dynamics. Homepage: http://cpmd.org URL: http://cpmd.org</details>|
	|[cppzmq](https://github.com/zeromq/cppzmq)|-|4.7.1|<details><summary>Expand</summary>Description: cppzmq is a C++ binding for libzmq Homepage: https://github.com/zeromq/cppzmq URL: https://github.com/zeromq/cppzmq</details>|
	|[cpu_features](https://github.com/google/cpu_features)|-|0.6.0|<details><summary>Expand</summary>Description: A cross-platform C library to retrieve CPU features (such as available instructions) at runtime. Homepage: https://github.com/google/cpu_features URL: https://github.com/google/cpu_features</details>|
	|[cram](https://bitheap.org/cram)|-|0.7|<details><summary>Expand</summary>Description: Cram is a functional testing framework for command line applications. Homepage: https://bitheap.org/cram URL: https://bitheap.org/cram</details>|
	|[crest](https://xtb-docs.readthedocs.io/en/latest/crest.html)|chem|2.11, 2.12, 3.0.1, 3.0.2|<details><summary>Expand</summary>Description: CREST is an utility/driver program for the xtb program. Originally it was designed as conformer sampling program, hence the abbreviation ConformerRotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNnxTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles. Homepage: https://xtb-docs.readthedocs.io/en/latest/crest.html URL: https://xtb-docs.readthedocs.io/en/latest/crest.html</details>|
	|[cromwell](http://cromwell.readthedocs.io/)|tools|58|<details><summary>Expand</summary>Description: Scientific workflow engine designed for simplicity and scalability. Trivially transition between one off use cases to massive scale production environments http://cromwell.readthedocs.io/ Homepage: https://cromwell.readthedocs.io/ URL: https://cromwell.readthedocs.io/</details>|
	|[csblast](https://github.com/soedinglab/csblast/)|-|2.2.4|<details><summary>Expand</summary>Description: Context-specific extension of BLAST that significantly improves sensitivity and alignment quality. Homepage: https://github.com/soedinglab/csblast/ URL: https://github.com/soedinglab/csblast/</details>|
	|[cslib](https://cslib.sandia.gov/index.html)|-|20180813|<details><summary>Expand</summary>Description: The client/server model is a messaging paradigm, used ubiquitously by web-based applications, where one or more clients (e.g. your phone) communicate with a server (e.g. a website). In the context of the CSlib, it can used to couple two scientific applications (apps) together where one app acts as the client and the other as the server. The client sends messages to the server requesting it perform specific computations and return the results. Homepage: https://cslib.sandia.gov/index.html URL: https://cslib.sandia.gov/index.html</details>|
	|[cst](https://www.cst.com/)|phys|2020|<details><summary>Expand</summary>Description: CST offers accurate, efficient computational solutions for electromagnetic design and analysis. CST simulation is user-friendly and enables users to choose the most appropriate method for the design and optimization of devices operating in a wide range of frequencies. Please note that we installed only Graphic frontend and command line interface. You need to provide your own license server in order to be able to run CST. Homepage: https://www.cst.com/ URL: https://www.cst.com/</details>|
	|[csvtk](https://github.com/shenwei356/csvtk)|-|0.23.0|<details><summary>Expand</summary>Description: Similar to FASTA/Q format in field of Bioinformatics, CSV/TSV formats are basic and ubiquitous file formats in both Bioinformatics and data science. People usually use spreadsheet software like MS Excel to process table data. However this is all by clicking and typing, which is not automated and is time-consuming to repeat, especially when you want to apply similar operations with different datasets or purposes. You can also accomplish some CSV/TSV manipulations using shell commands, but more code is needed to handle the header line. Shell commands do not support selecting columns with column names either. csvtk is convenient for rapid data investigation and also easy to integrate into analysis pipelines. It could save you lots of time in (not) writing Python/R scripts. Homepage: https://github.com/shenwei356/csvtk URL: https://github.com/shenwei356/csvtk</details>|
	|[ctffind](https://grigoriefflab.umassmed.edu/ctffind4)|chem|4.1.14|<details><summary>Expand</summary>Description: Program for finding CTFs of electron micrographs. Homepage: https://grigoriefflab.umassmed.edu/ctffind4 URL: https://grigoriefflab.umassmed.edu/ctffind4</details>|
	|[cuba](https://feynarts.de/cuba/)|-|4.2.2|<details><summary>Expand</summary>Description: Cuba a library for multidimensional numerical integration Homepage: https://feynarts.de/cuba/ URL: https://feynarts.de/cuba/</details>|
	|[cubegui](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.4|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube graphical report explorer. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubelib](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.4|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube general purpose C++ library component and command-line tools. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubewriter](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.3|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube high-performance C writer library component. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubic_interpolation](https://github.com/tudo-astroparticlephysics/cubic_interpolation)|-|0.1.5|<details><summary>Expand</summary>Description: A leightweight interpolation library based on boost and eigen. It provides the utilities to handle tables which are runtime intensive to build and reduces interpolation failures to axis transformation. Homepage: https://github.com/tudo-astroparticlephysics/cubic_interpolation URL: https://github.com/tudo-astroparticlephysics/cubic_interpolation</details>|
	|[cuda](https://developer.nvidia.com/cuda-toolkit)|tools|10.1, 10.2|<details><summary>Expand</summary>Description: CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. Homepage: https://developer.nvidia.com/cuda-toolkit URL: https://developer.nvidia.com/cuda-toolkit</details>|
	|[cudnn](https://developer.nvidia.com/cudnn)|math|8.0.3, 8.2.0, 8.6.0.163, 8.7.0.84, 8.9.5.29, 9.2.1.18, 9.5.1.17|<details><summary>Expand</summary>Description: The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. Homepage: https://developer.nvidia.com/cudnn URL: https://developer.nvidia.com/cudnn</details>|
	|[cufflinks](http://cole-trapnell-lab.github.io/cufflinks/)|bio|2.2.1|<details><summary>Expand</summary>Description: Transcript assembly, differential expression, and differential regulation for RNA-Seq Homepage: http://cole-trapnell-lab.github.io/cufflinks/ URL: http://cole-trapnell-lab.github.io/cufflinks/</details>|
	|[cuquantum](https://developer.nvidia.com/cuquantum-sdk)|-|23.06.1.8, 24.03.0.4|<details><summary>Expand</summary>Description: NVIDIA cuQuantum SDK is a high-performance library for quantum information science and beyond Homepage: https://developer.nvidia.com/cuquantum-sdk URL: https://developer.nvidia.com/cuquantum-sdk</details>|
	|[cusparselt](https://docs.nvidia.com/cuda/cusparselt/index.html)|-|0.4.0.7, 0.5.0.1, 0.6.1.0|<details><summary>Expand</summary>Description: NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix Homepage: https://docs.nvidia.com/cuda/cusparselt/index.html URL: https://docs.nvidia.com/cuda/cusparselt/index.html</details>|
	|[custom-ctypes](https://github.com/ComputeCanada/custom_ctypes/)|-|1.1, 1.2|<details><summary>Expand</summary>Description: This repository is to try to fix discovery of libraries with Python's ctypes module. It changes the behavior of find_library to return absolute paths to shared objects rather than just the names. Homepage: https://github.com/ComputeCanada/custom_ctypes/ URL: https://github.com/ComputeCanada/custom_ctypes/</details>|
	|[cutensor](https://developer.nvidia.com/cutensor)|-|1.5.0.3, 1.7.0.1, 2.0.0.7, 2.0.1.2|<details><summary>Expand</summary>Description: cuTENSOR is a high-performance CUDA library for tensor primitives. Homepage: https://developer.nvidia.com/cutensor URL: https://developer.nvidia.com/cutensor</details>|
	|[cvit](https://sourceforge.net/projects/cvit/)|-|1.2.1|<details><summary>Expand</summary>Description: A genomic linkage feature visualization tools based on Perl. Homepage: https://sourceforge.net/projects/cvit/ URL: https://sourceforge.net/projects/cvit/</details>|
	|[dakota](https://dakota.sandia.gov/)|tools|6.13|<details><summary>Expand</summary>Description: The Dakota project delivers both state-of-the-art research and robust, usable software for optimization and UQ. Broadly, the Dakota software's advanced parametric analyses enable design exploration, model calibration, risk analysis, and quantification of margins and uncertainty with computational models. The Dakota toolkit provides a flexible, extensible interface between such simulation codes and its iterative systems analysis methods, which include: - optimization with gradient and nongradient-based methods; - uncertainty quantification with sampling, reliability, stochastic expansion, and epistemic methods; - parameter estimation using nonlinear least squares (deterministic) or Bayesian inference (stochastic); and - sensitivity/variance analysis with design of experiments and parameter study methods. These capabilities may be used on their own or as components within advanced strategies such as hybrid optimization, surrogate-based optimization, mixed integer nonlinear programming, or optimization under uncertainty. Homepage: https://dakota.sandia.gov/ URL: https://dakota.sandia.gov/</details>|
	|[dalton](https://www.daltonprogram.org/www/citation.html.)|chem|2020.0, 2020.1|<details><summary>Expand</summary>Description: The Dalton code is a powerful tool for a wide range of molecular properties at different levels of theory. Any published work arising from use of one of the Dalton2016 programs must acknowledge that by a proper reference, https://www.daltonprogram.org/www/citation.html. Homepage: https://daltonprogram.org/ URL: https://daltonprogram.org/</details>|
	|[das_tool](https://github.com/cmks/DAS_Tool)|-|1.1.6|<details><summary>Expand</summary>Description: DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly. Homepage: https://github.com/cmks/DAS_Tool URL: https://github.com/cmks/DAS_Tool</details>|
	|[db](https://www.oracle.com/technetwork/products/berkeleydb)|-|18.1.32|<details><summary>Expand</summary>Description: Berkeley DB enables the development of custom data management solutions, without the overhead traditionally associated with such custom projects. Homepage: https://www.oracle.com/technetwork/products/berkeleydb URL: https://www.oracle.com/technetwork/products/berkeleydb</details>|
	|[dcm2niix](https://github.com/rordenlab/dcm2niix)|bio|1.0.20200331, 1.0.20230411|<details><summary>Expand</summary>Description: dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format. Homepage: https://github.com/rordenlab/dcm2niix URL: https://github.com/rordenlab/dcm2niix</details>|
	|[dcmtk](https://dicom.offis.de/dcmtk)|-|3.6.7|<details><summary>Expand</summary>Description: DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers. Homepage: https://dicom.offis.de/dcmtk URL: https://dicom.offis.de/dcmtk</details>|
	|[ddt-cpu](https://www.linaroforge.com)|tools|20.2, 22.0.1, 23.1.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining MAP and DDT. Homepage: https://www.linaroforge.com URL: https://www.linaroforge.com</details>|
	|[ddt-gpu](https://www.linaroforge.com)|tools|20.2, 22.0.1, 23.1.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining MAP and DDT, with support for CUDA. Homepage: https://www.linaroforge.com URL: https://www.linaroforge.com</details>|
	|[dealii](http://www.dealii.org/)|math|9.2.0, 9.3.1|<details><summary>Expand</summary>Description: A C++ software library supporting the creation of finite element codes and an open community of users and developers. Homepage: http://www.dealii.org/ URL: http://www.dealii.org/</details>|
	|[deepvariant](https://github.com/google/deepvariant)|-|1.8.0|<details><summary>Expand</summary>Description: DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file. DeepVariant supports germline variant-calling in diploid organisms. Homepage: https://github.com/google/deepvariant URL: https://github.com/google/deepvariant</details>|
	|[deepvariant-gpu](https://github.com/google/deepvariant)|-|1.8.0|<details><summary>Expand</summary>Description: DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file. DeepVariant supports germline variant-calling in diploid organisms. Homepage: https://github.com/google/deepvariant URL: https://github.com/google/deepvariant</details>|
	|[delft3d](https://svn.oss.deltares.nl)|chem|62441|<details><summary>Expand</summary>Description: Delft3D is a powerful modelling suite focusing primarily on coastal, estuarine, river, rural and urban environments Delft3D FM Suite can simulate storm surges, hurricanes, tsunamis, detailed flows and water levels, waves, sediment transport and morphology, water quality and ecology, and is capable of handling the interactions between these processes. Homepage: https://svn.oss.deltares.nl URL: https://svn.oss.deltares.nl</details>|
	|[delly](https://github.com/dellytools/delly/)|-|0.8.5, 1.1.6, 1.1.8|<details><summary>Expand</summary>Description: Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome. Homepage: https://github.com/dellytools/delly/ URL: https://github.com/dellytools/delly/</details>|
	|[detonate](http://deweylab.biostat.wisc.edu/detonate/)|bio|1.11|<details><summary>Expand</summary>Description: DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation) consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds of genomic sequences. Homepage: http://deweylab.biostat.wisc.edu/detonate/ URL: http://deweylab.biostat.wisc.edu/detonate/</details>|
	|[dftbplus](https://www.dftb-plus.info)|chem|21.1, 21.2, 24.1|<details><summary>Expand</summary>Description: DFTB+ is a fast and efficient versatile quantum mechanical simulation package. It is based on the Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been developed for the DFTB framework so far. Using DFTB+ you can carry out quantum mechanical simulations like with ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of magnitude in speed. Homepage: https://www.dftb-plus.info URL: https://www.dftb-plus.info</details>|
	|[dftd3-lib](https://github.com/dftbplus/dftd3-lib)|chem|0.10|<details><summary>Expand</summary>Description: This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers. The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been converted to free format and encapsulated into modules. Homepage: https://github.com/dftbplus/dftd3-lib URL: https://github.com/dftbplus/dftd3-lib</details>|
	|[dftd4](https://github.com/dftd4/dftd4/)|chem|3.3.0, 3.6.0, 3.7.0|<details><summary>Expand</summary>Description: DFTD4 van-der-Waals functional is generally Applicable Atomic-Charge Dependent London Dispersion Correction. Homepage: https://github.com/dftd4/dftd4/ URL: https://github.com/dftd4/dftd4/</details>|
	|[diamond](https://github.com/bbuchfink/diamond)|bio|0.9.36, 2.0.4, 2.0.9, 2.0.13, 2.0.15, 2.1.6, 2.1.7, 2.1.8|<details><summary>Expand</summary>Description: Accelerated BLAST compatible local sequence aligner Homepage: https://github.com/bbuchfink/diamond URL: https://github.com/bbuchfink/diamond</details>|
	|[dirac](http://www.diracprogram.org)|-|21.1, 23.0|<details><summary>Expand</summary>Description: DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations Homepage: http://www.diracprogram.org URL: http://www.diracprogram.org</details>|
	|[dl_monte](https://gitlab.com/dl_monte)|-|2.07|<details><summary>Expand</summary>Description: General-purpose Monte Carlo molecular simulation Homepage: https://gitlab.com/dl_monte URL: https://gitlab.com/dl_monte</details>|
	|[dl_poly4](http://www.scd.stfc.ac.uk/SCD/44516.aspx)|chem|4.10.0, 5.1.0|<details><summary>Expand</summary>Description: DL_POLY is a general purpose classical molecular dynamics (MD) simulation software developed at Daresbury Laboratory by I.T. Todorov and W. Smith. Homepage: http://www.scd.stfc.ac.uk/SCD/44516.aspx URL: http://www.scd.stfc.ac.uk/SCD/44516.aspx</details>|
	|[dmalloc](http://dmalloc.com/)|tools|5.5.2|<details><summary>Expand</summary>Description: The debug memory allocation or dmalloc library has been designed as a drop in replacement for the system s malloc, realloc, calloc, free and other memory management routines while providing powerful debugging facilities configurable at runtime. These facilities include such things as memory-leak tracking, fence-post write detection, file/line number reporting, and general logging of statistics. The library is reasonably portable having been run successfully on at least the following operating systems: AIX, BSD/OS, DG/UX, Free/Net/OpenBSD, GNU/Hurd, HPUX, Irix, Linux, MS-DOG, NeXT, OSF, SCO, Solaris, SunOS, Ultrix, Unixware, Windoze, and even Unicos on a Cray T3E. It also provides support for the debugging of threaded programs. The package includes the library, configuration scripts, debug utility application, test program, and documentation. Homepage: http://dmalloc.com/ URL: http://dmalloc.com/</details>|
	|[dnmtools](https://github.com/smithlabcode/dnmtools)|-|1.4.4|<details><summary>Expand</summary>Description: DNMTools is a set of tools for analyzing DNA methylation data from high-throughput sequencing experiments, especially whole genome bisulfite sequencing (WGBS), but also reduced representation bisulfite sequencing (RRBS). These tools focus on overcoming the computing challenges imposed by the scale of genome-wide DNA methylation data, which is usually the early parts of data analysis. Homepage: https://github.com/smithlabcode/dnmtools URL: https://github.com/smithlabcode/dnmtools</details>|
	|[dorado](https://github.com/nanoporetech/dorado)|-|0.2.1, 0.2.2, 0.3.0, 0.4.1, 0.4.3, 0.5.3, 0.6.1, 0.7.0, 0.7.2, 0.8.0, 0.8.3, 0.9.5|<details><summary>Expand</summary>Description: Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads. Homepage: https://github.com/nanoporetech/dorado URL: https://github.com/nanoporetech/dorado</details>|
	|[dotnet-core](https://www.microsoft.com/net/)|-|3.1.8, 5.0.12, 6.0.0, 8.0.407, 9.0.202|<details><summary>Expand</summary>Description: .NET is a free, cross-platform, open source developer platform for building many different types of applications. With .NET, you can use multiple languages, editors, and libraries to build for web, mobile, desktop, gaming, and IoT. Contains the SDK and the Runtime. Homepage: https://www.microsoft.com/net/ URL: https://www.microsoft.com/net/</details>|
	|[double-conversion](https://github.com/google/double-conversion)|-|3.1.5, 3.2.1|<details><summary>Expand</summary>Description: Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles. Homepage: https://github.com/google/double-conversion URL: https://github.com/google/double-conversion</details>|
	|[dpc++](https://github.com/intel/llvm)|-|2022-06, 2022-12|<details><summary>Expand</summary>Description: DPC++ is a LLVM-based compiler project that implements compiler and runtime support for the SYCL language. Homepage: https://github.com/intel/llvm URL: https://github.com/intel/llvm</details>|
	|[dragmap](https://github.com/Illumina/DRAGMAP)|-|1.3.0|<details><summary>Expand</summary>Description: Dragmap is the Dragen mapper/aligner Open Source Software. Homepage: https://github.com/Illumina/DRAGMAP URL: https://github.com/Illumina/DRAGMAP</details>|
	|[dssp](https://swift.cmbi.umcn.nl/gv/dssp/)|chem|2.3.0, 3.1.4|<details><summary>Expand</summary>Description: The DSSP algorithm is the standard method for assigning secondary structure to the amino acids of a protein, given the atomic-resolution coordinates of the protein. If you use DSSP, please quote these two articles: 1) A series of PDB related databases for everyday needs. Wouter G Touw, Coos Baakman, Jon Black, Tim AH te Beek, E Krieger, Robbie P Joosten, Gert Vriend. Nucleic Acids Research 2015 January; 43(Database issue): D364-D368. 2) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. Kabsch W, Sander C, Biopolymers. 1983 22 2577-2637. PMID: 6667333; UI: 84128824. Homepage: https://swift.cmbi.umcn.nl/gv/dssp/ URL: https://swift.cmbi.umcn.nl/gv/dssp/</details>|
	|[ecbuild](https://ecbuild.readthedocs.io/)|-|3.8.0|<details><summary>Expand</summary>Description: A CMake-based build system, consisting of a collection of CMake macros and functions that ease the managing of software build systems Homepage: https://ecbuild.readthedocs.io/ URL: https://ecbuild.readthedocs.io/</details>|
	|[eccodes](https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home)|geo|2.15.0, 2.19.0, 2.22.1, 2.25.0, 2.31.0|<details><summary>Expand</summary>Description: ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding). Homepage: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home URL: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</details>|
	|[edirect](https://www.ncbi.nlm.nih.gov/books/NBK25501/)|-|20.9.20231210|<details><summary>Expand</summary>Description: Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases from a Unix terminal window. Search terms are entered as command-line arguments. Individual operations are connected with Unix pipes to construct multi-step queries. Selected records can then be retrieved in a variety of formats. Homepage: https://www.ncbi.nlm.nih.gov/books/NBK25501/ URL: https://www.ncbi.nlm.nih.gov/books/NBK25501/</details>|
	|[eigen](https://eigen.tuxfamily.org)|math|3.3.7, 3.4.0|<details><summary>Expand</summary>Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms. Homepage: https://eigen.tuxfamily.org URL: https://eigen.tuxfamily.org</details>|
	|[eigensoft](http://www.hsph.harvard.edu/alkes-price/software/)|bio|7.2.1|<details><summary>Expand</summary>Description: The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al. 2006) and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal components analysis to explicitly model ancestry differences between cases and controls along continuous axes of variation; the resulting correction is specific to a candidate marker's variation in frequency across ancestral populations, minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT package has a built-in plotting script and supports multiple file formats and quantitative phenotypes. Homepage: http://www.hsph.harvard.edu/alkes-price/software/ URL: http://www.hsph.harvard.edu/alkes-price/software/</details>|
	|[elastix](http://elastix.isi.uu.nl/)|-|5.0.1|<details><summary>Expand</summary>Description: elastix: a toolbox for rigid and nonrigid registration of images. Homepage: http://elastix.isi.uu.nl/ URL: http://elastix.isi.uu.nl/</details>|
	|[elixir](https://elixir-lang.org/)|tools|1.13|<details><summary>Expand</summary>Description: Elixir is a dynamic, functional language designed for building scalable and maintainable applications. Homepage: https://elixir-lang.org/ URL: https://elixir-lang.org/</details>|
	|[elmerfem](https://www.csc.fi/web/elmer)|-|scc20, 8.4, 9.0|<details><summary>Expand</summary>Description: Elmer is an open source multiphysical simulation software mainly developed by CSC - IT Center for Science (CSC). Elmer development was started 1995 in collaboration with Finnish Universities, research institutes and industry. After it's open source publication in 2005, the use and development of Elmer has become international. Elmer includes physical models of fluid dynamics, structural mechanics, electromagnetics, heat transfer and acoustics, for example. These are described by partial differential equations which Elmer solves by the Finite Element Method (FEM). Homepage: https://www.csc.fi/web/elmer URL: https://www.csc.fi/web/elmer</details>|
	|[elpa](https://elpa.mpcdf.mpg.de/)|math|2020.05.001, 2023.05.001, 2024.05.001|<details><summary>Expand</summary>Description: Eigenvalue SoLvers for Petaflop-Applications. Homepage: https://elpa.mpcdf.mpg.de/ URL: https://elpa.mpcdf.mpg.de/</details>|
	|[emboss](https://emboss.sourceforge.net/)|bio|6.6.0|<details><summary>Expand</summary>Description: EMBOSS is 'The European Molecular Biology Open Software Suite' . EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology (e.g. EMBnet) user community. Homepage: https://emboss.sourceforge.net/ URL: https://emboss.sourceforge.net/</details>|
	|[embree](https://github.com/embree/embree)|-|2.17.7, 3.11.0, 3.13.5, 4.3.0|<details><summary>Expand</summary>Description: Intel Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Intel Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Intel Embree supports runtime code selection to choose the traversal and build algorithms that best matches the instruction set of your CPU. Homepage: https://github.com/embree/embree URL: https://github.com/embree/embree</details>|
	|[energyplus](https://energyplus.net/downloads)|-|9.3.0, 23.2.0|<details><summary>Expand</summary>Description: EnergyPlus is a whole building energy simulation program that engineers, architects, and researchers use to model both energy consumptionfor heating, cooling, ventilation, lighting and plug and process loadsand water use in buildings. Homepage: https://energyplus.net/downloads URL: https://energyplus.net/downloads</details>|
	|[erlangotp](http://www.erlang.org/)|tools|23.3, 24.2|<details><summary>Expand</summary>Description: Erlang is a general-purpose concurrent, garbage-collected programming language and runtime system. Homepage: http://www.erlang.org/ URL: http://www.erlang.org/</details>|
	|[esmf](https://www.earthsystemcog.org/projects/esmf/)|geo|8.0.1, 8.2.0, 8.4.0, 8.6.0, 8.7.0, 8.8.0|<details><summary>Expand</summary>Description: The Earth System Modeling Framework (ESMF) is a suite of software tools for developing high-performance, multi-component Earth science modeling applications. Homepage: https://www.earthsystemcog.org/projects/esmf/ URL: https://www.earthsystemcog.org/projects/esmf/</details>|
	|[espeak-ng](https://github.com/espeak-ng/espeak-ng)|-|1.51|<details><summary>Expand</summary>Description: The eSpeak NG is a compact open source software text-to-speech synthesizer for Linux, Windows, Android and other operating systems. It supports more than 100 languages and accents. It is based on the eSpeak engine created by Jonathan Duddington. Homepage: https://github.com/espeak-ng/espeak-ng URL: https://github.com/espeak-ng/espeak-ng</details>|
	|[etsf_io](http://www.etsf.eu/resources/software/libraries_and_tools)|io|1.0.4|<details><summary>Expand</summary>Description: A library of F90 routines to read/write the ETSF file format has been written. It is called ETSF_IO and available under LGPL. Homepage: http://www.etsf.eu/resources/software/libraries_and_tools URL: http://www.etsf.eu/resources/software/libraries_and_tools</details>|
	|[etsf_io-mpi](http://www.etsf.eu/resources/software/libraries_and_tools)|-|1.0.4|<details><summary>Expand</summary>Description: A library of F90 routines to read/write the ETSF file format has been written. It is called ETSF_IO and available under LGPL. Homepage: http://www.etsf.eu/resources/software/libraries_and_tools URL: http://www.etsf.eu/resources/software/libraries_and_tools</details>|
	|[exonerate](https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate)|bio|2.4.0|<details><summary>Expand</summary>Description: Exonerate is a generic tool for pairwise sequence comparison. It allows you to align sequences using a many alignment models, using either exhaustive dynamic programming, or a variety of heuristics. Homepage: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate URL: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</details>|
	|[expat](https://libexpat.github.io)|tools|2.2.9, 2.2.10, 2.4.1|<details><summary>Expand</summary>Description: Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags) Homepage: https://libexpat.github.io URL: https://libexpat.github.io</details>|
	|[faiss](https://github.com/facebookresearch/faiss)|-|1.6.5, 1.7.1, 1.7.3, 1.7.4, 1.8.0|<details><summary>Expand</summary>Description: Faiss is a library for efficient similarity search and clustering of dense vectors. It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM. It also contains supporting code for evaluation and parameter tuning. Faiss is written in C++ with complete wrappers for Python/numpy. Some of the most useful algorithms are implemented on the GPU. Homepage: https://github.com/facebookresearch/faiss URL: https://github.com/facebookresearch/faiss</details>|
	|[fann](http://leenissen.dk)|-|2.2.0|<details><summary>Expand</summary>Description: Fast Artificial Neural Network Library is a free open source neural network library, which implements multilayer artificial neural networks in C with support for both fully connected and sparsely connected networks. Homepage: http://leenissen.dk URL: http://leenissen.dk</details>|
	|[fasta](http://fasta.bioch.virginia.edu)|bio|36.3.8h|<details><summary>Expand</summary>Description: The FASTA programs find regions of local or global (new) similarity between protein or DNA sequences, either by searching Protein or DNA databases, or by identifying local duplications within a sequence. Homepage: http://fasta.bioch.virginia.edu URL: http://fasta.bioch.virginia.edu</details>|
	|[fastahack](https://github.com/ekg/fastahack)|-|1.0.0|<details><summary>Expand</summary>Description: Utilities for indexing and sequence extraction from FASTA files. Homepage: https://github.com/ekg/fastahack URL: https://github.com/ekg/fastahack</details>|
	|[fastani](https://github.com/ParBLiSS/FastANI)|-|1.32|<details><summary>Expand</summary>Description: FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies. Homepage: https://github.com/ParBLiSS/FastANI URL: https://github.com/ParBLiSS/FastANI</details>|
	|[fastme](http://www.atgc-montpellier.fr/fastme/)|bio|2.1.6.2|<details><summary>Expand</summary>Description: FastME: a comprehensive, accurate and fast distance-based phylogeny inference program. Homepage: http://www.atgc-montpellier.fr/fastme/ URL: http://www.atgc-montpellier.fr/fastme/</details>|
	|[fastp](https://github.com/OpenGene/fastp)|bio|0.20.1, 0.23.1, 0.23.4, 0.24.0|<details><summary>Expand</summary>Description: A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance. Homepage: https://github.com/OpenGene/fastp URL: https://github.com/OpenGene/fastp</details>|
	|[fastq-join](https://github.com/brwnj/fastq-join)|-|1.3.1|<details><summary>Expand</summary>Description: fastq-join: Joins two paired-end reads on the overlapping ends. Homepage: https://github.com/brwnj/fastq-join URL: https://github.com/brwnj/fastq-join</details>|
	|[fastq-multx](https://github.com/brwnj/fastq-multx)|-|1.4.0|<details><summary>Expand</summary>Description: fastq-multx Homepage: https://github.com/brwnj/fastq-multx URL: https://github.com/brwnj/fastq-multx</details>|
	|[fastq-tools](https://homes.cs.washington.edu/~dcjones/fastq-tools/)|-|0.8|<details><summary>Expand</summary>Description: This package provides a number of small and efficient programs to perform common tasks with high throughput sequencing data in the FASTQ format. All of the programs work with typical FASTQ files as well as gzipped FASTQ files. Homepage: https://homes.cs.washington.edu/~dcjones/fastq-tools/ URL: https://homes.cs.washington.edu/~dcjones/fastq-tools/</details>|
	|[fastq_screen](http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/)|bio|0.11.4|<details><summary>Expand</summary>Description: FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect. Homepage: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/ URL: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/</details>|
	|[fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)|bio|0.11.9, 0.12.0, 0.12.1|<details><summary>Expand</summary>Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline. Homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</details>|
	|[fastsimcoal2](http://cmpg.unibe.ch/software/fastsimcoal2/)|bio|2.6.0.3, 2.7.0.9|<details><summary>Expand</summary>Description: While preserving all the simulation flexibility of simcoal2, fastsimcoal is now implemented under a faster continous-time sequential Markovian coalescent approximation, allowing it to efficiently generate genetic diversity for different types of markers along large genomic regions, for both present or ancient samples. It includes a parameter sampler allowing its integration into Bayesian or likelihood parameter estimation procedure. fastsimcoal can handle very complex evolutionary scenarios including an arbitrary migration matrix between samples, historical events allowing for population resize, population fusion and fission, admixture events, changes in migration matrix, or changes in population growth rates. The time of sampling can be specified independently for each sample, allowing for serial sampling in the same or in different populations. Homepage: http://cmpg.unibe.ch/software/fastsimcoal2/ URL: http://cmpg.unibe.ch/software/fastsimcoal2/</details>|
	|[fastspar](https://github.com/scwatts/fastspar)|-|1.0.0|<details><summary>Expand</summary>Description: FastSpar is a C++ implementation of the SparCC algorithm which is up to several thousand times faster than the original Python2 release and uses much less memory. The FastSpar implementation provides threading support and a p-value estimator which accounts for the possibility of repetitious data permutations (see this paper for further details). Homepage: https://github.com/scwatts/fastspar URL: https://github.com/scwatts/fastspar</details>|
	|[fasttree](http://www.microbesonline.org/fasttree/)|bio|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fasttree-double](http://www.microbesonline.org/fasttree/)|-|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fastx-toolkit](http://hannonlab.cshl.edu/fastx_toolkit/)|bio|0.0.14|<details><summary>Expand</summary>Description: The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing. Homepage: http://hannonlab.cshl.edu/fastx_toolkit/ URL: http://hannonlab.cshl.edu/fastx_toolkit/</details>|
	|[fcl](https://github.com/flexible-collision-library/fcl)|-|0.7.0|<details><summary>Expand</summary>Description: FCL is a library for performing three types of proximity queries on a pair of geometric models composed of triangles. Homepage: https://github.com/flexible-collision-library/fcl URL: https://github.com/flexible-collision-library/fcl</details>|
	|[fds](https://pages.nist.gov/fds-smv/)|-|6.7.5, 6.7.6, 6.7.7, 6.7.8, 6.7.9, 6.8.0|<details><summary>Expand</summary>Description: Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows, with an emphasis on smoke and heat transport from fires. Homepage: https://pages.nist.gov/fds-smv/ URL: https://pages.nist.gov/fds-smv/</details>|
	|[febio](https://github.com/febiosoftware/FEBio)|-|4.7|<details><summary>Expand</summary>Description: FEBio is a nonlinear finite element (FE) solver that is specifically designed for biomechanical applications. It offers modeling scenarios, constitutive models and boundary conditions that are relevant to many research areas in biomechanics. All features can be used together seamlessly, giving the user a powerful tool for solving 3D problems in computational biomechanics. The software is open-source, and pre-compiled executables for Windows, OS-X and Linux platforms are available. Homepage: https://github.com/febiosoftware/FEBio URL: https://github.com/febiosoftware/FEBio</details>|
	|[fermi-lite](https://github.com/lh3/fermi-lite)|-|20190320|<details><summary>Expand</summary>Description: Standalone C library for assembling Illumina short reads in small regions. Homepage: https://github.com/lh3/fermi-lite URL: https://github.com/lh3/fermi-lite</details>|
	|[ferret](http://ferret.pmel.noaa.gov/)|vis|7.3, 7.6.0|<details><summary>Expand</summary>Description: Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets. Ferret 7.6.0 is the final available version of Classic Ferret. The developers recommend using PyFerret instead. Homepage: http://ferret.pmel.noaa.gov/ URL: http://ferret.pmel.noaa.gov/</details>|
	|[ffmpeg](https://www.ffmpeg.org/)|-|4.2.2, 4.3.2|<details><summary>Expand</summary>Description: A complete, cross-platform solution to record, convert and stream audio and video. Homepage: https://www.ffmpeg.org/ URL: https://www.ffmpeg.org/</details>|
	|[fftw](https://www.fftw.org)|math|3.3.8, 3.3.10|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://www.fftw.org URL: https://www.fftw.org</details>|
	|[fftw-mpi](https://www.fftw.org)|math|3.3.8, 3.3.9, 3.3.10|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://www.fftw.org URL: https://www.fftw.org</details>|
	|[filevercmp](https://github.com/ekg/filevercmp)|-|20191210|<details><summary>Expand</summary>Description: filevercmp function as in sort --version-sort. Homepage: https://github.com/ekg/filevercmp URL: https://github.com/ekg/filevercmp</details>|
	|[filtlong](https://github.com/rrwick/Filtlong)|bio|0.2.0, 0.2.1|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/rrwick/Filtlong URL: https://github.com/rrwick/Filtlong</details>|
	|[flash](https://ccb.jhu.edu/software/FLASH/)|-|1.2.11|<details><summary>Expand</summary>Description: FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. Homepage: https://ccb.jhu.edu/software/FLASH/ URL: https://ccb.jhu.edu/software/FLASH/</details>|
	|[flashpca](https://github.com/gabraham/flashpca/)|-|2.0|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/gabraham/flashpca/ URL: https://github.com/gabraham/flashpca/</details>|
	|[flatbuffers](https://github.com/google/flatbuffers/)|-|22.9.24|<details><summary>Expand</summary>Description: FlatBuffers: Memory Efficient Serialization Library Homepage: https://github.com/google/flatbuffers/ URL: https://github.com/google/flatbuffers/</details>|
	|[flexiblas](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)|-|3.0.4, 3.2.0, 3.3.1, 3.4.4|<details><summary>Expand</summary>Description: FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it. Homepage: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release URL: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release</details>|
	|[flint](http://www.flintlib.org/)|math|2.7.1, 2.9.0, 3.0.0|<details><summary>Expand</summary>Description: FLINT (Fast Library for Number Theory) is a C library in support of computations in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs, factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides various low-level routines for fast arithmetic. FLINT is extensively documented and tested. Homepage: http://www.flintlib.org/ URL: http://www.flintlib.org/</details>|
	|[fmlrc2](https://github.com/HudsonAlpha/fmlrc2)|-|0.1.8|<details><summary>Expand</summary>Description: FMLRC v2, based on the same methodology used by the original FMLRC. In benchmarks, the results between FMLRC v1 and v2 are nearly identical, but tests have shown that v2 uses approximately 50% of the run and CPU time compared to v1. Homepage: https://github.com/HudsonAlpha/fmlrc2 URL: https://github.com/HudsonAlpha/fmlrc2</details>|
	|[fmm3d](https://fmm3d.readthedocs.io)|-|1.0.1, 1.0.4|<details><summary>Expand</summary>Description: Flatiron Institute Fast Multipole Libraries: a set of libraries to compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine. Homepage: https://fmm3d.readthedocs.io URL: https://fmm3d.readthedocs.io</details>|
	|[fmriprep](https://fmriprep.org/)|-|23.0.2, 23.1.3|<details><summary>Expand</summary>Description: fMRIPrep is a NiPreps (NeuroImaging PREProcessing toolS) application (www.nipreps.org) for the preprocessing of task-based and resting-state functional MRI (fMRI). Homepage: https://fmriprep.org/ URL: https://fmriprep.org/</details>|
	|[fmt](http://fmtlib.net/)|-|5.3.0, 6.2.1, 7.0.3, 9.1.0|<details><summary>Expand</summary>Description: fmt (formerly cppformat) is an open-source formatting library. Homepage: http://fmtlib.net/ URL: http://fmtlib.net/</details>|
	|[fpc](https://www.freepascal.org)|tools|3.2.2|<details><summary>Expand</summary>Description: Free Pascal is a mature, versatile, open source Pascal compiler. Homepage: https://www.freepascal.org URL: https://www.freepascal.org</details>|
	|[fplll](https://github.com/fplll/fplll)|-|5.4.5|<details><summary>Expand</summary>Description: fplll contains implementations of several lattice algorithms. The implementation relies on floating-point orthogonalization, and the 1982 paper from Lenstra, Lenstra Jr. and Lovasz (LLL) is central to the code, hence the name. Homepage: https://github.com/fplll/fplll URL: https://github.com/fplll/fplll</details>|
	|[fraggenescan](http://omics.informatics.indiana.edu/FragGeneScan/)|bio|1.30, 1.31|<details><summary>Expand</summary>Description: FragGeneScan is an application for finding (fragmented) genes in short reads. Homepage: http://omics.informatics.indiana.edu/FragGeneScan/ URL: http://omics.informatics.indiana.edu/FragGeneScan/</details>|
	|[freebayes](https://github.com/freebayes/freebayes)|bio|1.2.0, 1.3.6, 1.3.7|<details><summary>Expand</summary>Description: FreeBayes is a Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs (single-nucleotide polymorphisms), indels (insertions and deletions), MNPs (multi-nucleotide polymorphisms), and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment. Homepage: https://github.com/freebayes/freebayes URL: https://github.com/freebayes/freebayes</details>|
	|[freesasa](https://freesasa.github.io)|-|2.1.0|<details><summary>Expand</summary>Description: FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA). Homepage: https://freesasa.github.io URL: https://freesasa.github.io</details>|
	|[freesurfer](https://surfer.nmr.mgh.harvard.edu/)|bio|5.3.0, 7.4.1|<details><summary>Expand</summary>Description: FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data. FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data. Homepage: https://surfer.nmr.mgh.harvard.edu/ URL: https://surfer.nmr.mgh.harvard.edu/</details>|
	|[freexl](https://www.gaia-gis.it/fossil/freexl/index)|tools|1.0.5, 2.0.0|<details><summary>Expand</summary>Description: FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. Homepage: https://www.gaia-gis.it/fossil/freexl/index URL: https://www.gaia-gis.it/fossil/freexl/index</details>|
	|[fsl](https://www.fmrib.ox.ac.uk/fsl/)|bio|6.0.3, 6.0.4, 6.0.7.7|<details><summary>Expand</summary>Description: FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data. Homepage: https://www.fmrib.ox.ac.uk/fsl/ URL: https://www.fmrib.ox.ac.uk/fsl/</details>|
	|[fsom](https://github.com/ekg/fsom)|-|20141119, 20151117|<details><summary>Expand</summary>Description: A tiny C library for managing SOM (Self-Organizing Maps) neural networks. Homepage: https://github.com/ekg/fsom URL: https://github.com/ekg/fsom</details>|
	|[g2clib](https://github.com/NOAA-EMC/NCEPLIBS-g2c)|geo|1.6.0, 1.8.0|<details><summary>Expand</summary>Description: This library contains C decoder/encoder routines for GRIB edition 2. This is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-g2c URL: https://github.com/NOAA-EMC/NCEPLIBS-g2c</details>|
	|[g2lib](https://github.com/NOAA-EMC/NCEPLIBS-g2/)|geo|3.1.0, 3.4.8|<details><summary>Expand</summary>Description: Library contains GRIB2 encoder/decoder and search/indexing routines. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-g2/ URL: https://github.com/NOAA-EMC/NCEPLIBS-g2/</details>|
	|[ga](http://hpc.pnl.gov/globalarrays/)|tools|5.7.2|<details><summary>Expand</summary>Description: Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model. It provides primitives for one-sided communication (Get, Put, Accumulate) and Atomic Operations (read increment). It supports blocking and non-blocking primtives, and supports location consistency. I8 version. Homepage: http://hpc.pnl.gov/globalarrays/ URL: http://hpc.pnl.gov/globalarrays/</details>|
	|[gamess-us](https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html)|chem|20210930-R2P1, 2020.2, 20220930-R2, 20230630-R1, 20230930-R2|<details><summary>Expand</summary>Description: The General Atomic and Molecular Electronic Structure System (GAMESS) is a general ab initio quantum chemistry package. This version of GAMESS-US is built with Intel compilers, mixed DDI, and OpenMPI 4.1. Homepage: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html URL: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html</details>|
	|[gapcloser](https://sourceforge.net/projects/soapdenovo2/files/GapCloser/)|-|1.12-r6|<details><summary>Expand</summary>Description: GapCloser is designed to close the gaps emerging during the scaffolding process by SOAPdenovo or other assembler, using the abundant pair relationships of short reads. Homepage: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/ URL: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/</details>|
	|[gate](http://www.opengatecollaboration.org/)|bio|9.3, 9.4|<details><summary>Expand</summary>Description: GATE is an advanced opensource software developed by the international OpenGATE collaboration and dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography Homepage: http://www.opengatecollaboration.org/ URL: http://www.opengatecollaboration.org/</details>|
	|[gatk](http://www.broadinstitute.org/gatk/)|bio|4.1.8.0, 4.1.8.1, 4.2.2.0, 4.2.4.0, 4.2.5.0, 4.4.0.0, 4.6.1.0|<details><summary>Expand</summary>Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size. Homepage: http://www.broadinstitute.org/gatk/ URL: http://www.broadinstitute.org/gatk/</details>|
	|[gazebo](https://gazebosim.org)|-|11.7.0|<details><summary>Expand</summary>Description: This is the Gazebo simulator. Gazebo simulates multiple robots in a 3D environment, with extensive dynamic interaction between objects. Homepage: https://gazebosim.org URL: https://gazebosim.org</details>|
	|[gblocks](http://molevol.cmima.csic.es/castresana/Gblocks.html)|-|0.91b|<details><summary>Expand</summary>Description: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis Homepage: http://molevol.cmima.csic.es/castresana/Gblocks.html URL: http://molevol.cmima.csic.es/castresana/Gblocks.html</details>|
	|[gcc](https://gcc.gnu.org/)|tools|8.4.0, 9.3.0, 10.2.0, 10.3.0, 11.3.0, 12.3, 13.3|<details><summary>Expand</summary>Description: The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). Homepage: https://gcc.gnu.org/ URL: https://gcc.gnu.org/</details>|
	|[gclust](http://gclust.c.u-tokyo.ac.jp/)|-|355z3|<details><summary>Expand</summary>Description: Gclust software was developed to make clusters of protein sequences from all predicted protein sequences in a selected set of genomes. The clusters are homolog groups, but not ortholog clusters (see below for the distinction), and therefore, contain all homologous sequences encoded by the selected genomes. An ortholog cluster, such as the one in KEGG Orthologs or COG in NCBI, contains only a single sequence for each genome, and such a single representative is usually selected by a criterion called bi-directional best hit Homepage: http://gclust.c.u-tokyo.ac.jp/ URL: http://gclust.c.u-tokyo.ac.jp/</details>|
	|[gcta](https://yanglab.westlake.edu.cn/software/gcta/)|bio|1.26.0, 1.93.2, 1.94.1|<details><summary>Expand</summary>Description: GCTA (Genome-wide Complex Trait Analysis) is a software package, which was initially developed to estimate the proportion of phenotypic variance explained by all genome-wide SNPs for a complex trait but has been extensively extended for many other analyses of data from genome-wide association studies (GWASs). Homepage: https://yanglab.westlake.edu.cn/software/gcta/ URL: https://yanglab.westlake.edu.cn/software/gcta/</details>|
	|[gd](https://github.com/lstein/Perl-GD)|-|2.71, 2.77|<details><summary>Expand</summary>Description: GD.pm - Interface to Gd Graphics Library Homepage: https://github.com/lstein/Perl-GD URL: https://github.com/lstein/Perl-GD</details>|
	|[gdal](https://www.gdal.org/)|geo|3.0.4, 3.2.3, 3.4.1, 3.5.1, 3.7.2, 3.9.1|<details><summary>Expand</summary>Description: GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing. Homepage: https://www.gdal.org/ URL: https://www.gdal.org/</details>|
	|[gdcm](https://sourceforge.net/projects/gdcm)|-|2.6.8, 3.0.8|<details><summary>Expand</summary>Description: Grassroots DICOM: Cross-platform DICOM implementation Homepage: https://sourceforge.net/projects/gdcm URL: https://sourceforge.net/projects/gdcm</details>|
	|[gdrcopy](https://github.com/NVIDIA/gdrcopy)|-|2.1, 2.3.1, 2.4.1|<details><summary>Expand</summary>Description: A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology. Homepage: https://github.com/NVIDIA/gdrcopy URL: https://github.com/NVIDIA/gdrcopy</details>|
	|[geant4](http://geant4.cern.ch/)|phys|10.06, 10.7.3, 11.1.0, 11.1.2, 11.2.1, 11.3.0|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[geant4-data](https://geant4.cern.ch/)|-|10.7.3, 11.1.0, 11.1.2, 11.2.1, 11.3.0|<details><summary>Expand</summary>Description: Datasets for Geant4. Homepage: https://geant4.cern.ch/ URL: https://geant4.cern.ch/</details>|
	|[geant4-seq](http://geant4.cern.ch/)|-|11.1.0|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[geant4-topasmc3.9](http://geant4.cern.ch/)|-|10.7.3|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[gem](https://github.com/ECCC-ASTD-MRD/gem)|-|5.1.1|<details><summary>Expand</summary>Description: The Global Environmental Multiscale Model (GEM) is an integrated forecasting and data assimilation system developed by the Atmospheric Numerical Prediction Research Section, Meteorological Research Division, of Environment and Climate Change Canada. Homepage: https://github.com/ECCC-ASTD-MRD/gem URL: https://github.com/ECCC-ASTD-MRD/gem</details>|
	|[gemma](http://veda.cs.uiuc.edu/Seq2Expr/)|bio|0.98.3, 0.98.5|<details><summary>Expand</summary>Description: GEMMA is a software toolkit for fast application of linear mixed models (LMMs) and related models to genome-wide association studies (GWAS) and other large-scale data sets. Homepage: http://veda.cs.uiuc.edu/Seq2Expr/ URL: http://veda.cs.uiuc.edu/Seq2Expr/</details>|
	|[gengetopt](https://www.gnu.org/software/gengetopt/gengetopt.html)|-|2.23|<details><summary>Expand</summary>Description: Gengetopt is a tool to write command line option parsing code for C programs. Homepage: https://www.gnu.org/software/gengetopt/gengetopt.html URL: https://www.gnu.org/software/gengetopt/gengetopt.html</details>|
	|[genmap](https://github.com/cpockrandt/genmap)|-|1.3.0|<details><summary>Expand</summary>Description: GenMap - Fast and Exact Computation of Genome Mappability GenMap computes the uniqueness of k-mers for each position in the genome while allowing for up to e mismatches. More formally, the uniqueness or (k,e)-mappability can be described for every position as the reciprocal value of how often each k-mer occurs approximately in the genome, i.e., with up to e mismatches. Hence, a mappability value of 1 at position i indicates that the k-mer in the sequence at position i occurs only once in the sequence with up to e errors. A low mappability value indicates that this k-mer belongs to a repetitive region. GenMap can be applied to single or multiple genomes and helps finding regions that are unique or shared by many or all genomes. Homepage: https://github.com/cpockrandt/genmap URL: https://github.com/cpockrandt/genmap</details>|
	|[genometools](http://genometools.org)|bio|1.6.1|<details><summary>Expand</summary>Description: A comprehensive software library for efficient processing of structured genome annotations. Homepage: http://genometools.org URL: http://genometools.org</details>|
	|[gentoo](https://wiki.gentoo.org/wiki/Project:Prefix)|-|2020, 2023|<details><summary>Expand</summary>Description: Geopsy team is developing, distributing and maintaining open source software for geophysical research and applications. Born during SESAME European Project, it has provided tools for processing ambient vibrations with site characterization in mind since 2005. Progressively, more conventional techniques (such as MASW or refraction) are included to offer a high quality, comprehensive and free platform for the interpretation of geophysical experiments. Homepage: https://www.geopsy.org/ URL: https://www.geopsy.org/</details>|
	|[geos](https://trac.osgeo.org/geos)|geo|3.7.3, 3.8.1, 3.9.1, 3.10.2, 3.12.0|<details><summary>Expand</summary>Description: GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS) Homepage: https://trac.osgeo.org/geos URL: https://trac.osgeo.org/geos</details>|
	|[gerris](http://gfs.sourceforge.net/wiki/index.php/Main_Page)|-|20131206|<details><summary>Expand</summary>Description: Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow Homepage: http://gfs.sourceforge.net/wiki/index.php/Main_Page URL: http://gfs.sourceforge.net/wiki/index.php/Main_Page</details>|
	|[gffcompare](https://ccb.jhu.edu/software/stringtie/gffcompare.shtml)|-|0.12.6|<details><summary>Expand</summary>Description: GffCompare provides classification and reference annotation mapping and matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files. Homepage: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml URL: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml</details>|
	|[gffread](https://github.com/gpertea/gffread)|-|0.11.7, 0.12.3|<details><summary>Expand</summary>Description: GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and more. Homepage: https://github.com/gpertea/gffread URL: https://github.com/gpertea/gffread</details>|
	|[gflags](https://github.com/gflags/gflags)|-|2.2.2|<details><summary>Expand</summary>Description: The gflags package contains a C++ library that implements commandline flags processing. It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. Homepage: https://github.com/gflags/gflags URL: https://github.com/gflags/gflags</details>|
	|[gibbs2](https://aoterodelaroza.github.io/gibbs2/)|-|1.0|<details><summary>Expand</summary>Description: Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary conditions of temperature and pressure. Gibbs2 uses the results of periodic solid-state quantum-mechanical calculations, specifically the energy-volume curve and possibly the harmonic phonon frequencies, to compute the thermodynamic properties of the solid within the framework of the quasiharmonic approximation. Homepage: https://aoterodelaroza.github.io/gibbs2/ URL: https://aoterodelaroza.github.io/gibbs2/</details>|
	|[ginkgo](https://ginkgo-project.github.io/)|-|1.6.0|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://ginkgo-project.github.io/ URL: https://ginkgo-project.github.io/</details>|
	|[git-annex](http://git-annex.branchable.com/)|tools|8.20200810, 10.20221003, 10.20231129|<details><summary>Expand</summary>Description: git-annex allows managing files with git, without checking the file contents into git. While that may seem paradoxical, it is useful when dealing with files larger than git can currently easily handle, whether due to limitations in memory, time, or disk space. Homepage: http://git-annex.branchable.com/ URL: http://git-annex.branchable.com/</details>|
	|[git-lfs](https://git-lfs.github.com)|-|2.11.0, 3.3.0, 3.4.0|<details><summary>Expand</summary>Description: Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com Homepage: https://git-lfs.github.com URL: https://git-lfs.github.com</details>|
	|[givaro](http://givaro.forge.imag.fr/)|-|4.2.0|<details><summary>Expand</summary>Description: C++ library for arithmetic and algebraic computations Homepage: http://givaro.forge.imag.fr/ URL: http://givaro.forge.imag.fr/</details>|
	|[glew](http://glew.sourceforge.net/)|-|2.1.0, 2.2.0|<details><summary>Expand</summary>Description: The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform. Homepage: http://glew.sourceforge.net/ URL: http://glew.sourceforge.net/</details>|
	|[glfw](https://www.glfw.org)|-|3.3.2, 3.3.8, 3.4|<details><summary>Expand</summary>Description: GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop Homepage: https://www.glfw.org URL: https://www.glfw.org</details>|
	|[glimmerhmm](https://ccb.jhu.edu/software/glimmerhmm)|-|3.0.4|<details><summary>Expand</summary>Description: GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model. Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models. Homepage: https://ccb.jhu.edu/software/glimmerhmm URL: https://ccb.jhu.edu/software/glimmerhmm</details>|
	|[glimpse](https://odelaneau.github.io/GLIMPSE/)|-|1.1.1, 2.0.0|<details><summary>Expand</summary>Description: GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies. Main features of the method: 1. Accurate imputed genotype calls. Our method takes advantage of reference panels to produce high quality genotype calls. 2. Accurate phasing. GLIMPSE outputs accurate phased haplotypes for the low-coverage sequenced dataset. 3. Low-coverage sequencing outperforms SNP arrays. Imputation using low-coverage sequencing data is competitive to SNP array imputation. Results for European and African-American populations are interactively available on the website. 4. A cost-effective paradigm. GLIMPSE realises whole genome imputation from the HRC reference panel for less than 1$. 5. An easy-to-use framework for diploid, haploid and mixed ploidy imputation. Our method is able to perform imputation of samples with different ploidy without the need to split the data. Homepage: https://odelaneau.github.io/GLIMPSE/ URL: https://odelaneau.github.io/GLIMPSE/</details>|
	|[glm](https://github.com/g-truc/glm)|vis|0.9.9.8|<details><summary>Expand</summary>Description: OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on the OpenGL Shading Language (GLSL) specifications. Homepage: https://github.com/g-truc/glm URL: https://github.com/g-truc/glm</details>|
	|[globalarrays](https://hpc.pnl.gov/globalarrays)|-|5.7.2, 5.8|<details><summary>Expand</summary>Description: Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model Homepage: https://hpc.pnl.gov/globalarrays URL: https://hpc.pnl.gov/globalarrays</details>|
	|[glost](https://github.com/cea-hpc/glost)|tools|0.3.1|<details><summary>Expand</summary>Description: This is GLOST, the Greedy Launcher Of Small Tasks. Homepage: https://github.com/cea-hpc/glost URL: https://github.com/cea-hpc/glost</details>|
	|[glpk](https://www.gnu.org/software/glpk/)|math|4.65, 5.0|<details><summary>Expand</summary>Description: The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems. It is a set of routines written in ANSI C and organized in the form of a callable library. Homepage: https://www.gnu.org/software/glpk/ URL: https://www.gnu.org/software/glpk/</details>|
	|[gmap-gsnap](http://research-pub.gene.com/gmap/)|bio|2024-10-20, 2020-11-14, 2024-08-20, 2019-09-12|<details><summary>Expand</summary>Description: GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences GSNAP: Genomic Short-read Nucleotide Alignment Program Homepage: http://research-pub.gene.com/gmap/ URL: http://research-pub.gene.com/gmap/</details>|
	|[gmp](https://gmplib.org/)|-|6.2.0|<details><summary>Expand</summary>Description: GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers. Homepage: https://gmplib.org/ URL: https://gmplib.org/</details>|
	|[gmsh](https://gmsh.info/)|phys|4.7.0, 4.10.5, 4.11.1, 4.12.2|<details><summary>Expand</summary>Description: Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor. Homepage: https://gmsh.info/ URL: https://gmsh.info/</details>|
	|[gmt](https://www.generic-mapping-tools.org/)|geo|6.5.0|<details><summary>Expand</summary>Description: GMT is an open source collection of about 80 command-line tools for manipulating geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting, etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another 40 more specialized and discipline-specific tools. Homepage: https://www.generic-mapping-tools.org/ URL: https://www.generic-mapping-tools.org/</details>|
	|[gmtk](http://melodi.ee.washington.edu/)|-|1.4.4|<details><summary>Expand</summary>Description: GMTK: The Graphical Models Toolkit Homepage: http://melodi.ee.washington.edu/ URL: http://melodi.ee.washington.edu/</details>|
	|[gnina](https://github.com/gnina/gnina)|-|1.0.1|<details><summary>Expand</summary>Description: gnina (pronounced NEE-na) is a molecular docking program with integrated support for scoring and optimizing ligands using convolutional neural networks. It is a fork of smina, which is a fork of AutoDock Vina. Homepage: https://github.com/gnina/gnina URL: https://github.com/gnina/gnina</details>|
	|[gnuplot](http://gnuplot.sourceforge.net)|vis|5.2.8, 5.4.2, 5.4.6, 5.4.8|<details><summary>Expand</summary>Description: Portable interactive, function plotting utility Homepage: http://gnuplot.sourceforge.net URL: http://gnuplot.sourceforge.net</details>|
	|[go](https://www.golang.org)|tools|1.14.1, 1.18.3, 1.21.3|<details><summary>Expand</summary>Description: Go is an open source programming language that makes it easy to build simple, reliable, and efficient software. Homepage: https://www.golang.org URL: https://www.golang.org</details>|
	|[gomc](https://gomc-wsu.org/)|-|2.75a|<details><summary>Expand</summary>Description: GOMC - GPU Optimized Monte Carlo is a parallel molecular simulation code designed for high-performance simulation of large systems. Homepage: https://gomc-wsu.org/ URL: https://gomc-wsu.org/</details>|
	|[googlebenchmark](https://github.com/google/benchmark)|-|1.7.1, 1.8.3|<details><summary>Expand</summary>Description: Google's library to benchmark code snippets, similar to unit tests. Homepage: https://github.com/google/benchmark URL: https://github.com/google/benchmark</details>|
	|[googletest](https://github.com/google/googletest)|tools|1.10.0, 1.13.0, 1.14.0|<details><summary>Expand</summary>Description: Google's framework for writing C++ tests on a variety of platforms Homepage: https://github.com/google/googletest URL: https://github.com/google/googletest</details>|
	|[grace](http://freecode.com/projects/grace)|vis|5.99.0|<details><summary>Expand</summary>Description: Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif. Homepage: http://freecode.com/projects/grace URL: http://freecode.com/projects/grace</details>|
	|[graph-tool](https://graph-tool.skewed.de/)|-|2.37, 2.45, 2.56|<details><summary>Expand</summary>Description: Graph-tool is an efficient Python module for manipulation and statistical analysis of graphs (a.k.a. networks). Contrary to most other python modules with similar functionality, the core data structures and algorithms are implemented in C++, making extensive use of template metaprogramming, based heavily on the Boost Graph Library. This confers it a level of performance that is comparable (both in memory usage and computation time) to that of a pure C/C++ library. Homepage: https://graph-tool.skewed.de/ URL: https://graph-tool.skewed.de/</details>|
	|[grass](https://grass.osgeo.org)|geo|7.8.4, 8.2.1|<details><summary>Expand</summary>Description: The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization Homepage: https://grass.osgeo.org URL: https://grass.osgeo.org</details>|
	|[groff](https://www.gnu.org/software/groff)|-|1.22.4|<details><summary>Expand</summary>Description: Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands and produces formatted output. Homepage: https://www.gnu.org/software/groff URL: https://www.gnu.org/software/groff</details>|
	|[gromacs](http://www.gromacs.org)|chem|2016.6, 2020.4, 2020.6, 2021.2, 2021.4, 2021.6, 2022.2, 2022.3, 2023.2, 2023.3, 2023.5, 2024.1, 2024.4|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a GPU enabled build, containing both MPI and threadMPI builds. Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-colvars](http://colvars.github.io/)|chem|2020.6|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version has been patched with 'Colvars', a collective variables module for molecular simulation programs. This is a CPU only build, containing both MPI and threadMPI builds. - CC-Wiki: GROMACS Homepage: http://colvars.github.io/ URL: http://colvars.github.io/</details>|
	|[gromacs-cp2k](https://www.cp2k.org/tools:gromacs)|chem|2022.2|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This version has been compiled with CP2K for QM/MM calculations. Note: only 'gmx_mpi' and 'gmx_mpi_d' executables are available. See also: https://www.cp2k.org/tools:gromacs - CC-Wiki: GROMACS Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-ls](https://vanegaslab.org/software)|-|2016.3|<details><summary>Expand</summary>Description: GROMACS-LS is a modified version of GROMACS that has been modified to enable the calculation of local stress fields from molecular dynamics simulations. This is a CPU only build. Note that GROMACS-LS does not support MPI, GPUs(CUDA) or Hardware accelleration (SIMD) and is therefore much slower than unmodified versions of GROMACS. CC-Wiki: GROMACS Homepage: https://vanegaslab.org/software URL: https://vanegaslab.org/software</details>|
	|[gromacs-plumed](http://www.gromacs.org)|chem|2019.6, 2020.7, 2021.2, 2021.4, 2021.6, 2021.7, 2022.3, 2022.6, 2023.5|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version contains the PLUMED patches in order to run metadynamics. - CC-Wiki: GROMACS Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-ramd](https://kbbox.h-its.org/toolbox/methods/molecular-simulation/random-acceleration-molecular-dynamics-ramd/)|chem|2024.1-RAMD-2.1, 2020.5-RAMD-2.0|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version has been modified to support RAMD, which is used to identify ligand exit routes from the buried binding pockets of receptors and investigate the mechanism of ligand dissociation. https://kbbox.h-its.org/toolbox/methods/molecular-simulation/random-acceleration-molecular-dynamics-ramd/ This is a CPU only build, containing both MPI and threadMPI builds. CC-Wiki: GROMACS Homepage: https://github.com/HITS-MCM/gromacs-ramd URL: https://github.com/HITS-MCM/gromacs-ramd</details>|
	|[gromacs-swaxs](https://cbjh.gitlab.io/gromacs-swaxs-docs/)|chem|2021.7-0.5.1|<details><summary>Expand</summary>Description: Modified GROMACS for small-angle scattering calculations (SAXS/WAXS/SANS). This is a GPU enabled build, containing both MPI and threadMPI builds. - CC-Wiki: GROMACS Homepage: https://cbjh.gitlab.io/gromacs-swaxs-docs/ URL: https://cbjh.gitlab.io/gromacs-swaxs-docs/</details>|
	|[gsl](https://www.gnu.org/software/gsl/)|math|1.16, 2.6, 2.7|<details><summary>Expand</summary>Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting. Homepage: https://www.gnu.org/software/gsl/ URL: https://www.gnu.org/software/gsl/</details>|
	|[gsl-lite](https://github.com/gsl-lite/gsl-lite)|-|0.40.0, 0.41.0|<details><summary>Expand</summary>Description: A single-file header-only version of ISO C++ Guidelines Support Library (GSL) for C++98, C++11, and later Homepage: https://github.com/gsl-lite/gsl-lite URL: https://github.com/gsl-lite/gsl-lite</details>|
	|[gts](http://gts.sourceforge.net/)|-|20121130|<details><summary>Expand</summary>Description: GTS stands for the GNU Triangulated Surface Library. It is an Open Source Free Software Library intended to provide a set of useful functions to deal with 3D surfaces meshed with interconnected triangles. Homepage: http://gts.sourceforge.net/ URL: http://gts.sourceforge.net/</details>|
	|[gudhi](https://gudhi.inria.fr/)|-|3.4.1, 3.7.1|<details><summary>Expand</summary>Description: The GUDHI library is a generic open source C++ library, with a Python interface, for Topological Data Analysis (TDA) and Higher Dimensional Geometry Understanding. The library offers state-of-the-art data structures and algorithms to construct simplicial complexes and compute persistent homology. The library comes with data sets, demos, examples and test suites. Homepage: https://gudhi.inria.fr/ URL: https://gudhi.inria.fr/</details>|
	|[guile](https://www.gnu.org/software/guile/)|tools|2.2.2|<details><summary>Expand</summary>Description: Guile is a programming language, designed to help programmers create flexible applications that can be extended by users or other programmers with plug-ins, modules, or scripts. Homepage: https://www.gnu.org/software/guile/ URL: https://www.gnu.org/software/guile/</details>|
	|[gurobi](https://www.gurobi.com)|math|9.0.3, 9.1.0, 9.1.2, 9.5.0, 9.5.2, 10.0.1, 10.0.2, 10.0.3, 11.0.0, 11.0.1, 11.0.3, 12.0.0|<details><summary>Expand</summary>Description: The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms. Homepage: https://www.gurobi.com URL: https://www.gurobi.com</details>|
	|[hal](https://github.com/ComparativeGenomicsToolkit/hal)|bio|2.2|<details><summary>Expand</summary>Description: HAL is a structure to efficiently store and index multiple genome alignments and ancestral reconstructions. HAL is a graph-based representation which provides several advantages over matrix/block-based formats such as MAF, such as improved scalability and the ability to perform queries with respect to an arbitrary reference or subtree. This package includes the HAL API and several analysis and conversion tools which are described below. HAL files are presently stored in either HDF5 or mmap format, but we note that the tools and most of the API are format-independent, so other databases could be implemented in the future. Homepage: https://github.com/ComparativeGenomicsToolkit/hal URL: https://github.com/ComparativeGenomicsToolkit/hal</details>|
	|[hapgen2](https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html)|-|2.2.0|<details><summary>Expand</summary>Description: 'HAPGEN2' simulates case control datasets at SNP markers. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html</details>|
	|[haploview](https://www.broadinstitute.org/haploview/haploview)|-|4.2|<details><summary>Expand</summary>Description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses. Homepage: https://www.broadinstitute.org/haploview/haploview URL: https://www.broadinstitute.org/haploview/haploview</details>|
	|[harminv](https://github.com/NanoComp/harminv)|math|1.4.1, 1.4.2|<details><summary>Expand</summary>Description: Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion - given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those sinusoids. Homepage: https://github.com/NanoComp/harminv URL: https://github.com/NanoComp/harminv</details>|
	|[hdf](https://www.hdfgroup.org/products/hdf4/)|io|4.2.15, 4.2.16|<details><summary>Expand</summary>Description: HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. Homepage: https://www.hdfgroup.org/products/hdf4/ URL: https://www.hdfgroup.org/products/hdf4/</details>|
	|[hdf-eos5](https://hdfeos.org)|-|5.1.16|<details><summary>Expand</summary>Description: HDF is the prescribed format for standard data products that are derived from EOS missions. HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura. Two versions of HDF-EOS libraries: HDF-EOS2 (v2.20) based on HDF4 and HDF-EOS5 (v5.1.16) based on HDF5 are developed. Homepage: https://hdfeos.org URL: https://hdfeos.org</details>|
	|[hdf-fortran](https://www.hdfgroup.org/products/hdf4/)|-|4.2.15|<details><summary>Expand</summary>Description: HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. Homepage: https://www.hdfgroup.org/products/hdf4/ URL: https://www.hdfgroup.org/products/hdf4/</details>|
	|[hdf5](https://portal.hdfgroup.org/display/support)|io|1.8.22, 1.10.6, 1.10.7, 1.12.1, 1.12.2, 1.14.2, 1.14.5|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdf5-mpi](https://portal.hdfgroup.org/display/support)|io|1.10.6, 1.10.7, 1.12.1, 1.12.2, 1.14.2, 1.14.4, 1.14.5|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdfview](https://support.hdfgroup.org/products/java/hdfview/)|-|2.14|<details><summary>Expand</summary>Description: HDFView is a visual tool for browsing and editing HDF4 and HDF5 files. Homepage: https://support.hdfgroup.org/products/java/hdfview/ URL: https://support.hdfgroup.org/products/java/hdfview/</details>|
	|[healpix](http://healpix.sourceforge.net/)|-|3.81, 3.82, 3.83|<details><summary>Expand</summary>Description: Hierarchical Equal Area isoLatitude Pixelation of a sphere. Homepage: http://healpix.sourceforge.net/ URL: http://healpix.sourceforge.net/</details>|
	|[heasoft](https://heasarc.nasa.gov/lheasoft/)|phys|6.32.1|<details><summary>Expand</summary>Description: A Unified Release of the FTOOLS and XANADU Software Packages XANADU High-level, multi-mission tasks for X-ray astronomical spectral, timing, and imaging data analysis FTOOLS General and mission-specific tools to manipulate FITS files FITSIO Core library responsible for reading and writing FITS files (distributed with FTOOLS) fv General FITS file browser/editor/plotter with a graphical user interface (distributed with FTOOLS) XSTAR Tool for calculating the physical conditions and emission spectra of photoionized gases Homepage: https://heasarc.nasa.gov/lheasoft/ URL: https://heasarc.nasa.gov/lheasoft/</details>|
	|[hh-suite](https://github.com/soedinglab/hh-suite)|bio|3.3.0|<details><summary>Expand</summary>Description: HH-suite is an open-source software package for sensitive protein sequence searching. It contains programs that can search for similar protein sequences in protein sequence databases. Homepage: https://github.com/soedinglab/hh-suite URL: https://github.com/soedinglab/hh-suite</details>|
	|[hifiasm](https://github.com/chhylp123/hifiasm)|-|0.16.1, 0.19.5|<details><summary>Expand</summary>Description: Hifiasm: a haplotype-resolved assembler for accurate Hifi reads. Homepage: https://github.com/chhylp123/hifiasm URL: https://github.com/chhylp123/hifiasm</details>|
	|[highfive](https://github.com/BlueBrain/HighFive)|-|2.9.0|<details><summary>Expand</summary>Description: HighFive is a modern header-only C++11 friendly interface for libhdf5. Homepage: https://github.com/BlueBrain/HighFive URL: https://github.com/BlueBrain/HighFive</details>|
	|[hisat2](https://daehwankimlab.github.io/hisat2/)|bio|2.2.1|<details><summary>Expand</summary>Description: HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome). Homepage: https://daehwankimlab.github.io/hisat2/ URL: https://daehwankimlab.github.io/hisat2/</details>|
	|[hmmer](http://hmmer.org/)|bio|3.2.1, 3.3.2, 3.4|<details><summary>Expand</summary>Description: HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs). Compared to BLAST, FASTA, and other sequence alignment and database search tools based on older scoring methodology, HMMER aims to be significantly more accurate and more able to detect remote homologs because of the strength of its underlying mathematical models. In the past, this strength came at significant computational expense, but in the new HMMER3 project, HMMER is now essentially as fast as BLAST. Homepage: http://hmmer.org/ URL: http://hmmer.org/</details>|
	|[hoomd-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|2.9.3, 3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[hoomd-blue-mpi](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[hpcspades](http://cab.spbu.ru/software/spades/)|-|3.15.0|<details><summary>Expand</summary>Description: Genome assembler for single-cell and isolates data sets Homepage: http://cab.spbu.ru/software/spades/ URL: http://cab.spbu.ru/software/spades/</details>|
	|[hpctoolkit](http://hpctoolkit.org/)|-|2020.08|<details><summary>Expand</summary>Description: HPCToolkit is an integrated suite of tools for measurement and analysis of program performance on computers ranging from multicore desktop systems to supercomputers. Homepage: http://hpctoolkit.org/ URL: http://hpctoolkit.org/</details>|
	|[hpl](https://www.netlib.org/benchmark/hpl/)|tools|2.3|<details><summary>Expand</summary>Description: HPL is a software package that solves a (random) dense linear system in double precision (64 bits) arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available implementation of the High Performance Computing Linpack Benchmark. Homepage: https://www.netlib.org/benchmark/hpl/ URL: https://www.netlib.org/benchmark/hpl/</details>|
	|[htslib](https://www.htslib.org/)|bio|1.9, 1.10.2, 1.11, 1.13, 1.14, 1.15.1, 1.16, 1.17, 1.18, 1.19|<details><summary>Expand</summary>Description: A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[hwloc](https://www.open-mpi.org/projects/hwloc/)|-|2.4.0, 2.7.1, 2.9.1, 2.10.0|<details><summary>Expand</summary>Description: The Portable Hardware Locality (hwloc) software package provides a portable abstraction (across OS, versions, architectures, ...) of the hierarchical topology of modern architectures, including NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various system attributes such as cache and memory information as well as the locality of I/O devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering information about modern computing hardware so as to exploit it accordingly and efficiently. Homepage: https://www.open-mpi.org/projects/hwloc/ URL: https://www.open-mpi.org/projects/hwloc/</details>|
	|[hyphy](https://veg.github.io/hyphy-site/)|bio|2.5.26, 2.5.49|<details><summary>Expand</summary>Description: HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package for the analysis of genetic sequences (in particular the inference of natural selection) using techniques in phylogenetics, molecular evolution, and machine learning Homepage: https://veg.github.io/hyphy-site/ URL: https://veg.github.io/hyphy-site/</details>|
	|[hypre](https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods)|math|2.20.0|<details><summary>Expand</summary>Description: Hypre is a library for solving large, sparse linear systems of equations on massively parallel computers. The problems of interest arise in the simulation codes being developed at LLNL and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences. Homepage: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods URL: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods</details>|
	|[idba-ud](http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/)|-|1.1.3|<details><summary>Expand</summary>Description: IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm. IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth contigs are removed iteratively with cutoff threshold from low to high to reduce the errors in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD can iterate k value of de Bruijn graph to a very large value with less gaps and less branches to form long contigs in both low-depth and high-depth regions. Homepage: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/ URL: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/</details>|
	|[igblast](https://www.ncbi.nlm.nih.gov/igblast/intro.html)|bio|1.17.0, 1.18.0|<details><summary>Expand</summary>Description: IgBLAST was developed at NCBI to facilitate analysis of immunoglobulin variable domain sequences (IgBLAST has recently been extended to perform analysis for T cell receptor (TR) sequences). It uses BLAST search algorithm. Homepage: https://www.ncbi.nlm.nih.gov/igblast/intro.html URL: https://www.ncbi.nlm.nih.gov/igblast/intro.html</details>|
	|[ignition](https://ignitionrobotics.org/)|-|citadel|<details><summary>Expand</summary>Description: Tools and libraries for robotics applications. Homepage: https://ignitionrobotics.org/ URL: https://ignitionrobotics.org/</details>|
	|[igraph](https://igraph.org)|math|0.8.2, 0.9.10, 0.10.2, 0.10.7, 0.10.13|<details><summary>Expand</summary>Description: igraph is a collection of network analysis tools with the emphasis on efficiency, portability and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++. Homepage: https://igraph.org URL: https://igraph.org</details>|
	|[igv](https://www.broadinstitute.org/software/igv/)|-|2.9.2|<details><summary>Expand</summary>Description: This package contains command line utilities for preprocessing, computing feature count density (coverage), sorting, and indexing data files. Homepage: https://www.broadinstitute.org/software/igv/ URL: https://www.broadinstitute.org/software/igv/</details>|
	|[ijulia-kernel](https://julialang.org/)|-|1.5, 1.8, 1.10|<details><summary>Expand</summary>Description: IJulia kernelspec for Julia 1.10 Homepage: https://julialang.org/ URL: https://julialang.org/</details>|
	|[ima3](https://github.com/jodyhey/IMa3)|-|1.12, 20210120|<details><summary>Expand</summary>Description: IMa3 is the newest in the IM sequence of programs. It can be used to solve a fundamental problem in evolutionary genetics, which is to jointly consider phylogenetic history and pouplation genetic history, including gene exchange. IMa3 can be used to estimate the rooted phylogenetic tree for multiple populations, and does so while integrating over all possible Isolation-with-Migration models. For a given phylogenetic tree IMa3 addresses the same model as IMa2. Like IMa2-p, IMa3 can run on multiple processors. Homepage: https://github.com/jodyhey/IMa3 URL: https://github.com/jodyhey/IMa3</details>|
	|[imb](https://software.intel.com/en-us/articles/intel-mpi-benchmarks)|-|2021.3|<details><summary>Expand</summary>Description: The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and global communication operations for a range of message sizes Homepage: https://software.intel.com/en-us/articles/intel-mpi-benchmarks URL: https://software.intel.com/en-us/articles/intel-mpi-benchmarks</details>|
	|[imkl](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)|math|2020.1.217, 2021.2.0, 2021.4.0, 2022.1.0, 2023.2.0, 2024.1.0, 2024.2.0|<details><summary>Expand</summary>Description: Intel oneAPI Math Kernel Library Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</details>|
	|[impute2](http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)|bio|2.3.2|<details><summary>Expand</summary>Description: IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation and haplotype phasing program based on ideas from Howie et al. 2009 Homepage: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html URL: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</details>|
	|[impute5](https://jmarchini.org/software/#impute-5)|-|1.1.5|<details><summary>Expand</summary>Description: IMPUTE 5 is a genotype imputation method that can scale to reference panels with millions of samples. This method continues to refine the observation made in the IMPUTE2 method, that accuracy is optimized via use of a custom subset of haplotypes when imputing each individual. It achieves fast, accurate, and memory-efficient imputation by selecting haplotypes using the Positional Burrows Wheeler Transform (PBWT). By using the PBWT data structure at genotyped markers, IMPUTE 5 identifies locally best matching haplotypes and long identical by state segments. The method then uses the selected haplotypes as conditioning states within the IMPUTE model. IMPUTE5 is up to 30x faster than MINIMAC4 and up to 3x faster than BEAGLE5.1 Homepage: https://jmarchini.org/software/#impute-5 URL: https://jmarchini.org/software/#impute-5</details>|
	|[infernal](http://eddylab.org/infernal/)|bio|1.1.3, 1.1.4, 1.1.5|<details><summary>Expand</summary>Description: Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. Homepage: http://eddylab.org/infernal/ URL: http://eddylab.org/infernal/</details>|
	|[intel](https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html)|tools|2020.1.217, 2021.2.0, 2022.1.0, 2023.2.1, 2024.2.0|<details><summary>Expand</summary>Description: Intel C, C++ and Fortran compilers (classic and oneAPI) Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</details>|
	|[intel-opencl](https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html)|-|2021.2.0|<details><summary>Expand</summary>Description: Intel's OpenCL CPU runtime Homepage: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html URL: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html</details>|
	|[intelmpi](https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html)|mpi|2019.7.217, 2021.2.0, 2021.9.0|<details><summary>Expand</summary>Description: Intel MPI Library, compatible with MPICH ABI Homepage: https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html URL: https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html</details>|
	|[intelxed](https://intelxed.github.io/)|-|12.0.1|<details><summary>Expand</summary>Description: The X86 Encoder Decoder (XED), is a software library (and associated headers) for encoding and decoding X86 (IA32 and Intel64) instructions. Homepage: https://intelxed.github.io/ URL: https://intelxed.github.io/</details>|
	|[interproscan](http://www.ebi.ac.uk/interpro/)|bio|5.52-86.0, 5.64-96.0, 5.63-95.0, 5.53-87.0, 5.73-104.0, 5.56-89.0, 5.50-84.0, 5.55-88.0|<details><summary>Expand</summary>Description: InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[interproscan_data](http://www.ebi.ac.uk/interpro/)|-|5.63-95.0, 5.73-104.0, 5.64-96.0|<details><summary>Expand</summary>Description: This module containes only InterProScan data part. InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[intervaltree](https://github.com/ekg/intervaltree)|-|0.1|<details><summary>Expand</summary>Description: An interval tree can be used to efficiently find a set of numeric intervals overlapping or containing another interval. This library provides a basic implementation of an interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. Homepage: https://github.com/ekg/intervaltree URL: https://github.com/ekg/intervaltree</details>|
	|[ioapi](https://www.cmascenter.org/ioapi/)|-|3.2-2020111|<details><summary>Expand</summary>Description: The Models-3/EDSS Input/Output Applications Programming Interface (I/O API) provides the environmental model developer with an easy-to-learn, easy-to-use programming library for data storage and access, available from both Fortran and C. The same routines can be used for both file storage (using netCDF files) and model coupling (using PVM mailboxes). It is the standard data access library for both the NCSC/CMAS's EDSS project and EPA's Models-3, CMAQ, and SMOKE, as well as various other atmospheric and hydrological modeling systems. Homepage: https://www.cmascenter.org/ioapi/ URL: https://www.cmascenter.org/ioapi/</details>|
	|[ipopt](https://projects.coin-or.org/Ipopt)|-|3.14.11, 3.14.14|<details><summary>Expand</summary>Description: IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt) is an open source software package for large-scale nonlinear optimization. Homepage: https://projects.coin-or.org/Ipopt URL: https://projects.coin-or.org/Ipopt</details>|
	|[ipp](https://software.intel.com/en-us/articles/intel-ipp/)|tools|2020.1.217|<details><summary>Expand</summary>Description: Intel Integrated Performance Primitives (Intel IPP) is an extensive library of multicore-ready, highly optimized software functions for multimedia, data processing, and communications applications. Intel IPP offers thousands of optimized functions covering frequently used fundamental algorithms. Homepage: https://software.intel.com/en-us/articles/intel-ipp/ URL: https://software.intel.com/en-us/articles/intel-ipp/</details>|
	|[ipykernel](https://ipython.org/)|-|2020a, 2020b, 2021a, 2022a, 2023a, 2023b, 2024a, 2024b, 2025a|<details><summary>Expand</summary>Description: Bundle which contains ipykernel 2025a and its dependencies Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[ipython-kernel](https://ipython.org/)|-|2.7, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12, 3.13|<details><summary>Expand</summary>Description: IPython kernelspec for Python 3.13 Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[iq-tree](http://www.iqtree.org/)|bio|1.6.12, 2.0.7, 2.1.2, 2.2.1, 2.2.2.7, 2.3.6|<details><summary>Expand</summary>Description: Efficient phylogenomic software by maximum likelihood Homepage: http://www.iqtree.org/ URL: http://www.iqtree.org/</details>|
	|[ir-kernel](https://irkernel.github.io)|-|4.2, 4.4|<details><summary>Expand</summary>Description: The R kernel for the 'Jupyter' environment executes R code which the front-end (Jupyter Notebook or other front-ends) submits to the kernel via the network. Homepage: https://irkernel.github.io URL: https://irkernel.github.io</details>|
	|[irfinder](https://github.com/williamritchie/IRFinder/)|-|1.3.1|<details><summary>Expand</summary>Description: Basic STAR (Spliced Transcripts Alignment to a Reference) STAR workflow consists of 2 steps: Generating genoms indexes files and mapping reads to the genomes. Homepage: https://github.com/williamritchie/IRFinder/ URL: https://github.com/williamritchie/IRFinder/</details>|
	|[isce2](https://github.com/isce-framework/isce2)|-|2.6.3|<details><summary>Expand</summary>Description: Interferometric synthetic aperture radar Scientific Computing Environment (ISCE) is a framework designed for the purpose of processing Interferometric Synthetic Aperture Radar (InSAR) data. The framework aspects of it have been designed as a general software development framework. It may have additional utility in a general sense for building other types of software packages. Homepage: https://github.com/isce-framework/isce2 URL: https://github.com/isce-framework/isce2</details>|
	|[ispc](https://ispc.github.io/)|-|1.10.0, 1.13.0, 1.18.0, 1.21.1|Description: Intel SPMD Program Compilers; An open-source compiler for high-performance SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language, with extensions for 'single program, multiple data' (SPMD) programming. Under the SPMD model, the programmer writes a program that generally appears to be a regular serial program, though the execution model is actually that a number of program instances execute in parallel on the hardware. Homepage: https://ispc.github.io/ , https://github.com/ispc/ispc/ URL: https://ispc.github.io/ , https://github.com/ispc/ispc/<br /><br /><br /></div> \|- \| align="center" \| [https://software.intel.com/en-us/intel-trace-analyzer/ itac] \| align="center" \| tools \| align="center" \| 2021.5.0 \|<details><summary>Expand</summary> <div class="mw-collapsible mw-collapsed" style="white-space: pre-line;"><br />Description: The Intel Trace Collector is a low-overhead tracing library that performs event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application activities gathered by the Intel Trace Collector through graphical displays. Homepage: https://software.intel.com/en-us/intel-trace-analyzer/ URL: https://software.intel.com/en-us/intel-trace-analyzer/</details>|
	|[itk](https://itk.org)|vis|4.13.3, 5.0.1, 5.1.2, 5.2.1, 5.3.0, 5.4.0|<details><summary>Expand</summary>Description: Insight Segmentation and Registration Toolkit (ITK) provides an extensive suite of software tools for registering and segmenting multidimensional imaging data. Homepage: https://itk.org URL: https://itk.org</details>|
	|[ivar](https://github.com/andersen-lab/ivar)|-|1.4.2|<details><summary>Expand</summary>Description: iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. Homepage: https://github.com/andersen-lab/ivar URL: https://github.com/andersen-lab/ivar</details>|
	|[jags](http://mcmc-jags.sourceforge.net/)|math|4.3.0, 4.3.2|<details><summary>Expand</summary>Description: JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation Homepage: http://mcmc-jags.sourceforge.net/ URL: http://mcmc-jags.sourceforge.net/</details>|
	|[jasper](https://www.ece.uvic.ca/~frodo/jasper/)|vis|2.0.16, 4.0.0|<details><summary>Expand</summary>Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard. Homepage: https://www.ece.uvic.ca/~frodo/jasper/ URL: https://www.ece.uvic.ca/~frodo/jasper/</details>|
	|[java](http://openjdk.java.net)|tools|11.0.16_8, 1.8.0_192, 1.8.0_292, 11.0.2, 11.0.22, 13.0.2, 14.0.2, 17.0.2, 17.0.6, 21.0.1|<details><summary>Expand</summary>Description: Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers. Homepage: http://openjdk.java.net URL: http://openjdk.java.net</details>|
	|[jbigkit](https://www.cl.cam.ac.uk/~mgk25/jbigkit/)|-|2.1|<details><summary>Expand</summary>Description: JBIG-KIT is a software implementation of the JBIG1 data compression standard (ITU-T T.82), which was designed for bi-level image data, such as scanned documents. Homepage: https://www.cl.cam.ac.uk/~mgk25/jbigkit/ URL: https://www.cl.cam.ac.uk/~mgk25/jbigkit/</details>|
	|[jdftx](http://jdftx.org/index.html)|chem|1.7.0|<details><summary>Expand</summary>Description: JDFTx is a plane-wave density-functional theory (DFT) code designed to be as easy to develop with as it is easy to use. It is distributed under the GPL license (version 3 or higher) and publications resulting from its use must cite: R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory Homepage: http://jdftx.org/index.html URL: http://jdftx.org/index.html</details>|
	|[jellyfish](http://www.genome.umd.edu/jellyfish.html)|bio|2.3.0|<details><summary>Expand</summary>Description: Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. Homepage: http://www.genome.umd.edu/jellyfish.html URL: http://www.genome.umd.edu/jellyfish.html</details>|
	|[jemalloc](http://jemalloc.net)|-|5.2.1, 5.3.0|<details><summary>Expand</summary>Description: jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support. Homepage: http://jemalloc.net URL: http://jemalloc.net</details>|
	|[json-fortran](https://github.com/jacobwilliams/json-fortran)|-|8.3.0|<details><summary>Expand</summary>Description: JSON-Fortran: A Modern Fortran JSON API Homepage: https://github.com/jacobwilliams/json-fortran URL: https://github.com/jacobwilliams/json-fortran</details>|
	|[jsoncpp](https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html)|-|1.9.4, 1.9.5|<details><summary>Expand</summary>Description: JsonCpp is a C++ library that allows manipulating JSON values, including serialization and deserialization to and from strings. It can also preserve existing comment in unserialization/serialization steps, making it a convenient format to store user input files. Homepage: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html URL: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html</details>|
	|[judy](http://judy.sourceforge.net/)|-|1.0.5|<details><summary>Expand</summary>Description: A C library that implements a dynamic array. Homepage: http://judy.sourceforge.net/ URL: http://judy.sourceforge.net/</details>|
	|[julia](https://julialang.org)|tools|1.4.1, 1.5.2, 1.6.0, 1.6.1, 1.7.0, 1.8.1, 1.8.5, 1.9.1, 1.9.3, 1.10.0, 1.11.3|<details><summary>Expand</summary>Description: Julia is a high-level, high-performance dynamic programming language for numerical computing Homepage: https://julialang.org URL: https://julialang.org</details>|
	|[kahip](https://kahip.github.io/)|-|3.14, 3.16|<details><summary>Expand</summary>Description: The graph partitioning framework KaHIP -- Karlsruhe High Quality Partitioning. Homepage: https://kahip.github.io/ URL: https://kahip.github.io/</details>|
	|[kahypar](https://kahypar.org/)|-|1.3.2|<details><summary>Expand</summary>Description: KaHyPar is a multilevel hypergraph partitioning framework providing direct k-way and recursive bisection based partitioning algorithms that compute solutions of very high quality. Homepage: https://kahypar.org/ URL: https://kahypar.org/</details>|
	|[kaiju](http://kaiju.binf.ku.dk/)|bio|1.6.2, 1.7.4, 1.10.1|<details><summary>Expand</summary>Description: Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments Homepage: http://kaiju.binf.ku.dk/ URL: http://kaiju.binf.ku.dk/</details>|
	|[kalign](https://github.com/TimoLassmann/kalign)|-|2.03, 3.3.5|<details><summary>Expand</summary>Description: Kalign is a fast multiple sequence alignment program for biological sequences. Homepage: https://github.com/TimoLassmann/kalign URL: https://github.com/TimoLassmann/kalign</details>|
	|[kallisto](https://pachterlab.github.io/kallisto/)|bio|0.46.1|<details><summary>Expand</summary>Description: kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. Homepage: https://pachterlab.github.io/kallisto/ URL: https://pachterlab.github.io/kallisto/</details>|
	|[kentutils](http://genome.cse.ucsc.edu/)|bio|401|<details><summary>Expand</summary>Description: Kent utilities: collection of tools used by the UCSC genome browser. Homepage: http://genome.cse.ucsc.edu/ URL: http://genome.cse.ucsc.edu/</details>|
	|[kim-api](https://openkim.org/)|chem|2.1.3, 2.3.0|<details><summary>Expand</summary>Description: Open Knowledgebase of Interatomic Models. KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations. This is a library that can be used by simulation programs to get access to the models in the OpenKIM database. This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running kim-api-collections-management install user MODELNAME or kim-api-collections-management install user OpenKIM to install them all. Homepage: https://openkim.org/ URL: https://openkim.org/</details>|
	|[kma](https://bitbucket.org/genomicepidemiology/kma)|-|1.3.0, 1.3.25, 1.4.14|<details><summary>Expand</summary>Description: KMA is a mapping method designed to map raw reads directly against redundant databases, in an ultra-fast manner using seed and extend. Homepage: https://bitbucket.org/genomicepidemiology/kma URL: https://bitbucket.org/genomicepidemiology/kma</details>|
	|[kmergenie](http://kmergenie.bx.psu.edu/)|-|1.7051|<details><summary>Expand</summary>Description: KmerGenie estimates the best k-mer length for genome de novo assembly. Homepage: http://kmergenie.bx.psu.edu/ URL: http://kmergenie.bx.psu.edu/</details>|
	|[komplexity](https://github.com/eclarke/komplexity)|-|0.3.6|<details><summary>Expand</summary>Description: A command-line tool built in Rust to quickly calculate and/or mask low-complexity sequences from a FAST[A/Q] file. This uses the number of unique k-mers over a sequence divided by the length to assess complexity. Homepage: https://github.com/eclarke/komplexity URL: https://github.com/eclarke/komplexity</details>|
	|[kraken](https://ccb.jhu.edu/software/kraken/)|bio|1.1.1|<details><summary>Expand</summary>Description: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. Homepage: https://ccb.jhu.edu/software/kraken/ URL: https://ccb.jhu.edu/software/kraken/</details>|
	|[kraken2](https://github.com/DerrickWood/kraken2/wiki)|bio|2.0.9-beta, 2.1.1, 2.1.2, 2.1.3|<details><summary>Expand</summary>Description: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. Homepage: https://github.com/DerrickWood/kraken2/wiki URL: https://github.com/DerrickWood/kraken2/wiki</details>|
	|[krakenuniq](https://github.com/fbreitwieser/krakenuniq)|-|1.0.2|<details><summary>Expand</summary>Description: krakenuniq: confident and fast metagenomics classification using unique k-mer counts. False-positive identifications are a significant problem in metagenomics classification. KrakenUniq (formerly KrakenHLL) is a novel metagenomics classifier that combines the fast k-mer-based classification of Kraken with an efficient algorithm for assessing the coverage of unique k-mers found in each species in a dataset. On various test datasets, KrakenUniq gives better recall and precision than other methods and effectively classifies and distinguishes pathogens with low abundance from false positives in infectious disease samples. By using the probabilistic cardinality estimator HyperLogLog, KrakenUniq runs as fast as Kraken and requires little additional memory. Homepage: https://github.com/fbreitwieser/krakenuniq URL: https://github.com/fbreitwieser/krakenuniq</details>|
	|[kronatools](https://github.com/marbl/Krona/wiki/KronaTools)|-|2.8|<details><summary>Expand</summary>Description: Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files. Homepage: https://github.com/marbl/Krona/wiki/KronaTools URL: https://github.com/marbl/Krona/wiki/KronaTools</details>|
	|[lammps-omp](https://www.lammps.org)|chem|20201029, 20210929, 20220623, 20230802, 20240829|<details><summary>Expand</summary>Description: LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality. Homepage: https://www.lammps.org URL: https://www.lammps.org</details>|
	|[last](http://last.cbrc.jp/)|bio|1145|<details><summary>Expand</summary>Description: LAST finds similar regions between sequences. Homepage: http://last.cbrc.jp/ URL: http://last.cbrc.jp/</details>|
	|[lastz](https://www.bx.psu.edu/~rsharris/lastz/)|-|1.04.03|<details><summary>Expand</summary>Description: LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS sequencing technologies such as Roche 454. Homepage: https://www.bx.psu.edu/~rsharris/lastz/ URL: https://www.bx.psu.edu/~rsharris/lastz/</details>|
	|[latte](https://github.com/lanl/LATTE)|chem|1.2.1|<details><summary>Expand</summary>Description: Open source density functional tight binding molecular dynamics. Homepage: https://github.com/lanl/LATTE URL: https://github.com/lanl/LATTE</details>|
	|[ldc](https://wiki.dlang.org/LDC)|-|0.17.6, 1.26.0, 1.30.0|<details><summary>Expand</summary>Description: The LLVM-based D Compiler Homepage: https://wiki.dlang.org/LDC URL: https://wiki.dlang.org/LDC</details>|
	|[leptonica](http://www.leptonica.org)|-|1.82.0|<details><summary>Expand</summary>Description: Leptonica is a collection of pedagogically-oriented open source software that is broadly useful for image processing and image analysis applications. Homepage: http://www.leptonica.org URL: http://www.leptonica.org</details>|
	|[lerc](https://github.com/Esri/lerc)|-|4.0.0|<details><summary>Expand</summary>Description: LERC is an open-source image or raster format which supports rapid encoding and decoding for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding, so the precision of the original input image is preserved (within user defined error bounds). Homepage: https://github.com/Esri/lerc URL: https://github.com/Esri/lerc</details>|
	|[leveldb](https://github.com/google/leveldb)|tools|1.22|<details><summary>Expand</summary>Description: LevelDB is a fast key-value storage library written at Google that provides an ordered mapping from string keys to string values. Homepage: https://github.com/google/leveldb URL: https://github.com/google/leveldb</details>|
	|[lhapdf](https://lhapdf.hepforge.org/index.html)|-|6.4.0|<details><summary>Expand</summary>Description: LHAPDF is a general purpose C++ interpolator, used for evaluating PDFs from discretised data files. Previous versions of LHAPDF were written in Fortran 77/90 Homepage: https://lhapdf.hepforge.org/index.html URL: https://lhapdf.hepforge.org/index.html</details>|
	|[libaec](https://gitlab.dkrz.de/k202009/libaec)|-|1.0.6|<details><summary>Expand</summary>Description: Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa. Homepage: https://gitlab.dkrz.de/k202009/libaec URL: https://gitlab.dkrz.de/k202009/libaec</details>|
	|[libbeef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|-|0.1.2|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: https://github.com/vossjo/libbeef URL: https://github.com/vossjo/libbeef</details>|
	|[libbigwig](https://github.com/dpryan79/libBigWig)|-|0.4.6|<details><summary>Expand</summary>Description: A C library for handling bigWig files Homepage: https://github.com/dpryan79/libBigWig URL: https://github.com/dpryan79/libBigWig</details>|
	|[libccd](https://github.com/danfis/libccd)|-|2.1|<details><summary>Expand</summary>Description: libccd is library for a collision detection between two convex shapes. libccd implements variation on GilbertJohnsonKeerthi algorithm plus Expand Polytope Algorithm (EPA) and also implements algorithm Minkowski Portal Refinement (MPR, a.k.a. XenoCollide) as described in Game Programming Gems 7. Homepage: https://github.com/danfis/libccd URL: https://github.com/danfis/libccd</details>|
	|[libcdms](https://github.com/CDAT/libcdms/)|-|3.1.2|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcdms/ URL: https://github.com/CDAT/libcdms/</details>|
	|[libcerf](https://jugit.fz-juelich.de/mlz/libcerf)|math|1.13, 1.17, 2.1, 2.4|<details><summary>Expand</summary>Description: libcerf is a self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions. Homepage: https://jugit.fz-juelich.de/mlz/libcerf URL: https://jugit.fz-juelich.de/mlz/libcerf</details>|
	|[libcf](https://github.com/CDAT/libcf/)|-|1.0.3|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcf/ URL: https://github.com/CDAT/libcf/</details>|
	|[libctl](https://libctl.readthedocs.io/en/latest/)|tools|4.5.0, 4.5.1|<details><summary>Expand</summary>Description: libctl is a free Guile-based library implementing flexible control files for scientific simulations. Homepage: https://libctl.readthedocs.io/en/latest/ URL: https://libctl.readthedocs.io/en/latest/</details>|
	|[libdap](https://www.opendap.org/software/libdap)|-|3.20.6, 3.20.11|<details><summary>Expand</summary>Description: A C++ SDK which contains an implementation of DAP 2.0 and DAP4.0. This includes both Client- and Server-side support classes. Homepage: https://www.opendap.org/software/libdap URL: https://www.opendap.org/software/libdap</details>|
	|[libdivsufsort](https://github.com/y-256/libdivsufsort)|-|2.0.1|<details><summary>Expand</summary>Description: libdivsufsort is a software library that implements a lightweight suffix array construction algorithm. Homepage: https://github.com/y-256/libdivsufsort URL: https://github.com/y-256/libdivsufsort</details>|
	|[libdrs](https://github.com/CDAT/libdrs/)|-|3.1.2|<details><summary>Expand</summary>Description: PCMDI's old DRS format implementation Homepage: https://github.com/CDAT/libdrs/ URL: https://github.com/CDAT/libdrs/</details>|
	|[libevent](https://libevent.org/)|-|2.1.11|<details><summary>Expand</summary>Description: The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached. Furthermore, libevent also support callbacks due to signals or regular timeouts. Homepage: https://libevent.org/ URL: https://libevent.org/</details>|
	|[libfabric](https://ofiwg.github.io/libfabric/)|-|1.10.1, 1.11.0, 1.12.1, 1.15.1, 1.18.0, 1.21.0|<details><summary>Expand</summary>Description: Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. Homepage: https://ofiwg.github.io/libfabric/ URL: https://ofiwg.github.io/libfabric/</details>|
	|[libfdf](https://launchpad.net/libfdf)|-|0.2.2|<details><summary>Expand</summary>Description: LibFDF provides a Fortran API to parse files in the Flexible Data Format (FDF). Homepage: https://launchpad.net/libfdf URL: https://launchpad.net/libfdf</details>|
	|[libffi](https://sourceware.org/libffi/)|-|3.3|<details><summary>Expand</summary>Description: The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time. Homepage: https://sourceware.org/libffi/ URL: https://sourceware.org/libffi/</details>|
	|[libgd](https://libgd.github.io)|vis|2.3.0, 2.3.3|<details><summary>Expand</summary>Description: GD is an open source code library for the dynamic creation of images by programmers. Homepage: https://libgd.github.io URL: https://libgd.github.io</details>|
	|[libgdsii](https://github.com/HomerReid/libGDSII)|-|0.21|<details><summary>Expand</summary>Description: libGDSII is a C++ library for working with GDSII binary data files, intended primarily for use with the computational electromagnetism codes scuff-em and meep but sufficiently general-purpose to allow other uses as well. Homepage: https://github.com/HomerReid/libGDSII URL: https://github.com/HomerReid/libGDSII</details>|
	|[libgeotiff](https://directory.fsf.org/wiki/Libgeotiff)|-|1.5.1, 1.6.0, 1.7.1|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libgeotiff-proj901](https://directory.fsf.org/wiki/Libgeotiff)|-|1.7.1|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libgridxc](https://launchpad.net/libgridxc)|chem|0.8.0|<details><summary>Expand</summary>Description: A library to compute the exchange and correlation energy and potential in spherical (i.e. an atom) or periodic systems. It is based on SiestaXC. Homepage: https://launchpad.net/libgridxc URL: https://launchpad.net/libgridxc</details>|
	|[libgridxc-mpi](https://launchpad.net/libgridxc)|-|0.8.0|<details><summary>Expand</summary>Description: A library to compute the exchange and correlation energy and potential in spherical (i.e. an atom) or periodic systems. It is based on SiestaXC. Homepage: https://launchpad.net/libgridxc URL: https://launchpad.net/libgridxc</details>|
	|[libgtextutils](http://hannonlab.cshl.edu/fastx_toolkit/)|bio|0.7|<details><summary>Expand</summary>Description: ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream Homepage: http://hannonlab.cshl.edu/fastx_toolkit/ URL: http://hannonlab.cshl.edu/fastx_toolkit/</details>|
	|[libharu](https://github.com/libharu/libharu/)|-|2.3.0|<details><summary>Expand</summary>Description: libHaru is a free, cross platform, open source library for generating PDF files. Homepage: https://github.com/libharu/libharu/ URL: https://github.com/libharu/libharu/</details>|
	|[libint](https://github.com/evaleev/libint)|chem|1.1.6, 2.0.3, 2.4.2, 2.6.0, 2.7.2|<details><summary>Expand</summary>Description: Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory. Homepage: https://github.com/evaleev/libint URL: https://github.com/evaleev/libint</details>|
	|[libmaus2](https://github.com/gt1/libmaus2)|bio|2.0.499|<details><summary>Expand</summary>Description: libmaus2 is a collection of data structures and algorithms. Homepage: https://github.com/gt1/libmaus2 URL: https://github.com/gt1/libmaus2</details>|
	|[libmbd](https://github.com/libmbd/libmbd)|-|0.12.8|<details><summary>Expand</summary>Description: A general-purpose package for scalable quantum many-body dispersion calculations. Homepage: https://github.com/libmbd/libmbd URL: https://github.com/libmbd/libmbd</details>|
	|[libmesh](https://github.com/libMesh/libmesh)|math|1.7.1, 1.7.5|<details><summary>Expand</summary>Description: The libMesh library provides a framework for the numerical simulation of partial differential equations using arbitrary unstructured discretizations on serial and parallel platforms. A major goal of the library is to provide support for adaptive mesh refinement (AMR) computations in parallel while allowing a research scientist to focus on the physics they are modeling. Homepage: https://github.com/libMesh/libmesh URL: https://github.com/libMesh/libmesh</details>|
	|[libpsml](https://gitlab.com/siesta-project/libraries/libpsml)|-|1.1.8, 1.1.12|<details><summary>Expand</summary>Description: A library to handle pseudopotentials in PSML format Homepage: https://gitlab.com/siesta-project/libraries/libpsml URL: https://gitlab.com/siesta-project/libraries/libpsml</details>|
	|[libraw](https://www.libraw.org/)|-|0.21.2|<details><summary>Expand</summary>Description: The LibRaw library provides a simple and unified interface for extracting out of RAW files generated by digital photo cameras the following: - RAW data (pixel values) - Metadata necessary for processing RAW (geometry, CFA / Bayer pattern, black level, white balance, etc.) - Embedded preview / thumbnail. Homepage: https://www.libraw.org/ URL: https://www.libraw.org/</details>|
	|[librdkafka](https://github.com/edenhill/librdkafka)|-|1.5.2-RC1|<details><summary>Expand</summary>Description: librdkafka is a C library implementation of the Apache Kafka protocol, providing Producer, Consumer and Admin clients. It was designed with message delivery reliability and high performance in mind, current figures exceed 1 million msgs/second for the producer and 3 million msgs/second for the consumer. librdkafka is licensed under the 2-clause BSD license. Homepage: https://github.com/edenhill/librdkafka URL: https://github.com/edenhill/librdkafka</details>|
	|[librttopo](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[librttopo-proj9](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[libspatialindex](https://libspatialindex.org)|phys|1.8.5, 1.9.3|<details><summary>Expand</summary>Description: C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API Homepage: https://libspatialindex.org URL: https://libspatialindex.org</details>|
	|[libspatialite](https://www.gaia-gis.it/fossil/libspatialite/home)|phys|4.3.0a, 5.0.1, 5.1.0|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libspatialite-proj9](https://www.gaia-gis.it/fossil/libspatialite/home)|-|5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libspatialite-proj901](https://www.gaia-gis.it/fossil/libspatialite/home)|-|5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libstatgen](https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen)|-|20190330|<details><summary>Expand</summary>Description: Useful set of classes for creating statistical genetic programs. Homepage: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen URL: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen</details>|
	|[liburing](https://github.com/axboe/liburing)|-|2.3|<details><summary>Expand</summary>Description: liburing provides helpers to setup and teardown io_uring instances, and also a simplified interface for applications that don't need (or want) to deal with the full kernel side implementation. Homepage: https://github.com/axboe/liburing URL: https://github.com/axboe/liburing</details>|
	|[libvori](https://brehm-research.de/libvori.php)|-|220621|<details><summary>Expand</summary>Description: C++ library implementing the Voronoi integration as well as the compressed bqb file format. The present version of libvori is a very early development version, which is hard-coded to work with the CP2k program package. Homepage: https://brehm-research.de/libvori.php URL: https://brehm-research.de/libvori.php</details>|
	|[libxc](https://libxc.gitlab.io)|chem|2.2.2, 3.0.0, 3.0.1, 4.3.4, 5.0.0, 5.1.3, 5.2.3, 6.2.2|<details><summary>Expand</summary>Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals. Homepage: https://libxc.gitlab.io URL: https://libxc.gitlab.io</details>|
	|[libxslt](http://xmlsoft.org/)|-|1.1.29, 1.1.34|<details><summary>Expand</summary>Description: Libxslt is the XSLT C library developed for the GNOME project (but usable outside of the Gnome platform). Homepage: http://xmlsoft.org/ URL: http://xmlsoft.org/</details>|
	|[libxsmm](https://github.com/hfp/libxsmm)|math|1.16.1, 1.17|<details><summary>Expand</summary>Description: LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86). Homepage: https://github.com/hfp/libxsmm URL: https://github.com/hfp/libxsmm</details>|
	|[libyaml](https://pyyaml.org/wiki/LibYAML)|-|0.2.5|<details><summary>Expand</summary>Description: LibYAML is a YAML parser and emitter written in C. Homepage: https://pyyaml.org/wiki/LibYAML URL: https://pyyaml.org/wiki/LibYAML</details>|
	|[liknorm](https://github.com/limix/liknorm)|tools|1.5.1|<details><summary>Expand</summary>Description: C library for computing moments of the product of an exponential-family likelihood with a Normal distribution. Homepage: https://github.com/limix/liknorm URL: https://github.com/limix/liknorm</details>|
	|[limix-bgen](https://github.com/limix/bgen)|-|3.0.3|<details><summary>Expand</summary>Description: A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3. Homepage: https://github.com/limix/bgen URL: https://github.com/limix/bgen</details>|
	|[links](http://www.bcgsc.ca/platform/bioinfo/software/arcs).)|bio|1.8.6, 2.0.1|<details><summary>Expand</summary>Description: LINKS is a scalable genomics application for scaffolding or re-scaffolding genome assembly drafts with long reads, such as those produced by Oxford Nanopore Technologies Ltd and Pacific Biosciences. It provides a generic alignment-free framework for scaffolding and can work on any sequences. It is versatile and supports not only long sequences as a source of long-range information, but also MPET pairs and linked-reads, such as those from the 10X Genomics GemCode and Chromium platform, via ARCS (http://www.bcgsc.ca/platform/bioinfo/software/arcs). Fill gaps in LINKS-derived scaffolds using Sealer (http://www.bcgsc.ca/platform/bioinfo/software/sealer). Homepage: http://www.bcgsc.ca/platform/bioinfo/software/links URL: http://www.bcgsc.ca/platform/bioinfo/software/links</details>|
	|[littlecms](http://www.littlecms.com/)|-|2.9, 2.11|<details><summary>Expand</summary>Description: Little CMS intends to be an OPEN SOURCE small-footprint color management engine, with special focus on accuracy and performance. Homepage: http://www.littlecms.com/ URL: http://www.littlecms.com/</details>|
	|[lldb](https://lldb.llvm.org/)|-|11.0.0|<details><summary>Expand</summary>Description: The debugger component of the LLVM project Homepage: https://lldb.llvm.org/ URL: https://lldb.llvm.org/</details>|
	|[llvm](https://llvm.org/)|tools|8.0.1, 9.0.1, 13.0.1, 14.0.3, 16.0.6, 18.1.8|<details><summary>Expand</summary>Description: The LLVM Core libraries provide a modern source- and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!) These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR"). The LLVM Core libraries are well documented, and it is particularly easy to invent your own language (or port an existing compiler) to use LLVM as an optimizer and code generator. Homepage: https://llvm.org/ URL: https://llvm.org/</details>|
	|[lmdb](https://symas.com/lmdb)|-|0.9.24, 0.9.31|<details><summary>Expand</summary>Description: LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance of a pure in-memory database while retaining the persistence of standard disk-based databases. Homepage: https://symas.com/lmdb URL: https://symas.com/lmdb</details>|
	|[lpsolve](https://sourceforge.net/projects/lpsolve/)|math|5.5.2.5|<details><summary>Expand</summary>Description: Mixed Integer Linear Programming (MILP) solver Homepage: https://sourceforge.net/projects/lpsolve/ URL: https://sourceforge.net/projects/lpsolve/</details>|
	|[ls-dyna](http://www.lstc.com/products/ls-dyna)|phys|12.2.1|<details><summary>Expand</summary>Description: LS-DYNA is a general-purpose finite element program capable of simulating complex real world problems. It is used by the automobile, aerospace, construction, military, manufacturing, and bioengineering industries. LS-DYNA is optimized for shared and distributed memory Unix, Linux, and Windows based, platforms, and it is fully QA'd by LSTC. The code's origins lie in highly nonlinear, transient dynamic finite element analysis using explicit time integration. Homepage: http://www.lstc.com/products/ls-dyna URL: http://www.lstc.com/products/ls-dyna</details>|
	|[ls-dyna-mpi](http://www.lstc.com/products/ls-dyna)|phys|12.2.1|<details><summary>Expand</summary>Description: LS-DYNA is a general-purpose finite element program capable of simulating complex real world problems. It is used by the automobile, aerospace, construction, military, manufacturing, and bioengineering industries. LS-DYNA is optimized for shared and distributed memory Unix, Linux, and Windows based, platforms, and it is fully QA'd by LSTC. The code's origins lie in highly nonlinear, transient dynamic finite element analysis using explicit time integration. Homepage: http://www.lstc.com/products/ls-dyna URL: http://www.lstc.com/products/ls-dyna</details>|
	|[ls-opt](http://www.lstc.com/products/ls-opt)|phys|7.0.0|<details><summary>Expand</summary>Description: LS-OPT is a standalone Design Optimization and Probabilistic Analysis package with an interface to LS-DYNA. Homepage: http://www.lstc.com/products/ls-opt URL: http://www.lstc.com/products/ls-opt</details>|
	|[ls-prepost](http://lstc.com/products/ls-prepost)|-|4.8.11, 4.9.9|<details><summary>Expand</summary>Description: LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA. Homepage: http://lstc.com/products/ls-prepost URL: http://lstc.com/products/ls-prepost</details>|
	|[lsd2](https://github.com/tothuhien/lsd2)|-|1.9.7, 2.3, 2.4.1|<details><summary>Expand</summary>Description: Least-squares methods to estimate rates and dates from phylogenies Homepage: https://github.com/tothuhien/lsd2 URL: https://github.com/tothuhien/lsd2</details>|
	|[ltr_retriever](https://github.com/oushujun/LTR_retriever)|-|2.9.0|<details><summary>Expand</summary>Description: LTR_retriever is a command line program (in Perl) for accurate identification of LTR retrotransposons (LTR-RTs) from outputs of LTRharvest, LTR_FINDER, MGEScan 3.0.0, LTR_STRUC, and LtrDetector, and generates non-redundant LTR-RT library for genome annotations. By default, the program will generate whole-genome LTR-RT annotation and the LTR Assembly Index (LAI) for evaluations of the assembly continuity of the input genome. Homepage: https://github.com/oushujun/LTR_retriever URL: https://github.com/oushujun/LTR_retriever</details>|
	|[lumpy](https://github.com/arq5x/lumpy-sv)|bio|0.2.13|<details><summary>Expand</summary>Description: A probabilistic framework for structural variant discovery. Homepage: https://github.com/arq5x/lumpy-sv URL: https://github.com/arq5x/lumpy-sv</details>|
	|[m2](https://github.com/Macaulay2/M2)|-|1.19.1, 1.21|<details><summary>Expand</summary>Description: Macaulay2 is a system for computing in commutative algebra, algebraic geometry and related fields. The system was originally written by Dan Grayson and Mike Stillman. David Eisenbud joined the project a number of years ago, and many users are writing packages for the system, and some are contributing source code. Homepage: https://github.com/Macaulay2/M2 URL: https://github.com/Macaulay2/M2</details>|
	|[maeparser](https://github.com/schrodinger/maeparser)|chem|1.2.4|<details><summary>Expand</summary>Description: maeparser is a parser for Schrodinger Maestro files. Homepage: https://github.com/schrodinger/maeparser URL: https://github.com/schrodinger/maeparser</details>|
	|[maffilter](https://jydu.github.io/maffilter/)|-|2020.11|<details><summary>Expand</summary>Description: Genome Alignment Processor Homepage: https://jydu.github.io/maffilter/ URL: https://jydu.github.io/maffilter/</details>|
	|[mafft](https://mafft.cbrc.jp/alignment/software/source.html)|bio|7.471, 7.526|<details><summary>Expand</summary>Description: MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc. Homepage: https://mafft.cbrc.jp/alignment/software/source.html URL: https://mafft.cbrc.jp/alignment/software/source.html</details>|
	|[mafft-mpi](https://mafft.cbrc.jp/alignment/software/source.html)|-|7.471, 7.526|<details><summary>Expand</summary>Description: MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc. Homepage: https://mafft.cbrc.jp/alignment/software/source.html URL: https://mafft.cbrc.jp/alignment/software/source.html</details>|
	|[magma](https://icl.cs.utk.edu/magma/)|math|2.5.4, 2.6.1, 2.7.1, 2.7.2, 2.8.0|<details><summary>Expand</summary>Description: The MAGMA project aims to develop a dense linear algebra library similar to LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems. Homepage: https://icl.cs.utk.edu/magma/ URL: https://icl.cs.utk.edu/magma/</details>|
	|[magma-gene-analysis](https://ctg.cncr.nl/software/magma)|-|1.10|<details><summary>Expand</summary>Description: MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous GWAS or meta-analysis. Homepage: https://ctg.cncr.nl/software/magma URL: https://ctg.cncr.nl/software/magma</details>|
	|[makeinfo](https://www.gnu.org/software/texinfo/)|-|6.7|<details><summary>Expand</summary>Description: makeinfo is part of the Texinfo project, the official documentation format of the GNU project. Homepage: https://www.gnu.org/software/texinfo/ URL: https://www.gnu.org/software/texinfo/</details>|
	|[maker](http://www.yandell-lab.org/software/maker.html)|bio|3.01.03|<details><summary>Expand</summary>Description: MAKER is a portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases. MAKER identifies repeats, aligns ESTs and proteins to a genome, produces ab-initio gene predictions and automatically synthesizes these data into gene annotations having evidence-based quality values. Homepage: http://www.yandell-lab.org/software/maker.html URL: http://www.yandell-lab.org/software/maker.html</details>|
	|[manta](https://github.com/Illumina/manta)|bio|1.6.0|<details><summary>Expand</summary>Description: Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. It is optimized for analysis of germline variation in small sets of individuals and somatic variation in tumor/normal sample pairs. Manta discovers, assembles and scores large-scale SVs, medium-sized indels and large insertions within a single efficient workflow. Homepage: https://github.com/Illumina/manta URL: https://github.com/Illumina/manta</details>|
	|[marginpolish](https://github.com/UCSC-nanopore-cgl/MarginPolish)|-|1.3.0|<details><summary>Expand</summary>Description: MarginPolish is a graph-based assembly polisher. It iteratively finds multiple probable alignment paths for run-length-encoded reads and uses these to generate a refined sequence. It takes as input a FASTA assembly and an indexed BAM (ONT reads aligned to the assembly), and it produces a polished FASTA assembly. Homepage: https://github.com/UCSC-nanopore-cgl/MarginPolish URL: https://github.com/UCSC-nanopore-cgl/MarginPolish</details>|
	|[mariadb](https://docs.computecanada.ca/wiki/Database_servers.)|tools|10.4.13, 10.6.12, 11.5.0|<details><summary>Expand</summary>Description: MariaDB An enhanced, drop-in replacement for MySQL For more information please take a look at https://docs.computecanada.ca/wiki/Database_servers. Homepage: https://mariadb.org/ URL: https://mariadb.org/</details>|
	|[mariadb-connector-c](https://downloads.mariadb.org/connector-c/)|-|3.1.7, 3.3.7|<details><summary>Expand</summary>Description: MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases. Homepage: https://downloads.mariadb.org/connector-c/ URL: https://downloads.mariadb.org/connector-c/</details>|
	|[mash](http://mash.readthedocs.org)|-|2.3|<details><summary>Expand</summary>Description: Fast genome and metagenome distance estimation using MinHash Homepage: http://mash.readthedocs.org URL: http://mash.readthedocs.org</details>|
	|[masurca](http://www.genome.umd.edu/masurca.html)|bio|3.4.1, 4.0.1, 4.0.3, 4.1.0|<details><summary>Expand</summary>Description: MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454, Pacbio and Nanopore). Homepage: http://www.genome.umd.edu/masurca.html URL: http://www.genome.umd.edu/masurca.html</details>|
	|[matio](https://sourceforge.net/projects/matio/)|io|1.5.19, 1.5.26|<details><summary>Expand</summary>Description: matio is an C library for reading and writing Matlab MAT files. Homepage: https://sourceforge.net/projects/matio/ URL: https://sourceforge.net/projects/matio/</details>|
	|[matlab](http://www.mathworks.com/products/matlab)|tools|2020a, 2020b, 2020b.4, 2020b.6, 2021a.1, 2021a.5, 2021b.3, 2022a, 2022b.2, 2023a.3, 2023b.2, 2024b.1|<details><summary>Expand</summary>Description: MATLAB is a high-level language and interactive environment that enables you to perform computationally intensive tasks faster than with traditional programming languages such as C, C++, and Fortran. Homepage: http://www.mathworks.com/products/matlab URL: http://www.mathworks.com/products/matlab</details>|
	|[maven](https://maven.apache.org/index.html)|tools|3.6.3, 3.9.6|<details><summary>Expand</summary>Description: Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. Homepage: https://maven.apache.org/index.html URL: https://maven.apache.org/index.html</details>|
	|[maxbin](https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html)|bio|2.2.7|<details><summary>Expand</summary>Description: MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm. Homepage: https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html URL: https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</details>|
	|[mcl](http://micans.org/mcl/)|math|14.137|<details><summary>Expand</summary>Description: The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for networks (also known as graphs) based on simulation of (stochastic) flow in graphs. The algorithm was invented/discovered by Stijn van Dongen at the Centre for Mathematics and Computer Science (also known as CWI) in the Netherlands. MCL has been applied in a number of different domains, mostly in bioinformatics. Homepage: http://micans.org/mcl/ URL: http://micans.org/mcl/</details>|
	|[mcr](https://www.mathworks.com/products/compiler/mcr/)|tools|R2022b, R2021b, R2020b, R2024a|<details><summary>Expand</summary>Description: The MATLAB Runtime is a standalone set of shared libraries that enables the execution of compiled MATLAB applications or components on computers that do not have MATLAB installed. Homepage: https://www.mathworks.com/products/compiler/mcr/ URL: https://www.mathworks.com/products/compiler/mcr/</details>|
	|[mctc-lib](https://grimme-lab.github.io/mctc-lib)|-|0.3.1|<details><summary>Expand</summary>Description: Common tool chain for working with molecular structure data in various applications. This library provides a unified way to perform operations on molecular structure data, like reading and writing to common geometry file formats. Homepage: https://grimme-lab.github.io/mctc-lib URL: https://grimme-lab.github.io/mctc-lib</details>|
	|[meep](https://meep.readthedocs.io/en/latest/)|phys|1.16.1, 1.24.0, 1.25.0, 1.29.0|<details><summary>Expand</summary>Description: Meep is a free and open-source software package for electromagnetics simulation via the finite-difference time-domain (FDTD) method spanning a broad range of applications. Homepage: https://meep.readthedocs.io/en/latest/ URL: https://meep.readthedocs.io/en/latest/</details>|
	|[megahit](https://github.com/voutcn/megahit)|bio|1.2.9|<details><summary>Expand</summary>Description: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph Homepage: https://github.com/voutcn/megahit URL: https://github.com/voutcn/megahit</details>|
	|[meme](http://meme-suite.org)|-|5.2.0, 5.4.1, 5.5.0, 5.5.5, 5.5.7|<details><summary>Expand</summary>Description: The MEME Suite allows you to: - discover motifs using MEME, DREME (DNA only) or GLAM2 on groups of related DNA or protein sequences, - search sequence databases with motifs using MAST, FIMO, MCAST or GLAM2SCAN, - compare a motif to all motifs in a database of motifs, - associate motifs with Gene Ontology terms via their putative target genes, - analyse motif enrichment using SpaMo or CentriMo. Homepage: http://meme-suite.org URL: http://meme-suite.org</details>|
	|[mesa-astrophysics](https://docs.mesastar.org)|phys|r24.03.1|<details><summary>Expand</summary>Description: MESA Modules for Experiments in Stellar Astrophysics Homepage: https://docs.mesastar.org URL: https://docs.mesastar.org</details>|
	|[mesquite](https://software.sandia.gov/mesquite/)|math|2.3.0|<details><summary>Expand</summary>Description: Mesh-Quality Improvement Library Homepage: https://software.sandia.gov/mesquite/ URL: https://software.sandia.gov/mesquite/</details>|
	|[met](https://dtcenter.org/)|phys|9.1.1|<details><summary>Expand</summary>Description: Model Evaluation Tools (MET) is a highly configurable, state-of-the-art suite of verification tools designed for the verification and evaluation of numerical weather forecasts. Although MET was developed at the Developmental Testbed Center (DTC) to be used with the Weather Research and Forecasting (WRF) modeling system, the data formats and standards used allow MET to be run on the output of other modeling systems as well. Homepage: https://dtcenter.org/ URL: https://dtcenter.org/</details>|
	|[meta-farm](https://docs.computecanada.ca/wiki/META_package_for_serial_farming)|-|1.0.2, 1.0.3|<details><summary>Expand</summary>Description: Meta-farm is a suite of scripts designed in-house to fully automate throughput computing (running a large number of related serial/parallel/GPU jobs - so called farming). Homepage: https://docs.computecanada.ca/wiki/META_package_for_serial_farming URL: https://docs.computecanada.ca/wiki/META_package_for_serial_farming</details>|
	|[metabat](https://bitbucket.org/berkeleylab/metabat)|bio|2.12.1, 2.14, 2.17|<details><summary>Expand</summary>Description: An efficient tool for accurately reconstructing single genomes from complex microbial communities Homepage: https://bitbucket.org/berkeleylab/metabat URL: https://bitbucket.org/berkeleylab/metabat</details>|
	|[metaeuk](https://metaeuk.soedinglab.org)|-|4-a0f584d, 6|<details><summary>Expand</summary>Description: MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic metagenomic contigs. Homepage: https://metaeuk.soedinglab.org URL: https://metaeuk.soedinglab.org</details>|
	|[metageneannotator](http://metagene.nig.ac.jp/)|-|20080819|<details><summary>Expand</summary>Description: MetaGeneAnnotator is a gene-finding program for prokaryote and phage. Homepage: http://metagene.nig.ac.jp/ URL: http://metagene.nig.ac.jp/</details>|
	|[metagenome-atlas](https://github.com/metagenome-atlas/atlas)|-|2.4.3, 2.5.0|<details><summary>Expand</summary>Description: Metagenome-atlas is a easy-to-use metagenomic pipeline based on snakemake. It handles all steps from QC, Assembly, Binning, to Annotation. Homepage: https://github.com/metagenome-atlas/atlas URL: https://github.com/metagenome-atlas/atlas</details>|
	|[metal](http://www.sph.umich.edu/csg/abecasis/metal/)|bio|2011-03-25|<details><summary>Expand</summary>Description: METAL provides a computationally efficient tool for meta-analysis of genome-wide association scans, which is a commonly used approach for improving power complex traits gene mapping studies. METAL provides a rich scripting interface and implements efficient memory management to allow analyses of very large data sets and to support a variety of input file formats. Availability and implementation: METAL, including source code, documentation, examples, and executables, is available at: http://www.sph.umich.edu/csg/abecasis/metal/ If you use Metal please fill out the registration form on the page: http://csg.sph.umich.edu/abecasis/metal/download/ Contact: goncalo@umich.edu Homepage: http://www.sph.umich.edu/csg/abecasis/metal/ URL: http://www.sph.umich.edu/csg/abecasis/metal/</details>|
	|[metaxa2](https://microbiology.se/software/metaxa2/)|-|2.2|<details><summary>Expand</summary>Description: Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically Homepage: https://microbiology.se/software/metaxa2/ URL: https://microbiology.se/software/metaxa2/</details>|
	|[methyldackel](https://github.com/dpryan79/MethylDackel)|-|0.6.1|<details><summary>Expand</summary>Description: A (mostly) universal methylation extractor for BS-seq experiments. Homepage: https://github.com/dpryan79/MethylDackel URL: https://github.com/dpryan79/MethylDackel</details>|
	|[metis](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|math|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[metis-64idx](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|-|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[migrate-n](http://popgen.sc.fsu.edu/Migrate/Migrate-n.html)|-|4.4.4|<details><summary>Expand</summary>Description: Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data. It uses a coalescent theory approach taking into account history of mutations and uncertainty of the genealogy. Homepage: http://popgen.sc.fsu.edu/Migrate/Migrate-n.html URL: http://popgen.sc.fsu.edu/Migrate/Migrate-n.html</details>|
	|[mii](https://github.com/codeandkey/mii)|-|1.1.1, 1.1.2|<details><summary>Expand</summary>Description: A smart search engine for module environments. Homepage: https://github.com/codeandkey/mii URL: https://github.com/codeandkey/mii</details>|
	|[minc-toolkit](http://bic-mni.github.io/)|bio|1.9.18.1, 1.9.18.3|<details><summary>Expand</summary>Description: Medical Imaging NetCDF Toolkit Homepage: http://bic-mni.github.io/ URL: http://bic-mni.github.io/</details>|
	|[minced](https://github.com/ctSkennerton/minced)|-|0.4.2|<details><summary>Expand</summary>Description: Mining CRISPRs in Environmental Datasets Homepage: https://github.com/ctSkennerton/minced URL: https://github.com/ctSkennerton/minced</details>|
	|[minia](https://github.com/GATB/gatb-core)|bio|3.2.6|<details><summary>Expand</summary>Description: GATB - The Genome Analysis Toolbox with de-Bruijn graph. Homepage: https://github.com/GATB/gatb-core URL: https://github.com/GATB/gatb-core</details>|
	|[miniasm](https://github.com/lh3/minimap2)|bio|0.3-20191007|<details><summary>Expand</summary>Description: Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format. Different from mainstream assemblers, miniasm does not have a consensus step. It simply concatenates pieces of read sequences to generate the final unitig sequences. Thus the per-base error rate is similar to the raw input reads. Homepage: https://github.com/lh3/minimap2 URL: https://github.com/lh3/minimap2</details>|
	|[minimac3](http://genome.sph.umich.edu/wiki/Minimac3)|bio|2.0.1|<details><summary>Expand</summary>Description: Minimac3 is a lower memory and more computationally efficient implementation of the genotype imputation algorithms in minimac and minimac2. Minimac3 is designed to handle very large reference panels in a more computationally efficient way with no loss of accuracy. Homepage: http://genome.sph.umich.edu/wiki/Minimac3 URL: http://genome.sph.umich.edu/wiki/Minimac3</details>|
	|[minimac4](https://genome.sph.umich.edu/wiki/Minimac4)|-|1.0.2|<details><summary>Expand</summary>Description: Minimac4 is a latest version in the series of genotype imputation software - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010). Minimac4 is a lower memory and more computationally efficient implementation of the original algorithms with comparable imputation quality. Homepage: https://genome.sph.umich.edu/wiki/Minimac4 URL: https://genome.sph.umich.edu/wiki/Minimac4</details>|
	|[minimap2](https://github.com/lh3/minimap2)|bio|2.17, 2.18, 2.24, 2.26, 2.28|<details><summary>Expand</summary>Description: Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment. Homepage: https://github.com/lh3/minimap2 URL: https://github.com/lh3/minimap2</details>|
	|[minpath](https://omics.informatics.indiana.edu/MinPath)|-|1.6|<details><summary>Expand</summary>Description: MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological pathways for a query dataset. Homepage: https://omics.informatics.indiana.edu/MinPath URL: https://omics.informatics.indiana.edu/MinPath</details>|
	|[mixcr](http://mixcr.readthedocs.io/en/latest/quickstart.html)|bio|4.1.2, 4.7.0|<details><summary>Expand</summary>Description: MiXCR is a universal framework that processes big immunome data from raw sequences to quantitated clonotypes. MiXCR efficiently handles paired- and single-end reads, considers sequence quality, corrects PCR errors and identifies germline hypermutations. The software supports both partial- and full-length profiling and employs all available RNA or DNA information, including sequences upstream of V and downstream of J gene segments. Homepage: http://mixcr.readthedocs.io/en/latest/quickstart.html URL: http://mixcr.readthedocs.io/en/latest/quickstart.html</details>|
	|[mmg](https://www.mmgtools.org/)|-|5.7.3|<details><summary>Expand</summary>Description: Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries: the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation and generation of a triangulation from a set of points or from given boundary edges the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries. Homepage: https://www.mmgtools.org/ URL: https://www.mmgtools.org/</details>|
	|[mmseqs2](https://mmseqs.com)|-|15-6f452, 14-7e284, 13-45111|<details><summary>Expand</summary>Description: MMseqs2: ultra fast and sensitive search and clustering suite Homepage: https://mmseqs.com URL: https://mmseqs.com</details>|
	|[modflow](https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs)|-|6.5.0|<details><summary>Expand</summary>Description: MODFLOW is the USGS's modular hydrologic model. MODFLOW is considered an international standard for simulating and predicting groundwater conditions and groundwater/surface-water interactions. Homepage: https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs URL: https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs</details>|
	|[molden](http://www.cmbi.ru.nl/molden/)|chem|6.5|<details><summary>Expand</summary>Description: Molden is a package for displaying Molecular Density from the Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical packages Mopac/Ampac Homepage: http://www.cmbi.ru.nl/molden/ URL: http://www.cmbi.ru.nl/molden/</details>|
	|[mono](https://www.mono-project.com/)|tools|6.12.0.122|<details><summary>Expand</summary>Description: An open source, cross-platform, implementation of C# and the CLR that is binary compatible with Microsoft.NET. Homepage: https://www.mono-project.com/ URL: https://www.mono-project.com/</details>|
	|[mothur](http://www.mothur.org/)|bio|1.44.3, 1.46.1, 1.47.0, 1.48.0|<details><summary>Expand</summary>Description: Mothur is a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community. Homepage: http://www.mothur.org/ URL: http://www.mothur.org/</details>|
	|[mpas](https://mpas-dev.github.io)|-|7.0|<details><summary>Expand</summary>Description: The Model for Prediction Across Scales (MPAS) is a collaborative project for developing atmosphere, ocean and other earth-system simulation components for use in climate, regional climate and weather studies. Homepage: https://mpas-dev.github.io URL: https://mpas-dev.github.io</details>|
	|[mpb](https://mpb.readthedocs.io/en/latest)|-|1.11.1|<details><summary>Expand</summary>Description: MPB is a software package to compute definite-frequency eigenstates of Maxwell's equations in periodic dielectric structures. It can compute optical dispersion relations and eigenstates for structures such as strip waveguides and optical fibers. MPB is well suited for the study of photonic crystals: periodic dielectric structures exhibiting a band gap in their optical modes, prohibiting propagation of light in that frequency range. Homepage: https://mpb.readthedocs.io/en/latest URL: https://mpb.readthedocs.io/en/latest</details>|
	|[mpb-mpi](https://mpb.readthedocs.io/en/latest)|-|1.11.1|<details><summary>Expand</summary>Description: MPB is a software package to compute definite-frequency eigenstates of Maxwell's equations in periodic dielectric structures. It can compute optical dispersion relations and eigenstates for structures such as strip waveguides and optical fibers. MPB is well suited for the study of photonic crystals: periodic dielectric structures exhibiting a band gap in their optical modes, prohibiting propagation of light in that frequency range. Homepage: https://mpb.readthedocs.io/en/latest URL: https://mpb.readthedocs.io/en/latest</details>|
	|[mpi4py](https://github.com/mpi4py/mpi4py)|tools|3.0.3, 3.1.2, 3.1.3, 3.1.4, 3.1.6, 4.0.0, 4.0.3|<details><summary>Expand</summary>Description: MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for the Python programming language, allowing any Python program to exploit multiple processors. Homepage: https://github.com/mpi4py/mpi4py URL: https://github.com/mpi4py/mpi4py</details>|
	|[mpsolve](https://numpi.dm.unipi.it/software/mpsolve)|-|3.2.1|<details><summary>Expand</summary>Description: MPSolve is a C package to solve polynomials and secular equations. It released under the terms of the GNU General public license as it specified in the COPYING file inside the source directory. Homepage: https://numpi.dm.unipi.it/software/mpsolve URL: https://numpi.dm.unipi.it/software/mpsolve</details>|
	|[mrbayes](https://nbisweden.github.io/MrBayes/)|bio|3.2.7|<details><summary>Expand</summary>Description: MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. Homepage: https://nbisweden.github.io/MrBayes/ URL: https://nbisweden.github.io/MrBayes/</details>|
	|[mrcc](https://www.mrcc.hu/index.php)|-|20230828|<details><summary>Expand</summary>Description: MRCC is a suite of ab initio and density functional quantum chemistry programs for high-accuracy electronic structure calculations developed and maintained by the quantum chemistry research group at the Department of Physical Chemistry and Materials Science, TU Budapest. Its special feature, the use of automated programming tools enabled the development of tensor manipulation routines which are independent of the number of indices of the corresponding tensors, thus significantly simplifying the general implementation of quantum chemical methods. Homepage: https://www.mrcc.hu/index.php URL: https://www.mrcc.hu/index.php</details>|
	|[mrtrix](http://www.brain.org.au/software/index.html#mrtrix)|bio|3.0.1, 3.0.4|<details><summary>Expand</summary>Description: MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines. Homepage: http://www.brain.org.au/software/index.html#mrtrix URL: http://www.brain.org.au/software/index.html#mrtrix</details>|
	|[mscg](https://github.com/uchicago-voth/MSCG-release)|chem|1.7.3.1|<details><summary>Expand</summary>Description: The Multiscale Coarse-Graining (MS-CG) method is a variational force-matching technique developed by the Voth group. This project serves as the workhorse implementation of the method used as a standard force-matching code. The development of MS-CG is supported by academic research grants. If you use this package, please cite at least one of these papers: - S. Izvekov and G. A. Voth, J. Chem. Phys. 123, 134105 (2005). - W. G. Noid, et al., J. Chem. Phys. 128, 244114 (2008). - W. G. Noid et al., J. Chem. Phys. 128, 244115 (2008). - L. Lu, S. Izvekov, A. Das, H. C. Andersen, and G. A. Voth, Journal of Chemical Theory and Computation, 6(3), 954-965 (2010). Homepage: https://github.com/uchicago-voth/MSCG-release URL: https://github.com/uchicago-voth/MSCG-release</details>|
	|[msmc2](https://github.com/stschiff/msmc)|-|2.1.3|<details><summary>Expand</summary>Description: This program implements MSMC2, a method to infer population size history and population separation history from whole genome sequencing data. For a general guide, please refer to version 1: https://github.com/stschiff/msmc Binaries are attached to the releases (under the "Releases" tab within github). Homepage: https://github.com/stschiff/msmc URL: https://github.com/stschiff/msmc</details>|
	|[mstore](https://github.com/grimme-lab/mstore)|-|0.2.0, 0.3.0|<details><summary>Expand</summary>Description: Molecular structure store for testing Homepage: https://github.com/grimme-lab/mstore URL: https://github.com/grimme-lab/mstore</details>|
	|[mujoco](https://mujoco.org/)|-|2.2.0, 2.2.2, 2.3.6, 3.0.1, 3.1.6, 3.3.0|<details><summary>Expand</summary>Description: MuJoCo stands for Multi-Joint dynamics with Contact. It is a general purpose physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, machine learning, and other areas which demand fast and accurate simulation of articulated structures interacting with their environment. Homepage: https://mujoco.org/ URL: https://mujoco.org/</details>|
	|[multicharge](https://github.com/grimme-lab/multicharge)|-|0.3.0|<details><summary>Expand</summary>Description: Electronegativity equilibration model for atomic partial charges. Homepage: https://github.com/grimme-lab/multicharge URL: https://github.com/grimme-lab/multicharge</details>|
	|[multichoose](https://github.com/ekg/multichoose)|-|1.0.3|<details><summary>Expand</summary>Description: generate multiset combinations (n multichoose k). Homepage: https://github.com/ekg/multichoose URL: https://github.com/ekg/multichoose</details>|
	|[mummer](https://mummer.sourceforge.net/)|bio|4.0.0beta2, 4.0.0rc1|<details><summary>Expand</summary>Description: MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it. Homepage: https://mummer.sourceforge.net/ URL: https://mummer.sourceforge.net/</details>|
	|[mumps-metis](https://graal.ens-lyon.fr/MUMPS/)|tools|5.2.1|<details><summary>Expand</summary>Description: A parallel sparse direct solver Homepage: https://graal.ens-lyon.fr/MUMPS/ URL: https://graal.ens-lyon.fr/MUMPS/</details>|
	|[mumps-parmetis](http://graal.ens-lyon.fr/MUMPS/)|tools|5.3.5|<details><summary>Expand</summary>Description: A parallel sparse direct solver Homepage: http://graal.ens-lyon.fr/MUMPS/ URL: http://graal.ens-lyon.fr/MUMPS/</details>|
	|[muparser](http://beltoforion.de/article.php?a=muparser)|math|2.2.5, 2.3.2, 2.3.4|<details><summary>Expand</summary>Description: muParser is an extensible high performance math expression parser library written in C++. It works by transforming a mathematical expression into bytecode and precalculating constant parts of the expression. Homepage: http://beltoforion.de/article.php?a=muparser URL: http://beltoforion.de/article.php?a=muparser</details>|
	|[muparserx](https://www.google.ca/search?q=muparserx)|-|4.0.12|<details><summary>Expand</summary>Description: A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more. Homepage: beltoforion.de/en/muparserx URL: beltoforion.de/en/muparserx</details>|
	|[muscle](https://drive5.com/muscle/)|bio|3.8.1551|<details><summary>Expand</summary>Description: MUSCLE is one of the best-performing multiple alignment programs according to published benchmark tests, with accuracy and speed that are consistently better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users learn everything they need to know about MUSCLE in a few minutes-only a handful of command-line options are needed to perform common alignment tasks. Homepage: https://drive5.com/muscle/ URL: https://drive5.com/muscle/</details>|
	|[mustang](http://www.csse.monash.edu.au/~karun/Site/mustang.html)|-|3.2.4|<details><summary>Expand</summary>Description: MUSTANG (MUltiple STructural AligNment AlGorithm), for the alignment of multiple protein structures. Given a set of protein structures, the program constructs a multiple alignment using the spatial information of the C atoms in the set. Homepage: http://www.csse.monash.edu.au/~karun/Site/mustang.html URL: http://www.csse.monash.edu.au/~karun/Site/mustang.html</details>|
	|[mysql](http://www.mysql.com/)|tools|5.7.36, 8.3.0|<details><summary>Expand</summary>Description: MySQL is one of the world's most widely used open-source relational database management system (RDBMS). Homepage: http://www.mysql.com/ URL: http://www.mysql.com/</details>|
	|[n2p2](https://compphysvienna.github.io/n2p2/)|-|2.2.0|<details><summary>Expand</summary>Description: A neural network potential package: high-dimensional neural network potentials in computational physics and chemistry. Homepage: https://compphysvienna.github.io/n2p2/ URL: https://compphysvienna.github.io/n2p2/</details>|
	|[namd-multicore](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14, 3.0.1|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ofi](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ofi-smp](http://www.ks.uiuc.edu/Research/namd/)|-|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ucx](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14, 3.0.1|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ucx-smp](http://www.ks.uiuc.edu/Research/namd/)|-|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[nanoflann](https://github.com/jlblancoc/nanoflann)|-|1.3.2|<details><summary>Expand</summary>Description: nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies Homepage: https://github.com/jlblancoc/nanoflann URL: https://github.com/jlblancoc/nanoflann</details>|
	|[nanopolish](https://github.com/jts/nanopolish)|bio|0.11.2, 0.13.2|<details><summary>Expand</summary>Description: Software package for signal-level analysis of Oxford Nanopore sequencing data. Nanopolish can calculate an improved consensus sequence for a draft genome assembly, detect base modifications, call SNPs and indels with respect to a reference genome and more. Homepage: https://github.com/jts/nanopolish URL: https://github.com/jts/nanopolish</details>|
	|[nauty](http://pallini.di.uniroma1.it/)|-|2.6r12|<details><summary>Expand</summary>Description: nauty and Traces are programs for computing automorphism groups of graphs and digraphs. They can also produce a canonical label. Homepage: http://pallini.di.uniroma1.it/ URL: http://pallini.di.uniroma1.it/</details>|
	|[nccl](https://developer.nvidia.com/nccl)|tools|2.7.8, 2.8.4, 2.11.4, 2.12.12, 2.18.3, 2.18.5, 2.22.3, 2.26.2|<details><summary>Expand</summary>Description: The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs. Homepage: https://developer.nvidia.com/nccl URL: https://developer.nvidia.com/nccl</details>|
	|[nciplot](https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html)|-|4.0-20200624|<details><summary>Expand</summary>Description: NCIPLOT is a program for revealing non covalent interactions based on the reduced density gradient. Homepage: https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html URL: https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html</details>|
	|[ncl](https://www.ncl.ucar.edu)|vis|6.6.2|<details><summary>Expand</summary>Description: NCL is an interpreted language designed specifically for scientific data analysis and visualization. Homepage: https://www.ncl.ucar.edu URL: https://www.ncl.ucar.edu</details>|
	|[nco](https://nco.sourceforge.net)|io|4.9.5, 5.0.6, 5.1.7|<details><summary>Expand</summary>Description: manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5 Homepage: https://nco.sourceforge.net URL: https://nco.sourceforge.net</details>|
	|[ncview](http://meteora.ucsd.edu/~pierce/ncview_home_page.html)|vis|2.1.8|<details><summary>Expand</summary>Description: Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc. Homepage: http://meteora.ucsd.edu/~pierce/ncview_home_page.html URL: http://meteora.ucsd.edu/~pierce/ncview_home_page.html</details>|
	|[nektar++](https://www.nektar.info/)|math|5.0.1, 5.3.0, 5.7.0|<details><summary>Expand</summary>Description: Nektar++ is a tensor product based finite element package designed to allow one to construct efficient classical low polynomial order h-type solvers (where h is the size of the finite element) as well as higher p-order piecewise polynomial order solvers. Homepage: https://www.nektar.info/ URL: https://www.nektar.info/</details>|
	|[netcdf](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0, 4.9.0, 4.9.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++-mpi](http://www.unidata.ucar.edu/software/netcdf/)|io|4.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: http://www.unidata.ucar.edu/software/netcdf/ URL: http://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++4](https://www.unidata.ucar.edu/software/netcdf/)|io|4.3.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++4-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.3.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3, 4.6.0, 4.6.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3, 4.6.0, 4.6.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0, 4.8.1, 4.9.0, 4.9.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[neuron](http://www.neuron.yale.edu/neuron)|bio|7.8.2, 8.0.0, 8.2.3|<details><summary>Expand</summary>Description: Empirically-based simulations of neurons and networks of neurons. Homepage: http://www.neuron.yale.edu/neuron URL: http://www.neuron.yale.edu/neuron</details>|
	|[newmat](http://www.robertnz.net/index.html)|-|11|<details><summary>Expand</summary>Description: Newmat C++ library is intended for scientists and engineers who need to manipulate a variety of types of matrices using standard matrix operations. Emphasis is on the kind of operations needed in statistical calculations such as least squares, linear equation solve and eigenvalues. Homepage: http://www.robertnz.net/index.html URL: http://www.robertnz.net/index.html</details>|
	|[nextdenovo](https://github.com/Nextomics/NextDenovo)|-|2.5.2|<details><summary>Expand</summary>Description: NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio HiFi reads), but requires significantly less computing resources and storages. After assembly, the per-base accuracy is about 98-99.8%, to further improve single base accuracy, try NextPolish. Homepage: https://github.com/Nextomics/NextDenovo URL: https://github.com/Nextomics/NextDenovo</details>|
	|[nextflow](https://www.nextflow.io/)|-|20.10.0, 21.04.3, 21.10.3, 22.04.3, 22.10.6, 22.10.8, 23.04.3, 23.10.0, 24.04.4|<details><summary>Expand</summary>Description: Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data Homepage: https://www.nextflow.io/ URL: https://www.nextflow.io/</details>|
	|[nextgenmap](http://cibiv.github.io/NextGenMap/)|bio|0.5.5|<details><summary>Expand</summary>Description: NextGenMap is a flexible highly sensitive short read mapping tool that handles much higher mismatch rates than comparable algorithms while still outperforming them in terms of runtime. Homepage: http://cibiv.github.io/NextGenMap/ URL: http://cibiv.github.io/NextGenMap/</details>|
	|[nextstrain.cli](https://github.com/nextstrain/cli/tree/3.1.0)|-|3.1.0|<details><summary>Expand</summary>Description: This is the source code repository for a program called nextstrain, the Nextstrain command-line interface (CLI). It aims to provide a consistent way to run and visualize pathogen builds and access Nextstrain components like Augur and Auspice across computing environments such as Docker, Conda, and AWS Batch. Homepage: https://github.com/nextstrain/cli/tree/3.1.0 URL: https://github.com/nextstrain/cli/tree/3.1.0</details>|
	|[ngmlr](https://github.com/philres/ngmlr)|-|0.2.7|<details><summary>Expand</summary>Description: Ngmlr is a long-read mapper designed to align PacBilo or Oxford Nanopore to a reference genome with a focus on reads that span structural variations. Homepage: https://github.com/philres/ngmlr URL: https://github.com/philres/ngmlr</details>|
	|[ngs](https://github.com/ncbi/ngs)|-|2.10.5|<details><summary>Expand</summary>Description: NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from Next Generation Sequencing. Homepage: https://github.com/ncbi/ngs URL: https://github.com/ncbi/ngs</details>|
	|[ngt](https://github.com/yahoojapan/NGT)|-|2.0.13|<details><summary>Expand</summary>Description: NGT provides commands and a library for performing high-speed approximate nearest neighbor searches against a large volume of data in high dimensional vector data space (several ten to several thousand dimensions). Homepage: https://github.com/yahoojapan/NGT URL: https://github.com/yahoojapan/NGT</details>|
	|[nibabel](https://nipy.github.io/nibabel)|-|5.2.0|<details><summary>Expand</summary>Description: NiBabel provides read/write access to some common medical and neuroimaging file formats, including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI. Homepage: https://nipy.github.io/nibabel URL: https://nipy.github.io/nibabel</details>|
	|[ninja-phylogenetics](https://github.com/TravisWheelerLab/NINJA)|-|0.97-cluster_only|<details><summary>Expand</summary>Description: Nearly Infinite Neighbor Joining Application Compute correct neighbor-joining phylogenies for inputs of more than 10,000 sequences. This is a C++/SSE port of the original Java code described in: Wheeler, T.J. 2009. Large-scale neighbor-joining with NINJA. In S.L. Salzberg and T. Warnow (Eds.), Proceedings of the 9th Workshop on Algorithms in Bioinformatics. WABI 2009, pp. 375-389. Springer, Berlin. (LNCS webpage,preprint) The Java version of NINJA was the fastest available tool computing neighbor-joining phylogenies ( 10x faster than the fastest implemenation of the canonical neighbor-joining algorithm - QuickTree ) at the time of it's release. In addition to generating phylogenies, Ninja can be used to output pairwise distances using several common sequence distance measures, and cluster sequences using a nearest-neighbor approach. Homepage: https://github.com/TravisWheelerLab/NINJA URL: https://github.com/TravisWheelerLab/NINJA</details>|
	|[nlopt](https://nlopt.readthedocs.io/en/latest/)|math|2.6.1, 2.6.2, 2.7.0, 2.7.1, 2.9.1|<details><summary>Expand</summary>Description: NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms. Homepage: https://nlopt.readthedocs.io/en/latest/ URL: https://nlopt.readthedocs.io/en/latest/</details>|
	|[nodejs](https://nodejs.org)|tools|12.16.1, 15.2.1, 18.17.1, 20.16.0|<details><summary>Expand</summary>Description: Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices. Homepage: https://nodejs.org URL: https://nodejs.org</details>|
	|[nspr](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)|-|4.25|<details><summary>Expand</summary>Description: Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</details>|
	|[nss](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)|-|3.51|<details><summary>Expand</summary>Description: Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</details>|
	|[ntedit](https://github.com/bcgsc/ntEdit)|-|1.3.2|<details><summary>Expand</summary>Description: ntEdit is a fast and scalable genomics application for polishing genome assembly drafts. Homepage: https://github.com/bcgsc/ntEdit URL: https://github.com/bcgsc/ntEdit</details>|
	|[ntl](http://www.shoup.net/ntl/)|math|11.4.3|<details><summary>Expand</summary>Description: NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields. Homepage: http://www.shoup.net/ntl/ URL: http://www.shoup.net/ntl/</details>|
	|[nvbio](https://github.com/NVlabs/nvbio)|-|1.1.00.20200205|<details><summary>Expand</summary>Description: NVBIO is a library of reusable components designed by NVIDIA Corporation to accelerate bioinformatics applications using CUDA. Homepage: https://github.com/NVlabs/nvbio URL: https://github.com/NVlabs/nvbio</details>|
	|[nvhpc](https://developer.nvidia.com/hpc-sdk/)|tools|20.7, 22.1, 22.7, 23.7, 23.9, 25.1|<details><summary>Expand</summary>Description: C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI) Homepage: https://developer.nvidia.com/hpc-sdk/ URL: https://developer.nvidia.com/hpc-sdk/</details>|
	|[nwchem](http://www.nwchem-sw.org)|chem|7.0.2-p1, 6.8.1|<details><summary>Expand</summary>Description: NWChem aims to provide its users with computational chemistry tools that are scalable both in their ability to treat large scientific computational chemistry problems efficiently, and in their use of available parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters. NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties and relativity. Homepage: http://www.nwchem-sw.org URL: http://www.nwchem-sw.org</details>|
	|[oases](https://github.com/dzerbino/oases)|-|0.2.09|<details><summary>Expand</summary>Description: Oases is a de novo transcriptome assembler designed to produce transcripts from short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly. Homepage: https://github.com/dzerbino/oases URL: https://github.com/dzerbino/oases</details>|
	|[occt](https://www.opencascade.com/)|-|7.4.0, 7.7.1, 7.8.1|<details><summary>Expand</summary>Description: Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications. Homepage: https://www.opencascade.com/ URL: https://www.opencascade.com/</details>|
	|[octave](https://www.gnu.org/software/octave/)|tools|5.2.0, 7.2.0|<details><summary>Expand</summary>Description: GNU Octave is a high-level interpreted language, primarily intended for numerical computations. Homepage: https://www.gnu.org/software/octave/ URL: https://www.gnu.org/software/octave/</details>|
	|[octomap](https://octomap.github.io/)|-|1.10.0|<details><summary>Expand</summary>Description: The OctoMap library implements a 3D occupancy grid mapping approach, providing data structures and mapping algorithms in C++ particularly suited for robotics. Homepage: https://octomap.github.io/ URL: https://octomap.github.io/</details>|
	|[octopus](http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page)|chem|10.1|<details><summary>Expand</summary>Description: Octopus is a scientific program aimed at the ab initio virtual experimentation on a hopefully ever-increasing range of system types. Electrons are described quantum-mechanically within density-functional theory (DFT), in its time-dependent form (TDDFT) when doing simulations in time. Nuclei are described classically as point particles. Electron-nucleus interaction is described within the pseudopotential approximation. Homepage: http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page URL: http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page</details>|
	|[octopus-bio](https://luntergroup.github.io/octopus/)|-|0.7.4|<details><summary>Expand</summary>Description: Octopus is a mapping-based variant caller that implements several calling models within a unified haplotype-aware framework. Octopus takes inspiration from particle filtering by constructing a tree of haplotypes and dynamically pruning and extending the tree based on haplotype posterior probabilities in a sequential manner. This allows octopus to implicitly consider all possible haplotypes at a given loci in reasonable time. Homepage: https://luntergroup.github.io/octopus/ URL: https://luntergroup.github.io/octopus/</details>|
	|[ogre](https://ogrecave.github.io/ogre/)|-|1.10.12, 1.12.12|<details><summary>Expand</summary>Description: OGRE (Object-Oriented Graphics Rendering Engine) is a scene-oriented, flexible 3D engine written in C++ designed to make it easier and more intuitive for developers to produce games and demos utilising 3D hardware. The class library abstracts all the details of using the underlying system libraries like Direct3D and OpenGL and provides an interface based on world objects and other intuitive classes. Homepage: https://ogrecave.github.io/ogre/ URL: https://ogrecave.github.io/ogre/</details>|
	|[oligoarrayaux](http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux)|-|3.8|<details><summary>Expand</summary>Description: OligoArrayAux is a subset of the UNAFold package for use with OligoArray. Homepage: http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux URL: http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux</details>|
	|[one-dnn](https://github.com/oneapi-src/oneDNN)|-|1.8, 2.4.3|<details><summary>Expand</summary>Description: Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) ONE-DNN) Homepage: https://github.com/oneapi-src/oneDNN URL: https://github.com/oneapi-src/oneDNN</details>|
	|[opa-psm2](https://github.com/cornelisnetworks/opa-psm2/)|-|11.2.185, 11.2.206|<details><summary>Expand</summary>Description: Low-level user-space communications interface for the Intel(R) OPA family of products. Homepage: https://github.com/cornelisnetworks/opa-psm2/ URL: https://github.com/cornelisnetworks/opa-psm2/</details>|
	|[opari2](https://www.score-p.org)|-|2.0.5|<details><summary>Expand</summary>Description: OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a source-to-source instrumentation tool for OpenMP and hybrid codes. It surrounds OpenMP directives and runtime library calls with calls to the POMP2 measurement interface. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[openbabel](https://openbabel.org)|chem|3.1.1|<details><summary>Expand</summary>Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas. Homepage: https://openbabel.org URL: https://openbabel.org</details>|
	|[openbabel-omp](https://openbabel.org)|chem|3.1.1|<details><summary>Expand</summary>Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas. Homepage: https://openbabel.org URL: https://openbabel.org</details>|
	|[openblas](https://www.openblas.net/)|math|0.3.9, 0.3.17, 0.3.18, 0.3.20, 0.3.24, 0.3.27|<details><summary>Expand</summary>Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version. Homepage: https://www.openblas.net/ URL: https://www.openblas.net/</details>|
	|[opencarp](https://opencarp.org)|-|4.0|<details><summary>Expand</summary>Description: openCARP is an open cardiac electrophysiology simulator for in-silico experiments. Homepage: https://opencarp.org URL: https://opencarp.org</details>|
	|[opencascade](https://www.opencascade.com/open-cascade-technology/)|-|7.5.0, 7.5.2|<details><summary>Expand</summary>Description: Open CASCADE Technology (OCCT) is a world-known 3D modeling kernel, freely available in open source. It's a software development kit (SDK) intended for the development of applications dealing with 3D CAD data. OCCT includes a set of C++ class libraries providing services for 3D surface and solid modeling, visualization, data exchange and rapid application development. Homepage: https://www.opencascade.com/open-cascade-technology/ URL: https://www.opencascade.com/open-cascade-technology/</details>|
	|[opencoarrays](https://github.com/sourceryinstitute/opencoarrays)|-|2.9.2|<details><summary>Expand</summary>Description: OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran. Homepage: https://github.com/sourceryinstitute/opencoarrays URL: https://github.com/sourceryinstitute/opencoarrays</details>|
	|[opencolorio](http://opencolorio.org/)|vis|1.1.1, 2.2.1, 2.3.0|<details><summary>Expand</summary>Description: OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production with an emphasis on visual effects and computer animation. Homepage: http://opencolorio.org/ URL: http://opencolorio.org/</details>|
	|[opencv](http://opencv.org/)|vis|4.4.0, 4.5.1, 4.5.5, 4.6.0, 4.7.0, 4.8.0, 4.8.1, 4.9.0, 4.10.0-2, 4.11.0|<details><summary>Expand</summary>Description: OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. OpenCV was built to provide a common infrastructure for computer vision applications and to accelerate the use of machine perception in the commercial products. Homepage: http://opencv.org/ URL: http://opencv.org/</details>|
	|[openexr](https://www.openexr.com/)|vis|2.5.2, 3.1.5|<details><summary>Expand</summary>Description: OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light and Magic for use in computer imaging applications Homepage: https://www.openexr.com/ URL: https://www.openexr.com/</details>|
	|[openfast](https://openfast.readthedocs.io/en/main/index.html)|-|3.1.0, 3.3.0, 3.5.0|<details><summary>Expand</summary>Description: OpenFAST is a multi-physics, multi-fidelity tool for simulating the coupled dynamic response of wind turbines. Practically speaking, OpenFAST is the framework (or glue code) that couples computational modules for aerodynamics, hydrodynamics for offshore structures, control and electrical system (servo) dynamics, and structural dynamics to enable coupled nonlinear aero-hydro-servo-elastic simulation in the time domain. OpenFAST enables the analysis of a range of wind turbine configurations, including two- or three-blade horizontal-axis rotor, pitch or stall regulation, rigid or teetering hub, upwind or downwind rotor, and lattice or tubular tower. The wind turbine can be modeled on land or offshore on fixed-bottom or floating substructures. Homepage: https://openfast.readthedocs.io/en/main/index.html URL: https://openfast.readthedocs.io/en/main/index.html</details>|
	|[openfoam](https://www.openfoam.com/)|phys|6, 8, 9, 10, 11, v2006, v2012, v2112, v2206, v2212, v2306, v2312, v2406|<details><summary>Expand</summary>Description: OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. Homepage: https://www.openfoam.com/ URL: https://www.openfoam.com/</details>|
	|[openfoam-extend](http://www.extend-project.de/)|phys|4.1|<details><summary>Expand</summary>Description: OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. Homepage: http://www.extend-project.de/ URL: http://www.extend-project.de/</details>|
	|[openimagedenoise](https://github.com/OpenImageDenoise/oidn)|-|1.2.2, 1.4.3, 2.0.1, 2.1.0|<details><summary>Expand</summary>Description: Intel Open Image Denoise is an open source library of high-performance, high-quality denoising filters for images rendered with ray tracing. Homepage: https://github.com/OpenImageDenoise/oidn URL: https://github.com/OpenImageDenoise/oidn</details>|
	|[openimageio](https://openimageio.org/)|vis|2.1.17.0, 2.4.7.1, 2.4.14.0|<details><summary>Expand</summary>Description: OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities, and applications. Homepage: https://openimageio.org/ URL: https://openimageio.org/</details>|
	|[openmc](https://docs.openmc.org/en/stable/)|-|0.13.2, 0.13.3, 0.15.0|<details><summary>Expand</summary>Description: OpenMC is a community-developed Monte Carlo neutron and photon transport simulation code. It is capable of performing fixed source, k-eigenvalue, and subcritical multiplication calculations on models built using either a constructive solid geometry or CAD representation. OpenMC supports both continuous-energy and multigroup transport. The continuous-energy particle interaction data is based on a native HDF5 format that can be generated from ACE files produced by NJOY. Parallelism is enabled via a hybrid MPI and OpenMP programming model. Homepage: https://docs.openmc.org/en/stable/ URL: https://docs.openmc.org/en/stable/</details>|
	|[openmm](http://openmm.org)|chem|7.5.0, 7.6.0, 7.7.0, 8.0.0, 8.1.1, 8.2.0|<details><summary>Expand</summary>Description: OpenMM is a toolkit for molecular simulation. Homepage: http://openmm.org URL: http://openmm.org</details>|
	|[openmm-alphafold](http://openmm.org)|-|7.5.1|<details><summary>Expand</summary>Description: OpenMM is a toolkit for molecular simulation. Homepage: http://openmm.org URL: http://openmm.org</details>|
	|[openmm-plumed](https://github.com/openmm/openmm-plumed/)|-|1.0|<details><summary>Expand</summary>Description: This project provides a connection between OpenMM and PLUMED. It allows you to bias or analyze an OpenMM simulation based on collective variables. Homepage: https://github.com/openmm/openmm-plumed/ URL: https://github.com/openmm/openmm-plumed/</details>|
	|[openmolcas](https://gitlab.com/Molcas/OpenMolcas)|chem|20.10|<details><summary>Expand</summary>Description: OpenMolcas is a quantum chemistry software package Homepage: https://gitlab.com/Molcas/OpenMolcas URL: https://gitlab.com/Molcas/OpenMolcas</details>|
	|[openmpi](https://www.open-mpi.org/)|mpi|4.0.3, 4.0.5, 4.1.1, 4.1.4, 4.1.5, 5.0.3|<details><summary>Expand</summary>Description: The Open MPI Project is an open source MPI-3 implementation. Homepage: https://www.open-mpi.org/ URL: https://www.open-mpi.org/</details>|
	|[openmx](http://www.openmx-square.org/)|chem|3.9, 3.9.9|<details><summary>Expand</summary>Description: OpenMX (Open source package for Material eXplorer) is a software package for nano-scale material simulations based on density functional theories (DFT), norm-conserving pseudopotentials, and pseudo-atomic localized basis functions. The methods and algorithms used in OpenMX and their implementation are carefully designed for the realization of large-scale ab initio electronic structure calculations on parallel computers based on the MPI or MPI/OpenMP hybrid parallelism. Homepage: http://www.openmx-square.org/ URL: http://www.openmx-square.org/</details>|
	|[openpgm](https://code.google.com/p/openpgm/)|tools|5.2.122|<details><summary>Expand</summary>Description: OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol that enables receivers to detect loss, request retransmission of lost data, or notify an application of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible for ensuring all data is received, absolving the sender of reception responsibility. Homepage: https://code.google.com/p/openpgm/ URL: https://code.google.com/p/openpgm/</details>|
	|[openrefine](https://github.com/OpenRefine/OpenRefine)|-|3.4.1|<details><summary>Expand</summary>Description: OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it, and augment it with data coming from the web. Homepage: https://github.com/OpenRefine/OpenRefine URL: https://github.com/OpenRefine/OpenRefine</details>|
	|[opensees](http://opensees.berkeley.edu/)|-|3.2.0, 3.5.0|<details><summary>Expand</summary>Description: OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes. This version is built using DISTRIBUTED_MP platform setting. Homepage: http://opensees.berkeley.edu/ URL: http://opensees.berkeley.edu/</details>|
	|[opensim](https://simtk.org/projects/opensim)|-|4.5.1|<details><summary>Expand</summary>Description: OpenSim is a freely available, user extensible software system that lets users develop models of musculoskeletal structures and create dynamic simulations of movement. Homepage: https://simtk.org/projects/opensim URL: https://simtk.org/projects/opensim</details>|
	|[openslide](https://github.com/openslide/openslide)|vis|3.4.1, 4.0.0|<details><summary>Expand</summary>Description: C library for reading virtual slide images Homepage: https://github.com/openslide/openslide URL: https://github.com/openslide/openslide</details>|
	|[opensubdiv](https://github.com/PixarAnimationStudios/OpenSubdiv)|-|3.4.4|<details><summary>Expand</summary>Description: OpenSubdiv is a set of open source libraries that implement high performance subdivision surface (subdiv) evaluation on massively parallel CPU and GPU architectures. Homepage: https://github.com/PixarAnimationStudios/OpenSubdiv URL: https://github.com/PixarAnimationStudios/OpenSubdiv</details>|
	|[openvdb](https://github.com/AcademySoftwareFoundation/openvdb)|-|7.0.0, 10.0.1, 10.1.0|<details><summary>Expand</summary>Description: OpenVDB is an open source C++ library comprising a novel hierarchical data structure and a large suite of tools for the efficient storage and manipulation of sparse volumetric data discretized on three-dimensional grids. It was developed by DreamWorks Animation for use in volumetric applications typically encountered in feature film production. Homepage: https://github.com/AcademySoftwareFoundation/openvdb URL: https://github.com/AcademySoftwareFoundation/openvdb</details>|
	|[openvkl](https://github.com/OpenVKL/openvkl)|-|0.10.0, 1.3.1, 1.3.2|<details><summary>Expand</summary>Description: Intel Open Volume Kernel Library (Intel Open VKL) is a collection of high-performance volume computation kernels, developed at Intel. The target users of Open VKL are graphics application engineers who want to improve the performance of their volume rendering applications by leveraging Open VKL's performance-optimized kernels, which include volume traversal and sampling functionality for a variety of volumetric data formats. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Homepage: https://github.com/OpenVKL/openvkl URL: https://github.com/OpenVKL/openvkl</details>|
	|[opera-ms](https://github.com/CSB5/OPERA-MS)|-|0.9.0-20200802|<details><summary>Expand</summary>Description: OPERA-MS is a hybrid metagenomic assembler which combines the advantages of short and long-read technologies to provide high quality assemblies, addressing issues of low contiguity for short-read only assemblies, and low base-pair quality for long-read only assemblies. Homepage: https://github.com/CSB5/OPERA-MS URL: https://github.com/CSB5/OPERA-MS</details>|
	|[optix](https://developer.nvidia.com/optix)|-|6.5.0, 7.7.0, 8.0.0|<details><summary>Expand</summary>Description: OptiX is NVIDIA SDK for easy ray tracing performance. It provides a simple framework for accessing the GPU's massive ray tracing power using state-of-the-art GPU algorithms. Homepage: https://developer.nvidia.com/optix URL: https://developer.nvidia.com/optix</details>|
	|[orad](https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html)|-|2.6.1|<details><summary>Expand</summary>Description: DRAGEN ORA Decompression software decompresses *.fastq.ora files that have been compressed using DRAGEN ORA Compression software with the default human reference and with the additional supported references. Homepage: https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html URL: https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html</details>|
	|[orca](http://cec.mpg.de/forum/)|chem|4.2.1, 5.0.1, 5.0.2, 5.0.3, 5.0.4, 6.0.0, 6.0.1|<details><summary>Expand</summary>Description: ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects. Homepage: http://cec.mpg.de/forum/ URL: http://cec.mpg.de/forum/</details>|
	|[orthomcl](http://orthomcl.org/)|bio|2.0.9|<details><summary>Expand</summary>Description: OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences. Homepage: http://orthomcl.org/ URL: http://orthomcl.org/</details>|
	|[ospray](https://github.com/ospray/ospray)|-|1.8.5, 2.2.0, 2.10.0, 2.12.0|<details><summary>Expand</summary>Description: Intel OSPRay is an open source, scalable, and portable ray tracing engine for high-performance, high-fidelity visualization on Intel Architecture CPUs. Homepage: https://github.com/ospray/ospray URL: https://github.com/ospray/ospray</details>|
	|[osrm-backend](https://github.com/Project-OSRM/osrm-backend)|-|5.26.0|<details><summary>Expand</summary>Description: Open Source Routing Machine Homepage: https://github.com/Project-OSRM/osrm-backend URL: https://github.com/Project-OSRM/osrm-backend</details>|
	|[osu-micro-benchmarks](https://mvapich.cse.ohio-state.edu/benchmarks/)|tools|5.6.2, 5.6.3, 5.9, 7.1-1, 7.2, 7.4|<details><summary>Expand</summary>Description: OSU Micro-Benchmarks Homepage: https://mvapich.cse.ohio-state.edu/benchmarks/ URL: https://mvapich.cse.ohio-state.edu/benchmarks/</details>|
	|[otf2](https://www.score-p.org)|-|2.2|<details><summary>Expand</summary>Description: The Open Trace Format 2 is a highly scalable, memory efficient event trace data format plus support library. It is the new standard trace format for Scalasca, Vampir, and TAU and is open for other tools. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[ovito](https://www.ovito.org)|-|3.3.3|<details><summary>Expand</summary>Description: OVITO is a scientific visualization and analysis software for atomistic and particle simulation data. It helps scientists gain better insights into materials phenomena and physical processes. The program is freely available for all major platforms under an open source license. It has served in a growing number of computational simulation studies as a powerful tool to analyze, understand and illustrate simulation results. Homepage: https://www.ovito.org URL: https://www.ovito.org</details>|
	|[p4est](http://www.p4est.org/)|math|2.2, 2.3.2, 2.8.5, 2.8.6|<details><summary>Expand</summary>Description: The p4est software library enables the dynamic management of a collection of adaptive octrees, conveniently called a forest of octrees. p4est is designed to work in parallel and scales to hundreds of thousands of processor cores. It is free software released under GNU General Public Licence version 2, or (at your option) any later version. Homepage: http://www.p4est.org/ URL: http://www.p4est.org/</details>|
	|[packmol](http://m3g.iqm.unicamp.br/packmol)|chem|20.3.3|<details><summary>Expand</summary>Description: Packing Optimization for Molecular Dynamics Simulations Homepage: http://m3g.iqm.unicamp.br/packmol URL: http://m3g.iqm.unicamp.br/packmol</details>|
	|[pagmo](https://esa.github.io/pagmo2)|-|2.18.0|<details><summary>Expand</summary>Description: pagmo is a C++ scientific library for massively parallel optimization. Homepage: https://esa.github.io/pagmo2 URL: https://esa.github.io/pagmo2</details>|
	|[pairix](https://github.com/4dn-dcic/pairix)|-|0.3.7|<details><summary>Expand</summary>Description: 1D/2D indexing and querying on bgzipped text file with a pair of genomic coordinates. Homepage: https://github.com/4dn-dcic/pairix URL: https://github.com/4dn-dcic/pairix</details>|
	|[paml](http://abacus.gene.ucl.ac.uk/software/paml.html)|bio|4.9j, 4.10.7|<details><summary>Expand</summary>Description: PAML is a package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood. Homepage: http://abacus.gene.ucl.ac.uk/software/paml.html URL: http://abacus.gene.ucl.ac.uk/software/paml.html</details>|
	|[panther](https://www.google.ca/search?q=panther)|-|14.1|<details><summary>Expand</summary>Description: Panther is the database part of InterProScan software. It contains only data. Homepage: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/ URL: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/</details>|
	|[papi](https://icl.cs.utk.edu/projects/papi/)|tools|6.0.0|<details><summary>Expand</summary>Description: PAPI provides the tool designer and application engineer with a consistent interface and methodology for use of the performance counter hardware found in most major microprocessors. PAPI enables software engineers to see, in near real time, the relation between software performance and processor events. In addition Component PAPI provides access to a collection of components that expose performance measurement opportunites across the hardware and software stack. Homepage: https://icl.cs.utk.edu/projects/papi/ URL: https://icl.cs.utk.edu/projects/papi/</details>|
	|[parallelio](https://github.com/NCAR/ParallelIO)|-|2.5.4, 2.6.3|<details><summary>Expand</summary>Description: A high-level Parallel I/O Library for structured grid applications Homepage: https://github.com/NCAR/ParallelIO URL: https://github.com/NCAR/ParallelIO</details>|
	|[parasail](https://github.com/jeffdaily/parasail)|-|2.5, 2.6.2|<details><summary>Expand</summary>Description: parasail is a SIMD C (C99) library containing implementations of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global pairwise sequence alignment algorithms. Homepage: https://github.com/jeffdaily/parasail URL: https://github.com/jeffdaily/parasail</details>|
	|[paraview](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0, 5.11.2, 5.13.1|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[paraview-offscreen](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[paraview-offscreen-gpu](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[pari-gp](http://pari.math.u-bordeaux.fr)|-|2.13.0|<details><summary>Expand</summary>Description: PARI/GP is a widely used computer algebra system designed for fast computations in number theory (factorizations, algebraic number theory, elliptic curves...), but also contains a large number of other useful functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc., and a lot of transcendental functions. PARI is also available as a C library to allow for faster computations. Homepage: http://pari.math.u-bordeaux.fr URL: http://pari.math.u-bordeaux.fr</details>|
	|[parmetis](http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview)|math|4.0.3|<details><summary>Expand</summary>Description: ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview</details>|
	|[parmgridgen](http://www-users.cs.umn.edu/~moulitsa/software.html)|math|1.0|<details><summary>Expand</summary>Description: ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen, that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited for geometric multigrid methods. Homepage: http://www-users.cs.umn.edu/~moulitsa/software.html URL: http://www-users.cs.umn.edu/~moulitsa/software.html</details>|
	|[parsplice](https://gitlab.com/exaalt/parsplice/)|-|1.1|<details><summary>Expand</summary>Description: Parallel Trajectory Splicing AMD Code Homepage: https://gitlab.com/exaalt/parsplice/ URL: https://gitlab.com/exaalt/parsplice/</details>|
	|[pbbam](http://pbbam.readthedocs.io/)|bio|1.0.7, 1.7.0|<details><summary>Expand</summary>Description: The pbbam software package provides components to create, query, and edit PacBio BAM files and associated indices. Homepage: http://pbbam.readthedocs.io/ URL: http://pbbam.readthedocs.io/</details>|
	|[pbcopper](https://github.com/PacificBiosciences/pbcopper)|-|1.4.0, 1.9.0|<details><summary>Expand</summary>Description: The pbcopper library provides a suite of data structures, algorithms, and utilities for C++ applications. Homepage: https://github.com/PacificBiosciences/pbcopper URL: https://github.com/PacificBiosciences/pbcopper</details>|
	|[pbmm2](https://github.com/PacificBiosciences/pbmm2)|-|1.7.0|<details><summary>Expand</summary>Description: A minimap2 frontend for PacBio native data formats Homepage: https://github.com/PacificBiosciences/pbmm2 URL: https://github.com/PacificBiosciences/pbmm2</details>|
	|[pbsuite](https://sourceforge.net/p/pb-jelly/wiki/Home/)|bio|15.8.24|<details><summary>Expand</summary>Description: PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or long 454 reads in fasta format) to high-confidence draft assembles. Homepage: https://sourceforge.net/p/pb-jelly/wiki/Home/ URL: https://sourceforge.net/p/pb-jelly/wiki/Home/</details>|
	|[pcre](https://www.pcre.org/)|-|8.44|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[pcre2](https://www.pcre.org/)|-|10.34|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[pdal](https://pdal.io/)|-|2.7.1|<details><summary>Expand</summary>Description: PDAL is Point Data Abstraction Library. It is a C/C++ open source library and applications for translating and processing point cloud data. It is not limited to LiDAR data, although the focus and impetus for many of the tools in the library have their origins in LiDAR. Homepage: https://pdal.io/ URL: https://pdal.io/</details>|
	|[pdt](https://www.cs.uoregon.edu/research/pdt/)|-|3.25.1|<details><summary>Expand</summary>Description: Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program knowledge accessible to developers of static and dynamic analysis tools. PDT implements a standard program representation, the program database (PDB), that can be accessed in a uniform way through a class library supporting common PDB operations. Homepage: https://www.cs.uoregon.edu/research/pdt/ URL: https://www.cs.uoregon.edu/research/pdt/</details>|
	|[pear](https://cme.h-its.org/exelixis/web/software/pear/)|bio|0.9.11|<details><summary>Expand</summary>Description: PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. It is fully parallelized and can run with as low as just a few kilobytes of memory. Homepage: https://cme.h-its.org/exelixis/web/software/pear/ URL: https://cme.h-its.org/exelixis/web/software/pear/</details>|
	|[penncnv](https://penncnv.openbioinformatics.org/)|bio|1.0.5|<details><summary>Expand</summary>Description: A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format, it can also handle other types of SNP arrays and oligonucleotide arrays. Homepage: https://penncnv.openbioinformatics.org/ URL: https://penncnv.openbioinformatics.org/</details>|
	|[percolator](https://github.com/percolator)|bio|3.06|<details><summary>Expand</summary>Description: Semi-supervised learning for peptide identification from shotgun proteomics datasets Homepage: https://github.com/percolator URL: https://github.com/percolator</details>|
	|[perl](https://www.perl.org/)|tools|5.30.2, 5.36.1|<details><summary>Expand</summary>Description: Larry Wall's Practical Extraction and Report Language Includes a small selection of extra CPAN packages for core functionality. Homepage: https://www.perl.org/ URL: https://www.perl.org/</details>|
	|[petsc](http://www.mcs.anl.gov/petsc)|tools|3.12.4, 3.13.3, 3.13.6, 3.14.1, 3.15.0, 3.17.1, 3.19.2, 3.20.0, 3.21.6|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-64bits](http://www.mcs.anl.gov/petsc)|tools|3.17.1, 3.19.2, 3.21.6|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-complex](http://www.mcs.anl.gov/petsc)|tools|3.20.0|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-pardiso](http://www.mcs.anl.gov/petsc)|-|3.17.1|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. PETSc with the Intel MKL Pardiso and CPardiso solver. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-pardiso-64bits](http://www.mcs.anl.gov/petsc)|-|3.17.1|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. PETSc with the Intel MKL Pardiso and CPardiso solver, and 64 bits indices. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[pfamscan](http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/)|-|1.6|<details><summary>Expand</summary>Description: PfamScan is used to search a FASTA sequence against a library of Pfam HMM. Homepage: http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/ URL: http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</details>|
	|[pfft](https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en)|math|1.0.8-alpha|<details><summary>Expand</summary>Description: PFFT is a software library for computing massively parallel, fast Fourier transformations on distributed memory architectures. PFFT can be understood as a generalization of FFTW-MPI to multidimensional data decomposition. Homepage: https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en URL: https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en</details>|
	|[pflotran](https://www.pflotran.org/)|-|4.0, 5.0.0, 6.0.0|<details><summary>Expand</summary>Description: PFLOTRAN is an open source, state-of-the-art massively parallel subsurface flow and reactive transport code. PFLOTRAN solves a system of generally nonlinear partial differential equations describing multiphase, multicomponent and multiscale reactive flow and transport in porous materials. The code is designed to run on massively parallel computing architectures as well as workstations and laptops. Homepage: https://www.pflotran.org/ URL: https://www.pflotran.org/</details>|
	|[pftoolsv3](https://github.com/sib-swiss/pftools3)|-|3.2.12|<details><summary>Expand</summary>Description: A suite of tools to build and search generalized profiles (protein and DNA). Homepage: https://github.com/sib-swiss/pftools3 URL: https://github.com/sib-swiss/pftools3</details>|
	|[pgplot](https://sites.astro.caltech.edu/~tjp/pgplot/)|vis|5.2, 5.2.2|<details><summary>Expand</summary>Description: The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable, device-independent graphics package for making simple scientific graphs. It is intended for making graphical images of publication quality with minimum effort on the part of the user. For most applications, the program can be device-independent, and the output can be directed to the appropriate device at run time. Homepage: https://sites.astro.caltech.edu/~tjp/pgplot/ URL: https://sites.astro.caltech.edu/~tjp/pgplot/</details>|
	|[phast](http://compgen.cshl.edu/phast/)|-|1.6|<details><summary>Expand</summary>Description: PHAST is a freely available software package for comparative and evolutionary genomics. Homepage: http://compgen.cshl.edu/phast/ URL: http://compgen.cshl.edu/phast/</details>|
	|[phon](http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/)|-|1.48|<details><summary>Expand</summary>Description: The program PHON calculates force constant matrices and phonon frequencies in crystals. From the frequencies it also calculates various thermodynamic quantities, like the Helmholtz free energy, the entropy, the specific heat and the internal energy of the harmonic crystal. The procedure is based on the small displacement method, and can be used in combination with any program capable to calculate forces on the atoms of the crystal. Homepage: http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/ URL: http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/</details>|
	|[photospline](https://github.com/icecube/photospline)|-|2.2.1|<details><summary>Expand</summary>Description: Detector response to a high-energy physics process is often estimated by Monte Carlo simulation. For purposes of data analysis, the results of this simulation are typically stored in large multi-dimensional histograms, which can quickly become unwieldy in terms of size or numerically problematic due to unfilled bins or interpolation artifacts. Photospline is a library that uses the penalized spline technique to efficiently compute, store, and evaluate B-spline representations of such tables. Homepage: https://github.com/icecube/photospline URL: https://github.com/icecube/photospline</details>|
	|[phylip](http://evolution.genetics.washington.edu/phylip)|bio|3.698|<details><summary>Expand</summary>Description: PHYLIP is a free package of programs for inferring phylogenies. Homepage: http://evolution.genetics.washington.edu/phylip URL: http://evolution.genetics.washington.edu/phylip</details>|
	|[phylobayes](http://megasun.bch.umontreal.ca/People/lartillot/www/download.html)|bio|4.1c|<details><summary>Expand</summary>Description: A Bayesian software for phylogenetic reconstruction using mixture models Homepage: http://megasun.bch.umontreal.ca/People/lartillot/www/download.html URL: http://megasun.bch.umontreal.ca/People/lartillot/www/download.html</details>|
	|[phylobayes-mpi](https://github.com/bayesiancook/pbmpi)|bio|20180420, 20201026|<details><summary>Expand</summary>Description: A Bayesian software for phylogenetic reconstruction using mixture models Homepage: https://github.com/bayesiancook/pbmpi URL: https://github.com/bayesiancook/pbmpi</details>|
	|[phylokit](https://github.com/pranjalv123/phylokit)|-|1.0|<details><summary>Expand</summary>Description: C++ library for high performance phylogenetics Homepage: https://github.com/pranjalv123/phylokit URL: https://github.com/pranjalv123/phylokit</details>|
	|[phyml](https://github.com/stephaneguindon/phyml)|-|3.3.20190321|<details><summary>Expand</summary>Description: Phylogenetic estimation using (Maximum) Likelihood Homepage: https://github.com/stephaneguindon/phyml URL: https://github.com/stephaneguindon/phyml</details>|
	|[phyx](https://github.com/FePhyFoFum/phyx)|bio|1.1|<details><summary>Expand</summary>Description: phyx performs phylogenetics analyses on trees and sequences. Homepage: https://github.com/FePhyFoFum/phyx URL: https://github.com/FePhyFoFum/phyx</details>|
	|[picard](https://broadinstitute.github.io/picard/)|bio|2.23.3, 2.26.3, 3.1.0|<details><summary>Expand</summary>Description: A set of tools (in Java) for working with next generation sequencing data in the BAM format. Homepage: https://broadinstitute.github.io/picard/ URL: https://broadinstitute.github.io/picard/</details>|
	|[pilercr](https://www.drive5.com/pilercr/)|-|1.06|<details><summary>Expand</summary>Description: PILERCR is public domain software for finding CRISPR repeats. It is part of the PILER family of repeat analysis tools. Homepage: https://www.drive5.com/pilercr/ URL: https://www.drive5.com/pilercr/</details>|
	|[pilon](https://github.com/broadinstitute/pilon)|bio|1.23, 1.24|<details><summary>Expand</summary>Description: Pilon is an automated genome assembly improvement and variant detection tool Homepage: https://github.com/broadinstitute/pilon URL: https://github.com/broadinstitute/pilon</details>|
	|[pindel](http://gmt.genome.wustl.edu/packages/pindel/)|bio|0.2.5b9-20170508|<details><summary>Expand</summary>Description: Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data. It uses a pattern growth approach to identify the breakpoints of these variants from paired-end short reads. Homepage: http://gmt.genome.wustl.edu/packages/pindel/ URL: http://gmt.genome.wustl.edu/packages/pindel/</details>|
	|[platanus](http://platanus.bio.titech.ac.jp/)|bio|1.2.4|<details><summary>Expand</summary>Description: PLATform for Assembling NUcleotide Sequences Homepage: http://platanus.bio.titech.ac.jp/ URL: http://platanus.bio.titech.ac.jp/</details>|
	|[platanus-allee](http://platanus.bio.titech.ac.jp/)|-|2.2.2|<details><summary>Expand</summary>Description: Platanus-allee is an assembler derived from Platanus assembler, however, it was developed with another concept. Platanus-allee tries to construct each haplotype sequence from the beginning and pair them as homologous chromosomes, while Platanus constructs consensus sequence of homologous chromosomes at first and tries to split into each haplotype sequence. Therefore, Platanus-allee marks better performance for highly heterozygous species genome or highly diverged genomic regions. Homepage: http://platanus.bio.titech.ac.jp/ URL: http://platanus.bio.titech.ac.jp/</details>|
	|[plink](https://www.cog-genomics.org/plink/2.0/)|bio|2.00a3.6, 1.9b_6.21-x86_64, 2.00a5.8, 2.00-10252019-avx2, 2.00-20231024-avx2|<details><summary>Expand</summary>Description: Whole-genome association analysis toolset Homepage: https://www.cog-genomics.org/plink/2.0/ URL: https://www.cog-genomics.org/plink/2.0/</details>|
	|[plplot](http://plplot.sourceforge.net)|-|5.15.0|<details><summary>Expand</summary>Description: PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer. Homepage: http://plplot.sourceforge.net URL: http://plplot.sourceforge.net</details>|
	|[plumed](https://www.plumed.org)|chem|2.6.1, 2.6.2, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.7.6, 2.8.1, 2.8.3, 2.8.4, 2.9.0, 2.9.2|<details><summary>Expand</summary>Description: PLUMED is an open source library for free energy calculations in molecular systems which works together with some of the most popular molecular dynamics engines. Free energy calculations can be performed as a function of many order parameters with a particular focus on biological problems, using state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD. The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. Homepage: https://www.plumed.org URL: https://www.plumed.org</details>|
	|[pluto](https://plutojl.org/)|-|0.14.7|<details><summary>Expand</summary>Description: A Pluto notebook is made up of small blocks of Julia code (cells) and together they form a reactive notebook. When you change a variable, Pluto automatically re-runs the cells that refer to it. Cells can even be placed in arbitrary order - intelligent syntax analysis figures out the dependencies between them and takes care of execution. Homepage: https://plutojl.org/ URL: https://plutojl.org/</details>|
	|[pmix](https://pmix.org/)|-|3.1.5, 3.2.3, 4.1.2, 4.2.4, 5.0.2|<details><summary>Expand</summary>Description: Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. Homepage: https://pmix.org/ URL: https://pmix.org/</details>|
	|[pnetcdf](https://trac.mcs.anl.gov/projects/parallel-netcdf)|io|1.9.0, 1.10.0, 1.12.2, 1.12.3|<details><summary>Expand</summary>Description: PnetCDF is a high-performance parallel I/O library for accessing files in format compatibility with Unidata's NetCDF, specifically the formats of CDF-1, 2, and 5. The CDF-5 file format, an extension of CDF-2, supports unsigned data types and uses 64-bit integers to allow users to define large dimensions, attributes, and variables (&gt; 2B array elements). Homepage: https://trac.mcs.anl.gov/projects/parallel-netcdf URL: https://trac.mcs.anl.gov/projects/parallel-netcdf</details>|
	|[podman](https://podman.io)|-|4.1.1, 4.5.0, 4.9.5|<details><summary>Expand</summary>Description: Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Homepage: https://podman.io URL: https://podman.io</details>|
	|[poplddecay](https://github.com/BGI-shenzhen/PopLDdecay)|-|3.41|<details><summary>Expand</summary>Description: PopLDdecay: a fast and effective tool for linkage disequilibrium decay analysis based on variant call format(VCF) files. Homepage: https://github.com/BGI-shenzhen/PopLDdecay URL: https://github.com/BGI-shenzhen/PopLDdecay</details>|
	|[popscle](https://github.com/statgen/popscle)|-|0.1-beta|<details><summary>Expand</summary>Description: A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools Homepage: https://github.com/statgen/popscle URL: https://github.com/statgen/popscle</details>|
	|[portaudio](http://www.portaudio.com/)|-|190600_20161030, 190700_20210406|<details><summary>Expand</summary>Description: PortAudio is a free, cross-platform, open-source, audio I/O library. It lets you write simple audio programs in 'C' or C++ that will compile and run on many platforms including Windows, Macintosh OS X, and Unix (OSS/ALSA). It is intended to promote the exchange of audio software between developers on different platforms. Many applications use PortAudio for Audio I/O. Homepage: http://www.portaudio.com/ URL: http://www.portaudio.com/</details>|
	|[postgresql](https://www.postgresql.org/)|tools|12.4, 13.2, 14.2, 16.0|<details><summary>Expand</summary>Description: PostgreSQL is a powerful, open source object-relational database system. It is fully ACID compliant, has full support for foreign keys, joins, views, triggers, and stored procedures (in multiple languages). It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP. It also supports storage of binary large objects, including pictures, sounds, or video. It has native programming interfaces for C/C++, Java, .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation. Homepage: https://www.postgresql.org/ URL: https://www.postgresql.org/</details>|
	|[pov-ray](https://www.povray.org/)|-|3.8.0-x.10064738, 3.7.0.10|<details><summary>Expand</summary>Description: The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program which generates images from a text-based scene description, and is available for a variety of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning three-dimensional graphics. The source code is available for those wanting to do their own ports. Homepage: https://www.povray.org/ URL: https://www.povray.org/</details>|
	|[pplacer](http://matsen.fhcrc.org/pplacer/)|bio|1.1.alpha19|<details><summary>Expand</summary>Description: Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give useful information about uncertainty, and to offer advanced visualization and downstream analysis. Homepage: http://matsen.fhcrc.org/pplacer/ URL: http://matsen.fhcrc.org/pplacer/</details>|
	|[prank](http://wasabiapp.org/software/prank/)|bio|170427|<details><summary>Expand</summary>Description: PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences. PRANK is based on a novel algorithm that treats insertions correctly and avoids over-estimation of the number of deletion events. Homepage: http://wasabiapp.org/software/prank/ URL: http://wasabiapp.org/software/prank/</details>|
	|[primer3](https://primer3.org/)|-|2.5.0|<details><summary>Expand</summary>Description: Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction'). PCR is an essential and ubiquitous tool in genetics and molecular biology. Primer3 can also design hybridization probes and sequencing primers. Homepage: https://primer3.org/ URL: https://primer3.org/</details>|
	|[primme](http://www.cs.wm.edu/~andreas/software/)|-|3.2|<details><summary>Expand</summary>Description: PRIMME, pronounced as prime, is a high-performance library for computing a few eigenvalues/eigenvectors, and singular values/vectors. PRIMME is especially optimized for large, difficult problems. Real symmetric and complex Hermitian problems, standard A x = \lambda x and generalized A x = \lambda B x, are supported. Besides, standard eigenvalue problems with a normal matrix are supported. It can find largest, smallest, or interior singular/eigenvalues, and can use preconditioning to accelerate convergence. Homepage: http://www.cs.wm.edu/~andreas/software/ URL: http://www.cs.wm.edu/~andreas/software/</details>|
	|[prinseq](http://prinseq.sourceforge.net)|bio|0.20.4|<details><summary>Expand</summary>Description: A bioinformatics tool to PRe-process and show INformation of SEQuence data. Homepage: http://prinseq.sourceforge.net URL: http://prinseq.sourceforge.net</details>|
	|[prism](https://www.prismmodelchecker.org/)|-|4.7|<details><summary>Expand</summary>Description: PRISM is a probabilistic model checker, a tool for formal modelling and analysis of systems that exhibit random or probabilistic behaviour. It has been used to analyse systems from many different application domains, including communication and multimedia protocols, randomised distributed algorithms, security protocols, biological systems and many others. Homepage: https://www.prismmodelchecker.org/ URL: https://www.prismmodelchecker.org/</details>|
	|[prodigal](https://github.com/hyattpd/Prodigal/)|bio|2.6.3|<details><summary>Expand</summary>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee. Homepage: https://github.com/hyattpd/Prodigal/ URL: https://github.com/hyattpd/Prodigal/</details>|
	|[prodigal-gv](https://github.com/apcamargo/prodigal-gv)|-|2.6.3|<details><summary>Expand</summary>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee. Homepage: https://github.com/apcamargo/prodigal-gv URL: https://github.com/apcamargo/prodigal-gv</details>|
	|[proj](https://proj.org)|geo|4.9.3, 6.3.2, 7.0.0, 7.0.1, 7.2.1, 8.0.0, 9.0.0, 9.0.1, 9.2.0, 9.4.1|<details><summary>Expand</summary>Description: Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates Homepage: https://proj.org URL: https://proj.org</details>|
	|[proj4-fortran](https://github.com/mhagdorn/proj4-fortran)|geo|1.0|<details><summary>Expand</summary>Description: proj4-fortran is a f77 and f90 wrappers for proj4, a cartograohic projections library. Homepage: https://github.com/mhagdorn/proj4-fortran URL: https://github.com/mhagdorn/proj4-fortran</details>|
	|[prokka](https://www.vicbioinformatics.com/software.prokka.shtml)|-|1.14.5|<details><summary>Expand</summary>Description: Prokka is a software tool for the rapid annotation of prokaryotic genomes. Homepage: https://www.vicbioinformatics.com/software.prokka.shtml URL: https://www.vicbioinformatics.com/software.prokka.shtml</details>|
	|[protobuf](https://github.com/google/protobuf/)|tools|3.12.3, 3.19.4, 3.21.3, 24.4|<details><summary>Expand</summary>Description: Google Protocol Buffers Homepage: https://github.com/google/protobuf/ URL: https://github.com/google/protobuf/</details>|
	|[prrte](https://docs.prrte.org/)|-|3.0.5|<details><summary>Expand</summary>Description: PRRTE is the PMIx Reference RunTime Environment Homepage: https://docs.prrte.org/ URL: https://docs.prrte.org/</details>|
	|[psfex](https://psfex.readthedocs.io/en/latest/index.html)|-|3.21.1|<details><summary>Expand</summary>Description: A computer program that extracts precise models of the Point Spread Functions (PSFs) from images processed by SExtractor and measures the quality of images. The generated PSF models can be used for model-fitting photometry or morphological analyses. Homepage: https://psfex.readthedocs.io/en/latest/index.html URL: https://psfex.readthedocs.io/en/latest/index.html</details>|
	|[psi4](http://www.psicode.org/)|chem|1.3.2, 1.4, 1.5, 1.9, 1.9.1|<details><summary>Expand</summary>Description: PSI4 is an open-source suite of ab initio quantum chemistry programs designed for efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform computations with more than 2500 basis functions running serially or in parallel. Homepage: http://www.psicode.org/ URL: http://www.psicode.org/</details>|
	|[psipred](http://bioinf.cs.ucl.ac.uk)|-|4.02|<details><summary>Expand</summary>Description: Accurate protein secondary structure prediction Homepage: http://bioinf.cs.ucl.ac.uk URL: http://bioinf.cs.ucl.ac.uk</details>|
	|[psmc](https://github.com/lh3/psmc)|bio|0.6.5|<details><summary>Expand</summary>Description: This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model. Homepage: https://github.com/lh3/psmc URL: https://github.com/lh3/psmc</details>|
	|[ptex](https://ptex.us/)|-|2.3.2|<details><summary>Expand</summary>Description: Ptex is a texture mapping system developed by Walt Disney Animation Studios for production-quality rendering: Homepage: https://ptex.us/ URL: https://ptex.us/</details>|
	|[pullseq](https://github.com/bcthomas/pullseq)|-|1.0.2|<details><summary>Expand</summary>Description: Utility program for extracting sequences from a fasta/fastq file Homepage: https://github.com/bcthomas/pullseq URL: https://github.com/bcthomas/pullseq</details>|
	|[pyne](http://pyne.io/)|-|0.7.3, 0.7.7|<details><summary>Expand</summary>Description: PyNE is a suite of tools to aid in computational nuclear science and engineering. PyNE seeks to provide native implementations of common nuclear algorithms, as well as Python bindings and I/O support for other industry standard nuclear codes. Homepage: http://pyne.io/ URL: http://pyne.io/</details>|
	|[pypy](https://pypy.org/index.html)|tools|5.8.0, 7.3.3|<details><summary>Expand</summary>Description: PyPy is a fast, compliant alternative implementation of the Python language (2.7.13 and 3.5.3). Homepage: https://pypy.org/index.html URL: https://pypy.org/index.html</details>|
	|[pytest](https://pytest.org)|-|6.1.2, 6.2.5, 7.0.1, 7.4.0, 8.2.2|<details><summary>Expand</summary>Description: pytest: simple powerful testing with Python Homepage: https://pytest.org URL: https://pytest.org</details>|
	|[python](https://python.org/)|tools|2.7.18, 3.6.10, 3.7.7, 3.7.9, 3.8.2, 3.8.10, 3.9.6, 3.10.2, 3.10.13, 3.11.2, 3.11.5, 3.12.4, 3.13.2|<details><summary>Expand</summary>Description: Python is a programming language that lets you work more quickly and integrate your systems more effectively. Homepage: https://python.org/ URL: https://python.org/</details>|
	|[python-build-bundle](http://python.org/)|-|2022a, 2023a, 2023b, 2024a, 2025a|<details><summary>Expand</summary>Description: Bundle which contains Python packages useful for building Homepage: http://python.org/ URL: http://python.org/</details>|
	|[q6](https://github.com/qusers/Q6)|-|6.0|<details><summary>Expand</summary>Description: Q is a set of Molecular Dynamics (MD) tools tailored to the following specific kinds of free energy calculations: Free Energy Perturbation (FEP) Empirical Valence Bond (EVB) Linear Interaction Energies (LIE) Quantum Classical Path (QCP) Homepage: https://github.com/qusers/Q6 URL: https://github.com/qusers/Q6</details>|
	|[qca](https://userbase.kde.org/QCA)|geo|2.3.0, 2.3.5, 2.3.7|<details><summary>Expand</summary>Description: Taking a hint from the similarly-named Java Cryptography Architecture, QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes and conventions. QCA separates the API from the implementation, using plugins known as Providers. The advantage of this model is to allow applications to avoid linking to or explicitly depending on any particular cryptographic library. This allows one to easily change or upgrade crypto implementations without even needing to recompile the application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX. Homepage: https://userbase.kde.org/QCA URL: https://userbase.kde.org/QCA</details>|
	|[qcint](http://wiki.sunqm.net/libcint)|-|4.4.6, 5.1.5, 5.1.7, 5.1.8, 6.1.2|<details><summary>Expand</summary>Description: libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform. Homepage: http://wiki.sunqm.net/libcint URL: http://wiki.sunqm.net/libcint</details>|
	|[qctool](https://www.well.ox.ac.uk/~gav/qctool_v2/)|-|2.2.0|<details><summary>Expand</summary>Description: QCTOOL is a command-line utility program for manipulation and quality control of gwas datasets and other genome-wide data. Homepage: https://www.well.ox.ac.uk/~gav/qctool_v2/ URL: https://www.well.ox.ac.uk/~gav/qctool_v2/</details>|
	|[qgis](http://www.qgis.org/)|geo|3.10.6, 3.22.14, 3.28.12|<details><summary>Expand</summary>Description: QGIS is a user friendly Open Source Geographic Information System (GIS) Homepage: http://www.qgis.org/ URL: http://www.qgis.org/</details>|
	|[qhull](http://www.qhull.org)|math|2019.1, 2020.2|<details><summary>Expand</summary>Description: Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull. Homepage: http://www.qhull.org URL: http://www.qhull.org</details>|
	|[qiime2](http://qiime2.org/)|-|2021.11, 2023.5, 2024.5|<details><summary>Expand</summary>Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. Homepage: http://qiime2.org/ URL: http://qiime2.org/</details>|
	|[qjson](http://qjson.sourceforge.net/)|tools|0.9.0|<details><summary>Expand</summary>Description: QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa. Homepage: http://qjson.sourceforge.net/ URL: http://qjson.sourceforge.net/</details>|
	|[qrupdate](https://sourceforge.net/projects/qrupdate/)|math|1.1.2|<details><summary>Expand</summary>Description: qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions. Homepage: https://sourceforge.net/projects/qrupdate/ URL: https://sourceforge.net/projects/qrupdate/</details>|
	|[qscintilla](https://www.riverbankcomputing.com/software/qscintilla)|tools|2.11.2, 2.11.6, 2.14.1|<details><summary>Expand</summary>Description: QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control Homepage: https://www.riverbankcomputing.com/software/qscintilla URL: https://www.riverbankcomputing.com/software/qscintilla</details>|
	|[qt](https://qt.io/)|tools|5.12.8, 5.15.2, 5.15.8, 5.15.11, 6.5.3|<details><summary>Expand</summary>Description: Qt is a comprehensive cross-platform C++ application framework. Homepage: https://qt.io/ URL: https://qt.io/</details>|
	|[qt5webkit](https://github.com/qt/qtwebkit)|-|5.212.0-alpha4|<details><summary>Expand</summary>Description: Qt Port of WebKit. WebKit is an open source web browser engine. Homepage: https://github.com/qt/qtwebkit URL: https://github.com/qt/qtwebkit</details>|
	|[qtkeychain](https://github.com/frankosterfeld/qtkeychain)|-|0.9.1, 0.13.2, 0.14.1|<details><summary>Expand</summary>Description: Platform-independent Qt API for storing passwords securely. Homepage: https://github.com/frankosterfeld/qtkeychain URL: https://github.com/frankosterfeld/qtkeychain</details>|
	|[qtltools](https://qtltools.github.io/qtltools/)|-|1.3.1|<details><summary>Expand</summary>Description: QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) in few easy-to-perform steps. Homepage: https://qtltools.github.io/qtltools/ URL: https://qtltools.github.io/qtltools/</details>|
	|[qualimap](http://qualimap.bioinfo.cipf.es/)|-|2.3|<details><summary>Expand</summary>Description: Qualimap 2 is a platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data and its derivatives like feature counts. Homepage: http://qualimap.bioinfo.cipf.es/ URL: http://qualimap.bioinfo.cipf.es/</details>|
	|[quantumatk](https://www.synopsys.com/silicon/quantumatk.html)|-|2019.12, 2022.03, 2024.09|<details><summary>Expand</summary>Description: QuantumATK is a complete atomistic simulation toolkit. QuantumATK addresses key applications in the semiconductor industry and is a core component of the Synopsys Design-Technology Co-Optimization (DTCO) solution. Atomic-scale modeling tools in QuantumATK range from classical force fields for handling large and to some extent more realistic materials systems to ab initio tools that provide highly accurate results for smaller systems. Homepage: https://www.synopsys.com/silicon/quantumatk.html URL: https://www.synopsys.com/silicon/quantumatk.html</details>|
	|[quantumespresso](https://www.quantum-espresso.org)|chem|6.5, 6.6, 6.7, 6.8, 7.0, 7.1, 7.2, 7.3.1|<details><summary>Expand</summary>Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). Homepage: https://www.quantum-espresso.org URL: https://www.quantum-espresso.org</details>|
	|[quast](https://github.com/ablab/quast)|-|5.2.0|<details><summary>Expand</summary>Description: QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison. Homepage: https://github.com/ablab/quast URL: https://github.com/ablab/quast</details>|
	|[qwt](http://qwt.sourceforge.net/)|tools|6.1.4, 6.2.0|<details><summary>Expand</summary>Description: The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background. Homepage: http://qwt.sourceforge.net/ URL: http://qwt.sourceforge.net/</details>|
	|[qwtpolar](http://qwtpolar.sourceforge.net/)|tools|1.1.1|<details><summary>Expand</summary>Description: The QwtPolar library contains classes for displaying values on a polar coordinate system. Homepage: http://qwtpolar.sourceforge.net/ URL: http://qwtpolar.sourceforge.net/</details>|
	|[r](https://www.r-project.org/)|tools|4.0.0, 4.0.2, 4.0.5, 4.1.0, 4.1.2, 4.2.1, 4.2.2, 4.3.1, 4.4.0|<details><summary>Expand</summary>Description: R is a free software environment for statistical computing and graphics. Homepage: https://www.r-project.org/ URL: https://www.r-project.org/</details>|
	|[r-bundle-bioconductor](https://bioconductor.org)|bio|3.12, 3.14, 3.16, 3.17, 3.18, 3.20|<details><summary>Expand</summary>Description: Bioconductor provides tools for the analysis and coprehension of high-throughput genomic data. Homepage: https://bioconductor.org URL: https://bioconductor.org</details>|
	|[r2r](https://sourceforge.net/projects/weinberg-r2r/files/latest/download/)|-|1.0.6|<details><summary>Expand</summary>Description: Weinberg-R2R: Faster creation of aesthetic RNA secondary structure diagrams Homepage: https://sourceforge.net/projects/weinberg-r2r/files/latest/download/ URL: https://sourceforge.net/projects/weinberg-r2r/files/latest/download/</details>|
	|[racket](https://racket-lang.org/)|-|7.8|<details><summary>Expand</summary>Description: Racket, the Programming Language Homepage: https://racket-lang.org/ URL: https://racket-lang.org/</details>|
	|[racon](https://github.com/lbcb-sci/racon)|bio|1.4.3, 1.4.13, 1.5.0|<details><summary>Expand</summary>Description: Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. Homepage: https://github.com/lbcb-sci/racon URL: https://github.com/lbcb-sci/racon</details>|
	|[randomlib](http://randomlib.sourceforge.net/html/)|-|1.10|<details><summary>Expand</summary>Description: RandomLib is a C++ interface to the Mersenne Twister random number generator, MT19937 and to the SIMD-oriented Fast Mersenne Twister random number generator, SFMT19937. It provides convenient access to random integers and reals at a variety of precisions. RandomLib also contains new algorithms which permit exact sampling from the normal and discrete normal distributions (provided that the underlying generator is perfect). The emphasis in this implementation is on providing a reliable source of random numbers for scientific applications where there's a premium on accuracy, repeatability, portability, and ease of use. By default, this library uses SFMT's improved method for seeding the generator and it allows access to both the 32-bit and 64-bit versions of MT19937 and SFMT19937 and includes implementations of the SFMT19937 using SSE2 and AltiVec instructions. Homepage: http://randomlib.sourceforge.net/html/ URL: http://randomlib.sourceforge.net/html/</details>|
	|[rapidjson](https://rapidjson.org)|-|1.1.0|<details><summary>Expand</summary>Description: A fast JSON parser/generator for C++ with both SAX/DOM style API Homepage: https://rapidjson.org URL: https://rapidjson.org</details>|
	|[rapsearch2](https://omics.informatics.indiana.edu/mg/RAPSearch2)|-|2.24|<details><summary>Expand</summary>Description: RAPSearch stands for Reduced Alphabet based Protein similarity Search Homepage: https://omics.informatics.indiana.edu/mg/RAPSearch2 URL: https://omics.informatics.indiana.edu/mg/RAPSearch2</details>|
	|[raqm](https://github.com/HOST-Oman/libraqm)|-|0.9.0, 0.10.1|<details><summary>Expand</summary>Description: Raqm is a small library that encapsulates the logic for complex text layout and provides a convenient API. Homepage: https://github.com/HOST-Oman/libraqm URL: https://github.com/HOST-Oman/libraqm</details>|
	|[raspa2](https://iraspa.org/raspa/)|-|2.0.47|<details><summary>Expand</summary>Description: RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials. Homepage: https://iraspa.org/raspa/ URL: https://iraspa.org/raspa/</details>|
	|[raspa3](https://iraspa.org/raspa/)|-|3.0.3|<details><summary>Expand</summary>Description: RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials. Homepage: https://iraspa.org/raspa/ URL: https://iraspa.org/raspa/</details>|
	|[ratatosk](https://github.com/DecodeGenetics/Ratatosk)|-|0.7.6.3|<details><summary>Expand</summary>Description: Phased hybrid error correction of long reads using colored de Bruijn graphs Homepage: https://github.com/DecodeGenetics/Ratatosk URL: https://github.com/DecodeGenetics/Ratatosk</details>|
	|[raven](https://github.com/lbcb-sci/raven)|-|1.5.0|<details><summary>Expand</summary>Description: Raven is a de novo genome assembler for long uncorrected reads. Homepage: https://github.com/lbcb-sci/raven URL: https://github.com/lbcb-sci/raven</details>|
	|[raxml](https://github.com/stamatak/standard-RAxML)|bio|8.2.12|<details><summary>Expand</summary>Description: RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. Homepage: https://github.com/stamatak/standard-RAxML URL: https://github.com/stamatak/standard-RAxML</details>|
	|[raxml-ng](https://github.com/amkozlov/raxml-ng)|-|1.0.1, 1.2.0|<details><summary>Expand</summary>Description: RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree. Homepage: https://github.com/amkozlov/raxml-ng URL: https://github.com/amkozlov/raxml-ng</details>|
	|[raxml-pthreads](https://github.com/stamatak/standard-RAxML)|-|8.2.12|<details><summary>Expand</summary>Description: RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. Homepage: https://github.com/stamatak/standard-RAxML URL: https://github.com/stamatak/standard-RAxML</details>|
	|[ray](https://github.com/zorino/ray)|bio|3.0.1|<details><summary>Expand</summary>Description: Ray -- Parallel genome assemblies for parallel DNA sequencing. Homepage: https://github.com/zorino/ray URL: https://github.com/zorino/ray</details>|
	|[rcorrector](https://github.com/mourisl/Rcorrector)|-|1.0.4, 1.0.6|<details><summary>Expand</summary>Description: Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data. Homepage: https://github.com/mourisl/Rcorrector URL: https://github.com/mourisl/Rcorrector</details>|
	|[rdkit](http://www.rdkit.org/)|chem|2019.03.4, 2020.09.1b1, 2021.03.3, 2021.09.3, 2021.09.4, 2022.09.4, 2023.09.3, 2023.09.5, 2024.03.4, 2024.03.5, 2024.09.6|<details><summary>Expand</summary>Description: RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python. Homepage: http://www.rdkit.org/ URL: http://www.rdkit.org/</details>|
	|[re2c](https://re2c.org/)|-|1.3|<details><summary>Expand</summary>Description: re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons. Homepage: https://re2c.org/ URL: https://re2c.org/</details>|
	|[recon](http://eddylab.org/software/recon/)|bio|1.08|<details><summary>Expand</summary>Description: Proper identification of repetitive sequences is an essential step in genome analysis. The RECON package performs de novo identification and classification of repeat sequence families from genomic sequences. The underlying algorithm is based on extensions to the usual approach of single linkage clustering of local pairwise alignments between genomic sequences. Specifically, our extensions use multiple alignment information to define the boundaries of individual copies of the repeats and to distinguish homologous but distinct repeat element families. RECON should be useful for first-pass automatic classification of repeats in newly sequenced genomes. Homepage: http://eddylab.org/software/recon/ URL: http://eddylab.org/software/recon/</details>|
	|[reduce](https://github.com/rlabduke/reduce)|-|4.14|<details><summary>Expand</summary>Description: Reduce - tool for adding and correcting hydrogens in PDB files Homepage: https://github.com/rlabduke/reduce URL: https://github.com/rlabduke/reduce</details>|
	|[reframe](https://github.com/eth-cscs/reframe)|-|3.12.0|<details><summary>Expand</summary>Description: ReFrame is a framework for writing regression tests for HPC systems. Homepage: https://github.com/eth-cscs/reframe URL: https://github.com/eth-cscs/reframe</details>|
	|[regenie](https://doi.org/10.1038/s41588-021-00870-7)|-|3.2.1|<details><summary>Expand</summary>Description: regenie is a C++ program for whole genome regression modelling of large genome-wide association studies. It is developed and supported by a team of scientists at the Regeneron Genetics Center. The method has the following properties It works on quantitative and binary traits, including binary traits with unbalanced case-control ratios It can process multiple phenotypes at once For binary traits it supports Firth logistic regression and an SPA test It can perform gene/region-based burden tests It is fast and memory efficient It supports the BGEN, PLINK bed/bim/fam and PLINK2 pgen/pvar/psam genetic data formats It is ideally suited for implementation in Apache Spark (see GLOW) Citation Mbatchou, J., Barnard, L., Backman, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. Nat Genet 53, 10971103 (2021). https://doi.org/10.1038/s41588-021-00870-7 Homepage: https://rgcgithub.github.io/regenie/ URL: https://rgcgithub.github.io/regenie/</details>|
	|[regtools](https://regtools.readthedocs.org)|-|1.0.0|<details><summary>Expand</summary>Description: RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context. Homepage: https://regtools.readthedocs.org URL: https://regtools.readthedocs.org</details>|
	|[relion](https://www3.mrc-lmb.cam.ac.uk/relion/)|chem|3.1.1, 4.0.0|<details><summary>Expand</summary>Description: RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer program for Maximum A Posteriori refinement of (multiple) 3D reconstructions or 2D class averages in cryo-electron microscopy. It is developed in the research group of Sjors Scheres at the MRC Laboratory of Molecular Biology. Homepage: https://www3.mrc-lmb.cam.ac.uk/relion/ URL: https://www3.mrc-lmb.cam.ac.uk/relion/</details>|
	|[repasthpc](https://repast.github.io/)|bio|2.2.0, 2.3.0|<details><summary>Expand</summary>Description: The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years: Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based modeling system that is designed for use on large computing clusters and supercomputers. Homepage: https://repast.github.io/ URL: https://repast.github.io/</details>|
	|[repeatmasker](http://www.repeatmasker.org/)|bio|4.1.2-p1, 4.1.1|<details><summary>Expand</summary>Description: RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. Homepage: http://www.repeatmasker.org/ URL: http://www.repeatmasker.org/</details>|
	|[repeatmodeler](http://repeatmasker.org/RepeatModeler/)|bio|2.0.1, 2.0.2a, 2.0.3|<details><summary>Expand</summary>Description: RepeatModeler is a de-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats. Homepage: http://repeatmasker.org/RepeatModeler/ URL: http://repeatmasker.org/RepeatModeler/</details>|
	|[repeatscout](https://bix.ucsd.edu/repeatscout/)|bio|1.0.5|<details><summary>Expand</summary>Description: RepeatScout is a tool to discover repetitive substrings in DNA. Homepage: https://bix.ucsd.edu/repeatscout/ URL: https://bix.ucsd.edu/repeatscout/</details>|
	|[revbayes](https://revbayes.github.io/)|-|1.1.1|<details><summary>Expand</summary>Description: RevBayes provides an interactive environment for statistical computation in phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in evolutionary biology, particularly phylogenetics. Homepage: https://revbayes.github.io/ URL: https://revbayes.github.io/</details>|
	|[rgaugury](https://bitbucket.org/yaanlpc/rgaugury/src/master/)|-|2.1.3|<details><summary>Expand</summary>Description: This project aims to develop a sophisticated informatics tools for large scale of resistance gene analogs identification, including major groups of RGAs, such as NBS encoding genes, receptor like proteins and receptor like protein kinases. It is thus named as RGA augury or RGAugury standalone pipeline. Homepage: https://bitbucket.org/yaanlpc/rgaugury/src/master/ URL: https://bitbucket.org/yaanlpc/rgaugury/src/master/</details>|
	|[rkcommon](https://github.com/ospray/rkcommon)|-|1.4.2, 1.10.0, 1.11.0|<details><summary>Expand</summary>Description: Intel RenderKit common library Homepage: https://github.com/ospray/rkcommon URL: https://github.com/ospray/rkcommon</details>|
	|[rmats](http://rnaseq-mats.sourceforge.net/)|-|4.1.1, 4.1.2, 4.3.0|<details><summary>Expand</summary>Description: MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data. The statistical model of MATS calculates the P-value and false discovery rate that the difference in the isoform ratio of a gene between two conditions exceeds a given user-defined threshold. From the RNA-Seq data, MATS can automatically detect and analyze alternative splicing events corresponding to all major types of alternative splicing patterns. MATS handles replicate RNA-Seq data from both paired and unpaired study design. Homepage: http://rnaseq-mats.sourceforge.net/ URL: http://rnaseq-mats.sourceforge.net/</details>|
	|[rmblast](http://www.repeatmasker.org/RMBlast.html)|bio|2.10.0, 2.14.1|<details><summary>Expand</summary>Description: RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn' for use with RepeatMasker and RepeatModeler. Homepage: http://www.repeatmasker.org/RMBlast.html URL: http://www.repeatmasker.org/RMBlast.html</details>|
	|[rnacode](https://wash.github.io/rnacode/)|-|0.3|<details><summary>Expand</summary>Description: RNAcode - Analyze the protein coding potential in multiple sequence alignments Homepage: https://wash.github.io/rnacode/ URL: https://wash.github.io/rnacode/</details>|
	|[rnammer](http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer)|bio|1.2|<details><summary>Expand</summary>Description: RNAmmer 1.2 predicts 5s/8s, 16s/18s, and 23s/28s ribosomal RNA in full genome sequences. The method is described in detail in the following article: RNammer: consistent annotation of rRNA genes in genomic sequences. Lagesen K, Hallin PF, Roedland E, Staerfeldt HH, Rognes T Ussery DW. Nucleic Acids Res. Apr 22, 2007. Homepage: http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer URL: http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer</details>|
	|[roary](https://sanger-pathogens.github.io/Roary/)|bio|3.13.0|<details><summary>Expand</summary>Description: Roary is a high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome. Homepage: https://sanger-pathogens.github.io/Roary/ URL: https://sanger-pathogens.github.io/Roary/</details>|
	|[rocksdb](https://github.com/facebook/rocksdb)|-|7.10.2|<details><summary>Expand</summary>Description: A library that provides an embeddable, persistent key-value store for fast storage. Homepage: https://github.com/facebook/rocksdb URL: https://github.com/facebook/rocksdb</details>|
	|[root](https://root.cern.ch/drupal/)|tools|6.20.04, 6.24.06, 6.26.06, 6.28.06|<details><summary>Expand</summary>Description: The ROOT system provides a set of OO frameworks with all the functionality needed to handle and analyze large amounts of data in a very efficient way. Homepage: https://root.cern.ch/drupal/ URL: https://root.cern.ch/drupal/</details>|
	|[rosetta](https://www.rosettacommons.org)|chem|3.10, 3.12, 3.13, 2019.21.60746|<details><summary>Expand</summary>Description: Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids. This is a weekly release. Homepage: https://www.rosettacommons.org URL: https://www.rosettacommons.org</details>|
	|[rsem](http://deweylab.github.io/RSEM/)|bio|1.3.3|<details><summary>Expand</summary>Description: RNA-Seq by Expectation-Maximization Homepage: http://deweylab.github.io/RSEM/ URL: http://deweylab.github.io/RSEM/</details>|
	|[rstudio-server](https://www.rstudio.com/)|tools|4.1, 4.2, 4.3, 4.4|<details><summary>Expand</summary>Description: RStudio Server enables you to provide a browser based interface to a version of R running on a remote Linux server, bringing the power and productivity of the RStudio IDE to server-based deployments of R. Homepage: https://www.rstudio.com/ URL: https://www.rstudio.com/</details>|
	|[rtk](https://itk.org)|bio|1.4.0|<details><summary>Expand</summary>Description: Insighteconstruction Toolkit (RTK) is an open-source and cross-platform software for fast circular cone-beam CT reconstruction based on the Insight Toolkit (ITK). RTK is developped by the RTK consortium. RTK provides or will provide Basic operators for reconstruction, e.g., filtering, forward, projection and backprojection Multithreaded CPU and GPU versions Tools for respiratory motion correction I/O for several scanners Preprocessing of raw data for scatter correction Homepage: https://itk.org URL: https://itk.org</details>|
	|[ruamel.yaml](https://sourceforge.net/projects/ruamel-yaml)|-|0.17.21|<details><summary>Expand</summary>Description: ruamel.yaml is a YAML 1.2 loader/dumper package for Python. Homepage: https://sourceforge.net/projects/ruamel-yaml URL: https://sourceforge.net/projects/ruamel-yaml</details>|
	|[ruby](https://www.ruby-lang.org)|tools|2.7.1, 3.2.2|<details><summary>Expand</summary>Description: Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. Homepage: https://www.ruby-lang.org URL: https://www.ruby-lang.org</details>|
	|[rust](https://www.rust-lang.org)|tools|1.47.0, 1.53.0, 1.59.0, 1.65.0, 1.70.0, 1.76.0, 1.85.0|<details><summary>Expand</summary>Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. Homepage: https://www.rust-lang.org URL: https://www.rust-lang.org</details>|
	|[sabre](https://github.com/najoshi/sabre)|bio|1.00|<details><summary>Expand</summary>Description: A barcode demultiplexing and trimming tool for FastQ files Homepage: https://github.com/najoshi/sabre URL: https://github.com/najoshi/sabre</details>|
	|[saga](https://saga-gis.sourceforge.io/)|-|8.2.2|<details><summary>Expand</summary>Description: System for automated geoscientific analyses Homepage: https://saga-gis.sourceforge.io/ URL: https://saga-gis.sourceforge.io/</details>|
	|[sagemath](http://www.sagemath.org/)|tools|9.1, 9.3|<details><summary>Expand</summary>Description: SageMath is a free open-source mathematics software system licensed under the GPL. It builds on top of many existing open-source packages: NumPy, SciPy, matplotlib, Sympy, Maxima, GAP, FLINT, R and many more. Access their combined power through a common, Python-based language or directly via interfaces or wrappers. Homepage: http://www.sagemath.org/ URL: http://www.sagemath.org/</details>|
	|[salmon](https://github.com/COMBINE-lab/salmon)|bio|1.3.0, 1.4.0, 1.10.2|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Homepage: https://github.com/COMBINE-lab/salmon URL: https://github.com/COMBINE-lab/salmon</details>|
	|[salmonbeta](https://github.com/COMBINE-lab/salmon/)|bio|0.6.0|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Salmon achieves is accuracy and speed via a number of different innovations, including the use of quasi-mapping (accurate but fast-to-compute proxies for traditional read alignments), and massively-parallel stochastic collapsed variational inference. The result is a versatile tool that fits nicely into many differnt pipelines. For example, you can choose to make use of our quasi-mapping algorithm by providing Salmon with raw sequencing reads, or, if it is more convenient, you can provide Salmon with regular alignments (e.g. an unsorted BAM file produced with your favorite aligner), and it will use the same wicked-fast, state-of-the-art inference algorithm to estimate transcript-level abundances for your experiment. Homepage: https://github.com/COMBINE-lab/salmon/ URL: https://github.com/COMBINE-lab/salmon/</details>|
	|[sambamba](https://lomereiter.github.io/sambamba/)|bio|0.8.0, 1.0.1|<details><summary>Expand</summary>Description: Sambamba is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Current functionality is an important subset of samtools functionality, including view, index, sort, markdup, and depth. Homepage: https://lomereiter.github.io/sambamba/ URL: https://lomereiter.github.io/sambamba/</details>|
	|[samblaster](https://github.com/GregoryFaust/samblaster)|bio|0.1.26|<details><summary>Expand</summary>Description: samblaster: a tool to mark duplicates and extract discordant and split reads from sam files Homepage: https://github.com/GregoryFaust/samblaster URL: https://github.com/GregoryFaust/samblaster</details>|
	|[samstat](http://samstat.sourceforge.net/)|-|1.5.1|<details><summary>Expand</summary>Description: Displaying sequence statistics for next generation sequencing Homepage: http://samstat.sourceforge.net/ URL: http://samstat.sourceforge.net/</details>|
	|[samtools](https://www.htslib.org/)|bio|0.1.20, 1.10, 1.11, 1.12, 1.13, 1.15.1, 1.16.1, 1.17, 1.18, 1.20|<details><summary>Expand</summary>Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format. Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[savvy](https://github.com/statgen/savvy)|-|2.1.0|<details><summary>Expand</summary>Description: Interface to various variant calling formats. Homepage: https://github.com/statgen/savvy URL: https://github.com/statgen/savvy</details>|
	|[sbt](http://www.scala-sbt.org/)|tools|1.3.13|<details><summary>Expand</summary>Description: sbt is a build tool for Scala, Java, and more. It requires Java 1.8 or later. sbt uses a small number of concepts to support flexible and powerful build definitions. There are not that many concepts, but sbt is not exactly like other build systems and there are details you will stumble on if you haven t read the documentation. The Getting Started Guide covers the concepts you need to know to create and maintain an sbt build definition. It is highly recommended to read the Getting Started Guide. Homepage: http://www.scala-sbt.org/ URL: http://www.scala-sbt.org/</details>|
	|[scalapack](https://www.netlib.org/scalapack/)|math|2.1.0, 2.2.0|<details><summary>Expand</summary>Description: The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines redesigned for distributed memory MIMD parallel computers. Homepage: https://www.netlib.org/scalapack/ URL: https://www.netlib.org/scalapack/</details>|
	|[scalasca](https://www.scalasca.org/)|-|2.5|<details><summary>Expand</summary>Description: Scalasca is a software tool that supports the performance optimization of parallel programs by measuring and analyzing their runtime behavior. The analysis identifies potential performance bottlenecks -- in particular those concerning communication and synchronization -- and offers guidance in exploring their causes. Homepage: https://www.scalasca.org/ URL: https://www.scalasca.org/</details>|
	|[scipoptsuite](https://scipopt.org/)|-|9.0.0|<details><summary>Expand</summary>Description: SCIP is currently one of the fastest non-commercial solvers for mixed integer programming (MIP) and mixed integer nonlinear programming (MINLP). It is also a framework for constraint integer programming and branch-cut-and-price. It allows for total control of the solution process and the access of detailed information down to the guts of the solver. Homepage: https://scipopt.org/ URL: https://scipopt.org/</details>|
	|[scipy-stack](http://python.org/)|math|2020a, 2020b, 2021a, 2022a, 2023a, 2023b, 2024a, 2024b, 2025a|<details><summary>Expand</summary>Description: Bundle which contains the Scientific Python stack, including Cycler, mpmath, numpy, scipy, sympy, pandas, matplotlib, ipython_genutils, traitlets, ptyprocess, pathlib2, pickleshare, pexpect, simplegeneric, ipython, ipykernel, jupyter_client, jupyter_core, pyzmq, tornado, futures and ipyparallel. Homepage: http://python.org/ URL: http://python.org/</details>|
	|[score-p](https://www.score-p.org)|-|6.0|<details><summary>Expand</summary>Description: The Score-P measurement infrastructure is a highly scalable and easy-to-use tool suite for profiling, event tracing, and online analysis of HPC applications. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[scotch](https://www.labri.fr/perso/pelegrin/scotch/)|math|6.0.9, 7.0.3|<details><summary>Expand</summary>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. Homepage: https://www.labri.fr/perso/pelegrin/scotch/ URL: https://www.labri.fr/perso/pelegrin/scotch/</details>|
	|[scotch-no-thread](https://www.labri.fr/perso/pelegrin/scotch/)|-|6.1.2, 7.0.3|<details><summary>Expand</summary>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. This version has no threading support. Homepage: https://www.labri.fr/perso/pelegrin/scotch/ URL: https://www.labri.fr/perso/pelegrin/scotch/</details>|
	|[scythe](https://github.com/ucdavis-bioinformatics/scythe)|-|0.994|<details><summary>Expand</summary>Description: Scythe uses a Naive Bayesian approach to classify contaminant substrings in sequence reads. It considers quality information, which can make it robust in picking out 3'-end adapters, which often include poor quality bases. Homepage: https://github.com/ucdavis-bioinformatics/scythe URL: https://github.com/ucdavis-bioinformatics/scythe</details>|
	|[sdl2](http://www.libsdl.org/)|tools|2.0.16|<details><summary>Expand</summary>Description: SDL: Simple DirectMedia Layer, a cross-platform multimedia library Homepage: http://www.libsdl.org/ URL: http://www.libsdl.org/</details>|
	|[sdsl](https://github.com/simongog/sdsl-lite)|-|2.1.1-20191211|<details><summary>Expand</summary>Description: The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing succinct data structures. Homepage: https://github.com/simongog/sdsl-lite URL: https://github.com/simongog/sdsl-lite</details>|
	|[selscan](https://github.com/szpiech/selscan)|bio|2.0.3|<details><summary>Expand</summary>Description: selscan -- a program to calculate EHH-based scans for positive selection in genomes selscan currently implements EHH, iHS, XP-EHH, nSL, and iHH12 and requires phased data. It should be run separately for each chromosome and population (or population pair for XP-EHH). selscan is 'dumb' with respect ancestral/derived coding and simply expects haplotype data to be coded 0/1. Unstandardized iHS scores are thus reported as ln(iHH1/iHH0) based on the coding you have provided. Homepage: https://github.com/szpiech/selscan URL: https://github.com/szpiech/selscan</details>|
	|[seqan-library](https://www.seqan.de/)|-|2.4.0|<details><summary>Expand</summary>Description: SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences with the focus on biological data Homepage: https://www.seqan.de/ URL: https://www.seqan.de/</details>|
	|[seqkit](https://bioinf.shenwei.me/seqkit/)|-|0.13.2, 0.15.0, 2.3.1, 2.5.1|<details><summary>Expand</summary>Description: FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. This project describes a cross-platform ultrafast comprehensive toolkit for FASTA/Q processing. SeqKit provides executable binary files for all major operating systems, including Windows, Linux, and Mac OS X, and can be directly used without any dependencies or pre-configurations. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations. Homepage: https://bioinf.shenwei.me/seqkit/ URL: https://bioinf.shenwei.me/seqkit/</details>|
	|[seqlib](https://github.com/walaj/SeqLib)|-|1.2.0|<details><summary>Expand</summary>Description: C++ interface to HTSlib, BWA-MEM and Fermi. Homepage: https://github.com/walaj/SeqLib URL: https://github.com/walaj/SeqLib</details>|
	|[seqtk](https://github.com/lh3/seqtk/)|bio|1.4|<details><summary>Expand</summary>Description: Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip. Homepage: https://github.com/lh3/seqtk/ URL: https://github.com/lh3/seqtk/</details>|
	|[shapeit](https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html)|-|2.r904|<details><summary>Expand</summary>Description: SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</details>|
	|[shapeit4](https://odelaneau.github.io/shapeit4/)|-|4.2.0, 4.2.1|<details><summary>Expand</summary>Description: SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. Homepage: https://odelaneau.github.io/shapeit4/ URL: https://odelaneau.github.io/shapeit4/</details>|
	|[shapeit5](https://odelaneau.github.io/shapeit5/)|-|5.1.1|<details><summary>Expand</summary>Description: SHAPEIT5 estimates haplotypes in large datasets, with a special focus on rare variants. Homepage: https://odelaneau.github.io/shapeit5/ URL: https://odelaneau.github.io/shapeit5/</details>|
	|[shasta](https://github.com/chanzuckerberg/shasta)|-|0.10.0|<details><summary>Expand</summary>Description: The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include: Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k 10). Homepage: https://github.com/chanzuckerberg/shasta URL: https://github.com/chanzuckerberg/shasta</details>|
	|[shengbte](http://www.shengbte.org/)|phys|1.1.1, 1.5.0|<details><summary>Expand</summary>Description: ShengBTE is a software package for solving the Boltzmann Transport Equation for phonons. Also installed is the 'thirdorder' package of Python scripts. Homepage: http://www.shengbte.org/ URL: http://www.shengbte.org/</details>|
	|[shrinkwrap](https://github.com/jonathonl/shrinkwrap)|-|1.2.0|<details><summary>Expand</summary>Description: A std::streambuf wrapper for compression formats. Homepage: https://github.com/jonathonl/shrinkwrap URL: https://github.com/jonathonl/shrinkwrap</details>|
	|[sibelia](http://bioinf.spbau.ru/en/sibelia)|-|3.0.7|<details><summary>Expand</summary>Description: Sibelia: A comparative genomics tool: It assists biologists in analysing the genomic variations that correlate with pathogens, or the genomic changes that help microorganisms adapt in different environments. Sibelia will also be helpful for the evolutionary and genome rearrangement studies for multiple strains of microorganisms. Homepage: http://bioinf.spbau.ru/en/sibelia URL: http://bioinf.spbau.ru/en/sibelia</details>|
	|[sickle](https://github.com/najoshi/sickle)|bio|1.33|<details><summary>Expand</summary>Description: Windowed Adaptive Trimming for fastq files using quality Homepage: https://github.com/najoshi/sickle URL: https://github.com/najoshi/sickle</details>|
	|[siesta](http://departments.icmab.es/leem/siesta)|chem|4.1-MaX-3.0, 4.0.1, 4.1-b4, 4.1.5|<details><summary>Expand</summary>Description: SIESTA is both a method and its computer program implementation, to perform efficient electronic structure calculations and ab initio molecular dynamics simulations of molecules and solids. Homepage: http://departments.icmab.es/leem/siesta URL: http://departments.icmab.es/leem/siesta</details>|
	|[signalp](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp)|bio|4.1f|<details><summary>Expand</summary>Description: SignalP predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive bacteria, Gram-negative bacteria, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp</details>|
	|[simbac](https://github.com/tbrown91/SimBac)|-|0.1a|<details><summary>Expand</summary>Description: SimBac simulates bacterial genomes with the clonal genealogy under a coalescent model with recombination. Homepage: https://github.com/tbrown91/SimBac URL: https://github.com/tbrown91/SimBac</details>|
	|[simbody](https://github.com/simbody/simbody)|-|3.7|<details><summary>Expand</summary>Description: Simbody is a high-performance, open-source toolkit for science- and engineering-quality simulation of articulated mechanisms, including biomechanical structures such as human and animal skeletons, mechanical systems like robots, vehicles, and machines, and anything else that can be described as a set of rigid bodies interconnected by joints, influenced by forces and motions, and restricted by constraints. Simbody includes a multibody dynamics library for modeling motion in generalized/internal coordinates in O(n) time. This is sometimes called a Featherstone-style physics engine. Homepage: https://github.com/simbody/simbody URL: https://github.com/simbody/simbody</details>|
	|[simka](https://gatb.inria.fr/software/simka/)|-|1.5.3|<details><summary>Expand</summary>Description: Simka is a de novo comparative metagenomics tool. Simka represents each dataset as a k-mer spectrum and compute several classical ecological distances between them. Homepage: https://gatb.inria.fr/software/simka/ URL: https://gatb.inria.fr/software/simka/</details>|
	|[simnibs](https://simnibs.github.io/simnibs)|-|4.1.0|<details><summary>Expand</summary>Description: SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation Homepage: https://simnibs.github.io/simnibs URL: https://simnibs.github.io/simnibs</details>|
	|[simple-dftd3](https://github.com/dftd3/simple-dftd3)|-|0.7.0, 1.0.0|<details><summary>Expand</summary>Description: reimplementation of the DFT-D3 program Homepage: https://github.com/dftd3/simple-dftd3 URL: https://github.com/dftd3/simple-dftd3</details>|
	|[singular](https://www.singular.uni-kl.de/)|-|4.2.1|<details><summary>Expand</summary>Description: Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory. Homepage: https://www.singular.uni-kl.de/ URL: https://www.singular.uni-kl.de/</details>|
	|[sionlib](https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html)|-|1.7.6|<details><summary>Expand</summary>Description: SIONlib is a scalable I/O library for parallel access to task-local files. The library not only supports writing and reading binary data to or from several thousands of processors into a single or a small number of physical files, but also provides global open and close functions to access SIONlib files in parallel. This package provides a stripped-down installation of SIONlib for use with performance tools (e.g., Score-P), with renamed symbols to avoid conflicts when an application using SIONlib itself is linked against a tool requiring a different SIONlib version. Homepage: https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html URL: https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html</details>|
	|[slepc](https://slepc.upv.es/)|-|3.14.2, 3.17.2, 3.19.2, 3.20.1, 3.21.2|<details><summary>Expand</summary>Description: SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. Homepage: https://slepc.upv.es/ URL: https://slepc.upv.es/</details>|
	|[slepc-complex](https://slepc.upv.es/)|-|3.20.1|<details><summary>Expand</summary>Description: SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. Homepage: https://slepc.upv.es/ URL: https://slepc.upv.es/</details>|
	|[slicer](https://www.slicer.org/)|-|4.11.20210226|<details><summary>Expand</summary>Description: 3D Slicer is an open source software platform for medical image informatics, image processing, and three-dimensional visualization. Built over two decades through support from the National Institutes of Health and a worldwide developer community, Slicer brings free, powerful cross-platform processing tools to physicians, researchers, and the general public. Homepage: https://www.slicer.org/ URL: https://www.slicer.org/</details>|
	|[slim](https://messerlab.org/slim)|-|3.4.0, 4.0.1, 4.3|<details><summary>Expand</summary>Description: SLiM is an evolutionary simulation framework that combines a powerful engine for population genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios. Homepage: https://messerlab.org/slim URL: https://messerlab.org/slim</details>|
	|[slurm-completion](https://slurm.schedmd.com/)|-|23.02.7|<details><summary>Expand</summary>Description: bash completion for Slurm client commands Homepage: https://slurm.schedmd.com/ URL: https://slurm.schedmd.com/</details>|
	|[smartdenovo](https://github.com/ruanjue/smartdenovo)|-|20180219|<details><summary>Expand</summary>Description: SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy. Homepage: https://github.com/ruanjue/smartdenovo URL: https://github.com/ruanjue/smartdenovo</details>|
	|[smithwaterman](https://github.com/ekg/smithwaterman)|-|20160702|<details><summary>Expand</summary>Description: smith-waterman-gotoh alignment algorithm. Homepage: https://github.com/ekg/smithwaterman URL: https://github.com/ekg/smithwaterman</details>|
	|[smrtlink-sequel2](https://www.pacb.com/support/software-downloads/)|-|13.1.0.221970|<details><summary>Expand</summary>Description: PacBio's open-source SMRT Analysis software suite is designed for use with Single Molecule, Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through the extensive set of APIs provided Homepage: https://www.pacb.com/support/software-downloads/ URL: https://www.pacb.com/support/software-downloads/</details>|
	|[snap](http://korflab.ucdavis.edu/software.html)|bio|2017-05-17|<details><summary>Expand</summary>Description: SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser. Homepage: http://korflab.ucdavis.edu/software.html URL: http://korflab.ucdavis.edu/software.html</details>|
	|[snappy](https://github.com/google/snappy)|tools|1.1.8|<details><summary>Expand</summary>Description: Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression. Homepage: https://github.com/google/snappy URL: https://github.com/google/snappy</details>|
	|[sniffles](https://github.com/fritzsedlazeck/Sniffles)|-|1.0.12b|<details><summary>Expand</summary>Description: Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+) using evidence from split-read alignments, high-mismatch regions, and coverage analysis. Please note the current version of Sniffles requires sorted output from BWA-MEM (use -M and -x parameter), Minimap2 (sam file with Cigar and MD string) or NGMLR. Homepage: https://github.com/fritzsedlazeck/Sniffles URL: https://github.com/fritzsedlazeck/Sniffles</details>|
	|[snp-sites](https://github.com/sanger-pathogens/snp-sites)|-|2.5.1|<details><summary>Expand</summary>Description: Extracting single nucleotide polymorphisms (SNPs) from a large whole genome alignment is now a routine task, but existing tools have failed to scale efficiently with the increased size of studies. These tools are slow, memory inefficient and are installed through non-standard procedures. We present SNP-sites which can rapidly extract SNPs from a multi-FASTA alignment using modest resources and can output results in multiple formats for downstream analysis. SNPs can be extracted from a 8.3 GB alignment file (1,842 taxa, 22,618 sites) in 267 seconds using 59 MB of RAM and 1 CPU core, making it feasible to run on modest computers. Homepage: https://github.com/sanger-pathogens/snp-sites URL: https://github.com/sanger-pathogens/snp-sites</details>|
	|[snpeff](https://pcingola.github.io/SnpEff/)|bio|5.0, 5.2|<details><summary>Expand</summary>Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes). Homepage: https://pcingola.github.io/SnpEff/ URL: https://pcingola.github.io/SnpEff/</details>|
	|[soapdenovo-trans](http://soap.genomics.org.cn/SOAPdenovo-Trans.html)|-|1.0.4|<details><summary>Expand</summary>Description: SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. Homepage: http://soap.genomics.org.cn/SOAPdenovo-Trans.html URL: http://soap.genomics.org.cn/SOAPdenovo-Trans.html</details>|
	|[soapdenovo2](http://soap.genomics.org.cn/index.html)|bio|r242|<details><summary>Expand</summary>Description: SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for human-sized genomes. The program is specially designed to assemble Illumina short reads. It creates new opportunities for building reference sequences and carrying out accurate analyses of unexplored genomes in a cost effective way. SOAPdenovo2 is the successor of SOAPdenovo. Homepage: http://soap.genomics.org.cn/index.html URL: http://soap.genomics.org.cn/index.html</details>|
	|[soci](http://soci.sourceforge.net/)|-|4.0.2, 4.0.3|<details><summary>Expand</summary>Description: SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the regular C++ code, staying entirely within the Standard C++. Homepage: http://soci.sourceforge.net/ URL: http://soci.sourceforge.net/</details>|
	|[sortmerna](http://bioinfo.lifl.fr/RNA/sortmerna/)|bio|4.2.0, 4.3.4, 4.3.6|<details><summary>Expand</summary>Description: SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads. Homepage: http://bioinfo.lifl.fr/RNA/sortmerna/ URL: http://bioinfo.lifl.fr/RNA/sortmerna/</details>|
	|[soundtouch](https://www.surina.net/soundtouch/)|-|2.3.3|<details><summary>Expand</summary>Description: SoundTouch is an open-source audio processing library for changing the Tempo, Pitch and Playback Rates of audio streams or audio files. The library additionally supports estimating stable beats-per-minute rates for audio tracks. Homepage: https://www.surina.net/soundtouch/ URL: https://www.surina.net/soundtouch/</details>|
	|[spades](http://cab.spbu.ru/software/spades/)|bio|3.14.1, 3.15.1, 3.15.3, 3.15.4, 4.0.0|<details><summary>Expand</summary>Description: Genome assembler for single-cell and isolates data sets Homepage: http://cab.spbu.ru/software/spades/ URL: http://cab.spbu.ru/software/spades/</details>|
	|[spark](http://spark.apache.org)|tools|3.0.0, 3.3.0|<details><summary>Expand</summary>Description: Spark is Hadoop MapReduce done in memory Homepage: http://spark.apache.org URL: http://spark.apache.org</details>|
	|[sparsehash](https://github.com/sparsehash/sparsehash)|-|2.0.4|<details><summary>Expand</summary>Description: An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed. Homepage: https://github.com/sparsehash/sparsehash URL: https://github.com/sparsehash/sparsehash</details>|
	|[spdlog](https://github.com/gabime/spdlog)|-|1.9.2, 1.11.0|<details><summary>Expand</summary>Description: Very fast, header-only/compiled, C++ logging library. Homepage: https://github.com/gabime/spdlog URL: https://github.com/gabime/spdlog</details>|
	|[spectra](https://spectralib.org/)|-|0.9.0, 1.0.1|<details><summary>Expand</summary>Description: Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library. Homepage: https://spectralib.org/ URL: https://spectralib.org/</details>|
	|[spglib](https://github.com/spglib/spglib)|chem|1.16.0, 2.0.2|<details><summary>Expand</summary>Description: Spglib is a C library for finding and handling crystal symmetries. Homepage: https://github.com/spglib/spglib URL: https://github.com/spglib/spglib</details>|
	|[spoa](https://github.com/rvaser/spoa)|-|3.4.0|<details><summary>Expand</summary>Description: Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm which is used to generate consensus sequences Homepage: https://github.com/rvaser/spoa URL: https://github.com/rvaser/spoa</details>|
	|[sprng](http://www.sprng.org/)|math|5.0|<details><summary>Expand</summary>Description: Scalable Parallel Pseudo Random Number Generators Library Homepage: http://www.sprng.org/ URL: http://www.sprng.org/</details>|
	|[sqlite](https://www.sqlite.org/)|-|3.38.5, 3.43.1|<details><summary>Expand</summary>Description: SQLite: SQL Database Engine in a C Library Homepage: https://www.sqlite.org/ URL: https://www.sqlite.org/</details>|
	|[sra-toolkit](https://github.com/ncbi/sra-tools)|bio|2.10.8, 3.0.0, 3.0.9|<details><summary>Expand</summary>Description: The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to programmatically access data housed within SRA and convert it from the SRA format Homepage: https://github.com/ncbi/sra-tools URL: https://github.com/ncbi/sra-tools</details>|
	|[srprism](https://github.com/ncbi/SRPRISM)|-|3.1.2|<details><summary>Expand</summary>Description: Single Read Paired Read Indel Substitution Minimizer Homepage: https://github.com/ncbi/SRPRISM URL: https://github.com/ncbi/SRPRISM</details>|
	|[ssw](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library)|-|1.1, 1.2.5|<details><summary>Expand</summary>Description: SSW is a fast implementation of the Smith-Waterman algorithm, which uses the Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the instruction level. SSW library provides an API that can be flexibly used by programs written in C, C++ and other languages. We also provide a software that can do protein and genome alignment directly. Current version of our implementation is ~50 times faster than an ordinary Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and location heuristically. Homepage: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library URL: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library</details>|
	|[stacks](http://creskolab.uoregon.edu/stacks/)|bio|2.53, 2.55, 2.57, 2.59, 2.60, 2.62, 2.64, 2.66, 2.67|<details><summary>Expand</summary>Description: Stacks is a software pipeline for building loci from short-read sequences, such as those generated on the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting population genomics and phylogeography. Homepage: http://creskolab.uoregon.edu/stacks/ URL: http://creskolab.uoregon.edu/stacks/</details>|
	|[star](https://github.com/alexdobin/STAR)|bio|2.7.5a, 2.7.8a, 2.7.9a, 2.7.11a, 2.7.11b|<details><summary>Expand</summary>Description: STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays. Homepage: https://github.com/alexdobin/STAR URL: https://github.com/alexdobin/STAR</details>|
	|[star-fusion](https://github.com/STAR-Fusion/STAR-Fusion)|-|1.10.0|<details><summary>Expand</summary>Description: STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set. Homepage: https://github.com/STAR-Fusion/STAR-Fusion URL: https://github.com/STAR-Fusion/STAR-Fusion</details>|
	|[starccm](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010-R8, 16.02.008-R8, 16.02.009-R8, 16.04.007-R8, 16.04.012-R8, 16.06.008-R8, 17.02.007-R8, 17.02.008-R8, 17.04.007-R8, 17.04.008-R8, 17.06.007-R8, 17.06.008-R8, 18.02.008-R8, 18.04.008-R8, 18.06.006-R8, 19.04.007-R8|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses double precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[starccm-mixed](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010, 16.02.008, 16.02.009, 16.04.007, 16.04.012, 16.06.008, 17.02.007, 17.02.008, 17.04.007, 17.04.008, 17.06.007, 17.06.008, 18.02.008, 18.04.008, 18.06.006, 19.04.007|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses mixed precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[statistics-r](https://metacpan.org/pod/Statistics::R)|-|0.34|<details><summary>Expand</summary>Description: Perl interface with the R statistical program Homepage: https://metacpan.org/pod/Statistics::R URL: https://metacpan.org/pod/Statistics::R</details>|
	|[stringtie](https://ccb.jhu.edu/software/stringtie/)|bio|2.1.3, 2.1.5, 2.2.1, 2.2.3|<details><summary>Expand</summary>Description: StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts Homepage: https://ccb.jhu.edu/software/stringtie/ URL: https://ccb.jhu.edu/software/stringtie/</details>|
	|[structure](http://web.stanford.edu/group/pritchardlab/software/)|bio|2.3.4|<details><summary>Expand</summary>Description: The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. Homepage: http://web.stanford.edu/group/pritchardlab/software/ URL: http://web.stanford.edu/group/pritchardlab/software/</details>|
	|[su2](http://su2.stanford.edu)|math|7.0.8, 7.5.1|<details><summary>Expand</summary>Description: An open-source collection of software tools written in C++ for performing Partial Differential Equation (PDE) analysis and solving PDE-constrained optimization problems. The toolset is designed with computational fluid dynamics and aerodynamic shape optimization in mind. Homepage: http://su2.stanford.edu URL: http://su2.stanford.edu</details>|
	|[subread](http://subread.sourceforge.net/)|bio|2.0.3, 2.0.6|<details><summary>Expand</summary>Description: High performance read alignment, quantification and mutation discovery Homepage: http://subread.sourceforge.net/ URL: http://subread.sourceforge.net/</details>|
	|[suitesparse](https://faculty.cse.tamu.edu/davis/suitesparse.html)|math|5.7.1, 5.10.1, 5.13.0, 7.6.0|<details><summary>Expand</summary>Description: SuiteSparse is a collection of libraries to manipulate sparse matrices. Homepage: https://faculty.cse.tamu.edu/davis/suitesparse.html URL: https://faculty.cse.tamu.edu/davis/suitesparse.html</details>|
	|[sumo](https://github.com/eclipse/sumo/)|-|1.7.0, 1.9.0, 1.9.2, 1.15.0|<details><summary>Expand</summary>Description: Eclipse Sumo is an open source, highly portable, microscopic and continuous traffic simulation package designed to handle large road networks. Homepage: https://github.com/eclipse/sumo/ URL: https://github.com/eclipse/sumo/</details>|
	|[sundials](https://computing.llnl.gov/projects/sundials)|-|2.7.0, 5.3.0, 5.7.0, 6.4.1, 6.6.0|<details><summary>Expand</summary>Description: SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers Homepage: https://computing.llnl.gov/projects/sundials URL: https://computing.llnl.gov/projects/sundials</details>|
	|[superlu](http://crd-legacy.lbl.gov/~xiaoye/SuperLU/)|math|5.2.1|<details><summary>Expand</summary>Description: SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines. Homepage: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/ URL: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/</details>|
	|[supernova](https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome)|bio|2.1.1|<details><summary>Expand</summary>Description: Supernova is delivered as a single, self-contained tar file that can be unpacked anywhere on your system. It bundles all of its own required software dependencies, which are pre-compiled to run on a range of Linux distributions. Homepage: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome URL: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome</details>|
	|[survivor](https://github.com/fritzsedlazeck/SURVIVOR)|-|1.0.7|<details><summary>Expand</summary>Description: SURVIVOR is a tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples, and includes various methods to reformat or summarize SVs. Homepage: https://github.com/fritzsedlazeck/SURVIVOR URL: https://github.com/fritzsedlazeck/SURVIVOR</details>|
	|[svaba](https://github.com/walaj/svaba)|-|1.1.0|<details><summary>Expand</summary>Description: Structural variation and indel detection by local assembly Homepage: https://github.com/walaj/svaba URL: https://github.com/walaj/svaba</details>|
	|[swat+](https://swatplus.gitbook.io)|-|60.5.1|<details><summary>Expand</summary>Description: The Soil and Water Assessment Tool (SWAT) is a small watershed to river basin-scale model used to simulate the quality and quantity of surface and ground water and predict the environmental impact of land use, land management practices, and climate change. In order to face present and future challenges in water resources modeling SWAT code has undergone major modifications over the past few years, resulting in SWAT+, a completely revised version of the model. SWAT+ provides a more flexible spatial representation of interactions and processes within a watershed. Homepage: https://swatplus.gitbook.io URL: https://swatplus.gitbook.io</details>|
	|[swi-prolog](https://www.swi-prolog.org/)|-|9.0.3|<details><summary>Expand</summary>Description: SWI-Prolog offers a comprehensive free Prolog environment. Homepage: https://www.swi-prolog.org/ URL: https://www.swi-prolog.org/</details>|
	|[swig](http://www.swig.org/)|-|4.0.1, 4.0.2, 4.1.1, 4.2.1|<details><summary>Expand</summary>Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. Homepage: http://www.swig.org/ URL: http://www.swig.org/</details>|
	|[symengine](https://github.com/symengine/symengine)|-|0.6.0, 0.8.1, 0.9.0, 0.10.1, 0.11.1, 0.11.2, 0.13.0, 0.14.0|<details><summary>Expand</summary>Description: SymEngine is a standalone fast C++ symbolic manipulation library. Homepage: https://github.com/symengine/symengine URL: https://github.com/symengine/symengine</details>|
	|[tabix](http://samtools.sourceforge.net)|bio|0.2.6|<details><summary>Expand</summary>Description: Generic indexer for TAB-delimited genome position files Homepage: http://samtools.sourceforge.net URL: http://samtools.sourceforge.net</details>|
	|[tabixpp](https://github.com/ekg/tabixpp)|-|1.1.0, 1.1.2|<details><summary>Expand</summary>Description: C++ wrapper to tabix indexer Homepage: https://github.com/ekg/tabixpp URL: https://github.com/ekg/tabixpp</details>|
	|[tau](http://tau.uoregon.edu)|tools|2.30.1|<details><summary>Expand</summary>Description: The TAU Performance System is a portable profiling and tracing toolkit for performance analysis of parallel programs written in Fortran, C, C++, Java, Python. Homepage: http://tau.uoregon.edu URL: http://tau.uoregon.edu</details>|
	|[taudem](https://github.com/dtarb/TauDEM/wiki)|-|5.3.8|<details><summary>Expand</summary>Description: TauDEM (Terrain Analysis Using Digital Elevation Models) is a suite of Digital Elevation Model (DEM) tools for the extraction and analysis of hydrologic information from topography as represented by a DEM. Homepage: https://github.com/dtarb/TauDEM/wiki URL: https://github.com/dtarb/TauDEM/wiki</details>|
	|[taxonkit](https://bioinf.shenwei.me/taxonkit/)|-|0.6.2|<details><summary>Expand</summary>Description: TaxonKit - A Cross-platform and Efficient NCBI Taxonomy Toolkit TaxonKit is a command-line toolkit for rapid manipulation of NCBI taxonomy data. It provides executable binary files for major operating systems including Windows, Linux, and Mac OS X, and can be directly used without any dependencies nor local database buiding. Homepage: https://bioinf.shenwei.me/taxonkit/ URL: https://bioinf.shenwei.me/taxonkit/</details>|
	|[tbb](https://github.com/oneapi-src/oneTBB)|tools|2020.2, 2021.8.0, 2021.10.0|<details><summary>Expand</summary>Description: Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability. Homepage: https://github.com/oneapi-src/oneTBB URL: https://github.com/oneapi-src/oneTBB</details>|
	|[tbl2asn](https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/)|-|25.8|<details><summary>Expand</summary>Description: Tbl2asn is a command-line program that automates the creation of sequence records for submission to GenBank Homepage: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/ URL: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/</details>|
	|[tblite](https://github.com/tblite/tblite)|-|0.3.0|<details><summary>Expand</summary>Description: Light-weight tight-binding framework Homepage: https://github.com/tblite/tblite URL: https://github.com/tblite/tblite</details>|
	|[tensorrt](https://developer.nvidia.com/tensorrt)|-|8.6.1.6|<details><summary>Expand</summary>Description: NVIDIA TensorRT is a platform for high-performance deep learning inference Homepage: https://developer.nvidia.com/tensorrt URL: https://developer.nvidia.com/tensorrt</details>|
	|[tesseract](https://github.com/tesseract-ocr/tesseract)|-|4.1.3, 5.0.1|<details><summary>Expand</summary>Description: Tesseract is an optical character recognition engine Homepage: https://github.com/tesseract-ocr/tesseract URL: https://github.com/tesseract-ocr/tesseract</details>|
	|[thrift](https://thrift.apache.org/)|-|0.13.0, 0.14.2, 0.16.0, 0.18.1, 0.19.0|<details><summary>Expand</summary>Description: The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages. Homepage: https://thrift.apache.org/ URL: https://thrift.apache.org/</details>|
	|[tmhmm](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm)|bio|2.0c|<details><summary>Expand</summary>Description: TMHMM 2.0 predicts transmembrane helices in proteins. The method is described in: Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes. Anders Krogh and Bjorn Larsson, Gunnar von Heijne, and Erik L.L. Sonnhammer. J. Mol. Biol. 305:567-580, 2001. and: A hidden Markov model for predicting transmembrane helices in protein sequences. Erik L.L. Sonnhammer, Gunnar von Heijne, and Anders Krogh. In J. Glasgow et al., eds.: Proc. of ISMB6, pages 175-182. AAAI Press, 1998. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm</details>|
	|[togl](https://sourceforge.net/projects/togl/)|-|2.0|<details><summary>Expand</summary>Description: A Tcl/Tk widget for OpenGL rendering. Homepage: https://sourceforge.net/projects/togl/ URL: https://sourceforge.net/projects/togl/</details>|
	|[tophat](http://ccb.jhu.edu/software/tophat/)|bio|2.1.2|<details><summary>Expand</summary>Description: TopHat is a fast splice junction mapper for RNA-Seq reads. Homepage: http://ccb.jhu.edu/software/tophat/ URL: http://ccb.jhu.edu/software/tophat/</details>|
	|[towhee](http://towhee.sourceforge.net/)|chem|8.2.3|<details><summary>Expand</summary>Description: Towhee is a Monte Carlo molecular simulation code originally designed for the prediction of fluid phase equilibria using atom-based force fields and the Gibbs ensemble with particular attention paid to algorithms addressing molecule conformation sampling. The code has subsequently been extended to several ensembles, many different force fields, and solid (or at least porous) phases. Homepage: http://towhee.sourceforge.net/ URL: http://towhee.sourceforge.net/</details>|
	|[tpp](http://ww.peptideatlas.org/)|-|6.0.0|<details><summary>Expand</summary>Description: The Trans-Proteomic Pipeline (TPP) is a collection of integrated tools for MS/MS proteomics, developed at the SPC. On cedar we can also provide TPP web interface (tpp_gui) per group upon user request Homepage: http://ww.peptideatlas.org/ URL: http://ww.peptideatlas.org/</details>|
	|[trans-abyss](http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/)|bio|2.0.1|<details><summary>Expand</summary>Description: de novo assembly of RNA-Seq data using ABySS Homepage: http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/ URL: http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/</details>|
	|[transdecoder](https://transdecoder.github.io/)|bio|5.5.0, 5.7.1|<details><summary>Expand</summary>Description: TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks. Homepage: https://transdecoder.github.io/ URL: https://transdecoder.github.io/</details>|
	|[transrate](http://hibberdlab.com/transrate/)|bio|1.0.3|<details><summary>Expand</summary>Description: Transrate is software for de-novo transcriptome assembly quality analysis. It examines your assembly in detail and compares it to experimental evidence such as the sequencing reads, reporting quality scores for contigs and assemblies. This allows you to choose between assemblers and parameters, filter out the bad contigs from an assembly, and help decide when to stop trying to improve the assembly. Homepage: http://hibberdlab.com/transrate/ URL: http://hibberdlab.com/transrate/</details>|
	|[travis-analyzer](http://www.travis-analyzer.de/)|-|220729|<details><summary>Expand</summary>Description: TRAVIS is a free tool for analyzing and visualizing trajectories from all kinds of Molecular Dynamics or Monte Carlo simulations. Homepage: http://www.travis-analyzer.de/ URL: http://www.travis-analyzer.de/</details>|
	|[treemix](http://bitbucket.org/nygcresearch/treemix)|-|1.13|<details><summary>Expand</summary>Description: TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations. Homepage: http://bitbucket.org/nygcresearch/treemix URL: http://bitbucket.org/nygcresearch/treemix</details>|
	|[trf](https://tandem.bu.edu/trf/trf.html)|bio|4.09.1|<details><summary>Expand</summary>Description: A tandem repeat in DNA is two or more adjacent, approximate copies of a pattern of nucleotides. Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences. In order to use the program, the user submits a sequence in FASTA format. There is no need to specify the pattern, the size of the pattern or any other parameter. The output consists of two files: a repeat table file and an alignment file. The repeat table, viewable in a web browser, contains information about each repeat, including its location, size, number of copies and nucleotide content. Clicking on the location indices for one of the table entries opens a second browser page that shows an alignment of the copies against a consensus pattern. The program is very fast, analyzing sequences on the order of .5Mb in just a few seconds. Submitted sequences may be of arbitrary length. Repeats with pattern size in the range from 1 to 2000 bases are detected. Homepage: https://tandem.bu.edu/trf/trf.html URL: https://tandem.bu.edu/trf/trf.html</details>|
	|[trilinos](https://trilinos.org)|tools|13.0.1, 13.2.0, 13.4.0, 13.4.1, 15.1.1|<details><summary>Expand</summary>Description: The Trilinos Project is an effort to develop algorithms and enabling technologies within an object-oriented software framework for the solution of large-scale, complex multi-physics engineering and scientific problems. A unique design feature of Trilinos is its focus on packages. Homepage: https://trilinos.org URL: https://trilinos.org</details>|
	|[trimal](http://trimal.cgenomics.org/)|bio|1.4|<details><summary>Expand</summary>Description: trimAl is a tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment Homepage: http://trimal.cgenomics.org/ URL: http://trimal.cgenomics.org/</details>|
	|[trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)|bio|0.39|<details><summary>Expand</summary>Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.The selection of trimming steps and their associated parameters are supplied on the command line. Homepage: http://www.usadellab.org/cms/?page=trimmomatic URL: http://www.usadellab.org/cms/?page=trimmomatic</details>|
	|[trinity](http://trinityrnaseq.github.io/)|bio|2.11.0, 2.12.0, 2.13.2, 2.14.0|<details><summary>Expand</summary>Description: Trinity represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads. Homepage: http://trinityrnaseq.github.io/ URL: http://trinityrnaseq.github.io/</details>|
	|[trinotate](https://trinotate.github.io/)|bio|3.2.1, 3.2.2, 4.0.0|<details><summary>Expand</summary>Description: Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms. Homepage: https://trinotate.github.io/ URL: https://trinotate.github.io/</details>|
	|[triqs](https://triqs.github.io/triqs)|-|3.1.1, 3.3.1|<details><summary>Expand</summary>Description: TRIQS (Toolbox for Research on Interacting Quantum Systems) is a scientific project providing a set of C++ and Python libraries to develop new tools for the study of interacting quantum systems. Homepage: https://triqs.github.io/triqs URL: https://triqs.github.io/triqs</details>|
	|[trnascan-se](http://lowelab.ucsc.edu/)|bio|2.0.7, 2.0.12|<details><summary>Expand</summary>Description: tRNAscan-SE: An improved tool for transfer RNA detection, Patricia Chan and Todd Lowe, School of Engineering, University of California, Santa Cruz, CA. tRNAscan-SE was written in the PERL (version 5.0) script language. Input consists of DNA or RNA sequences in FASTA format. tRNA predictions are output in standard tabular or ACeDB format. tRNAscan-SE does no tRNA detection itself, but instead combines the strengths of three independent tRNA prediction programs by negotiating the flow of information between them, performing a limited amount of post-processing, and outputting the results in one of several formats. Homepage: http://lowelab.ucsc.edu/ URL: http://lowelab.ucsc.edu/</details>|
	|[ucc](https://www.openucx.org/)|-|1.0.0, 1.2.0, 1.3.0|<details><summary>Expand</summary>Description: UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and feature-rich for current and emerging programming models and runtimes. Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucc-cuda](https://www.openucx.org/)|-|1.0.0, 1.2.0, 1.3.0|<details><summary>Expand</summary>Description: UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and feature-rich for current and emerging programming models and runtimes. This module adds the UCC CUDA support. Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucx](https://www.openucx.org/)|-|1.8.0, 1.9.0, 1.10.0, 1.12.1, 1.14.1, 1.16.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucx-cuda](http://www.openucx.org/)|-|1.12.1, 1.14.1, 1.16.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications This module adds the UCX CUDA support. Homepage: http://www.openucx.org/ URL: http://www.openucx.org/</details>|
	|[udunits](https://www.unidata.ucar.edu/software/udunits/)|tools|2.2.26, 2.2.28|<details><summary>Expand</summary>Description: UDUNITS supports conversion of unit specifications between formatted and binary forms, arithmetic manipulation of units, and conversion of values between compatible scales of measurement. Homepage: https://www.unidata.ucar.edu/software/udunits/ URL: https://www.unidata.ucar.edu/software/udunits/</details>|
	|[unixodbc](https://www.unixodbc.org)|-|2.3.9|<details><summary>Expand</summary>Description: unixODBC provides a uniform interface between application and database driver Homepage: https://www.unixodbc.org URL: https://www.unixodbc.org</details>|
	|[valgrind-mpi](https://valgrind.org)|tools|3.16.1, 3.21.0|<details><summary>Expand</summary>Description: Valgrind: Debugging and profiling tools Homepage: https://valgrind.org URL: https://valgrind.org</details>|
	|[varscan](https://github.com/dkoboldt/varscan)|-|2.4.2, 2.4.6|<details><summary>Expand</summary>Description: Variant calling and somatic mutation/CNV detection for next-generation sequencing data Homepage: https://github.com/dkoboldt/varscan URL: https://github.com/dkoboldt/varscan</details>|
	|[vaspkit](https://vaspkit.com/)|-|1.5.1|<details><summary>Expand</summary>Description: VASPKIT aims at providing a powerful and user-friendly interface to perform high throughput analysis of various material properties from the raw calculated data using the widely-used VASP code. Homepage: https://vaspkit.com/ URL: https://vaspkit.com/</details>|
	|[vcflib](https://github.com/vcflib/vcflib)|-|1.0.1, 1.0.3, 1.0.9|<details><summary>Expand</summary>Description: vcflib provides methods to manipulate and interpret sequence variation as it can be described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended to concisely describe reference-indexed genetic variations between individuals. Homepage: https://github.com/vcflib/vcflib URL: https://github.com/vcflib/vcflib</details>|
	|[vcftools](https://vcftools.github.io)|bio|0.1.16|<details><summary>Expand</summary>Description: The aim of VCFtools is to provide easily accessible methods for working with complex genetic variation data in the form of VCF files. Homepage: https://vcftools.github.io URL: https://vcftools.github.io</details>|
	|[velvet](http://www.ebi.ac.uk/~zerbino/velvet/)|bio|1.2.10|<details><summary>Expand</summary>Description: Sequence assembler for very short reads Homepage: http://www.ebi.ac.uk/~zerbino/velvet/ URL: http://www.ebi.ac.uk/~zerbino/velvet/</details>|
	|[viennarna](http://www.tbi.univie.ac.at/RNA/)|bio|2.5.0, 2.5.1|<details><summary>Expand</summary>Description: The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures. Homepage: http://www.tbi.univie.ac.at/RNA/ URL: http://www.tbi.univie.ac.at/RNA/</details>|
	|[virsorter](https://github.com/simroux/VirSorter)|-|1.0.6|<details><summary>Expand</summary>Description: VirSorter: mining viral signal from microbial genomic data. Homepage: https://github.com/simroux/VirSorter URL: https://github.com/simroux/VirSorter</details>|
	|[visit](https://wci.llnl.gov/simulation/computer-codes/visit)|vis|2.13.3, 3.2.1|<details><summary>Expand</summary>Description: VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool. Homepage: https://wci.llnl.gov/simulation/computer-codes/visit URL: https://wci.llnl.gov/simulation/computer-codes/visit</details>|
	|[vmatch](http://www.vmatch.de/)|-|2.3.1|<details><summary>Expand</summary>Description: Vmatch, a versatile software tool for efficiently solving large scale sequence matching tasks. Vmatch subsumes the software tool REPuter, but is much more general, with a very flexible user interface, and improved space and time requirements. Homepage: http://www.vmatch.de/ URL: http://www.vmatch.de/</details>|
	|[vmd](https://www.ks.uiuc.edu/Research/vmd)|vis|1.9.3, 1.9.4a43, 1.9.4a57|<details><summary>Expand</summary>Description: VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. Homepage: https://www.ks.uiuc.edu/Research/vmd URL: https://www.ks.uiuc.edu/Research/vmd</details>|
	|[voro++](http://math.lbl.gov/voro++/)|math|0.4.6|<details><summary>Expand</summary>Description: Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles. Homepage: http://math.lbl.gov/voro++/ URL: http://math.lbl.gov/voro++/</details>|
	|[vsearch](https://github.com/torognes/vsearch)|bio|2.13.3, 2.13.4, 2.15.2, 2.21.1|<details><summary>Expand</summary>Description: VSEARCH supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, rereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering, conversion and merging of paired-end reads. Homepage: https://github.com/torognes/vsearch URL: https://github.com/torognes/vsearch</details>|
	|[vtk](http://www.vtk.org)|vis|8.2.0, 9.0.1, 9.1.0, 9.3.0|<details><summary>Expand</summary>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. Homepage: http://www.vtk.org URL: http://www.vtk.org</details>|
	|[vtk-mpi](http://www.vtk.org)|vis|9.0.1, 9.0.3|<details><summary>Expand</summary>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. Homepage: http://www.vtk.org URL: http://www.vtk.org</details>|
	|[vtune](https://software.intel.com/en-us/vtune)|tools|2020.1, 2022.2|<details><summary>Expand</summary>Description: Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran, Assembly and Java. Homepage: https://software.intel.com/en-us/vtune URL: https://software.intel.com/en-us/vtune</details>|
	|[w3emc](https://github.com/NOAA-EMC/NCEPLIBS-w3emc/)|-|2.11.0|<details><summary>Expand</summary>Description: This library contains Fortran decoder/encoder routines for GRIB edition 1. GRIdded Binary or General Regularly-distributed Information in Binary form (GRIB) is a World Meteorological Organization (WMO) standard for gridded data. This is part of the NCEPLIBS project. Homepage: hthttps://github.com/NOAA-EMC/NCEPLIBS-w3emc/ URL: hthttps://github.com/NOAA-EMC/NCEPLIBS-w3emc/</details>|
	|[wannier90](http://www.wannier.org/)|chem|1.2, 3.1.0|<details><summary>Expand</summary>Description: The Maximally-Localised Generalised Wannier Functions calculates the counterpart of localized molecular orbitals for crystalline systems from the wavefuction. Homepage: http://www.wannier.org/ URL: http://www.wannier.org/</details>|
	|[wannier90-abinit](http://www.wannier.org)|chem|2.0.1.1|<details><summary>Expand</summary>Description: A tool for obtaining maximally-localised Wannier functions Homepage: http://www.wannier.org URL: http://www.wannier.org</details>|
	|[wanniertools](http://www.wanniertools.com)|-|2.7.0|<details><summary>Expand</summary>Description: WannierTools is an open source software that studies the physical properties of given tight-binding model. Homepage: http://www.wanniertools.com URL: http://www.wanniertools.com</details>|
	|[wasp](https://code.ornl.gov/neams-workbench/wasp)|-|3.1.4, 4.0.3, 4.2.0|<details><summary>Expand</summary>Description: The Workbench Analysis Sequence Processor (WASP) was developed to streamline lexing, parsing, access, validation, and analysis of ascii text files. Homepage: https://code.ornl.gov/neams-workbench/wasp URL: https://code.ornl.gov/neams-workbench/wasp</details>|
	|[wcslib](https://www.atnf.csiro.au/people/mcalabre/WCS/)|-|8.3|<details><summary>Expand</summary>Description: The FITS "World Coordinate System" (WCS) standard defines keywords and usage that provide for the description of astronomical coordinate systems in a FITS image header. Homepage: https://www.atnf.csiro.au/people/mcalabre/WCS/ URL: https://www.atnf.csiro.au/people/mcalabre/WCS/</details>|
	|[wfa2](https://github.com/smarco/WFA2-lib)|-|2.3.4|<details><summary>Expand</summary>Description: The wavefront alignment (WFA) algorithm is an exact gap-affine algorithm that takes advantage of homologous regions between the sequences to accelerate the alignment process. Homepage: https://github.com/smarco/WFA2-lib URL: https://github.com/smarco/WFA2-lib</details>|
	|[wgrib2](http://www.cpc.noaa.gov/products/wesley/wgrib2/)|geo|3.0.0|<details><summary>Expand</summary>Description: A program to decode/manipulate GRIB-2 files. Homepage: http://www.cpc.noaa.gov/products/wesley/wgrib2/ URL: http://www.cpc.noaa.gov/products/wesley/wgrib2/</details>|
	|[winnowmap](https://github.com/marbl/Winnowmap)|-|2.03|<details><summary>Expand</summary>Description: Winnowmap is a long-read mapping algorithm, and a result of our exploration into superior minimizer sampling techniques. Homepage: https://github.com/marbl/Winnowmap URL: https://github.com/marbl/Winnowmap</details>|
	|[wps](http://www.wrf-model.org)|geo|4.1, 4.2|<details><summary>Expand</summary>Description: WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale numerical weather prediction system designed to serve both operational forecasting and atmospheric research needs. Homepage: http://www.wrf-model.org URL: http://www.wrf-model.org</details>|
	|[wrf](https://www.wrf-model.org)|geo|4.1.3, 4.2.1, 4.3.3|<details><summary>Expand</summary>Description: The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale numerical weather prediction system designed to serve both operational forecasting and atmospheric research needs. Homepage: https://www.wrf-model.org URL: https://www.wrf-model.org</details>|
	|[wtdbg2](https://github.com/ruanjue/wtdbg2)|-|2.5|<details><summary>Expand</summary>Description: Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy. Homepage: https://github.com/ruanjue/wtdbg2 URL: https://github.com/ruanjue/wtdbg2</details>|
	|[wxwidgets](https://www.wxwidgets.org)|tools|3.1.4, 3.1.7|<details><summary>Expand</summary>Description: wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI. Homepage: https://www.wxwidgets.org URL: https://www.wxwidgets.org</details>|
	|[xbraid](https://github.com/XBraid/xbraid)|-|3.1.0|<details><summary>Expand</summary>Description: XBraid is a parallel-in-time software package. It implements an optimal-scaling multigrid solver for the (non)linear systems that arise from the discretization of problems with evolutionary behavior. Homepage: https://github.com/XBraid/xbraid URL: https://github.com/XBraid/xbraid</details>|
	|[xcrysden](http://www.xcrysden.org/)|vis|1.5.60, 1.6.2|<details><summary>Expand</summary>Description: XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. Homepage: http://www.xcrysden.org/ URL: http://www.xcrysden.org/</details>|
	|[xdrfile](http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library)|tools|1.1.4|<details><summary>Expand</summary>Description: library for reading and writing xtc, edr and trr files Homepage: http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library URL: http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library</details>|
	|[xerces-c++](https://xerces.apache.org/xerces-c/)|tools|3.2.2, 3.2.3|<details><summary>Expand</summary>Description: Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs. Homepage: https://xerces.apache.org/xerces-c/ URL: https://xerces.apache.org/xerces-c/</details>|
	|[xml-libxml](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)|-|2.0205, 2.0208|<details><summary>Expand</summary>Description: Perl binding for libxml2 Homepage: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod URL: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</details>|
	|[xmlf90](https://gitlab.com/siesta-project/libraries/xmlf90)|tools|1.5.4, 1.6.2|<details><summary>Expand</summary>Description: A fast XML parser and generator in Fortran Homepage: https://gitlab.com/siesta-project/libraries/xmlf90 URL: https://gitlab.com/siesta-project/libraries/xmlf90</details>|
	|[xtb](https://xtb-docs.readthedocs.io)|chem|6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.6.1, 6.7.1|<details><summary>Expand</summary>Description: xtb - An extended tight-binding semi-empirical program package. Homepage: https://xtb-docs.readthedocs.io URL: https://xtb-docs.readthedocs.io</details>|
	|[xtensor](https://github.com/xtensor-stack/xtensor)|-|0.24.2, 0.25.0|<details><summary>Expand</summary>Description: xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions. Homepage: https://github.com/xtensor-stack/xtensor URL: https://github.com/xtensor-stack/xtensor</details>|
	|[yambo](http://www.yambo-code.org)|chem|5.0.4|<details><summary>Expand</summary>Description: Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics. Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf. Homepage: http://www.yambo-code.org URL: http://www.yambo-code.org</details>|
	|[yaml-cpp](https://github.com/jbeder/yaml-cpp)|-|0.7.0|<details><summary>Expand</summary>Description: yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec Homepage: https://github.com/jbeder/yaml-cpp URL: https://github.com/jbeder/yaml-cpp</details>|
	|[yaxt](https://www.dkrz.de/redmine/projects/yaxt)|tools|0.9.0, 0.9.1, 0.10.0|<details><summary>Expand</summary>Description: Yet Another eXchange Tool Homepage: https://www.dkrz.de/redmine/projects/yaxt URL: https://www.dkrz.de/redmine/projects/yaxt</details>|
	|[zeo++](https://www.zeoplusplus.org/)|-|0.3|<details><summary>Expand</summary>Description: Zeo++ is a software package for analysis of crystalline porous materials. Zeo++ can be used to perform geometry-based analysis of structure and topology of the void space inside a material, to assemble or alternate structures as well as to generate structure representations to be used in structure similarity calculations. Zeo++ can be used to either analyze a single structure or perform high-throughput analysis of a large database. Homepage: https://www.zeoplusplus.org/ URL: https://www.zeoplusplus.org/</details>|
	|[zoltan](https://sandialabs.github.io/Zoltan/)|-|3.901|<details><summary>Expand</summary>Description: Zoltan Dynamic Load Balancing and Graph Algorithm Toolkit Homepage: https://sandialabs.github.io/Zoltan/ URL: https://sandialabs.github.io/Zoltan/</details>|

=== "AVX 2"

	|**Module**|**Type**|**Versions**|**Description**|
	|:----:|:----:|:----:|:----:|
	|[4ti2](https://4ti2.github.io/)|-|1.6.9|<details><summary>Expand</summary>Description: A software package for algebraic, geometric and combinatorial problems on linear spaces Homepage: https://4ti2.github.io/ URL: https://4ti2.github.io/</details>|
	|[abaqus](http://www.simulia.com/products/abaqus_fea.html)|phys|2021|<details><summary>Expand</summary>Description: Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA. Homepage: http://www.simulia.com/products/abaqus_fea.html URL: http://www.simulia.com/products/abaqus_fea.html</details>|
	|[abinit](https://www.abinit.org/)|chem|9.2.2, 9.6.2, 10.0.3, 10.2.3|<details><summary>Expand</summary>Description: ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. Homepage: https://www.abinit.org/ URL: https://www.abinit.org/</details>|
	|[abricate](https://github.com/tseemann/abricate)|-|1.0.0|<details><summary>Expand</summary>Description: Mass screening of contigs for antimicrobial and virulence genes Homepage: https://github.com/tseemann/abricate URL: https://github.com/tseemann/abricate</details>|
	|[abseil](https://abseil.io/)|-|20230125.3|<details><summary>Expand</summary>Description: Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives. Homepage: https://abseil.io/ URL: https://abseil.io/</details>|
	|[abyss](https://www.bcgsc.ca/platform/bioinfo/software/abyss)|bio|2.1.5, 2.2.5, 2.3.7|<details><summary>Expand</summary>Description: Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler Homepage: https://www.bcgsc.ca/platform/bioinfo/software/abyss URL: https://www.bcgsc.ca/platform/bioinfo/software/abyss</details>|
	|[actc](https://sourceforge.net/projects/actc)|-|1.1|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[admixture](https://dalexander.github.io/admixture/)|bio|1.3.0|<details><summary>Expand</summary>Description: ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. Homepage: https://dalexander.github.io/admixture/ URL: https://dalexander.github.io/admixture/</details>|
	|[adol-c](https://projects.coin-or.org/ADOL-C)|-|2.7.2|<details><summary>Expand</summary>Description: The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C. Homepage: https://projects.coin-or.org/ADOL-C URL: https://projects.coin-or.org/ADOL-C</details>|
	|[adtree](https://github.com/tudelft3d/AdTree)|-|1.1.2|<details><summary>Expand</summary>Description: AdTree: Accurate, Detailed, and Automatic Modelling of Laser-Scanned Trees Homepage: https://github.com/tudelft3d/AdTree URL: https://github.com/tudelft3d/AdTree</details>|
	|[advisor](https://software.intel.com/intel-advisor-xe)|tools|2020.3|<details><summary>Expand</summary>Description: Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain Homepage: https://software.intel.com/intel-advisor-xe URL: https://software.intel.com/intel-advisor-xe</details>|
	|[afni](http://afni.nimh.nih.gov/)|bio|20.3.05, 21.2.10, 22.1.12, 23.1.00, 23.1.08, 24.1.03|<details><summary>Expand</summary>Description: AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data - a technique for mapping human brain activity. Homepage: http://afni.nimh.nih.gov/ URL: http://afni.nimh.nih.gov/</details>|
	|[alevin-fry](https://github.com/COMBINE-lab/alevin-fry)|-|0.8.2|<details><summary>Expand</summary>Description: alevin-fry is an efficient and flexible tool for processing single-cell sequencing data, currently focused on single-cell transcriptomics and feature barcoding. Homepage: https://github.com/COMBINE-lab/alevin-fry URL: https://github.com/COMBINE-lab/alevin-fry</details>|
	|[alm](https://github.com/ttadano/ALM)|-|2.0.0_dev.2|<details><summary>Expand</summary>Description: Spglib is a software for calculating harmonic and anharmonic interatomic force constants in solids and molecules Homepage: https://github.com/ttadano/ALM URL: https://github.com/ttadano/ALM</details>|
	|[almosthere](https://github.com/horta/almosthere)|-|1.0.10, 1.0.15|<details><summary>Expand</summary>Description: Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows, Linux, and macOS. It is non-blocking as the progress update is done via a dedicated, lightweight thread, as to not impair the performance of the calling program. Homepage: https://github.com/horta/almosthere URL: https://github.com/horta/almosthere</details>|
	|[alpscore](https://github.com/ALPSCore/ALPSCore)|phys|2.2.0|<details><summary>Expand</summary>Description: The ALPSCore project, based on the ALPS (Algorithms and Libraries for Physics Simulations) project, provides generic algorithms and utilities for physics problems. It strives to increase software reuse in the physics community. For copyright see COPYRIGHT.TXT For licensing see LICENSE.TXT For acknowledgment in scientific publications see ACKNOWLEDGE.TXT Homepage: https://github.com/ALPSCore/ALPSCore URL: https://github.com/ALPSCore/ALPSCore</details>|
	|[amber](http://ambermd.org/amber.html)|chem|20.12-20.15, 20.9-20.15, 22.5-23.5, 18.14-18.17|<details><summary>Expand</summary>Description: Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction. Homepage: http://ambermd.org/amber.html URL: http://ambermd.org/amber.html</details>|
	|[ambertools](https://ambermd.org/AmberTools.php)|chem|20, 21, 22, 23, 23.5|<details><summary>Expand</summary>Description: AmberTools consists of several independently developed packages that work well by themselves, and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations, with either explicit water or generalized Born solvent models. Homepage: https://ambermd.org/AmberTools.php URL: https://ambermd.org/AmberTools.php</details>|
	|[amos](http://amos.sourceforge.net)|-|3.1.0|<details><summary>Expand</summary>Description: The AMOS consortium is committed to the development of open-source whole genome assembly software Homepage: http://amos.sourceforge.net URL: http://amos.sourceforge.net</details>|
	|[ampl-mp](https://github.com/ampl/mp)|-|3.1.0|<details><summary>Expand</summary>Description: An open-source library for mathematical programming. Homepage: https://github.com/ampl/mp URL: https://github.com/ampl/mp</details>|
	|[amrfinderplus](https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus)|-|3.11.18, 3.11.26, 3.12.8|<details><summary>Expand</summary>Description: This software and the accompanying database identify acquired antimicrobial resistance genes in bacterial protein and/or assembled nucleotide sequences as well as known resistance-associated point mutations for several taxa. With AMRFinderPlus we added select members of additional classes of genes such as virulence factors, biocide, heat, acid, and metal resistance genes. Homepage: https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus URL: https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus</details>|
	|[amrplusplus](https://megares.meglab.org/amrplusplus/latest/html/v2)|-|2.0-20200114|<details><summary>Expand</summary>Description: AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. Homepage: https://megares.meglab.org/amrplusplus/latest/html/v2 URL: https://megares.meglab.org/amrplusplus/latest/html/v2</details>|
	|[andi](https://github.com/evolbioinf/andi/)|-|0.14|<details><summary>Expand</summary>Description: This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes. Homepage: https://github.com/evolbioinf/andi/ URL: https://github.com/evolbioinf/andi/</details>|
	|[angsd](http://www.popgen.dk/angsd)|bio|0.933, 0.935, 0.936, 0.939, 0.940|<details><summary>Expand</summary>Description: Program for analysing NGS data. Homepage: http://www.popgen.dk/angsd URL: http://www.popgen.dk/angsd</details>|
	|[annovar](http://annovar.openbioinformatics.org/en/latest/)|bio|20191024|<details><summary>Expand</summary>Description: ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). Homepage: http://annovar.openbioinformatics.org/en/latest/ URL: http://annovar.openbioinformatics.org/en/latest/</details>|
	|[anserini](https://github.com/castorini/anserini)|-|0.9.4|<details><summary>Expand</summary>Description: An open-source information retreival toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h Homepage: https://github.com/castorini/anserini URL: https://github.com/castorini/anserini</details>|
	|[ansys](http://www.ansys.com)|phys|2024R1.03, 2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2, 2024R1|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ansysedt](http://www.ansys.com)|-|2021R2, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ant](https://ant.apache.org/)|tools|1.9.15, 1.10.8, 1.10.14|<details><summary>Expand</summary>Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. Homepage: https://ant.apache.org/ URL: https://ant.apache.org/</details>|
	|[antlr](https://www.antlr2.org/)|-|2.7.7|<details><summary>Expand</summary>Description: ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. Homepage: https://www.antlr2.org/ URL: https://www.antlr2.org/</details>|
	|[ants](https://stnava.github.io/ANTs/)|vis|2.3.2, 2.3.5, 2.4.4, 2.5.0|<details><summary>Expand</summary>Description: ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing, interpreting and visualizing multidimensional data. Homepage: https://stnava.github.io/ANTs/ URL: https://stnava.github.io/ANTs/</details>|
	|[any2fasta](https://github.com/tseemann/any2fasta)|-|0.4.2|<details><summary>Expand</summary>Description: Convert various sequence formats to FASTA Homepage: https://github.com/tseemann/any2fasta URL: https://github.com/tseemann/any2fasta</details>|
	|[apbs](http://www.poissonboltzmann.org/apbs)|chem|1.3|<details><summary>Expand</summary>Description: APBS is a software package for modeling biomolecular solvation through solution of the Poisson-Boltzmann equation (PBE), one of the most popular continuum models for describing electrostatic interactions between molecular solutes in salty, aqueous media. Homepage: http://www.poissonboltzmann.org/apbs URL: http://www.poissonboltzmann.org/apbs</details>|
	|[apptainer](https://apptainer.org)|-|1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4, 1.3.4, 1.3.5|<details><summary>Expand</summary>Description: Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. Homepage: https://apptainer.org URL: https://apptainer.org</details>|
	|[aragorn](http://www.ansikte.se/ARAGORN/)|-|1.2.38, 1.2.41|<details><summary>Expand</summary>Description: a program to detect tRNA genes and tmRNA genes in nucleotide sequences Homepage: http://www.ansikte.se/ARAGORN/ URL: http://www.ansikte.se/ARAGORN/</details>|
	|[arb](https://arblib.org/)|-|2.19.0|<details><summary>Expand</summary>Description: Arb is a C library for arbitrary-precision interval arithmetic. It has full support for both real and complex numbers. The library is thread-safe, portable, and extensively tested. Homepage: https://arblib.org/ URL: https://arblib.org/</details>|
	|[arcs](https://github.com/bcgsc/arcs)|bio|1.2.1, 1.2.5, 1.2.7|<details><summary>Expand</summary>Description: Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. Homepage: https://github.com/bcgsc/arcs URL: https://github.com/bcgsc/arcs</details>|
	|[argtable](http://argtable.sourceforge.net/)|tools|2.13|<details><summary>Expand</summary>Description: Argtable is an ANSI C library for parsing GNU style command line options with a minimum of fuss. Homepage: http://argtable.sourceforge.net/ URL: http://argtable.sourceforge.net/</details>|
	|[arioc](https://sourceforge.net/projects/actc)|-|1.43|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[arks](https://github.com/bcgsc/arcs)|-|1.0.4|<details><summary>Expand</summary>Description: Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. Homepage: https://github.com/bcgsc/arcs URL: https://github.com/bcgsc/arcs</details>|
	|[armadillo](https://arma.sourceforge.net/)|math|9.900.2, 12.6.4|<details><summary>Expand</summary>Description: Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions. Homepage: https://arma.sourceforge.net/ URL: https://arma.sourceforge.net/</details>|
	|[arpack-ng](https://github.com/opencollab/arpack-ng)|math|3.7.0, 3.8.0, 3.9.0, 3.9.1|<details><summary>Expand</summary>Description: ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems. Homepage: https://github.com/opencollab/arpack-ng URL: https://github.com/opencollab/arpack-ng</details>|
	|[arrayfire](https://arrayfire.com/)|-|3.7.3, 3.8.2, 3.9.0|<details><summary>Expand</summary>Description: ArrayFire is a general-purpose library that simplifies the process of developing software that targets parallel and massively-parallel architectures including CPUs, GPUs, and other hardware acceleration devices. Homepage: https://arrayfire.com/ URL: https://arrayfire.com/</details>|
	|[arrow](https://arrow.apache.org)|tools|0.16.0, 0.17.1, 1.0.0, 2.0.0, 5.0.0, 7.0.0, 8.0.0, 9.0.0, 10.0.1, 11.0.0, 12.0.1, 13.0.0, 14.0.0, 14.0.1, 15.0.1, 16.1.0, 17.0.0, 18.1.0, 19.0.1|<details><summary>Expand</summary>Description: Apache Arrow is a cross-language development platform for in-memory data. Homepage: https://arrow.apache.org URL: https://arrow.apache.org</details>|
	|[ascp](http://downloads.asperasoft.com/)|tools|3.5.4|<details><summary>Expand</summary>Description: Aspera ASCP is a high speed file transfer software Homepage: http://downloads.asperasoft.com/ URL: http://downloads.asperasoft.com/</details>|
	|[aspect](https://aspect.geodynamics.org/)|-|2.3.0, 2.4.0|<details><summary>Expand</summary>Description: ASPECT: Advanced Solver for Problems in Earth's ConvecTion. An extensible code written in C++ to support research in simulating convection in the Earth's mantle and elsewhere to provide the geosciences with a well-documented and extensible code base for their research needs and to create an open, inclusive, participatory community providing users and developers with a state-of-the-art, comprehensive software that performs well while being simple to extend. Homepage: https://aspect.geodynamics.org/ URL: https://aspect.geodynamics.org/</details>|
	|[assimp](https://github.com/assimp/assimp)|-|5.0.0, 5.2.5|<details><summary>Expand</summary>Description: Open Asset Import Library (assimp) is a library to import and export various 3d-model-formats including scene-post-processing to generate missing render data. Homepage: https://github.com/assimp/assimp URL: https://github.com/assimp/assimp</details>|
	|[astrid](https://github.com/pranjalv123/ASTRID)|-|2.2.1|<details><summary>Expand</summary>Description: ASTRID-2 is a method for estimating species trees from gene trees. Homepage: https://github.com/pranjalv123/ASTRID URL: https://github.com/pranjalv123/ASTRID</details>|
	|[atat](https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/)|-|3.36|<details><summary>Expand</summary>Description: ATAT is a generic name that refers to a collection of alloy theory tools developped by Axel van de Walle, in collaboration with various research groups and with various sources of financial support. Homepage: https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/ URL: https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/</details>|
	|[atom](https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html)|chem|4.2.7_2|<details><summary>Expand</summary>Description: ATOM is the name of a program originally written (circa 1982) by Sverre Froyen at the University of California at Berkeley, modified starting in 1990 by Norman Troullier and Jose Luis Martins at the University of Minnesota, and currently maintained by Alberto Garcia, who added some features and made substantial structural changes to the April 1990 (5.0) Minnesota version while at Berkeley and elsewhere. Homepage: https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html URL: https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html</details>|
	|[atomicrex](http://atomicrex.gitlab.io/atomicrex/)|chem|1.0.20181114|<details><summary>Expand</summary>Description: atomicrex is a versatile tool for the construction of advanced atomistic models. It is written in C++ and Python. It was primarily developed to fit interatomic potential models. Thanks to its flexible generic structure its application range, however, is much larger. In a general sense, it allows one to develop models that describe a given property as a function of an atomic (or atom-like) configuration. The property in question can be scalar or vectorial in nature, and could represent e.g., total energies and forces, or eventually electronic eigen energies. It thus provides already the basic framework for constructing in the future for example tight binding models. Homepage: http://atomicrex.gitlab.io/atomicrex/ URL: http://atomicrex.gitlab.io/atomicrex/</details>|
	|[atompaw](http://users.wfu.edu/natalie/papers/pwpaw/man.html)|chem|4.1.0.6, 4.2.0.3|<details><summary>Expand</summary>Description: AtomPAW is a Projector-Augmented Wave Dataset Generator that can be used both as a standalone program and a library. Homepage: http://users.wfu.edu/natalie/papers/pwpaw/man.html URL: http://users.wfu.edu/natalie/papers/pwpaw/man.html</details>|
	|[augustus](https://bioinf.uni-greifswald.de/augustus/)|bio|3.3.3, 3.4.0, 3.5.0|<details><summary>Expand</summary>Description: AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences Homepage: https://bioinf.uni-greifswald.de/augustus/ URL: https://bioinf.uni-greifswald.de/augustus/</details>|
	|[autodiff](https://github.com/autodiff/autodiff)|-|0.6.0, 0.6.12|<details><summary>Expand</summary>Description: autodiff is a C++17 library that uses modern and advanced programming techniques to enable automatic computation of derivatives in an efficient, easy, and intuitive way. Homepage: https://github.com/autodiff/autodiff URL: https://github.com/autodiff/autodiff</details>|
	|[autodock-gpu](https://github.com/ccsb-scripps/AutoDock-GPU)|-|1.5.3, 1.6|<details><summary>Expand</summary>Description: OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure. Homepage: https://github.com/ccsb-scripps/AutoDock-GPU URL: https://github.com/ccsb-scripps/AutoDock-GPU</details>|
	|[autodock-vina](https://vina.scripps.edu/)|-|1.2.6|<details><summary>Expand</summary>Description: AutoDock Vina is an open-source program for doing molecular docking. Homepage: https://vina.scripps.edu/ URL: https://vina.scripps.edu/</details>|
	|[autodock_vina](http://vina.scripps.edu/index.html)|chem|1.1.2|<details><summary>Expand</summary>Description: AutoDock Vina is an open-source program for doing molecular docking. Homepage: http://vina.scripps.edu/index.html URL: http://vina.scripps.edu/index.html</details>|
	|[bacio](https://github.com/NOAA-EMC/NCEPLIBS-bacio/)|-|2.6.0|<details><summary>Expand</summary>Description: This library performs binary I/O for the NCEP models, processing unformatted byte-addressable data records, and transforming the little endian files and big endian files. This is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-bacio/ URL: https://github.com/NOAA-EMC/NCEPLIBS-bacio/</details>|
	|[bamm](http://bamm-project.org/)|bio|2.5.0|<details><summary>Expand</summary>Description: BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates. It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of lineage diversification and trait evolution. Homepage: http://bamm-project.org/ URL: http://bamm-project.org/</details>|
	|[bamtools](https://github.com/pezmaster31/bamtools)|bio|2.5.1, 2.5.2|<details><summary>Expand</summary>Description: BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files. Homepage: https://github.com/pezmaster31/bamtools URL: https://github.com/pezmaster31/bamtools</details>|
	|[bamutil](http://genome.sph.umich.edu/wiki/BamUtil)|bio|1.0.14|<details><summary>Expand</summary>Description: BamUtil is a repository that contains several programs that perform operations on SAM/BAM files. All of these programs are built into a single executable, bam. Homepage: http://genome.sph.umich.edu/wiki/BamUtil URL: http://genome.sph.umich.edu/wiki/BamUtil</details>|
	|[barracuda](https://sourceforge.net/projects/seqbarracuda/)|-|0.7.107h|<details><summary>Expand</summary>Description: Barracuda is a high-speed sequence aligner based on Sanger's BWA and utilizes the latest Nvidia CUDA architecture for accelerating alignments of sequence reads generated by next-generation sequencers. Homepage: https://sourceforge.net/projects/seqbarracuda/ URL: https://sourceforge.net/projects/seqbarracuda/</details>|
	|[barrnap](https://github.com/tseemann/barrnap)|-|0.9|<details><summary>Expand</summary>Description: Barrnap predicts the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), metazoan mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S). It takes FASTA DNA sequence as input, and write GFF3 as output. It uses the new nhmmer tool that comes with HMMER 3.1 for HMM searching in RNA:DNA style. Multithreading is supported and one can expect roughly linear speed-ups with more CPUs. Homepage: https://github.com/tseemann/barrnap URL: https://github.com/tseemann/barrnap</details>|
	|[bayesass](http://www.rannala.org/?page_id=245)|bio|3.0.4|<details><summary>Expand</summary>Description: BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes Homepage: http://www.rannala.org/?page_id=245 URL: http://www.rannala.org/?page_id=245</details>|
	|[bayesass3-snps](http://www.rannala.org/inference-of-recent-migration)|-|1.1|<details><summary>Expand</summary>Description: Modification of BayesAss 3.0.4 to allow handling of large SNP datasets generated via methods such as RADseq protocols. This is an enhancement of the code produced by the Rannala Research Group: http://www.rannala.org/inference-of-recent-migration Most changes will not be obvious to the average user. However, the most important changes will now allow the user to input large SNP datasets that previously would have caused BayesAss to crash with a "segmentation fault" error. The upper limit of 420 loci has been removed. Homepage: https://github.com/stevemussmann/BayesAss3-SNPs URL: https://github.com/stevemussmann/BayesAss3-SNPs</details>|
	|[bayescan](http://cmpg.unibe.ch/software/BayeScan/)|bio|2.1|<details><summary>Expand</summary>Description: BayeScan aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. Homepage: http://cmpg.unibe.ch/software/BayeScan/ URL: http://cmpg.unibe.ch/software/BayeScan/</details>|
	|[baypass](http://www1.montpellier.inra.fr/CBGP/software/baypass/)|bio|2.1, 2.2|<details><summary>Expand</summary>Description: The package BayPass is a population genomics software which is primarily aimed at identifying genetic markers subjected to selection and/or associated to population-specific covariates (e.g., environmental variables, quantitative or categorical phenotypic characteristics). Homepage: http://www1.montpellier.inra.fr/CBGP/software/baypass/ URL: http://www1.montpellier.inra.fr/CBGP/software/baypass/</details>|
	|[bazel](https://bazel.io/)|tools|3.6.0|<details><summary>Expand</summary>Description: Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software. Homepage: https://bazel.io/ URL: https://bazel.io/</details>|
	|[bbmap](https://sourceforge.net/projects/bbmap/)|bio|37.78, 38.86, 39.06|<details><summary>Expand</summary>Description: BBMap short read aligner, and other bioinformatic tools. Homepage: https://sourceforge.net/projects/bbmap/ URL: https://sourceforge.net/projects/bbmap/</details>|
	|[bcftools](https://www.htslib.org/)|bio|1.9, 1.10.2, 1.11, 1.13, 1.16, 1.18, 1.19|<details><summary>Expand</summary>Description: Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[bcl-convert](https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html)|-|4.2.4, 4.2.7|<details><summary>Expand</summary>Description: The Illumina BCL Convert v4.0 is a standalone local software app that converts the Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files. Homepage: https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html URL: https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html</details>|
	|[bcl2fastq2](https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html)|bio|2.20.0|<details><summary>Expand</summary>Description: bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by Illumina sequencing systems to standard FASTQ file formats for downstream analysis. Homepage: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html URL: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html</details>|
	|[beagle](https://faculty.washington.edu/browning/beagle/beagle.html)|-|5.4-240301|<details><summary>Expand</summary>Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers. Homepage: https://faculty.washington.edu/browning/beagle/beagle.html URL: https://faculty.washington.edu/browning/beagle/beagle.html</details>|
	|[beagle-lib](https://github.com/beagle-dev/beagle-lib)|bio|3.1.2, 4.0.0, 4.0.1|<details><summary>Expand</summary>Description: beagle-lib is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. Homepage: https://github.com/beagle-dev/beagle-lib URL: https://github.com/beagle-dev/beagle-lib</details>|
	|[beast](http://beast2.org)|bio|2.6.3, 2.7.5|<details><summary>Expand</summary>Description: BEAST is a cross-platform program for Bayesian MCMC analysis of molecular sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted proportional to its posterior probability. Homepage: http://beast2.org URL: http://beast2.org</details>|
	|[bedops](http://bedops.readthedocs.io/en/latest/index.html)|-|2.4.39, 2.4.41|<details><summary>Expand</summary>Description: BEDOPS is an open-source command-line toolkit that performs highly efficient and scalable Boolean and other set operations, statistical calculations, archiving, conversion and other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for distributing whole-genome analyses across a computational cluster. Homepage: http://bedops.readthedocs.io/en/latest/index.html URL: http://bedops.readthedocs.io/en/latest/index.html</details>|
	|[bedtools](https://github.com/arq5x/bedtools2)|bio|2.29.2, 2.30.0, 2.31.0|<details><summary>Expand</summary>Description: The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM. Homepage: https://github.com/arq5x/bedtools2 URL: https://github.com/arq5x/bedtools2</details>|
	|[beef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|chem|0.1.1|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: http://suncat.stanford.edu/facility/software/functional URL: http://suncat.stanford.edu/facility/software/functional</details>|
	|[berkeleygw](https://arxiv.org/abs/1111.4429).)|phys|2.1.0, 3.0.1, 4.0|<details><summary>Expand</summary>Description: The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle properties and the optical responses of a large variety of materials from bulk periodic crystals to nanostructures such as slabs, wires and molecules. We ask that you please cite the following papers when using results obtained from the BerkeleyGW package: - Mark S. Hybertsen and Steven G. Louie, Electron correlation in semiconductors and insulators: Band gaps and quasiparticle energies, Phys. Rev. B 34, 5390 (1986). - Michael Rohlfing and Steven G. Louie, Electron-hole excitations and optical spectra from first principles, Phys. Rev. B 62, 4927 (2000). - Jack Deslippe, Georgy Samsonidze, David A. Strubbe, Manish Jain, Marvin L. Cohen, and Steven G. Louie, BerkeleyGW: A Massively Parallel Computer Package for the Calculation of the Quasiparticle and Optical Properties of Materials and Nanostructures, Comput. Phys. Commun. 183, 1269 (2012) (https://arxiv.org/abs/1111.4429). Homepage: http://www.berkeleygw.org URL: http://www.berkeleygw.org</details>|
	|[bgcdist](https://sites.google.com/site/bgcsoftware/)|-|1.03|<details><summary>Expand</summary>Description: bgc implements Bayesian estimation of genomic clines to quantify introgression at many loci. Models are available for linked loci, genotype uncertainty and sequence errors. Models for genotype uncertainty and sequence error can be used to estimate genomic cline parameters from next-generation sequence data. Homepage: https://sites.google.com/site/bgcsoftware/ URL: https://sites.google.com/site/bgcsoftware/</details>|
	|[bgen-lib](https://enkre.net/cgi-bin/code/bgen)|-|1.1.7|<details><summary>Expand</summary>Description: This repository contains a reference implementation of the BGEN format, written in C++. The library can be used as the basis for BGEN support in other software, or as a reference for developers writing their own implementations of the BGEN format. What's included? This repository contains the library itself, a set of example data files, and a number of example programs (e.g. bgen_to_vcf) that demonstrate the use of the library API. In addition, a number of utilities built using the library are also included in this repository: bgenix - a tool to index and efficiently retrieve subsets of a BGEN file. cat-bgen - a tool to efficiently concatenate BGEN files. edit-bgen - a tool to edit BGEN file metadata. An R package called rbgen is also constructed in the build directory. See the rbgen wiki page for more information on using this package. Homepage: https://enkre.net/cgi-bin/code/bgen URL: https://enkre.net/cgi-bin/code/bgen</details>|
	|[bifrost](https://github.com/pmelsted/bifrost)|-|1.0.5|<details><summary>Expand</summary>Description: Parallel construction, indexing and querying of colored and compacted de Bruijn graphs Homepage: https://github.com/pmelsted/bifrost URL: https://github.com/pmelsted/bifrost</details>|
	|[bigdft](http://bigdft.org/Wiki/index.php?title=BigDFT_website)|chem|1.8.3, 1.9.5|<details><summary>Expand</summary>Description: BigDFT is a DFT massively parallel electronic structure code using a wavelet basis set with the capability to use a linear scaling method. Wavelets form a real space basis set distributed on an adaptive mesh (two levels of resolution in our implementation). GTH or HGH pseudopotentials are used to remove the core electrons. The code BigDFT is available in ABINIT v5.5 and higher but can also be downloaded in a standalone version from the website. Thanks to our Poisson solver based on a Green function formalism, periodic systems, surfaces and isolated systems can be simulated with explicit boundary conditions. The Poisson solver can also be downloaded and used independently and is integrated in ABINIT, Octopus and CP2K. The code is free software, available under GNU-GPL license and the BigDFT developer community encourages anyone willing to contribute to join the team. Homepage: http://bigdft.org/Wiki/index.php?title=BigDFT_website URL: http://bigdft.org/Wiki/index.php?title=BigDFT_website</details>|
	|[bio-searchio-hmmer](http://hmmer.org.)|-|1.7.3|<details><summary>Expand</summary>Description: Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org. Homepage: https://metacpan.org/pod/Bio::SearchIO::hmmer3 URL: https://metacpan.org/pod/Bio::SearchIO::hmmer3</details>|
	|[bioawk](https://github.com/lh3/bioawk)|-|1.0|<details><summary>Expand</summary>Description: Bioawk is an extension to Brian Kernighan's awk, adding the support of several common biological data formats, including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. Homepage: https://github.com/lh3/bioawk URL: https://github.com/lh3/bioawk</details>|
	|[biobloomtools](https://github.com/bcgsc/biobloom)|-|2.3.2-20200731, 2.3.3|<details><summary>Expand</summary>Description: BioBloom Tools (BBT) provides the means to create filters for a given reference and then to categorize sequences. This methodology is faster than alignment but does not provide mapping locations. BBT was initially intended to be used for pre-processing and QC applications like contamination detection, but is flexible to accommodate other purposes. This tool is intended to be a pipeline component to replace costly alignment steps. Homepage: https://github.com/bcgsc/biobloom URL: https://github.com/bcgsc/biobloom</details>|
	|[bioperl](https://bioperl.org/)|bio|1.7.7, 1.7.8|<details><summary>Expand</summary>Description: Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects. Homepage: https://bioperl.org/ URL: https://bioperl.org/</details>|
	|[biopp](https://github.com/BioPP/bpp-core)|-|2.4.1|<details><summary>Expand</summary>Description: Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics. Bio++ is Object Oriented and is designed to be both easy to use and computer efficient. Bio++ intends to help programmers to write computer expensive programs, by providing them a set of re-usable tools. Homepage: https://github.com/BioPP/bpp-core URL: https://github.com/BioPP/bpp-core</details>|
	|[bismark](http://www.bioinformatics.babraham.ac.uk/projects/bismark/)|-|0.22.3, 0.24.1|<details><summary>Expand</summary>Description: A tool to map bisulfite converted sequence reads and determine cytosine methylation states Homepage: http://www.bioinformatics.babraham.ac.uk/projects/bismark/ URL: http://www.bioinformatics.babraham.ac.uk/projects/bismark/</details>|
	|[bison](https://www.gnu.org/software/bison)|-|3.7.1, 3.8.2|<details><summary>Expand</summary>Description: Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables. Homepage: https://www.gnu.org/software/bison URL: https://www.gnu.org/software/bison</details>|
	|[blacs](http://www.netlib.org/blacs/)|math|1.1|<details><summary>Expand</summary>Description: The BLACS (Basic Linear Algebra Communication Subprograms) project is an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface that may be implemented efficiently and uniformly across a large range of distributed memory platforms. Homepage: http://www.netlib.org/blacs/ URL: http://www.netlib.org/blacs/</details>|
	|[blasr](https://github.com/PacificBiosciences/blasr)|bio|5.3.3|<details><summary>Expand</summary>Description: The PacBio long read aligner Homepage: https://github.com/PacificBiosciences/blasr URL: https://github.com/PacificBiosciences/blasr</details>|
	|[blasr_libcpp](https://github.com/PacificBiosciences/blasr_libcpp)|bio|5.3.4|<details><summary>Expand</summary>Description: Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for analyzing PacBio sequences Homepage: https://github.com/PacificBiosciences/blasr_libcpp URL: https://github.com/PacificBiosciences/blasr_libcpp</details>|
	|[blast](http://blast.ncbi.nlm.nih.gov/)|bio|2.2.26|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: http://blast.ncbi.nlm.nih.gov/ URL: http://blast.ncbi.nlm.nih.gov/</details>|
	|[blast+](https://blast.ncbi.nlm.nih.gov/)|bio|2.10.1, 2.11.0, 2.12.0, 2.13.0, 2.14.0, 2.14.1|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: https://blast.ncbi.nlm.nih.gov/ URL: https://blast.ncbi.nlm.nih.gov/</details>|
	|[blat](https://genome.ucsc.edu/goldenPath/help/blatSpec.html)|bio|3.5, 3.7|<details><summary>Expand</summary>Description: BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more. Homepage: https://genome.ucsc.edu/goldenPath/help/blatSpec.html URL: https://genome.ucsc.edu/goldenPath/help/blatSpec.html</details>|
	|[blender](https://www.blender.org/)|vis|2.92.0, 3.6.0, 4.0.2, 4.2.8|<details><summary>Expand</summary>Description: Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. Homepage: https://www.blender.org/ URL: https://www.blender.org/</details>|
	|[blis](https://github.com/flame/blis/)|-|0.8.1, 0.9.0, 1.0|<details><summary>Expand</summary>Description: BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries. Homepage: https://github.com/flame/blis/ URL: https://github.com/flame/blis/</details>|
	|[blitz++](https://github.com/blitzpp/blitz)|tools|1.0.2|<details><summary>Expand</summary>Description: Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++ with a speed comparable to Fortran implementations, while preserving an object-oriented interface Homepage: https://github.com/blitzpp/blitz URL: https://github.com/blitzpp/blitz</details>|
	|[blosc](https://www.blosc.org/)|-|1.17.1, 1.21.3|<details><summary>Expand</summary>Description: Blosc, an extremely fast, multi-threaded, meta-compressor library Homepage: https://www.blosc.org/ URL: https://www.blosc.org/</details>|
	|[blosc2](https://www.blosc.org/)|-|2.10.3|<details><summary>Expand</summary>Description: Blosc, an extremely fast, multi-threaded, meta-compressor library Homepage: https://www.blosc.org/ URL: https://www.blosc.org/</details>|
	|[bmtagger](https://www.google.ca/search?q=bmtagger)|-|3.101|<details><summary>Expand</summary>Description: Best Match Tagger for removing human reads from metagenomics datasets Homepage: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/ URL: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/</details>|
	|[bolt-lmm](https://data.broadinstitute.org/alkesgroup/BOLT-LMM/)|bio|2.3.4, 2.4|<details><summary>Expand</summary>Description: The BOLT-LMM software package currently consists of two main algorithms, the BOLT-LMM algorithm for mixed model association testing, and the BOLT-REML algorithm for variance components analysis (i.e., partitioning of SNP-heritability and estimation of genetic correlations). The BOLT-LMM algorithm computes statistics for testing association between phenotype and genotypes using a linear mixed model (LMM). The BOLT-REML algorithm estimates heritability explained by genotyped SNPs and genetic correlations among multiple traits measured on the same set of individuals. Like the GCTA software. Homepage: https://data.broadinstitute.org/alkesgroup/BOLT-LMM/ URL: https://data.broadinstitute.org/alkesgroup/BOLT-LMM/</details>|
	|[boltztrap](http://www.icams.de/content/departments/cmat/boltztrap/)|phys|1.2.5|<details><summary>Expand</summary>Description: Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport coefficients. Homepage: http://www.icams.de/content/departments/cmat/boltztrap/ URL: http://www.icams.de/content/departments/cmat/boltztrap/</details>|
	|[boost](https://www.boost.org/)|tools|1.72.0, 1.76.0, 1.80.0, 1.82.0, 1.85.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-build](https://www.boost.org/)|-|1.80.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-mpi](https://www.boost.org/)|tools|1.72.0, 1.80.0, 1.82.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[bowtie](http://bowtie-bio.sourceforge.net/index.shtml)|bio|1.3.0|<details><summary>Expand</summary>Description: Bowtie is an ultrafast, memory-efficient short read aligner. It aligns short DNA sequences (reads) to the human genome. Homepage: http://bowtie-bio.sourceforge.net/index.shtml URL: http://bowtie-bio.sourceforge.net/index.shtml</details>|
	|[bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)|bio|2.4.1, 2.4.2, 2.4.4, 2.5.1, 2.5.2, 2.5.4|<details><summary>Expand</summary>Description: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. Homepage: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml URL: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</details>|
	|[bpp](https://github.com/bpp/bpp)|-|4.3.8, 4.6.1|<details><summary>Expand</summary>Description: The aim of this project is to implement a versatile high-performance version of the BPP software. BPP implements four methods: * Estimation of the parameters of species divergence times and population sizes under the multi-species coalescent (MSC) model when the species phylogeny is given * Inference of the species tree when the assignments are given by the user * Species delimitation using a user-specified guide treed * Joint species delimitation and species tree estimation Homepage: https://github.com/bpp/bpp URL: https://github.com/bpp/bpp</details>|
	|[bracken](https://ccb.jhu.edu/software/bracken/)|-|2.6.0, 2.7, 3.0|<details><summary>Expand</summary>Description: Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample. Braken uses the taxonomy labels assigned by Kraken, a highly accurate metagenomics classification algorithm, to estimate the number of reads originating from each species present in a sample. Kraken classifies reads to the best matching location in the taxonomic tree, but does not estimate abundances of species. We use the Kraken database itself to derive probabilities that describe how much sequence from each genome is identical to other genomes in the database, and combine this information with the assignments for a particular sample to estimate abundance at the species level, the genus level, or above. Combined with the Kraken classifier, Bracken produces accurate species- and genus-level abundance estimates even when a sample contains two or more near-identical species. NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the default kmer length is different depending on the version of Kraken used. If you use Kraken 1 defaults, specify 31 as the kmer length. If you use Kraken 2 defaults, specify 35 as the kmer length. Homepage: https://ccb.jhu.edu/software/bracken/ URL: https://ccb.jhu.edu/software/bracken/</details>|
	|[breseq](https://barricklab.org/breseq)|-|0.35.2, 0.36.1, 0.38.2|<details><summary>Expand</summary>Description: breseq is a computational pipeline for the analysis of short-read re-sequencing data Homepage: https://barricklab.org/breseq URL: https://barricklab.org/breseq</details>|
	|[brotli](https://github.com/google/brotli)|-|1.0.9|<details><summary>Expand</summary>Description: Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932. Homepage: https://github.com/google/brotli URL: https://github.com/google/brotli</details>|
	|[brunsli](https://github.com/google/brunsli/)|-|0.1|<details><summary>Expand</summary>Description: Brunsli is a lossless JPEG repacking library. Homepage: https://github.com/google/brunsli/ URL: https://github.com/google/brunsli/</details>|
	|[btllib](https://github.com/bcgsc/btllib)|-|1.6.2|<details><summary>Expand</summary>Description: Bioinformatics Technology Lab common code library Homepage: https://github.com/bcgsc/btllib URL: https://github.com/bcgsc/btllib</details>|
	|[bufr](https://github.com/NOAA-EMC/NCEPLIBS-bufr/)|-|12.0.1|<details><summary>Expand</summary>Description: The NCEPLIBS-bufr library contains routines and utilites for working with the WMO BUFR format. It is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-bufr/ URL: https://github.com/NOAA-EMC/NCEPLIBS-bufr/</details>|
	|[bufrlib](https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB)|geo|11.3.0.2|<details><summary>Expand</summary>Description: NCEP BUFRLIB software to encode or decode BUFR messages. It is not intended to be a primer on the BUFR code form itself. Homepage: https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB URL: https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB</details>|
	|[bullet](http://bulletphysics.org/)|-|2.89, 3.24|<details><summary>Expand</summary>Description: Bullet professional 3D Game Multiphysics Library provides state of the art collision detection, soft body and rigid body dynamics. Homepage: http://bulletphysics.org/ URL: http://bulletphysics.org/</details>|
	|[bustools](https://github.com/BUStools/bustools)|-|0.40.0|<details><summary>Expand</summary>Description: bustools is a program for manipulating BUS files for single cell RNA-Seq datasets. It can be used to error correct barcodes, collapse UMIs, produce gene count or transcript compatibility count matrices, and is useful for many other tasks. See the kallisto bustools website for examples and instructions on how to use bustools as part of a single-cell RNA-seq workflow. Homepage: https://github.com/BUStools/bustools URL: https://github.com/BUStools/bustools</details>|
	|[bwa](http://bio-bwa.sourceforge.net/)|bio|0.7.17, 0.7.18|<details><summary>Expand</summary>Description: Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. Homepage: http://bio-bwa.sourceforge.net/ URL: http://bio-bwa.sourceforge.net/</details>|
	|[bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)|-|2.2.1|<details><summary>Expand</summary>Description: The tool bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case, dataset and the running machine. The original bwa was developed by Heng Li (@lh3). Performance enhancement in bwa-mem2 was primarily done by Vasimuddin Md (@yuk12) and Sanchit Misra (@sanchit-misra) from Parallel Computing Lab, Intel. bwa-mem2 is distributed under the MIT license. Homepage: https://github.com/bwa-mem2/bwa-mem2 URL: https://github.com/bwa-mem2/bwa-mem2</details>|
	|[bwidget](https://core.tcl-lang.org/bwidget/home)|-|1.9.14|<details><summary>Expand</summary>Description: The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces. Homepage: https://core.tcl-lang.org/bwidget/home URL: https://core.tcl-lang.org/bwidget/home</details>|
	|[caf](http://actor-framework.org/)|-|0.18.3|<details><summary>Expand</summary>Description: An open source implementation of the actor model in C++ Homepage: http://actor-framework.org/ URL: http://actor-framework.org/</details>|
	|[cafe5](https://github.com/hahnlab/CAFE5)|-|5.1.0|<details><summary>Expand</summary>Description: Software for Computational Analysis of gene Family Evolution The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa. Homepage: https://github.com/hahnlab/CAFE5 URL: https://github.com/hahnlab/CAFE5</details>|
	|[campari](http://campari.sourceforge.net/V5/index.html)|-|5.0|<details><summary>Expand</summary>Description: CAMPARI is a joint package for performing and analyzing molecular simulations, in particular of systems of biological relevance. It focuses on a wide availability of algorithms for (advanced) sampling and is capable of combining Monte Carlo and molecular dynamics in seamless fashion. Homepage: http://campari.sourceforge.net/V5/index.html URL: http://campari.sourceforge.net/V5/index.html</details>|
	|[cantera](https://github.com/Cantera/cantera)|chem|2.4.0, 2.5.1, 2.6.0|<details><summary>Expand</summary>Description: Chemical kinetics, thermodynamics, and transport tool suite Homepage: https://github.com/Cantera/cantera URL: https://github.com/Cantera/cantera</details>|
	|[canu](https://github.com/marbl/canu)|bio|2.0, 2.1.1, 2.2|<details><summary>Expand</summary>Description: Canu is a fork of the Celera Assembler, designed for high-noise single-molecule sequencing (such as the PacBio RS II or Oxford Nanopore MinION). Canu is a hierarchical assembly pipeline which runs in four steps: Detect overlaps in high-noise sequences using MHAP Generate corrected sequence consensus Trim corrected sequences Assemble trimmed corrected sequences Homepage: https://github.com/marbl/canu URL: https://github.com/marbl/canu</details>|
	|[cap3](http://seq.cs.iastate.edu/)|bio|20151002|<details><summary>Expand</summary>Description: CAP3 assembly program Homepage: http://seq.cs.iastate.edu/ URL: http://seq.cs.iastate.edu/</details>|
	|[capnproto](https://capnproto.org)|-|0.7.0, 1.0.2|<details><summary>Expand</summary>Description: Cap'n Proto is an insanely fast data interchange format and capability-based RPC system. Homepage: https://capnproto.org URL: https://capnproto.org</details>|
	|[casacore](https://github.com/casacore/casacore)|-|3.6.1|<details><summary>Expand</summary>Description: A suite of C++ libraries for radio astronomy data processing. The ephemerides data needs to be in DATA_DIR and the location must be specified at runtime. Thus user's can update them. Homepage: https://github.com/casacore/casacore URL: https://github.com/casacore/casacore</details>|
	|[casadi](https://github.com/casadi/casadi)|-|3.6.3, 3.6.7|<details><summary>Expand</summary>Description: CasADi is an open-source tool for nonlinear optimization and algorithmic differentiation. Homepage: https://github.com/casadi/casadi URL: https://github.com/casadi/casadi</details>|
	|[casper](http://best.snu.ac.kr/casper/)|bio|0.8.2|<details><summary>Expand</summary>Description: CASPER (Context-Aware Scheme for Paired-End Read) is state-of-the art merging tool in terms of accuracy and robustness. Using this sophisticated merging method, we could get elongated reads from the forward and reverse reads. Homepage: http://best.snu.ac.kr/casper/ URL: http://best.snu.ac.kr/casper/</details>|
	|[catch2](https://github.com/catchorg/Catch2)|-|2.11.0, 3.2.1|<details><summary>Expand</summary>Description: A modern, C++-native, header-only, test framework for unit-tests, TDD and BDD - using C++11, C++14, C++17 and later (or C++03 on the Catch1.x branch) Homepage: https://github.com/catchorg/Catch2 URL: https://github.com/catchorg/Catch2</details>|
	|[ccfits](http://heasarc.gsfc.nasa.gov/fitsio/CCfits/)|vis|2.5|<details><summary>Expand</summary>Description: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. Homepage: http://heasarc.gsfc.nasa.gov/fitsio/CCfits/ URL: http://heasarc.gsfc.nasa.gov/fitsio/CCfits/</details>|
	|[ccsm](http://www.cesm.ucar.edu/models/ccsm4.0/)|geo|4_0_a02|<details><summary>Expand</summary>Description: The Community Climate System Model (CCSM) is a coupled climate model for simulating the earth's climate system. Composed of four separate models simultaneously simulating the earth's atmosphere, ocean, land surface and sea-ice, and one central coupler component, the CCSM allows researchers to conduct fundamental research into the earth's past, present and future climate states. Homepage: http://www.cesm.ucar.edu/models/ccsm4.0/ URL: http://www.cesm.ucar.edu/models/ccsm4.0/</details>|
	|[cd-hit](http://weizhongli-lab.org/cd-hit/)|bio|4.8.1|<details><summary>Expand</summary>Description: CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences. Homepage: http://weizhongli-lab.org/cd-hit/ URL: http://weizhongli-lab.org/cd-hit/</details>|
	|[cdbfasta](https://sourceforge.net/projects/cdbfasta)|-|0.99|<details><summary>Expand</summary>Description: Fasta file indexing and retrival tool Homepage: https://sourceforge.net/projects/cdbfasta URL: https://sourceforge.net/projects/cdbfasta</details>|
	|[cdo](https://code.zmaw.de/projects/cdo)|geo|1.9.8, 1.9.10, 2.0.4, 2.0.5, 2.2.1, 2.2.2|<details><summary>Expand</summary>Description: CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data. Homepage: https://code.zmaw.de/projects/cdo URL: https://code.zmaw.de/projects/cdo</details>|
	|[cegma](https://github.com/KorfLab/CEGMA_v2)|-|2.5|<details><summary>Expand</summary>Description: CEGMA (Core Eukaryotic Genes Mapping Approach) is a pipeline for building a set of high reliable set of gene annotations in virtually any eukaryotic genome. The strategy relies on a simple fact: some highly conserved proteins are encoded in essentially all eukaryotic genomes. We use the KOGs database to build a set of these highly conserved ubiquitous proteins. We define a set of 458 core proteins, and the protocol, CEGMA, to find orthologs of the core proteins in new genomes and to determine their exon-intron structures. The procedure uses information from the core genes of six model organisms by first using TBLASTN to identify candidate regions in a new genome. It then proposes and redefines gene structures using a iterative combination of GeneWise, HMMER and geneid. The system includes the use of a profile for each core gene to ensure the reliability of the final predicted gene structure. CEGMA source code, compiled binaries and documentation are available under the GNU GENERAL PUBLIC LICENSE. Homepage: https://github.com/KorfLab/CEGMA_v2 URL: https://github.com/KorfLab/CEGMA_v2</details>|
	|[cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)|bio|2.1.0|<details><summary>Expand</summary>Description: Cell Ranger is a set of analysis pipelines that process Chromium single cell 3-RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis. Cell Ranger 1.2 and later support libraries generated by the Chromium Single Cell 3i-v1 and v2 reagent kits, whereas Cell Ranger 1.1 and earlier do not support v2 libraries. Cell Ranger includes four main gene expression pipelines: - cellranger mkfastq wraps Illumina's bcl2fastq to correctly demultiplex Chromium-prepared sequencing samples and to convert barcode and read data to FASTQ files. - cellranger count takes FASTQ files from cellranger mkfastq and performs alignment, filtering, and UMI counting. - cellranger aggr aggregates outputs from multiple runs of cellranger count, normalizing those runs to the same sequencing depth and then recomputing the gene-barcode matrices and analysis on the combined data. - cellranger reanalyze takes gene-barcode matrices produced by cellranger count or cellranger aggr and reruns the dimensionality reduction, clustering, and gene expression algorithms using tunable parameter settings. These pipelines combine Chromium-specific algorithms with the widely used RNA-seq aligner STAR. Output is delivered in standard BAM, MEX, CSV, HDF5 and HTML formats that are augmented with cellular information. Homepage: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome URL: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</details>|
	|[cellsnp-lite](https://github.com/single-cell-genetics/cellsnp-lite)|-|1.2.2|<details><summary>Expand</summary>Description: cellsnp-lite was initially designed to pileup the expressed alleles in single-cell or bulk RNA-seq data, which can be directly used for donor deconvolution in multiplexed single-cell RNA-seq data, particularly with vireo, which assigns cells to donors and detects doublets, even without genotyping reference. Now besides RNA-seq data, cellsnp-lite could also be applied on DNA-seq and ATAC-seq data, either in bulk or single-cell. Homepage: https://github.com/single-cell-genetics/cellsnp-lite URL: https://github.com/single-cell-genetics/cellsnp-lite</details>|
	|[centrifuge](https://github.com/infphilo/centrifuge)|bio|1.0.4-beta, 1.0.4.2|<details><summary>Expand</summary>Description: [Centrifuge] is a novel microbial classification engine that enables rapid, accurate and sensitive labeling of reads and quantification of species on desktop computers. The system uses a novel indexing scheme based on the Burrows-Wheeler transform (BWT) and the Ferragina-Manzini (FM) index, optimized specifically for the metagenomic classification problem. Centrifuge requires a relatively small index (4.7 GB for all complete bacterial and viral genomes plus the human genome) and classifies sequences at very high speed, allowing it to process the millions of reads from a typical high-throughput DNA sequencing run within a few minutes. Together these advances enable timely and accurate analysis of large metagenomics data sets on conventional desktop computers Homepage: https://github.com/infphilo/centrifuge URL: https://github.com/infphilo/centrifuge</details>|
	|[cereal](https://uscilab.github.io/cereal/)|-|1.3.0, 1.3.2|<details><summary>Expand</summary>Description: cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or used standalone. Homepage: https://uscilab.github.io/cereal/ URL: https://uscilab.github.io/cereal/</details>|
	|[ceres-solver](https://ceres-solver.googlesource.com/ceres-solver/)|-|1.14.0|<details><summary>Expand</summary>Description: Ceres Solver is an open source C++ library for modeling and solving large, complicated optimization problems. It is a feature rich, mature and performant library which has been used in production at Google since 2010. Ceres Solver can solve two kinds of problems: - Non-linear Least Squares problems with bounds constraints. - General unconstrained optimization problems. Homepage: https://ceres-solver.googlesource.com/ceres-solver/ URL: https://ceres-solver.googlesource.com/ceres-solver/</details>|
	|[cesm](http://www.cesm.ucar.edu/models/cesm2/)|geo|2.1.3|<details><summary>Expand</summary>Description: The Community Earth System Model (CESM) is a fully-coupled, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states. Homepage: http://www.cesm.ucar.edu/models/cesm2/ URL: http://www.cesm.ucar.edu/models/cesm2/</details>|
	|[cfitsio](https://heasarc.gsfc.nasa.gov/fitsio/)|vis|3.41, 3.48, 3.49, 4.1.0, 4.3.0, 4.5.0|<details><summary>Expand</summary>Description: CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format. Homepage: https://heasarc.gsfc.nasa.gov/fitsio/ URL: https://heasarc.gsfc.nasa.gov/fitsio/</details>|
	|[cfour](http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download)|chem|2.1|<details><summary>Expand</summary>Description: Serial version of CFOUR (Coupled-Cluster techniques for Computational Chemistry) is a program package for performing high-level quantum chemical calculations on atoms and molecules. The major strength of the program suite is its rather sophisticated arsenal of high-level ab initio methods for the calculation of atomic and molecular properties. Virtually all approaches based on Muller-Plesset (MP) perturbation theory and the coupled-cluster approximation (CC) are available; most of these have complementary analytic derivative approaches within the package as well. CFOUR is not a commercial code. It is rather a program that is undergoing development; new techniques and improvements are constantly being made (mainly in the groups of John F. Stanton (until the end of 2016 at the University of Texas at Austin, USA, now at the Quantum Theory Project of the University of Florida at Gainesville, USA) and Juergen Gauss (Universitat Mainz, Germany)). For non-commercial purposes there is no charge to obtain CFOUR; one must simply sign a license agreement. http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download Homepage: http://www.cfour.de/ URL: http://www.cfour.de/</details>|
	|[cfour-mpi](http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download)|chem|2.1|<details><summary>Expand</summary>Description: CFOUR (Coupled-Cluster techniques for Computational Chemistry) is a program package for performing high-level quantum chemical calculations on atoms and molecules. The major strength of the program suite is its rather sophisticated arsenal of high-level ab initio methods for the calculation of atomic and molecular properties. Virtually all approaches based on Muller-Plesset (MP) perturbation theory and the coupled-cluster approximation (CC) are available; most of these have complementary analytic derivative approaches within the package as well. CFOUR is not a commercial code. It is rather a program that is undergoing development; new techniques and improvements are constantly being made (mainly in the groups of John F. Stanton (until the end of 2016 at the University of Texas at Austin, USA, now at the Quantum Theory Project of the University of Florida at Gainesville, USA) and Juergen Gauss (Universitat Mainz, Germany)). For non-commercial purposes there is no charge to obtain CFOUR; one must simply sign a license agreement. http://slater.chemie.uni-mainz.de/cfour/index.php?n=Main.Download Homepage: http://www.cfour.de/ URL: http://www.cfour.de/</details>|
	|[cgal](https://www.cgal.org/)|math|4.14.3, 5.2.4, 5.5.2|<details><summary>Expand</summary>Description: The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library. Homepage: https://www.cgal.org/ URL: https://www.cgal.org/</details>|
	|[cgns](https://cgns.github.io/)|phys|3.4.1, 4.1.0, 4.1.2, 4.5.0|<details><summary>Expand</summary>Description: The CGNS system is designed to facilitate the exchange of data between sites and applications, and to help stabilize the archiving of aerodynamic data. Homepage: https://cgns.github.io/ URL: https://cgns.github.io/</details>|
	|[chapel-multicore](https://chapel-lang.org)|-|1.25.0, 1.31.0, 2.3.0, 2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ofi](https://chapel-lang.org)|-|1.25.0, 1.31.0|<details><summary>Expand</summary>Description: Chapel is an emerging parallel programming language whose design and development is being led by Cray Inc. Chapel is designed to improve the productivity of high-end computer users while also serving as a portable parallel programming model that can be used on commodity clusters or desktop multicore systems. Chapel strives to vastly improve the programmability of large-scale parallel computers while matching or beating the performance and portability of current programming models like MPI. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ucx](https://chapel-lang.org)|-|1.25.0, 1.31.0, 2.3.0, 2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[chapel-ucx-cuda](https://chapel-lang.org)|-|2.3.0, 2.4.0|<details><summary>Expand</summary>Description: Chapel is a programming language designed for productive parallel computing on large-scale systems. Homepage: https://chapel-lang.org URL: https://chapel-lang.org</details>|
	|[charm-gems](https://github.com/simnibs/charm-gems)|-|1.3.3|<details><summary>Expand</summary>Description: This repository contains the gems C++ code and python bindings used in Freesurfer's Sequence-Adaptive Multimodal SEGmentation (SAMSEG) and in SimNIBS 4.0 Complete Head Anatomy Reconstruction Method (CHARM) to create individualized head models for electric field simulations. Homepage: https://github.com/simnibs/charm-gems URL: https://github.com/simnibs/charm-gems</details>|
	|[chemps2](https://github.com/SebWouters/CheMPS2)|chem|1.8.9|<details><summary>Expand</summary>Description: CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry. Homepage: https://github.com/SebWouters/CheMPS2 URL: https://github.com/SebWouters/CheMPS2</details>|
	|[circos](http://www.circos.ca/)|vis|0.69-9|<details><summary>Expand</summary>Description: Circos is a software package for visualizing data and information. It visualizes data in a circular layout - this makes Circos ideal for exploring relationships between objects or positions. Homepage: http://www.circos.ca/ URL: http://www.circos.ca/</details>|
	|[clang](https://clang.llvm.org/)|tools|9.0.1, 11.0.0, 13.0.1, 15.0.2, 17.0.6, 18.1.8|<details><summary>Expand</summary>Description: C, C++, Objective-C compiler, based on LLVM. Does not include C++ standard library -- use libstdc++ from GCC. Homepage: https://clang.llvm.org/ URL: https://clang.llvm.org/</details>|
	|[clhep](https://proj-clhep.web.cern.ch/proj-clhep/)|math|2.4.1.3, 2.4.4.0, 2.4.6.2, 2.4.7.1|<details><summary>Expand</summary>Description: The CLHEP project is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra. CLHEP is structured in a set of packages independent of any external package. Homepage: https://proj-clhep.web.cern.ch/proj-clhep/ URL: https://proj-clhep.web.cern.ch/proj-clhep/</details>|
	|[clustal-omega](http://www.clustal.org/omega/)|bio|1.2.4|<details><summary>Expand</summary>Description: Clustal Omega is a multiple sequence alignment program for proteins. It produces biologically meaningful multiple sequence alignments of divergent sequences. Evolutionary relationships can be seen via viewing Cladograms or Phylograms Homepage: http://www.clustal.org/omega/ URL: http://www.clustal.org/omega/</details>|
	|[cmake](https://www.cmake.org)|tools|3.18.4, 3.20.1, 3.21.4, 3.22.1, 3.23.1, 3.27.7, 3.31.0|<details><summary>Expand</summary>Description: CMake, the cross-platform, open-source build system. CMake is a family of tools designed to build, test and package software. Homepage: https://www.cmake.org URL: https://www.cmake.org</details>|
	|[cnvnator](https://github.com/abyzovlab/CNVnator)|bio|0.4.1|<details><summary>Expand</summary>Description: a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads Homepage: https://github.com/abyzovlab/CNVnator URL: https://github.com/abyzovlab/CNVnator</details>|
	|[code-server](https://github.com/cdr/code-server)|-|3.5.0, 3.12.0, 4.92.2|<details><summary>Expand</summary>Description: Run VS Code on any machine anywhere and access it in the browser. Homepage: https://github.com/cdr/code-server URL: https://github.com/cdr/code-server</details>|
	|[coinmp](https://projects.coin-or.org/CoinMP)|-|1.8.4|<details><summary>Expand</summary>Description: CoinMP is a C-API library that supports most of the functionality of CLP (Coin LP), CBC (Coin Branch-and-Cut), and CGL (Cut Generation Library) projects. Homepage: https://projects.coin-or.org/CoinMP URL: https://projects.coin-or.org/CoinMP</details>|
	|[colmap](https://github.com/colmap/colmap)|-|3.6|<details><summary>Expand</summary>Description: COLMAP is a general-purpose Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline with a graphical and command-line interface. It offers a wide range of features for reconstruction of ordered and unordered image collections. The software is licensed under the new BSD license. Homepage: https://github.com/colmap/colmap URL: https://github.com/colmap/colmap</details>|
	|[combblas](https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/)|-|1.6.2|<details><summary>Expand</summary>Description: The Combinatorial BLAS (CombBLAS) is an extensible distributed-memory parallel graph library offering a small but powerful set of linear algebra primitives specifically targeting graph analytics. Homepage: https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/ URL: https://people.eecs.berkeley.edu/~aydin/CombBLAS/html/</details>|
	|[comsol](https://www.comsol.com)|phys|5.6, 6.0.0.405, 6.1.0.357, 6.2.0.415, 6.3|<details><summary>Expand</summary>Description: COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems. Homepage: https://www.comsol.com URL: https://www.comsol.com</details>|
	|[coordgenlibs](https://github.com/schrodinger/coordgenlibs)|chem|1.4.2|<details><summary>Expand</summary>Description: Schrodinger-developed 2D Coordinate Generation Homepage: https://github.com/schrodinger/coordgenlibs URL: https://github.com/schrodinger/coordgenlibs</details>|
	|[coretran](https://github.com/leonfoks/coretran#compile)|-|1.0.1|<details><summary>Expand</summary>Description: An easy to follow library to make Fortran easier in general with wrapped interfaces, sorting routines, kD-Trees, and other algorithms to handle scientific data and concepts. The library contains core fortran routines and object-oriented classes. Homepage: https://github.com/leonfoks/coretran#compile URL: https://github.com/leonfoks/coretran#compile</details>|
	|[corset](https://github.com/Oshlack/Corset/wiki)|bio|1.09|<details><summary>Expand</summary>Description: Corset is a command-line software program to go from a de novo transcriptome assembly to gene-level counts. Our software takes a set of reads that have been multi-mapped to the transcriptome (where multiple alignments per read were reported) and hierarchically clusters the transcripts based on the proportion of shared reads and expression patterns. It will report the clusters and gene-level counts for each sample, which are easily tested for differential expression with count based tools such as edgeR and DESeq. Homepage: https://github.com/Oshlack/Corset/wiki URL: https://github.com/Oshlack/Corset/wiki</details>|
	|[cp2k](https://www.cp2k.org/)|chem|7.1, 8.2, 9.1, 2023.1|<details><summary>Expand</summary>Description: CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials. Homepage: https://www.cp2k.org/ URL: https://www.cp2k.org/</details>|
	|[cpmd](http://cpmd.org)|chem|4.3|<details><summary>Expand</summary>Description: The CPMD code is a parallelized plane wave / pseudopotential implementation of Density Functional Theory, particularly designed for ab-initio molecular dynamics. Homepage: http://cpmd.org URL: http://cpmd.org</details>|
	|[cppzmq](https://github.com/zeromq/cppzmq)|-|4.7.1|<details><summary>Expand</summary>Description: cppzmq is a C++ binding for libzmq Homepage: https://github.com/zeromq/cppzmq URL: https://github.com/zeromq/cppzmq</details>|
	|[cpu_features](https://github.com/google/cpu_features)|-|0.6.0|<details><summary>Expand</summary>Description: A cross-platform C library to retrieve CPU features (such as available instructions) at runtime. Homepage: https://github.com/google/cpu_features URL: https://github.com/google/cpu_features</details>|
	|[cram](https://bitheap.org/cram)|-|0.7|<details><summary>Expand</summary>Description: Cram is a functional testing framework for command line applications. Homepage: https://bitheap.org/cram URL: https://bitheap.org/cram</details>|
	|[crest](https://xtb-docs.readthedocs.io/en/latest/crest.html)|chem|2.11, 2.12, 3.0.1, 3.0.2|<details><summary>Expand</summary>Description: CREST is an utility/driver program for the xtb program. Originally it was designed as conformer sampling program, hence the abbreviation ConformerRotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNnxTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles. Homepage: https://xtb-docs.readthedocs.io/en/latest/crest.html URL: https://xtb-docs.readthedocs.io/en/latest/crest.html</details>|
	|[cromwell](http://cromwell.readthedocs.io/)|tools|58|<details><summary>Expand</summary>Description: Scientific workflow engine designed for simplicity and scalability. Trivially transition between one off use cases to massive scale production environments http://cromwell.readthedocs.io/ Homepage: https://cromwell.readthedocs.io/ URL: https://cromwell.readthedocs.io/</details>|
	|[csblast](https://github.com/soedinglab/csblast/)|-|2.2.4|<details><summary>Expand</summary>Description: Context-specific extension of BLAST that significantly improves sensitivity and alignment quality. Homepage: https://github.com/soedinglab/csblast/ URL: https://github.com/soedinglab/csblast/</details>|
	|[cslib](https://cslib.sandia.gov/index.html)|-|20180813|<details><summary>Expand</summary>Description: The client/server model is a messaging paradigm, used ubiquitously by web-based applications, where one or more clients (e.g. your phone) communicate with a server (e.g. a website). In the context of the CSlib, it can used to couple two scientific applications (apps) together where one app acts as the client and the other as the server. The client sends messages to the server requesting it perform specific computations and return the results. Homepage: https://cslib.sandia.gov/index.html URL: https://cslib.sandia.gov/index.html</details>|
	|[cst](https://www.cst.com/)|phys|2020|<details><summary>Expand</summary>Description: CST offers accurate, efficient computational solutions for electromagnetic design and analysis. CST simulation is user-friendly and enables users to choose the most appropriate method for the design and optimization of devices operating in a wide range of frequencies. Please note that we installed only Graphic frontend and command line interface. You need to provide your own license server in order to be able to run CST. Homepage: https://www.cst.com/ URL: https://www.cst.com/</details>|
	|[csvtk](https://github.com/shenwei356/csvtk)|-|0.23.0|<details><summary>Expand</summary>Description: Similar to FASTA/Q format in field of Bioinformatics, CSV/TSV formats are basic and ubiquitous file formats in both Bioinformatics and data science. People usually use spreadsheet software like MS Excel to process table data. However this is all by clicking and typing, which is not automated and is time-consuming to repeat, especially when you want to apply similar operations with different datasets or purposes. You can also accomplish some CSV/TSV manipulations using shell commands, but more code is needed to handle the header line. Shell commands do not support selecting columns with column names either. csvtk is convenient for rapid data investigation and also easy to integrate into analysis pipelines. It could save you lots of time in (not) writing Python/R scripts. Homepage: https://github.com/shenwei356/csvtk URL: https://github.com/shenwei356/csvtk</details>|
	|[ctffind](https://grigoriefflab.umassmed.edu/ctffind4)|chem|4.1.14|<details><summary>Expand</summary>Description: Program for finding CTFs of electron micrographs. Homepage: https://grigoriefflab.umassmed.edu/ctffind4 URL: https://grigoriefflab.umassmed.edu/ctffind4</details>|
	|[cuba](https://feynarts.de/cuba/)|-|4.2.2|<details><summary>Expand</summary>Description: Cuba a library for multidimensional numerical integration Homepage: https://feynarts.de/cuba/ URL: https://feynarts.de/cuba/</details>|
	|[cubegui](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.4|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube graphical report explorer. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubelib](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.4|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube general purpose C++ library component and command-line tools. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubewriter](https://www.scalasca.org/software/cube-4.x/download.html)|-|4.4.3|<details><summary>Expand</summary>Description: Cube, which is used as performance report explorer for Scalasca and Score-P, is a generic tool for displaying a multi-dimensional performance space consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree can be collapsed or expanded to achieve the desired level of granularity. This module provides the Cube high-performance C writer library component. Homepage: https://www.scalasca.org/software/cube-4.x/download.html URL: https://www.scalasca.org/software/cube-4.x/download.html</details>|
	|[cubic_interpolation](https://github.com/tudo-astroparticlephysics/cubic_interpolation)|-|0.1.5|<details><summary>Expand</summary>Description: A leightweight interpolation library based on boost and eigen. It provides the utilities to handle tables which are runtime intensive to build and reduces interpolation failures to axis transformation. Homepage: https://github.com/tudo-astroparticlephysics/cubic_interpolation URL: https://github.com/tudo-astroparticlephysics/cubic_interpolation</details>|
	|[cuda](https://developer.nvidia.com/cuda-toolkit)|tools|10.1, 10.2|<details><summary>Expand</summary>Description: CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. Homepage: https://developer.nvidia.com/cuda-toolkit URL: https://developer.nvidia.com/cuda-toolkit</details>|
	|[cudnn](https://developer.nvidia.com/cudnn)|math|8.0.3, 8.2.0, 8.6.0.163, 8.7.0.84, 8.9.5.29, 9.2.1.18, 9.5.1.17|<details><summary>Expand</summary>Description: The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. Homepage: https://developer.nvidia.com/cudnn URL: https://developer.nvidia.com/cudnn</details>|
	|[cufflinks](http://cole-trapnell-lab.github.io/cufflinks/)|bio|2.2.1|<details><summary>Expand</summary>Description: Transcript assembly, differential expression, and differential regulation for RNA-Seq Homepage: http://cole-trapnell-lab.github.io/cufflinks/ URL: http://cole-trapnell-lab.github.io/cufflinks/</details>|
	|[cuquantum](https://developer.nvidia.com/cuquantum-sdk)|-|22.05.0.41, 23.06.1.8, 24.03.0.4|<details><summary>Expand</summary>Description: NVIDIA cuQuantum SDK is a high-performance library for quantum information science and beyond Homepage: https://developer.nvidia.com/cuquantum-sdk URL: https://developer.nvidia.com/cuquantum-sdk</details>|
	|[cusparselt](https://docs.nvidia.com/cuda/cusparselt/index.html)|-|0.4.0.7, 0.5.0.1, 0.6.1.0|<details><summary>Expand</summary>Description: NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix Homepage: https://docs.nvidia.com/cuda/cusparselt/index.html URL: https://docs.nvidia.com/cuda/cusparselt/index.html</details>|
	|[custom-ctypes](https://github.com/ComputeCanada/custom_ctypes/)|-|1.1, 1.2|<details><summary>Expand</summary>Description: This repository is to try to fix discovery of libraries with Python's ctypes module. It changes the behavior of find_library to return absolute paths to shared objects rather than just the names. Homepage: https://github.com/ComputeCanada/custom_ctypes/ URL: https://github.com/ComputeCanada/custom_ctypes/</details>|
	|[cutensor](https://developer.nvidia.com/cutensor)|-|1.5.0.3, 1.7.0.1, 2.0.0.7, 2.0.1.2|<details><summary>Expand</summary>Description: cuTENSOR is a high-performance CUDA library for tensor primitives. Homepage: https://developer.nvidia.com/cutensor URL: https://developer.nvidia.com/cutensor</details>|
	|[cvit](https://sourceforge.net/projects/cvit/)|-|1.2.1|<details><summary>Expand</summary>Description: A genomic linkage feature visualization tools based on Perl. Homepage: https://sourceforge.net/projects/cvit/ URL: https://sourceforge.net/projects/cvit/</details>|
	|[dakota](https://dakota.sandia.gov/)|tools|6.13|<details><summary>Expand</summary>Description: The Dakota project delivers both state-of-the-art research and robust, usable software for optimization and UQ. Broadly, the Dakota software's advanced parametric analyses enable design exploration, model calibration, risk analysis, and quantification of margins and uncertainty with computational models. The Dakota toolkit provides a flexible, extensible interface between such simulation codes and its iterative systems analysis methods, which include: - optimization with gradient and nongradient-based methods; - uncertainty quantification with sampling, reliability, stochastic expansion, and epistemic methods; - parameter estimation using nonlinear least squares (deterministic) or Bayesian inference (stochastic); and - sensitivity/variance analysis with design of experiments and parameter study methods. These capabilities may be used on their own or as components within advanced strategies such as hybrid optimization, surrogate-based optimization, mixed integer nonlinear programming, or optimization under uncertainty. Homepage: https://dakota.sandia.gov/ URL: https://dakota.sandia.gov/</details>|
	|[dalton](https://www.daltonprogram.org/www/citation.html.)|chem|2020.0, 2020.1|<details><summary>Expand</summary>Description: The Dalton code is a powerful tool for a wide range of molecular properties at different levels of theory. Any published work arising from use of one of the Dalton2016 programs must acknowledge that by a proper reference, https://www.daltonprogram.org/www/citation.html. Homepage: https://daltonprogram.org/ URL: https://daltonprogram.org/</details>|
	|[das_tool](https://github.com/cmks/DAS_Tool)|-|1.1.6|<details><summary>Expand</summary>Description: DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly. Homepage: https://github.com/cmks/DAS_Tool URL: https://github.com/cmks/DAS_Tool</details>|
	|[db](https://www.oracle.com/technetwork/products/berkeleydb)|-|18.1.32|<details><summary>Expand</summary>Description: Berkeley DB enables the development of custom data management solutions, without the overhead traditionally associated with such custom projects. Homepage: https://www.oracle.com/technetwork/products/berkeleydb URL: https://www.oracle.com/technetwork/products/berkeleydb</details>|
	|[dcm2niix](https://github.com/rordenlab/dcm2niix)|bio|1.0.20200331, 1.0.20210317, 1.0.20230411|<details><summary>Expand</summary>Description: dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format. Homepage: https://github.com/rordenlab/dcm2niix URL: https://github.com/rordenlab/dcm2niix</details>|
	|[dcmtk](https://dicom.offis.de/dcmtk)|-|3.6.7|<details><summary>Expand</summary>Description: DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers. Homepage: https://dicom.offis.de/dcmtk URL: https://dicom.offis.de/dcmtk</details>|
	|[ddt-cpu](https://www.linaroforge.com)|tools|20.2, 22.0.1, 23.1.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining MAP and DDT. Homepage: https://www.linaroforge.com URL: https://www.linaroforge.com</details>|
	|[ddt-gpu](https://www.linaroforge.com)|tools|20.2, 22.0.1, 23.1.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining MAP and DDT, with support for CUDA. Homepage: https://www.linaroforge.com URL: https://www.linaroforge.com</details>|
	|[dealii](http://www.dealii.org/)|math|9.2.0, 9.3.1, 9.4.1|<details><summary>Expand</summary>Description: A C++ software library supporting the creation of finite element codes and an open community of users and developers. Homepage: http://www.dealii.org/ URL: http://www.dealii.org/</details>|
	|[deepvariant](https://github.com/google/deepvariant)|-|1.8.0|<details><summary>Expand</summary>Description: DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file. DeepVariant supports germline variant-calling in diploid organisms. Homepage: https://github.com/google/deepvariant URL: https://github.com/google/deepvariant</details>|
	|[deepvariant-gpu](https://github.com/google/deepvariant)|-|1.8.0|<details><summary>Expand</summary>Description: DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file. DeepVariant supports germline variant-calling in diploid organisms. Homepage: https://github.com/google/deepvariant URL: https://github.com/google/deepvariant</details>|
	|[delft3d](https://svn.oss.deltares.nl)|chem|62441|<details><summary>Expand</summary>Description: Delft3D is a powerful modelling suite focusing primarily on coastal, estuarine, river, rural and urban environments Delft3D FM Suite can simulate storm surges, hurricanes, tsunamis, detailed flows and water levels, waves, sediment transport and morphology, water quality and ecology, and is capable of handling the interactions between these processes. Homepage: https://svn.oss.deltares.nl URL: https://svn.oss.deltares.nl</details>|
	|[delly](https://github.com/dellytools/delly/)|-|0.8.5, 1.1.6, 1.1.8|<details><summary>Expand</summary>Description: Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome. Homepage: https://github.com/dellytools/delly/ URL: https://github.com/dellytools/delly/</details>|
	|[detonate](http://deweylab.biostat.wisc.edu/detonate/)|bio|1.11|<details><summary>Expand</summary>Description: DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation) consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds of genomic sequences. Homepage: http://deweylab.biostat.wisc.edu/detonate/ URL: http://deweylab.biostat.wisc.edu/detonate/</details>|
	|[dftbplus](https://www.dftb-plus.info)|chem|21.1, 21.2, 24.1|<details><summary>Expand</summary>Description: DFTB+ is a fast and efficient versatile quantum mechanical simulation package. It is based on the Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been developed for the DFTB framework so far. Using DFTB+ you can carry out quantum mechanical simulations like with ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of magnitude in speed. Homepage: https://www.dftb-plus.info URL: https://www.dftb-plus.info</details>|
	|[dftd3-lib](https://github.com/dftbplus/dftd3-lib)|chem|0.10|<details><summary>Expand</summary>Description: This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers. The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been converted to free format and encapsulated into modules. Homepage: https://github.com/dftbplus/dftd3-lib URL: https://github.com/dftbplus/dftd3-lib</details>|
	|[dftd4](https://github.com/dftd4/dftd4/)|chem|3.3.0, 3.6.0, 3.7.0|<details><summary>Expand</summary>Description: DFTD4 van-der-Waals functional is generally Applicable Atomic-Charge Dependent London Dispersion Correction. Homepage: https://github.com/dftd4/dftd4/ URL: https://github.com/dftd4/dftd4/</details>|
	|[diamond](https://github.com/bbuchfink/diamond)|bio|0.9.36, 2.0.4, 2.0.9, 2.0.13, 2.0.15, 2.1.6, 2.1.7, 2.1.8|<details><summary>Expand</summary>Description: Accelerated BLAST compatible local sequence aligner Homepage: https://github.com/bbuchfink/diamond URL: https://github.com/bbuchfink/diamond</details>|
	|[dirac](http://www.diracprogram.org)|-|21.1, 23.0|<details><summary>Expand</summary>Description: DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations Homepage: http://www.diracprogram.org URL: http://www.diracprogram.org</details>|
	|[dl_monte](https://gitlab.com/dl_monte)|-|2.07|<details><summary>Expand</summary>Description: General-purpose Monte Carlo molecular simulation Homepage: https://gitlab.com/dl_monte URL: https://gitlab.com/dl_monte</details>|
	|[dl_poly4](http://www.scd.stfc.ac.uk/SCD/44516.aspx)|chem|4.10.0, 5.1.0|<details><summary>Expand</summary>Description: DL_POLY is a general purpose classical molecular dynamics (MD) simulation software developed at Daresbury Laboratory by I.T. Todorov and W. Smith. Homepage: http://www.scd.stfc.ac.uk/SCD/44516.aspx URL: http://www.scd.stfc.ac.uk/SCD/44516.aspx</details>|
	|[dmalloc](http://dmalloc.com/)|tools|5.5.2|<details><summary>Expand</summary>Description: The debug memory allocation or dmalloc library has been designed as a drop in replacement for the system s malloc, realloc, calloc, free and other memory management routines while providing powerful debugging facilities configurable at runtime. These facilities include such things as memory-leak tracking, fence-post write detection, file/line number reporting, and general logging of statistics. The library is reasonably portable having been run successfully on at least the following operating systems: AIX, BSD/OS, DG/UX, Free/Net/OpenBSD, GNU/Hurd, HPUX, Irix, Linux, MS-DOG, NeXT, OSF, SCO, Solaris, SunOS, Ultrix, Unixware, Windoze, and even Unicos on a Cray T3E. It also provides support for the debugging of threaded programs. The package includes the library, configuration scripts, debug utility application, test program, and documentation. Homepage: http://dmalloc.com/ URL: http://dmalloc.com/</details>|
	|[dnmtools](https://github.com/smithlabcode/dnmtools)|-|1.4.4|<details><summary>Expand</summary>Description: DNMTools is a set of tools for analyzing DNA methylation data from high-throughput sequencing experiments, especially whole genome bisulfite sequencing (WGBS), but also reduced representation bisulfite sequencing (RRBS). These tools focus on overcoming the computing challenges imposed by the scale of genome-wide DNA methylation data, which is usually the early parts of data analysis. Homepage: https://github.com/smithlabcode/dnmtools URL: https://github.com/smithlabcode/dnmtools</details>|
	|[dorado](https://github.com/nanoporetech/dorado)|-|0.2.1, 0.2.2, 0.3.0, 0.4.1, 0.4.3, 0.5.3, 0.6.1, 0.7.0, 0.7.2, 0.8.0, 0.8.3, 0.9.5|<details><summary>Expand</summary>Description: Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads. Homepage: https://github.com/nanoporetech/dorado URL: https://github.com/nanoporetech/dorado</details>|
	|[dotnet-core](https://www.microsoft.com/net/)|-|3.1.8, 5.0.12, 6.0.0, 8.0.407, 9.0.202|<details><summary>Expand</summary>Description: .NET is a free, cross-platform, open source developer platform for building many different types of applications. With .NET, you can use multiple languages, editors, and libraries to build for web, mobile, desktop, gaming, and IoT. Contains the SDK and the Runtime. Homepage: https://www.microsoft.com/net/ URL: https://www.microsoft.com/net/</details>|
	|[double-conversion](https://github.com/google/double-conversion)|-|3.1.5, 3.2.1|<details><summary>Expand</summary>Description: Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles. Homepage: https://github.com/google/double-conversion URL: https://github.com/google/double-conversion</details>|
	|[dpc++](https://github.com/intel/llvm)|-|2022-06, 2022-12|<details><summary>Expand</summary>Description: DPC++ is a LLVM-based compiler project that implements compiler and runtime support for the SYCL language. Homepage: https://github.com/intel/llvm URL: https://github.com/intel/llvm</details>|
	|[dragmap](https://github.com/Illumina/DRAGMAP)|-|1.3.0|<details><summary>Expand</summary>Description: Dragmap is the Dragen mapper/aligner Open Source Software. Homepage: https://github.com/Illumina/DRAGMAP URL: https://github.com/Illumina/DRAGMAP</details>|
	|[dssp](https://swift.cmbi.umcn.nl/gv/dssp/)|chem|2.3.0, 3.1.4|<details><summary>Expand</summary>Description: The DSSP algorithm is the standard method for assigning secondary structure to the amino acids of a protein, given the atomic-resolution coordinates of the protein. If you use DSSP, please quote these two articles: 1) A series of PDB related databases for everyday needs. Wouter G Touw, Coos Baakman, Jon Black, Tim AH te Beek, E Krieger, Robbie P Joosten, Gert Vriend. Nucleic Acids Research 2015 January; 43(Database issue): D364-D368. 2) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. Kabsch W, Sander C, Biopolymers. 1983 22 2577-2637. PMID: 6667333; UI: 84128824. Homepage: https://swift.cmbi.umcn.nl/gv/dssp/ URL: https://swift.cmbi.umcn.nl/gv/dssp/</details>|
	|[ecbuild](https://ecbuild.readthedocs.io/)|-|3.8.0|<details><summary>Expand</summary>Description: A CMake-based build system, consisting of a collection of CMake macros and functions that ease the managing of software build systems Homepage: https://ecbuild.readthedocs.io/ URL: https://ecbuild.readthedocs.io/</details>|
	|[eccodes](https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home)|geo|2.15.0, 2.19.0, 2.21.0, 2.22.1, 2.25.0, 2.31.0|<details><summary>Expand</summary>Description: ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding). Homepage: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home URL: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</details>|
	|[edirect](https://www.ncbi.nlm.nih.gov/books/NBK25501/)|-|20.9.20231210|<details><summary>Expand</summary>Description: Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases from a Unix terminal window. Search terms are entered as command-line arguments. Individual operations are connected with Unix pipes to construct multi-step queries. Selected records can then be retrieved in a variety of formats. Homepage: https://www.ncbi.nlm.nih.gov/books/NBK25501/ URL: https://www.ncbi.nlm.nih.gov/books/NBK25501/</details>|
	|[eigen](https://eigen.tuxfamily.org)|math|3.3.7, 3.4.0|<details><summary>Expand</summary>Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms. Homepage: https://eigen.tuxfamily.org URL: https://eigen.tuxfamily.org</details>|
	|[eigensoft](http://www.hsph.harvard.edu/alkes-price/software/)|bio|7.2.1|<details><summary>Expand</summary>Description: The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al. 2006) and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal components analysis to explicitly model ancestry differences between cases and controls along continuous axes of variation; the resulting correction is specific to a candidate marker's variation in frequency across ancestral populations, minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT package has a built-in plotting script and supports multiple file formats and quantitative phenotypes. Homepage: http://www.hsph.harvard.edu/alkes-price/software/ URL: http://www.hsph.harvard.edu/alkes-price/software/</details>|
	|[elastix](http://elastix.isi.uu.nl/)|-|5.0.1|<details><summary>Expand</summary>Description: elastix: a toolbox for rigid and nonrigid registration of images. Homepage: http://elastix.isi.uu.nl/ URL: http://elastix.isi.uu.nl/</details>|
	|[elixir](https://elixir-lang.org/)|tools|1.13|<details><summary>Expand</summary>Description: Elixir is a dynamic, functional language designed for building scalable and maintainable applications. Homepage: https://elixir-lang.org/ URL: https://elixir-lang.org/</details>|
	|[elmerfem](https://www.csc.fi/web/elmer)|-|scc20, 8.4, 9.0|<details><summary>Expand</summary>Description: Elmer is an open source multiphysical simulation software mainly developed by CSC - IT Center for Science (CSC). Elmer development was started 1995 in collaboration with Finnish Universities, research institutes and industry. After it's open source publication in 2005, the use and development of Elmer has become international. Elmer includes physical models of fluid dynamics, structural mechanics, electromagnetics, heat transfer and acoustics, for example. These are described by partial differential equations which Elmer solves by the Finite Element Method (FEM). Homepage: https://www.csc.fi/web/elmer URL: https://www.csc.fi/web/elmer</details>|
	|[elpa](https://elpa.mpcdf.mpg.de/)|math|2020.05.001, 2023.05.001, 2024.05.001|<details><summary>Expand</summary>Description: Eigenvalue SoLvers for Petaflop-Applications. Homepage: https://elpa.mpcdf.mpg.de/ URL: https://elpa.mpcdf.mpg.de/</details>|
	|[emboss](https://emboss.sourceforge.net/)|bio|6.6.0|<details><summary>Expand</summary>Description: EMBOSS is 'The European Molecular Biology Open Software Suite' . EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology (e.g. EMBnet) user community. Homepage: https://emboss.sourceforge.net/ URL: https://emboss.sourceforge.net/</details>|
	|[embree](https://github.com/embree/embree)|-|2.17.7, 3.11.0, 3.13.5, 4.3.0|<details><summary>Expand</summary>Description: Intel Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Intel Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Intel Embree supports runtime code selection to choose the traversal and build algorithms that best matches the instruction set of your CPU. Homepage: https://github.com/embree/embree URL: https://github.com/embree/embree</details>|
	|[energyplus](https://energyplus.net/downloads)|-|9.3.0, 23.2.0|<details><summary>Expand</summary>Description: EnergyPlus is a whole building energy simulation program that engineers, architects, and researchers use to model both energy consumptionfor heating, cooling, ventilation, lighting and plug and process loadsand water use in buildings. Homepage: https://energyplus.net/downloads URL: https://energyplus.net/downloads</details>|
	|[erlangotp](http://www.erlang.org/)|tools|23.3, 24.2|<details><summary>Expand</summary>Description: Erlang is a general-purpose concurrent, garbage-collected programming language and runtime system. Homepage: http://www.erlang.org/ URL: http://www.erlang.org/</details>|
	|[esmf](https://www.earthsystemcog.org/projects/esmf/)|geo|8.0.1, 8.2.0, 8.4.0, 8.6.0, 8.7.0, 8.8.0|<details><summary>Expand</summary>Description: The Earth System Modeling Framework (ESMF) is a suite of software tools for developing high-performance, multi-component Earth science modeling applications. Homepage: https://www.earthsystemcog.org/projects/esmf/ URL: https://www.earthsystemcog.org/projects/esmf/</details>|
	|[espeak-ng](https://github.com/espeak-ng/espeak-ng)|-|1.51|<details><summary>Expand</summary>Description: The eSpeak NG is a compact open source software text-to-speech synthesizer for Linux, Windows, Android and other operating systems. It supports more than 100 languages and accents. It is based on the eSpeak engine created by Jonathan Duddington. Homepage: https://github.com/espeak-ng/espeak-ng URL: https://github.com/espeak-ng/espeak-ng</details>|
	|[etsf_io](http://www.etsf.eu/resources/software/libraries_and_tools)|io|1.0.4|<details><summary>Expand</summary>Description: A library of F90 routines to read/write the ETSF file format has been written. It is called ETSF_IO and available under LGPL. Homepage: http://www.etsf.eu/resources/software/libraries_and_tools URL: http://www.etsf.eu/resources/software/libraries_and_tools</details>|
	|[etsf_io-mpi](http://www.etsf.eu/resources/software/libraries_and_tools)|-|1.0.4|<details><summary>Expand</summary>Description: A library of F90 routines to read/write the ETSF file format has been written. It is called ETSF_IO and available under LGPL. Homepage: http://www.etsf.eu/resources/software/libraries_and_tools URL: http://www.etsf.eu/resources/software/libraries_and_tools</details>|
	|[examl](https://github.com/stamatak/ExaML)|-|3.0.22|<details><summary>Expand</summary>Description: Exascale Maximum Likelihood (ExaML) code for phylogenetic inference using MPI. This code implements the popular RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. Homepage: https://github.com/stamatak/ExaML URL: https://github.com/stamatak/ExaML</details>|
	|[exonerate](https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate)|bio|2.4.0|<details><summary>Expand</summary>Description: Exonerate is a generic tool for pairwise sequence comparison. It allows you to align sequences using a many alignment models, using either exhaustive dynamic programming, or a variety of heuristics. Homepage: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate URL: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</details>|
	|[expat](https://libexpat.github.io)|tools|2.2.9, 2.2.10, 2.4.1|<details><summary>Expand</summary>Description: Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags) Homepage: https://libexpat.github.io URL: https://libexpat.github.io</details>|
	|[expect](https://core.tcl.tk/expect/index)|-|5.45.4|<details><summary>Expand</summary>Description: Expect is a tool for automating interactive applications such as telnet, ftp, passwd, fsck, rlogin, tip, etc. Expect really makes this stuff trivial. Expect is also useful for testing these same applications. Homepage: https://core.tcl.tk/expect/index URL: https://core.tcl.tk/expect/index</details>|
	|[faiss](https://github.com/facebookresearch/faiss)|-|1.6.5, 1.7.1, 1.7.3, 1.7.4, 1.8.0|<details><summary>Expand</summary>Description: Faiss is a library for efficient similarity search and clustering of dense vectors. It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM. It also contains supporting code for evaluation and parameter tuning. Faiss is written in C++ with complete wrappers for Python/numpy. Some of the most useful algorithms are implemented on the GPU. Homepage: https://github.com/facebookresearch/faiss URL: https://github.com/facebookresearch/faiss</details>|
	|[falcon](https://github.com/PacificBiosciences/FALCON)|bio|1.8.8|<details><summary>Expand</summary>Description: Falcon: a set of tools for fast aligning long reads for consensus and assembly Homepage: https://github.com/PacificBiosciences/FALCON URL: https://github.com/PacificBiosciences/FALCON</details>|
	|[fann](http://leenissen.dk)|-|2.2.0|<details><summary>Expand</summary>Description: Fast Artificial Neural Network Library is a free open source neural network library, which implements multilayer artificial neural networks in C with support for both fully connected and sparsely connected networks. Homepage: http://leenissen.dk URL: http://leenissen.dk</details>|
	|[fasta](http://fasta.bioch.virginia.edu)|bio|36.3.8h|<details><summary>Expand</summary>Description: The FASTA programs find regions of local or global (new) similarity between protein or DNA sequences, either by searching Protein or DNA databases, or by identifying local duplications within a sequence. Homepage: http://fasta.bioch.virginia.edu URL: http://fasta.bioch.virginia.edu</details>|
	|[fastahack](https://github.com/ekg/fastahack)|-|1.0.0|<details><summary>Expand</summary>Description: Utilities for indexing and sequence extraction from FASTA files. Homepage: https://github.com/ekg/fastahack URL: https://github.com/ekg/fastahack</details>|
	|[fastani](https://github.com/ParBLiSS/FastANI)|-|1.32|<details><summary>Expand</summary>Description: FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies. Homepage: https://github.com/ParBLiSS/FastANI URL: https://github.com/ParBLiSS/FastANI</details>|
	|[fastme](http://www.atgc-montpellier.fr/fastme/)|bio|2.1.6.2|<details><summary>Expand</summary>Description: FastME: a comprehensive, accurate and fast distance-based phylogeny inference program. Homepage: http://www.atgc-montpellier.fr/fastme/ URL: http://www.atgc-montpellier.fr/fastme/</details>|
	|[fastp](https://github.com/OpenGene/fastp)|bio|0.20.1, 0.23.1, 0.23.4, 0.24.0|<details><summary>Expand</summary>Description: A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance. Homepage: https://github.com/OpenGene/fastp URL: https://github.com/OpenGene/fastp</details>|
	|[fastq-join](https://github.com/brwnj/fastq-join)|-|1.3.1|<details><summary>Expand</summary>Description: fastq-join: Joins two paired-end reads on the overlapping ends. Homepage: https://github.com/brwnj/fastq-join URL: https://github.com/brwnj/fastq-join</details>|
	|[fastq-multx](https://github.com/brwnj/fastq-multx)|-|1.4.0|<details><summary>Expand</summary>Description: fastq-multx Homepage: https://github.com/brwnj/fastq-multx URL: https://github.com/brwnj/fastq-multx</details>|
	|[fastq-tools](https://homes.cs.washington.edu/~dcjones/fastq-tools/)|-|0.8|<details><summary>Expand</summary>Description: This package provides a number of small and efficient programs to perform common tasks with high throughput sequencing data in the FASTQ format. All of the programs work with typical FASTQ files as well as gzipped FASTQ files. Homepage: https://homes.cs.washington.edu/~dcjones/fastq-tools/ URL: https://homes.cs.washington.edu/~dcjones/fastq-tools/</details>|
	|[fastq_screen](http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/)|bio|0.11.4|<details><summary>Expand</summary>Description: FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect. Homepage: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/ URL: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/</details>|
	|[fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)|bio|0.11.9, 0.12.0, 0.12.1|<details><summary>Expand</summary>Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline. Homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</details>|
	|[fastsimcoal2](http://cmpg.unibe.ch/software/fastsimcoal2/)|bio|2.6.0.3, 2.7.0.9|<details><summary>Expand</summary>Description: While preserving all the simulation flexibility of simcoal2, fastsimcoal is now implemented under a faster continous-time sequential Markovian coalescent approximation, allowing it to efficiently generate genetic diversity for different types of markers along large genomic regions, for both present or ancient samples. It includes a parameter sampler allowing its integration into Bayesian or likelihood parameter estimation procedure. fastsimcoal can handle very complex evolutionary scenarios including an arbitrary migration matrix between samples, historical events allowing for population resize, population fusion and fission, admixture events, changes in migration matrix, or changes in population growth rates. The time of sampling can be specified independently for each sample, allowing for serial sampling in the same or in different populations. Homepage: http://cmpg.unibe.ch/software/fastsimcoal2/ URL: http://cmpg.unibe.ch/software/fastsimcoal2/</details>|
	|[fastspar](https://github.com/scwatts/fastspar)|-|1.0.0|<details><summary>Expand</summary>Description: FastSpar is a C++ implementation of the SparCC algorithm which is up to several thousand times faster than the original Python2 release and uses much less memory. The FastSpar implementation provides threading support and a p-value estimator which accounts for the possibility of repetitious data permutations (see this paper for further details). Homepage: https://github.com/scwatts/fastspar URL: https://github.com/scwatts/fastspar</details>|
	|[fasttree](http://www.microbesonline.org/fasttree/)|bio|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fasttree-double](http://www.microbesonline.org/fasttree/)|-|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fastx-toolkit](http://hannonlab.cshl.edu/fastx_toolkit/)|bio|0.0.14|<details><summary>Expand</summary>Description: The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing. Homepage: http://hannonlab.cshl.edu/fastx_toolkit/ URL: http://hannonlab.cshl.edu/fastx_toolkit/</details>|
	|[fcl](https://github.com/flexible-collision-library/fcl)|-|0.7.0|<details><summary>Expand</summary>Description: FCL is a library for performing three types of proximity queries on a pair of geometric models composed of triangles. Homepage: https://github.com/flexible-collision-library/fcl URL: https://github.com/flexible-collision-library/fcl</details>|
	|[fds](https://pages.nist.gov/fds-smv/)|-|6.7.5, 6.7.6, 6.7.7, 6.7.8, 6.7.9, 6.8.0|<details><summary>Expand</summary>Description: Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows, with an emphasis on smoke and heat transport from fires. Homepage: https://pages.nist.gov/fds-smv/ URL: https://pages.nist.gov/fds-smv/</details>|
	|[febio](https://github.com/febiosoftware/FEBio)|-|4.7|<details><summary>Expand</summary>Description: FEBio is a nonlinear finite element (FE) solver that is specifically designed for biomechanical applications. It offers modeling scenarios, constitutive models and boundary conditions that are relevant to many research areas in biomechanics. All features can be used together seamlessly, giving the user a powerful tool for solving 3D problems in computational biomechanics. The software is open-source, and pre-compiled executables for Windows, OS-X and Linux platforms are available. Homepage: https://github.com/febiosoftware/FEBio URL: https://github.com/febiosoftware/FEBio</details>|
	|[fermi-lite](https://github.com/lh3/fermi-lite)|-|20190320|<details><summary>Expand</summary>Description: Standalone C library for assembling Illumina short reads in small regions. Homepage: https://github.com/lh3/fermi-lite URL: https://github.com/lh3/fermi-lite</details>|
	|[ferret](http://ferret.pmel.noaa.gov/)|vis|7.3, 7.6.0|<details><summary>Expand</summary>Description: Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets. Ferret 7.6.0 is the final available version of Classic Ferret. The developers recommend using PyFerret instead. Homepage: http://ferret.pmel.noaa.gov/ URL: http://ferret.pmel.noaa.gov/</details>|
	|[ffmpeg](https://www.ffmpeg.org/)|-|4.2.2, 4.3.2|<details><summary>Expand</summary>Description: A complete, cross-platform solution to record, convert and stream audio and video. Homepage: https://www.ffmpeg.org/ URL: https://www.ffmpeg.org/</details>|
	|[fftw](https://www.fftw.org)|math|3.3.8, 3.3.10|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://www.fftw.org URL: https://www.fftw.org</details>|
	|[fftw-mpi](https://www.fftw.org)|math|3.3.8, 3.3.9, 3.3.10|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://www.fftw.org URL: https://www.fftw.org</details>|
	|[filevercmp](https://github.com/ekg/filevercmp)|-|20191210|<details><summary>Expand</summary>Description: filevercmp function as in sort --version-sort. Homepage: https://github.com/ekg/filevercmp URL: https://github.com/ekg/filevercmp</details>|
	|[filtlong](https://github.com/rrwick/Filtlong)|bio|0.2.0, 0.2.1|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/rrwick/Filtlong URL: https://github.com/rrwick/Filtlong</details>|
	|[flash](https://ccb.jhu.edu/software/FLASH/)|-|1.2.11|<details><summary>Expand</summary>Description: FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. Homepage: https://ccb.jhu.edu/software/FLASH/ URL: https://ccb.jhu.edu/software/FLASH/</details>|
	|[flashpca](https://github.com/gabraham/flashpca/)|-|2.0|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/gabraham/flashpca/ URL: https://github.com/gabraham/flashpca/</details>|
	|[flatbuffers](https://github.com/google/flatbuffers/)|-|22.9.24|<details><summary>Expand</summary>Description: FlatBuffers: Memory Efficient Serialization Library Homepage: https://github.com/google/flatbuffers/ URL: https://github.com/google/flatbuffers/</details>|
	|[fleur](http://www.flapw.de/)|-|MaX-R4|<details><summary>Expand</summary>Description: FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code, based on density-functional theory. Homepage: http://www.flapw.de/ URL: http://www.flapw.de/</details>|
	|[flexiblas](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)|-|3.0.4, 3.2.0, 3.3.1, 3.4.4|<details><summary>Expand</summary>Description: FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it. Homepage: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release URL: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release</details>|
	|[flint](http://www.flintlib.org/)|math|2.7.1, 2.9.0, 3.0.0|<details><summary>Expand</summary>Description: FLINT (Fast Library for Number Theory) is a C library in support of computations in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs, factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides various low-level routines for fast arithmetic. FLINT is extensively documented and tested. Homepage: http://www.flintlib.org/ URL: http://www.flintlib.org/</details>|
	|[fmlrc2](https://github.com/HudsonAlpha/fmlrc2)|-|0.1.8|<details><summary>Expand</summary>Description: FMLRC v2, based on the same methodology used by the original FMLRC. In benchmarks, the results between FMLRC v1 and v2 are nearly identical, but tests have shown that v2 uses approximately 50% of the run and CPU time compared to v1. Homepage: https://github.com/HudsonAlpha/fmlrc2 URL: https://github.com/HudsonAlpha/fmlrc2</details>|
	|[fmm3d](https://fmm3d.readthedocs.io)|-|1.0.1, 1.0.4|<details><summary>Expand</summary>Description: Flatiron Institute Fast Multipole Libraries: a set of libraries to compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine. Homepage: https://fmm3d.readthedocs.io URL: https://fmm3d.readthedocs.io</details>|
	|[fmriprep](https://fmriprep.org/)|-|23.0.2, 23.1.3|<details><summary>Expand</summary>Description: fMRIPrep is a NiPreps (NeuroImaging PREProcessing toolS) application (www.nipreps.org) for the preprocessing of task-based and resting-state functional MRI (fMRI). Homepage: https://fmriprep.org/ URL: https://fmriprep.org/</details>|
	|[fmt](http://fmtlib.net/)|-|5.3.0, 6.2.1, 7.0.3, 9.1.0|<details><summary>Expand</summary>Description: fmt (formerly cppformat) is an open-source formatting library. Homepage: http://fmtlib.net/ URL: http://fmtlib.net/</details>|
	|[fpc](https://www.freepascal.org)|tools|3.2.2|<details><summary>Expand</summary>Description: Free Pascal is a mature, versatile, open source Pascal compiler. Homepage: https://www.freepascal.org URL: https://www.freepascal.org</details>|
	|[fplll](https://github.com/fplll/fplll)|-|5.4.5|<details><summary>Expand</summary>Description: fplll contains implementations of several lattice algorithms. The implementation relies on floating-point orthogonalization, and the 1982 paper from Lenstra, Lenstra Jr. and Lovasz (LLL) is central to the code, hence the name. Homepage: https://github.com/fplll/fplll URL: https://github.com/fplll/fplll</details>|
	|[fpocket](https://github.com/Discngine/fpocket)|-|3.1.4.2|<details><summary>Expand</summary>Description: fpocket is a very fast open source protein pocket detection algorithm based on Voronoi tessellation. Homepage: https://github.com/Discngine/fpocket URL: https://github.com/Discngine/fpocket</details>|
	|[fraggenescan](http://omics.informatics.indiana.edu/FragGeneScan/)|bio|1.30, 1.31|<details><summary>Expand</summary>Description: FragGeneScan is an application for finding (fragmented) genes in short reads. Homepage: http://omics.informatics.indiana.edu/FragGeneScan/ URL: http://omics.informatics.indiana.edu/FragGeneScan/</details>|
	|[freebayes](https://github.com/freebayes/freebayes)|bio|1.2.0, 1.3.6, 1.3.7|<details><summary>Expand</summary>Description: FreeBayes is a Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs (single-nucleotide polymorphisms), indels (insertions and deletions), MNPs (multi-nucleotide polymorphisms), and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment. Homepage: https://github.com/freebayes/freebayes URL: https://github.com/freebayes/freebayes</details>|
	|[freesasa](https://freesasa.github.io)|-|2.1.0|<details><summary>Expand</summary>Description: FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA). Homepage: https://freesasa.github.io URL: https://freesasa.github.io</details>|
	|[freesurfer](https://surfer.nmr.mgh.harvard.edu/)|bio|5.3.0, 7.4.1|<details><summary>Expand</summary>Description: FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data. FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data. Homepage: https://surfer.nmr.mgh.harvard.edu/ URL: https://surfer.nmr.mgh.harvard.edu/</details>|
	|[freexl](https://www.gaia-gis.it/fossil/freexl/index)|tools|1.0.5, 2.0.0|<details><summary>Expand</summary>Description: FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. Homepage: https://www.gaia-gis.it/fossil/freexl/index URL: https://www.gaia-gis.it/fossil/freexl/index</details>|
	|[fsl](https://www.fmrib.ox.ac.uk/fsl/)|bio|6.0.3, 6.0.4, 6.0.7.7|<details><summary>Expand</summary>Description: FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data. Homepage: https://www.fmrib.ox.ac.uk/fsl/ URL: https://www.fmrib.ox.ac.uk/fsl/</details>|
	|[fsom](https://github.com/ekg/fsom)|-|20141119, 20151117|<details><summary>Expand</summary>Description: A tiny C library for managing SOM (Self-Organizing Maps) neural networks. Homepage: https://github.com/ekg/fsom URL: https://github.com/ekg/fsom</details>|
	|[g2clib](https://github.com/NOAA-EMC/NCEPLIBS-g2c)|geo|1.6.0, 1.8.0|<details><summary>Expand</summary>Description: This library contains C decoder/encoder routines for GRIB edition 2. This is part of the NCEPLIBS project. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-g2c URL: https://github.com/NOAA-EMC/NCEPLIBS-g2c</details>|
	|[g2lib](https://github.com/NOAA-EMC/NCEPLIBS-g2/)|geo|3.1.0, 3.4.8|<details><summary>Expand</summary>Description: Library contains GRIB2 encoder/decoder and search/indexing routines. Homepage: https://github.com/NOAA-EMC/NCEPLIBS-g2/ URL: https://github.com/NOAA-EMC/NCEPLIBS-g2/</details>|
	|[ga](http://hpc.pnl.gov/globalarrays/)|tools|5.7.2|<details><summary>Expand</summary>Description: Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model. It provides primitives for one-sided communication (Get, Put, Accumulate) and Atomic Operations (read increment). It supports blocking and non-blocking primtives, and supports location consistency. I8 version. Homepage: http://hpc.pnl.gov/globalarrays/ URL: http://hpc.pnl.gov/globalarrays/</details>|
	|[gamess-us](https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html)|chem|20210930-R2P1, 2020.2, 20220930-R2, 20230630-R1, 20230930-R2|<details><summary>Expand</summary>Description: The General Atomic and Molecular Electronic Structure System (GAMESS) is a general ab initio quantum chemistry package. This version of GAMESS-US is built with Intel compilers, mixed DDI, and OpenMPI 4.1. Homepage: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html URL: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html</details>|
	|[gapcloser](https://sourceforge.net/projects/soapdenovo2/files/GapCloser/)|-|1.12-r6|<details><summary>Expand</summary>Description: GapCloser is designed to close the gaps emerging during the scaffolding process by SOAPdenovo or other assembler, using the abundant pair relationships of short reads. Homepage: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/ URL: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/</details>|
	|[gate](http://www.opengatecollaboration.org/)|bio|9.3, 9.4|<details><summary>Expand</summary>Description: GATE is an advanced opensource software developed by the international OpenGATE collaboration and dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography Homepage: http://www.opengatecollaboration.org/ URL: http://www.opengatecollaboration.org/</details>|
	|[gatk](http://www.broadinstitute.org/gatk/)|bio|4.1.8.0, 4.1.8.1, 4.2.2.0, 4.2.4.0, 4.2.5.0, 4.4.0.0, 4.6.1.0|<details><summary>Expand</summary>Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size. Homepage: http://www.broadinstitute.org/gatk/ URL: http://www.broadinstitute.org/gatk/</details>|
	|[gazebo](https://gazebosim.org)|-|11.7.0|<details><summary>Expand</summary>Description: This is the Gazebo simulator. Gazebo simulates multiple robots in a 3D environment, with extensive dynamic interaction between objects. Homepage: https://gazebosim.org URL: https://gazebosim.org</details>|
	|[gblocks](http://molevol.cmima.csic.es/castresana/Gblocks.html)|-|0.91b|<details><summary>Expand</summary>Description: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis Homepage: http://molevol.cmima.csic.es/castresana/Gblocks.html URL: http://molevol.cmima.csic.es/castresana/Gblocks.html</details>|
	|[gcc](https://gcc.gnu.org/)|tools|8.4.0, 9.3.0, 10.2.0, 10.3.0, 11.3.0, 12.3, 13.3|<details><summary>Expand</summary>Description: The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). Homepage: https://gcc.gnu.org/ URL: https://gcc.gnu.org/</details>|
	|[gclust](http://gclust.c.u-tokyo.ac.jp/)|-|355z3|<details><summary>Expand</summary>Description: Gclust software was developed to make clusters of protein sequences from all predicted protein sequences in a selected set of genomes. The clusters are homolog groups, but not ortholog clusters (see below for the distinction), and therefore, contain all homologous sequences encoded by the selected genomes. An ortholog cluster, such as the one in KEGG Orthologs or COG in NCBI, contains only a single sequence for each genome, and such a single representative is usually selected by a criterion called bi-directional best hit Homepage: http://gclust.c.u-tokyo.ac.jp/ URL: http://gclust.c.u-tokyo.ac.jp/</details>|
	|[gcta](https://yanglab.westlake.edu.cn/software/gcta/)|bio|1.26.0, 1.93.2, 1.94.1|<details><summary>Expand</summary>Description: GCTA (Genome-wide Complex Trait Analysis) is a software package, which was initially developed to estimate the proportion of phenotypic variance explained by all genome-wide SNPs for a complex trait but has been extensively extended for many other analyses of data from genome-wide association studies (GWASs). Homepage: https://yanglab.westlake.edu.cn/software/gcta/ URL: https://yanglab.westlake.edu.cn/software/gcta/</details>|
	|[gd](https://github.com/lstein/Perl-GD)|-|2.71, 2.77|<details><summary>Expand</summary>Description: GD.pm - Interface to Gd Graphics Library Homepage: https://github.com/lstein/Perl-GD URL: https://github.com/lstein/Perl-GD</details>|
	|[gdal](https://www.gdal.org/)|geo|3.0.4, 3.2.3, 3.4.1, 3.4.3, 3.5.1, 3.7.2, 3.9.1|<details><summary>Expand</summary>Description: GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing. Homepage: https://www.gdal.org/ URL: https://www.gdal.org/</details>|
	|[gdcm](https://sourceforge.net/projects/gdcm)|-|2.6.8, 3.0.8|<details><summary>Expand</summary>Description: Grassroots DICOM: Cross-platform DICOM implementation Homepage: https://sourceforge.net/projects/gdcm URL: https://sourceforge.net/projects/gdcm</details>|
	|[gdrcopy](https://github.com/NVIDIA/gdrcopy)|-|2.1, 2.3, 2.3.1, 2.4.1|<details><summary>Expand</summary>Description: A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology. Homepage: https://github.com/NVIDIA/gdrcopy URL: https://github.com/NVIDIA/gdrcopy</details>|
	|[geant4](http://geant4.cern.ch/)|phys|10.06, 10.7.3, 11.1.0, 11.1.2, 11.2.1, 11.3.0|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[geant4-data](https://geant4.cern.ch/)|-|10.7.3, 11.1.0, 11.1.2, 11.2.1, 11.3.0|<details><summary>Expand</summary>Description: Datasets for Geant4. Homepage: https://geant4.cern.ch/ URL: https://geant4.cern.ch/</details>|
	|[geant4-seq](http://geant4.cern.ch/)|-|11.1.0|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[geant4-topasmc3.9](http://geant4.cern.ch/)|-|10.7.3|<details><summary>Expand</summary>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science. Homepage: http://geant4.cern.ch/ URL: http://geant4.cern.ch/</details>|
	|[gem](https://github.com/ECCC-ASTD-MRD/gem)|-|5.1.1|<details><summary>Expand</summary>Description: The Global Environmental Multiscale Model (GEM) is an integrated forecasting and data assimilation system developed by the Atmospheric Numerical Prediction Research Section, Meteorological Research Division, of Environment and Climate Change Canada. Homepage: https://github.com/ECCC-ASTD-MRD/gem URL: https://github.com/ECCC-ASTD-MRD/gem</details>|
	|[gemma](http://veda.cs.uiuc.edu/Seq2Expr/)|bio|0.98.3, 0.98.5|<details><summary>Expand</summary>Description: GEMMA is a software toolkit for fast application of linear mixed models (LMMs) and related models to genome-wide association studies (GWAS) and other large-scale data sets. Homepage: http://veda.cs.uiuc.edu/Seq2Expr/ URL: http://veda.cs.uiuc.edu/Seq2Expr/</details>|
	|[geneid](https://github.com/guigolab/geneid)|-|1.4.5|<details><summary>Expand</summary>Description: program to predict genic elements as splice sites, exons or genes, along eukaryotic DNA sequences. geneid offers also some rudimentary support to integrate predictions for multiple sources. The program is written in ANSI C, and runs on UNIX operating systems. Installation, setup and usage of geneid is very easy, and there is a range of options to configure output predictions and program behaviour. Homepage: https://github.com/guigolab/geneid URL: https://github.com/guigolab/geneid</details>|
	|[genewise](http://www.ebi.ac.uk/~birney/wise2)|-|2.4.1|<details><summary>Expand</summary>Description: Package focused on comparisons of biopolymers, commonly DNA sequence and protein sequence. It is a collection of algorithms which generally differ from the usualy, **standard** bioinformatics comparison methods. Probably the most used algorithm in Wise2 is **genewise**, which is the comparison of DNA sequence at the level of its protein translation. This comparison allows the simultaneous prediction of say gene structure with homology based alignment Homepage: http://www.ebi.ac.uk/~birney/wise2 URL: http://www.ebi.ac.uk/~birney/wise2</details>|
	|[gengetopt](https://www.gnu.org/software/gengetopt/gengetopt.html)|-|2.23|<details><summary>Expand</summary>Description: Gengetopt is a tool to write command line option parsing code for C programs. Homepage: https://www.gnu.org/software/gengetopt/gengetopt.html URL: https://www.gnu.org/software/gengetopt/gengetopt.html</details>|
	|[genmap](https://github.com/cpockrandt/genmap)|-|1.3.0|<details><summary>Expand</summary>Description: GenMap - Fast and Exact Computation of Genome Mappability GenMap computes the uniqueness of k-mers for each position in the genome while allowing for up to e mismatches. More formally, the uniqueness or (k,e)-mappability can be described for every position as the reciprocal value of how often each k-mer occurs approximately in the genome, i.e., with up to e mismatches. Hence, a mappability value of 1 at position i indicates that the k-mer in the sequence at position i occurs only once in the sequence with up to e errors. A low mappability value indicates that this k-mer belongs to a repetitive region. GenMap can be applied to single or multiple genomes and helps finding regions that are unique or shared by many or all genomes. Homepage: https://github.com/cpockrandt/genmap URL: https://github.com/cpockrandt/genmap</details>|
	|[genometools](http://genometools.org)|bio|1.6.1|<details><summary>Expand</summary>Description: A comprehensive software library for efficient processing of structured genome annotations. Homepage: http://genometools.org URL: http://genometools.org</details>|
	|[gentoo](https://wiki.gentoo.org/wiki/Project:Prefix)|-|2020, 2023|<details><summary>Expand</summary>Description: Geopsy team is developing, distributing and maintaining open source software for geophysical research and applications. Born during SESAME European Project, it has provided tools for processing ambient vibrations with site characterization in mind since 2005. Progressively, more conventional techniques (such as MASW or refraction) are included to offer a high quality, comprehensive and free platform for the interpretation of geophysical experiments. Homepage: https://www.geopsy.org/ URL: https://www.geopsy.org/</details>|
	|[geos](https://trac.osgeo.org/geos)|geo|3.7.3, 3.8.1, 3.9.1, 3.10.2, 3.12.0|<details><summary>Expand</summary>Description: GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS) Homepage: https://trac.osgeo.org/geos URL: https://trac.osgeo.org/geos</details>|
	|[gerris](http://gfs.sourceforge.net/wiki/index.php/Main_Page)|-|20131206|<details><summary>Expand</summary>Description: Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow Homepage: http://gfs.sourceforge.net/wiki/index.php/Main_Page URL: http://gfs.sourceforge.net/wiki/index.php/Main_Page</details>|
	|[gffcompare](https://ccb.jhu.edu/software/stringtie/gffcompare.shtml)|-|0.12.6|<details><summary>Expand</summary>Description: GffCompare provides classification and reference annotation mapping and matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files. Homepage: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml URL: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml</details>|
	|[gffread](https://github.com/gpertea/gffread)|-|0.11.7, 0.12.3|<details><summary>Expand</summary>Description: GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and more. Homepage: https://github.com/gpertea/gffread URL: https://github.com/gpertea/gffread</details>|
	|[gflags](https://github.com/gflags/gflags)|-|2.2.2|<details><summary>Expand</summary>Description: The gflags package contains a C++ library that implements commandline flags processing. It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. Homepage: https://github.com/gflags/gflags URL: https://github.com/gflags/gflags</details>|
	|[gibbs2](https://aoterodelaroza.github.io/gibbs2/)|-|1.0|<details><summary>Expand</summary>Description: Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary conditions of temperature and pressure. Gibbs2 uses the results of periodic solid-state quantum-mechanical calculations, specifically the energy-volume curve and possibly the harmonic phonon frequencies, to compute the thermodynamic properties of the solid within the framework of the quasiharmonic approximation. Homepage: https://aoterodelaroza.github.io/gibbs2/ URL: https://aoterodelaroza.github.io/gibbs2/</details>|
	|[ginkgo](https://ginkgo-project.github.io/)|-|1.6.0|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: https://ginkgo-project.github.io/ URL: https://ginkgo-project.github.io/</details>|
	|[git-annex](http://git-annex.branchable.com/)|tools|8.20200810, 10.20221003, 10.20231129|<details><summary>Expand</summary>Description: git-annex allows managing files with git, without checking the file contents into git. While that may seem paradoxical, it is useful when dealing with files larger than git can currently easily handle, whether due to limitations in memory, time, or disk space. Homepage: http://git-annex.branchable.com/ URL: http://git-annex.branchable.com/</details>|
	|[git-lfs](https://git-lfs.github.com)|-|2.11.0, 3.3.0, 3.4.0|<details><summary>Expand</summary>Description: Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com Homepage: https://git-lfs.github.com URL: https://git-lfs.github.com</details>|
	|[givaro](http://givaro.forge.imag.fr/)|-|4.2.0|<details><summary>Expand</summary>Description: C++ library for arithmetic and algebraic computations Homepage: http://givaro.forge.imag.fr/ URL: http://givaro.forge.imag.fr/</details>|
	|[glew](http://glew.sourceforge.net/)|-|2.1.0, 2.2.0|<details><summary>Expand</summary>Description: The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform. Homepage: http://glew.sourceforge.net/ URL: http://glew.sourceforge.net/</details>|
	|[glfw](https://www.glfw.org)|-|3.3.2, 3.3.8, 3.4|<details><summary>Expand</summary>Description: GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop Homepage: https://www.glfw.org URL: https://www.glfw.org</details>|
	|[glimmerhmm](https://ccb.jhu.edu/software/glimmerhmm)|-|3.0.4|<details><summary>Expand</summary>Description: GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model. Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models. Homepage: https://ccb.jhu.edu/software/glimmerhmm URL: https://ccb.jhu.edu/software/glimmerhmm</details>|
	|[glimpse](https://odelaneau.github.io/GLIMPSE/)|-|1.1.1, 2.0.0|<details><summary>Expand</summary>Description: GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies. Main features of the method: 1. Accurate imputed genotype calls. Our method takes advantage of reference panels to produce high quality genotype calls. 2. Accurate phasing. GLIMPSE outputs accurate phased haplotypes for the low-coverage sequenced dataset. 3. Low-coverage sequencing outperforms SNP arrays. Imputation using low-coverage sequencing data is competitive to SNP array imputation. Results for European and African-American populations are interactively available on the website. 4. A cost-effective paradigm. GLIMPSE realises whole genome imputation from the HRC reference panel for less than 1$. 5. An easy-to-use framework for diploid, haploid and mixed ploidy imputation. Our method is able to perform imputation of samples with different ploidy without the need to split the data. Homepage: https://odelaneau.github.io/GLIMPSE/ URL: https://odelaneau.github.io/GLIMPSE/</details>|
	|[glm](https://github.com/g-truc/glm)|vis|0.9.9.8|<details><summary>Expand</summary>Description: OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on the OpenGL Shading Language (GLSL) specifications. Homepage: https://github.com/g-truc/glm URL: https://github.com/g-truc/glm</details>|
	|[globalarrays](https://hpc.pnl.gov/globalarrays)|-|5.7.2, 5.8|<details><summary>Expand</summary>Description: Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model Homepage: https://hpc.pnl.gov/globalarrays URL: https://hpc.pnl.gov/globalarrays</details>|
	|[glost](https://github.com/cea-hpc/glost)|tools|0.3.1|<details><summary>Expand</summary>Description: This is GLOST, the Greedy Launcher Of Small Tasks. Homepage: https://github.com/cea-hpc/glost URL: https://github.com/cea-hpc/glost</details>|
	|[glpk](https://www.gnu.org/software/glpk/)|math|4.65, 5.0|<details><summary>Expand</summary>Description: The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems. It is a set of routines written in ANSI C and organized in the form of a callable library. Homepage: https://www.gnu.org/software/glpk/ URL: https://www.gnu.org/software/glpk/</details>|
	|[gmap-gsnap](http://research-pub.gene.com/gmap/)|bio|2024-10-20, 2020-11-14, 2019-09-12, 2024-08-20|<details><summary>Expand</summary>Description: GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences GSNAP: Genomic Short-read Nucleotide Alignment Program Homepage: http://research-pub.gene.com/gmap/ URL: http://research-pub.gene.com/gmap/</details>|
	|[gmp](https://gmplib.org/)|-|6.2.0|<details><summary>Expand</summary>Description: GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers. Homepage: https://gmplib.org/ URL: https://gmplib.org/</details>|
	|[gmsh](https://gmsh.info/)|phys|4.7.0, 4.10.5, 4.11.1, 4.12.2, 4.13.1|<details><summary>Expand</summary>Description: Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor. Homepage: https://gmsh.info/ URL: https://gmsh.info/</details>|
	|[gmt](https://www.generic-mapping-tools.org/)|geo|6.4.0, 6.5.0|<details><summary>Expand</summary>Description: GMT is an open source collection of about 80 command-line tools for manipulating geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting, etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another 40 more specialized and discipline-specific tools. Homepage: https://www.generic-mapping-tools.org/ URL: https://www.generic-mapping-tools.org/</details>|
	|[gmtk](http://melodi.ee.washington.edu/)|-|1.4.4|<details><summary>Expand</summary>Description: GMTK: The Graphical Models Toolkit Homepage: http://melodi.ee.washington.edu/ URL: http://melodi.ee.washington.edu/</details>|
	|[gnina](https://github.com/gnina/gnina)|-|1.0.1|<details><summary>Expand</summary>Description: gnina (pronounced NEE-na) is a molecular docking program with integrated support for scoring and optimizing ligands using convolutional neural networks. It is a fork of smina, which is a fork of AutoDock Vina. Homepage: https://github.com/gnina/gnina URL: https://github.com/gnina/gnina</details>|
	|[gnuplot](http://gnuplot.sourceforge.net)|vis|5.2.8, 5.4.2, 5.4.6, 5.4.8|<details><summary>Expand</summary>Description: Portable interactive, function plotting utility Homepage: http://gnuplot.sourceforge.net URL: http://gnuplot.sourceforge.net</details>|
	|[go](https://www.golang.org)|tools|1.14.1, 1.18.3, 1.21.3|<details><summary>Expand</summary>Description: Go is an open source programming language that makes it easy to build simple, reliable, and efficient software. Homepage: https://www.golang.org URL: https://www.golang.org</details>|
	|[gomc](https://gomc-wsu.org/)|-|2.75a|<details><summary>Expand</summary>Description: GOMC - GPU Optimized Monte Carlo is a parallel molecular simulation code designed for high-performance simulation of large systems. Homepage: https://gomc-wsu.org/ URL: https://gomc-wsu.org/</details>|
	|[googlebenchmark](https://github.com/google/benchmark)|-|1.7.1, 1.8.3|<details><summary>Expand</summary>Description: Google's library to benchmark code snippets, similar to unit tests. Homepage: https://github.com/google/benchmark URL: https://github.com/google/benchmark</details>|
	|[googletest](https://github.com/google/googletest)|tools|1.10.0, 1.13.0, 1.14.0|<details><summary>Expand</summary>Description: Google's framework for writing C++ tests on a variety of platforms Homepage: https://github.com/google/googletest URL: https://github.com/google/googletest</details>|
	|[grace](http://freecode.com/projects/grace)|vis|5.99.0|<details><summary>Expand</summary>Description: Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif. Homepage: http://freecode.com/projects/grace URL: http://freecode.com/projects/grace</details>|
	|[graph-tool](https://graph-tool.skewed.de/)|-|2.37, 2.45, 2.56|<details><summary>Expand</summary>Description: Graph-tool is an efficient Python module for manipulation and statistical analysis of graphs (a.k.a. networks). Contrary to most other python modules with similar functionality, the core data structures and algorithms are implemented in C++, making extensive use of template metaprogramming, based heavily on the Boost Graph Library. This confers it a level of performance that is comparable (both in memory usage and computation time) to that of a pure C/C++ library. Homepage: https://graph-tool.skewed.de/ URL: https://graph-tool.skewed.de/</details>|
	|[grass](https://grass.osgeo.org)|geo|7.8.4, 8.2.1|<details><summary>Expand</summary>Description: The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization Homepage: https://grass.osgeo.org URL: https://grass.osgeo.org</details>|
	|[groff](https://www.gnu.org/software/groff)|-|1.22.4|<details><summary>Expand</summary>Description: Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands and produces formatted output. Homepage: https://www.gnu.org/software/groff URL: https://www.gnu.org/software/groff</details>|
	|[gromacs](http://www.gromacs.org)|chem|2016.6, 2020.4, 2020.6, 2021.2, 2021.4, 2021.6, 2022.2, 2022.3, 2023.2, 2023.3, 2023.5, 2024.1, 2024.4|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a GPU enabled build, containing both MPI and threadMPI builds. Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-colvars](http://colvars.github.io/)|chem|2020.6|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version has been patched with 'Colvars', a collective variables module for molecular simulation programs. This is a GPU enabled build, containing both MPI and threadMPI builds. - CC-Wiki: GROMACS Homepage: http://colvars.github.io/ URL: http://colvars.github.io/</details>|
	|[gromacs-cp2k](https://www.cp2k.org/tools:gromacs)|chem|2022.2|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This version has been compiled with CP2K for QM/MM calculations. Note: only 'gmx_mpi' and 'gmx_mpi_d' executables are available. See also: https://www.cp2k.org/tools:gromacs - CC-Wiki: GROMACS Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-ls](https://vanegaslab.org/software)|-|2016.3|<details><summary>Expand</summary>Description: GROMACS-LS is a modified version of GROMACS that has been modified to enable the calculation of local stress fields from molecular dynamics simulations. This is a CPU only build. Note that GROMACS-LS does not support MPI, GPUs(CUDA) or Hardware accelleration (SIMD) and is therefore much slower than unmodified versions of GROMACS. CC-Wiki: GROMACS Homepage: https://vanegaslab.org/software URL: https://vanegaslab.org/software</details>|
	|[gromacs-plumed](http://www.gromacs.org)|chem|2019.6, 2020.7, 2021.2, 2021.4, 2021.6, 2021.7, 2022.3, 2022.6, 2023.5|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version contains the PLUMED patches in order to run metadynamics. - CC-Wiki: GROMACS Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gromacs-ramd](https://kbbox.h-its.org/toolbox/methods/molecular-simulation/random-acceleration-molecular-dynamics-ramd/)|chem|2020.5-RAMD-2.0, 2024.1-RAMD-2.1|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics. This version has been modified to support RAMD, which is used to identify ligand exit routes from the buried binding pockets of receptors and investigate the mechanism of ligand dissociation. https://kbbox.h-its.org/toolbox/methods/molecular-simulation/random-acceleration-molecular-dynamics-ramd/ This is a GPU enabled build, containing both MPI and threadMPI binaries. Homepage: https://github.com/HITS-MCM/gromacs-ramd URL: https://github.com/HITS-MCM/gromacs-ramd</details>|
	|[gromacs-swaxs](https://cbjh.gitlab.io/gromacs-swaxs-docs/)|chem|2021.7-0.5.1|<details><summary>Expand</summary>Description: Modified GROMACS for small-angle scattering calculations (SAXS/WAXS/SANS). This is a CPU only build, containing both MPI and threadMPI builds. - CC-Wiki: GROMACS Homepage: https://cbjh.gitlab.io/gromacs-swaxs-docs/ URL: https://cbjh.gitlab.io/gromacs-swaxs-docs/</details>|
	|[gsl](https://www.gnu.org/software/gsl/)|math|1.16, 2.6, 2.7|<details><summary>Expand</summary>Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting. Homepage: https://www.gnu.org/software/gsl/ URL: https://www.gnu.org/software/gsl/</details>|
	|[gsl-lite](https://github.com/gsl-lite/gsl-lite)|-|0.40.0, 0.41.0|<details><summary>Expand</summary>Description: A single-file header-only version of ISO C++ Guidelines Support Library (GSL) for C++98, C++11, and later Homepage: https://github.com/gsl-lite/gsl-lite URL: https://github.com/gsl-lite/gsl-lite</details>|
	|[gts](http://gts.sourceforge.net/)|-|20121130|<details><summary>Expand</summary>Description: GTS stands for the GNU Triangulated Surface Library. It is an Open Source Free Software Library intended to provide a set of useful functions to deal with 3D surfaces meshed with interconnected triangles. Homepage: http://gts.sourceforge.net/ URL: http://gts.sourceforge.net/</details>|
	|[gudhi](https://gudhi.inria.fr/)|-|3.4.1, 3.7.1|<details><summary>Expand</summary>Description: The GUDHI library is a generic open source C++ library, with a Python interface, for Topological Data Analysis (TDA) and Higher Dimensional Geometry Understanding. The library offers state-of-the-art data structures and algorithms to construct simplicial complexes and compute persistent homology. The library comes with data sets, demos, examples and test suites. Homepage: https://gudhi.inria.fr/ URL: https://gudhi.inria.fr/</details>|
	|[guile](https://www.gnu.org/software/guile/)|tools|2.2.2|<details><summary>Expand</summary>Description: Guile is a programming language, designed to help programmers create flexible applications that can be extended by users or other programmers with plug-ins, modules, or scripts. Homepage: https://www.gnu.org/software/guile/ URL: https://www.gnu.org/software/guile/</details>|
	|[gurobi](https://www.gurobi.com)|math|9.0.3, 9.1.0, 9.1.2, 9.5.0, 9.5.2, 10.0.1, 10.0.2, 10.0.3, 11.0.0, 11.0.1, 11.0.3, 12.0.0|<details><summary>Expand</summary>Description: The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms. Homepage: https://www.gurobi.com URL: https://www.gurobi.com</details>|
	|[h4toh5](http://www.hdfgroup.org/h4toh5/)|io|2.2.5|<details><summary>Expand</summary>Description: The h4toh5 software consists of the h4toh5 and h5toh4 command-line utilities, as well as a conversion library for converting between individual HDF4 and HDF5 objects. Homepage: http://www.hdfgroup.org/h4toh5/ URL: http://www.hdfgroup.org/h4toh5/</details>|
	|[hal](https://github.com/ComparativeGenomicsToolkit/hal)|bio|2.2|<details><summary>Expand</summary>Description: HAL is a structure to efficiently store and index multiple genome alignments and ancestral reconstructions. HAL is a graph-based representation which provides several advantages over matrix/block-based formats such as MAF, such as improved scalability and the ability to perform queries with respect to an arbitrary reference or subtree. This package includes the HAL API and several analysis and conversion tools which are described below. HAL files are presently stored in either HDF5 or mmap format, but we note that the tools and most of the API are format-independent, so other databases could be implemented in the future. Homepage: https://github.com/ComparativeGenomicsToolkit/hal URL: https://github.com/ComparativeGenomicsToolkit/hal</details>|
	|[hapgen2](https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html)|-|2.2.0|<details><summary>Expand</summary>Description: 'HAPGEN2' simulates case control datasets at SNP markers. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html</details>|
	|[haploview](https://www.broadinstitute.org/haploview/haploview)|-|4.2|<details><summary>Expand</summary>Description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses. Homepage: https://www.broadinstitute.org/haploview/haploview URL: https://www.broadinstitute.org/haploview/haploview</details>|
	|[harminv](https://github.com/NanoComp/harminv)|math|1.4.1, 1.4.2|<details><summary>Expand</summary>Description: Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion - given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those sinusoids. Homepage: https://github.com/NanoComp/harminv URL: https://github.com/NanoComp/harminv</details>|
	|[hdf](https://www.hdfgroup.org/products/hdf4/)|io|4.2.15, 4.2.16|<details><summary>Expand</summary>Description: HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. Homepage: https://www.hdfgroup.org/products/hdf4/ URL: https://www.hdfgroup.org/products/hdf4/</details>|
	|[hdf-eos5](https://hdfeos.org)|-|5.1.16|<details><summary>Expand</summary>Description: HDF is the prescribed format for standard data products that are derived from EOS missions. HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura. Two versions of HDF-EOS libraries: HDF-EOS2 (v2.20) based on HDF4 and HDF-EOS5 (v5.1.16) based on HDF5 are developed. Homepage: https://hdfeos.org URL: https://hdfeos.org</details>|
	|[hdf-fortran](https://www.hdfgroup.org/products/hdf4/)|-|4.2.15|<details><summary>Expand</summary>Description: HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. Homepage: https://www.hdfgroup.org/products/hdf4/ URL: https://www.hdfgroup.org/products/hdf4/</details>|
	|[hdf5](https://portal.hdfgroup.org/display/support)|io|1.8.22, 1.10.6, 1.10.7, 1.12.1, 1.12.2, 1.14.2, 1.14.5|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdf5-mpi](https://portal.hdfgroup.org/display/support)|io|1.8.22, 1.10.6, 1.10.7, 1.12.1, 1.12.2, 1.14.2, 1.14.4, 1.14.5|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdfview](https://support.hdfgroup.org/products/java/hdfview/)|-|2.14|<details><summary>Expand</summary>Description: HDFView is a visual tool for browsing and editing HDF4 and HDF5 files. Homepage: https://support.hdfgroup.org/products/java/hdfview/ URL: https://support.hdfgroup.org/products/java/hdfview/</details>|
	|[healpix](http://healpix.sourceforge.net/)|-|3.81, 3.82, 3.83|<details><summary>Expand</summary>Description: Hierarchical Equal Area isoLatitude Pixelation of a sphere. Homepage: http://healpix.sourceforge.net/ URL: http://healpix.sourceforge.net/</details>|
	|[heasoft](https://heasarc.nasa.gov/lheasoft/)|phys|6.32.1|<details><summary>Expand</summary>Description: A Unified Release of the FTOOLS and XANADU Software Packages XANADU High-level, multi-mission tasks for X-ray astronomical spectral, timing, and imaging data analysis FTOOLS General and mission-specific tools to manipulate FITS files FITSIO Core library responsible for reading and writing FITS files (distributed with FTOOLS) fv General FITS file browser/editor/plotter with a graphical user interface (distributed with FTOOLS) XSTAR Tool for calculating the physical conditions and emission spectra of photoionized gases Homepage: https://heasarc.nasa.gov/lheasoft/ URL: https://heasarc.nasa.gov/lheasoft/</details>|
	|[hh-suite](https://github.com/soedinglab/hh-suite)|bio|3.3.0|<details><summary>Expand</summary>Description: HH-suite is an open-source software package for sensitive protein sequence searching. It contains programs that can search for similar protein sequences in protein sequence databases. Homepage: https://github.com/soedinglab/hh-suite URL: https://github.com/soedinglab/hh-suite</details>|
	|[hifiasm](https://github.com/chhylp123/hifiasm)|-|0.16.1, 0.19.5|<details><summary>Expand</summary>Description: Hifiasm: a haplotype-resolved assembler for accurate Hifi reads. Homepage: https://github.com/chhylp123/hifiasm URL: https://github.com/chhylp123/hifiasm</details>|
	|[highfive](https://github.com/BlueBrain/HighFive)|-|2.9.0|<details><summary>Expand</summary>Description: HighFive is a modern header-only C++11 friendly interface for libhdf5. Homepage: https://github.com/BlueBrain/HighFive URL: https://github.com/BlueBrain/HighFive</details>|
	|[hisat2](https://daehwankimlab.github.io/hisat2/)|bio|2.2.1|<details><summary>Expand</summary>Description: HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome). Homepage: https://daehwankimlab.github.io/hisat2/ URL: https://daehwankimlab.github.io/hisat2/</details>|
	|[hmmer](http://hmmer.org/)|bio|3.2.1, 3.3.2, 3.4|<details><summary>Expand</summary>Description: HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs). Compared to BLAST, FASTA, and other sequence alignment and database search tools based on older scoring methodology, HMMER aims to be significantly more accurate and more able to detect remote homologs because of the strength of its underlying mathematical models. In the past, this strength came at significant computational expense, but in the new HMMER3 project, HMMER is now essentially as fast as BLAST. Homepage: http://hmmer.org/ URL: http://hmmer.org/</details>|
	|[hoomd-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|2.9.3, 3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[hoomd-blue-mpi](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[hpcspades](http://cab.spbu.ru/software/spades/)|-|3.15.0|<details><summary>Expand</summary>Description: Genome assembler for single-cell and isolates data sets Homepage: http://cab.spbu.ru/software/spades/ URL: http://cab.spbu.ru/software/spades/</details>|
	|[hpctoolkit](http://hpctoolkit.org/)|-|2020.08|<details><summary>Expand</summary>Description: HPCToolkit is an integrated suite of tools for measurement and analysis of program performance on computers ranging from multicore desktop systems to supercomputers. Homepage: http://hpctoolkit.org/ URL: http://hpctoolkit.org/</details>|
	|[hpl](https://www.netlib.org/benchmark/hpl/)|tools|2.3|<details><summary>Expand</summary>Description: HPL is a software package that solves a (random) dense linear system in double precision (64 bits) arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available implementation of the High Performance Computing Linpack Benchmark. Homepage: https://www.netlib.org/benchmark/hpl/ URL: https://www.netlib.org/benchmark/hpl/</details>|
	|[htslib](https://www.htslib.org/)|bio|1.9, 1.10.2, 1.11, 1.13, 1.14, 1.15.1, 1.16, 1.17, 1.18, 1.19|<details><summary>Expand</summary>Description: A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[hwloc](https://www.open-mpi.org/projects/hwloc/)|-|2.4.0, 2.7.1, 2.9.1, 2.10.0|<details><summary>Expand</summary>Description: The Portable Hardware Locality (hwloc) software package provides a portable abstraction (across OS, versions, architectures, ...) of the hierarchical topology of modern architectures, including NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various system attributes such as cache and memory information as well as the locality of I/O devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering information about modern computing hardware so as to exploit it accordingly and efficiently. Homepage: https://www.open-mpi.org/projects/hwloc/ URL: https://www.open-mpi.org/projects/hwloc/</details>|
	|[hyphy](https://veg.github.io/hyphy-site/)|bio|2.5.26, 2.5.49|<details><summary>Expand</summary>Description: HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package for the analysis of genetic sequences (in particular the inference of natural selection) using techniques in phylogenetics, molecular evolution, and machine learning Homepage: https://veg.github.io/hyphy-site/ URL: https://veg.github.io/hyphy-site/</details>|
	|[hypre](https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods)|math|2.20.0|<details><summary>Expand</summary>Description: Hypre is a library for solving large, sparse linear systems of equations on massively parallel computers. The problems of interest arise in the simulation codes being developed at LLNL and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences. Homepage: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods URL: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods</details>|
	|[idba-ud](http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/)|-|1.1.3|<details><summary>Expand</summary>Description: IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm. IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth contigs are removed iteratively with cutoff threshold from low to high to reduce the errors in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD can iterate k value of de Bruijn graph to a very large value with less gaps and less branches to form long contigs in both low-depth and high-depth regions. Homepage: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/ URL: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/</details>|
	|[igblast](https://www.ncbi.nlm.nih.gov/igblast/intro.html)|bio|1.17.0, 1.18.0|<details><summary>Expand</summary>Description: IgBLAST was developed at NCBI to facilitate analysis of immunoglobulin variable domain sequences (IgBLAST has recently been extended to perform analysis for T cell receptor (TR) sequences). It uses BLAST search algorithm. Homepage: https://www.ncbi.nlm.nih.gov/igblast/intro.html URL: https://www.ncbi.nlm.nih.gov/igblast/intro.html</details>|
	|[ignition](https://ignitionrobotics.org/)|-|citadel|<details><summary>Expand</summary>Description: Tools and libraries for robotics applications. Homepage: https://ignitionrobotics.org/ URL: https://ignitionrobotics.org/</details>|
	|[igraph](https://igraph.org)|math|0.8.2, 0.9.10, 0.10.2, 0.10.7, 0.10.13|<details><summary>Expand</summary>Description: igraph is a collection of network analysis tools with the emphasis on efficiency, portability and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++. Homepage: https://igraph.org URL: https://igraph.org</details>|
	|[igv](https://www.broadinstitute.org/software/igv/)|-|2.9.2|<details><summary>Expand</summary>Description: This package contains command line utilities for preprocessing, computing feature count density (coverage), sorting, and indexing data files. Homepage: https://www.broadinstitute.org/software/igv/ URL: https://www.broadinstitute.org/software/igv/</details>|
	|[ijulia-kernel](https://julialang.org/)|-|1.5, 1.8, 1.10|<details><summary>Expand</summary>Description: IJulia kernelspec for Julia 1.10 Homepage: https://julialang.org/ URL: https://julialang.org/</details>|
	|[ima3](https://github.com/jodyhey/IMa3)|-|1.12, 20210120|<details><summary>Expand</summary>Description: IMa3 is the newest in the IM sequence of programs. It can be used to solve a fundamental problem in evolutionary genetics, which is to jointly consider phylogenetic history and pouplation genetic history, including gene exchange. IMa3 can be used to estimate the rooted phylogenetic tree for multiple populations, and does so while integrating over all possible Isolation-with-Migration models. For a given phylogenetic tree IMa3 addresses the same model as IMa2. Like IMa2-p, IMa3 can run on multiple processors. Homepage: https://github.com/jodyhey/IMa3 URL: https://github.com/jodyhey/IMa3</details>|
	|[imb](https://software.intel.com/en-us/articles/intel-mpi-benchmarks)|-|2021.3|<details><summary>Expand</summary>Description: The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and global communication operations for a range of message sizes Homepage: https://software.intel.com/en-us/articles/intel-mpi-benchmarks URL: https://software.intel.com/en-us/articles/intel-mpi-benchmarks</details>|
	|[imkl](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)|math|2020.1.217, 2021.2.0, 2021.4.0, 2022.1.0, 2023.2.0, 2024.1.0, 2024.2.0|<details><summary>Expand</summary>Description: Intel oneAPI Math Kernel Library Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</details>|
	|[impute2](http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)|bio|2.3.2|<details><summary>Expand</summary>Description: IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation and haplotype phasing program based on ideas from Howie et al. 2009 Homepage: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html URL: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</details>|
	|[impute5](https://jmarchini.org/software/#impute-5)|-|1.1.5|<details><summary>Expand</summary>Description: IMPUTE 5 is a genotype imputation method that can scale to reference panels with millions of samples. This method continues to refine the observation made in the IMPUTE2 method, that accuracy is optimized via use of a custom subset of haplotypes when imputing each individual. It achieves fast, accurate, and memory-efficient imputation by selecting haplotypes using the Positional Burrows Wheeler Transform (PBWT). By using the PBWT data structure at genotyped markers, IMPUTE 5 identifies locally best matching haplotypes and long identical by state segments. The method then uses the selected haplotypes as conditioning states within the IMPUTE model. IMPUTE5 is up to 30x faster than MINIMAC4 and up to 3x faster than BEAGLE5.1 Homepage: https://jmarchini.org/software/#impute-5 URL: https://jmarchini.org/software/#impute-5</details>|
	|[infernal](http://eddylab.org/infernal/)|bio|1.1.3, 1.1.4, 1.1.5|<details><summary>Expand</summary>Description: Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. Homepage: http://eddylab.org/infernal/ URL: http://eddylab.org/infernal/</details>|
	|[intel](https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html)|tools|2020.1.217, 2021.2.0, 2022.1.0, 2023.2.1, 2024.2.0|<details><summary>Expand</summary>Description: Intel C, C++ and Fortran compilers (classic and oneAPI) Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</details>|
	|[intel-opencl](https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html)|-|2021.2.0|<details><summary>Expand</summary>Description: Intel's OpenCL CPU runtime Homepage: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html URL: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html</details>|
	|[intelmpi](https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html)|mpi|2019.7.217, 2021.2.0, 2021.6.0, 2021.9.0|<details><summary>Expand</summary>Description: Intel MPI Library, compatible with MPICH ABI Homepage: https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html URL: https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html</details>|
	|[intelxed](https://intelxed.github.io/)|-|12.0.1|<details><summary>Expand</summary>Description: The X86 Encoder Decoder (XED), is a software library (and associated headers) for encoding and decoding X86 (IA32 and Intel64) instructions. Homepage: https://intelxed.github.io/ URL: https://intelxed.github.io/</details>|
	|[interproscan](http://www.ebi.ac.uk/interpro/)|bio|5.73-104.0, 5.55-88.0, 5.50-84.0, 5.52-86.0, 5.53-87.0, 5.56-89.0, 5.64-96.0, 5.63-95.0|<details><summary>Expand</summary>Description: InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[interproscan_data](http://www.ebi.ac.uk/interpro/)|-|5.63-95.0, 5.64-96.0, 5.73-104.0|<details><summary>Expand</summary>Description: This module containes only InterProScan data part. InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[intervaltree](https://github.com/ekg/intervaltree)|-|0.1|<details><summary>Expand</summary>Description: An interval tree can be used to efficiently find a set of numeric intervals overlapping or containing another interval. This library provides a basic implementation of an interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. Homepage: https://github.com/ekg/intervaltree URL: https://github.com/ekg/intervaltree</details>|
	|[ioapi](https://www.cmascenter.org/ioapi/)|-|3.2-2020111, 3.2-20200828|<details><summary>Expand</summary>Description: The Models-3/EDSS Input/Output Applications Programming Interface (I/O API) provides the environmental model developer with an easy-to-learn, easy-to-use programming library for data storage and access, available from both Fortran and C. The same routines can be used for both file storage (using netCDF files) and model coupling (using PVM mailboxes). It is the standard data access library for both the NCSC/CMAS's EDSS project and EPA's Models-3, CMAQ, and SMOKE, as well as various other atmospheric and hydrological modeling systems. Homepage: https://www.cmascenter.org/ioapi/ URL: https://www.cmascenter.org/ioapi/</details>|
	|[ipopt](https://projects.coin-or.org/Ipopt)|-|3.14.11, 3.14.14|<details><summary>Expand</summary>Description: IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt) is an open source software package for large-scale nonlinear optimization. Homepage: https://projects.coin-or.org/Ipopt URL: https://projects.coin-or.org/Ipopt</details>|
	|[ipp](https://software.intel.com/en-us/articles/intel-ipp/)|tools|2020.1.217|<details><summary>Expand</summary>Description: Intel Integrated Performance Primitives (Intel IPP) is an extensive library of multicore-ready, highly optimized software functions for multimedia, data processing, and communications applications. Intel IPP offers thousands of optimized functions covering frequently used fundamental algorithms. Homepage: https://software.intel.com/en-us/articles/intel-ipp/ URL: https://software.intel.com/en-us/articles/intel-ipp/</details>|
	|[ipykernel](https://ipython.org/)|-|2020a, 2020b, 2021a, 2022a, 2023a, 2023b, 2024a, 2024b, 2025a|<details><summary>Expand</summary>Description: Bundle which contains ipykernel 2025a and its dependencies Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[ipython-kernel](https://ipython.org/)|-|2.7, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12, 3.13|<details><summary>Expand</summary>Description: IPython kernelspec for Python 3.13 Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[iq-tree](http://www.iqtree.org/)|bio|1.6.12, 2.0.7, 2.1.2, 2.2.1, 2.2.2.7, 2.3.6|<details><summary>Expand</summary>Description: Efficient phylogenomic software by maximum likelihood Homepage: http://www.iqtree.org/ URL: http://www.iqtree.org/</details>|
	|[ir-kernel](https://irkernel.github.io)|-|4.2, 4.4|<details><summary>Expand</summary>Description: The R kernel for the 'Jupyter' environment executes R code which the front-end (Jupyter Notebook or other front-ends) submits to the kernel via the network. Homepage: https://irkernel.github.io URL: https://irkernel.github.io</details>|
	|[irfinder](https://github.com/williamritchie/IRFinder/)|-|1.3.1|<details><summary>Expand</summary>Description: Basic STAR (Spliced Transcripts Alignment to a Reference) STAR workflow consists of 2 steps: Generating genoms indexes files and mapping reads to the genomes. Homepage: https://github.com/williamritchie/IRFinder/ URL: https://github.com/williamritchie/IRFinder/</details>|
	|[isce2](https://github.com/isce-framework/isce2)|-|2.6.3|<details><summary>Expand</summary>Description: Interferometric synthetic aperture radar Scientific Computing Environment (ISCE) is a framework designed for the purpose of processing Interferometric Synthetic Aperture Radar (InSAR) data. The framework aspects of it have been designed as a general software development framework. It may have additional utility in a general sense for building other types of software packages. Homepage: https://github.com/isce-framework/isce2 URL: https://github.com/isce-framework/isce2</details>|
	|[ispc](https://ispc.github.io/)|-|1.10.0, 1.13.0, 1.18.0, 1.21.1|Description: Intel SPMD Program Compilers; An open-source compiler for high-performance SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language, with extensions for 'single program, multiple data' (SPMD) programming. Under the SPMD model, the programmer writes a program that generally appears to be a regular serial program, though the execution model is actually that a number of program instances execute in parallel on the hardware. Homepage: https://ispc.github.io/ , https://github.com/ispc/ispc/ URL: https://ispc.github.io/ , https://github.com/ispc/ispc/<br /><br /><br /></div> \|- \| align="center" \| [https://software.intel.com/en-us/intel-trace-analyzer/ itac] \| align="center" \| tools \| align="center" \| 2021.5.0 \|<details><summary>Expand</summary> <div class="mw-collapsible mw-collapsed" style="white-space: pre-line;"><br />Description: The Intel Trace Collector is a low-overhead tracing library that performs event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application activities gathered by the Intel Trace Collector through graphical displays. Homepage: https://software.intel.com/en-us/intel-trace-analyzer/ URL: https://software.intel.com/en-us/intel-trace-analyzer/</details>|
	|[itk](https://itk.org)|vis|4.13.3, 5.0.1, 5.1.2, 5.2.1, 5.3.0, 5.4.0|<details><summary>Expand</summary>Description: Insight Segmentation and Registration Toolkit (ITK) provides an extensive suite of software tools for registering and segmenting multidimensional imaging data. Homepage: https://itk.org URL: https://itk.org</details>|
	|[ivar](https://github.com/andersen-lab/ivar)|-|1.4.2|<details><summary>Expand</summary>Description: iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. Homepage: https://github.com/andersen-lab/ivar URL: https://github.com/andersen-lab/ivar</details>|
	|[jags](http://mcmc-jags.sourceforge.net/)|math|4.3.0, 4.3.2|<details><summary>Expand</summary>Description: JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation Homepage: http://mcmc-jags.sourceforge.net/ URL: http://mcmc-jags.sourceforge.net/</details>|
	|[jasper](https://www.ece.uvic.ca/~frodo/jasper/)|vis|2.0.16, 4.0.0|<details><summary>Expand</summary>Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard. Homepage: https://www.ece.uvic.ca/~frodo/jasper/ URL: https://www.ece.uvic.ca/~frodo/jasper/</details>|
	|[java](http://openjdk.java.net)|tools|1.8.0_292, 1.8.0_192, 11.0.16_8, 11.0.2, 11.0.22, 13.0.2, 14.0.2, 17.0.2, 17.0.6, 21.0.1|<details><summary>Expand</summary>Description: Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers. Homepage: http://openjdk.java.net URL: http://openjdk.java.net</details>|
	|[jbigkit](https://www.cl.cam.ac.uk/~mgk25/jbigkit/)|-|2.1|<details><summary>Expand</summary>Description: JBIG-KIT is a software implementation of the JBIG1 data compression standard (ITU-T T.82), which was designed for bi-level image data, such as scanned documents. Homepage: https://www.cl.cam.ac.uk/~mgk25/jbigkit/ URL: https://www.cl.cam.ac.uk/~mgk25/jbigkit/</details>|
	|[jdftx](http://jdftx.org/index.html)|chem|1.7.0|<details><summary>Expand</summary>Description: JDFTx is a plane-wave density-functional theory (DFT) code designed to be as easy to develop with as it is easy to use. It is distributed under the GPL license (version 3 or higher) and publications resulting from its use must cite: R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory Homepage: http://jdftx.org/index.html URL: http://jdftx.org/index.html</details>|
	|[jellyfish](http://www.genome.umd.edu/jellyfish.html)|bio|2.3.0|<details><summary>Expand</summary>Description: Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. Homepage: http://www.genome.umd.edu/jellyfish.html URL: http://www.genome.umd.edu/jellyfish.html</details>|
	|[jemalloc](http://jemalloc.net)|-|5.2.1, 5.3.0|<details><summary>Expand</summary>Description: jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support. Homepage: http://jemalloc.net URL: http://jemalloc.net</details>|
	|[json-fortran](https://github.com/jacobwilliams/json-fortran)|-|8.3.0|<details><summary>Expand</summary>Description: JSON-Fortran: A Modern Fortran JSON API Homepage: https://github.com/jacobwilliams/json-fortran URL: https://github.com/jacobwilliams/json-fortran</details>|
	|[jsoncpp](https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html)|-|1.9.4, 1.9.5|<details><summary>Expand</summary>Description: JsonCpp is a C++ library that allows manipulating JSON values, including serialization and deserialization to and from strings. It can also preserve existing comment in unserialization/serialization steps, making it a convenient format to store user input files. Homepage: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html URL: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html</details>|
	|[judy](http://judy.sourceforge.net/)|-|1.0.5|<details><summary>Expand</summary>Description: A C library that implements a dynamic array. Homepage: http://judy.sourceforge.net/ URL: http://judy.sourceforge.net/</details>|
	|[julia](https://julialang.org)|tools|1.4.1, 1.5.2, 1.6.0, 1.6.1, 1.6.2, 1.7.0, 1.8.1, 1.8.5, 1.9.1, 1.9.3, 1.10.0, 1.11.3|<details><summary>Expand</summary>Description: Julia is a high-level, high-performance dynamic programming language for numerical computing Homepage: https://julialang.org URL: https://julialang.org</details>|
	|[kahip](https://kahip.github.io/)|-|3.14, 3.16|<details><summary>Expand</summary>Description: The graph partitioning framework KaHIP -- Karlsruhe High Quality Partitioning. Homepage: https://kahip.github.io/ URL: https://kahip.github.io/</details>|
	|[kahypar](https://kahypar.org/)|-|1.3.2|<details><summary>Expand</summary>Description: KaHyPar is a multilevel hypergraph partitioning framework providing direct k-way and recursive bisection based partitioning algorithms that compute solutions of very high quality. Homepage: https://kahypar.org/ URL: https://kahypar.org/</details>|
	|[kaiju](http://kaiju.binf.ku.dk/)|bio|1.6.2, 1.7.4, 1.10.1|<details><summary>Expand</summary>Description: Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments Homepage: http://kaiju.binf.ku.dk/ URL: http://kaiju.binf.ku.dk/</details>|
	|[kalign](https://github.com/TimoLassmann/kalign)|-|2.03, 3.3.5|<details><summary>Expand</summary>Description: Kalign is a fast multiple sequence alignment program for biological sequences. Homepage: https://github.com/TimoLassmann/kalign URL: https://github.com/TimoLassmann/kalign</details>|
	|[kallisto](https://pachterlab.github.io/kallisto/)|bio|0.46.1|<details><summary>Expand</summary>Description: kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. Homepage: https://pachterlab.github.io/kallisto/ URL: https://pachterlab.github.io/kallisto/</details>|
	|[kentutils](http://genome.cse.ucsc.edu/)|bio|401, 453|<details><summary>Expand</summary>Description: Kent utilities: collection of tools used by the UCSC genome browser. Homepage: http://genome.cse.ucsc.edu/ URL: http://genome.cse.ucsc.edu/</details>|
	|[kim-api](https://openkim.org/)|chem|2.1.3, 2.3.0|<details><summary>Expand</summary>Description: Open Knowledgebase of Interatomic Models. KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations. This is a library that can be used by simulation programs to get access to the models in the OpenKIM database. This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running kim-api-collections-management install user MODELNAME or kim-api-collections-management install user OpenKIM to install them all. Homepage: https://openkim.org/ URL: https://openkim.org/</details>|
	|[kma](https://bitbucket.org/genomicepidemiology/kma)|-|1.3.0, 1.3.25, 1.4.14|<details><summary>Expand</summary>Description: KMA is a mapping method designed to map raw reads directly against redundant databases, in an ultra-fast manner using seed and extend. Homepage: https://bitbucket.org/genomicepidemiology/kma URL: https://bitbucket.org/genomicepidemiology/kma</details>|
	|[kmergenie](http://kmergenie.bx.psu.edu/)|-|1.7051|<details><summary>Expand</summary>Description: KmerGenie estimates the best k-mer length for genome de novo assembly. Homepage: http://kmergenie.bx.psu.edu/ URL: http://kmergenie.bx.psu.edu/</details>|
	|[komplexity](https://github.com/eclarke/komplexity)|-|0.3.6|<details><summary>Expand</summary>Description: A command-line tool built in Rust to quickly calculate and/or mask low-complexity sequences from a FAST[A/Q] file. This uses the number of unique k-mers over a sequence divided by the length to assess complexity. Homepage: https://github.com/eclarke/komplexity URL: https://github.com/eclarke/komplexity</details>|
	|[kraken](https://ccb.jhu.edu/software/kraken/)|bio|1.1.1|<details><summary>Expand</summary>Description: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. Homepage: https://ccb.jhu.edu/software/kraken/ URL: https://ccb.jhu.edu/software/kraken/</details>|
	|[kraken2](https://github.com/DerrickWood/kraken2/wiki)|bio|2.0.9-beta, 2.1.1, 2.1.2, 2.1.3|<details><summary>Expand</summary>Description: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. Homepage: https://github.com/DerrickWood/kraken2/wiki URL: https://github.com/DerrickWood/kraken2/wiki</details>|
	|[krakenuniq](https://github.com/fbreitwieser/krakenuniq)|-|1.0.2|<details><summary>Expand</summary>Description: krakenuniq: confident and fast metagenomics classification using unique k-mer counts. False-positive identifications are a significant problem in metagenomics classification. KrakenUniq (formerly KrakenHLL) is a novel metagenomics classifier that combines the fast k-mer-based classification of Kraken with an efficient algorithm for assessing the coverage of unique k-mers found in each species in a dataset. On various test datasets, KrakenUniq gives better recall and precision than other methods and effectively classifies and distinguishes pathogens with low abundance from false positives in infectious disease samples. By using the probabilistic cardinality estimator HyperLogLog, KrakenUniq runs as fast as Kraken and requires little additional memory. Homepage: https://github.com/fbreitwieser/krakenuniq URL: https://github.com/fbreitwieser/krakenuniq</details>|
	|[kronatools](https://github.com/marbl/Krona/wiki/KronaTools)|-|2.8|<details><summary>Expand</summary>Description: Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files. Homepage: https://github.com/marbl/Krona/wiki/KronaTools URL: https://github.com/marbl/Krona/wiki/KronaTools</details>|
	|[lammps-omp](https://www.lammps.org)|chem|20201029, 20210929, 20220623, 20230802, 20240829|<details><summary>Expand</summary>Description: LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality. Homepage: https://www.lammps.org URL: https://www.lammps.org</details>|
	|[last](http://last.cbrc.jp/)|bio|1145|<details><summary>Expand</summary>Description: LAST finds similar regions between sequences. Homepage: http://last.cbrc.jp/ URL: http://last.cbrc.jp/</details>|
	|[lastz](https://www.bx.psu.edu/~rsharris/lastz/)|-|1.04.03|<details><summary>Expand</summary>Description: LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS sequencing technologies such as Roche 454. Homepage: https://www.bx.psu.edu/~rsharris/lastz/ URL: https://www.bx.psu.edu/~rsharris/lastz/</details>|
	|[latte](https://github.com/lanl/LATTE)|chem|1.2.1|<details><summary>Expand</summary>Description: Open source density functional tight binding molecular dynamics. Homepage: https://github.com/lanl/LATTE URL: https://github.com/lanl/LATTE</details>|
	|[ldc](https://wiki.dlang.org/LDC)|-|0.17.6, 1.26.0, 1.30.0|<details><summary>Expand</summary>Description: The LLVM-based D Compiler Homepage: https://wiki.dlang.org/LDC URL: https://wiki.dlang.org/LDC</details>|
	|[leptonica](http://www.leptonica.org)|-|1.82.0|<details><summary>Expand</summary>Description: Leptonica is a collection of pedagogically-oriented open source software that is broadly useful for image processing and image analysis applications. Homepage: http://www.leptonica.org URL: http://www.leptonica.org</details>|
	|[lerc](https://github.com/Esri/lerc)|-|4.0.0|<details><summary>Expand</summary>Description: LERC is an open-source image or raster format which supports rapid encoding and decoding for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding, so the precision of the original input image is preserved (within user defined error bounds). Homepage: https://github.com/Esri/lerc URL: https://github.com/Esri/lerc</details>|
	|[leveldb](https://github.com/google/leveldb)|tools|1.22|<details><summary>Expand</summary>Description: LevelDB is a fast key-value storage library written at Google that provides an ordered mapping from string keys to string values. Homepage: https://github.com/google/leveldb URL: https://github.com/google/leveldb</details>|
	|[lhapdf](https://lhapdf.hepforge.org/index.html)|-|6.4.0|<details><summary>Expand</summary>Description: LHAPDF is a general purpose C++ interpolator, used for evaluating PDFs from discretised data files. Previous versions of LHAPDF were written in Fortran 77/90 Homepage: https://lhapdf.hepforge.org/index.html URL: https://lhapdf.hepforge.org/index.html</details>|
	|[libaec](https://gitlab.dkrz.de/k202009/libaec)|-|1.0.6|<details><summary>Expand</summary>Description: Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa. Homepage: https://gitlab.dkrz.de/k202009/libaec URL: https://gitlab.dkrz.de/k202009/libaec</details>|
	|[libbeef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|-|0.1.2|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: https://github.com/vossjo/libbeef URL: https://github.com/vossjo/libbeef</details>|
	|[libbigwig](https://github.com/dpryan79/libBigWig)|-|0.4.6|<details><summary>Expand</summary>Description: A C library for handling bigWig files Homepage: https://github.com/dpryan79/libBigWig URL: https://github.com/dpryan79/libBigWig</details>|
	|[libccd](https://github.com/danfis/libccd)|-|2.1|<details><summary>Expand</summary>Description: libccd is library for a collision detection between two convex shapes. libccd implements variation on GilbertJohnsonKeerthi algorithm plus Expand Polytope Algorithm (EPA) and also implements algorithm Minkowski Portal Refinement (MPR, a.k.a. XenoCollide) as described in Game Programming Gems 7. Homepage: https://github.com/danfis/libccd URL: https://github.com/danfis/libccd</details>|
	|[libcdms](https://github.com/CDAT/libcdms/)|-|3.1.2|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcdms/ URL: https://github.com/CDAT/libcdms/</details>|
	|[libcerf](https://jugit.fz-juelich.de/mlz/libcerf)|math|1.13, 1.17, 2.1, 2.4|<details><summary>Expand</summary>Description: libcerf is a self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions. Homepage: https://jugit.fz-juelich.de/mlz/libcerf URL: https://jugit.fz-juelich.de/mlz/libcerf</details>|
	|[libcf](https://github.com/CDAT/libcf/)|-|1.0.3|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcf/ URL: https://github.com/CDAT/libcf/</details>|
	|[libctl](https://libctl.readthedocs.io/en/latest/)|tools|4.5.0, 4.5.1|<details><summary>Expand</summary>Description: libctl is a free Guile-based library implementing flexible control files for scientific simulations. Homepage: https://libctl.readthedocs.io/en/latest/ URL: https://libctl.readthedocs.io/en/latest/</details>|
	|[libdap](https://www.opendap.org/software/libdap)|-|3.20.6, 3.20.11|<details><summary>Expand</summary>Description: A C++ SDK which contains an implementation of DAP 2.0 and DAP4.0. This includes both Client- and Server-side support classes. Homepage: https://www.opendap.org/software/libdap URL: https://www.opendap.org/software/libdap</details>|
	|[libdivsufsort](https://github.com/y-256/libdivsufsort)|-|2.0.1|<details><summary>Expand</summary>Description: libdivsufsort is a software library that implements a lightweight suffix array construction algorithm. Homepage: https://github.com/y-256/libdivsufsort URL: https://github.com/y-256/libdivsufsort</details>|
	|[libdrs](https://github.com/CDAT/libdrs/)|-|3.1.2|<details><summary>Expand</summary>Description: PCMDI's old DRS format implementation Homepage: https://github.com/CDAT/libdrs/ URL: https://github.com/CDAT/libdrs/</details>|
	|[libevent](https://libevent.org/)|-|2.1.11|<details><summary>Expand</summary>Description: The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached. Furthermore, libevent also support callbacks due to signals or regular timeouts. Homepage: https://libevent.org/ URL: https://libevent.org/</details>|
	|[libfabric](https://ofiwg.github.io/libfabric/)|-|1.10.1, 1.11.0, 1.12.1, 1.15.1, 1.18.0, 1.21.0|<details><summary>Expand</summary>Description: Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. Homepage: https://ofiwg.github.io/libfabric/ URL: https://ofiwg.github.io/libfabric/</details>|
	|[libfdf](https://launchpad.net/libfdf)|-|0.2.2|<details><summary>Expand</summary>Description: LibFDF provides a Fortran API to parse files in the Flexible Data Format (FDF). Homepage: https://launchpad.net/libfdf URL: https://launchpad.net/libfdf</details>|
	|[libffi](https://sourceware.org/libffi/)|-|3.3|<details><summary>Expand</summary>Description: The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time. Homepage: https://sourceware.org/libffi/ URL: https://sourceware.org/libffi/</details>|
	|[libgd](https://libgd.github.io)|vis|2.3.0, 2.3.3|<details><summary>Expand</summary>Description: GD is an open source code library for the dynamic creation of images by programmers. Homepage: https://libgd.github.io URL: https://libgd.github.io</details>|
	|[libgdsii](https://github.com/HomerReid/libGDSII)|-|0.21|<details><summary>Expand</summary>Description: libGDSII is a C++ library for working with GDSII binary data files, intended primarily for use with the computational electromagnetism codes scuff-em and meep but sufficiently general-purpose to allow other uses as well. Homepage: https://github.com/HomerReid/libGDSII URL: https://github.com/HomerReid/libGDSII</details>|
	|[libgeotiff](https://directory.fsf.org/wiki/Libgeotiff)|-|1.5.1, 1.6.0, 1.7.1|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libgeotiff-proj901](https://directory.fsf.org/wiki/Libgeotiff)|-|1.7.1|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libgridxc](https://launchpad.net/libgridxc)|chem|0.8.0|<details><summary>Expand</summary>Description: A library to compute the exchange and correlation energy and potential in spherical (i.e. an atom) or periodic systems. It is based on SiestaXC. Homepage: https://launchpad.net/libgridxc URL: https://launchpad.net/libgridxc</details>|
	|[libgridxc-mpi](https://launchpad.net/libgridxc)|-|0.8.0|<details><summary>Expand</summary>Description: A library to compute the exchange and correlation energy and potential in spherical (i.e. an atom) or periodic systems. It is based on SiestaXC. Homepage: https://launchpad.net/libgridxc URL: https://launchpad.net/libgridxc</details>|
	|[libgtextutils](http://hannonlab.cshl.edu/fastx_toolkit/)|bio|0.7|<details><summary>Expand</summary>Description: ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream Homepage: http://hannonlab.cshl.edu/fastx_toolkit/ URL: http://hannonlab.cshl.edu/fastx_toolkit/</details>|
	|[libharu](https://github.com/libharu/libharu/)|-|2.3.0|<details><summary>Expand</summary>Description: libHaru is a free, cross platform, open source library for generating PDF files. Homepage: https://github.com/libharu/libharu/ URL: https://github.com/libharu/libharu/</details>|
	|[libint](https://github.com/evaleev/libint)|chem|1.1.6, 2.0.3, 2.4.2, 2.6.0, 2.7.2|<details><summary>Expand</summary>Description: Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory. Homepage: https://github.com/evaleev/libint URL: https://github.com/evaleev/libint</details>|
	|[libmaus2](https://github.com/gt1/libmaus2)|bio|2.0.499|<details><summary>Expand</summary>Description: libmaus2 is a collection of data structures and algorithms. Homepage: https://github.com/gt1/libmaus2 URL: https://github.com/gt1/libmaus2</details>|
	|[libmbd](https://github.com/libmbd/libmbd)|-|0.12.8|<details><summary>Expand</summary>Description: A general-purpose package for scalable quantum many-body dispersion calculations. Homepage: https://github.com/libmbd/libmbd URL: https://github.com/libmbd/libmbd</details>|
	|[libmesh](https://github.com/libMesh/libmesh)|math|1.7.1, 1.7.5|<details><summary>Expand</summary>Description: The libMesh library provides a framework for the numerical simulation of partial differential equations using arbitrary unstructured discretizations on serial and parallel platforms. A major goal of the library is to provide support for adaptive mesh refinement (AMR) computations in parallel while allowing a research scientist to focus on the physics they are modeling. Homepage: https://github.com/libMesh/libmesh URL: https://github.com/libMesh/libmesh</details>|
	|[libpsml](https://gitlab.com/siesta-project/libraries/libpsml)|-|1.1.8, 1.1.12|<details><summary>Expand</summary>Description: A library to handle pseudopotentials in PSML format Homepage: https://gitlab.com/siesta-project/libraries/libpsml URL: https://gitlab.com/siesta-project/libraries/libpsml</details>|
	|[libraw](https://www.libraw.org/)|-|0.21.2|<details><summary>Expand</summary>Description: The LibRaw library provides a simple and unified interface for extracting out of RAW files generated by digital photo cameras the following: - RAW data (pixel values) - Metadata necessary for processing RAW (geometry, CFA / Bayer pattern, black level, white balance, etc.) - Embedded preview / thumbnail. Homepage: https://www.libraw.org/ URL: https://www.libraw.org/</details>|
	|[librdkafka](https://github.com/edenhill/librdkafka)|-|1.5.2-RC1|<details><summary>Expand</summary>Description: librdkafka is a C library implementation of the Apache Kafka protocol, providing Producer, Consumer and Admin clients. It was designed with message delivery reliability and high performance in mind, current figures exceed 1 million msgs/second for the producer and 3 million msgs/second for the consumer. librdkafka is licensed under the 2-clause BSD license. Homepage: https://github.com/edenhill/librdkafka URL: https://github.com/edenhill/librdkafka</details>|
	|[librttopo](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[librttopo-proj9](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[libspatialindex](https://libspatialindex.org)|phys|1.8.5, 1.9.3|<details><summary>Expand</summary>Description: C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API Homepage: https://libspatialindex.org URL: https://libspatialindex.org</details>|
	|[libspatialite](https://www.gaia-gis.it/fossil/libspatialite/home)|phys|4.3.0a, 5.0.1, 5.1.0|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libspatialite-proj9](https://www.gaia-gis.it/fossil/libspatialite/home)|-|5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libspatialite-proj901](https://www.gaia-gis.it/fossil/libspatialite/home)|-|5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libstatgen](https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen)|-|20190330|<details><summary>Expand</summary>Description: Useful set of classes for creating statistical genetic programs. Homepage: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen URL: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen</details>|
	|[liburing](https://github.com/axboe/liburing)|-|2.3|<details><summary>Expand</summary>Description: liburing provides helpers to setup and teardown io_uring instances, and also a simplified interface for applications that don't need (or want) to deal with the full kernel side implementation. Homepage: https://github.com/axboe/liburing URL: https://github.com/axboe/liburing</details>|
	|[libvori](https://brehm-research.de/libvori.php)|-|220621|<details><summary>Expand</summary>Description: C++ library implementing the Voronoi integration as well as the compressed bqb file format. The present version of libvori is a very early development version, which is hard-coded to work with the CP2k program package. Homepage: https://brehm-research.de/libvori.php URL: https://brehm-research.de/libvori.php</details>|
	|[libxc](https://libxc.gitlab.io)|chem|2.2.2, 3.0.0, 3.0.1, 4.3.4, 5.0.0, 5.1.3, 5.2.3, 6.2.2, 7.0.0|<details><summary>Expand</summary>Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals. Homepage: https://libxc.gitlab.io URL: https://libxc.gitlab.io</details>|
	|[libxslt](http://xmlsoft.org/)|-|1.1.29, 1.1.34|<details><summary>Expand</summary>Description: Libxslt is the XSLT C library developed for the GNOME project (but usable outside of the Gnome platform). Homepage: http://xmlsoft.org/ URL: http://xmlsoft.org/</details>|
	|[libxsmm](https://github.com/hfp/libxsmm)|math|1.16.1, 1.17|<details><summary>Expand</summary>Description: LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86). Homepage: https://github.com/hfp/libxsmm URL: https://github.com/hfp/libxsmm</details>|
	|[libyaml](https://pyyaml.org/wiki/LibYAML)|-|0.2.5|<details><summary>Expand</summary>Description: LibYAML is a YAML parser and emitter written in C. Homepage: https://pyyaml.org/wiki/LibYAML URL: https://pyyaml.org/wiki/LibYAML</details>|
	|[liknorm](https://github.com/limix/liknorm)|tools|1.5.1|<details><summary>Expand</summary>Description: C library for computing moments of the product of an exponential-family likelihood with a Normal distribution. Homepage: https://github.com/limix/liknorm URL: https://github.com/limix/liknorm</details>|
	|[limix-bgen](https://github.com/limix/bgen)|-|3.0.3, 4.1.0|<details><summary>Expand</summary>Description: A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3. Homepage: https://github.com/limix/bgen URL: https://github.com/limix/bgen</details>|
	|[links](http://www.bcgsc.ca/platform/bioinfo/software/arcs).)|bio|1.8.6, 2.0.1|<details><summary>Expand</summary>Description: LINKS is a scalable genomics application for scaffolding or re-scaffolding genome assembly drafts with long reads, such as those produced by Oxford Nanopore Technologies Ltd and Pacific Biosciences. It provides a generic alignment-free framework for scaffolding and can work on any sequences. It is versatile and supports not only long sequences as a source of long-range information, but also MPET pairs and linked-reads, such as those from the 10X Genomics GemCode and Chromium platform, via ARCS (http://www.bcgsc.ca/platform/bioinfo/software/arcs). Fill gaps in LINKS-derived scaffolds using Sealer (http://www.bcgsc.ca/platform/bioinfo/software/sealer). Homepage: http://www.bcgsc.ca/platform/bioinfo/software/links URL: http://www.bcgsc.ca/platform/bioinfo/software/links</details>|
	|[littlecms](http://www.littlecms.com/)|-|2.9, 2.11|<details><summary>Expand</summary>Description: Little CMS intends to be an OPEN SOURCE small-footprint color management engine, with special focus on accuracy and performance. Homepage: http://www.littlecms.com/ URL: http://www.littlecms.com/</details>|
	|[lldb](https://lldb.llvm.org/)|-|11.0.0|<details><summary>Expand</summary>Description: The debugger component of the LLVM project Homepage: https://lldb.llvm.org/ URL: https://lldb.llvm.org/</details>|
	|[llvm](https://llvm.org/)|tools|8.0.1, 9.0.1, 11.1.0, 13.0.1, 14.0.3, 16.0.6, 18.1.8|<details><summary>Expand</summary>Description: The LLVM Core libraries provide a modern source- and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!) These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR"). The LLVM Core libraries are well documented, and it is particularly easy to invent your own language (or port an existing compiler) to use LLVM as an optimizer and code generator. Homepage: https://llvm.org/ URL: https://llvm.org/</details>|
	|[lmdb](https://symas.com/lmdb)|-|0.9.24, 0.9.31|<details><summary>Expand</summary>Description: LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance of a pure in-memory database while retaining the persistence of standard disk-based databases. Homepage: https://symas.com/lmdb URL: https://symas.com/lmdb</details>|
	|[lpsolve](https://sourceforge.net/projects/lpsolve/)|math|5.5.2.5|<details><summary>Expand</summary>Description: Mixed Integer Linear Programming (MILP) solver Homepage: https://sourceforge.net/projects/lpsolve/ URL: https://sourceforge.net/projects/lpsolve/</details>|
	|[ls-dyna](http://www.lstc.com/products/ls-dyna)|phys|12.0, 12.2.1, 13.0, 13.1.1|<details><summary>Expand</summary>Description: LS-DYNA is a general-purpose finite element program capable of simulating complex real world problems. It is used by the automobile, aerospace, construction, military, manufacturing, and bioengineering industries. LS-DYNA is optimized for shared and distributed memory Unix, Linux, and Windows based, platforms, and it is fully QA'd by LSTC. The code's origins lie in highly nonlinear, transient dynamic finite element analysis using explicit time integration. Homepage: http://www.lstc.com/products/ls-dyna URL: http://www.lstc.com/products/ls-dyna</details>|
	|[ls-dyna-mpi](http://www.lstc.com/products/ls-dyna)|phys|12.0, 12.2.1, 13.0, 13.1.1|<details><summary>Expand</summary>Description: LS-DYNA is a general-purpose finite element program capable of simulating complex real world problems. It is used by the automobile, aerospace, construction, military, manufacturing, and bioengineering industries. LS-DYNA is optimized for shared and distributed memory Unix, Linux, and Windows based, platforms, and it is fully QA'd by LSTC. The code's origins lie in highly nonlinear, transient dynamic finite element analysis using explicit time integration. Homepage: http://www.lstc.com/products/ls-dyna URL: http://www.lstc.com/products/ls-dyna</details>|
	|[ls-opt](http://www.lstc.com/products/ls-opt)|phys|7.0.0|<details><summary>Expand</summary>Description: LS-OPT is a standalone Design Optimization and Probabilistic Analysis package with an interface to LS-DYNA. Homepage: http://www.lstc.com/products/ls-opt URL: http://www.lstc.com/products/ls-opt</details>|
	|[ls-prepost](http://lstc.com/products/ls-prepost)|-|4.8.11, 4.9.9|<details><summary>Expand</summary>Description: LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA. Homepage: http://lstc.com/products/ls-prepost URL: http://lstc.com/products/ls-prepost</details>|
	|[lsd2](https://github.com/tothuhien/lsd2)|-|1.9.7, 2.3, 2.4.1|<details><summary>Expand</summary>Description: Least-squares methods to estimate rates and dates from phylogenies Homepage: https://github.com/tothuhien/lsd2 URL: https://github.com/tothuhien/lsd2</details>|
	|[ltr_retriever](https://github.com/oushujun/LTR_retriever)|-|2.9.0|<details><summary>Expand</summary>Description: LTR_retriever is a command line program (in Perl) for accurate identification of LTR retrotransposons (LTR-RTs) from outputs of LTRharvest, LTR_FINDER, MGEScan 3.0.0, LTR_STRUC, and LtrDetector, and generates non-redundant LTR-RT library for genome annotations. By default, the program will generate whole-genome LTR-RT annotation and the LTR Assembly Index (LAI) for evaluations of the assembly continuity of the input genome. Homepage: https://github.com/oushujun/LTR_retriever URL: https://github.com/oushujun/LTR_retriever</details>|
	|[lumpy](https://github.com/arq5x/lumpy-sv)|bio|0.2.13|<details><summary>Expand</summary>Description: A probabilistic framework for structural variant discovery. Homepage: https://github.com/arq5x/lumpy-sv URL: https://github.com/arq5x/lumpy-sv</details>|
	|[m2](https://github.com/Macaulay2/M2)|-|1.19.1, 1.21|<details><summary>Expand</summary>Description: Macaulay2 is a system for computing in commutative algebra, algebraic geometry and related fields. The system was originally written by Dan Grayson and Mike Stillman. David Eisenbud joined the project a number of years ago, and many users are writing packages for the system, and some are contributing source code. Homepage: https://github.com/Macaulay2/M2 URL: https://github.com/Macaulay2/M2</details>|
	|[maeparser](https://github.com/schrodinger/maeparser)|chem|1.2.4|<details><summary>Expand</summary>Description: maeparser is a parser for Schrodinger Maestro files. Homepage: https://github.com/schrodinger/maeparser URL: https://github.com/schrodinger/maeparser</details>|
	|[maffilter](https://jydu.github.io/maffilter/)|-|2020.11|<details><summary>Expand</summary>Description: Genome Alignment Processor Homepage: https://jydu.github.io/maffilter/ URL: https://jydu.github.io/maffilter/</details>|
	|[mafft](https://mafft.cbrc.jp/alignment/software/source.html)|bio|7.471, 7.526|<details><summary>Expand</summary>Description: MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc. Homepage: https://mafft.cbrc.jp/alignment/software/source.html URL: https://mafft.cbrc.jp/alignment/software/source.html</details>|
	|[mafft-mpi](https://mafft.cbrc.jp/alignment/software/source.html)|-|7.471, 7.526|<details><summary>Expand</summary>Description: MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc. Homepage: https://mafft.cbrc.jp/alignment/software/source.html URL: https://mafft.cbrc.jp/alignment/software/source.html</details>|
	|[magma](https://icl.cs.utk.edu/magma/)|math|2.5.4, 2.6.1, 2.7.1, 2.7.2, 2.8.0|<details><summary>Expand</summary>Description: The MAGMA project aims to develop a dense linear algebra library similar to LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems. Homepage: https://icl.cs.utk.edu/magma/ URL: https://icl.cs.utk.edu/magma/</details>|
	|[magma-gene-analysis](https://ctg.cncr.nl/software/magma)|-|1.10|<details><summary>Expand</summary>Description: MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous GWAS or meta-analysis. Homepage: https://ctg.cncr.nl/software/magma URL: https://ctg.cncr.nl/software/magma</details>|
	|[makeinfo](https://www.gnu.org/software/texinfo/)|-|6.7|<details><summary>Expand</summary>Description: makeinfo is part of the Texinfo project, the official documentation format of the GNU project. Homepage: https://www.gnu.org/software/texinfo/ URL: https://www.gnu.org/software/texinfo/</details>|
	|[maker](http://www.yandell-lab.org/software/maker.html)|bio|3.01.03|<details><summary>Expand</summary>Description: MAKER is a portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases. MAKER identifies repeats, aligns ESTs and proteins to a genome, produces ab-initio gene predictions and automatically synthesizes these data into gene annotations having evidence-based quality values. Homepage: http://www.yandell-lab.org/software/maker.html URL: http://www.yandell-lab.org/software/maker.html</details>|
	|[manta](https://github.com/Illumina/manta)|bio|1.6.0|<details><summary>Expand</summary>Description: Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. It is optimized for analysis of germline variation in small sets of individuals and somatic variation in tumor/normal sample pairs. Manta discovers, assembles and scores large-scale SVs, medium-sized indels and large insertions within a single efficient workflow. Homepage: https://github.com/Illumina/manta URL: https://github.com/Illumina/manta</details>|
	|[marginpolish](https://github.com/UCSC-nanopore-cgl/MarginPolish)|-|1.3.0|<details><summary>Expand</summary>Description: MarginPolish is a graph-based assembly polisher. It iteratively finds multiple probable alignment paths for run-length-encoded reads and uses these to generate a refined sequence. It takes as input a FASTA assembly and an indexed BAM (ONT reads aligned to the assembly), and it produces a polished FASTA assembly. Homepage: https://github.com/UCSC-nanopore-cgl/MarginPolish URL: https://github.com/UCSC-nanopore-cgl/MarginPolish</details>|
	|[mariadb](https://docs.computecanada.ca/wiki/Database_servers.)|tools|10.4.13, 10.6.12, 11.5.0|<details><summary>Expand</summary>Description: MariaDB An enhanced, drop-in replacement for MySQL For more information please take a look at https://docs.computecanada.ca/wiki/Database_servers. Homepage: https://mariadb.org/ URL: https://mariadb.org/</details>|
	|[mariadb-connector-c](https://downloads.mariadb.org/connector-c/)|-|3.1.7, 3.3.7|<details><summary>Expand</summary>Description: MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases. Homepage: https://downloads.mariadb.org/connector-c/ URL: https://downloads.mariadb.org/connector-c/</details>|
	|[mash](http://mash.readthedocs.org)|-|2.3|<details><summary>Expand</summary>Description: Fast genome and metagenome distance estimation using MinHash Homepage: http://mash.readthedocs.org URL: http://mash.readthedocs.org</details>|
	|[masurca](http://www.genome.umd.edu/masurca.html)|bio|3.4.1, 4.0.1, 4.0.3, 4.1.0|<details><summary>Expand</summary>Description: MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454, Pacbio and Nanopore). Homepage: http://www.genome.umd.edu/masurca.html URL: http://www.genome.umd.edu/masurca.html</details>|
	|[matio](https://sourceforge.net/projects/matio/)|io|1.5.19, 1.5.26|<details><summary>Expand</summary>Description: matio is an C library for reading and writing Matlab MAT files. Homepage: https://sourceforge.net/projects/matio/ URL: https://sourceforge.net/projects/matio/</details>|
	|[matlab](http://www.mathworks.com/products/matlab)|tools|2020a, 2020b.4, 2020b.6, 2021a.1, 2021a.5, 2021b.3, 2022a, 2022b.2, 2023a.3, 2023b.2, 2024b.1|<details><summary>Expand</summary>Description: MATLAB is a high-level language and interactive environment that enables you to perform computationally intensive tasks faster than with traditional programming languages such as C, C++, and Fortran. Homepage: http://www.mathworks.com/products/matlab URL: http://www.mathworks.com/products/matlab</details>|
	|[maven](https://maven.apache.org/index.html)|tools|3.6.3, 3.9.6|<details><summary>Expand</summary>Description: Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. Homepage: https://maven.apache.org/index.html URL: https://maven.apache.org/index.html</details>|
	|[maxbin](https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html)|bio|2.2.7|<details><summary>Expand</summary>Description: MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm. Homepage: https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html URL: https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</details>|
	|[mcl](http://micans.org/mcl/)|math|14.137|<details><summary>Expand</summary>Description: The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for networks (also known as graphs) based on simulation of (stochastic) flow in graphs. The algorithm was invented/discovered by Stijn van Dongen at the Centre for Mathematics and Computer Science (also known as CWI) in the Netherlands. MCL has been applied in a number of different domains, mostly in bioinformatics. Homepage: http://micans.org/mcl/ URL: http://micans.org/mcl/</details>|
	|[mcr](https://www.mathworks.com/products/compiler/mcr/)|tools|R2021b, R2020b, R2022b, R2024a|<details><summary>Expand</summary>Description: The MATLAB Runtime is a standalone set of shared libraries that enables the execution of compiled MATLAB applications or components on computers that do not have MATLAB installed. Homepage: https://www.mathworks.com/products/compiler/mcr/ URL: https://www.mathworks.com/products/compiler/mcr/</details>|
	|[mctc-lib](https://grimme-lab.github.io/mctc-lib)|-|0.3.1|<details><summary>Expand</summary>Description: Common tool chain for working with molecular structure data in various applications. This library provides a unified way to perform operations on molecular structure data, like reading and writing to common geometry file formats. Homepage: https://grimme-lab.github.io/mctc-lib URL: https://grimme-lab.github.io/mctc-lib</details>|
	|[meep](https://meep.readthedocs.io/en/latest/)|phys|1.16.1, 1.24.0, 1.25.0, 1.29.0|<details><summary>Expand</summary>Description: Meep is a free and open-source software package for electromagnetics simulation via the finite-difference time-domain (FDTD) method spanning a broad range of applications. Homepage: https://meep.readthedocs.io/en/latest/ URL: https://meep.readthedocs.io/en/latest/</details>|
	|[megahit](https://github.com/voutcn/megahit)|bio|1.2.9|<details><summary>Expand</summary>Description: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph Homepage: https://github.com/voutcn/megahit URL: https://github.com/voutcn/megahit</details>|
	|[meme](http://meme-suite.org)|-|5.2.0, 5.4.1, 5.5.0, 5.5.5, 5.5.7|<details><summary>Expand</summary>Description: The MEME Suite allows you to: - discover motifs using MEME, DREME (DNA only) or GLAM2 on groups of related DNA or protein sequences, - search sequence databases with motifs using MAST, FIMO, MCAST or GLAM2SCAN, - compare a motif to all motifs in a database of motifs, - associate motifs with Gene Ontology terms via their putative target genes, - analyse motif enrichment using SpaMo or CentriMo. Homepage: http://meme-suite.org URL: http://meme-suite.org</details>|
	|[mesa-astrophysics](https://docs.mesastar.org)|phys|r24.03.1|<details><summary>Expand</summary>Description: MESA Modules for Experiments in Stellar Astrophysics Homepage: https://docs.mesastar.org URL: https://docs.mesastar.org</details>|
	|[mesquite](https://software.sandia.gov/mesquite/)|math|2.3.0|<details><summary>Expand</summary>Description: Mesh-Quality Improvement Library Homepage: https://software.sandia.gov/mesquite/ URL: https://software.sandia.gov/mesquite/</details>|
	|[met](https://dtcenter.org/)|phys|9.1.1|<details><summary>Expand</summary>Description: Model Evaluation Tools (MET) is a highly configurable, state-of-the-art suite of verification tools designed for the verification and evaluation of numerical weather forecasts. Although MET was developed at the Developmental Testbed Center (DTC) to be used with the Weather Research and Forecasting (WRF) modeling system, the data formats and standards used allow MET to be run on the output of other modeling systems as well. Homepage: https://dtcenter.org/ URL: https://dtcenter.org/</details>|
	|[meta-farm](https://docs.computecanada.ca/wiki/META_package_for_serial_farming)|-|1.0.2, 1.0.3|<details><summary>Expand</summary>Description: Meta-farm is a suite of scripts designed in-house to fully automate throughput computing (running a large number of related serial/parallel/GPU jobs - so called farming). Homepage: https://docs.computecanada.ca/wiki/META_package_for_serial_farming URL: https://docs.computecanada.ca/wiki/META_package_for_serial_farming</details>|
	|[metabat](https://bitbucket.org/berkeleylab/metabat)|bio|2.12.1, 2.14, 2.17|<details><summary>Expand</summary>Description: An efficient tool for accurately reconstructing single genomes from complex microbial communities Homepage: https://bitbucket.org/berkeleylab/metabat URL: https://bitbucket.org/berkeleylab/metabat</details>|
	|[metaeuk](https://metaeuk.soedinglab.org)|-|4-a0f584d, 6|<details><summary>Expand</summary>Description: MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic metagenomic contigs. Homepage: https://metaeuk.soedinglab.org URL: https://metaeuk.soedinglab.org</details>|
	|[metageneannotator](http://metagene.nig.ac.jp/)|-|20080819|<details><summary>Expand</summary>Description: MetaGeneAnnotator is a gene-finding program for prokaryote and phage. Homepage: http://metagene.nig.ac.jp/ URL: http://metagene.nig.ac.jp/</details>|
	|[metagenome-atlas](https://github.com/metagenome-atlas/atlas)|-|2.4.3, 2.5.0|<details><summary>Expand</summary>Description: Metagenome-atlas is a easy-to-use metagenomic pipeline based on snakemake. It handles all steps from QC, Assembly, Binning, to Annotation. Homepage: https://github.com/metagenome-atlas/atlas URL: https://github.com/metagenome-atlas/atlas</details>|
	|[metal](http://www.sph.umich.edu/csg/abecasis/metal/)|bio|2011-03-25|<details><summary>Expand</summary>Description: METAL provides a computationally efficient tool for meta-analysis of genome-wide association scans, which is a commonly used approach for improving power complex traits gene mapping studies. METAL provides a rich scripting interface and implements efficient memory management to allow analyses of very large data sets and to support a variety of input file formats. Availability and implementation: METAL, including source code, documentation, examples, and executables, is available at: http://www.sph.umich.edu/csg/abecasis/metal/ If you use Metal please fill out the registration form on the page: http://csg.sph.umich.edu/abecasis/metal/download/ Contact: goncalo@umich.edu Homepage: http://www.sph.umich.edu/csg/abecasis/metal/ URL: http://www.sph.umich.edu/csg/abecasis/metal/</details>|
	|[metamaps](https://github.com/DiltheyLab/MetaMaps)|-|2020.03.11|<details><summary>Expand</summary>Description: MetaMaps is tool specifically developed for the analysis of long-read (PacBio/Oxford Nanopore) metagenomic datasets. It simultaenously carries out read assignment and sample composition estimation. It is faster than classical exact alignment-based approaches, and its output is more information-rich than that of kmer-spectra-based methods. For example, each MetaMaps alignment comes with an approximate alignment location, an estimated alignment identity and a mapping quality. The approximate mapping algorithm employed by MetaMaps is based on MashMap. MetaMaps adds a mapping quality model and EM-based estimation of sample composition. Homepage: https://github.com/DiltheyLab/MetaMaps URL: https://github.com/DiltheyLab/MetaMaps</details>|
	|[metaxa2](https://microbiology.se/software/metaxa2/)|-|2.2|<details><summary>Expand</summary>Description: Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically Homepage: https://microbiology.se/software/metaxa2/ URL: https://microbiology.se/software/metaxa2/</details>|
	|[methyldackel](https://github.com/dpryan79/MethylDackel)|-|0.6.1|<details><summary>Expand</summary>Description: A (mostly) universal methylation extractor for BS-seq experiments. Homepage: https://github.com/dpryan79/MethylDackel URL: https://github.com/dpryan79/MethylDackel</details>|
	|[metis](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|math|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[metis-64idx](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|-|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[migrate-n](http://popgen.sc.fsu.edu/Migrate/Migrate-n.html)|-|4.4.4|<details><summary>Expand</summary>Description: Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data. It uses a coalescent theory approach taking into account history of mutations and uncertainty of the genealogy. Homepage: http://popgen.sc.fsu.edu/Migrate/Migrate-n.html URL: http://popgen.sc.fsu.edu/Migrate/Migrate-n.html</details>|
	|[mii](https://github.com/codeandkey/mii)|-|1.1.1, 1.1.2|<details><summary>Expand</summary>Description: A smart search engine for module environments. Homepage: https://github.com/codeandkey/mii URL: https://github.com/codeandkey/mii</details>|
	|[minc-toolkit](http://bic-mni.github.io/)|bio|1.9.18.1, 1.9.18.3|<details><summary>Expand</summary>Description: Medical Imaging NetCDF Toolkit Homepage: http://bic-mni.github.io/ URL: http://bic-mni.github.io/</details>|
	|[minced](https://github.com/ctSkennerton/minced)|-|0.4.2|<details><summary>Expand</summary>Description: Mining CRISPRs in Environmental Datasets Homepage: https://github.com/ctSkennerton/minced URL: https://github.com/ctSkennerton/minced</details>|
	|[minia](https://github.com/GATB/gatb-core)|bio|3.2.6|<details><summary>Expand</summary>Description: GATB - The Genome Analysis Toolbox with de-Bruijn graph. Homepage: https://github.com/GATB/gatb-core URL: https://github.com/GATB/gatb-core</details>|
	|[miniasm](https://github.com/lh3/minimap2)|bio|0.3-20191007|<details><summary>Expand</summary>Description: Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format. Different from mainstream assemblers, miniasm does not have a consensus step. It simply concatenates pieces of read sequences to generate the final unitig sequences. Thus the per-base error rate is similar to the raw input reads. Homepage: https://github.com/lh3/minimap2 URL: https://github.com/lh3/minimap2</details>|
	|[minimac3](http://genome.sph.umich.edu/wiki/Minimac3)|bio|2.0.1|<details><summary>Expand</summary>Description: Minimac3 is a lower memory and more computationally efficient implementation of the genotype imputation algorithms in minimac and minimac2. Minimac3 is designed to handle very large reference panels in a more computationally efficient way with no loss of accuracy. Homepage: http://genome.sph.umich.edu/wiki/Minimac3 URL: http://genome.sph.umich.edu/wiki/Minimac3</details>|
	|[minimac4](https://genome.sph.umich.edu/wiki/Minimac4)|-|1.0.2|<details><summary>Expand</summary>Description: Minimac4 is a latest version in the series of genotype imputation software - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010). Minimac4 is a lower memory and more computationally efficient implementation of the original algorithms with comparable imputation quality. Homepage: https://genome.sph.umich.edu/wiki/Minimac4 URL: https://genome.sph.umich.edu/wiki/Minimac4</details>|
	|[minimap2](https://github.com/lh3/minimap2)|bio|2.17, 2.18, 2.24, 2.26, 2.28|<details><summary>Expand</summary>Description: Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment. Homepage: https://github.com/lh3/minimap2 URL: https://github.com/lh3/minimap2</details>|
	|[minpath](https://omics.informatics.indiana.edu/MinPath)|-|1.6|<details><summary>Expand</summary>Description: MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological pathways for a query dataset. Homepage: https://omics.informatics.indiana.edu/MinPath URL: https://omics.informatics.indiana.edu/MinPath</details>|
	|[mixcr](http://mixcr.readthedocs.io/en/latest/quickstart.html)|bio|4.1.2, 4.7.0|<details><summary>Expand</summary>Description: MiXCR is a universal framework that processes big immunome data from raw sequences to quantitated clonotypes. MiXCR efficiently handles paired- and single-end reads, considers sequence quality, corrects PCR errors and identifies germline hypermutations. The software supports both partial- and full-length profiling and employs all available RNA or DNA information, including sequences upstream of V and downstream of J gene segments. Homepage: http://mixcr.readthedocs.io/en/latest/quickstart.html URL: http://mixcr.readthedocs.io/en/latest/quickstart.html</details>|
	|[mmg](https://www.mmgtools.org/)|-|5.7.3|<details><summary>Expand</summary>Description: Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries: the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation and generation of a triangulation from a set of points or from given boundary edges the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries. Homepage: https://www.mmgtools.org/ URL: https://www.mmgtools.org/</details>|
	|[mmseqs2](https://mmseqs.com)|-|14-7e284, 15-6f452, 13-45111|<details><summary>Expand</summary>Description: MMseqs2: ultra fast and sensitive search and clustering suite Homepage: https://mmseqs.com URL: https://mmseqs.com</details>|
	|[modflow](https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs)|-|6.5.0|<details><summary>Expand</summary>Description: MODFLOW is the USGS's modular hydrologic model. MODFLOW is considered an international standard for simulating and predicting groundwater conditions and groundwater/surface-water interactions. Homepage: https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs URL: https://www.usgs.gov/mission-areas/water-resources/science/modflow-and-related-programs</details>|
	|[molden](http://www.cmbi.ru.nl/molden/)|chem|6.5|<details><summary>Expand</summary>Description: Molden is a package for displaying Molecular Density from the Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical packages Mopac/Ampac Homepage: http://www.cmbi.ru.nl/molden/ URL: http://www.cmbi.ru.nl/molden/</details>|
	|[mono](https://www.mono-project.com/)|tools|6.12.0.122|<details><summary>Expand</summary>Description: An open source, cross-platform, implementation of C# and the CLR that is binary compatible with Microsoft.NET. Homepage: https://www.mono-project.com/ URL: https://www.mono-project.com/</details>|
	|[mothur](http://www.mothur.org/)|bio|1.44.3, 1.46.1, 1.47.0, 1.48.0|<details><summary>Expand</summary>Description: Mothur is a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community. Homepage: http://www.mothur.org/ URL: http://www.mothur.org/</details>|
	|[mpas](https://mpas-dev.github.io)|-|7.0|<details><summary>Expand</summary>Description: The Model for Prediction Across Scales (MPAS) is a collaborative project for developing atmosphere, ocean and other earth-system simulation components for use in climate, regional climate and weather studies. Homepage: https://mpas-dev.github.io URL: https://mpas-dev.github.io</details>|
	|[mpb](https://mpb.readthedocs.io/en/latest)|-|1.11.1|<details><summary>Expand</summary>Description: MPB is a software package to compute definite-frequency eigenstates of Maxwell's equations in periodic dielectric structures. It can compute optical dispersion relations and eigenstates for structures such as strip waveguides and optical fibers. MPB is well suited for the study of photonic crystals: periodic dielectric structures exhibiting a band gap in their optical modes, prohibiting propagation of light in that frequency range. Homepage: https://mpb.readthedocs.io/en/latest URL: https://mpb.readthedocs.io/en/latest</details>|
	|[mpb-mpi](https://mpb.readthedocs.io/en/latest)|-|1.11.1|<details><summary>Expand</summary>Description: MPB is a software package to compute definite-frequency eigenstates of Maxwell's equations in periodic dielectric structures. It can compute optical dispersion relations and eigenstates for structures such as strip waveguides and optical fibers. MPB is well suited for the study of photonic crystals: periodic dielectric structures exhibiting a band gap in their optical modes, prohibiting propagation of light in that frequency range. Homepage: https://mpb.readthedocs.io/en/latest URL: https://mpb.readthedocs.io/en/latest</details>|
	|[mpi4py](https://github.com/mpi4py/mpi4py)|tools|3.0.3, 3.1.2, 3.1.3, 3.1.4, 3.1.6, 4.0.0, 4.0.3|<details><summary>Expand</summary>Description: MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for the Python programming language, allowing any Python program to exploit multiple processors. Homepage: https://github.com/mpi4py/mpi4py URL: https://github.com/mpi4py/mpi4py</details>|
	|[mpsolve](https://numpi.dm.unipi.it/software/mpsolve)|-|3.2.1|<details><summary>Expand</summary>Description: MPSolve is a C package to solve polynomials and secular equations. It released under the terms of the GNU General public license as it specified in the COPYING file inside the source directory. Homepage: https://numpi.dm.unipi.it/software/mpsolve URL: https://numpi.dm.unipi.it/software/mpsolve</details>|
	|[mrbayes](https://nbisweden.github.io/MrBayes/)|bio|3.2.7|<details><summary>Expand</summary>Description: MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. Homepage: https://nbisweden.github.io/MrBayes/ URL: https://nbisweden.github.io/MrBayes/</details>|
	|[mrcc](https://www.mrcc.hu/index.php)|-|20230828|<details><summary>Expand</summary>Description: MRCC is a suite of ab initio and density functional quantum chemistry programs for high-accuracy electronic structure calculations developed and maintained by the quantum chemistry research group at the Department of Physical Chemistry and Materials Science, TU Budapest. Its special feature, the use of automated programming tools enabled the development of tensor manipulation routines which are independent of the number of indices of the corresponding tensors, thus significantly simplifying the general implementation of quantum chemical methods. Homepage: https://www.mrcc.hu/index.php URL: https://www.mrcc.hu/index.php</details>|
	|[mrtrix](http://www.brain.org.au/software/index.html#mrtrix)|bio|3.0.1, 3.0.4|<details><summary>Expand</summary>Description: MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines. Homepage: http://www.brain.org.au/software/index.html#mrtrix URL: http://www.brain.org.au/software/index.html#mrtrix</details>|
	|[mscg](https://github.com/uchicago-voth/MSCG-release)|chem|1.7.3.1|<details><summary>Expand</summary>Description: The Multiscale Coarse-Graining (MS-CG) method is a variational force-matching technique developed by the Voth group. This project serves as the workhorse implementation of the method used as a standard force-matching code. The development of MS-CG is supported by academic research grants. If you use this package, please cite at least one of these papers: - S. Izvekov and G. A. Voth, J. Chem. Phys. 123, 134105 (2005). - W. G. Noid, et al., J. Chem. Phys. 128, 244114 (2008). - W. G. Noid et al., J. Chem. Phys. 128, 244115 (2008). - L. Lu, S. Izvekov, A. Das, H. C. Andersen, and G. A. Voth, Journal of Chemical Theory and Computation, 6(3), 954-965 (2010). Homepage: https://github.com/uchicago-voth/MSCG-release URL: https://github.com/uchicago-voth/MSCG-release</details>|
	|[msmc2](https://github.com/stschiff/msmc)|-|2.1.3|<details><summary>Expand</summary>Description: This program implements MSMC2, a method to infer population size history and population separation history from whole genome sequencing data. For a general guide, please refer to version 1: https://github.com/stschiff/msmc Binaries are attached to the releases (under the "Releases" tab within github). Homepage: https://github.com/stschiff/msmc URL: https://github.com/stschiff/msmc</details>|
	|[mstore](https://github.com/grimme-lab/mstore)|-|0.2.0, 0.3.0|<details><summary>Expand</summary>Description: Molecular structure store for testing Homepage: https://github.com/grimme-lab/mstore URL: https://github.com/grimme-lab/mstore</details>|
	|[mujoco](https://mujoco.org/)|-|2.2.0, 2.2.2, 2.3.6, 3.0.1, 3.1.6, 3.3.0|<details><summary>Expand</summary>Description: MuJoCo stands for Multi-Joint dynamics with Contact. It is a general purpose physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, machine learning, and other areas which demand fast and accurate simulation of articulated structures interacting with their environment. Homepage: https://mujoco.org/ URL: https://mujoco.org/</details>|
	|[multicharge](https://github.com/grimme-lab/multicharge)|-|0.3.0|<details><summary>Expand</summary>Description: Electronegativity equilibration model for atomic partial charges. Homepage: https://github.com/grimme-lab/multicharge URL: https://github.com/grimme-lab/multicharge</details>|
	|[multichoose](https://github.com/ekg/multichoose)|-|1.0.3|<details><summary>Expand</summary>Description: generate multiset combinations (n multichoose k). Homepage: https://github.com/ekg/multichoose URL: https://github.com/ekg/multichoose</details>|
	|[mummer](https://mummer.sourceforge.net/)|bio|4.0.0beta2, 4.0.0rc1|<details><summary>Expand</summary>Description: MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it. Homepage: https://mummer.sourceforge.net/ URL: https://mummer.sourceforge.net/</details>|
	|[mumps-metis](https://mumps-solver.org/)|tools|5.2.1, 5.4.1, 5.7.3|<details><summary>Expand</summary>Description: A parallel sparse direct solver Homepage: https://mumps-solver.org/ URL: https://mumps-solver.org/</details>|
	|[mumps-parmetis](https://mumps-solver.org/)|tools|5.3.5, 5.4.1, 5.7.3|<details><summary>Expand</summary>Description: A parallel sparse direct solver Homepage: https://mumps-solver.org/ URL: https://mumps-solver.org/</details>|
	|[muparser](http://beltoforion.de/article.php?a=muparser)|math|2.2.5, 2.3.2, 2.3.4|<details><summary>Expand</summary>Description: muParser is an extensible high performance math expression parser library written in C++. It works by transforming a mathematical expression into bytecode and precalculating constant parts of the expression. Homepage: http://beltoforion.de/article.php?a=muparser URL: http://beltoforion.de/article.php?a=muparser</details>|
	|[muparserx](https://www.google.ca/search?q=muparserx)|-|4.0.12|<details><summary>Expand</summary>Description: A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more. Homepage: beltoforion.de/en/muparserx URL: beltoforion.de/en/muparserx</details>|
	|[muscle](https://drive5.com/muscle/)|bio|3.8.1551|<details><summary>Expand</summary>Description: MUSCLE is one of the best-performing multiple alignment programs according to published benchmark tests, with accuracy and speed that are consistently better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users learn everything they need to know about MUSCLE in a few minutes-only a handful of command-line options are needed to perform common alignment tasks. Homepage: https://drive5.com/muscle/ URL: https://drive5.com/muscle/</details>|
	|[mustang](http://www.csse.monash.edu.au/~karun/Site/mustang.html)|-|3.2.4|<details><summary>Expand</summary>Description: MUSTANG (MUltiple STructural AligNment AlGorithm), for the alignment of multiple protein structures. Given a set of protein structures, the program constructs a multiple alignment using the spatial information of the C atoms in the set. Homepage: http://www.csse.monash.edu.au/~karun/Site/mustang.html URL: http://www.csse.monash.edu.au/~karun/Site/mustang.html</details>|
	|[mysql](http://www.mysql.com/)|tools|5.7.36, 8.3.0|<details><summary>Expand</summary>Description: MySQL is one of the world's most widely used open-source relational database management system (RDBMS). Homepage: http://www.mysql.com/ URL: http://www.mysql.com/</details>|
	|[n2p2](https://compphysvienna.github.io/n2p2/)|-|2.2.0|<details><summary>Expand</summary>Description: A neural network potential package: high-dimensional neural network potentials in computational physics and chemistry. Homepage: https://compphysvienna.github.io/n2p2/ URL: https://compphysvienna.github.io/n2p2/</details>|
	|[namd-multicore](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14, 3.0.1|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ofi](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ofi-smp](http://www.ks.uiuc.edu/Research/namd/)|-|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ucx](http://www.ks.uiuc.edu/Research/namd/)|chem|2.14, 3.0.1|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[namd-ucx-smp](http://www.ks.uiuc.edu/Research/namd/)|-|2.14|<details><summary>Expand</summary>Description: NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems. Homepage: http://www.ks.uiuc.edu/Research/namd/ URL: http://www.ks.uiuc.edu/Research/namd/</details>|
	|[nanoflann](https://github.com/jlblancoc/nanoflann)|-|1.3.2|<details><summary>Expand</summary>Description: nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies Homepage: https://github.com/jlblancoc/nanoflann URL: https://github.com/jlblancoc/nanoflann</details>|
	|[nanopolish](https://github.com/jts/nanopolish)|bio|0.11.2, 0.13.2|<details><summary>Expand</summary>Description: Software package for signal-level analysis of Oxford Nanopore sequencing data. Nanopolish can calculate an improved consensus sequence for a draft genome assembly, detect base modifications, call SNPs and indels with respect to a reference genome and more. Homepage: https://github.com/jts/nanopolish URL: https://github.com/jts/nanopolish</details>|
	|[nauty](http://pallini.di.uniroma1.it/)|-|2.6r12|<details><summary>Expand</summary>Description: nauty and Traces are programs for computing automorphism groups of graphs and digraphs. They can also produce a canonical label. Homepage: http://pallini.di.uniroma1.it/ URL: http://pallini.di.uniroma1.it/</details>|
	|[nccl](https://developer.nvidia.com/nccl)|tools|2.7.8, 2.8.4, 2.11.4, 2.12.12, 2.18.3, 2.18.5, 2.22.3, 2.26.2|<details><summary>Expand</summary>Description: The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs. Homepage: https://developer.nvidia.com/nccl URL: https://developer.nvidia.com/nccl</details>|
	|[nciplot](https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html)|-|4.0-20200624|<details><summary>Expand</summary>Description: NCIPLOT is a program for revealing non covalent interactions based on the reduced density gradient. Homepage: https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html URL: https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html</details>|
	|[ncl](https://www.ncl.ucar.edu)|vis|6.6.2|<details><summary>Expand</summary>Description: NCL is an interpreted language designed specifically for scientific data analysis and visualization. Homepage: https://www.ncl.ucar.edu URL: https://www.ncl.ucar.edu</details>|
	|[nco](https://nco.sourceforge.net)|io|4.9.5, 5.0.6, 5.1.7|<details><summary>Expand</summary>Description: manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5 Homepage: https://nco.sourceforge.net URL: https://nco.sourceforge.net</details>|
	|[ncview](http://meteora.ucsd.edu/~pierce/ncview_home_page.html)|vis|2.1.8|<details><summary>Expand</summary>Description: Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc. Homepage: http://meteora.ucsd.edu/~pierce/ncview_home_page.html URL: http://meteora.ucsd.edu/~pierce/ncview_home_page.html</details>|
	|[nektar++](https://www.nektar.info/)|math|5.0.1, 5.3.0, 5.7.0|<details><summary>Expand</summary>Description: Nektar++ is a tensor product based finite element package designed to allow one to construct efficient classical low polynomial order h-type solvers (where h is the size of the finite element) as well as higher p-order piecewise polynomial order solvers. Homepage: https://www.nektar.info/ URL: https://www.nektar.info/</details>|
	|[netcdf](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0, 4.9.0, 4.9.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++-mpi](http://www.unidata.ucar.edu/software/netcdf/)|io|4.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: http://www.unidata.ucar.edu/software/netcdf/ URL: http://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++4](https://www.unidata.ucar.edu/software/netcdf/)|io|4.3.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-c++4-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.3.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3, 4.6.0, 4.6.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3, 4.6.0, 4.6.1|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0, 4.8.1, 4.9.0, 4.9.2|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[neuron](http://www.neuron.yale.edu/neuron)|bio|7.8.2, 8.0.0, 8.2.3|<details><summary>Expand</summary>Description: Empirically-based simulations of neurons and networks of neurons. Homepage: http://www.neuron.yale.edu/neuron URL: http://www.neuron.yale.edu/neuron</details>|
	|[newmat](http://www.robertnz.net/index.html)|-|11|<details><summary>Expand</summary>Description: Newmat C++ library is intended for scientists and engineers who need to manipulate a variety of types of matrices using standard matrix operations. Emphasis is on the kind of operations needed in statistical calculations such as least squares, linear equation solve and eigenvalues. Homepage: http://www.robertnz.net/index.html URL: http://www.robertnz.net/index.html</details>|
	|[nextdenovo](https://github.com/Nextomics/NextDenovo)|-|2.5.2|<details><summary>Expand</summary>Description: NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio HiFi reads), but requires significantly less computing resources and storages. After assembly, the per-base accuracy is about 98-99.8%, to further improve single base accuracy, try NextPolish. Homepage: https://github.com/Nextomics/NextDenovo URL: https://github.com/Nextomics/NextDenovo</details>|
	|[nextflow](https://www.nextflow.io/)|-|20.10.0, 21.04.3, 21.10.3, 22.04.3, 22.10.6, 22.10.8, 23.04.3, 23.10.0, 24.04.4|<details><summary>Expand</summary>Description: Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data Homepage: https://www.nextflow.io/ URL: https://www.nextflow.io/</details>|
	|[nextgenmap](http://cibiv.github.io/NextGenMap/)|bio|0.5.5|<details><summary>Expand</summary>Description: NextGenMap is a flexible highly sensitive short read mapping tool that handles much higher mismatch rates than comparable algorithms while still outperforming them in terms of runtime. Homepage: http://cibiv.github.io/NextGenMap/ URL: http://cibiv.github.io/NextGenMap/</details>|
	|[nextstrain.cli](https://github.com/nextstrain/cli/tree/3.1.0)|-|3.1.0|<details><summary>Expand</summary>Description: This is the source code repository for a program called nextstrain, the Nextstrain command-line interface (CLI). It aims to provide a consistent way to run and visualize pathogen builds and access Nextstrain components like Augur and Auspice across computing environments such as Docker, Conda, and AWS Batch. Homepage: https://github.com/nextstrain/cli/tree/3.1.0 URL: https://github.com/nextstrain/cli/tree/3.1.0</details>|
	|[ngmlr](https://github.com/philres/ngmlr)|-|0.2.7|<details><summary>Expand</summary>Description: Ngmlr is a long-read mapper designed to align PacBilo or Oxford Nanopore to a reference genome with a focus on reads that span structural variations. Homepage: https://github.com/philres/ngmlr URL: https://github.com/philres/ngmlr</details>|
	|[ngs](https://github.com/ncbi/ngs)|-|2.10.5|<details><summary>Expand</summary>Description: NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from Next Generation Sequencing. Homepage: https://github.com/ncbi/ngs URL: https://github.com/ncbi/ngs</details>|
	|[ngt](https://github.com/yahoojapan/NGT)|-|2.0.13|<details><summary>Expand</summary>Description: NGT provides commands and a library for performing high-speed approximate nearest neighbor searches against a large volume of data in high dimensional vector data space (several ten to several thousand dimensions). Homepage: https://github.com/yahoojapan/NGT URL: https://github.com/yahoojapan/NGT</details>|
	|[nibabel](https://nipy.github.io/nibabel)|-|5.2.0|<details><summary>Expand</summary>Description: NiBabel provides read/write access to some common medical and neuroimaging file formats, including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI. Homepage: https://nipy.github.io/nibabel URL: https://nipy.github.io/nibabel</details>|
	|[ninja-phylogenetics](https://github.com/TravisWheelerLab/NINJA)|-|0.97-cluster_only|<details><summary>Expand</summary>Description: Nearly Infinite Neighbor Joining Application Compute correct neighbor-joining phylogenies for inputs of more than 10,000 sequences. This is a C++/SSE port of the original Java code described in: Wheeler, T.J. 2009. Large-scale neighbor-joining with NINJA. In S.L. Salzberg and T. Warnow (Eds.), Proceedings of the 9th Workshop on Algorithms in Bioinformatics. WABI 2009, pp. 375-389. Springer, Berlin. (LNCS webpage,preprint) The Java version of NINJA was the fastest available tool computing neighbor-joining phylogenies ( 10x faster than the fastest implemenation of the canonical neighbor-joining algorithm - QuickTree ) at the time of it's release. In addition to generating phylogenies, Ninja can be used to output pairwise distances using several common sequence distance measures, and cluster sequences using a nearest-neighbor approach. Homepage: https://github.com/TravisWheelerLab/NINJA URL: https://github.com/TravisWheelerLab/NINJA</details>|
	|[nlopt](https://nlopt.readthedocs.io/en/latest/)|math|2.6.1, 2.6.2, 2.7.0, 2.7.1, 2.9.1|<details><summary>Expand</summary>Description: NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms. Homepage: https://nlopt.readthedocs.io/en/latest/ URL: https://nlopt.readthedocs.io/en/latest/</details>|
	|[nodejs](https://nodejs.org)|tools|12.16.1, 15.2.1, 16.14.0, 18.17.1, 20.16.0|<details><summary>Expand</summary>Description: Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices. Homepage: https://nodejs.org URL: https://nodejs.org</details>|
	|[nspr](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)|-|4.25|<details><summary>Expand</summary>Description: Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</details>|
	|[nss](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)|-|3.51|<details><summary>Expand</summary>Description: Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</details>|
	|[ntedit](https://github.com/bcgsc/ntEdit)|-|1.3.2|<details><summary>Expand</summary>Description: ntEdit is a fast and scalable genomics application for polishing genome assembly drafts. Homepage: https://github.com/bcgsc/ntEdit URL: https://github.com/bcgsc/ntEdit</details>|
	|[ntl](http://www.shoup.net/ntl/)|math|11.4.3|<details><summary>Expand</summary>Description: NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields. Homepage: http://www.shoup.net/ntl/ URL: http://www.shoup.net/ntl/</details>|
	|[nvbio](https://github.com/NVlabs/nvbio)|-|1.1.00.20200205|<details><summary>Expand</summary>Description: NVBIO is a library of reusable components designed by NVIDIA Corporation to accelerate bioinformatics applications using CUDA. Homepage: https://github.com/NVlabs/nvbio URL: https://github.com/NVlabs/nvbio</details>|
	|[nvhpc](https://developer.nvidia.com/hpc-sdk/)|tools|20.7, 22.1, 22.7, 23.7, 23.9, 25.1|<details><summary>Expand</summary>Description: C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI) Homepage: https://developer.nvidia.com/hpc-sdk/ URL: https://developer.nvidia.com/hpc-sdk/</details>|
	|[nwchem](http://www.nwchem-sw.org)|chem|7.0.2-p1, 6.8.1, 7.0.0, 7.0.2|<details><summary>Expand</summary>Description: NWChem aims to provide its users with computational chemistry tools that are scalable both in their ability to treat large scientific computational chemistry problems efficiently, and in their use of available parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters. NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties and relativity. Homepage: http://www.nwchem-sw.org URL: http://www.nwchem-sw.org</details>|
	|[oases](https://github.com/dzerbino/oases)|-|0.2.09|<details><summary>Expand</summary>Description: Oases is a de novo transcriptome assembler designed to produce transcripts from short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly. Homepage: https://github.com/dzerbino/oases URL: https://github.com/dzerbino/oases</details>|
	|[occt](https://www.opencascade.com/)|-|7.4.0, 7.7.1, 7.8.1|<details><summary>Expand</summary>Description: Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications. Homepage: https://www.opencascade.com/ URL: https://www.opencascade.com/</details>|
	|[octave](https://www.gnu.org/software/octave/)|tools|5.2.0, 7.2.0|<details><summary>Expand</summary>Description: GNU Octave is a high-level interpreted language, primarily intended for numerical computations. Homepage: https://www.gnu.org/software/octave/ URL: https://www.gnu.org/software/octave/</details>|
	|[octomap](https://octomap.github.io/)|-|1.10.0|<details><summary>Expand</summary>Description: The OctoMap library implements a 3D occupancy grid mapping approach, providing data structures and mapping algorithms in C++ particularly suited for robotics. Homepage: https://octomap.github.io/ URL: https://octomap.github.io/</details>|
	|[octopus](http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page)|chem|10.1, 10.3|<details><summary>Expand</summary>Description: Octopus is a scientific program aimed at the ab initio virtual experimentation on a hopefully ever-increasing range of system types. Electrons are described quantum-mechanically within density-functional theory (DFT), in its time-dependent form (TDDFT) when doing simulations in time. Nuclei are described classically as point particles. Electron-nucleus interaction is described within the pseudopotential approximation. Homepage: http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page URL: http://www.tddft.org/programs/octopus/wiki/index.php/Main_Page</details>|
	|[octopus-bio](https://luntergroup.github.io/octopus/)|-|0.7.4|<details><summary>Expand</summary>Description: Octopus is a mapping-based variant caller that implements several calling models within a unified haplotype-aware framework. Octopus takes inspiration from particle filtering by constructing a tree of haplotypes and dynamically pruning and extending the tree based on haplotype posterior probabilities in a sequential manner. This allows octopus to implicitly consider all possible haplotypes at a given loci in reasonable time. Homepage: https://luntergroup.github.io/octopus/ URL: https://luntergroup.github.io/octopus/</details>|
	|[ogre](https://ogrecave.github.io/ogre/)|-|1.10.12, 1.12.12|<details><summary>Expand</summary>Description: OGRE (Object-Oriented Graphics Rendering Engine) is a scene-oriented, flexible 3D engine written in C++ designed to make it easier and more intuitive for developers to produce games and demos utilising 3D hardware. The class library abstracts all the details of using the underlying system libraries like Direct3D and OpenGL and provides an interface based on world objects and other intuitive classes. Homepage: https://ogrecave.github.io/ogre/ URL: https://ogrecave.github.io/ogre/</details>|
	|[oligoarrayaux](http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux)|-|3.8|<details><summary>Expand</summary>Description: OligoArrayAux is a subset of the UNAFold package for use with OligoArray. Homepage: http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux URL: http://unafold.rna.albany.edu/?q=DINAMelt/OligoArrayAux</details>|
	|[one-dnn](https://github.com/oneapi-src/oneDNN)|-|1.8, 2.4.3|<details><summary>Expand</summary>Description: Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) ONE-DNN) Homepage: https://github.com/oneapi-src/oneDNN URL: https://github.com/oneapi-src/oneDNN</details>|
	|[opa-psm2](https://github.com/cornelisnetworks/opa-psm2/)|-|11.2.185, 11.2.206|<details><summary>Expand</summary>Description: Low-level user-space communications interface for the Intel(R) OPA family of products. Homepage: https://github.com/cornelisnetworks/opa-psm2/ URL: https://github.com/cornelisnetworks/opa-psm2/</details>|
	|[opari2](https://www.score-p.org)|-|2.0.5|<details><summary>Expand</summary>Description: OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a source-to-source instrumentation tool for OpenMP and hybrid codes. It surrounds OpenMP directives and runtime library calls with calls to the POMP2 measurement interface. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[openbabel](https://openbabel.org)|chem|3.1.1|<details><summary>Expand</summary>Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas. Homepage: https://openbabel.org URL: https://openbabel.org</details>|
	|[openbabel-omp](https://openbabel.org)|chem|3.1.1|<details><summary>Expand</summary>Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas. Homepage: https://openbabel.org URL: https://openbabel.org</details>|
	|[openblas](https://www.openblas.net/)|math|0.3.9, 0.3.17, 0.3.18, 0.3.20, 0.3.24, 0.3.27|<details><summary>Expand</summary>Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version. Homepage: https://www.openblas.net/ URL: https://www.openblas.net/</details>|
	|[opencarp](https://opencarp.org)|-|4.0|<details><summary>Expand</summary>Description: openCARP is an open cardiac electrophysiology simulator for in-silico experiments. Homepage: https://opencarp.org URL: https://opencarp.org</details>|
	|[opencascade](https://www.opencascade.com/open-cascade-technology/)|-|7.5.0, 7.5.2|<details><summary>Expand</summary>Description: Open CASCADE Technology (OCCT) is a world-known 3D modeling kernel, freely available in open source. It's a software development kit (SDK) intended for the development of applications dealing with 3D CAD data. OCCT includes a set of C++ class libraries providing services for 3D surface and solid modeling, visualization, data exchange and rapid application development. Homepage: https://www.opencascade.com/open-cascade-technology/ URL: https://www.opencascade.com/open-cascade-technology/</details>|
	|[opencoarrays](https://github.com/sourceryinstitute/opencoarrays)|-|2.9.2|<details><summary>Expand</summary>Description: OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran. Homepage: https://github.com/sourceryinstitute/opencoarrays URL: https://github.com/sourceryinstitute/opencoarrays</details>|
	|[opencolorio](http://opencolorio.org/)|vis|1.1.1, 2.2.1, 2.3.0|<details><summary>Expand</summary>Description: OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production with an emphasis on visual effects and computer animation. Homepage: http://opencolorio.org/ URL: http://opencolorio.org/</details>|
	|[opencv](http://opencv.org/)|vis|4.4.0, 4.5.1, 4.5.5, 4.6.0, 4.7.0, 4.8.0, 4.8.1, 4.9.0, 4.10.0, 4.10.0-2, 4.11.0|<details><summary>Expand</summary>Description: OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. OpenCV was built to provide a common infrastructure for computer vision applications and to accelerate the use of machine perception in the commercial products. Homepage: http://opencv.org/ URL: http://opencv.org/</details>|
	|[openexr](https://www.openexr.com/)|vis|2.5.2, 3.1.5|<details><summary>Expand</summary>Description: OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light and Magic for use in computer imaging applications Homepage: https://www.openexr.com/ URL: https://www.openexr.com/</details>|
	|[openfast](https://openfast.readthedocs.io/en/main/index.html)|-|3.1.0, 3.3.0, 3.5.0|<details><summary>Expand</summary>Description: OpenFAST is a multi-physics, multi-fidelity tool for simulating the coupled dynamic response of wind turbines. Practically speaking, OpenFAST is the framework (or glue code) that couples computational modules for aerodynamics, hydrodynamics for offshore structures, control and electrical system (servo) dynamics, and structural dynamics to enable coupled nonlinear aero-hydro-servo-elastic simulation in the time domain. OpenFAST enables the analysis of a range of wind turbine configurations, including two- or three-blade horizontal-axis rotor, pitch or stall regulation, rigid or teetering hub, upwind or downwind rotor, and lattice or tubular tower. The wind turbine can be modeled on land or offshore on fixed-bottom or floating substructures. Homepage: https://openfast.readthedocs.io/en/main/index.html URL: https://openfast.readthedocs.io/en/main/index.html</details>|
	|[openfoam](https://www.openfoam.com/)|phys|8, 9, 10, 11, v2006, v2012, v2112, v2206, v2212, v2306, v2312, v2406|<details><summary>Expand</summary>Description: OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. Homepage: https://www.openfoam.com/ URL: https://www.openfoam.com/</details>|
	|[openfoam-extend](http://www.extend-project.de/)|phys|4.1|<details><summary>Expand</summary>Description: OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. Homepage: http://www.extend-project.de/ URL: http://www.extend-project.de/</details>|
	|[openimagedenoise](https://github.com/OpenImageDenoise/oidn)|-|1.2.2, 1.4.3, 2.0.1, 2.1.0|<details><summary>Expand</summary>Description: Intel Open Image Denoise is an open source library of high-performance, high-quality denoising filters for images rendered with ray tracing. Homepage: https://github.com/OpenImageDenoise/oidn URL: https://github.com/OpenImageDenoise/oidn</details>|
	|[openimageio](https://openimageio.org/)|vis|2.1.17.0, 2.4.7.1, 2.4.14.0|<details><summary>Expand</summary>Description: OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities, and applications. Homepage: https://openimageio.org/ URL: https://openimageio.org/</details>|
	|[openmc](https://docs.openmc.org/en/stable/)|-|0.13.2, 0.13.3, 0.15.0|<details><summary>Expand</summary>Description: OpenMC is a community-developed Monte Carlo neutron and photon transport simulation code. It is capable of performing fixed source, k-eigenvalue, and subcritical multiplication calculations on models built using either a constructive solid geometry or CAD representation. OpenMC supports both continuous-energy and multigroup transport. The continuous-energy particle interaction data is based on a native HDF5 format that can be generated from ACE files produced by NJOY. Parallelism is enabled via a hybrid MPI and OpenMP programming model. Homepage: https://docs.openmc.org/en/stable/ URL: https://docs.openmc.org/en/stable/</details>|
	|[openmm](http://openmm.org)|chem|7.5.0, 7.6.0, 7.7.0, 8.0.0, 8.1.1, 8.2.0|<details><summary>Expand</summary>Description: OpenMM is a toolkit for molecular simulation. Homepage: http://openmm.org URL: http://openmm.org</details>|
	|[openmm-alphafold](http://openmm.org)|-|7.5.1|<details><summary>Expand</summary>Description: OpenMM is a toolkit for molecular simulation. Homepage: http://openmm.org URL: http://openmm.org</details>|
	|[openmm-plumed](https://github.com/openmm/openmm-plumed/)|-|1.0|<details><summary>Expand</summary>Description: This project provides a connection between OpenMM and PLUMED. It allows you to bias or analyze an OpenMM simulation based on collective variables. Homepage: https://github.com/openmm/openmm-plumed/ URL: https://github.com/openmm/openmm-plumed/</details>|
	|[openmolcas](https://gitlab.com/Molcas/OpenMolcas)|chem|20.10|<details><summary>Expand</summary>Description: OpenMolcas is a quantum chemistry software package Homepage: https://gitlab.com/Molcas/OpenMolcas URL: https://gitlab.com/Molcas/OpenMolcas</details>|
	|[openmpi](https://www.open-mpi.org/)|mpi|4.0.3, 4.0.5, 4.1.1, 4.1.4, 4.1.5, 5.0.3|<details><summary>Expand</summary>Description: The Open MPI Project is an open source MPI-3 implementation. Homepage: https://www.open-mpi.org/ URL: https://www.open-mpi.org/</details>|
	|[openmx](http://www.openmx-square.org/)|chem|3.9, 3.9.9|<details><summary>Expand</summary>Description: OpenMX (Open source package for Material eXplorer) is a software package for nano-scale material simulations based on density functional theories (DFT), norm-conserving pseudopotentials, and pseudo-atomic localized basis functions. The methods and algorithms used in OpenMX and their implementation are carefully designed for the realization of large-scale ab initio electronic structure calculations on parallel computers based on the MPI or MPI/OpenMP hybrid parallelism. Homepage: http://www.openmx-square.org/ URL: http://www.openmx-square.org/</details>|
	|[openpgm](https://code.google.com/p/openpgm/)|tools|5.2.122|<details><summary>Expand</summary>Description: OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM) specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol that enables receivers to detect loss, request retransmission of lost data, or notify an application of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible for ensuring all data is received, absolving the sender of reception responsibility. Homepage: https://code.google.com/p/openpgm/ URL: https://code.google.com/p/openpgm/</details>|
	|[openrefine](https://github.com/OpenRefine/OpenRefine)|-|3.4.1|<details><summary>Expand</summary>Description: OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it, and augment it with data coming from the web. Homepage: https://github.com/OpenRefine/OpenRefine URL: https://github.com/OpenRefine/OpenRefine</details>|
	|[opensees](http://opensees.berkeley.edu/)|-|3.2.0, 3.4.0, 3.5.0|<details><summary>Expand</summary>Description: OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes. This version is built using DISTRIBUTED_MP platform setting. Homepage: http://opensees.berkeley.edu/ URL: http://opensees.berkeley.edu/</details>|
	|[opensim](https://simtk.org/projects/opensim)|-|4.5.1|<details><summary>Expand</summary>Description: OpenSim is a freely available, user extensible software system that lets users develop models of musculoskeletal structures and create dynamic simulations of movement. Homepage: https://simtk.org/projects/opensim URL: https://simtk.org/projects/opensim</details>|
	|[openslide](https://github.com/openslide/openslide)|vis|3.4.1, 4.0.0|<details><summary>Expand</summary>Description: C library for reading virtual slide images Homepage: https://github.com/openslide/openslide URL: https://github.com/openslide/openslide</details>|
	|[opensubdiv](https://github.com/PixarAnimationStudios/OpenSubdiv)|-|3.4.4|<details><summary>Expand</summary>Description: OpenSubdiv is a set of open source libraries that implement high performance subdivision surface (subdiv) evaluation on massively parallel CPU and GPU architectures. Homepage: https://github.com/PixarAnimationStudios/OpenSubdiv URL: https://github.com/PixarAnimationStudios/OpenSubdiv</details>|
	|[openvdb](https://github.com/AcademySoftwareFoundation/openvdb)|-|7.0.0, 10.0.1, 10.1.0|<details><summary>Expand</summary>Description: OpenVDB is an open source C++ library comprising a novel hierarchical data structure and a large suite of tools for the efficient storage and manipulation of sparse volumetric data discretized on three-dimensional grids. It was developed by DreamWorks Animation for use in volumetric applications typically encountered in feature film production. Homepage: https://github.com/AcademySoftwareFoundation/openvdb URL: https://github.com/AcademySoftwareFoundation/openvdb</details>|
	|[openvkl](https://github.com/OpenVKL/openvkl)|-|0.10.0, 1.3.1, 1.3.2|<details><summary>Expand</summary>Description: Intel Open Volume Kernel Library (Intel Open VKL) is a collection of high-performance volume computation kernels, developed at Intel. The target users of Open VKL are graphics application engineers who want to improve the performance of their volume rendering applications by leveraging Open VKL's performance-optimized kernels, which include volume traversal and sampling functionality for a variety of volumetric data formats. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Homepage: https://github.com/OpenVKL/openvkl URL: https://github.com/OpenVKL/openvkl</details>|
	|[opera-ms](https://github.com/CSB5/OPERA-MS)|-|0.9.0-20200802|<details><summary>Expand</summary>Description: OPERA-MS is a hybrid metagenomic assembler which combines the advantages of short and long-read technologies to provide high quality assemblies, addressing issues of low contiguity for short-read only assemblies, and low base-pair quality for long-read only assemblies. Homepage: https://github.com/CSB5/OPERA-MS URL: https://github.com/CSB5/OPERA-MS</details>|
	|[optix](https://developer.nvidia.com/optix)|-|6.5.0, 7.7.0, 8.0.0|<details><summary>Expand</summary>Description: OptiX is NVIDIA SDK for easy ray tracing performance. It provides a simple framework for accessing the GPU's massive ray tracing power using state-of-the-art GPU algorithms. Homepage: https://developer.nvidia.com/optix URL: https://developer.nvidia.com/optix</details>|
	|[orad](https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html)|-|2.6.1|<details><summary>Expand</summary>Description: DRAGEN ORA Decompression software decompresses *.fastq.ora files that have been compressed using DRAGEN ORA Compression software with the default human reference and with the additional supported references. Homepage: https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html URL: https://support.illumina.com/downloads/dragen_decompression_software_v261_software-linux.html</details>|
	|[orca](http://cec.mpg.de/forum/)|chem|4.2.1, 5.0.1, 5.0.2, 5.0.3, 5.0.4, 6.0.0, 6.0.1|<details><summary>Expand</summary>Description: ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects. Homepage: http://cec.mpg.de/forum/ URL: http://cec.mpg.de/forum/</details>|
	|[orthomcl](http://orthomcl.org/)|bio|2.0.9|<details><summary>Expand</summary>Description: OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences. Homepage: http://orthomcl.org/ URL: http://orthomcl.org/</details>|
	|[ospray](https://github.com/ospray/ospray)|-|1.8.5, 2.2.0, 2.10.0, 2.12.0|<details><summary>Expand</summary>Description: Intel OSPRay is an open source, scalable, and portable ray tracing engine for high-performance, high-fidelity visualization on Intel Architecture CPUs. Homepage: https://github.com/ospray/ospray URL: https://github.com/ospray/ospray</details>|
	|[osrm-backend](https://github.com/Project-OSRM/osrm-backend)|-|5.26.0|<details><summary>Expand</summary>Description: Open Source Routing Machine Homepage: https://github.com/Project-OSRM/osrm-backend URL: https://github.com/Project-OSRM/osrm-backend</details>|
	|[osu-micro-benchmarks](https://mvapich.cse.ohio-state.edu/benchmarks/)|tools|5.6.2, 5.6.3, 5.9, 7.1-1, 7.2, 7.4|<details><summary>Expand</summary>Description: OSU Micro-Benchmarks Homepage: https://mvapich.cse.ohio-state.edu/benchmarks/ URL: https://mvapich.cse.ohio-state.edu/benchmarks/</details>|
	|[otf2](https://www.score-p.org)|-|2.2|<details><summary>Expand</summary>Description: The Open Trace Format 2 is a highly scalable, memory efficient event trace data format plus support library. It is the new standard trace format for Scalasca, Vampir, and TAU and is open for other tools. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[ovito](https://www.ovito.org)|-|3.3.3|<details><summary>Expand</summary>Description: OVITO is a scientific visualization and analysis software for atomistic and particle simulation data. It helps scientists gain better insights into materials phenomena and physical processes. The program is freely available for all major platforms under an open source license. It has served in a growing number of computational simulation studies as a powerful tool to analyze, understand and illustrate simulation results. Homepage: https://www.ovito.org URL: https://www.ovito.org</details>|
	|[p4est](http://www.p4est.org/)|math|2.2, 2.3.2, 2.8.5, 2.8.6|<details><summary>Expand</summary>Description: The p4est software library enables the dynamic management of a collection of adaptive octrees, conveniently called a forest of octrees. p4est is designed to work in parallel and scales to hundreds of thousands of processor cores. It is free software released under GNU General Public Licence version 2, or (at your option) any later version. Homepage: http://www.p4est.org/ URL: http://www.p4est.org/</details>|
	|[packmol](http://m3g.iqm.unicamp.br/packmol)|chem|20.3.3|<details><summary>Expand</summary>Description: Packing Optimization for Molecular Dynamics Simulations Homepage: http://m3g.iqm.unicamp.br/packmol URL: http://m3g.iqm.unicamp.br/packmol</details>|
	|[pagmo](https://esa.github.io/pagmo2)|-|2.18.0|<details><summary>Expand</summary>Description: pagmo is a C++ scientific library for massively parallel optimization. Homepage: https://esa.github.io/pagmo2 URL: https://esa.github.io/pagmo2</details>|
	|[pairix](https://github.com/4dn-dcic/pairix)|-|0.3.7|<details><summary>Expand</summary>Description: 1D/2D indexing and querying on bgzipped text file with a pair of genomic coordinates. Homepage: https://github.com/4dn-dcic/pairix URL: https://github.com/4dn-dcic/pairix</details>|
	|[paml](http://abacus.gene.ucl.ac.uk/software/paml.html)|bio|4.9j, 4.10.7|<details><summary>Expand</summary>Description: PAML is a package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood. Homepage: http://abacus.gene.ucl.ac.uk/software/paml.html URL: http://abacus.gene.ucl.ac.uk/software/paml.html</details>|
	|[panther](https://www.google.ca/search?q=panther)|-|14.1|<details><summary>Expand</summary>Description: Panther is the database part of InterProScan software. It contains only data. Homepage: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/ URL: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/</details>|
	|[papi](https://icl.cs.utk.edu/projects/papi/)|tools|6.0.0|<details><summary>Expand</summary>Description: PAPI provides the tool designer and application engineer with a consistent interface and methodology for use of the performance counter hardware found in most major microprocessors. PAPI enables software engineers to see, in near real time, the relation between software performance and processor events. In addition Component PAPI provides access to a collection of components that expose performance measurement opportunites across the hardware and software stack. Homepage: https://icl.cs.utk.edu/projects/papi/ URL: https://icl.cs.utk.edu/projects/papi/</details>|
	|[parallelio](https://github.com/NCAR/ParallelIO)|-|2.5.4, 2.6.3|<details><summary>Expand</summary>Description: A high-level Parallel I/O Library for structured grid applications Homepage: https://github.com/NCAR/ParallelIO URL: https://github.com/NCAR/ParallelIO</details>|
	|[parasail](https://github.com/jeffdaily/parasail)|-|2.5, 2.6.2|<details><summary>Expand</summary>Description: parasail is a SIMD C (C99) library containing implementations of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global pairwise sequence alignment algorithms. Homepage: https://github.com/jeffdaily/parasail URL: https://github.com/jeffdaily/parasail</details>|
	|[paraview](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0, 5.11.2, 5.13.1|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[paraview-offscreen](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[paraview-offscreen-gpu](https://www.paraview.org)|vis|5.8.0, 5.9.1, 5.10.0, 5.11.0|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[pari-gp](http://pari.math.u-bordeaux.fr)|-|2.13.0|<details><summary>Expand</summary>Description: PARI/GP is a widely used computer algebra system designed for fast computations in number theory (factorizations, algebraic number theory, elliptic curves...), but also contains a large number of other useful functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc., and a lot of transcendental functions. PARI is also available as a C library to allow for faster computations. Homepage: http://pari.math.u-bordeaux.fr URL: http://pari.math.u-bordeaux.fr</details>|
	|[parmetis](http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview)|math|4.0.3|<details><summary>Expand</summary>Description: ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview</details>|
	|[parmgridgen](http://www-users.cs.umn.edu/~moulitsa/software.html)|math|1.0|<details><summary>Expand</summary>Description: ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen, that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited for geometric multigrid methods. Homepage: http://www-users.cs.umn.edu/~moulitsa/software.html URL: http://www-users.cs.umn.edu/~moulitsa/software.html</details>|
	|[parsplice](https://gitlab.com/exaalt/parsplice/)|-|1.1|<details><summary>Expand</summary>Description: Parallel Trajectory Splicing AMD Code Homepage: https://gitlab.com/exaalt/parsplice/ URL: https://gitlab.com/exaalt/parsplice/</details>|
	|[pbbam](http://pbbam.readthedocs.io/)|bio|1.0.7, 1.7.0|<details><summary>Expand</summary>Description: The pbbam software package provides components to create, query, and edit PacBio BAM files and associated indices. Homepage: http://pbbam.readthedocs.io/ URL: http://pbbam.readthedocs.io/</details>|
	|[pbcopper](https://github.com/PacificBiosciences/pbcopper)|-|1.4.0, 1.9.0|<details><summary>Expand</summary>Description: The pbcopper library provides a suite of data structures, algorithms, and utilities for C++ applications. Homepage: https://github.com/PacificBiosciences/pbcopper URL: https://github.com/PacificBiosciences/pbcopper</details>|
	|[pbmm2](https://github.com/PacificBiosciences/pbmm2)|-|1.7.0|<details><summary>Expand</summary>Description: A minimap2 frontend for PacBio native data formats Homepage: https://github.com/PacificBiosciences/pbmm2 URL: https://github.com/PacificBiosciences/pbmm2</details>|
	|[pbsuite](https://sourceforge.net/p/pb-jelly/wiki/Home/)|bio|15.8.24|<details><summary>Expand</summary>Description: PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or long 454 reads in fasta format) to high-confidence draft assembles. Homepage: https://sourceforge.net/p/pb-jelly/wiki/Home/ URL: https://sourceforge.net/p/pb-jelly/wiki/Home/</details>|
	|[pcre](https://www.pcre.org/)|-|8.44|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[pcre2](https://www.pcre.org/)|-|10.34|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[pdal](https://pdal.io/)|-|2.7.1|<details><summary>Expand</summary>Description: PDAL is Point Data Abstraction Library. It is a C/C++ open source library and applications for translating and processing point cloud data. It is not limited to LiDAR data, although the focus and impetus for many of the tools in the library have their origins in LiDAR. Homepage: https://pdal.io/ URL: https://pdal.io/</details>|
	|[pdt](https://www.cs.uoregon.edu/research/pdt/)|-|3.25.1|<details><summary>Expand</summary>Description: Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program knowledge accessible to developers of static and dynamic analysis tools. PDT implements a standard program representation, the program database (PDB), that can be accessed in a uniform way through a class library supporting common PDB operations. Homepage: https://www.cs.uoregon.edu/research/pdt/ URL: https://www.cs.uoregon.edu/research/pdt/</details>|
	|[pear](https://cme.h-its.org/exelixis/web/software/pear/)|bio|0.9.11|<details><summary>Expand</summary>Description: PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. It is fully parallelized and can run with as low as just a few kilobytes of memory. Homepage: https://cme.h-its.org/exelixis/web/software/pear/ URL: https://cme.h-its.org/exelixis/web/software/pear/</details>|
	|[penncnv](https://penncnv.openbioinformatics.org/)|bio|1.0.5|<details><summary>Expand</summary>Description: A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format, it can also handle other types of SNP arrays and oligonucleotide arrays. Homepage: https://penncnv.openbioinformatics.org/ URL: https://penncnv.openbioinformatics.org/</details>|
	|[percolator](https://github.com/percolator)|bio|3.06|<details><summary>Expand</summary>Description: Semi-supervised learning for peptide identification from shotgun proteomics datasets Homepage: https://github.com/percolator URL: https://github.com/percolator</details>|
	|[perl](https://www.perl.org/)|tools|5.30.2, 5.36.1|<details><summary>Expand</summary>Description: Larry Wall's Practical Extraction and Report Language Includes a small selection of extra CPAN packages for core functionality. Homepage: https://www.perl.org/ URL: https://www.perl.org/</details>|
	|[petsc](http://www.mcs.anl.gov/petsc)|tools|3.12.4, 3.13.3, 3.13.6, 3.14.1, 3.15.0, 3.17.1, 3.19.2, 3.20.0, 3.21.6|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-64bits](http://www.mcs.anl.gov/petsc)|tools|3.17.1, 3.19.2, 3.21.6|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-complex](http://www.mcs.anl.gov/petsc)|tools|3.20.0|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-pardiso](http://www.mcs.anl.gov/petsc)|-|3.17.1|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. PETSc with the Intel MKL Pardiso and CPardiso solver. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[petsc-pardiso-64bits](http://www.mcs.anl.gov/petsc)|-|3.17.1|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. PETSc with the Intel MKL Pardiso and CPardiso solver, and 64 bits indices. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[pfamscan](http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/)|-|1.6|<details><summary>Expand</summary>Description: PfamScan is used to search a FASTA sequence against a library of Pfam HMM. Homepage: http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/ URL: http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</details>|
	|[pfft](https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en)|math|1.0.8-alpha|<details><summary>Expand</summary>Description: PFFT is a software library for computing massively parallel, fast Fourier transformations on distributed memory architectures. PFFT can be understood as a generalization of FFTW-MPI to multidimensional data decomposition. Homepage: https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en URL: https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en</details>|
	|[pflotran](https://www.pflotran.org/)|-|4.0, 5.0.0, 6.0.0|<details><summary>Expand</summary>Description: PFLOTRAN is an open source, state-of-the-art massively parallel subsurface flow and reactive transport code. PFLOTRAN solves a system of generally nonlinear partial differential equations describing multiphase, multicomponent and multiscale reactive flow and transport in porous materials. The code is designed to run on massively parallel computing architectures as well as workstations and laptops. Homepage: https://www.pflotran.org/ URL: https://www.pflotran.org/</details>|
	|[pftoolsv3](https://github.com/sib-swiss/pftools3)|-|3.2.12|<details><summary>Expand</summary>Description: A suite of tools to build and search generalized profiles (protein and DNA). Homepage: https://github.com/sib-swiss/pftools3 URL: https://github.com/sib-swiss/pftools3</details>|
	|[pgplot](https://sites.astro.caltech.edu/~tjp/pgplot/)|vis|5.2, 5.2.2|<details><summary>Expand</summary>Description: The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable, device-independent graphics package for making simple scientific graphs. It is intended for making graphical images of publication quality with minimum effort on the part of the user. For most applications, the program can be device-independent, and the output can be directed to the appropriate device at run time. Homepage: https://sites.astro.caltech.edu/~tjp/pgplot/ URL: https://sites.astro.caltech.edu/~tjp/pgplot/</details>|
	|[phast](http://compgen.cshl.edu/phast/)|-|1.6|<details><summary>Expand</summary>Description: PHAST is a freely available software package for comparative and evolutionary genomics. Homepage: http://compgen.cshl.edu/phast/ URL: http://compgen.cshl.edu/phast/</details>|
	|[phon](http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/)|-|1.48|<details><summary>Expand</summary>Description: The program PHON calculates force constant matrices and phonon frequencies in crystals. From the frequencies it also calculates various thermodynamic quantities, like the Helmholtz free energy, the entropy, the specific heat and the internal energy of the harmonic crystal. The procedure is based on the small displacement method, and can be used in combination with any program capable to calculate forces on the atoms of the crystal. Homepage: http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/ URL: http://www.homepages.ucl.ac.uk/~ucfbdxa/phon/</details>|
	|[photospline](https://github.com/icecube/photospline)|-|2.2.1|<details><summary>Expand</summary>Description: Detector response to a high-energy physics process is often estimated by Monte Carlo simulation. For purposes of data analysis, the results of this simulation are typically stored in large multi-dimensional histograms, which can quickly become unwieldy in terms of size or numerically problematic due to unfilled bins or interpolation artifacts. Photospline is a library that uses the penalized spline technique to efficiently compute, store, and evaluate B-spline representations of such tables. Homepage: https://github.com/icecube/photospline URL: https://github.com/icecube/photospline</details>|
	|[phylip](http://evolution.genetics.washington.edu/phylip)|bio|3.698|<details><summary>Expand</summary>Description: PHYLIP is a free package of programs for inferring phylogenies. Homepage: http://evolution.genetics.washington.edu/phylip URL: http://evolution.genetics.washington.edu/phylip</details>|
	|[phylobayes](http://megasun.bch.umontreal.ca/People/lartillot/www/download.html)|bio|4.1c|<details><summary>Expand</summary>Description: A Bayesian software for phylogenetic reconstruction using mixture models Homepage: http://megasun.bch.umontreal.ca/People/lartillot/www/download.html URL: http://megasun.bch.umontreal.ca/People/lartillot/www/download.html</details>|
	|[phylobayes-mpi](https://github.com/bayesiancook/pbmpi)|bio|20180420, 20201026|<details><summary>Expand</summary>Description: A Bayesian software for phylogenetic reconstruction using mixture models Homepage: https://github.com/bayesiancook/pbmpi URL: https://github.com/bayesiancook/pbmpi</details>|
	|[phylokit](https://github.com/pranjalv123/phylokit)|-|1.0|<details><summary>Expand</summary>Description: C++ library for high performance phylogenetics Homepage: https://github.com/pranjalv123/phylokit URL: https://github.com/pranjalv123/phylokit</details>|
	|[phyml](https://github.com/stephaneguindon/phyml)|-|3.3.20190321|<details><summary>Expand</summary>Description: Phylogenetic estimation using (Maximum) Likelihood Homepage: https://github.com/stephaneguindon/phyml URL: https://github.com/stephaneguindon/phyml</details>|
	|[phyx](https://github.com/FePhyFoFum/phyx)|bio|1.1|<details><summary>Expand</summary>Description: phyx performs phylogenetics analyses on trees and sequences. Homepage: https://github.com/FePhyFoFum/phyx URL: https://github.com/FePhyFoFum/phyx</details>|
	|[picard](https://broadinstitute.github.io/picard/)|bio|2.23.3, 2.26.3, 3.1.0|<details><summary>Expand</summary>Description: A set of tools (in Java) for working with next generation sequencing data in the BAM format. Homepage: https://broadinstitute.github.io/picard/ URL: https://broadinstitute.github.io/picard/</details>|
	|[pilercr](https://www.drive5.com/pilercr/)|-|1.06|<details><summary>Expand</summary>Description: PILERCR is public domain software for finding CRISPR repeats. It is part of the PILER family of repeat analysis tools. Homepage: https://www.drive5.com/pilercr/ URL: https://www.drive5.com/pilercr/</details>|
	|[pilon](https://github.com/broadinstitute/pilon)|bio|1.23, 1.24|<details><summary>Expand</summary>Description: Pilon is an automated genome assembly improvement and variant detection tool Homepage: https://github.com/broadinstitute/pilon URL: https://github.com/broadinstitute/pilon</details>|
	|[pindel](http://gmt.genome.wustl.edu/packages/pindel/)|bio|0.2.5b9-20170508|<details><summary>Expand</summary>Description: Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data. It uses a pattern growth approach to identify the breakpoints of these variants from paired-end short reads. Homepage: http://gmt.genome.wustl.edu/packages/pindel/ URL: http://gmt.genome.wustl.edu/packages/pindel/</details>|
	|[platanus](http://platanus.bio.titech.ac.jp/)|bio|1.2.4|<details><summary>Expand</summary>Description: PLATform for Assembling NUcleotide Sequences Homepage: http://platanus.bio.titech.ac.jp/ URL: http://platanus.bio.titech.ac.jp/</details>|
	|[platanus-allee](http://platanus.bio.titech.ac.jp/)|-|2.2.2|<details><summary>Expand</summary>Description: Platanus-allee is an assembler derived from Platanus assembler, however, it was developed with another concept. Platanus-allee tries to construct each haplotype sequence from the beginning and pair them as homologous chromosomes, while Platanus constructs consensus sequence of homologous chromosomes at first and tries to split into each haplotype sequence. Therefore, Platanus-allee marks better performance for highly heterozygous species genome or highly diverged genomic regions. Homepage: http://platanus.bio.titech.ac.jp/ URL: http://platanus.bio.titech.ac.jp/</details>|
	|[plink](https://www.cog-genomics.org/plink/2.0/)|bio|2.00a5.8, 2.00-10252019-avx2, 2.00-20231024-avx2, 1.9b_6.21-x86_64, 2.00a3.6|<details><summary>Expand</summary>Description: Whole-genome association analysis toolset Homepage: https://www.cog-genomics.org/plink/2.0/ URL: https://www.cog-genomics.org/plink/2.0/</details>|
	|[plplot](http://plplot.sourceforge.net)|-|5.15.0|<details><summary>Expand</summary>Description: PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer. Homepage: http://plplot.sourceforge.net URL: http://plplot.sourceforge.net</details>|
	|[plumed](https://www.plumed.org)|chem|2.6.1, 2.6.2, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.7.6, 2.8.1, 2.8.3, 2.8.4, 2.9.0, 2.9.2|<details><summary>Expand</summary>Description: PLUMED is an open source library for free energy calculations in molecular systems which works together with some of the most popular molecular dynamics engines. Free energy calculations can be performed as a function of many order parameters with a particular focus on biological problems, using state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD. The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. Homepage: https://www.plumed.org URL: https://www.plumed.org</details>|
	|[pluto](https://plutojl.org/)|-|0.14.7|<details><summary>Expand</summary>Description: A Pluto notebook is made up of small blocks of Julia code (cells) and together they form a reactive notebook. When you change a variable, Pluto automatically re-runs the cells that refer to it. Cells can even be placed in arbitrary order - intelligent syntax analysis figures out the dependencies between them and takes care of execution. Homepage: https://plutojl.org/ URL: https://plutojl.org/</details>|
	|[pmix](https://pmix.org/)|-|3.1.5, 3.2.3, 4.1.2, 4.2.4, 5.0.2|<details><summary>Expand</summary>Description: Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. Homepage: https://pmix.org/ URL: https://pmix.org/</details>|
	|[pnetcdf](https://trac.mcs.anl.gov/projects/parallel-netcdf)|io|1.9.0, 1.10.0, 1.12.2, 1.12.3|<details><summary>Expand</summary>Description: PnetCDF is a high-performance parallel I/O library for accessing files in format compatibility with Unidata's NetCDF, specifically the formats of CDF-1, 2, and 5. The CDF-5 file format, an extension of CDF-2, supports unsigned data types and uses 64-bit integers to allow users to define large dimensions, attributes, and variables (&gt; 2B array elements). Homepage: https://trac.mcs.anl.gov/projects/parallel-netcdf URL: https://trac.mcs.anl.gov/projects/parallel-netcdf</details>|
	|[podman](https://podman.io)|-|4.1.1, 4.5.0, 4.9.5|<details><summary>Expand</summary>Description: Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Homepage: https://podman.io URL: https://podman.io</details>|
	|[poplddecay](https://github.com/BGI-shenzhen/PopLDdecay)|-|3.41, 3.43|<details><summary>Expand</summary>Description: PopLDdecay: a fast and effective tool for linkage disequilibrium decay analysis based on variant call format(VCF) files. Homepage: https://github.com/BGI-shenzhen/PopLDdecay URL: https://github.com/BGI-shenzhen/PopLDdecay</details>|
	|[popscle](https://github.com/statgen/popscle)|-|0.1-beta|<details><summary>Expand</summary>Description: A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools Homepage: https://github.com/statgen/popscle URL: https://github.com/statgen/popscle</details>|
	|[portaudio](http://www.portaudio.com/)|-|190700_20210406, 190600_20161030|<details><summary>Expand</summary>Description: PortAudio is a free, cross-platform, open-source, audio I/O library. It lets you write simple audio programs in 'C' or C++ that will compile and run on many platforms including Windows, Macintosh OS X, and Unix (OSS/ALSA). It is intended to promote the exchange of audio software between developers on different platforms. Many applications use PortAudio for Audio I/O. Homepage: http://www.portaudio.com/ URL: http://www.portaudio.com/</details>|
	|[postgresql](https://www.postgresql.org/)|tools|12.4, 13.2, 14.2, 15.3, 16.0|<details><summary>Expand</summary>Description: PostgreSQL is a powerful, open source object-relational database system. It is fully ACID compliant, has full support for foreign keys, joins, views, triggers, and stored procedures (in multiple languages). It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP. It also supports storage of binary large objects, including pictures, sounds, or video. It has native programming interfaces for C/C++, Java, .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation. Homepage: https://www.postgresql.org/ URL: https://www.postgresql.org/</details>|
	|[pov-ray](https://www.povray.org/)|-|3.8.0-x.10064738, 3.7.0.10|<details><summary>Expand</summary>Description: The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program which generates images from a text-based scene description, and is available for a variety of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning three-dimensional graphics. The source code is available for those wanting to do their own ports. Homepage: https://www.povray.org/ URL: https://www.povray.org/</details>|
	|[pplacer](http://matsen.fhcrc.org/pplacer/)|bio|1.1.alpha19|<details><summary>Expand</summary>Description: Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give useful information about uncertainty, and to offer advanced visualization and downstream analysis. Homepage: http://matsen.fhcrc.org/pplacer/ URL: http://matsen.fhcrc.org/pplacer/</details>|
	|[prank](http://wasabiapp.org/software/prank/)|bio|170427|<details><summary>Expand</summary>Description: PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences. PRANK is based on a novel algorithm that treats insertions correctly and avoids over-estimation of the number of deletion events. Homepage: http://wasabiapp.org/software/prank/ URL: http://wasabiapp.org/software/prank/</details>|
	|[primer3](https://primer3.org/)|-|2.5.0|<details><summary>Expand</summary>Description: Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction'). PCR is an essential and ubiquitous tool in genetics and molecular biology. Primer3 can also design hybridization probes and sequencing primers. Homepage: https://primer3.org/ URL: https://primer3.org/</details>|
	|[primme](http://www.cs.wm.edu/~andreas/software/)|-|3.2|<details><summary>Expand</summary>Description: PRIMME, pronounced as prime, is a high-performance library for computing a few eigenvalues/eigenvectors, and singular values/vectors. PRIMME is especially optimized for large, difficult problems. Real symmetric and complex Hermitian problems, standard A x = \lambda x and generalized A x = \lambda B x, are supported. Besides, standard eigenvalue problems with a normal matrix are supported. It can find largest, smallest, or interior singular/eigenvalues, and can use preconditioning to accelerate convergence. Homepage: http://www.cs.wm.edu/~andreas/software/ URL: http://www.cs.wm.edu/~andreas/software/</details>|
	|[prinseq](http://prinseq.sourceforge.net)|bio|0.20.4|<details><summary>Expand</summary>Description: A bioinformatics tool to PRe-process and show INformation of SEQuence data. Homepage: http://prinseq.sourceforge.net URL: http://prinseq.sourceforge.net</details>|
	|[prism](https://www.prismmodelchecker.org/)|-|4.7|<details><summary>Expand</summary>Description: PRISM is a probabilistic model checker, a tool for formal modelling and analysis of systems that exhibit random or probabilistic behaviour. It has been used to analyse systems from many different application domains, including communication and multimedia protocols, randomised distributed algorithms, security protocols, biological systems and many others. Homepage: https://www.prismmodelchecker.org/ URL: https://www.prismmodelchecker.org/</details>|
	|[prodigal](https://github.com/hyattpd/Prodigal/)|bio|2.6.3|<details><summary>Expand</summary>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee. Homepage: https://github.com/hyattpd/Prodigal/ URL: https://github.com/hyattpd/Prodigal/</details>|
	|[prodigal-gv](https://github.com/apcamargo/prodigal-gv)|-|2.6.3|<details><summary>Expand</summary>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee. Homepage: https://github.com/apcamargo/prodigal-gv URL: https://github.com/apcamargo/prodigal-gv</details>|
	|[proj](https://proj.org)|geo|4.9.3, 6.3.2, 7.0.0, 7.0.1, 7.2.1, 8.0.0, 9.0.0, 9.0.1, 9.2.0, 9.4.1|<details><summary>Expand</summary>Description: Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates Homepage: https://proj.org URL: https://proj.org</details>|
	|[proj4-fortran](https://github.com/mhagdorn/proj4-fortran)|geo|1.0|<details><summary>Expand</summary>Description: proj4-fortran is a f77 and f90 wrappers for proj4, a cartograohic projections library. Homepage: https://github.com/mhagdorn/proj4-fortran URL: https://github.com/mhagdorn/proj4-fortran</details>|
	|[prokka](https://www.vicbioinformatics.com/software.prokka.shtml)|-|1.14.5|<details><summary>Expand</summary>Description: Prokka is a software tool for the rapid annotation of prokaryotic genomes. Homepage: https://www.vicbioinformatics.com/software.prokka.shtml URL: https://www.vicbioinformatics.com/software.prokka.shtml</details>|
	|[protobuf](https://github.com/google/protobuf/)|tools|3.12.3, 3.19.4, 3.21.3, 24.4|<details><summary>Expand</summary>Description: Google Protocol Buffers Homepage: https://github.com/google/protobuf/ URL: https://github.com/google/protobuf/</details>|
	|[prrte](https://docs.prrte.org/)|-|3.0.5|<details><summary>Expand</summary>Description: PRRTE is the PMIx Reference RunTime Environment Homepage: https://docs.prrte.org/ URL: https://docs.prrte.org/</details>|
	|[psfex](https://psfex.readthedocs.io/en/latest/index.html)|-|3.21.1|<details><summary>Expand</summary>Description: A computer program that extracts precise models of the Point Spread Functions (PSFs) from images processed by SExtractor and measures the quality of images. The generated PSF models can be used for model-fitting photometry or morphological analyses. Homepage: https://psfex.readthedocs.io/en/latest/index.html URL: https://psfex.readthedocs.io/en/latest/index.html</details>|
	|[psi4](http://www.psicode.org/)|chem|1.3.2, 1.4, 1.5, 1.9, 1.9.1|<details><summary>Expand</summary>Description: PSI4 is an open-source suite of ab initio quantum chemistry programs designed for efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform computations with more than 2500 basis functions running serially or in parallel. Homepage: http://www.psicode.org/ URL: http://www.psicode.org/</details>|
	|[psipred](http://bioinf.cs.ucl.ac.uk)|-|4.02|<details><summary>Expand</summary>Description: Accurate protein secondary structure prediction Homepage: http://bioinf.cs.ucl.ac.uk URL: http://bioinf.cs.ucl.ac.uk</details>|
	|[psmc](https://github.com/lh3/psmc)|bio|0.6.5|<details><summary>Expand</summary>Description: This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model. Homepage: https://github.com/lh3/psmc URL: https://github.com/lh3/psmc</details>|
	|[ptex](https://ptex.us/)|-|2.3.2|<details><summary>Expand</summary>Description: Ptex is a texture mapping system developed by Walt Disney Animation Studios for production-quality rendering: Homepage: https://ptex.us/ URL: https://ptex.us/</details>|
	|[pullseq](https://github.com/bcthomas/pullseq)|-|1.0.2|<details><summary>Expand</summary>Description: Utility program for extracting sequences from a fasta/fastq file Homepage: https://github.com/bcthomas/pullseq URL: https://github.com/bcthomas/pullseq</details>|
	|[pyne](http://pyne.io/)|-|0.7.3, 0.7.7|<details><summary>Expand</summary>Description: PyNE is a suite of tools to aid in computational nuclear science and engineering. PyNE seeks to provide native implementations of common nuclear algorithms, as well as Python bindings and I/O support for other industry standard nuclear codes. Homepage: http://pyne.io/ URL: http://pyne.io/</details>|
	|[pypy](https://pypy.org/index.html)|tools|5.8.0, 7.3.3|<details><summary>Expand</summary>Description: PyPy is a fast, compliant alternative implementation of the Python language (2.7.13 and 3.5.3). Homepage: https://pypy.org/index.html URL: https://pypy.org/index.html</details>|
	|[pytest](https://pytest.org)|-|6.1.2, 6.2.5, 7.0.1, 7.4.0, 8.2.2|<details><summary>Expand</summary>Description: pytest: simple powerful testing with Python Homepage: https://pytest.org URL: https://pytest.org</details>|
	|[python](https://python.org/)|tools|2.7.18, 3.6.10, 3.7.7, 3.7.9, 3.8.2, 3.8.10, 3.9.6, 3.10.2, 3.10.13, 3.11.2, 3.11.5, 3.12.4, 3.13.2|<details><summary>Expand</summary>Description: Python is a programming language that lets you work more quickly and integrate your systems more effectively. Homepage: https://python.org/ URL: https://python.org/</details>|
	|[python-build-bundle](http://python.org/)|-|2022a, 2023a, 2023b, 2024a, 2025a|<details><summary>Expand</summary>Description: Bundle which contains Python packages useful for building Homepage: http://python.org/ URL: http://python.org/</details>|
	|[q6](https://github.com/qusers/Q6)|-|6.0|<details><summary>Expand</summary>Description: Q is a set of Molecular Dynamics (MD) tools tailored to the following specific kinds of free energy calculations: Free Energy Perturbation (FEP) Empirical Valence Bond (EVB) Linear Interaction Energies (LIE) Quantum Classical Path (QCP) Homepage: https://github.com/qusers/Q6 URL: https://github.com/qusers/Q6</details>|
	|[qca](https://userbase.kde.org/QCA)|geo|2.3.0, 2.3.5, 2.3.7|<details><summary>Expand</summary>Description: Taking a hint from the similarly-named Java Cryptography Architecture, QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes and conventions. QCA separates the API from the implementation, using plugins known as Providers. The advantage of this model is to allow applications to avoid linking to or explicitly depending on any particular cryptographic library. This allows one to easily change or upgrade crypto implementations without even needing to recompile the application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX. Homepage: https://userbase.kde.org/QCA URL: https://userbase.kde.org/QCA</details>|
	|[qcint](http://wiki.sunqm.net/libcint)|-|4.4.6, 5.1.5, 5.1.7, 5.1.8, 6.1.2|<details><summary>Expand</summary>Description: libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform. Homepage: http://wiki.sunqm.net/libcint URL: http://wiki.sunqm.net/libcint</details>|
	|[qctool](https://www.well.ox.ac.uk/~gav/qctool_v2/)|-|2.2.0|<details><summary>Expand</summary>Description: QCTOOL is a command-line utility program for manipulation and quality control of gwas datasets and other genome-wide data. Homepage: https://www.well.ox.ac.uk/~gav/qctool_v2/ URL: https://www.well.ox.ac.uk/~gav/qctool_v2/</details>|
	|[qgis](http://www.qgis.org/)|geo|3.10.6, 3.16.10, 3.22.14, 3.28.12|<details><summary>Expand</summary>Description: QGIS is a user friendly Open Source Geographic Information System (GIS) Homepage: http://www.qgis.org/ URL: http://www.qgis.org/</details>|
	|[qhull](http://www.qhull.org)|math|2019.1, 2020.2|<details><summary>Expand</summary>Description: Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull. Homepage: http://www.qhull.org URL: http://www.qhull.org</details>|
	|[qiime2](http://qiime2.org/)|-|2021.11, 2023.5, 2024.5|<details><summary>Expand</summary>Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. Homepage: http://qiime2.org/ URL: http://qiime2.org/</details>|
	|[qjson](http://qjson.sourceforge.net/)|tools|0.9.0|<details><summary>Expand</summary>Description: QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa. Homepage: http://qjson.sourceforge.net/ URL: http://qjson.sourceforge.net/</details>|
	|[qrupdate](https://sourceforge.net/projects/qrupdate/)|math|1.1.2|<details><summary>Expand</summary>Description: qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions. Homepage: https://sourceforge.net/projects/qrupdate/ URL: https://sourceforge.net/projects/qrupdate/</details>|
	|[qscintilla](https://www.riverbankcomputing.com/software/qscintilla)|tools|2.11.2, 2.11.6, 2.14.1|<details><summary>Expand</summary>Description: QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control Homepage: https://www.riverbankcomputing.com/software/qscintilla URL: https://www.riverbankcomputing.com/software/qscintilla</details>|
	|[qt](https://qt.io/)|tools|5.12.8, 5.15.2, 5.15.8, 5.15.11, 6.5.3|<details><summary>Expand</summary>Description: Qt is a comprehensive cross-platform C++ application framework. Homepage: https://qt.io/ URL: https://qt.io/</details>|
	|[qt5webkit](https://github.com/qt/qtwebkit)|-|5.212.0-alpha4|<details><summary>Expand</summary>Description: Qt Port of WebKit. WebKit is an open source web browser engine. Homepage: https://github.com/qt/qtwebkit URL: https://github.com/qt/qtwebkit</details>|
	|[qtkeychain](https://github.com/frankosterfeld/qtkeychain)|-|0.9.1, 0.13.2, 0.14.1|<details><summary>Expand</summary>Description: Platform-independent Qt API for storing passwords securely. Homepage: https://github.com/frankosterfeld/qtkeychain URL: https://github.com/frankosterfeld/qtkeychain</details>|
	|[qtltools](https://qtltools.github.io/qtltools/)|-|1.3.1|<details><summary>Expand</summary>Description: QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) in few easy-to-perform steps. Homepage: https://qtltools.github.io/qtltools/ URL: https://qtltools.github.io/qtltools/</details>|
	|[qualimap](http://qualimap.bioinfo.cipf.es/)|-|2.3|<details><summary>Expand</summary>Description: Qualimap 2 is a platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data and its derivatives like feature counts. Homepage: http://qualimap.bioinfo.cipf.es/ URL: http://qualimap.bioinfo.cipf.es/</details>|
	|[quantumatk](https://www.synopsys.com/silicon/quantumatk.html)|-|2019.12, 2020.09, 2022.03, 2024.09|<details><summary>Expand</summary>Description: QuantumATK is a complete atomistic simulation toolkit. QuantumATK addresses key applications in the semiconductor industry and is a core component of the Synopsys Design-Technology Co-Optimization (DTCO) solution. Atomic-scale modeling tools in QuantumATK range from classical force fields for handling large and to some extent more realistic materials systems to ab initio tools that provide highly accurate results for smaller systems. Homepage: https://www.synopsys.com/silicon/quantumatk.html URL: https://www.synopsys.com/silicon/quantumatk.html</details>|
	|[quantumespresso](https://www.quantum-espresso.org)|chem|6.5, 6.6, 6.7, 6.8, 7.0, 7.1, 7.2, 7.3.1|<details><summary>Expand</summary>Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). Homepage: https://www.quantum-espresso.org URL: https://www.quantum-espresso.org</details>|
	|[quast](https://github.com/ablab/quast)|-|5.0.2, 5.2.0|<details><summary>Expand</summary>Description: QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison. Homepage: https://github.com/ablab/quast URL: https://github.com/ablab/quast</details>|
	|[qwt](http://qwt.sourceforge.net/)|tools|6.1.4, 6.2.0|<details><summary>Expand</summary>Description: The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background. Homepage: http://qwt.sourceforge.net/ URL: http://qwt.sourceforge.net/</details>|
	|[qwtpolar](http://qwtpolar.sourceforge.net/)|tools|1.1.1|<details><summary>Expand</summary>Description: The QwtPolar library contains classes for displaying values on a polar coordinate system. Homepage: http://qwtpolar.sourceforge.net/ URL: http://qwtpolar.sourceforge.net/</details>|
	|[r](https://www.r-project.org/)|tools|4.0.0, 4.0.2, 4.0.5, 4.1.0, 4.1.2, 4.2.1, 4.2.2, 4.3.1, 4.4.0|<details><summary>Expand</summary>Description: R is a free software environment for statistical computing and graphics. Homepage: https://www.r-project.org/ URL: https://www.r-project.org/</details>|
	|[r-bundle-bioconductor](https://bioconductor.org)|bio|3.12, 3.14, 3.16, 3.17, 3.18, 3.20|<details><summary>Expand</summary>Description: Bioconductor provides tools for the analysis and coprehension of high-throughput genomic data. Homepage: https://bioconductor.org URL: https://bioconductor.org</details>|
	|[r2r](https://sourceforge.net/projects/weinberg-r2r/files/latest/download/)|-|1.0.6|<details><summary>Expand</summary>Description: Weinberg-R2R: Faster creation of aesthetic RNA secondary structure diagrams Homepage: https://sourceforge.net/projects/weinberg-r2r/files/latest/download/ URL: https://sourceforge.net/projects/weinberg-r2r/files/latest/download/</details>|
	|[racket](https://racket-lang.org/)|-|7.8|<details><summary>Expand</summary>Description: Racket, the Programming Language Homepage: https://racket-lang.org/ URL: https://racket-lang.org/</details>|
	|[racon](https://github.com/lbcb-sci/racon)|bio|1.4.3, 1.4.13, 1.5.0|<details><summary>Expand</summary>Description: Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. Homepage: https://github.com/lbcb-sci/racon URL: https://github.com/lbcb-sci/racon</details>|
	|[randomlib](http://randomlib.sourceforge.net/html/)|-|1.10|<details><summary>Expand</summary>Description: RandomLib is a C++ interface to the Mersenne Twister random number generator, MT19937 and to the SIMD-oriented Fast Mersenne Twister random number generator, SFMT19937. It provides convenient access to random integers and reals at a variety of precisions. RandomLib also contains new algorithms which permit exact sampling from the normal and discrete normal distributions (provided that the underlying generator is perfect). The emphasis in this implementation is on providing a reliable source of random numbers for scientific applications where there's a premium on accuracy, repeatability, portability, and ease of use. By default, this library uses SFMT's improved method for seeding the generator and it allows access to both the 32-bit and 64-bit versions of MT19937 and SFMT19937 and includes implementations of the SFMT19937 using SSE2 and AltiVec instructions. Homepage: http://randomlib.sourceforge.net/html/ URL: http://randomlib.sourceforge.net/html/</details>|
	|[rapidjson](https://rapidjson.org)|-|1.1.0|<details><summary>Expand</summary>Description: A fast JSON parser/generator for C++ with both SAX/DOM style API Homepage: https://rapidjson.org URL: https://rapidjson.org</details>|
	|[rapsearch2](https://omics.informatics.indiana.edu/mg/RAPSearch2)|-|2.24|<details><summary>Expand</summary>Description: RAPSearch stands for Reduced Alphabet based Protein similarity Search Homepage: https://omics.informatics.indiana.edu/mg/RAPSearch2 URL: https://omics.informatics.indiana.edu/mg/RAPSearch2</details>|
	|[raqm](https://github.com/HOST-Oman/libraqm)|-|0.9.0, 0.10.1|<details><summary>Expand</summary>Description: Raqm is a small library that encapsulates the logic for complex text layout and provides a convenient API. Homepage: https://github.com/HOST-Oman/libraqm URL: https://github.com/HOST-Oman/libraqm</details>|
	|[raspa2](https://iraspa.org/raspa/)|-|2.0.47|<details><summary>Expand</summary>Description: RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials. Homepage: https://iraspa.org/raspa/ URL: https://iraspa.org/raspa/</details>|
	|[raspa3](https://iraspa.org/raspa/)|-|3.0.3|<details><summary>Expand</summary>Description: RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials. Homepage: https://iraspa.org/raspa/ URL: https://iraspa.org/raspa/</details>|
	|[ratatosk](https://github.com/DecodeGenetics/Ratatosk)|-|0.7.6.3|<details><summary>Expand</summary>Description: Phased hybrid error correction of long reads using colored de Bruijn graphs Homepage: https://github.com/DecodeGenetics/Ratatosk URL: https://github.com/DecodeGenetics/Ratatosk</details>|
	|[raven](https://github.com/lbcb-sci/raven)|-|1.5.0|<details><summary>Expand</summary>Description: Raven is a de novo genome assembler for long uncorrected reads. Homepage: https://github.com/lbcb-sci/raven URL: https://github.com/lbcb-sci/raven</details>|
	|[raxml](https://github.com/stamatak/standard-RAxML)|bio|8.2.12|<details><summary>Expand</summary>Description: RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. Homepage: https://github.com/stamatak/standard-RAxML URL: https://github.com/stamatak/standard-RAxML</details>|
	|[raxml-ng](https://github.com/amkozlov/raxml-ng)|-|1.0.1, 1.2.0|<details><summary>Expand</summary>Description: RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree. Homepage: https://github.com/amkozlov/raxml-ng URL: https://github.com/amkozlov/raxml-ng</details>|
	|[raxml-pthreads](https://github.com/stamatak/standard-RAxML)|-|8.2.12|<details><summary>Expand</summary>Description: RAxML search algorithm for maximum likelihood based inference of phylogenetic trees. Homepage: https://github.com/stamatak/standard-RAxML URL: https://github.com/stamatak/standard-RAxML</details>|
	|[ray](https://github.com/zorino/ray)|bio|3.0.1|<details><summary>Expand</summary>Description: Ray -- Parallel genome assemblies for parallel DNA sequencing. Homepage: https://github.com/zorino/ray URL: https://github.com/zorino/ray</details>|
	|[rcorrector](https://github.com/mourisl/Rcorrector)|-|1.0.4, 1.0.6|<details><summary>Expand</summary>Description: Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data. Homepage: https://github.com/mourisl/Rcorrector URL: https://github.com/mourisl/Rcorrector</details>|
	|[rdkit](http://www.rdkit.org/)|chem|2019.03.4, 2020.09.1b1, 2021.03.3, 2021.09.3, 2021.09.4, 2022.09.4, 2023.09.3, 2023.09.5, 2024.03.4, 2024.03.5, 2024.09.6|<details><summary>Expand</summary>Description: RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python. Homepage: http://www.rdkit.org/ URL: http://www.rdkit.org/</details>|
	|[re2c](https://re2c.org/)|-|1.3|<details><summary>Expand</summary>Description: re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons. Homepage: https://re2c.org/ URL: https://re2c.org/</details>|
	|[recon](http://eddylab.org/software/recon/)|bio|1.08|<details><summary>Expand</summary>Description: Proper identification of repetitive sequences is an essential step in genome analysis. The RECON package performs de novo identification and classification of repeat sequence families from genomic sequences. The underlying algorithm is based on extensions to the usual approach of single linkage clustering of local pairwise alignments between genomic sequences. Specifically, our extensions use multiple alignment information to define the boundaries of individual copies of the repeats and to distinguish homologous but distinct repeat element families. RECON should be useful for first-pass automatic classification of repeats in newly sequenced genomes. Homepage: http://eddylab.org/software/recon/ URL: http://eddylab.org/software/recon/</details>|
	|[reduce](https://github.com/rlabduke/reduce)|-|4.14|<details><summary>Expand</summary>Description: Reduce - tool for adding and correcting hydrogens in PDB files Homepage: https://github.com/rlabduke/reduce URL: https://github.com/rlabduke/reduce</details>|
	|[reframe](https://github.com/eth-cscs/reframe)|-|3.12.0|<details><summary>Expand</summary>Description: ReFrame is a framework for writing regression tests for HPC systems. Homepage: https://github.com/eth-cscs/reframe URL: https://github.com/eth-cscs/reframe</details>|
	|[regenie](https://doi.org/10.1038/s41588-021-00870-7)|-|2.2.4, 3.0, 3.2.1|<details><summary>Expand</summary>Description: regenie is a C++ program for whole genome regression modelling of large genome-wide association studies. It is developed and supported by a team of scientists at the Regeneron Genetics Center. The method has the following properties It works on quantitative and binary traits, including binary traits with unbalanced case-control ratios It can process multiple phenotypes at once For binary traits it supports Firth logistic regression and an SPA test It can perform gene/region-based burden tests It is fast and memory efficient It supports the BGEN, PLINK bed/bim/fam and PLINK2 pgen/pvar/psam genetic data formats It is ideally suited for implementation in Apache Spark (see GLOW) Citation Mbatchou, J., Barnard, L., Backman, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. Nat Genet 53, 10971103 (2021). https://doi.org/10.1038/s41588-021-00870-7 Homepage: https://rgcgithub.github.io/regenie/ URL: https://rgcgithub.github.io/regenie/</details>|
	|[regtools](https://regtools.readthedocs.org)|-|1.0.0|<details><summary>Expand</summary>Description: RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context. Homepage: https://regtools.readthedocs.org URL: https://regtools.readthedocs.org</details>|
	|[relion](https://bitbucket.org/scheres/relion-3.0_beta.git)|chem|3.1.1, 4.0.0|<details><summary>Expand</summary>Description: RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer program for Maximum A Posteriori refinement of (multiple) 3D reconstructions or 2D class averages in cryo-electron microscopy. It is developed in the research group of Sjors Scheres at the MRC Laboratory of Molecular Biology. Homepage: https://bitbucket.org/scheres/relion-3.0_beta.git URL: https://bitbucket.org/scheres/relion-3.0_beta.git</details>|
	|[repasthpc](https://repast.github.io/)|bio|2.2.0, 2.3.0|<details><summary>Expand</summary>Description: The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years: Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based modeling system that is designed for use on large computing clusters and supercomputers. Homepage: https://repast.github.io/ URL: https://repast.github.io/</details>|
	|[repeatmasker](http://www.repeatmasker.org/)|bio|4.1.2-p1, 4.1.1, 4.1.4|<details><summary>Expand</summary>Description: RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. Homepage: http://www.repeatmasker.org/ URL: http://www.repeatmasker.org/</details>|
	|[repeatmodeler](http://repeatmasker.org/RepeatModeler/)|bio|2.0.1, 2.0.2a, 2.0.3|<details><summary>Expand</summary>Description: RepeatModeler is a de-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats. Homepage: http://repeatmasker.org/RepeatModeler/ URL: http://repeatmasker.org/RepeatModeler/</details>|
	|[repeatscout](https://bix.ucsd.edu/repeatscout/)|bio|1.0.5|<details><summary>Expand</summary>Description: RepeatScout is a tool to discover repetitive substrings in DNA. Homepage: https://bix.ucsd.edu/repeatscout/ URL: https://bix.ucsd.edu/repeatscout/</details>|
	|[revbayes](https://revbayes.github.io/)|-|1.1.1|<details><summary>Expand</summary>Description: RevBayes provides an interactive environment for statistical computation in phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in evolutionary biology, particularly phylogenetics. Homepage: https://revbayes.github.io/ URL: https://revbayes.github.io/</details>|
	|[rgaugury](https://bitbucket.org/yaanlpc/rgaugury/src/master/)|-|2.1.3|<details><summary>Expand</summary>Description: This project aims to develop a sophisticated informatics tools for large scale of resistance gene analogs identification, including major groups of RGAs, such as NBS encoding genes, receptor like proteins and receptor like protein kinases. It is thus named as RGA augury or RGAugury standalone pipeline. Homepage: https://bitbucket.org/yaanlpc/rgaugury/src/master/ URL: https://bitbucket.org/yaanlpc/rgaugury/src/master/</details>|
	|[rkcommon](https://github.com/ospray/rkcommon)|-|1.4.2, 1.10.0, 1.11.0|<details><summary>Expand</summary>Description: Intel RenderKit common library Homepage: https://github.com/ospray/rkcommon URL: https://github.com/ospray/rkcommon</details>|
	|[rmats](http://rnaseq-mats.sourceforge.net/)|-|4.1.1, 4.1.2, 4.3.0|<details><summary>Expand</summary>Description: MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data. The statistical model of MATS calculates the P-value and false discovery rate that the difference in the isoform ratio of a gene between two conditions exceeds a given user-defined threshold. From the RNA-Seq data, MATS can automatically detect and analyze alternative splicing events corresponding to all major types of alternative splicing patterns. MATS handles replicate RNA-Seq data from both paired and unpaired study design. Homepage: http://rnaseq-mats.sourceforge.net/ URL: http://rnaseq-mats.sourceforge.net/</details>|
	|[rmblast](http://www.repeatmasker.org/RMBlast.html)|bio|2.10.0, 2.14.1|<details><summary>Expand</summary>Description: RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn' for use with RepeatMasker and RepeatModeler. Homepage: http://www.repeatmasker.org/RMBlast.html URL: http://www.repeatmasker.org/RMBlast.html</details>|
	|[rnacode](https://wash.github.io/rnacode/)|-|0.3|<details><summary>Expand</summary>Description: RNAcode - Analyze the protein coding potential in multiple sequence alignments Homepage: https://wash.github.io/rnacode/ URL: https://wash.github.io/rnacode/</details>|
	|[rnammer](http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer)|bio|1.2|<details><summary>Expand</summary>Description: RNAmmer 1.2 predicts 5s/8s, 16s/18s, and 23s/28s ribosomal RNA in full genome sequences. The method is described in detail in the following article: RNammer: consistent annotation of rRNA genes in genomic sequences. Lagesen K, Hallin PF, Roedland E, Staerfeldt HH, Rognes T Ussery DW. Nucleic Acids Res. Apr 22, 2007. Homepage: http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer URL: http://www.cbs.dtu.dk/cgi-bin/sw_request?rnammer</details>|
	|[roary](https://sanger-pathogens.github.io/Roary/)|bio|3.13.0|<details><summary>Expand</summary>Description: Roary is a high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome. Homepage: https://sanger-pathogens.github.io/Roary/ URL: https://sanger-pathogens.github.io/Roary/</details>|
	|[rocksdb](https://github.com/facebook/rocksdb)|-|7.10.2|<details><summary>Expand</summary>Description: A library that provides an embeddable, persistent key-value store for fast storage. Homepage: https://github.com/facebook/rocksdb URL: https://github.com/facebook/rocksdb</details>|
	|[root](https://root.cern.ch/drupal/)|tools|6.20.04, 6.24.06, 6.26.06, 6.28.06|<details><summary>Expand</summary>Description: The ROOT system provides a set of OO frameworks with all the functionality needed to handle and analyze large amounts of data in a very efficient way. Homepage: https://root.cern.ch/drupal/ URL: https://root.cern.ch/drupal/</details>|
	|[rosetta](https://www.rosettacommons.org)|chem|3.10, 3.12, 3.13, 2019.21.60746|<details><summary>Expand</summary>Description: Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids. This is a weekly release. Homepage: https://www.rosettacommons.org URL: https://www.rosettacommons.org</details>|
	|[rsem](http://deweylab.github.io/RSEM/)|bio|1.3.3|<details><summary>Expand</summary>Description: RNA-Seq by Expectation-Maximization Homepage: http://deweylab.github.io/RSEM/ URL: http://deweylab.github.io/RSEM/</details>|
	|[rstudio-server](https://www.rstudio.com/)|tools|4.1, 4.2, 4.3, 4.4|<details><summary>Expand</summary>Description: RStudio Server enables you to provide a browser based interface to a version of R running on a remote Linux server, bringing the power and productivity of the RStudio IDE to server-based deployments of R. Homepage: https://www.rstudio.com/ URL: https://www.rstudio.com/</details>|
	|[rtk](https://itk.org)|bio|1.4.0|<details><summary>Expand</summary>Description: Insighteconstruction Toolkit (RTK) is an open-source and cross-platform software for fast circular cone-beam CT reconstruction based on the Insight Toolkit (ITK). RTK is developped by the RTK consortium. RTK provides or will provide Basic operators for reconstruction, e.g., filtering, forward, projection and backprojection Multithreaded CPU and GPU versions Tools for respiratory motion correction I/O for several scanners Preprocessing of raw data for scatter correction Homepage: https://itk.org URL: https://itk.org</details>|
	|[ruamel.yaml](https://sourceforge.net/projects/ruamel-yaml)|-|0.17.21|<details><summary>Expand</summary>Description: ruamel.yaml is a YAML 1.2 loader/dumper package for Python. Homepage: https://sourceforge.net/projects/ruamel-yaml URL: https://sourceforge.net/projects/ruamel-yaml</details>|
	|[ruby](https://www.ruby-lang.org)|tools|2.7.1, 3.2.2|<details><summary>Expand</summary>Description: Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. Homepage: https://www.ruby-lang.org URL: https://www.ruby-lang.org</details>|
	|[rust](https://www.rust-lang.org)|tools|1.47.0, 1.53.0, 1.59.0, 1.65.0, 1.70.0, 1.76.0, 1.85.0|<details><summary>Expand</summary>Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. Homepage: https://www.rust-lang.org URL: https://www.rust-lang.org</details>|
	|[sabre](https://github.com/najoshi/sabre)|bio|1.00|<details><summary>Expand</summary>Description: A barcode demultiplexing and trimming tool for FastQ files Homepage: https://github.com/najoshi/sabre URL: https://github.com/najoshi/sabre</details>|
	|[saga](https://saga-gis.sourceforge.io/)|-|8.2.2|<details><summary>Expand</summary>Description: System for automated geoscientific analyses Homepage: https://saga-gis.sourceforge.io/ URL: https://saga-gis.sourceforge.io/</details>|
	|[sagemath](http://www.sagemath.org/)|tools|9.1, 9.3|<details><summary>Expand</summary>Description: SageMath is a free open-source mathematics software system licensed under the GPL. It builds on top of many existing open-source packages: NumPy, SciPy, matplotlib, Sympy, Maxima, GAP, FLINT, R and many more. Access their combined power through a common, Python-based language or directly via interfaces or wrappers. Homepage: http://www.sagemath.org/ URL: http://www.sagemath.org/</details>|
	|[salmon](https://github.com/COMBINE-lab/salmon)|bio|1.3.0, 1.4.0, 1.7.0, 1.10.2|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Homepage: https://github.com/COMBINE-lab/salmon URL: https://github.com/COMBINE-lab/salmon</details>|
	|[salmonbeta](https://github.com/COMBINE-lab/salmon/)|bio|0.6.0|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Salmon achieves is accuracy and speed via a number of different innovations, including the use of quasi-mapping (accurate but fast-to-compute proxies for traditional read alignments), and massively-parallel stochastic collapsed variational inference. The result is a versatile tool that fits nicely into many differnt pipelines. For example, you can choose to make use of our quasi-mapping algorithm by providing Salmon with raw sequencing reads, or, if it is more convenient, you can provide Salmon with regular alignments (e.g. an unsorted BAM file produced with your favorite aligner), and it will use the same wicked-fast, state-of-the-art inference algorithm to estimate transcript-level abundances for your experiment. Homepage: https://github.com/COMBINE-lab/salmon/ URL: https://github.com/COMBINE-lab/salmon/</details>|
	|[sambamba](https://lomereiter.github.io/sambamba/)|bio|0.8.0, 1.0.1|<details><summary>Expand</summary>Description: Sambamba is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Current functionality is an important subset of samtools functionality, including view, index, sort, markdup, and depth. Homepage: https://lomereiter.github.io/sambamba/ URL: https://lomereiter.github.io/sambamba/</details>|
	|[samblaster](https://github.com/GregoryFaust/samblaster)|bio|0.1.26|<details><summary>Expand</summary>Description: samblaster: a tool to mark duplicates and extract discordant and split reads from sam files Homepage: https://github.com/GregoryFaust/samblaster URL: https://github.com/GregoryFaust/samblaster</details>|
	|[samstat](http://samstat.sourceforge.net/)|-|1.5.1|<details><summary>Expand</summary>Description: Displaying sequence statistics for next generation sequencing Homepage: http://samstat.sourceforge.net/ URL: http://samstat.sourceforge.net/</details>|
	|[samtools](https://www.htslib.org/)|bio|0.1.18, 0.1.20, 1.10, 1.11, 1.12, 1.13, 1.15.1, 1.16.1, 1.17, 1.18, 1.20|<details><summary>Expand</summary>Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format. Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[savvy](https://github.com/statgen/savvy)|-|2.1.0|<details><summary>Expand</summary>Description: Interface to various variant calling formats. Homepage: https://github.com/statgen/savvy URL: https://github.com/statgen/savvy</details>|
	|[sbt](http://www.scala-sbt.org/)|tools|1.3.13|<details><summary>Expand</summary>Description: sbt is a build tool for Scala, Java, and more. It requires Java 1.8 or later. sbt uses a small number of concepts to support flexible and powerful build definitions. There are not that many concepts, but sbt is not exactly like other build systems and there are details you will stumble on if you haven t read the documentation. The Getting Started Guide covers the concepts you need to know to create and maintain an sbt build definition. It is highly recommended to read the Getting Started Guide. Homepage: http://www.scala-sbt.org/ URL: http://www.scala-sbt.org/</details>|
	|[scalapack](https://www.netlib.org/scalapack/)|math|2.1.0, 2.2.0|<details><summary>Expand</summary>Description: The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines redesigned for distributed memory MIMD parallel computers. Homepage: https://www.netlib.org/scalapack/ URL: https://www.netlib.org/scalapack/</details>|
	|[scalasca](https://www.scalasca.org/)|-|2.5|<details><summary>Expand</summary>Description: Scalasca is a software tool that supports the performance optimization of parallel programs by measuring and analyzing their runtime behavior. The analysis identifies potential performance bottlenecks -- in particular those concerning communication and synchronization -- and offers guidance in exploring their causes. Homepage: https://www.scalasca.org/ URL: https://www.scalasca.org/</details>|
	|[scipoptsuite](https://scipopt.org/)|-|9.0.0|<details><summary>Expand</summary>Description: SCIP is currently one of the fastest non-commercial solvers for mixed integer programming (MIP) and mixed integer nonlinear programming (MINLP). It is also a framework for constraint integer programming and branch-cut-and-price. It allows for total control of the solution process and the access of detailed information down to the guts of the solver. Homepage: https://scipopt.org/ URL: https://scipopt.org/</details>|
	|[scipy-stack](http://python.org/)|math|2020a, 2020b, 2021a, 2022a, 2023a, 2023b, 2024a, 2024b, 2025a|<details><summary>Expand</summary>Description: Bundle which contains the Scientific Python stack, including Cycler, mpmath, numpy, scipy, sympy, pandas, matplotlib, ipython_genutils, traitlets, ptyprocess, pathlib2, pickleshare, pexpect, simplegeneric, ipython, ipykernel, jupyter_client, jupyter_core, pyzmq, tornado, futures and ipyparallel. Homepage: http://python.org/ URL: http://python.org/</details>|
	|[score-p](https://www.score-p.org)|-|6.0|<details><summary>Expand</summary>Description: The Score-P measurement infrastructure is a highly scalable and easy-to-use tool suite for profiling, event tracing, and online analysis of HPC applications. Homepage: https://www.score-p.org URL: https://www.score-p.org</details>|
	|[scotch](https://www.labri.fr/perso/pelegrin/scotch/)|math|6.0.9, 7.0.3|<details><summary>Expand</summary>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. Homepage: https://www.labri.fr/perso/pelegrin/scotch/ URL: https://www.labri.fr/perso/pelegrin/scotch/</details>|
	|[scotch-no-thread](https://www.labri.fr/perso/pelegrin/scotch/)|-|6.1.2, 7.0.3|<details><summary>Expand</summary>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. This version has no threading support. Homepage: https://www.labri.fr/perso/pelegrin/scotch/ URL: https://www.labri.fr/perso/pelegrin/scotch/</details>|
	|[scythe](https://github.com/ucdavis-bioinformatics/scythe)|-|0.994|<details><summary>Expand</summary>Description: Scythe uses a Naive Bayesian approach to classify contaminant substrings in sequence reads. It considers quality information, which can make it robust in picking out 3'-end adapters, which often include poor quality bases. Homepage: https://github.com/ucdavis-bioinformatics/scythe URL: https://github.com/ucdavis-bioinformatics/scythe</details>|
	|[sdl2](http://www.libsdl.org/)|tools|2.0.16|<details><summary>Expand</summary>Description: SDL: Simple DirectMedia Layer, a cross-platform multimedia library Homepage: http://www.libsdl.org/ URL: http://www.libsdl.org/</details>|
	|[sdsl](https://github.com/simongog/sdsl-lite)|-|2.1.1-20191211|<details><summary>Expand</summary>Description: The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing succinct data structures. Homepage: https://github.com/simongog/sdsl-lite URL: https://github.com/simongog/sdsl-lite</details>|
	|[selscan](https://github.com/szpiech/selscan)|bio|2.0.3|<details><summary>Expand</summary>Description: selscan -- a program to calculate EHH-based scans for positive selection in genomes selscan currently implements EHH, iHS, XP-EHH, nSL, and iHH12 and requires phased data. It should be run separately for each chromosome and population (or population pair for XP-EHH). selscan is 'dumb' with respect ancestral/derived coding and simply expects haplotype data to be coded 0/1. Unstandardized iHS scores are thus reported as ln(iHH1/iHH0) based on the coding you have provided. Homepage: https://github.com/szpiech/selscan URL: https://github.com/szpiech/selscan</details>|
	|[seqan-library](https://www.seqan.de/)|-|2.4.0|<details><summary>Expand</summary>Description: SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences with the focus on biological data Homepage: https://www.seqan.de/ URL: https://www.seqan.de/</details>|
	|[seqkit](https://bioinf.shenwei.me/seqkit/)|-|0.13.2, 0.15.0, 2.3.1, 2.5.1|<details><summary>Expand</summary>Description: FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. This project describes a cross-platform ultrafast comprehensive toolkit for FASTA/Q processing. SeqKit provides executable binary files for all major operating systems, including Windows, Linux, and Mac OS X, and can be directly used without any dependencies or pre-configurations. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations. Homepage: https://bioinf.shenwei.me/seqkit/ URL: https://bioinf.shenwei.me/seqkit/</details>|
	|[seqlib](https://github.com/walaj/SeqLib)|-|1.2.0|<details><summary>Expand</summary>Description: C++ interface to HTSlib, BWA-MEM and Fermi. Homepage: https://github.com/walaj/SeqLib URL: https://github.com/walaj/SeqLib</details>|
	|[seqtk](https://github.com/lh3/seqtk/)|bio|1.3, 1.4|<details><summary>Expand</summary>Description: Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip. Homepage: https://github.com/lh3/seqtk/ URL: https://github.com/lh3/seqtk/</details>|
	|[shapeit](https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html)|-|2.r904|<details><summary>Expand</summary>Description: SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</details>|
	|[shapeit4](https://odelaneau.github.io/shapeit4/)|-|4.2.0, 4.2.1|<details><summary>Expand</summary>Description: SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. Homepage: https://odelaneau.github.io/shapeit4/ URL: https://odelaneau.github.io/shapeit4/</details>|
	|[shapeit5](https://odelaneau.github.io/shapeit5/)|-|5.1.1|<details><summary>Expand</summary>Description: SHAPEIT5 estimates haplotypes in large datasets, with a special focus on rare variants. Homepage: https://odelaneau.github.io/shapeit5/ URL: https://odelaneau.github.io/shapeit5/</details>|
	|[shasta](https://github.com/chanzuckerberg/shasta)|-|0.10.0|<details><summary>Expand</summary>Description: The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include: Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k 10). Homepage: https://github.com/chanzuckerberg/shasta URL: https://github.com/chanzuckerberg/shasta</details>|
	|[shengbte](http://www.shengbte.org/)|phys|1.1.1, 1.5.0|<details><summary>Expand</summary>Description: ShengBTE is a software package for solving the Boltzmann Transport Equation for phonons. Also installed is the 'thirdorder' package of Python scripts. Homepage: http://www.shengbte.org/ URL: http://www.shengbte.org/</details>|
	|[shrinkwrap](https://github.com/jonathonl/shrinkwrap)|-|1.2.0|<details><summary>Expand</summary>Description: A std::streambuf wrapper for compression formats. Homepage: https://github.com/jonathonl/shrinkwrap URL: https://github.com/jonathonl/shrinkwrap</details>|
	|[sibelia](http://bioinf.spbau.ru/en/sibelia)|-|3.0.7|<details><summary>Expand</summary>Description: Sibelia: A comparative genomics tool: It assists biologists in analysing the genomic variations that correlate with pathogens, or the genomic changes that help microorganisms adapt in different environments. Sibelia will also be helpful for the evolutionary and genome rearrangement studies for multiple strains of microorganisms. Homepage: http://bioinf.spbau.ru/en/sibelia URL: http://bioinf.spbau.ru/en/sibelia</details>|
	|[sickle](https://github.com/najoshi/sickle)|bio|1.33|<details><summary>Expand</summary>Description: Windowed Adaptive Trimming for fastq files using quality Homepage: https://github.com/najoshi/sickle URL: https://github.com/najoshi/sickle</details>|
	|[siesta](https://gitlab.com/siesta-project)|chem|4.1-MaX-3.0, 4.0.1, 4.1-b4, 4.1.5, 5.0|<details><summary>Expand</summary>Description: SIESTA is both a method and its computer program implementation, to perform efficient electronic structure calculations and ab initio molecular dynamics simulations of molecules and solids. Homepage: https://gitlab.com/siesta-project URL: https://gitlab.com/siesta-project</details>|
	|[signalp](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp)|bio|4.1f|<details><summary>Expand</summary>Description: SignalP predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive bacteria, Gram-negative bacteria, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp</details>|
	|[silix](https://lbbe-web.univ-lyon1.fr/fr/SiLix)|-|1.3.0|<details><summary>Expand</summary>Description: SiLiX implements an ultra-efficient algorithm for the clustering of homologous sequences, based on single transitive links (single linkage) with alignment coverage constraints. Homepage: https://lbbe-web.univ-lyon1.fr/fr/SiLix URL: https://lbbe-web.univ-lyon1.fr/fr/SiLix</details>|
	|[simbac](https://github.com/tbrown91/SimBac)|-|0.1a|<details><summary>Expand</summary>Description: SimBac simulates bacterial genomes with the clonal genealogy under a coalescent model with recombination. Homepage: https://github.com/tbrown91/SimBac URL: https://github.com/tbrown91/SimBac</details>|
	|[simbody](https://github.com/simbody/simbody)|-|3.7|<details><summary>Expand</summary>Description: Simbody is a high-performance, open-source toolkit for science- and engineering-quality simulation of articulated mechanisms, including biomechanical structures such as human and animal skeletons, mechanical systems like robots, vehicles, and machines, and anything else that can be described as a set of rigid bodies interconnected by joints, influenced by forces and motions, and restricted by constraints. Simbody includes a multibody dynamics library for modeling motion in generalized/internal coordinates in O(n) time. This is sometimes called a Featherstone-style physics engine. Homepage: https://github.com/simbody/simbody URL: https://github.com/simbody/simbody</details>|
	|[simka](https://gatb.inria.fr/software/simka/)|-|1.5.3|<details><summary>Expand</summary>Description: Simka is a de novo comparative metagenomics tool. Simka represents each dataset as a k-mer spectrum and compute several classical ecological distances between them. Homepage: https://gatb.inria.fr/software/simka/ URL: https://gatb.inria.fr/software/simka/</details>|
	|[simnibs](https://simnibs.github.io/simnibs)|-|4.1.0|<details><summary>Expand</summary>Description: SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation Homepage: https://simnibs.github.io/simnibs URL: https://simnibs.github.io/simnibs</details>|
	|[simple-dftd3](https://github.com/dftd3/simple-dftd3)|-|0.7.0, 1.0.0|<details><summary>Expand</summary>Description: reimplementation of the DFT-D3 program Homepage: https://github.com/dftd3/simple-dftd3 URL: https://github.com/dftd3/simple-dftd3</details>|
	|[singular](https://www.singular.uni-kl.de/)|-|4.2.1|<details><summary>Expand</summary>Description: Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory. Homepage: https://www.singular.uni-kl.de/ URL: https://www.singular.uni-kl.de/</details>|
	|[sionlib](https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html)|-|1.7.6|<details><summary>Expand</summary>Description: SIONlib is a scalable I/O library for parallel access to task-local files. The library not only supports writing and reading binary data to or from several thousands of processors into a single or a small number of physical files, but also provides global open and close functions to access SIONlib files in parallel. This package provides a stripped-down installation of SIONlib for use with performance tools (e.g., Score-P), with renamed symbols to avoid conflicts when an application using SIONlib itself is linked against a tool requiring a different SIONlib version. Homepage: https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html URL: https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html</details>|
	|[slepc](https://slepc.upv.es/)|-|3.14.2, 3.17.2, 3.19.2, 3.20.1, 3.21.2|<details><summary>Expand</summary>Description: SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. Homepage: https://slepc.upv.es/ URL: https://slepc.upv.es/</details>|
	|[slepc-complex](https://slepc.upv.es/)|-|3.20.1|<details><summary>Expand</summary>Description: SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. Homepage: https://slepc.upv.es/ URL: https://slepc.upv.es/</details>|
	|[slicer](https://www.slicer.org/)|-|4.11.20210226|<details><summary>Expand</summary>Description: 3D Slicer is an open source software platform for medical image informatics, image processing, and three-dimensional visualization. Built over two decades through support from the National Institutes of Health and a worldwide developer community, Slicer brings free, powerful cross-platform processing tools to physicians, researchers, and the general public. Homepage: https://www.slicer.org/ URL: https://www.slicer.org/</details>|
	|[slim](https://messerlab.org/slim)|-|3.4.0, 4.0.1, 4.3|<details><summary>Expand</summary>Description: SLiM is an evolutionary simulation framework that combines a powerful engine for population genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios. Homepage: https://messerlab.org/slim URL: https://messerlab.org/slim</details>|
	|[slurm-completion](https://slurm.schedmd.com/)|-|23.02.7|<details><summary>Expand</summary>Description: bash completion for Slurm client commands Homepage: https://slurm.schedmd.com/ URL: https://slurm.schedmd.com/</details>|
	|[smartdenovo](https://github.com/ruanjue/smartdenovo)|-|20180219|<details><summary>Expand</summary>Description: SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy. Homepage: https://github.com/ruanjue/smartdenovo URL: https://github.com/ruanjue/smartdenovo</details>|
	|[smithwaterman](https://github.com/ekg/smithwaterman)|-|20160702|<details><summary>Expand</summary>Description: smith-waterman-gotoh alignment algorithm. Homepage: https://github.com/ekg/smithwaterman URL: https://github.com/ekg/smithwaterman</details>|
	|[smrtlink-sequel2](https://www.pacb.com/support/software-downloads/)|-|13.1.0.221970|<details><summary>Expand</summary>Description: PacBio's open-source SMRT Analysis software suite is designed for use with Single Molecule, Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through the extensive set of APIs provided Homepage: https://www.pacb.com/support/software-downloads/ URL: https://www.pacb.com/support/software-downloads/</details>|
	|[snap](http://korflab.ucdavis.edu/software.html)|bio|2017-05-17|<details><summary>Expand</summary>Description: SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser. Homepage: http://korflab.ucdavis.edu/software.html URL: http://korflab.ucdavis.edu/software.html</details>|
	|[snappy](https://github.com/google/snappy)|tools|1.1.8|<details><summary>Expand</summary>Description: Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression. Homepage: https://github.com/google/snappy URL: https://github.com/google/snappy</details>|
	|[sniffles](https://github.com/fritzsedlazeck/Sniffles)|-|1.0.12b|<details><summary>Expand</summary>Description: Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+) using evidence from split-read alignments, high-mismatch regions, and coverage analysis. Please note the current version of Sniffles requires sorted output from BWA-MEM (use -M and -x parameter), Minimap2 (sam file with Cigar and MD string) or NGMLR. Homepage: https://github.com/fritzsedlazeck/Sniffles URL: https://github.com/fritzsedlazeck/Sniffles</details>|
	|[snp-sites](https://github.com/sanger-pathogens/snp-sites)|-|2.5.1|<details><summary>Expand</summary>Description: Extracting single nucleotide polymorphisms (SNPs) from a large whole genome alignment is now a routine task, but existing tools have failed to scale efficiently with the increased size of studies. These tools are slow, memory inefficient and are installed through non-standard procedures. We present SNP-sites which can rapidly extract SNPs from a multi-FASTA alignment using modest resources and can output results in multiple formats for downstream analysis. SNPs can be extracted from a 8.3 GB alignment file (1,842 taxa, 22,618 sites) in 267 seconds using 59 MB of RAM and 1 CPU core, making it feasible to run on modest computers. Homepage: https://github.com/sanger-pathogens/snp-sites URL: https://github.com/sanger-pathogens/snp-sites</details>|
	|[snpeff](https://pcingola.github.io/SnpEff/)|bio|5.0, 5.2|<details><summary>Expand</summary>Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes). Homepage: https://pcingola.github.io/SnpEff/ URL: https://pcingola.github.io/SnpEff/</details>|
	|[soapdenovo-trans](http://soap.genomics.org.cn/SOAPdenovo-Trans.html)|-|1.0.4|<details><summary>Expand</summary>Description: SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. Homepage: http://soap.genomics.org.cn/SOAPdenovo-Trans.html URL: http://soap.genomics.org.cn/SOAPdenovo-Trans.html</details>|
	|[soapdenovo2](http://soap.genomics.org.cn/index.html)|bio|r242|<details><summary>Expand</summary>Description: SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for human-sized genomes. The program is specially designed to assemble Illumina short reads. It creates new opportunities for building reference sequences and carrying out accurate analyses of unexplored genomes in a cost effective way. SOAPdenovo2 is the successor of SOAPdenovo. Homepage: http://soap.genomics.org.cn/index.html URL: http://soap.genomics.org.cn/index.html</details>|
	|[soci](http://soci.sourceforge.net/)|-|4.0.2, 4.0.3|<details><summary>Expand</summary>Description: SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the regular C++ code, staying entirely within the Standard C++. Homepage: http://soci.sourceforge.net/ URL: http://soci.sourceforge.net/</details>|
	|[sortmerna](http://bioinfo.lifl.fr/RNA/sortmerna/)|bio|4.2.0, 4.3.4, 4.3.6|<details><summary>Expand</summary>Description: SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads. Homepage: http://bioinfo.lifl.fr/RNA/sortmerna/ URL: http://bioinfo.lifl.fr/RNA/sortmerna/</details>|
	|[soundtouch](https://www.surina.net/soundtouch/)|-|2.3.3|<details><summary>Expand</summary>Description: SoundTouch is an open-source audio processing library for changing the Tempo, Pitch and Playback Rates of audio streams or audio files. The library additionally supports estimating stable beats-per-minute rates for audio tracks. Homepage: https://www.surina.net/soundtouch/ URL: https://www.surina.net/soundtouch/</details>|
	|[spades](http://cab.spbu.ru/software/spades/)|bio|3.14.1, 3.15.1, 3.15.3, 3.15.4, 4.0.0|<details><summary>Expand</summary>Description: Genome assembler for single-cell and isolates data sets Homepage: http://cab.spbu.ru/software/spades/ URL: http://cab.spbu.ru/software/spades/</details>|
	|[spark](http://spark.apache.org)|tools|3.0.0, 3.3.0|<details><summary>Expand</summary>Description: Spark is Hadoop MapReduce done in memory Homepage: http://spark.apache.org URL: http://spark.apache.org</details>|
	|[sparsehash](https://github.com/sparsehash/sparsehash)|-|2.0.4|<details><summary>Expand</summary>Description: An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed. Homepage: https://github.com/sparsehash/sparsehash URL: https://github.com/sparsehash/sparsehash</details>|
	|[spdlog](https://github.com/gabime/spdlog)|-|1.9.2, 1.11.0|<details><summary>Expand</summary>Description: Very fast, header-only/compiled, C++ logging library. Homepage: https://github.com/gabime/spdlog URL: https://github.com/gabime/spdlog</details>|
	|[spectra](https://spectralib.org/)|-|0.9.0, 1.0.1|<details><summary>Expand</summary>Description: Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library. Homepage: https://spectralib.org/ URL: https://spectralib.org/</details>|
	|[spglib](https://github.com/spglib/spglib)|chem|1.16.0, 2.0.2|<details><summary>Expand</summary>Description: Spglib is a C library for finding and handling crystal symmetries. Homepage: https://github.com/spglib/spglib URL: https://github.com/spglib/spglib</details>|
	|[spoa](https://github.com/rvaser/spoa)|-|3.4.0|<details><summary>Expand</summary>Description: Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm which is used to generate consensus sequences Homepage: https://github.com/rvaser/spoa URL: https://github.com/rvaser/spoa</details>|
	|[sprng](http://www.sprng.org/)|math|5.0|<details><summary>Expand</summary>Description: Scalable Parallel Pseudo Random Number Generators Library Homepage: http://www.sprng.org/ URL: http://www.sprng.org/</details>|
	|[sqlite](https://www.sqlite.org/)|-|3.36, 3.38.5, 3.43.1|<details><summary>Expand</summary>Description: SQLite: SQL Database Engine in a C Library Homepage: https://www.sqlite.org/ URL: https://www.sqlite.org/</details>|
	|[sra-toolkit](https://github.com/ncbi/sra-tools)|bio|2.10.8, 3.0.0, 3.0.9|<details><summary>Expand</summary>Description: The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to programmatically access data housed within SRA and convert it from the SRA format Homepage: https://github.com/ncbi/sra-tools URL: https://github.com/ncbi/sra-tools</details>|
	|[srprism](https://github.com/ncbi/SRPRISM)|-|3.1.2|<details><summary>Expand</summary>Description: Single Read Paired Read Indel Substitution Minimizer Homepage: https://github.com/ncbi/SRPRISM URL: https://github.com/ncbi/SRPRISM</details>|
	|[ssw](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library)|-|1.1, 1.2.5|<details><summary>Expand</summary>Description: SSW is a fast implementation of the Smith-Waterman algorithm, which uses the Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the instruction level. SSW library provides an API that can be flexibly used by programs written in C, C++ and other languages. We also provide a software that can do protein and genome alignment directly. Current version of our implementation is ~50 times faster than an ordinary Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and location heuristically. Homepage: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library URL: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library</details>|
	|[stacks](http://creskolab.uoregon.edu/stacks/)|bio|2.53, 2.55, 2.57, 2.59, 2.60, 2.62, 2.64, 2.66, 2.67|<details><summary>Expand</summary>Description: Stacks is a software pipeline for building loci from short-read sequences, such as those generated on the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting population genomics and phylogeography. Homepage: http://creskolab.uoregon.edu/stacks/ URL: http://creskolab.uoregon.edu/stacks/</details>|
	|[star](https://github.com/alexdobin/STAR)|bio|2.7.5a, 2.7.8a, 2.7.9a, 2.7.11a, 2.7.11b|<details><summary>Expand</summary>Description: STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays. Homepage: https://github.com/alexdobin/STAR URL: https://github.com/alexdobin/STAR</details>|
	|[star-fusion](https://github.com/STAR-Fusion/STAR-Fusion)|-|1.10.0|<details><summary>Expand</summary>Description: STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set. Homepage: https://github.com/STAR-Fusion/STAR-Fusion URL: https://github.com/STAR-Fusion/STAR-Fusion</details>|
	|[starccm](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010-R8, 16.02.008-R8, 16.02.009-R8, 16.04.007-R8, 16.04.012-R8, 16.06.008-R8, 17.02.007-R8, 17.02.008-R8, 17.04.007-R8, 17.04.008-R8, 17.06.007-R8, 17.06.008-R8, 18.02.008-R8, 18.04.008-R8, 18.06.006-R8, 19.04.007-R8|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses double precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[starccm-mixed](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010, 16.02.008, 16.02.009, 16.04.007, 16.04.012, 16.06.008, 17.02.007, 17.02.008, 17.04.007, 17.04.008, 17.06.007, 17.06.008, 18.02.008, 18.04.008, 18.06.006, 19.04.007|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses mixed precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[statistics-r](https://metacpan.org/pod/Statistics::R)|-|0.34|<details><summary>Expand</summary>Description: Perl interface with the R statistical program Homepage: https://metacpan.org/pod/Statistics::R URL: https://metacpan.org/pod/Statistics::R</details>|
	|[stringtie](https://ccb.jhu.edu/software/stringtie/)|bio|2.1.3, 2.1.5, 2.2.1, 2.2.3|<details><summary>Expand</summary>Description: StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts Homepage: https://ccb.jhu.edu/software/stringtie/ URL: https://ccb.jhu.edu/software/stringtie/</details>|
	|[structure](http://web.stanford.edu/group/pritchardlab/software/)|bio|2.3.4|<details><summary>Expand</summary>Description: The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. Homepage: http://web.stanford.edu/group/pritchardlab/software/ URL: http://web.stanford.edu/group/pritchardlab/software/</details>|
	|[su2](http://su2.stanford.edu)|math|7.0.8, 7.5.1|<details><summary>Expand</summary>Description: An open-source collection of software tools written in C++ for performing Partial Differential Equation (PDE) analysis and solving PDE-constrained optimization problems. The toolset is designed with computational fluid dynamics and aerodynamic shape optimization in mind. Homepage: http://su2.stanford.edu URL: http://su2.stanford.edu</details>|
	|[subread](http://subread.sourceforge.net/)|bio|2.0.1, 2.0.3, 2.0.6|<details><summary>Expand</summary>Description: High performance read alignment, quantification and mutation discovery Homepage: http://subread.sourceforge.net/ URL: http://subread.sourceforge.net/</details>|
	|[suitesparse](https://faculty.cse.tamu.edu/davis/suitesparse.html)|math|5.7.1, 5.10.1, 5.13.0, 7.6.0|<details><summary>Expand</summary>Description: SuiteSparse is a collection of libraries to manipulate sparse matrices. Homepage: https://faculty.cse.tamu.edu/davis/suitesparse.html URL: https://faculty.cse.tamu.edu/davis/suitesparse.html</details>|
	|[sumo](https://github.com/eclipse/sumo/)|-|1.7.0, 1.9.0, 1.9.2, 1.15.0|<details><summary>Expand</summary>Description: Eclipse Sumo is an open source, highly portable, microscopic and continuous traffic simulation package designed to handle large road networks. Homepage: https://github.com/eclipse/sumo/ URL: https://github.com/eclipse/sumo/</details>|
	|[sundials](https://computing.llnl.gov/projects/sundials)|-|2.7.0, 5.3.0, 5.7.0, 6.4.1, 6.6.0|<details><summary>Expand</summary>Description: SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers Homepage: https://computing.llnl.gov/projects/sundials URL: https://computing.llnl.gov/projects/sundials</details>|
	|[superlu](http://crd-legacy.lbl.gov/~xiaoye/SuperLU/)|math|5.2.1|<details><summary>Expand</summary>Description: SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines. Homepage: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/ URL: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/</details>|
	|[supernova](https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome)|bio|2.1.1|<details><summary>Expand</summary>Description: Supernova is delivered as a single, self-contained tar file that can be unpacked anywhere on your system. It bundles all of its own required software dependencies, which are pre-compiled to run on a range of Linux distributions. Homepage: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome URL: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome</details>|
	|[survivor](https://github.com/fritzsedlazeck/SURVIVOR)|-|1.0.7|<details><summary>Expand</summary>Description: SURVIVOR is a tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples, and includes various methods to reformat or summarize SVs. Homepage: https://github.com/fritzsedlazeck/SURVIVOR URL: https://github.com/fritzsedlazeck/SURVIVOR</details>|
	|[svaba](https://github.com/walaj/svaba)|-|1.1.0|<details><summary>Expand</summary>Description: Structural variation and indel detection by local assembly Homepage: https://github.com/walaj/svaba URL: https://github.com/walaj/svaba</details>|
	|[swat+](https://swatplus.gitbook.io)|-|60.5.1|<details><summary>Expand</summary>Description: The Soil and Water Assessment Tool (SWAT) is a small watershed to river basin-scale model used to simulate the quality and quantity of surface and ground water and predict the environmental impact of land use, land management practices, and climate change. In order to face present and future challenges in water resources modeling SWAT code has undergone major modifications over the past few years, resulting in SWAT+, a completely revised version of the model. SWAT+ provides a more flexible spatial representation of interactions and processes within a watershed. Homepage: https://swatplus.gitbook.io URL: https://swatplus.gitbook.io</details>|
	|[swi-prolog](https://www.swi-prolog.org/)|-|9.0.3|<details><summary>Expand</summary>Description: SWI-Prolog offers a comprehensive free Prolog environment. Homepage: https://www.swi-prolog.org/ URL: https://www.swi-prolog.org/</details>|
	|[swig](http://www.swig.org/)|-|4.0.1, 4.0.2, 4.1.1, 4.2.1|<details><summary>Expand</summary>Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. Homepage: http://www.swig.org/ URL: http://www.swig.org/</details>|
	|[symengine](https://github.com/symengine/symengine)|-|0.6.0, 0.8.1, 0.9.0, 0.10.1, 0.11.1, 0.11.2, 0.13.0, 0.14.0|<details><summary>Expand</summary>Description: SymEngine is a standalone fast C++ symbolic manipulation library. Homepage: https://github.com/symengine/symengine URL: https://github.com/symengine/symengine</details>|
	|[tabix](http://samtools.sourceforge.net)|bio|0.2.6|<details><summary>Expand</summary>Description: Generic indexer for TAB-delimited genome position files Homepage: http://samtools.sourceforge.net URL: http://samtools.sourceforge.net</details>|
	|[tabixpp](https://github.com/ekg/tabixpp)|-|1.1.0, 1.1.2|<details><summary>Expand</summary>Description: C++ wrapper to tabix indexer Homepage: https://github.com/ekg/tabixpp URL: https://github.com/ekg/tabixpp</details>|
	|[tau](http://tau.uoregon.edu)|tools|2.30.1|<details><summary>Expand</summary>Description: The TAU Performance System is a portable profiling and tracing toolkit for performance analysis of parallel programs written in Fortran, C, C++, Java, Python. Homepage: http://tau.uoregon.edu URL: http://tau.uoregon.edu</details>|
	|[taudem](https://github.com/dtarb/TauDEM/wiki)|-|5.3.8|<details><summary>Expand</summary>Description: TauDEM (Terrain Analysis Using Digital Elevation Models) is a suite of Digital Elevation Model (DEM) tools for the extraction and analysis of hydrologic information from topography as represented by a DEM. Homepage: https://github.com/dtarb/TauDEM/wiki URL: https://github.com/dtarb/TauDEM/wiki</details>|
	|[taxonkit](https://bioinf.shenwei.me/taxonkit/)|-|0.6.2|<details><summary>Expand</summary>Description: TaxonKit - A Cross-platform and Efficient NCBI Taxonomy Toolkit TaxonKit is a command-line toolkit for rapid manipulation of NCBI taxonomy data. It provides executable binary files for major operating systems including Windows, Linux, and Mac OS X, and can be directly used without any dependencies nor local database buiding. Homepage: https://bioinf.shenwei.me/taxonkit/ URL: https://bioinf.shenwei.me/taxonkit/</details>|
	|[tbb](https://github.com/oneapi-src/oneTBB)|tools|2020.2, 2021.8.0, 2021.10.0|<details><summary>Expand</summary>Description: Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability. Homepage: https://github.com/oneapi-src/oneTBB URL: https://github.com/oneapi-src/oneTBB</details>|
	|[tbl2asn](https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/)|-|25.8|<details><summary>Expand</summary>Description: Tbl2asn is a command-line program that automates the creation of sequence records for submission to GenBank Homepage: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/ URL: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/</details>|
	|[tblite](https://github.com/tblite/tblite)|-|0.3.0|<details><summary>Expand</summary>Description: Light-weight tight-binding framework Homepage: https://github.com/tblite/tblite URL: https://github.com/tblite/tblite</details>|
	|[telemac-mascaret](http://www.opentelemac.org)|-|8p5r0|<details><summary>Expand</summary>Description: TELEMAC-MASCARET is an integrated suite of solvers for use in the field of free-surface flow. Having been used in the context of many studies throughout the world, it has become one of the major standards in its field. Homepage: http://www.opentelemac.org URL: http://www.opentelemac.org</details>|
	|[tensorrt](https://developer.nvidia.com/tensorrt)|-|8.6.1.6|<details><summary>Expand</summary>Description: NVIDIA TensorRT is a platform for high-performance deep learning inference Homepage: https://developer.nvidia.com/tensorrt URL: https://developer.nvidia.com/tensorrt</details>|
	|[tesseract](https://github.com/tesseract-ocr/tesseract)|-|4.1.3, 5.0.1|<details><summary>Expand</summary>Description: Tesseract is an optical character recognition engine Homepage: https://github.com/tesseract-ocr/tesseract URL: https://github.com/tesseract-ocr/tesseract</details>|
	|[thrift](https://thrift.apache.org/)|-|0.13.0, 0.14.2, 0.16.0, 0.18.1, 0.19.0|<details><summary>Expand</summary>Description: The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages. Homepage: https://thrift.apache.org/ URL: https://thrift.apache.org/</details>|
	|[tiledb](https://github.com/TileDB-Inc/TileDB)|-|2.17.1|<details><summary>Expand</summary>Description: TileDB is a powerful engine for storing and accessing dense and sparse multi-dimensional arrays, which can help you model any complex data efficiently. It is an embeddable C++ library that works on Linux, macOS, and Windows. Homepage: https://github.com/TileDB-Inc/TileDB URL: https://github.com/TileDB-Inc/TileDB</details>|
	|[tmhmm](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm)|bio|2.0c|<details><summary>Expand</summary>Description: TMHMM 2.0 predicts transmembrane helices in proteins. The method is described in: Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes. Anders Krogh and Bjorn Larsson, Gunnar von Heijne, and Erik L.L. Sonnhammer. J. Mol. Biol. 305:567-580, 2001. and: A hidden Markov model for predicting transmembrane helices in protein sequences. Erik L.L. Sonnhammer, Gunnar von Heijne, and Anders Krogh. In J. Glasgow et al., eds.: Proc. of ISMB6, pages 175-182. AAAI Press, 1998. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm</details>|
	|[togl](https://sourceforge.net/projects/togl/)|-|2.0|<details><summary>Expand</summary>Description: A Tcl/Tk widget for OpenGL rendering. Homepage: https://sourceforge.net/projects/togl/ URL: https://sourceforge.net/projects/togl/</details>|
	|[tophat](http://ccb.jhu.edu/software/tophat/)|bio|2.1.2|<details><summary>Expand</summary>Description: TopHat is a fast splice junction mapper for RNA-Seq reads. Homepage: http://ccb.jhu.edu/software/tophat/ URL: http://ccb.jhu.edu/software/tophat/</details>|
	|[towhee](http://towhee.sourceforge.net/)|chem|8.2.3|<details><summary>Expand</summary>Description: Towhee is a Monte Carlo molecular simulation code originally designed for the prediction of fluid phase equilibria using atom-based force fields and the Gibbs ensemble with particular attention paid to algorithms addressing molecule conformation sampling. The code has subsequently been extended to several ensembles, many different force fields, and solid (or at least porous) phases. Homepage: http://towhee.sourceforge.net/ URL: http://towhee.sourceforge.net/</details>|
	|[tpp](http://ww.peptideatlas.org/)|-|5.2.0, 6.0.0, 6.2.0|<details><summary>Expand</summary>Description: The Trans-Proteomic Pipeline (TPP) is a collection of integrated tools for MS/MS proteomics, developed at the SPC. On cedar we can also provide TPP web interface (tpp_gui) per group upon user request Homepage: http://ww.peptideatlas.org/ URL: http://ww.peptideatlas.org/</details>|
	|[trans-abyss](http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/)|bio|2.0.1|<details><summary>Expand</summary>Description: de novo assembly of RNA-Seq data using ABySS Homepage: http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/ URL: http://www.bcgsc.ca/platform/bioinfo/software/trans-abyss/</details>|
	|[transdecoder](https://transdecoder.github.io/)|bio|5.5.0, 5.7.1|<details><summary>Expand</summary>Description: TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks. Homepage: https://transdecoder.github.io/ URL: https://transdecoder.github.io/</details>|
	|[transrate](http://hibberdlab.com/transrate/)|bio|1.0.3|<details><summary>Expand</summary>Description: Transrate is software for de-novo transcriptome assembly quality analysis. It examines your assembly in detail and compares it to experimental evidence such as the sequencing reads, reporting quality scores for contigs and assemblies. This allows you to choose between assemblers and parameters, filter out the bad contigs from an assembly, and help decide when to stop trying to improve the assembly. Homepage: http://hibberdlab.com/transrate/ URL: http://hibberdlab.com/transrate/</details>|
	|[travis-analyzer](http://www.travis-analyzer.de/)|-|220729|<details><summary>Expand</summary>Description: TRAVIS is a free tool for analyzing and visualizing trajectories from all kinds of Molecular Dynamics or Monte Carlo simulations. Homepage: http://www.travis-analyzer.de/ URL: http://www.travis-analyzer.de/</details>|
	|[treemix](http://bitbucket.org/nygcresearch/treemix)|-|1.13|<details><summary>Expand</summary>Description: TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations. Homepage: http://bitbucket.org/nygcresearch/treemix URL: http://bitbucket.org/nygcresearch/treemix</details>|
	|[trf](https://tandem.bu.edu/trf/trf.html)|bio|4.09.1|<details><summary>Expand</summary>Description: A tandem repeat in DNA is two or more adjacent, approximate copies of a pattern of nucleotides. Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences. In order to use the program, the user submits a sequence in FASTA format. There is no need to specify the pattern, the size of the pattern or any other parameter. The output consists of two files: a repeat table file and an alignment file. The repeat table, viewable in a web browser, contains information about each repeat, including its location, size, number of copies and nucleotide content. Clicking on the location indices for one of the table entries opens a second browser page that shows an alignment of the copies against a consensus pattern. The program is very fast, analyzing sequences on the order of .5Mb in just a few seconds. Submitted sequences may be of arbitrary length. Repeats with pattern size in the range from 1 to 2000 bases are detected. Homepage: https://tandem.bu.edu/trf/trf.html URL: https://tandem.bu.edu/trf/trf.html</details>|
	|[trilinos](https://trilinos.org)|tools|13.0.1, 13.2.0, 13.4.0, 13.4.1, 15.1.1|<details><summary>Expand</summary>Description: The Trilinos Project is an effort to develop algorithms and enabling technologies within an object-oriented software framework for the solution of large-scale, complex multi-physics engineering and scientific problems. A unique design feature of Trilinos is its focus on packages. Homepage: https://trilinos.org URL: https://trilinos.org</details>|
	|[trimal](http://trimal.cgenomics.org/)|bio|1.4|<details><summary>Expand</summary>Description: trimAl is a tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment Homepage: http://trimal.cgenomics.org/ URL: http://trimal.cgenomics.org/</details>|
	|[trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)|bio|0.39|<details><summary>Expand</summary>Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.The selection of trimming steps and their associated parameters are supplied on the command line. Homepage: http://www.usadellab.org/cms/?page=trimmomatic URL: http://www.usadellab.org/cms/?page=trimmomatic</details>|
	|[trinity](http://trinityrnaseq.github.io/)|bio|2.11.0, 2.12.0, 2.13.2, 2.14.0|<details><summary>Expand</summary>Description: Trinity represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads. Homepage: http://trinityrnaseq.github.io/ URL: http://trinityrnaseq.github.io/</details>|
	|[trinotate](https://trinotate.github.io/)|bio|3.2.1, 3.2.2, 4.0.0|<details><summary>Expand</summary>Description: Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms. Homepage: https://trinotate.github.io/ URL: https://trinotate.github.io/</details>|
	|[triqs](https://triqs.github.io/triqs)|-|3.1.0, 3.1.1, 3.3.1|<details><summary>Expand</summary>Description: TRIQS (Toolbox for Research on Interacting Quantum Systems) is a scientific project providing a set of C++ and Python libraries to develop new tools for the study of interacting quantum systems. Homepage: https://triqs.github.io/triqs URL: https://triqs.github.io/triqs</details>|
	|[trnascan-se](http://lowelab.ucsc.edu/)|bio|2.0.7, 2.0.12|<details><summary>Expand</summary>Description: tRNAscan-SE: An improved tool for transfer RNA detection, Patricia Chan and Todd Lowe, School of Engineering, University of California, Santa Cruz, CA. tRNAscan-SE was written in the PERL (version 5.0) script language. Input consists of DNA or RNA sequences in FASTA format. tRNA predictions are output in standard tabular or ACeDB format. tRNAscan-SE does no tRNA detection itself, but instead combines the strengths of three independent tRNA prediction programs by negotiating the flow of information between them, performing a limited amount of post-processing, and outputting the results in one of several formats. Homepage: http://lowelab.ucsc.edu/ URL: http://lowelab.ucsc.edu/</details>|
	|[ucc](https://www.openucx.org/)|-|1.0.0, 1.2.0, 1.3.0|<details><summary>Expand</summary>Description: UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and feature-rich for current and emerging programming models and runtimes. Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucc-cuda](https://www.openucx.org/)|-|1.0.0, 1.2.0, 1.3.0|<details><summary>Expand</summary>Description: UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and feature-rich for current and emerging programming models and runtimes. This module adds the UCC CUDA support. Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucx](https://www.openucx.org/)|-|1.8.0, 1.9.0, 1.10.0, 1.12.1, 1.14.1, 1.16.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications Homepage: https://www.openucx.org/ URL: https://www.openucx.org/</details>|
	|[ucx-cuda](http://www.openucx.org/)|-|1.12.1, 1.14.1, 1.16.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications This module adds the UCX CUDA support. Homepage: http://www.openucx.org/ URL: http://www.openucx.org/</details>|
	|[udunits](https://www.unidata.ucar.edu/software/udunits/)|tools|2.2.26, 2.2.28|<details><summary>Expand</summary>Description: UDUNITS supports conversion of unit specifications between formatted and binary forms, arithmetic manipulation of units, and conversion of values between compatible scales of measurement. Homepage: https://www.unidata.ucar.edu/software/udunits/ URL: https://www.unidata.ucar.edu/software/udunits/</details>|
	|[unixodbc](https://www.unixodbc.org)|-|2.3.9|<details><summary>Expand</summary>Description: unixODBC provides a uniform interface between application and database driver Homepage: https://www.unixodbc.org URL: https://www.unixodbc.org</details>|
	|[usd](https://graphics.pixar.com/usd/docs/index.html)|-|21.05|<details><summary>Expand</summary>Description: Universal Scene Description (USD) is an efficient, scalable system for authoring, reading, and streaming time-sampled scene description for interchange between graphics applications. Homepage: https://graphics.pixar.com/usd/docs/index.html URL: https://graphics.pixar.com/usd/docs/index.html</details>|
	|[valgrind-mpi](https://valgrind.org)|tools|3.16.1, 3.21.0|<details><summary>Expand</summary>Description: Valgrind: Debugging and profiling tools Homepage: https://valgrind.org URL: https://valgrind.org</details>|
	|[varscan](https://github.com/dkoboldt/varscan)|-|2.4.2, 2.4.6|<details><summary>Expand</summary>Description: Variant calling and somatic mutation/CNV detection for next-generation sequencing data Homepage: https://github.com/dkoboldt/varscan URL: https://github.com/dkoboldt/varscan</details>|
	|[vaspkit](https://vaspkit.com/)|-|1.5.1|<details><summary>Expand</summary>Description: VASPKIT aims at providing a powerful and user-friendly interface to perform high throughput analysis of various material properties from the raw calculated data using the widely-used VASP code. Homepage: https://vaspkit.com/ URL: https://vaspkit.com/</details>|
	|[vcflib](https://github.com/vcflib/vcflib)|-|1.0.1, 1.0.3, 1.0.9|<details><summary>Expand</summary>Description: vcflib provides methods to manipulate and interpret sequence variation as it can be described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended to concisely describe reference-indexed genetic variations between individuals. Homepage: https://github.com/vcflib/vcflib URL: https://github.com/vcflib/vcflib</details>|
	|[vcftools](https://vcftools.github.io)|bio|0.1.16|<details><summary>Expand</summary>Description: The aim of VCFtools is to provide easily accessible methods for working with complex genetic variation data in the form of VCF files. Homepage: https://vcftools.github.io URL: https://vcftools.github.io</details>|
	|[velvet](http://www.ebi.ac.uk/~zerbino/velvet/)|bio|1.2.10|<details><summary>Expand</summary>Description: Sequence assembler for very short reads Homepage: http://www.ebi.ac.uk/~zerbino/velvet/ URL: http://www.ebi.ac.uk/~zerbino/velvet/</details>|
	|[viennarna](http://www.tbi.univie.ac.at/RNA/)|bio|2.4.17, 2.5.0, 2.5.1|<details><summary>Expand</summary>Description: The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures. Homepage: http://www.tbi.univie.ac.at/RNA/ URL: http://www.tbi.univie.ac.at/RNA/</details>|
	|[virsorter](https://github.com/simroux/VirSorter)|-|1.0.6|<details><summary>Expand</summary>Description: VirSorter: mining viral signal from microbial genomic data. Homepage: https://github.com/simroux/VirSorter URL: https://github.com/simroux/VirSorter</details>|
	|[visit](https://wci.llnl.gov/simulation/computer-codes/visit)|vis|2.13.3, 3.2.1|<details><summary>Expand</summary>Description: VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool. Homepage: https://wci.llnl.gov/simulation/computer-codes/visit URL: https://wci.llnl.gov/simulation/computer-codes/visit</details>|
	|[vmatch](http://www.vmatch.de/)|-|2.3.1|<details><summary>Expand</summary>Description: Vmatch, a versatile software tool for efficiently solving large scale sequence matching tasks. Vmatch subsumes the software tool REPuter, but is much more general, with a very flexible user interface, and improved space and time requirements. Homepage: http://www.vmatch.de/ URL: http://www.vmatch.de/</details>|
	|[vmd](https://www.ks.uiuc.edu/Research/vmd)|vis|1.9.3, 1.9.4a43, 1.9.4a57|<details><summary>Expand</summary>Description: VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. Homepage: https://www.ks.uiuc.edu/Research/vmd URL: https://www.ks.uiuc.edu/Research/vmd</details>|
	|[vmtk](http://www.vmtk.org)|-|1.4.0|<details><summary>Expand</summary>Description: vmtk is a collection of libraries and tools for 3D reconstruction, geometric analysis, mesh generation and surface data analysis for image-based modeling of blood vessels. Homepage: http://www.vmtk.org URL: http://www.vmtk.org</details>|
	|[voro++](http://math.lbl.gov/voro++/)|math|0.4.6|<details><summary>Expand</summary>Description: Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles. Homepage: http://math.lbl.gov/voro++/ URL: http://math.lbl.gov/voro++/</details>|
	|[vsearch](https://github.com/torognes/vsearch)|bio|2.13.3, 2.13.4, 2.15.2, 2.21.1|<details><summary>Expand</summary>Description: VSEARCH supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, rereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering, conversion and merging of paired-end reads. Homepage: https://github.com/torognes/vsearch URL: https://github.com/torognes/vsearch</details>|
	|[vtk](http://www.vtk.org)|vis|8.2.0, 9.0.1, 9.1.0, 9.3.0|<details><summary>Expand</summary>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. Homepage: http://www.vtk.org URL: http://www.vtk.org</details>|
	|[vtk-mpi](http://www.vtk.org)|vis|9.0.1, 9.0.3|<details><summary>Expand</summary>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. Homepage: http://www.vtk.org URL: http://www.vtk.org</details>|
	|[vtune](https://software.intel.com/en-us/vtune)|tools|2020.1, 2022.2|<details><summary>Expand</summary>Description: Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran, Assembly and Java. Homepage: https://software.intel.com/en-us/vtune URL: https://software.intel.com/en-us/vtune</details>|
	|[w3emc](https://github.com/NOAA-EMC/NCEPLIBS-w3emc/)|-|2.11.0|<details><summary>Expand</summary>Description: This library contains Fortran decoder/encoder routines for GRIB edition 1. GRIdded Binary or General Regularly-distributed Information in Binary form (GRIB) is a World Meteorological Organization (WMO) standard for gridded data. This is part of the NCEPLIBS project. Homepage: hthttps://github.com/NOAA-EMC/NCEPLIBS-w3emc/ URL: hthttps://github.com/NOAA-EMC/NCEPLIBS-w3emc/</details>|
	|[wannier90](http://www.wannier.org/)|chem|1.2, 3.1.0|<details><summary>Expand</summary>Description: The Maximally-Localised Generalised Wannier Functions calculates the counterpart of localized molecular orbitals for crystalline systems from the wavefuction. Homepage: http://www.wannier.org/ URL: http://www.wannier.org/</details>|
	|[wannier90-abinit](http://www.wannier.org)|chem|2.0.1.1|<details><summary>Expand</summary>Description: A tool for obtaining maximally-localised Wannier functions Homepage: http://www.wannier.org URL: http://www.wannier.org</details>|
	|[wanniertools](http://www.wanniertools.com)|-|2.7.0|<details><summary>Expand</summary>Description: WannierTools is an open source software that studies the physical properties of given tight-binding model. Homepage: http://www.wanniertools.com URL: http://www.wanniertools.com</details>|
	|[wasp](https://code.ornl.gov/neams-workbench/wasp)|-|3.1.4, 4.0.3, 4.2.0|<details><summary>Expand</summary>Description: The Workbench Analysis Sequence Processor (WASP) was developed to streamline lexing, parsing, access, validation, and analysis of ascii text files. Homepage: https://code.ornl.gov/neams-workbench/wasp URL: https://code.ornl.gov/neams-workbench/wasp</details>|
	|[wcslib](https://www.atnf.csiro.au/people/mcalabre/WCS/)|-|8.3|<details><summary>Expand</summary>Description: The FITS "World Coordinate System" (WCS) standard defines keywords and usage that provide for the description of astronomical coordinate systems in a FITS image header. Homepage: https://www.atnf.csiro.au/people/mcalabre/WCS/ URL: https://www.atnf.csiro.au/people/mcalabre/WCS/</details>|
	|[wfa2](https://github.com/smarco/WFA2-lib)|-|2.3.4|<details><summary>Expand</summary>Description: The wavefront alignment (WFA) algorithm is an exact gap-affine algorithm that takes advantage of homologous regions between the sequences to accelerate the alignment process. Homepage: https://github.com/smarco/WFA2-lib URL: https://github.com/smarco/WFA2-lib</details>|
	|[wgrib2](http://www.cpc.noaa.gov/products/wesley/wgrib2/)|geo|3.0.0|<details><summary>Expand</summary>Description: A program to decode/manipulate GRIB-2 files. Homepage: http://www.cpc.noaa.gov/products/wesley/wgrib2/ URL: http://www.cpc.noaa.gov/products/wesley/wgrib2/</details>|
	|[winnowmap](https://github.com/marbl/Winnowmap)|-|2.03|<details><summary>Expand</summary>Description: Winnowmap is a long-read mapping algorithm, and a result of our exploration into superior minimizer sampling techniques. Homepage: https://github.com/marbl/Winnowmap URL: https://github.com/marbl/Winnowmap</details>|
	|[wps](http://www.wrf-model.org)|geo|4.1, 4.2|<details><summary>Expand</summary>Description: WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale numerical weather prediction system designed to serve both operational forecasting and atmospheric research needs. Homepage: http://www.wrf-model.org URL: http://www.wrf-model.org</details>|
	|[wrf](https://www.wrf-model.org)|geo|4.1.3, 4.2.1, 4.3.3, 4.4|<details><summary>Expand</summary>Description: The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale numerical weather prediction system designed to serve both operational forecasting and atmospheric research needs. Homepage: https://www.wrf-model.org URL: https://www.wrf-model.org</details>|
	|[wrf-cmaq](https://www.epa.gov/cmaq)|-|5.4|<details><summary>Expand</summary>Description: CMAQ ( Community Multiscale Air Quality Modeling System) is a suite of Fortran 90 programs that work in concert to estimate ozone, PM, toxic compounds, and deposition throughout the troposphere. The model requires significant computational resources to run and therefore requires a multi-processor computer running Linux. Homepage: https://www.epa.gov/cmaq URL: https://www.epa.gov/cmaq</details>|
	|[wrf-co2](http://www.wrf-model.org)|geo|4.2.1|<details><summary>Expand</summary>Description: The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale numerical weather prediction system designed to serve both operational forecasting and atmospheric research needs. Compiled with the -DCLWRFGHG option that turns on the time-evolving CO2 Homepage: http://www.wrf-model.org URL: http://www.wrf-model.org</details>|
	|[wtdbg2](https://github.com/ruanjue/wtdbg2)|-|2.5|<details><summary>Expand</summary>Description: Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy. Homepage: https://github.com/ruanjue/wtdbg2 URL: https://github.com/ruanjue/wtdbg2</details>|
	|[wxwidgets](https://www.wxwidgets.org)|tools|3.1.4, 3.1.7|<details><summary>Expand</summary>Description: wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI. Homepage: https://www.wxwidgets.org URL: https://www.wxwidgets.org</details>|
	|[xbraid](https://github.com/XBraid/xbraid)|-|3.1.0|<details><summary>Expand</summary>Description: XBraid is a parallel-in-time software package. It implements an optimal-scaling multigrid solver for the (non)linear systems that arise from the discretization of problems with evolutionary behavior. Homepage: https://github.com/XBraid/xbraid URL: https://github.com/XBraid/xbraid</details>|
	|[xcrysden](http://www.xcrysden.org/)|vis|1.5.60, 1.6.2|<details><summary>Expand</summary>Description: XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. Homepage: http://www.xcrysden.org/ URL: http://www.xcrysden.org/</details>|
	|[xdrfile](http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library)|tools|1.1.4|<details><summary>Expand</summary>Description: library for reading and writing xtc, edr and trr files Homepage: http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library URL: http://www.gromacs.org/Developer_Zone/Programming_Guide/XTC_Library</details>|
	|[xerces-c++](http://xerces.apache.org/xerces-c/)|tools|3.2.2|<details><summary>Expand</summary>Description: Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs. Homepage: http://xerces.apache.org/xerces-c/ URL: http://xerces.apache.org/xerces-c/</details>|
	|[xml-libxml](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)|-|2.0205, 2.0208|<details><summary>Expand</summary>Description: Perl binding for libxml2 Homepage: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod URL: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</details>|
	|[xmlf90](https://gitlab.com/siesta-project/libraries/xmlf90)|tools|1.5.4, 1.6.2|<details><summary>Expand</summary>Description: A fast XML parser and generator in Fortran Homepage: https://gitlab.com/siesta-project/libraries/xmlf90 URL: https://gitlab.com/siesta-project/libraries/xmlf90</details>|
	|[xtb](https://xtb-docs.readthedocs.io)|chem|6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.6.1, 6.7.1|<details><summary>Expand</summary>Description: xtb - An extended tight-binding semi-empirical program package. Homepage: https://xtb-docs.readthedocs.io URL: https://xtb-docs.readthedocs.io</details>|
	|[xtensor](https://github.com/xtensor-stack/xtensor)|-|0.24.2, 0.25.0|<details><summary>Expand</summary>Description: xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions. Homepage: https://github.com/xtensor-stack/xtensor URL: https://github.com/xtensor-stack/xtensor</details>|
	|[yambo](http://www.yambo-code.org)|chem|4.5.0, 5.0.4|<details><summary>Expand</summary>Description: Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics. Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf. Homepage: http://www.yambo-code.org URL: http://www.yambo-code.org</details>|
	|[yaml-cpp](https://github.com/jbeder/yaml-cpp)|-|0.7.0|<details><summary>Expand</summary>Description: yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec Homepage: https://github.com/jbeder/yaml-cpp URL: https://github.com/jbeder/yaml-cpp</details>|
	|[yaxt](https://www.dkrz.de/redmine/projects/yaxt)|tools|0.9.0, 0.9.1, 0.10.0|<details><summary>Expand</summary>Description: Yet Another eXchange Tool Homepage: https://www.dkrz.de/redmine/projects/yaxt URL: https://www.dkrz.de/redmine/projects/yaxt</details>|
	|[zeo++](https://www.zeoplusplus.org/)|-|0.3|<details><summary>Expand</summary>Description: Zeo++ is a software package for analysis of crystalline porous materials. Zeo++ can be used to perform geometry-based analysis of structure and topology of the void space inside a material, to assemble or alternate structures as well as to generate structure representations to be used in structure similarity calculations. Zeo++ can be used to either analyze a single structure or perform high-throughput analysis of a large database. Homepage: https://www.zeoplusplus.org/ URL: https://www.zeoplusplus.org/</details>|
	|[zoltan](https://sandialabs.github.io/Zoltan/)|-|3.901|<details><summary>Expand</summary>Description: Zoltan Dynamic Load Balancing and Graph Algorithm Toolkit Homepage: https://sandialabs.github.io/Zoltan/ URL: https://sandialabs.github.io/Zoltan/</details>|

=== "AVX"

	|**Module**|**Type**|**Versions**|**Description**|
	|:----:|:----:|:----:|:----:|
	|[abaqus](http://www.simulia.com/products/abaqus_fea.html)|phys|2021|<details><summary>Expand</summary>Description: Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA. Homepage: http://www.simulia.com/products/abaqus_fea.html URL: http://www.simulia.com/products/abaqus_fea.html</details>|
	|[abyss](https://www.bcgsc.ca/platform/bioinfo/software/abyss)|bio|2.2.5|<details><summary>Expand</summary>Description: Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler Homepage: https://www.bcgsc.ca/platform/bioinfo/software/abyss URL: https://www.bcgsc.ca/platform/bioinfo/software/abyss</details>|
	|[actc](https://sourceforge.net/projects/actc)|-|1.1|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[admixture](http://software.genetics.ucla.edu/admixture)|bio|1.3.0|<details><summary>Expand</summary>Description: ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. Homepage: http://software.genetics.ucla.edu/admixture URL: http://software.genetics.ucla.edu/admixture</details>|
	|[advisor](https://software.intel.com/intel-advisor-xe)|tools|2020.3|<details><summary>Expand</summary>Description: Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain Homepage: https://software.intel.com/intel-advisor-xe URL: https://software.intel.com/intel-advisor-xe</details>|
	|[annovar](http://annovar.openbioinformatics.org/en/latest/)|bio|20191024|<details><summary>Expand</summary>Description: ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). Homepage: http://annovar.openbioinformatics.org/en/latest/ URL: http://annovar.openbioinformatics.org/en/latest/</details>|
	|[anserini](https://github.com/castorini/anserini)|-|0.9.4|<details><summary>Expand</summary>Description: An open-source information retreival toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h Homepage: https://github.com/castorini/anserini URL: https://github.com/castorini/anserini</details>|
	|[ansys](http://www.ansys.com)|phys|2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ansysedt](http://www.ansys.com)|-|2021R2, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ant](https://ant.apache.org/)|tools|1.9.15, 1.10.8|<details><summary>Expand</summary>Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. Homepage: https://ant.apache.org/ URL: https://ant.apache.org/</details>|
	|[antlr](https://www.antlr2.org/)|-|2.7.7|<details><summary>Expand</summary>Description: ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. Homepage: https://www.antlr2.org/ URL: https://www.antlr2.org/</details>|
	|[apptainer](https://apptainer.org)|-|1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4|<details><summary>Expand</summary>Description: Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. Homepage: https://apptainer.org URL: https://apptainer.org</details>|
	|[aragorn](http://mbio-serv2.mbioekol.lu.se/ARAGORN/)|-|1.2.38|<details><summary>Expand</summary>Description: a program to detect tRNA genes and tmRNA genes in nucleotide sequences Homepage: http://mbio-serv2.mbioekol.lu.se/ARAGORN/ URL: http://mbio-serv2.mbioekol.lu.se/ARAGORN/</details>|
	|[arrow](https://arrow.apache.org)|tools|0.16.0, 10.0.1, 12.0.1, 13.0.0|<details><summary>Expand</summary>Description: Apache Arrow is a cross-language development platform for in-memory data. Homepage: https://arrow.apache.org URL: https://arrow.apache.org</details>|
	|[ascp](http://downloads.asperasoft.com/)|tools|3.5.4|<details><summary>Expand</summary>Description: Aspera ASCP is a high speed file transfer software Homepage: http://downloads.asperasoft.com/ URL: http://downloads.asperasoft.com/</details>|
	|[autodock_vina](http://vina.scripps.edu/index.html)|chem|1.1.2|<details><summary>Expand</summary>Description: AutoDock Vina is an open-source program for doing molecular docking. Homepage: http://vina.scripps.edu/index.html URL: http://vina.scripps.edu/index.html</details>|
	|[bamtools](https://github.com/pezmaster31/bamtools)|bio|2.5.1|<details><summary>Expand</summary>Description: BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files. Homepage: https://github.com/pezmaster31/bamtools URL: https://github.com/pezmaster31/bamtools</details>|
	|[bbmap](https://sourceforge.net/projects/bbmap/)|bio|38.86|<details><summary>Expand</summary>Description: BBMap short read aligner, and other bioinformatic tools. Homepage: https://sourceforge.net/projects/bbmap/ URL: https://sourceforge.net/projects/bbmap/</details>|
	|[bcftools](https://www.htslib.org/)|bio|1.13|<details><summary>Expand</summary>Description: Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[beagle](https://faculty.washington.edu/browning/beagle/beagle.html)|-|5.4|<details><summary>Expand</summary>Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers. Homepage: https://faculty.washington.edu/browning/beagle/beagle.html URL: https://faculty.washington.edu/browning/beagle/beagle.html</details>|
	|[bedtools](https://github.com/arq5x/bedtools2)|bio|2.30.0|<details><summary>Expand</summary>Description: The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM. Homepage: https://github.com/arq5x/bedtools2 URL: https://github.com/arq5x/bedtools2</details>|
	|[beef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|chem|0.1.1|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: http://suncat.stanford.edu/facility/software/functional URL: http://suncat.stanford.edu/facility/software/functional</details>|
	|[bio-searchio-hmmer](http://hmmer.org.)|-|1.7.3|<details><summary>Expand</summary>Description: Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org. Homepage: https://metacpan.org/pod/Bio::SearchIO::hmmer3 URL: https://metacpan.org/pod/Bio::SearchIO::hmmer3</details>|
	|[bioawk](https://github.com/lh3/bioawk)|-|1.0|<details><summary>Expand</summary>Description: Bioawk is an extension to Brian Kernighan's awk, adding the support of several common biological data formats, including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. Homepage: https://github.com/lh3/bioawk URL: https://github.com/lh3/bioawk</details>|
	|[biobloomtools](https://github.com/bcgsc/biobloom)|-|2.3.3|<details><summary>Expand</summary>Description: BioBloom Tools (BBT) provides the means to create filters for a given reference and then to categorize sequences. This methodology is faster than alignment but does not provide mapping locations. BBT was initially intended to be used for pre-processing and QC applications like contamination detection, but is flexible to accommodate other purposes. This tool is intended to be a pipeline component to replace costly alignment steps. Homepage: https://github.com/bcgsc/biobloom URL: https://github.com/bcgsc/biobloom</details>|
	|[bioperl](https://bioperl.org/)|bio|1.7.7, 1.7.8|<details><summary>Expand</summary>Description: Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects. Homepage: https://bioperl.org/ URL: https://bioperl.org/</details>|
	|[blast](http://blast.ncbi.nlm.nih.gov/)|bio|2.2.26|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: http://blast.ncbi.nlm.nih.gov/ URL: http://blast.ncbi.nlm.nih.gov/</details>|
	|[blast+](https://blast.ncbi.nlm.nih.gov/)|bio|2.12.0, 2.14.0|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: https://blast.ncbi.nlm.nih.gov/ URL: https://blast.ncbi.nlm.nih.gov/</details>|
	|[blender](https://www.blender.org/)|vis|3.6.0|<details><summary>Expand</summary>Description: Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. Homepage: https://www.blender.org/ URL: https://www.blender.org/</details>|
	|[boost](https://www.boost.org/)|tools|1.72.0, 1.80.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-mpi](https://www.boost.org/)|tools|1.72.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)|bio|2.4.1, 2.5.1|<details><summary>Expand</summary>Description: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. Homepage: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml URL: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</details>|
	|[bustools](https://github.com/BUStools/bustools)|-|0.40.0|<details><summary>Expand</summary>Description: bustools is a program for manipulating BUS files for single cell RNA-Seq datasets. It can be used to error correct barcodes, collapse UMIs, produce gene count or transcript compatibility count matrices, and is useful for many other tasks. See the kallisto bustools website for examples and instructions on how to use bustools as part of a single-cell RNA-seq workflow. Homepage: https://github.com/BUStools/bustools URL: https://github.com/BUStools/bustools</details>|
	|[bwa](http://bio-bwa.sourceforge.net/)|bio|0.7.17|<details><summary>Expand</summary>Description: Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. Homepage: http://bio-bwa.sourceforge.net/ URL: http://bio-bwa.sourceforge.net/</details>|
	|[cafe5](https://github.com/hahnlab/CAFE5)|-|5.1.0|<details><summary>Expand</summary>Description: Software for Computational Analysis of gene Family Evolution The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa. Homepage: https://github.com/hahnlab/CAFE5 URL: https://github.com/hahnlab/CAFE5</details>|
	|[canu](https://github.com/marbl/canu)|bio|2.2|<details><summary>Expand</summary>Description: Canu is a fork of the Celera Assembler, designed for high-noise single-molecule sequencing (such as the PacBio RS II or Oxford Nanopore MinION). Canu is a hierarchical assembly pipeline which runs in four steps: Detect overlaps in high-noise sequences using MHAP Generate corrected sequence consensus Trim corrected sequences Assemble trimmed corrected sequences Homepage: https://github.com/marbl/canu URL: https://github.com/marbl/canu</details>|
	|[cap3](http://seq.cs.iastate.edu/)|bio|20151002|<details><summary>Expand</summary>Description: CAP3 assembly program Homepage: http://seq.cs.iastate.edu/ URL: http://seq.cs.iastate.edu/</details>|
	|[capnproto](https://capnproto.org)|-|0.7.0|<details><summary>Expand</summary>Description: Cap'n Proto is an insanely fast data interchange format and capability-based RPC system. Homepage: https://capnproto.org URL: https://capnproto.org</details>|
	|[catch2](https://github.com/catchorg/Catch2)|-|2.11.0|<details><summary>Expand</summary>Description: A modern, C++-native, header-only, test framework for unit-tests, TDD and BDD - using C++11, C++14, C++17 and later (or C++03 on the Catch1.x branch) Homepage: https://github.com/catchorg/Catch2 URL: https://github.com/catchorg/Catch2</details>|
	|[ccsm](http://www.cesm.ucar.edu/models/ccsm4.0/)|geo|4_0_a02|<details><summary>Expand</summary>Description: The Community Climate System Model (CCSM) is a coupled climate model for simulating the earth's climate system. Composed of four separate models simultaneously simulating the earth's atmosphere, ocean, land surface and sea-ice, and one central coupler component, the CCSM allows researchers to conduct fundamental research into the earth's past, present and future climate states. Homepage: http://www.cesm.ucar.edu/models/ccsm4.0/ URL: http://www.cesm.ucar.edu/models/ccsm4.0/</details>|
	|[cd-hit](http://weizhongli-lab.org/cd-hit/)|bio|4.8.1|<details><summary>Expand</summary>Description: CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences. Homepage: http://weizhongli-lab.org/cd-hit/ URL: http://weizhongli-lab.org/cd-hit/</details>|
	|[cdo](https://code.zmaw.de/projects/cdo)|geo|1.9.8, 1.9.10|<details><summary>Expand</summary>Description: CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data. Homepage: https://code.zmaw.de/projects/cdo URL: https://code.zmaw.de/projects/cdo</details>|
	|[cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)|bio|2.1.0|<details><summary>Expand</summary>Description: Cell Ranger is a set of analysis pipelines that process Chromium single cell 3-RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis. Cell Ranger 1.2 and later support libraries generated by the Chromium Single Cell 3i-v1 and v2 reagent kits, whereas Cell Ranger 1.1 and earlier do not support v2 libraries. Cell Ranger includes four main gene expression pipelines: - cellranger mkfastq wraps Illumina's bcl2fastq to correctly demultiplex Chromium-prepared sequencing samples and to convert barcode and read data to FASTQ files. - cellranger count takes FASTQ files from cellranger mkfastq and performs alignment, filtering, and UMI counting. - cellranger aggr aggregates outputs from multiple runs of cellranger count, normalizing those runs to the same sequencing depth and then recomputing the gene-barcode matrices and analysis on the combined data. - cellranger reanalyze takes gene-barcode matrices produced by cellranger count or cellranger aggr and reruns the dimensionality reduction, clustering, and gene expression algorithms using tunable parameter settings. These pipelines combine Chromium-specific algorithms with the widely used RNA-seq aligner STAR. Output is delivered in standard BAM, MEX, CSV, HDF5 and HTML formats that are augmented with cellular information. Homepage: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome URL: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</details>|
	|[cereal](http://uscilab.github.io/cereal/)|-|1.3.0|<details><summary>Expand</summary>Description: cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or used standalone. Homepage: http://uscilab.github.io/cereal/ URL: http://uscilab.github.io/cereal/</details>|
	|[cesm](http://www.cesm.ucar.edu/models/cesm2/)|geo|2.1.3|<details><summary>Expand</summary>Description: The Community Earth System Model (CESM) is a fully-coupled, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states. Homepage: http://www.cesm.ucar.edu/models/cesm2/ URL: http://www.cesm.ucar.edu/models/cesm2/</details>|
	|[cfitsio](https://heasarc.gsfc.nasa.gov/fitsio/)|vis|3.48, 4.1.0|<details><summary>Expand</summary>Description: CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format. Homepage: https://heasarc.gsfc.nasa.gov/fitsio/ URL: https://heasarc.gsfc.nasa.gov/fitsio/</details>|
	|[cgal](https://www.cgal.org/)|math|4.14.3, 5.5.2|<details><summary>Expand</summary>Description: The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library. Homepage: https://www.cgal.org/ URL: https://www.cgal.org/</details>|
	|[cmake](https://www.cmake.org)|tools|3.18.4, 3.20.1, 3.21.4, 3.22.1, 3.23.1, 3.27.7|<details><summary>Expand</summary>Description: CMake, the cross-platform, open-source build system. CMake is a family of tools designed to build, test and package software. Homepage: https://www.cmake.org URL: https://www.cmake.org</details>|
	|[code-server](https://github.com/cdr/code-server)|-|3.5.0, 3.12.0|<details><summary>Expand</summary>Description: Run VS Code on any machine anywhere and access it in the browser. Homepage: https://github.com/cdr/code-server URL: https://github.com/cdr/code-server</details>|
	|[coinmp](https://projects.coin-or.org/CoinMP)|-|1.8.4|<details><summary>Expand</summary>Description: CoinMP is a C-API library that supports most of the functionality of CLP (Coin LP), CBC (Coin Branch-and-Cut), and CGL (Cut Generation Library) projects. Homepage: https://projects.coin-or.org/CoinMP URL: https://projects.coin-or.org/CoinMP</details>|
	|[comsol](https://www.comsol.com)|phys|5.6, 6.0.0.405, 6.1, 6.1.0.357, 6.2|<details><summary>Expand</summary>Description: COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems. Homepage: https://www.comsol.com URL: https://www.comsol.com</details>|
	|[corset](https://github.com/Oshlack/Corset/wiki)|bio|1.09|<details><summary>Expand</summary>Description: Corset is a command-line software program to go from a de novo transcriptome assembly to gene-level counts. Our software takes a set of reads that have been multi-mapped to the transcriptome (where multiple alignments per read were reported) and hierarchically clusters the transcripts based on the proportion of shared reads and expression patterns. It will report the clusters and gene-level counts for each sample, which are easily tested for differential expression with count based tools such as edgeR and DESeq. Homepage: https://github.com/Oshlack/Corset/wiki URL: https://github.com/Oshlack/Corset/wiki</details>|
	|[cppzmq](https://github.com/zeromq/cppzmq)|-|4.7.1|<details><summary>Expand</summary>Description: cppzmq is a C++ binding for libzmq Homepage: https://github.com/zeromq/cppzmq URL: https://github.com/zeromq/cppzmq</details>|
	|[cromwell](http://cromwell.readthedocs.io/)|tools|58|<details><summary>Expand</summary>Description: Scientific workflow engine designed for simplicity and scalability. Trivially transition between one off use cases to massive scale production environments http://cromwell.readthedocs.io/ Homepage: https://cromwell.readthedocs.io/ URL: https://cromwell.readthedocs.io/</details>|
	|[cst](https://www.cst.com/)|phys|2020|<details><summary>Expand</summary>Description: CST offers accurate, efficient computational solutions for electromagnetic design and analysis. CST simulation is user-friendly and enables users to choose the most appropriate method for the design and optimization of devices operating in a wide range of frequencies. Please note that we installed only Graphic frontend and command line interface. You need to provide your own license server in order to be able to run CST. Homepage: https://www.cst.com/ URL: https://www.cst.com/</details>|
	|[csvtk](https://github.com/shenwei356/csvtk)|-|0.23.0|<details><summary>Expand</summary>Description: Similar to FASTA/Q format in field of Bioinformatics, CSV/TSV formats are basic and ubiquitous file formats in both Bioinformatics and data science. People usually use spreadsheet software like MS Excel to process table data. However this is all by clicking and typing, which is not automated and is time-consuming to repeat, especially when you want to apply similar operations with different datasets or purposes. You can also accomplish some CSV/TSV manipulations using shell commands, but more code is needed to handle the header line. Shell commands do not support selecting columns with column names either. csvtk is convenient for rapid data investigation and also easy to integrate into analysis pipelines. It could save you lots of time in (not) writing Python/R scripts. Homepage: https://github.com/shenwei356/csvtk URL: https://github.com/shenwei356/csvtk</details>|
	|[cuda](https://developer.nvidia.com/cuda-toolkit)|tools|10.2|<details><summary>Expand</summary>Description: CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. Homepage: https://developer.nvidia.com/cuda-toolkit URL: https://developer.nvidia.com/cuda-toolkit</details>|
	|[cudnn](https://developer.nvidia.com/cudnn)|math|8.0.3, 8.2.0, 8.6.0.163, 8.7.0.84, 8.9.5.29|<details><summary>Expand</summary>Description: The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. Homepage: https://developer.nvidia.com/cudnn URL: https://developer.nvidia.com/cudnn</details>|
	|[cufflinks](http://cole-trapnell-lab.github.io/cufflinks/)|bio|2.2.1|<details><summary>Expand</summary>Description: Transcript assembly, differential expression, and differential regulation for RNA-Seq Homepage: http://cole-trapnell-lab.github.io/cufflinks/ URL: http://cole-trapnell-lab.github.io/cufflinks/</details>|
	|[cusparselt](https://docs.nvidia.com/cuda/cusparselt/index.html)|-|0.4.0.7|<details><summary>Expand</summary>Description: NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix Homepage: https://docs.nvidia.com/cuda/cusparselt/index.html URL: https://docs.nvidia.com/cuda/cusparselt/index.html</details>|
	|[custom-ctypes](https://github.com/ComputeCanada/custom_ctypes/)|-|1.1|<details><summary>Expand</summary>Description: This repository is to try to fix discovery of libraries with Python's ctypes module. It changes the behavior of find_library to return absolute paths to shared objects rather than just the names. Homepage: https://github.com/ComputeCanada/custom_ctypes/ URL: https://github.com/ComputeCanada/custom_ctypes/</details>|
	|[cutensor](https://developer.nvidia.com/cutensor)|-|1.5.0.3, 1.7.0.1|<details><summary>Expand</summary>Description: cuTENSOR is a high-performance CUDA library for tensor primitives. Homepage: https://developer.nvidia.com/cutensor URL: https://developer.nvidia.com/cutensor</details>|
	|[cvit](https://sourceforge.net/projects/cvit/)|-|1.2.1|<details><summary>Expand</summary>Description: A genomic linkage feature visualization tools based on Perl. Homepage: https://sourceforge.net/projects/cvit/ URL: https://sourceforge.net/projects/cvit/</details>|
	|[ddt-cpu](http://www.allinea.com)|tools|20.2, 22.0.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining Arm MAP and Arm DDT. Homepage: http://www.allinea.com URL: http://www.allinea.com</details>|
	|[ddt-gpu](http://www.allinea.com)|tools|20.2, 22.0.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining Arm MAP and Arm DDT, with support for CUDA. Homepage: http://www.allinea.com URL: http://www.allinea.com</details>|
	|[delly](https://github.com/dellytools/delly)|-|0.8.5|<details><summary>Expand</summary>Description: Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. Homepage: https://github.com/dellytools/delly URL: https://github.com/dellytools/delly</details>|
	|[diamond](https://github.com/bbuchfink/diamond)|bio|2.1.6|<details><summary>Expand</summary>Description: Accelerated BLAST compatible local sequence aligner Homepage: https://github.com/bbuchfink/diamond URL: https://github.com/bbuchfink/diamond</details>|
	|[dl_monte](https://gitlab.com/dl_monte)|-|2.07|<details><summary>Expand</summary>Description: General-purpose Monte Carlo molecular simulation Homepage: https://gitlab.com/dl_monte URL: https://gitlab.com/dl_monte</details>|
	|[dorado](https://github.com/nanoporetech/dorado)|-|0.2.1, 0.2.2, 0.3.0|<details><summary>Expand</summary>Description: Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads. Homepage: https://github.com/nanoporetech/dorado URL: https://github.com/nanoporetech/dorado</details>|
	|[dotnet-core](https://www.microsoft.com/net/)|-|3.1.8, 5.0.12, 6.0.0|<details><summary>Expand</summary>Description: .NET is a free, cross-platform, open source developer platform for building many different types of applications. With .NET, you can use multiple languages, editors, and libraries to build for web, mobile, desktop, gaming, and IoT. Contains the SDK and the Runtime. Homepage: https://www.microsoft.com/net/ URL: https://www.microsoft.com/net/</details>|
	|[double-conversion](https://github.com/google/double-conversion)|-|3.1.5|<details><summary>Expand</summary>Description: Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles. Homepage: https://github.com/google/double-conversion URL: https://github.com/google/double-conversion</details>|
	|[dssp](https://swift.cmbi.umcn.nl/gv/dssp/)|chem|3.1.4|<details><summary>Expand</summary>Description: The DSSP algorithm is the standard method for assigning secondary structure to the amino acids of a protein, given the atomic-resolution coordinates of the protein. If you use DSSP, please quote these two articles: 1) A series of PDB related databases for everyday needs. Wouter G Touw, Coos Baakman, Jon Black, Tim AH te Beek, E Krieger, Robbie P Joosten, Gert Vriend. Nucleic Acids Research 2015 January; 43(Database issue): D364-D368. 2) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. Kabsch W, Sander C, Biopolymers. 1983 22 2577-2637. PMID: 6667333; UI: 84128824. Homepage: https://swift.cmbi.umcn.nl/gv/dssp/ URL: https://swift.cmbi.umcn.nl/gv/dssp/</details>|
	|[eccodes](https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home)|geo|2.15.0, 2.19.0, 2.21.0, 2.22.1|<details><summary>Expand</summary>Description: ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding). Homepage: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home URL: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</details>|
	|[eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page)|math|3.3.7|<details><summary>Expand</summary>Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms. Homepage: https://eigen.tuxfamily.org/index.php?title=Main_Page URL: https://eigen.tuxfamily.org/index.php?title=Main_Page</details>|
	|[elpa](https://elpa.rzg.mpg.de)|math|2020.05.001|<details><summary>Expand</summary>Description: Eigenvalue SoLvers for Petaflop-Applications . Homepage: https://elpa.rzg.mpg.de URL: https://elpa.rzg.mpg.de</details>|
	|[emboss](https://emboss.sourceforge.net/)|bio|6.6.0|<details><summary>Expand</summary>Description: EMBOSS is 'The European Molecular Biology Open Software Suite' . EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology (e.g. EMBnet) user community. Homepage: https://emboss.sourceforge.net/ URL: https://emboss.sourceforge.net/</details>|
	|[energyplus](https://energyplus.net/downloads)|-|9.3.0, 23.2.0|<details><summary>Expand</summary>Description: EnergyPlus is a whole building energy simulation program that engineers, architects, and researchers use to model both energy consumptionfor heating, cooling, ventilation, lighting and plug and process loadsand water use in buildings. Homepage: https://energyplus.net/downloads URL: https://energyplus.net/downloads</details>|
	|[expat](https://libexpat.github.io)|tools|2.2.9, 2.2.10, 2.4.8|<details><summary>Expand</summary>Description: Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags) Homepage: https://libexpat.github.io URL: https://libexpat.github.io</details>|
	|[falcon](https://github.com/PacificBiosciences/FALCON)|bio|1.8.8|<details><summary>Expand</summary>Description: Falcon: a set of tools for fast aligning long reads for consensus and assembly Homepage: https://github.com/PacificBiosciences/FALCON URL: https://github.com/PacificBiosciences/FALCON</details>|
	|[fastahack](https://github.com/ekg/fastahack)|-|1.0.0|<details><summary>Expand</summary>Description: Utilities for indexing and sequence extraction from FASTA files. Homepage: https://github.com/ekg/fastahack URL: https://github.com/ekg/fastahack</details>|
	|[fastani](https://github.com/ParBLiSS/FastANI)|-|1.32|<details><summary>Expand</summary>Description: FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies. Homepage: https://github.com/ParBLiSS/FastANI URL: https://github.com/ParBLiSS/FastANI</details>|
	|[fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)|bio|0.11.9, 0.12.0|<details><summary>Expand</summary>Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline. Homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</details>|
	|[fastsimcoal2](http://cmpg.unibe.ch/software/fastsimcoal2/)|bio|2.6.0.3, 2.7.0.9|<details><summary>Expand</summary>Description: While preserving all the simulation flexibility of simcoal2, fastsimcoal is now implemented under a faster continous-time sequential Markovian coalescent approximation, allowing it to efficiently generate genetic diversity for different types of markers along large genomic regions, for both present or ancient samples. It includes a parameter sampler allowing its integration into Bayesian or likelihood parameter estimation procedure. fastsimcoal can handle very complex evolutionary scenarios including an arbitrary migration matrix between samples, historical events allowing for population resize, population fusion and fission, admixture events, changes in migration matrix, or changes in population growth rates. The time of sampling can be specified independently for each sample, allowing for serial sampling in the same or in different populations. Homepage: http://cmpg.unibe.ch/software/fastsimcoal2/ URL: http://cmpg.unibe.ch/software/fastsimcoal2/</details>|
	|[fasttree](http://www.microbesonline.org/fasttree/)|bio|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fasttree-double](http://www.microbesonline.org/fasttree/)|-|2.1.11|<details><summary>Expand</summary>Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of time and memory. Homepage: http://www.microbesonline.org/fasttree/ URL: http://www.microbesonline.org/fasttree/</details>|
	|[fermi-lite](https://github.com/lh3/fermi-lite)|-|20190320|<details><summary>Expand</summary>Description: Standalone C library for assembling Illumina short reads in small regions. Homepage: https://github.com/lh3/fermi-lite URL: https://github.com/lh3/fermi-lite</details>|
	|[fftw](http://www.fftw.org)|math|3.3.8|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: http://www.fftw.org URL: http://www.fftw.org</details>|
	|[fftw-mpi](http://www.fftw.org)|math|3.3.8|<details><summary>Expand</summary>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data. Homepage: http://www.fftw.org URL: http://www.fftw.org</details>|
	|[filevercmp](https://github.com/ekg/filevercmp)|-|20191210|<details><summary>Expand</summary>Description: filevercmp function as in sort --version-sort. Homepage: https://github.com/ekg/filevercmp URL: https://github.com/ekg/filevercmp</details>|
	|[filtlong](https://github.com/rrwick/Filtlong)|bio|0.2.0|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/rrwick/Filtlong URL: https://github.com/rrwick/Filtlong</details>|
	|[flexiblas](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)|-|3.0.4|<details><summary>Expand</summary>Description: FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it. Homepage: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release URL: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release</details>|
	|[fmriprep](https://fmriprep.org/)|-|23.0.2, 23.1.3|<details><summary>Expand</summary>Description: fMRIPrep is a NiPreps (NeuroImaging PREProcessing toolS) application (www.nipreps.org) for the preprocessing of task-based and resting-state functional MRI (fMRI). Homepage: https://fmriprep.org/ URL: https://fmriprep.org/</details>|
	|[fmt](http://fmtlib.net/)|-|5.3.0, 6.2.1, 7.0.3|<details><summary>Expand</summary>Description: fmt (formerly cppformat) is an open-source formatting library. Homepage: http://fmtlib.net/ URL: http://fmtlib.net/</details>|
	|[freebayes](https://github.com/freebayes/freebayes)|bio|1.3.6|<details><summary>Expand</summary>Description: FreeBayes is a Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs (single-nucleotide polymorphisms), indels (insertions and deletions), MNPs (multi-nucleotide polymorphisms), and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment. Homepage: https://github.com/freebayes/freebayes URL: https://github.com/freebayes/freebayes</details>|
	|[freesasa](https://freesasa.github.io)|-|2.1.0|<details><summary>Expand</summary>Description: FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA). Homepage: https://freesasa.github.io URL: https://freesasa.github.io</details>|
	|[freesurfer](http://freesurfer.net/)|bio|5.3.0|<details><summary>Expand</summary>Description: FreeSurfer is a software package for the analysis and visualization of structural and functional neuroimaging data from cross-sectional or longitudinal studies. It is developed by the Laboratory for Computational Neuroimaging at the Athinoula A. Martinos Center for Biomedical Imaging. Homepage: http://freesurfer.net/ URL: http://freesurfer.net/</details>|
	|[freexl](https://www.gaia-gis.it/fossil/freexl/index)|tools|1.0.5|<details><summary>Expand</summary>Description: FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet. Homepage: https://www.gaia-gis.it/fossil/freexl/index URL: https://www.gaia-gis.it/fossil/freexl/index</details>|
	|[fsom](https://github.com/ekg/fsom)|-|20141119|<details><summary>Expand</summary>Description: A tiny C library for managing SOM (Self-Organizing Maps) neural networks. Homepage: https://github.com/ekg/fsom URL: https://github.com/ekg/fsom</details>|
	|[g2clib](https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/)|geo|1.6.0|<details><summary>Expand</summary>Description: Library contains GRIB2 encoder/decoder ('C' version). Homepage: https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/ URL: https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/</details>|
	|[gamess-us](https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html)|chem|20210930-R2P1, 2020.2|<details><summary>Expand</summary>Description: The General Atomic and Molecular Electronic Structure System (GAMESS) is a general ab initio quantum chemistry package. This version of GAMESS is built with Intel compilers and IntelMPI. No LibXC or MSUCC modules. Homepage: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html URL: https://www.msg.chem.iastate.edu/GAMESS/GAMESS.html</details>|
	|[gatk](http://www.broadinstitute.org/gatk/)|bio|4.1.8.0, 4.1.8.1, 4.2.2.0, 4.2.4.0, 4.2.5.0|<details><summary>Expand</summary>Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size. Homepage: http://www.broadinstitute.org/gatk/ URL: http://www.broadinstitute.org/gatk/</details>|
	|[gblocks](http://molevol.cmima.csic.es/castresana/Gblocks.html)|-|0.91b|<details><summary>Expand</summary>Description: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis Homepage: http://molevol.cmima.csic.es/castresana/Gblocks.html URL: http://molevol.cmima.csic.es/castresana/Gblocks.html</details>|
	|[gcc](https://gcc.gnu.org/)|tools|8.4.0, 9.3.0, 10.2.0, 10.3.0, 11.3.0|<details><summary>Expand</summary>Description: The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). Homepage: https://gcc.gnu.org/ URL: https://gcc.gnu.org/</details>|
	|[gcta](http://cnsgenomics.com/software/gcta/)|bio|1.93.2|<details><summary>Expand</summary>Description: GCTA (Genome-wide Complex Trait Analysis) was originally designed to estimate the proportion of phenotypic variance explained by all genome-wide SNPs for complex traits (the GREML method), and has subsequently extended for many other analyses to better understand the genetic architecture of complex traits. GCTA currently supports the following analyses. Homepage: http://cnsgenomics.com/software/gcta/ URL: http://cnsgenomics.com/software/gcta/</details>|
	|[gdal](https://www.gdal.org/)|geo|3.0.4, 3.2.3, 3.5.1|<details><summary>Expand</summary>Description: GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing. Homepage: https://www.gdal.org/ URL: https://www.gdal.org/</details>|
	|[gdrcopy](https://github.com/NVIDIA/gdrcopy)|-|2.1|<details><summary>Expand</summary>Description: A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology. Homepage: https://github.com/NVIDIA/gdrcopy URL: https://github.com/NVIDIA/gdrcopy</details>|
	|[geant4-data](https://geant4.cern.ch/)|-|10.7.3, 11.1.0|<details><summary>Expand</summary>Description: Datasets for Geant4. Homepage: https://geant4.cern.ch/ URL: https://geant4.cern.ch/</details>|
	|[genometools](http://genometools.org)|bio|1.6.1|<details><summary>Expand</summary>Description: A comprehensive software library for efficient processing of structured genome annotations. Homepage: http://genometools.org URL: http://genometools.org</details>|
	|[geos](https://trac.osgeo.org/geos)|geo|3.8.1, 3.9.1, 3.10.2|<details><summary>Expand</summary>Description: GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS) Homepage: https://trac.osgeo.org/geos URL: https://trac.osgeo.org/geos</details>|
	|[git-annex](http://git-annex.branchable.com/)|tools|8.20200810, 10.20221003|<details><summary>Expand</summary>Description: git-annex allows managing files with git, without checking the file contents into git. While that may seem paradoxical, it is useful when dealing with files larger than git can currently easily handle, whether due to limitations in memory, time, or disk space. Homepage: http://git-annex.branchable.com/ URL: http://git-annex.branchable.com/</details>|
	|[git-lfs](https://git-lfs.github.com)|-|2.11.0, 3.3.0|<details><summary>Expand</summary>Description: Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com Homepage: https://git-lfs.github.com URL: https://git-lfs.github.com</details>|
	|[glew](http://glew.sourceforge.net/)|-|2.1.0|<details><summary>Expand</summary>Description: The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform. Homepage: http://glew.sourceforge.net/ URL: http://glew.sourceforge.net/</details>|
	|[glm](https://github.com/g-truc/glm)|vis|0.9.9.8|<details><summary>Expand</summary>Description: OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on the OpenGL Shading Language (GLSL) specifications. Homepage: https://github.com/g-truc/glm URL: https://github.com/g-truc/glm</details>|
	|[glost](https://github.com/cea-hpc/glost)|tools|0.3.1|<details><summary>Expand</summary>Description: This is GLOST, the Greedy Launcher Of Small Tasks. Homepage: https://github.com/cea-hpc/glost URL: https://github.com/cea-hpc/glost</details>|
	|[gmt](https://www.generic-mapping-tools.org/)|geo|6.4.0|<details><summary>Expand</summary>Description: GMT is an open source collection of about 80 command-line tools for manipulating geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting, etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another 40 more specialized and discipline-specific tools. Homepage: https://www.generic-mapping-tools.org/ URL: https://www.generic-mapping-tools.org/</details>|
	|[gnuplot](http://gnuplot.sourceforge.net/)|vis|5.2.8|<details><summary>Expand</summary>Description: Portable interactive, function plotting utility Homepage: http://gnuplot.sourceforge.net/ URL: http://gnuplot.sourceforge.net/</details>|
	|[go](https://www.golang.org)|tools|1.14.1, 1.18.3, 1.21.3|<details><summary>Expand</summary>Description: Go is an open source programming language that makes it easy to build simple, reliable, and efficient software. Homepage: https://www.golang.org URL: https://www.golang.org</details>|
	|[googletest](https://github.com/google/googletest)|tools|1.10.0|<details><summary>Expand</summary>Description: Google's framework for writing C++ tests on a variety of platforms Homepage: https://github.com/google/googletest URL: https://github.com/google/googletest</details>|
	|[grace](http://freecode.com/projects/grace)|vis|5.99.0|<details><summary>Expand</summary>Description: Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif. Homepage: http://freecode.com/projects/grace URL: http://freecode.com/projects/grace</details>|
	|[gromacs](http://www.gromacs.org)|chem|2020.4, 2021.4, 2022.3|<details><summary>Expand</summary>Description: GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. This is a CPU only build, containing both MPI and threadMPI builds. - CC-Wiki: GROMACS Homepage: http://www.gromacs.org URL: http://www.gromacs.org</details>|
	|[gsl](https://www.gnu.org/software/gsl/)|math|2.6|<details><summary>Expand</summary>Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting. Homepage: https://www.gnu.org/software/gsl/ URL: https://www.gnu.org/software/gsl/</details>|
	|[gsl-lite](https://github.com/gsl-lite/gsl-lite)|-|0.40.0|<details><summary>Expand</summary>Description: A single-file header-only version of ISO C++ Guidelines Support Library (GSL) for C++98, C++11, and later Homepage: https://github.com/gsl-lite/gsl-lite URL: https://github.com/gsl-lite/gsl-lite</details>|
	|[gurobi](https://www.gurobi.com)|math|9.0.3, 9.1.0, 9.1.2, 9.5.0, 9.5.2, 10.0.1, 10.0.2, 10.0.3, 11.0.0|<details><summary>Expand</summary>Description: The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms. Homepage: https://www.gurobi.com URL: https://www.gurobi.com</details>|
	|[hapgen2](https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html)|-|2.2.0|<details><summary>Expand</summary>Description: 'HAPGEN2' simulates case control datasets at SNP markers. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html</details>|
	|[haploview](https://www.broadinstitute.org/haploview/haploview)|-|4.2|<details><summary>Expand</summary>Description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses. Homepage: https://www.broadinstitute.org/haploview/haploview URL: https://www.broadinstitute.org/haploview/haploview</details>|
	|[hdf](https://www.hdfgroup.org/products/hdf4/)|io|4.2.15|<details><summary>Expand</summary>Description: HDF (also known as HDF4) is a library and multi-object file format for storing and managing data between machines. Homepage: https://www.hdfgroup.org/products/hdf4/ URL: https://www.hdfgroup.org/products/hdf4/</details>|
	|[hdf5](https://portal.hdfgroup.org/display/support)|io|1.10.6, 1.10.7|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdf5-mpi](https://portal.hdfgroup.org/display/support)|io|1.10.6, 1.10.7|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdfview](https://support.hdfgroup.org/products/java/hdfview/)|-|2.14|<details><summary>Expand</summary>Description: HDFView is a visual tool for browsing and editing HDF4 and HDF5 files. Homepage: https://support.hdfgroup.org/products/java/hdfview/ URL: https://support.hdfgroup.org/products/java/hdfview/</details>|
	|[hh-suite](https://github.com/soedinglab/hh-suite)|bio|3.3.0|<details><summary>Expand</summary>Description: HH-suite is an open-source software package for sensitive protein sequence searching. It contains programs that can search for similar protein sequences in protein sequence databases. Homepage: https://github.com/soedinglab/hh-suite URL: https://github.com/soedinglab/hh-suite</details>|
	|[hisat2](https://daehwankimlab.github.io/hisat2/)|bio|2.2.1|<details><summary>Expand</summary>Description: HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome). Homepage: https://daehwankimlab.github.io/hisat2/ URL: https://daehwankimlab.github.io/hisat2/</details>|
	|[hmmer](http://hmmer.org/)|bio|3.3.2|<details><summary>Expand</summary>Description: HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs). Compared to BLAST, FASTA, and other sequence alignment and database search tools based on older scoring methodology, HMMER aims to be significantly more accurate and more able to detect remote homologs because of the strength of its underlying mathematical models. In the past, this strength came at significant computational expense, but in the new HMMER3 project, HMMER is now essentially as fast as BLAST. Homepage: http://hmmer.org/ URL: http://hmmer.org/</details>|
	|[hoomd-blue](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[hoomd-blue-mpi](http://glotzerlab.engin.umich.edu/hoomd-blue/)|-|3.6.0|<details><summary>Expand</summary>Description: HOOMD-blue is a general-purpose particle simulation toolkit for both GPUs and CPUs, with a Python interface. Homepage: http://glotzerlab.engin.umich.edu/hoomd-blue/ URL: http://glotzerlab.engin.umich.edu/hoomd-blue/</details>|
	|[htslib](https://www.htslib.org/)|bio|1.10.2, 1.13, 1.14|<details><summary>Expand</summary>Description: A C library for reading/writing high-throughput sequencing data. This package includes the utilities bgzip and tabix Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[hwloc](https://www.open-mpi.org/projects/hwloc/)|-|2.4.0, 2.7.1|<details><summary>Expand</summary>Description: The Portable Hardware Locality (hwloc) software package provides a portable abstraction (across OS, versions, architectures, ...) of the hierarchical topology of modern architectures, including NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various system attributes such as cache and memory information as well as the locality of I/O devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering information about modern computing hardware so as to exploit it accordingly and efficiently. Homepage: https://www.open-mpi.org/projects/hwloc/ URL: https://www.open-mpi.org/projects/hwloc/</details>|
	|[igblast](https://www.ncbi.nlm.nih.gov/igblast/intro.html)|bio|1.17.0, 1.18.0|<details><summary>Expand</summary>Description: IgBLAST was developed at NCBI to facilitate analysis of immunoglobulin variable domain sequences (IgBLAST has recently been extended to perform analysis for T cell receptor (TR) sequences). It uses BLAST search algorithm. Homepage: https://www.ncbi.nlm.nih.gov/igblast/intro.html URL: https://www.ncbi.nlm.nih.gov/igblast/intro.html</details>|
	|[igv](https://www.broadinstitute.org/software/igv/)|-|2.9.2|<details><summary>Expand</summary>Description: This package contains command line utilities for preprocessing, computing feature count density (coverage), sorting, and indexing data files. Homepage: https://www.broadinstitute.org/software/igv/ URL: https://www.broadinstitute.org/software/igv/</details>|
	|[imkl](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)|math|2020.1.217, 2021.2.0, 2021.4.0, 2022.1.0|<details><summary>Expand</summary>Description: Intel oneAPI Math Kernel Library Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</details>|
	|[impute2](http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)|bio|2.3.2|<details><summary>Expand</summary>Description: IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation and haplotype phasing program based on ideas from Howie et al. 2009 Homepage: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html URL: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</details>|
	|[impute5](https://jmarchini.org/software/#impute-5)|-|1.1.5|<details><summary>Expand</summary>Description: IMPUTE 5 is a genotype imputation method that can scale to reference panels with millions of samples. This method continues to refine the observation made in the IMPUTE2 method, that accuracy is optimized via use of a custom subset of haplotypes when imputing each individual. It achieves fast, accurate, and memory-efficient imputation by selecting haplotypes using the Positional Burrows Wheeler Transform (PBWT). By using the PBWT data structure at genotyped markers, IMPUTE 5 identifies locally best matching haplotypes and long identical by state segments. The method then uses the selected haplotypes as conditioning states within the IMPUTE model. IMPUTE5 is up to 30x faster than MINIMAC4 and up to 3x faster than BEAGLE5.1 Homepage: https://jmarchini.org/software/#impute-5 URL: https://jmarchini.org/software/#impute-5</details>|
	|[infernal](http://eddylab.org/infernal/)|bio|1.1.3|<details><summary>Expand</summary>Description: Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. Homepage: http://eddylab.org/infernal/ URL: http://eddylab.org/infernal/</details>|
	|[intel](https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html)|tools|2020.1.217, 2021.2.0, 2022.1.0|<details><summary>Expand</summary>Description: Intel C, C++ and Fortran compilers (classic and oneAPI) Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</details>|
	|[intel-opencl](https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html)|-|2021.2.0|<details><summary>Expand</summary>Description: Intel's OpenCL CPU runtime Homepage: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html URL: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html</details>|
	|[intelmpi](https://software.intel.com/en-us/intel-mpi-library/)|mpi|2019.7.217|<details><summary>Expand</summary>Description: Intel MPI Library, compatible with MPICH ABI Homepage: https://software.intel.com/en-us/intel-mpi-library/ URL: https://software.intel.com/en-us/intel-mpi-library/</details>|
	|[intelxed](https://intelxed.github.io/)|-|12.0.1|<details><summary>Expand</summary>Description: The X86 Encoder Decoder (XED), is a software library (and associated headers) for encoding and decoding X86 (IA32 and Intel64) instructions. Homepage: https://intelxed.github.io/ URL: https://intelxed.github.io/</details>|
	|[interproscan](http://www.ebi.ac.uk/interpro/)|bio|5.56-89.0, 5.55-88.0, 5.52-86.0, 5.53-87.0, 5.64-96.0, 5.50-84.0, 5.63-95.0|<details><summary>Expand</summary>Description: InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[interproscan_data](http://www.ebi.ac.uk/interpro/)|-|5.64-96.0, 5.63-95.0|<details><summary>Expand</summary>Description: This module containes only InterProScan data part. InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[intervaltree](https://github.com/ekg/intervaltree)|-|0.1|<details><summary>Expand</summary>Description: An interval tree can be used to efficiently find a set of numeric intervals overlapping or containing another interval. This library provides a basic implementation of an interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. Homepage: https://github.com/ekg/intervaltree URL: https://github.com/ekg/intervaltree</details>|
	|[ipp](https://software.intel.com/en-us/articles/intel-ipp/)|tools|2020.1.217|<details><summary>Expand</summary>Description: Intel Integrated Performance Primitives (Intel IPP) is an extensive library of multicore-ready, highly optimized software functions for multimedia, data processing, and communications applications. Intel IPP offers thousands of optimized functions covering frequently used fundamental algorithms. Homepage: https://software.intel.com/en-us/articles/intel-ipp/ URL: https://software.intel.com/en-us/articles/intel-ipp/</details>|
	|[ipykernel](https://ipython.org/)|-|2020a, 2020b, 2021a, 2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains ipykernel 2023b and its dependencies Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[ipython-kernel](https://ipython.org/)|-|2.7, 3.9, 3.10, 3.11|<details><summary>Expand</summary>Description: IPython kernelspec for Python 3.11 Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[iq-tree](http://www.iqtree.org/)|bio|2.2.1|<details><summary>Expand</summary>Description: Efficient phylogenomic software by maximum likelihood Homepage: http://www.iqtree.org/ URL: http://www.iqtree.org/</details>|
	|[itac](https://software.intel.com/en-us/intel-trace-analyzer/)|tools|2021.5.0|<details><summary>Expand</summary>Description: The Intel Trace Collector is a low-overhead tracing library that performs event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application activities gathered by the Intel Trace Collector through graphical displays. Homepage: https://software.intel.com/en-us/intel-trace-analyzer/ URL: https://software.intel.com/en-us/intel-trace-analyzer/</details>|
	|[jasper](https://www.ece.uvic.ca/~frodo/jasper/)|vis|2.0.16|<details><summary>Expand</summary>Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard. Homepage: https://www.ece.uvic.ca/~frodo/jasper/ URL: https://www.ece.uvic.ca/~frodo/jasper/</details>|
	|[java](http://openjdk.java.net)|tools|11.0.16_8, 1.8.0_192, 11.0.2, 13.0.2, 14.0.2, 17.0.2|<details><summary>Expand</summary>Description: Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers. Homepage: http://openjdk.java.net URL: http://openjdk.java.net</details>|
	|[jellyfish](http://www.genome.umd.edu/jellyfish.html)|bio|2.3.0|<details><summary>Expand</summary>Description: Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. Homepage: http://www.genome.umd.edu/jellyfish.html URL: http://www.genome.umd.edu/jellyfish.html</details>|
	|[jsoncpp](https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html)|-|1.9.4|<details><summary>Expand</summary>Description: JsonCpp is a C++ library that allows manipulating JSON values, including serialization and deserialization to and from strings. It can also preserve existing comment in unserialization/serialization steps, making it a convenient format to store user input files. Homepage: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html URL: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html</details>|
	|[judy](http://judy.sourceforge.net/)|-|1.0.5|<details><summary>Expand</summary>Description: A C library that implements a dynamic array. Homepage: http://judy.sourceforge.net/ URL: http://judy.sourceforge.net/</details>|
	|[julia](http://julialang.org)|tools|1.8.1|<details><summary>Expand</summary>Description: A high-level, high-performance dynamic language for technical computing. Homepage: http://julialang.org URL: http://julialang.org</details>|
	|[kentutils](http://genome.cse.ucsc.edu/)|bio|453|<details><summary>Expand</summary>Description: Kent utilities: collection of tools used by the UCSC genome browser. Homepage: http://genome.cse.ucsc.edu/ URL: http://genome.cse.ucsc.edu/</details>|
	|[kim-api](https://openkim.org/)|chem|2.1.3|<details><summary>Expand</summary>Description: Open Knowledgebase of Interatomic Models. KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations. This is a library that can be used by simulation programs to get access to the models in the OpenKIM database. This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running kim-api-collections-management install user MODELNAME or kim-api-collections-management install user OpenKIM to install them all. Homepage: https://openkim.org/ URL: https://openkim.org/</details>|
	|[kma](https://bitbucket.org/genomicepidemiology/kma)|-|1.3.0, 1.3.25, 1.4.14|<details><summary>Expand</summary>Description: KMA is a mapping method designed to map raw reads directly against redundant databases, in an ultra-fast manner using seed and extend. Homepage: https://bitbucket.org/genomicepidemiology/kma URL: https://bitbucket.org/genomicepidemiology/kma</details>|
	|[kraken2](https://github.com/DerrickWood/kraken2/wiki)|bio|2.1.2|<details><summary>Expand</summary>Description: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. Kraken aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm. Homepage: https://github.com/DerrickWood/kraken2/wiki URL: https://github.com/DerrickWood/kraken2/wiki</details>|
	|[lammps-omp](https://lammps.sandia.gov/)|chem|20210929|<details><summary>Expand</summary>Description: LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality. Homepage: https://lammps.sandia.gov/ URL: https://lammps.sandia.gov/</details>|
	|[ldc](https://wiki.dlang.org/LDC)|-|0.17.6|<details><summary>Expand</summary>Description: The LLVM-based D Compiler Homepage: https://wiki.dlang.org/LDC URL: https://wiki.dlang.org/LDC</details>|
	|[libcdms](https://github.com/CDAT/libcdms/)|-|3.1.2|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcdms/ URL: https://github.com/CDAT/libcdms/</details>|
	|[libcerf](https://jugit.fz-juelich.de/mlz/libcerf)|math|1.13|<details><summary>Expand</summary>Description: libcerf is a self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions. Homepage: https://jugit.fz-juelich.de/mlz/libcerf URL: https://jugit.fz-juelich.de/mlz/libcerf</details>|
	|[libdap](https://www.opendap.org/software/libdap)|-|3.20.6|<details><summary>Expand</summary>Description: A C++ SDK which contains an implementation of DAP 2.0 and DAP4.0. This includes both Client- and Server-side support classes. Homepage: https://www.opendap.org/software/libdap URL: https://www.opendap.org/software/libdap</details>|
	|[libdrs](https://github.com/CDAT/libdrs/)|-|3.1.2|<details><summary>Expand</summary>Description: PCMDI's old DRS format implementation Homepage: https://github.com/CDAT/libdrs/ URL: https://github.com/CDAT/libdrs/</details>|
	|[libevent](https://libevent.org/)|-|2.1.12|<details><summary>Expand</summary>Description: The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached. Furthermore, libevent also support callbacks due to signals or regular timeouts. Homepage: https://libevent.org/ URL: https://libevent.org/</details>|
	|[libfabric](https://ofiwg.github.io/libfabric/)|-|1.10.1, 1.12.1|<details><summary>Expand</summary>Description: Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. Homepage: https://ofiwg.github.io/libfabric/ URL: https://ofiwg.github.io/libfabric/</details>|
	|[libffi](https://sourceware.org/libffi/)|-|3.3|<details><summary>Expand</summary>Description: The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time. Homepage: https://sourceware.org/libffi/ URL: https://sourceware.org/libffi/</details>|
	|[libgd](https://libgd.github.io/)|vis|2.3.0|<details><summary>Expand</summary>Description: GD is an open source code library for the dynamic creation of images by programmers. Homepage: https://libgd.github.io/ URL: https://libgd.github.io/</details>|
	|[libgeotiff](https://directory.fsf.org/wiki/Libgeotiff)|-|1.5.1, 1.6.0|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libgeotiff-proj901](https://directory.fsf.org/wiki/Libgeotiff)|-|1.7.1|<details><summary>Expand</summary>Description: Library for reading and writing coordinate system information from/to GeoTIFF files Homepage: https://directory.fsf.org/wiki/Libgeotiff URL: https://directory.fsf.org/wiki/Libgeotiff</details>|
	|[libharu](https://github.com/libharu/libharu/)|-|2.3.0|<details><summary>Expand</summary>Description: libHaru is a free, cross platform, open source library for generating PDF files. Homepage: https://github.com/libharu/libharu/ URL: https://github.com/libharu/libharu/</details>|
	|[librttopo](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[librttopo-proj9](https://git.osgeo.org/gitea/rttopo/librttopo)|-|1.1.0|<details><summary>Expand</summary>Description: The RT Topology Library exposes an API to create and manage standard (ISO 13249 aka SQL/MM) topologies Homepage: https://git.osgeo.org/gitea/rttopo/librttopo URL: https://git.osgeo.org/gitea/rttopo/librttopo</details>|
	|[libspatialindex](https://libspatialindex.github.io)|phys|1.8.5|<details><summary>Expand</summary>Description: C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API Homepage: https://libspatialindex.github.io URL: https://libspatialindex.github.io</details>|
	|[libspatialite](https://www.gaia-gis.it/fossil/libspatialite/home)|phys|4.3.0a, 5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libspatialite-proj901](https://www.gaia-gis.it/fossil/libspatialite/home)|-|5.0.1|<details><summary>Expand</summary>Description: SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities. Homepage: https://www.gaia-gis.it/fossil/libspatialite/home URL: https://www.gaia-gis.it/fossil/libspatialite/home</details>|
	|[libstatgen](https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen)|-|20190330|<details><summary>Expand</summary>Description: Useful set of classes for creating statistical genetic programs. Homepage: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen URL: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen</details>|
	|[libxc](https://www.tddft.org/programs/libxc)|chem|4.3.4|<details><summary>Expand</summary>Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals. Homepage: https://www.tddft.org/programs/libxc URL: https://www.tddft.org/programs/libxc</details>|
	|[libxslt](http://xmlsoft.org/)|-|1.1.34|<details><summary>Expand</summary>Description: Libxslt is the XSLT C library developed for the GNOME project (but usable outside of the Gnome platform). Homepage: http://xmlsoft.org/ URL: http://xmlsoft.org/</details>|
	|[lldb](https://lldb.llvm.org/)|-|11.0.0|<details><summary>Expand</summary>Description: The debugger component of the LLVM project Homepage: https://lldb.llvm.org/ URL: https://lldb.llvm.org/</details>|
	|[llvm](https://llvm.org/)|tools|8.0.1, 9.0.1, 14.0.3|<details><summary>Expand</summary>Description: The LLVM Core libraries provide a modern source- and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!) These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR"). The LLVM Core libraries are well documented, and it is particularly easy to invent your own language (or port an existing compiler) to use LLVM as an optimizer and code generator. Homepage: https://llvm.org/ URL: https://llvm.org/</details>|
	|[lmdb](https://symas.com/lmdb)|-|0.9.24|<details><summary>Expand</summary>Description: LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance of a pure in-memory database while retaining the persistence of standard disk-based databases. Homepage: https://symas.com/lmdb URL: https://symas.com/lmdb</details>|
	|[ls-opt](http://www.lstc.com/products/ls-opt)|phys|7.0.0|<details><summary>Expand</summary>Description: LS-OPT is a standalone Design Optimization and Probabilistic Analysis package with an interface to LS-DYNA. Homepage: http://www.lstc.com/products/ls-opt URL: http://www.lstc.com/products/ls-opt</details>|
	|[ls-prepost](http://lstc.com/products/ls-prepost)|-|4.8.11, 4.9.9|<details><summary>Expand</summary>Description: LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA. Homepage: http://lstc.com/products/ls-prepost URL: http://lstc.com/products/ls-prepost</details>|
	|[lsd2](https://github.com/tothuhien/lsd2)|-|2.3|<details><summary>Expand</summary>Description: Least-squares methods to estimate rates and dates from phylogenies Homepage: https://github.com/tothuhien/lsd2 URL: https://github.com/tothuhien/lsd2</details>|
	|[mafft](https://mafft.cbrc.jp/alignment/software/source.html)|bio|7.471|<details><summary>Expand</summary>Description: MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of &lt;200 sequences), FFT-NS-2 (fast; for alignment of &lt;30,000 sequences), etc. Homepage: https://mafft.cbrc.jp/alignment/software/source.html URL: https://mafft.cbrc.jp/alignment/software/source.html</details>|
	|[mariadb](https://docs.computecanada.ca/wiki/Database_servers.)|tools|10.6.12|<details><summary>Expand</summary>Description: MariaDB An enhanced, drop-in replacement for MySQL For more information please take a look at https://docs.computecanada.ca/wiki/Database_servers. Homepage: https://mariadb.org/ URL: https://mariadb.org/</details>|
	|[mariadb-connector-c](https://downloads.mariadb.org/connector-c/)|-|3.1.7|<details><summary>Expand</summary>Description: MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases. Homepage: https://downloads.mariadb.org/connector-c/ URL: https://downloads.mariadb.org/connector-c/</details>|
	|[mash](http://mash.readthedocs.org)|-|2.3|<details><summary>Expand</summary>Description: Fast genome and metagenome distance estimation using MinHash Homepage: http://mash.readthedocs.org URL: http://mash.readthedocs.org</details>|
	|[matio](https://sourceforge.net/projects/matio/)|io|1.5.19|<details><summary>Expand</summary>Description: matio is an C library for reading and writing Matlab MAT files. Homepage: https://sourceforge.net/projects/matio/ URL: https://sourceforge.net/projects/matio/</details>|
	|[matlab](http://www.mathworks.com/products/matlab)|tools|2020a, 2020b.4, 2020b.6, 2021a.1, 2021a.5, 2021b.3, 2022a, 2022b.2, 2023a.3|<details><summary>Expand</summary>Description: MATLAB is a high-level language and interactive environment that enables you to perform computationally intensive tasks faster than with traditional programming languages such as C, C++, and Fortran. Homepage: http://www.mathworks.com/products/matlab URL: http://www.mathworks.com/products/matlab</details>|
	|[maven](https://maven.apache.org/index.html)|tools|3.6.3|<details><summary>Expand</summary>Description: Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. Homepage: https://maven.apache.org/index.html URL: https://maven.apache.org/index.html</details>|
	|[mcr](https://www.mathworks.com/products/compiler/mcr/)|tools|R2020b, R2021b, R2022b|<details><summary>Expand</summary>Description: The MATLAB Runtime is a standalone set of shared libraries that enables the execution of compiled MATLAB applications or components on computers that do not have MATLAB installed. Homepage: https://www.mathworks.com/products/compiler/mcr/ URL: https://www.mathworks.com/products/compiler/mcr/</details>|
	|[meta-farm](https://docs.computecanada.ca/wiki/META_package_for_serial_farming)|-|1.0.2|<details><summary>Expand</summary>Description: Meta-farm is a suite of scripts designed in-house to fully automate throughput computing (running a large number of related serial/parallel/GPU jobs - so called farming). Homepage: https://docs.computecanada.ca/wiki/META_package_for_serial_farming URL: https://docs.computecanada.ca/wiki/META_package_for_serial_farming</details>|
	|[metabat](https://bitbucket.org/berkeleylab/metabat)|bio|2.12.1|<details><summary>Expand</summary>Description: MetaBAT: A robust statistical framework for reconstructing genomes from metagenomic data Homepage: https://bitbucket.org/berkeleylab/metabat URL: https://bitbucket.org/berkeleylab/metabat</details>|
	|[metageneannotator](http://metagene.nig.ac.jp/)|-|20080819|<details><summary>Expand</summary>Description: MetaGeneAnnotator is a gene-finding program for prokaryote and phage. Homepage: http://metagene.nig.ac.jp/ URL: http://metagene.nig.ac.jp/</details>|
	|[metis](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|math|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[mii](https://github.com/codeandkey/mii)|-|1.1.1, 1.1.2|<details><summary>Expand</summary>Description: A smart search engine for module environments. Homepage: https://github.com/codeandkey/mii URL: https://github.com/codeandkey/mii</details>|
	|[minced](https://github.com/ctSkennerton/minced)|-|0.4.2|<details><summary>Expand</summary>Description: Mining CRISPRs in Environmental Datasets Homepage: https://github.com/ctSkennerton/minced URL: https://github.com/ctSkennerton/minced</details>|
	|[minimac3](http://genome.sph.umich.edu/wiki/Minimac3)|bio|2.0.1|<details><summary>Expand</summary>Description: Minimac3 is a lower memory and more computationally efficient implementation of the genotype imputation algorithms in minimac and minimac2. Minimac3 is designed to handle very large reference panels in a more computationally efficient way with no loss of accuracy. Homepage: http://genome.sph.umich.edu/wiki/Minimac3 URL: http://genome.sph.umich.edu/wiki/Minimac3</details>|
	|[minimap2](https://github.com/lh3/minimap2)|bio|2.17|<details><summary>Expand</summary>Description: Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment. Homepage: https://github.com/lh3/minimap2 URL: https://github.com/lh3/minimap2</details>|
	|[mixcr](http://mixcr.readthedocs.io/en/latest/quickstart.html)|bio|4.1.2|<details><summary>Expand</summary>Description: MiXCR is a universal framework that processes big immunome data from raw sequences to quantitated clonotypes. MiXCR efficiently handles paired- and single-end reads, considers sequence quality, corrects PCR errors and identifies germline hypermutations. The software supports both partial- and full-length profiling and employs all available RNA or DNA information, including sequences upstream of V and downstream of J gene segments. Homepage: http://mixcr.readthedocs.io/en/latest/quickstart.html URL: http://mixcr.readthedocs.io/en/latest/quickstart.html</details>|
	|[mpi4py](https://bitbucket.org/mpi4py/mpi4py)|tools|3.0.3, 3.1.3|<details><summary>Expand</summary>Description: MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for the Python programming language, allowing any Python program to exploit multiple processors. Homepage: https://bitbucket.org/mpi4py/mpi4py URL: https://bitbucket.org/mpi4py/mpi4py</details>|
	|[multichoose](https://github.com/ekg/multichoose)|-|1.0.3|<details><summary>Expand</summary>Description: generate multiset combinations (n multichoose k). Homepage: https://github.com/ekg/multichoose URL: https://github.com/ekg/multichoose</details>|
	|[mummer](http://mummer.sourceforge.net/)|bio|4.0.0beta2|<details><summary>Expand</summary>Description: MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it. Homepage: http://mummer.sourceforge.net/ URL: http://mummer.sourceforge.net/</details>|
	|[nanoflann](https://github.com/jlblancoc/nanoflann)|-|1.3.2|<details><summary>Expand</summary>Description: nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies Homepage: https://github.com/jlblancoc/nanoflann URL: https://github.com/jlblancoc/nanoflann</details>|
	|[nccl](https://developer.nvidia.com/nccl)|tools|2.7.8, 2.8.4, 2.11.4|<details><summary>Expand</summary>Description: The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs. Homepage: https://developer.nvidia.com/nccl URL: https://developer.nvidia.com/nccl</details>|
	|[nco](https://nco.sourceforge.net)|io|5.0.6|<details><summary>Expand</summary>Description: manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5 Homepage: https://nco.sourceforge.net URL: https://nco.sourceforge.net</details>|
	|[netcdf](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-fortran-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.5.2, 4.5.3|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4, 4.8.0|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[nextflow](https://www.nextflow.io)|-|20.10.0, 21.04.3, 21.10.3, 22.04.3, 22.10.6, 22.10.8, 23.04.3|<details><summary>Expand</summary>Description: Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data Homepage: https://www.nextflow.io URL: https://www.nextflow.io</details>|
	|[nextstrain.cli](https://github.com/nextstrain/cli/tree/3.1.0)|-|3.1.0|<details><summary>Expand</summary>Description: This is the source code repository for a program called nextstrain, the Nextstrain command-line interface (CLI). It aims to provide a consistent way to run and visualize pathogen builds and access Nextstrain components like Augur and Auspice across computing environments such as Docker, Conda, and AWS Batch. Homepage: https://github.com/nextstrain/cli/tree/3.1.0 URL: https://github.com/nextstrain/cli/tree/3.1.0</details>|
	|[ngmlr](https://github.com/philres/ngmlr)|-|0.2.7|<details><summary>Expand</summary>Description: Ngmlr is a long-read mapper designed to align PacBilo or Oxford Nanopore to a reference genome with a focus on reads that span structural variations. Homepage: https://github.com/philres/ngmlr URL: https://github.com/philres/ngmlr</details>|
	|[ngs](https://github.com/ncbi/ngs)|-|2.10.5|<details><summary>Expand</summary>Description: NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from Next Generation Sequencing. Homepage: https://github.com/ncbi/ngs URL: https://github.com/ncbi/ngs</details>|
	|[ninja-phylogenetics](https://github.com/TravisWheelerLab/NINJA)|-|0.97-cluster_only|<details><summary>Expand</summary>Description: Nearly Infinite Neighbor Joining Application Compute correct neighbor-joining phylogenies for inputs of more than 10,000 sequences. This is a C++/SSE port of the original Java code described in: Wheeler, T.J. 2009. Large-scale neighbor-joining with NINJA. In S.L. Salzberg and T. Warnow (Eds.), Proceedings of the 9th Workshop on Algorithms in Bioinformatics. WABI 2009, pp. 375-389. Springer, Berlin. (LNCS webpage,preprint) The Java version of NINJA was the fastest available tool computing neighbor-joining phylogenies ( 10x faster than the fastest implemenation of the canonical neighbor-joining algorithm - QuickTree ) at the time of it's release. In addition to generating phylogenies, Ninja can be used to output pairwise distances using several common sequence distance measures, and cluster sequences using a nearest-neighbor approach. Homepage: https://github.com/TravisWheelerLab/NINJA URL: https://github.com/TravisWheelerLab/NINJA</details>|
	|[nodejs](http://nodejs.org)|tools|16.14.0|<details><summary>Expand</summary>Description: Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices. Homepage: http://nodejs.org URL: http://nodejs.org</details>|
	|[nspr](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)|-|4.25, 4.29|<details><summary>Expand</summary>Description: Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</details>|
	|[nss](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)|-|3.51, 3.57|<details><summary>Expand</summary>Description: Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</details>|
	|[ntl](http://www.shoup.net/ntl/)|math|11.4.3|<details><summary>Expand</summary>Description: NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields. Homepage: http://www.shoup.net/ntl/ URL: http://www.shoup.net/ntl/</details>|
	|[nvhpc](https://developer.nvidia.com/hpc-sdk/)|tools|20.7, 22.1, 22.7, 23.7|<details><summary>Expand</summary>Description: C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI) Homepage: https://developer.nvidia.com/hpc-sdk/ URL: https://developer.nvidia.com/hpc-sdk/</details>|
	|[opa-psm2](https://github.com/cornelisnetworks/opa-psm2/)|-|11.2.185|<details><summary>Expand</summary>Description: Low-level user-space communications interface for the Intel(R) OPA family of products. Homepage: https://github.com/cornelisnetworks/opa-psm2/ URL: https://github.com/cornelisnetworks/opa-psm2/</details>|
	|[openblas](https://xianyi.github.com/OpenBLAS/)|math|0.3.9|<details><summary>Expand</summary>Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version. Homepage: https://xianyi.github.com/OpenBLAS/ URL: https://xianyi.github.com/OpenBLAS/</details>|
	|[openmm-alphafold](http://openmm.org)|-|7.5.1|<details><summary>Expand</summary>Description: OpenMM is a toolkit for molecular simulation. Homepage: http://openmm.org URL: http://openmm.org</details>|
	|[openmpi](https://www.open-mpi.org/)|mpi|4.0.3, 4.1.1|<details><summary>Expand</summary>Description: The Open MPI Project is an open source MPI-3 implementation. Homepage: https://www.open-mpi.org/ URL: https://www.open-mpi.org/</details>|
	|[openrefine](https://github.com/OpenRefine/OpenRefine)|-|3.4.1|<details><summary>Expand</summary>Description: OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it, and augment it with data coming from the web. Homepage: https://github.com/OpenRefine/OpenRefine URL: https://github.com/OpenRefine/OpenRefine</details>|
	|[optix](https://developer.nvidia.com/optix)|-|6.5.0, 7.7.0|<details><summary>Expand</summary>Description: OptiX is NVIDIA SDK for easy ray tracing performance. It provides a simple framework for accessing the GPU's massive ray tracing power using state-of-the-art GPU algorithms. Homepage: https://developer.nvidia.com/optix URL: https://developer.nvidia.com/optix</details>|
	|[orca](http://cec.mpg.de/forum/)|chem|4.2.1, 5.0.1, 5.0.2|<details><summary>Expand</summary>Description: ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects. Homepage: http://cec.mpg.de/forum/ URL: http://cec.mpg.de/forum/</details>|
	|[orthomcl](http://orthomcl.org/)|bio|2.0.9|<details><summary>Expand</summary>Description: OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences. Homepage: http://orthomcl.org/ URL: http://orthomcl.org/</details>|
	|[ovito](https://www.ovito.org)|-|3.3.3|<details><summary>Expand</summary>Description: OVITO is a scientific visualization and analysis software for atomistic and particle simulation data. It helps scientists gain better insights into materials phenomena and physical processes. The program is freely available for all major platforms under an open source license. It has served in a growing number of computational simulation studies as a powerful tool to analyze, understand and illustrate simulation results. Homepage: https://www.ovito.org URL: https://www.ovito.org</details>|
	|[p4est](http://www.p4est.org/)|math|2.2|<details><summary>Expand</summary>Description: The p4est software library enables the dynamic management of a collection of adaptive octrees, conveniently called a forest of octrees. p4est is designed to work in parallel and scales to hundreds of thousands of processor cores. It is free software released under GNU General Public Licence version 2, or (at your option) any later version. Homepage: http://www.p4est.org/ URL: http://www.p4est.org/</details>|
	|[packmol](http://m3g.iqm.unicamp.br/packmol)|chem|20.3.3|<details><summary>Expand</summary>Description: Packing Optimization for Molecular Dynamics Simulations Homepage: http://m3g.iqm.unicamp.br/packmol URL: http://m3g.iqm.unicamp.br/packmol</details>|
	|[paml](http://abacus.gene.ucl.ac.uk/software/paml.html)|bio|4.9j|<details><summary>Expand</summary>Description: PAML is a package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood. Homepage: http://abacus.gene.ucl.ac.uk/software/paml.html URL: http://abacus.gene.ucl.ac.uk/software/paml.html</details>|
	|[panther](https://www.google.ca/search?q=panther)|-|14.1|<details><summary>Expand</summary>Description: Panther is the database part of InterProScan software. It contains only data. Homepage: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/ URL: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/</details>|
	|[parasail](https://github.com/jeffdaily/parasail)|-|2.5|<details><summary>Expand</summary>Description: parasail is a SIMD C (C99) library containing implementations of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global pairwise sequence alignment algorithms. Homepage: https://github.com/jeffdaily/parasail URL: https://github.com/jeffdaily/parasail</details>|
	|[pcre2](https://www.pcre.org/)|-|10.34, 10.35|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[pear](https://cme.h-its.org/exelixis/web/software/pear/)|bio|0.9.11|<details><summary>Expand</summary>Description: PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. It is fully parallelized and can run with as low as just a few kilobytes of memory. Homepage: https://cme.h-its.org/exelixis/web/software/pear/ URL: https://cme.h-its.org/exelixis/web/software/pear/</details>|
	|[perl](https://www.perl.org/)|tools|5.30.2|<details><summary>Expand</summary>Description: Larry Wall's Practical Extraction and Report Language Homepage: https://www.perl.org/ URL: https://www.perl.org/</details>|
	|[petsc](http://www.mcs.anl.gov/petsc)|tools|3.14.1|<details><summary>Expand</summary>Description: PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations. Homepage: http://www.mcs.anl.gov/petsc URL: http://www.mcs.anl.gov/petsc</details>|
	|[picard](http://samtools.github.io/hts-specs))|bio|2.23.3, 2.26.3|<details><summary>Expand</summary>Description: A set of tools (in Java) for working with next generation sequencing data in the BAM (http://samtools.github.io/hts-specs) format. Homepage: http://broadinstitute.github.io/picard/ URL: http://broadinstitute.github.io/picard/</details>|
	|[pilon](https://github.com/broadinstitute/pilon)|bio|1.23, 1.24|<details><summary>Expand</summary>Description: Pilon is an automated genome assembly improvement and variant detection tool Homepage: https://github.com/broadinstitute/pilon URL: https://github.com/broadinstitute/pilon</details>|
	|[plink](https://www.cog-genomics.org/plink/1.9/)|bio|2.00-10252019-avx2, 1.9b_6.21-x86_64|<details><summary>Expand</summary>Description: plink-1.9-x86_64: Whole-genome association analysis toolset Homepage: https://www.cog-genomics.org/plink/1.9/ URL: https://www.cog-genomics.org/plink/1.9/</details>|
	|[pmix](https://pmix.org/)|-|3.1.5, 3.2.3|<details><summary>Expand</summary>Description: Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. Homepage: https://pmix.org/ URL: https://pmix.org/</details>|
	|[podman](https://podman.io)|-|4.1.1, 4.5.0|<details><summary>Expand</summary>Description: Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Homepage: https://podman.io URL: https://podman.io</details>|
	|[postgresql](https://www.postgresql.org/)|tools|12.4, 14.2|<details><summary>Expand</summary>Description: PostgreSQL is a powerful, open source object-relational database system. It is fully ACID compliant, has full support for foreign keys, joins, views, triggers, and stored procedures (in multiple languages). It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP. It also supports storage of binary large objects, including pictures, sounds, or video. It has native programming interfaces for C/C++, Java, .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation. Homepage: https://www.postgresql.org/ URL: https://www.postgresql.org/</details>|
	|[pplacer](http://matsen.fhcrc.org/pplacer/)|bio|1.1.alpha19|<details><summary>Expand</summary>Description: Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give useful information about uncertainty, and to offer advanced visualization and downstream analysis. Homepage: http://matsen.fhcrc.org/pplacer/ URL: http://matsen.fhcrc.org/pplacer/</details>|
	|[prodigal](https://github.com/hyattpd/Prodigal/)|bio|2.6.3|<details><summary>Expand</summary>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee. Homepage: https://github.com/hyattpd/Prodigal/ URL: https://github.com/hyattpd/Prodigal/</details>|
	|[proj](https://proj.org)|geo|7.0.0, 7.2.1, 8.0.0, 9.0.1|<details><summary>Expand</summary>Description: Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates Homepage: https://proj.org URL: https://proj.org</details>|
	|[prokka](https://www.vicbioinformatics.com/software.prokka.shtml)|-|1.14.5|<details><summary>Expand</summary>Description: Prokka is a software tool for the rapid annotation of prokaryotic genomes. Homepage: https://www.vicbioinformatics.com/software.prokka.shtml URL: https://www.vicbioinformatics.com/software.prokka.shtml</details>|
	|[protobuf](https://github.com/google/protobuf/)|tools|3.12.3, 3.19.4, 3.21.3|<details><summary>Expand</summary>Description: Google Protocol Buffers Homepage: https://github.com/google/protobuf/ URL: https://github.com/google/protobuf/</details>|
	|[pypy](https://pypy.org/index.html)|tools|5.8.0, 7.3.3|<details><summary>Expand</summary>Description: PyPy is a fast, compliant alternative implementation of the Python language (2.7.13 and 3.5.3). Homepage: https://pypy.org/index.html URL: https://pypy.org/index.html</details>|
	|[pytest](https://pytest.org)|-|6.1.2, 6.2.5, 7.0.1, 7.4.0|<details><summary>Expand</summary>Description: pytest: simple powerful testing with Python Homepage: https://pytest.org URL: https://pytest.org</details>|
	|[python](https://python.org/)|tools|2.7.18, 3.6.10, 3.7.7, 3.7.9, 3.8.2, 3.8.10, 3.9.6, 3.10.2, 3.11.2, 3.11.5|<details><summary>Expand</summary>Description: Python is a programming language that lets you work more quickly and integrate your systems more effectively. Homepage: https://python.org/ URL: https://python.org/</details>|
	|[python-build-bundle](http://python.org/)|-|2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains Python packages useful for building Homepage: http://python.org/ URL: http://python.org/</details>|
	|[qca](http://delta.affinix.com/qca/)|geo|2.3.0|<details><summary>Expand</summary>Description: Taking a hint from the similarly-named Java Cryptography Architecture, QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes and conventions. QCA separates the API from the implementation, using plugins known as Providers. The advantage of this model is to allow applications to avoid linking to or explicitly depending on any particular cryptographic library. This allows one to easily change or upgrade crypto implementations without even needing to recompile the application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX. Homepage: http://delta.affinix.com/qca/ URL: http://delta.affinix.com/qca/</details>|
	|[qgis](http://www.qgis.org/)|geo|3.16.10|<details><summary>Expand</summary>Description: QGIS is a user friendly Open Source Geographic Information System (GIS) Homepage: http://www.qgis.org/ URL: http://www.qgis.org/</details>|
	|[qiime2](http://qiime2.org/)|-|2021.11, 2023.5|<details><summary>Expand</summary>Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. Homepage: http://qiime2.org/ URL: http://qiime2.org/</details>|
	|[qjson](http://qjson.sourceforge.net/)|tools|0.9.0|<details><summary>Expand</summary>Description: QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa. Homepage: http://qjson.sourceforge.net/ URL: http://qjson.sourceforge.net/</details>|
	|[qscintilla](https://www.riverbankcomputing.com/software/qscintilla)|tools|2.11.2|<details><summary>Expand</summary>Description: QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control Homepage: https://www.riverbankcomputing.com/software/qscintilla URL: https://www.riverbankcomputing.com/software/qscintilla</details>|
	|[qt](https://qt.io/)|tools|5.12.8|<details><summary>Expand</summary>Description: Qt is a comprehensive cross-platform C++ application framework. Homepage: https://qt.io/ URL: https://qt.io/</details>|
	|[qt5webkit](https://github.com/qt/qtwebkit)|-|5.212.0-alpha4|<details><summary>Expand</summary>Description: Qt Port of WebKit. WebKit is an open source web browser engine. Homepage: https://github.com/qt/qtwebkit URL: https://github.com/qt/qtwebkit</details>|
	|[qtkeychain](https://github.com/frankosterfeld/qtkeychain)|-|0.9.1|<details><summary>Expand</summary>Description: Platform-independent Qt API for storing passwords securely. Homepage: https://github.com/frankosterfeld/qtkeychain URL: https://github.com/frankosterfeld/qtkeychain</details>|
	|[quantumatk](https://www.synopsys.com/silicon/quantumatk.html)|-|2019.12|<details><summary>Expand</summary>Description: QuantumATK is a complete atomistic simulation toolkit. QuantumATK addresses key applications in the semiconductor industry and is a core component of the Synopsys Design-Technology Co-Optimization (DTCO) solution. Atomic-scale modeling tools in QuantumATK range from classical force fields for handling large and to some extent more realistic materials systems to ab initio tools that provide highly accurate results for smaller systems. Homepage: https://www.synopsys.com/silicon/quantumatk.html URL: https://www.synopsys.com/silicon/quantumatk.html</details>|
	|[quantumespresso](https://www.quantum-espresso.org)|chem|6.6|<details><summary>Expand</summary>Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). Homepage: https://www.quantum-espresso.org URL: https://www.quantum-espresso.org</details>|
	|[qwt](https://qwt.sourceforge.net/)|tools|6.1.4|<details><summary>Expand</summary>Description: The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background. Homepage: https://qwt.sourceforge.net/ URL: https://qwt.sourceforge.net/</details>|
	|[qwtpolar](http://qwtpolar.sourceforge.net/)|tools|1.1.1|<details><summary>Expand</summary>Description: The QwtPolar library contains classes for displaying values on a polar coordinate system. Homepage: http://qwtpolar.sourceforge.net/ URL: http://qwtpolar.sourceforge.net/</details>|
	|[r](https://www.r-project.org/)|tools|4.0.2, 4.1.2, 4.2.1|<details><summary>Expand</summary>Description: R is a free software environment for statistical computing and graphics. Homepage: https://www.r-project.org/ URL: https://www.r-project.org/</details>|
	|[racon](https://github.com/lbcb-sci/racon)|bio|1.4.13|<details><summary>Expand</summary>Description: Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. Homepage: https://github.com/lbcb-sci/racon URL: https://github.com/lbcb-sci/racon</details>|
	|[rapidjson](https://rapidjson.org)|-|1.1.0|<details><summary>Expand</summary>Description: A fast JSON parser/generator for C++ with both SAX/DOM style API Homepage: https://rapidjson.org URL: https://rapidjson.org</details>|
	|[re2c](https://re2c.org/)|-|1.3|<details><summary>Expand</summary>Description: re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons. Homepage: https://re2c.org/ URL: https://re2c.org/</details>|
	|[reframe](https://github.com/eth-cscs/reframe)|-|3.12.0|<details><summary>Expand</summary>Description: ReFrame is a framework for writing regression tests for HPC systems. Homepage: https://github.com/eth-cscs/reframe URL: https://github.com/eth-cscs/reframe</details>|
	|[rmats](http://rnaseq-mats.sourceforge.net/)|-|4.1.1, 4.1.2|<details><summary>Expand</summary>Description: MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data. The statistical model of MATS calculates the P-value and false discovery rate that the difference in the isoform ratio of a gene between two conditions exceeds a given user-defined threshold. From the RNA-Seq data, MATS can automatically detect and analyze alternative splicing events corresponding to all major types of alternative splicing patterns. MATS handles replicate RNA-Seq data from both paired and unpaired study design. Homepage: http://rnaseq-mats.sourceforge.net/ URL: http://rnaseq-mats.sourceforge.net/</details>|
	|[ruby](https://www.ruby-lang.org)|tools|2.7.1|<details><summary>Expand</summary>Description: Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. Homepage: https://www.ruby-lang.org URL: https://www.ruby-lang.org</details>|
	|[rust](https://www.rust-lang.org)|tools|1.47.0, 1.53.0, 1.59.0, 1.65.0, 1.70.0|<details><summary>Expand</summary>Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. Homepage: https://www.rust-lang.org URL: https://www.rust-lang.org</details>|
	|[salmonbeta](https://github.com/COMBINE-lab/salmon/)|bio|0.6.0|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Salmon achieves is accuracy and speed via a number of different innovations, including the use of quasi-mapping (accurate but fast-to-compute proxies for traditional read alignments), and massively-parallel stochastic collapsed variational inference. The result is a versatile tool that fits nicely into many differnt pipelines. For example, you can choose to make use of our quasi-mapping algorithm by providing Salmon with raw sequencing reads, or, if it is more convenient, you can provide Salmon with regular alignments (e.g. an unsorted BAM file produced with your favorite aligner), and it will use the same wicked-fast, state-of-the-art inference algorithm to estimate transcript-level abundances for your experiment. Homepage: https://github.com/COMBINE-lab/salmon/ URL: https://github.com/COMBINE-lab/salmon/</details>|
	|[sambamba](https://lomereiter.github.io/sambamba/)|bio|0.8.0|<details><summary>Expand</summary>Description: Sambamba is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Current functionality is an important subset of samtools functionality, including view, index, sort, markdup, and depth. Homepage: https://lomereiter.github.io/sambamba/ URL: https://lomereiter.github.io/sambamba/</details>|
	|[samtools](https://www.htslib.org/)|bio|0.1.20, 1.10, 1.11, 1.13, 1.17|<details><summary>Expand</summary>Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format. Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[sbt](http://www.scala-sbt.org/)|tools|1.3.13|<details><summary>Expand</summary>Description: sbt is a build tool for Scala, Java, and more. It requires Java 1.8 or later. sbt uses a small number of concepts to support flexible and powerful build definitions. There are not that many concepts, but sbt is not exactly like other build systems and there are details you will stumble on if you haven t read the documentation. The Getting Started Guide covers the concepts you need to know to create and maintain an sbt build definition. It is highly recommended to read the Getting Started Guide. Homepage: http://www.scala-sbt.org/ URL: http://www.scala-sbt.org/</details>|
	|[scalapack](https://www.netlib.org/scalapack/)|math|2.1.0|<details><summary>Expand</summary>Description: The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines redesigned for distributed memory MIMD parallel computers. Homepage: https://www.netlib.org/scalapack/ URL: https://www.netlib.org/scalapack/</details>|
	|[scipy-stack](http://python.org/)|math|2020a, 2020b, 2021a, 2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains the Scientific Python stack, including Cycler, mpmath, numpy, scipy, sympy, pandas, matplotlib, ipython_genutils, traitlets, ptyprocess, pathlib2, pickleshare, pexpect, simplegeneric, ipython, ipykernel, jupyter_client, jupyter_core, pyzmq, tornado, futures and ipyparallel. Homepage: http://python.org/ URL: http://python.org/</details>|
	|[sdsl](https://github.com/simongog/sdsl-lite)|-|2.1.1-20191211|<details><summary>Expand</summary>Description: The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing succinct data structures. Homepage: https://github.com/simongog/sdsl-lite URL: https://github.com/simongog/sdsl-lite</details>|
	|[seqan-library](https://www.seqan.de/)|-|2.4.0|<details><summary>Expand</summary>Description: SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences with the focus on biological data Homepage: https://www.seqan.de/ URL: https://www.seqan.de/</details>|
	|[seqkit](https://bioinf.shenwei.me/seqkit/)|-|0.13.2, 0.15.0, 2.3.1|<details><summary>Expand</summary>Description: FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. This project describes a cross-platform ultrafast comprehensive toolkit for FASTA/Q processing. SeqKit provides executable binary files for all major operating systems, including Windows, Linux, and Mac OS X, and can be directly used without any dependencies or pre-configurations. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations. Homepage: https://bioinf.shenwei.me/seqkit/ URL: https://bioinf.shenwei.me/seqkit/</details>|
	|[seqlib](https://github.com/walaj/SeqLib)|-|1.2.0|<details><summary>Expand</summary>Description: C++ interface to HTSlib, BWA-MEM and Fermi. Homepage: https://github.com/walaj/SeqLib URL: https://github.com/walaj/SeqLib</details>|
	|[seqtk](https://github.com/lh3/seqtk/)|bio|1.3|<details><summary>Expand</summary>Description: Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip. Homepage: https://github.com/lh3/seqtk/ URL: https://github.com/lh3/seqtk/</details>|
	|[shapeit](https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html)|-|2.r904|<details><summary>Expand</summary>Description: SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</details>|
	|[shrinkwrap](https://github.com/jonathonl/shrinkwrap)|-|1.2.0|<details><summary>Expand</summary>Description: A std::streambuf wrapper for compression formats. Homepage: https://github.com/jonathonl/shrinkwrap URL: https://github.com/jonathonl/shrinkwrap</details>|
	|[signalp](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp)|bio|4.1f|<details><summary>Expand</summary>Description: SignalP predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive bacteria, Gram-negative bacteria, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp</details>|
	|[singular](https://www.singular.uni-kl.de/)|-|4.2.1|<details><summary>Expand</summary>Description: Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory. Homepage: https://www.singular.uni-kl.de/ URL: https://www.singular.uni-kl.de/</details>|
	|[slepc](https://slepc.upv.es/)|-|3.14.2|<details><summary>Expand</summary>Description: SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems. Homepage: https://slepc.upv.es/ URL: https://slepc.upv.es/</details>|
	|[slicer](https://www.slicer.org/)|-|4.11.20210226|<details><summary>Expand</summary>Description: 3D Slicer is an open source software platform for medical image informatics, image processing, and three-dimensional visualization. Built over two decades through support from the National Institutes of Health and a worldwide developer community, Slicer brings free, powerful cross-platform processing tools to physicians, researchers, and the general public. Homepage: https://www.slicer.org/ URL: https://www.slicer.org/</details>|
	|[smithwaterman](https://github.com/ekg/smithwaterman)|-|20160702|<details><summary>Expand</summary>Description: smith-waterman-gotoh alignment algorithm. Homepage: https://github.com/ekg/smithwaterman URL: https://github.com/ekg/smithwaterman</details>|
	|[sniffles](https://github.com/fritzsedlazeck/Sniffles)|-|1.0.12b|<details><summary>Expand</summary>Description: Sniffles is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+) using evidence from split-read alignments, high-mismatch regions, and coverage analysis. Please note the current version of Sniffles requires sorted output from BWA-MEM (use -M and -x parameter), Minimap2 (sam file with Cigar and MD string) or NGMLR. Homepage: https://github.com/fritzsedlazeck/Sniffles URL: https://github.com/fritzsedlazeck/Sniffles</details>|
	|[snpeff](https://pcingola.github.io/SnpEff/)|bio|5.0|<details><summary>Expand</summary>Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes). Homepage: https://pcingola.github.io/SnpEff/ URL: https://pcingola.github.io/SnpEff/</details>|
	|[soci](http://soci.sourceforge.net/)|-|4.0.2|<details><summary>Expand</summary>Description: SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the regular C++ code, staying entirely within the Standard C++. Homepage: http://soci.sourceforge.net/ URL: http://soci.sourceforge.net/</details>|
	|[spark](http://spark.apache.org)|tools|3.0.0, 3.3.0|<details><summary>Expand</summary>Description: Spark is Hadoop MapReduce done in memory Homepage: http://spark.apache.org URL: http://spark.apache.org</details>|
	|[sparsehash](https://github.com/sparsehash/sparsehash)|-|2.0.4|<details><summary>Expand</summary>Description: An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed. Homepage: https://github.com/sparsehash/sparsehash URL: https://github.com/sparsehash/sparsehash</details>|
	|[spectra](https://spectralib.org/index.html)|-|0.9.0|<details><summary>Expand</summary>Description: SpectrA stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library. SpectrA is implemented as a header-only C++ library, whose only dependency, Eigen, is also header-only. Hence SpectrA can be easily embedded in C++ projects that require calculating eigenvalues of large matrices. Homepage: https://spectralib.org/index.html URL: https://spectralib.org/index.html</details>|
	|[spoa](https://github.com/rvaser/spoa)|-|3.4.0|<details><summary>Expand</summary>Description: Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm which is used to generate consensus sequences Homepage: https://github.com/rvaser/spoa URL: https://github.com/rvaser/spoa</details>|
	|[sqlite](https://www.sqlite.org/)|-|3.38.5, 3.43.1|<details><summary>Expand</summary>Description: SQLite: SQL Database Engine in a C Library Homepage: https://www.sqlite.org/ URL: https://www.sqlite.org/</details>|
	|[sra-toolkit](https://github.com/ncbi/sra-tools)|bio|2.10.8, 3.0.0|<details><summary>Expand</summary>Description: The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to programmatically access data housed within SRA and convert it from the SRA format Homepage: https://github.com/ncbi/sra-tools URL: https://github.com/ncbi/sra-tools</details>|
	|[ssw](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library)|-|1.1|<details><summary>Expand</summary>Description: SSW is a fast implementation of the Smith-Waterman algorithm, which uses the Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the instruction level. SSW library provides an API that can be flexibly used by programs written in C, C++ and other languages. We also provide a software that can do protein and genome alignment directly. Current version of our implementation is ~50 times faster than an ordinary Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and location heuristically. Homepage: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library URL: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library</details>|
	|[starccm](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010-R8, 16.02.008-R8, 16.02.009-R8, 16.04.007-R8, 16.04.012-R8, 16.06.008-R8, 17.02.007-R8, 17.02.008-R8, 17.04.007-R8, 17.04.008-R8, 17.06.007-R8, 17.06.008-R8, 18.02.008-R8, 18.04.008-R8, 18.06.006-R8|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses double precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[starccm-mixed](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010, 16.02.008, 16.02.009, 16.04.007, 16.04.012, 16.06.008, 17.02.007, 17.02.008, 17.04.007, 17.04.008, 17.06.007, 17.06.008, 18.02.008, 18.04.008, 18.06.006|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses mixed precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[suitesparse](https://faculty.cse.tamu.edu/davis/suitesparse.html)|math|5.7.1|<details><summary>Expand</summary>Description: SuiteSparse is a collection of libraries manipulate sparse matrices. Homepage: https://faculty.cse.tamu.edu/davis/suitesparse.html URL: https://faculty.cse.tamu.edu/davis/suitesparse.html</details>|
	|[supernova](https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome)|bio|2.1.1|<details><summary>Expand</summary>Description: Supernova is delivered as a single, self-contained tar file that can be unpacked anywhere on your system. It bundles all of its own required software dependencies, which are pre-compiled to run on a range of Linux distributions. Homepage: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome URL: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome</details>|
	|[swi-prolog](https://www.swi-prolog.org/)|-|9.0.3|<details><summary>Expand</summary>Description: SWI-Prolog offers a comprehensive free Prolog environment. Homepage: https://www.swi-prolog.org/ URL: https://www.swi-prolog.org/</details>|
	|[swig](http://www.swig.org/)|-|4.0.1|<details><summary>Expand</summary>Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. Homepage: http://www.swig.org/ URL: http://www.swig.org/</details>|
	|[tabix](http://samtools.sourceforge.net)|bio|0.2.6|<details><summary>Expand</summary>Description: Generic indexer for TAB-delimited genome position files Homepage: http://samtools.sourceforge.net URL: http://samtools.sourceforge.net</details>|
	|[tabixpp](https://github.com/ekg/tabixpp)|-|1.1.0|<details><summary>Expand</summary>Description: C++ wrapper to tabix indexer Homepage: https://github.com/ekg/tabixpp URL: https://github.com/ekg/tabixpp</details>|
	|[taxonkit](https://bioinf.shenwei.me/taxonkit/)|-|0.6.2|<details><summary>Expand</summary>Description: TaxonKit - A Cross-platform and Efficient NCBI Taxonomy Toolkit TaxonKit is a command-line toolkit for rapid manipulation of NCBI taxonomy data. It provides executable binary files for major operating systems including Windows, Linux, and Mac OS X, and can be directly used without any dependencies nor local database buiding. Homepage: https://bioinf.shenwei.me/taxonkit/ URL: https://bioinf.shenwei.me/taxonkit/</details>|
	|[tbb](https://github.com/oneapi-src/oneTBB)|tools|2020.2, 2021.8.0|<details><summary>Expand</summary>Description: Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability. Homepage: https://github.com/oneapi-src/oneTBB URL: https://github.com/oneapi-src/oneTBB</details>|
	|[tbl2asn](https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/)|-|25.8|<details><summary>Expand</summary>Description: Tbl2asn is a command-line program that automates the creation of sequence records for submission to GenBank Homepage: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/ URL: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/</details>|
	|[tensorrt](https://developer.nvidia.com/tensorrt)|-|8.6.1.6|<details><summary>Expand</summary>Description: NVIDIA TensorRT is a platform for high-performance deep learning inference Homepage: https://developer.nvidia.com/tensorrt URL: https://developer.nvidia.com/tensorrt</details>|
	|[thrift](https://thrift.apache.org/)|-|0.13.0, 0.16.0, 0.18.1|<details><summary>Expand</summary>Description: The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages. Homepage: https://thrift.apache.org/ URL: https://thrift.apache.org/</details>|
	|[tmhmm](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm)|bio|2.0c|<details><summary>Expand</summary>Description: TMHMM 2.0 predicts transmembrane helices in proteins. The method is described in: Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes. Anders Krogh and Bjorn Larsson, Gunnar von Heijne, and Erik L.L. Sonnhammer. J. Mol. Biol. 305:567-580, 2001. and: A hidden Markov model for predicting transmembrane helices in protein sequences. Erik L.L. Sonnhammer, Gunnar von Heijne, and Anders Krogh. In J. Glasgow et al., eds.: Proc. of ISMB6, pages 175-182. AAAI Press, 1998. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm</details>|
	|[transdecoder](https://transdecoder.github.io/)|bio|5.5.0|<details><summary>Expand</summary>Description: TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks. Homepage: https://transdecoder.github.io/ URL: https://transdecoder.github.io/</details>|
	|[transrate](http://hibberdlab.com/transrate/)|bio|1.0.3|<details><summary>Expand</summary>Description: Transrate is software for de-novo transcriptome assembly quality analysis. It examines your assembly in detail and compares it to experimental evidence such as the sequencing reads, reporting quality scores for contigs and assemblies. This allows you to choose between assemblers and parameters, filter out the bad contigs from an assembly, and help decide when to stop trying to improve the assembly. Homepage: http://hibberdlab.com/transrate/ URL: http://hibberdlab.com/transrate/</details>|
	|[trilinos](https://trilinos.org)|tools|13.4.1|<details><summary>Expand</summary>Description: The Trilinos Project is an effort to develop algorithms and enabling technologies within an object-oriented software framework for the solution of large-scale, complex multi-physics engineering and scientific problems. A unique design feature of Trilinos is its focus on packages. Homepage: https://trilinos.org URL: https://trilinos.org</details>|
	|[trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)|bio|0.39|<details><summary>Expand</summary>Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.The selection of trimming steps and their associated parameters are supplied on the command line. Homepage: http://www.usadellab.org/cms/?page=trimmomatic URL: http://www.usadellab.org/cms/?page=trimmomatic</details>|
	|[ucx](http://www.openucx.org/)|-|1.8.0, 1.9.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications Homepage: http://www.openucx.org/ URL: http://www.openucx.org/</details>|
	|[udunits](https://www.unidata.ucar.edu/software/udunits/)|tools|2.2.26, 2.2.28|<details><summary>Expand</summary>Description: UDUNITS supports conversion of unit specifications between formatted and binary forms, arithmetic manipulation of units, and conversion of values between compatible scales of measurement. Homepage: https://www.unidata.ucar.edu/software/udunits/ URL: https://www.unidata.ucar.edu/software/udunits/</details>|
	|[unixodbc](https://www.unixodbc.org)|-|2.3.9|<details><summary>Expand</summary>Description: unixODBC provides a uniform interface between application and database driver Homepage: https://www.unixodbc.org URL: https://www.unixodbc.org</details>|
	|[varscan](https://github.com/dkoboldt/varscan)|-|2.4.2|<details><summary>Expand</summary>Description: Variant calling and somatic mutation/CNV detection for next-generation sequencing data Homepage: https://github.com/dkoboldt/varscan URL: https://github.com/dkoboldt/varscan</details>|
	|[vcflib](https://github.com/vcflib/vcflib)|-|1.0.3|<details><summary>Expand</summary>Description: vcflib provides methods to manipulate and interpret sequence variation as it can be described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended to concisely describe reference-indexed genetic variations between individuals. Homepage: https://github.com/vcflib/vcflib URL: https://github.com/vcflib/vcflib</details>|
	|[vcftools](https://vcftools.github.io)|bio|0.1.16|<details><summary>Expand</summary>Description: The aim of VCFtools is to provide easily accessible methods for working with complex genetic variation data in the form of VCF files. Homepage: https://vcftools.github.io URL: https://vcftools.github.io</details>|
	|[viennarna](http://www.tbi.univie.ac.at/RNA/)|bio|2.5.0|<details><summary>Expand</summary>Description: The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures. Homepage: http://www.tbi.univie.ac.at/RNA/ URL: http://www.tbi.univie.ac.at/RNA/</details>|
	|[visit](https://wci.llnl.gov/simulation/computer-codes/visit)|vis|2.13.3|<details><summary>Expand</summary>Description: VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool. Homepage: https://wci.llnl.gov/simulation/computer-codes/visit URL: https://wci.llnl.gov/simulation/computer-codes/visit</details>|
	|[voro++](http://math.lbl.gov/voro++/)|math|0.4.6|<details><summary>Expand</summary>Description: Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles. Homepage: http://math.lbl.gov/voro++/ URL: http://math.lbl.gov/voro++/</details>|
	|[vtk](https://www.vtk.org)|vis|8.2.0|<details><summary>Expand</summary>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation. Homepage: https://www.vtk.org URL: https://www.vtk.org</details>|
	|[vtune](https://software.intel.com/en-us/vtune)|tools|2020.1, 2022.2|<details><summary>Expand</summary>Description: Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran, Assembly and Java. Homepage: https://software.intel.com/en-us/vtune URL: https://software.intel.com/en-us/vtune</details>|
	|[wasp](https://code.ornl.gov/neams-workbench/wasp)|-|3.1.4, 4.0.3|<details><summary>Expand</summary>Description: The Workbench Analysis Sequence Processor (WASP) was developed to streamline lexing, parsing, access, validation, and analysis of ascii text files. Homepage: https://code.ornl.gov/neams-workbench/wasp URL: https://code.ornl.gov/neams-workbench/wasp</details>|
	|[xml-libxml](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)|-|2.0205|<details><summary>Expand</summary>Description: Perl binding for libxml2 Homepage: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod URL: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</details>|
	|[xmlf90](https://launchpad.net/xmlf90/)|tools|1.5.4|<details><summary>Expand</summary>Description: xmlf90 is a suite of libraries to handle XML in Fortran. It has two major components: - A XML parsing library. The parser was designed to be a useful tool in the extraction and analysis of data in the context of scientific computing, and thus the priorities were efficiency and the ability to deal with very large XML files while maintaining a small memory footprint. The most complete programming interface is based on the very successful SAX (Simple API for XML) model, although a partial DOM interface and a very experimental XPATH interface are also present. - A library (xmlf90-wxml) that facilitates the writing of well-formed XML, including such features as automatic start-tag completion, attribute pretty-printing, and element indentation. There are also helper routines to handle the output of numerical arrays. Homepage: https://launchpad.net/xmlf90/ URL: https://launchpad.net/xmlf90/</details>|
	|[xtensor](https://github.com/xtensor-stack/xtensor)|-|0.24.2|<details><summary>Expand</summary>Description: xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions. Homepage: https://github.com/xtensor-stack/xtensor URL: https://github.com/xtensor-stack/xtensor</details>|
	|[yaxt](https://www.dkrz.de/redmine/projects/yaxt)|tools|0.9.0, 0.9.1|<details><summary>Expand</summary>Description: Yet Another eXchange Tool Homepage: https://www.dkrz.de/redmine/projects/yaxt URL: https://www.dkrz.de/redmine/projects/yaxt</details>|

=== "SSE3"

	|**Module**|**Type**|**Versions**|**Description**|
	|:----:|:----:|:----:|:----:|
	|[abaqus](http://www.simulia.com/products/abaqus_fea.html)|phys|2021|<details><summary>Expand</summary>Description: Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA. Homepage: http://www.simulia.com/products/abaqus_fea.html URL: http://www.simulia.com/products/abaqus_fea.html</details>|
	|[actc](https://sourceforge.net/projects/actc)|-|1.1|<details><summary>Expand</summary>Description: ACTC converts independent triangles into triangle strips or fans. Homepage: https://sourceforge.net/projects/actc URL: https://sourceforge.net/projects/actc</details>|
	|[admixture](http://software.genetics.ucla.edu/admixture)|bio|1.3.0|<details><summary>Expand</summary>Description: ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. Homepage: http://software.genetics.ucla.edu/admixture URL: http://software.genetics.ucla.edu/admixture</details>|
	|[advisor](https://software.intel.com/intel-advisor-xe)|tools|2020.3|<details><summary>Expand</summary>Description: Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain Homepage: https://software.intel.com/intel-advisor-xe URL: https://software.intel.com/intel-advisor-xe</details>|
	|[annovar](http://annovar.openbioinformatics.org/en/latest/)|bio|20191024|<details><summary>Expand</summary>Description: ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). Homepage: http://annovar.openbioinformatics.org/en/latest/ URL: http://annovar.openbioinformatics.org/en/latest/</details>|
	|[anserini](https://github.com/castorini/anserini)|-|0.9.4|<details><summary>Expand</summary>Description: An open-source information retreival toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h Homepage: https://github.com/castorini/anserini URL: https://github.com/castorini/anserini</details>|
	|[ansys](http://www.ansys.com)|phys|2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ansysedt](http://www.ansys.com)|-|2021R2, 2023R2|<details><summary>Expand</summary>Description: ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater. Homepage: http://www.ansys.com URL: http://www.ansys.com</details>|
	|[ant](https://ant.apache.org/)|tools|1.9.15, 1.10.8|<details><summary>Expand</summary>Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. Homepage: https://ant.apache.org/ URL: https://ant.apache.org/</details>|
	|[apptainer](https://apptainer.org)|-|1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4|<details><summary>Expand</summary>Description: Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. Homepage: https://apptainer.org URL: https://apptainer.org</details>|
	|[ascp](http://downloads.asperasoft.com/)|tools|3.5.4|<details><summary>Expand</summary>Description: Aspera ASCP is a high speed file transfer software Homepage: http://downloads.asperasoft.com/ URL: http://downloads.asperasoft.com/</details>|
	|[autodock_vina](http://vina.scripps.edu/index.html)|chem|1.1.2|<details><summary>Expand</summary>Description: AutoDock Vina is an open-source program for doing molecular docking. Homepage: http://vina.scripps.edu/index.html URL: http://vina.scripps.edu/index.html</details>|
	|[bbmap](https://sourceforge.net/projects/bbmap/)|bio|38.86|<details><summary>Expand</summary>Description: BBMap short read aligner, and other bioinformatic tools. Homepage: https://sourceforge.net/projects/bbmap/ URL: https://sourceforge.net/projects/bbmap/</details>|
	|[beagle](https://faculty.washington.edu/browning/beagle/beagle.html)|-|5.4|<details><summary>Expand</summary>Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers. Homepage: https://faculty.washington.edu/browning/beagle/beagle.html URL: https://faculty.washington.edu/browning/beagle/beagle.html</details>|
	|[beef](http://dx.doi.org/10.1103/PhysRevB.85.235149.)|chem|0.1.1|<details><summary>Expand</summary>Description: BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. Homepage: http://suncat.stanford.edu/facility/software/functional URL: http://suncat.stanford.edu/facility/software/functional</details>|
	|[bioperl](https://bioperl.org/)|bio|1.7.7, 1.7.8|<details><summary>Expand</summary>Description: Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects. Homepage: https://bioperl.org/ URL: https://bioperl.org/</details>|
	|[blast](http://blast.ncbi.nlm.nih.gov/)|bio|2.2.26|<details><summary>Expand</summary>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences. Homepage: http://blast.ncbi.nlm.nih.gov/ URL: http://blast.ncbi.nlm.nih.gov/</details>|
	|[blender](https://www.blender.org/)|vis|3.6.0|<details><summary>Expand</summary>Description: Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. Homepage: https://www.blender.org/ URL: https://www.blender.org/</details>|
	|[blis](https://github.com/flame/blis/)|-|0.8.1|<details><summary>Expand</summary>Description: BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries. Homepage: https://github.com/flame/blis/ URL: https://github.com/flame/blis/</details>|
	|[blosc](https://www.blosc.org/)|-|1.17.1|<details><summary>Expand</summary>Description: Blosc, an extremely fast, multi-threaded, meta-compressor library Homepage: https://www.blosc.org/ URL: https://www.blosc.org/</details>|
	|[boost](https://www.boost.org/)|tools|1.72.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[boost-mpi](https://www.boost.org/)|tools|1.72.0|<details><summary>Expand</summary>Description: Boost provides free peer-reviewed portable C++ source libraries. Homepage: https://www.boost.org/ URL: https://www.boost.org/</details>|
	|[bustools](https://github.com/BUStools/bustools)|-|0.40.0|<details><summary>Expand</summary>Description: bustools is a program for manipulating BUS files for single cell RNA-Seq datasets. It can be used to error correct barcodes, collapse UMIs, produce gene count or transcript compatibility count matrices, and is useful for many other tasks. See the kallisto bustools website for examples and instructions on how to use bustools as part of a single-cell RNA-seq workflow. Homepage: https://github.com/BUStools/bustools URL: https://github.com/BUStools/bustools</details>|
	|[bwa](http://bio-bwa.sourceforge.net/)|bio|0.7.17|<details><summary>Expand</summary>Description: Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome. Homepage: http://bio-bwa.sourceforge.net/ URL: http://bio-bwa.sourceforge.net/</details>|
	|[cap3](http://seq.cs.iastate.edu/)|bio|20151002|<details><summary>Expand</summary>Description: CAP3 assembly program Homepage: http://seq.cs.iastate.edu/ URL: http://seq.cs.iastate.edu/</details>|
	|[catch2](https://github.com/catchorg/Catch2)|-|2.11.0|<details><summary>Expand</summary>Description: A modern, C++-native, header-only, test framework for unit-tests, TDD and BDD - using C++11, C++14, C++17 and later (or C++03 on the Catch1.x branch) Homepage: https://github.com/catchorg/Catch2 URL: https://github.com/catchorg/Catch2</details>|
	|[ccsm](http://www.cesm.ucar.edu/models/ccsm4.0/)|geo|4_0_a02|<details><summary>Expand</summary>Description: The Community Climate System Model (CCSM) is a coupled climate model for simulating the earth's climate system. Composed of four separate models simultaneously simulating the earth's atmosphere, ocean, land surface and sea-ice, and one central coupler component, the CCSM allows researchers to conduct fundamental research into the earth's past, present and future climate states. Homepage: http://www.cesm.ucar.edu/models/ccsm4.0/ URL: http://www.cesm.ucar.edu/models/ccsm4.0/</details>|
	|[cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)|bio|2.1.0|<details><summary>Expand</summary>Description: Cell Ranger is a set of analysis pipelines that process Chromium single cell 3-RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis. Cell Ranger 1.2 and later support libraries generated by the Chromium Single Cell 3i-v1 and v2 reagent kits, whereas Cell Ranger 1.1 and earlier do not support v2 libraries. Cell Ranger includes four main gene expression pipelines: - cellranger mkfastq wraps Illumina's bcl2fastq to correctly demultiplex Chromium-prepared sequencing samples and to convert barcode and read data to FASTQ files. - cellranger count takes FASTQ files from cellranger mkfastq and performs alignment, filtering, and UMI counting. - cellranger aggr aggregates outputs from multiple runs of cellranger count, normalizing those runs to the same sequencing depth and then recomputing the gene-barcode matrices and analysis on the combined data. - cellranger reanalyze takes gene-barcode matrices produced by cellranger count or cellranger aggr and reruns the dimensionality reduction, clustering, and gene expression algorithms using tunable parameter settings. These pipelines combine Chromium-specific algorithms with the widely used RNA-seq aligner STAR. Output is delivered in standard BAM, MEX, CSV, HDF5 and HTML formats that are augmented with cellular information. Homepage: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome URL: https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</details>|
	|[cereal](http://uscilab.github.io/cereal/)|-|1.3.0|<details><summary>Expand</summary>Description: cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or used standalone. Homepage: http://uscilab.github.io/cereal/ URL: http://uscilab.github.io/cereal/</details>|
	|[cesm](http://www.cesm.ucar.edu/models/cesm2/)|geo|2.1.3|<details><summary>Expand</summary>Description: The Community Earth System Model (CESM) is a fully-coupled, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states. Homepage: http://www.cesm.ucar.edu/models/cesm2/ URL: http://www.cesm.ucar.edu/models/cesm2/</details>|
	|[cgal](https://www.cgal.org/)|math|4.14.3, 5.5.2|<details><summary>Expand</summary>Description: The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library. Homepage: https://www.cgal.org/ URL: https://www.cgal.org/</details>|
	|[cmake](https://www.cmake.org)|tools|3.18.4, 3.20.1, 3.21.4, 3.22.1, 3.23.1, 3.27.7|<details><summary>Expand</summary>Description: CMake, the cross-platform, open-source build system. CMake is a family of tools designed to build, test and package software. Homepage: https://www.cmake.org URL: https://www.cmake.org</details>|
	|[code-server](https://github.com/cdr/code-server)|-|3.5.0, 3.12.0|<details><summary>Expand</summary>Description: Run VS Code on any machine anywhere and access it in the browser. Homepage: https://github.com/cdr/code-server URL: https://github.com/cdr/code-server</details>|
	|[coinmp](https://projects.coin-or.org/CoinMP)|-|1.8.4|<details><summary>Expand</summary>Description: CoinMP is a C-API library that supports most of the functionality of CLP (Coin LP), CBC (Coin Branch-and-Cut), and CGL (Cut Generation Library) projects. Homepage: https://projects.coin-or.org/CoinMP URL: https://projects.coin-or.org/CoinMP</details>|
	|[comsol](https://www.comsol.com)|phys|5.6, 6.0, 6.0.0.405, 6.1.0.357, 6.2|<details><summary>Expand</summary>Description: COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems. Homepage: https://www.comsol.com URL: https://www.comsol.com</details>|
	|[corset](https://github.com/Oshlack/Corset/wiki)|bio|1.09|<details><summary>Expand</summary>Description: Corset is a command-line software program to go from a de novo transcriptome assembly to gene-level counts. Our software takes a set of reads that have been multi-mapped to the transcriptome (where multiple alignments per read were reported) and hierarchically clusters the transcripts based on the proportion of shared reads and expression patterns. It will report the clusters and gene-level counts for each sample, which are easily tested for differential expression with count based tools such as edgeR and DESeq. Homepage: https://github.com/Oshlack/Corset/wiki URL: https://github.com/Oshlack/Corset/wiki</details>|
	|[cppzmq](https://github.com/zeromq/cppzmq)|-|4.7.1|<details><summary>Expand</summary>Description: cppzmq is a C++ binding for libzmq Homepage: https://github.com/zeromq/cppzmq URL: https://github.com/zeromq/cppzmq</details>|
	|[cromwell](http://cromwell.readthedocs.io/)|tools|58|<details><summary>Expand</summary>Description: Scientific workflow engine designed for simplicity and scalability. Trivially transition between one off use cases to massive scale production environments http://cromwell.readthedocs.io/ Homepage: https://cromwell.readthedocs.io/ URL: https://cromwell.readthedocs.io/</details>|
	|[cst](https://www.cst.com/)|phys|2020|<details><summary>Expand</summary>Description: CST offers accurate, efficient computational solutions for electromagnetic design and analysis. CST simulation is user-friendly and enables users to choose the most appropriate method for the design and optimization of devices operating in a wide range of frequencies. Please note that we installed only Graphic frontend and command line interface. You need to provide your own license server in order to be able to run CST. Homepage: https://www.cst.com/ URL: https://www.cst.com/</details>|
	|[csvtk](https://github.com/shenwei356/csvtk)|-|0.23.0|<details><summary>Expand</summary>Description: Similar to FASTA/Q format in field of Bioinformatics, CSV/TSV formats are basic and ubiquitous file formats in both Bioinformatics and data science. People usually use spreadsheet software like MS Excel to process table data. However this is all by clicking and typing, which is not automated and is time-consuming to repeat, especially when you want to apply similar operations with different datasets or purposes. You can also accomplish some CSV/TSV manipulations using shell commands, but more code is needed to handle the header line. Shell commands do not support selecting columns with column names either. csvtk is convenient for rapid data investigation and also easy to integrate into analysis pipelines. It could save you lots of time in (not) writing Python/R scripts. Homepage: https://github.com/shenwei356/csvtk URL: https://github.com/shenwei356/csvtk</details>|
	|[cuba](https://feynarts.de/cuba/)|-|4.2.2|<details><summary>Expand</summary>Description: Cuba a library for multidimensional numerical integration Homepage: https://feynarts.de/cuba/ URL: https://feynarts.de/cuba/</details>|
	|[cudnn](https://developer.nvidia.com/cudnn)|math|8.0.3, 8.2.0, 8.6.0.163, 8.7.0.84, 8.9.5.29|<details><summary>Expand</summary>Description: The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. Homepage: https://developer.nvidia.com/cudnn URL: https://developer.nvidia.com/cudnn</details>|
	|[cusparselt](https://docs.nvidia.com/cuda/cusparselt/index.html)|-|0.4.0.7|<details><summary>Expand</summary>Description: NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix Homepage: https://docs.nvidia.com/cuda/cusparselt/index.html URL: https://docs.nvidia.com/cuda/cusparselt/index.html</details>|
	|[custom-ctypes](https://github.com/ComputeCanada/custom_ctypes/)|-|1.1|<details><summary>Expand</summary>Description: This repository is to try to fix discovery of libraries with Python's ctypes module. It changes the behavior of find_library to return absolute paths to shared objects rather than just the names. Homepage: https://github.com/ComputeCanada/custom_ctypes/ URL: https://github.com/ComputeCanada/custom_ctypes/</details>|
	|[cutensor](https://developer.nvidia.com/cutensor)|-|1.5.0.3, 1.7.0.1|<details><summary>Expand</summary>Description: cuTENSOR is a high-performance CUDA library for tensor primitives. Homepage: https://developer.nvidia.com/cutensor URL: https://developer.nvidia.com/cutensor</details>|
	|[cvit](https://sourceforge.net/projects/cvit/)|-|1.2.1|<details><summary>Expand</summary>Description: A genomic linkage feature visualization tools based on Perl. Homepage: https://sourceforge.net/projects/cvit/ URL: https://sourceforge.net/projects/cvit/</details>|
	|[dakota](https://dakota.sandia.gov/)|tools|6.13|<details><summary>Expand</summary>Description: The Dakota project delivers both state-of-the-art research and robust, usable software for optimization and UQ. Broadly, the Dakota software's advanced parametric analyses enable design exploration, model calibration, risk analysis, and quantification of margins and uncertainty with computational models. The Dakota toolkit provides a flexible, extensible interface between such simulation codes and its iterative systems analysis methods, which include: - optimization with gradient and nongradient-based methods; - uncertainty quantification with sampling, reliability, stochastic expansion, and epistemic methods; - parameter estimation using nonlinear least squares (deterministic) or Bayesian inference (stochastic); and - sensitivity/variance analysis with design of experiments and parameter study methods. These capabilities may be used on their own or as components within advanced strategies such as hybrid optimization, surrogate-based optimization, mixed integer nonlinear programming, or optimization under uncertainty. Homepage: https://dakota.sandia.gov/ URL: https://dakota.sandia.gov/</details>|
	|[ddt-cpu](http://www.allinea.com)|tools|20.2, 22.0.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining Arm MAP and Arm DDT. Homepage: http://www.allinea.com URL: http://www.allinea.com</details>|
	|[ddt-gpu](http://www.allinea.com)|tools|20.2, 22.0.1|<details><summary>Expand</summary>Description: Profiler and debugger, combining Arm MAP and Arm DDT, with support for CUDA. Homepage: http://www.allinea.com URL: http://www.allinea.com</details>|
	|[delly](https://github.com/dellytools/delly)|-|0.8.5|<details><summary>Expand</summary>Description: Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. Homepage: https://github.com/dellytools/delly URL: https://github.com/dellytools/delly</details>|
	|[dorado](https://github.com/nanoporetech/dorado)|-|0.2.1, 0.2.2, 0.3.0|<details><summary>Expand</summary>Description: Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads. Homepage: https://github.com/nanoporetech/dorado URL: https://github.com/nanoporetech/dorado</details>|
	|[dotnet-core](https://www.microsoft.com/net/)|-|3.1.8, 5.0.12, 6.0.0|<details><summary>Expand</summary>Description: .NET is a free, cross-platform, open source developer platform for building many different types of applications. With .NET, you can use multiple languages, editors, and libraries to build for web, mobile, desktop, gaming, and IoT. Contains the SDK and the Runtime. Homepage: https://www.microsoft.com/net/ URL: https://www.microsoft.com/net/</details>|
	|[double-conversion](https://github.com/google/double-conversion)|-|3.1.5|<details><summary>Expand</summary>Description: Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles. Homepage: https://github.com/google/double-conversion URL: https://github.com/google/double-conversion</details>|
	|[eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page)|math|3.3.7|<details><summary>Expand</summary>Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms. Homepage: https://eigen.tuxfamily.org/index.php?title=Main_Page URL: https://eigen.tuxfamily.org/index.php?title=Main_Page</details>|
	|[elpa](https://elpa.rzg.mpg.de)|math|2020.05.001|<details><summary>Expand</summary>Description: Eigenvalue SoLvers for Petaflop-Applications . Homepage: https://elpa.rzg.mpg.de URL: https://elpa.rzg.mpg.de</details>|
	|[embree](https://github.com/embree/embree)|-|3.11.0|<details><summary>Expand</summary>Description: Intel Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Intel Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Intel Embree supports runtime code selection to choose the traversal and build algorithms that best matches the instruction set of your CPU. Homepage: https://github.com/embree/embree URL: https://github.com/embree/embree</details>|
	|[energyplus](https://energyplus.net/downloads)|-|9.3.0, 23.2.0|<details><summary>Expand</summary>Description: EnergyPlus is a whole building energy simulation program that engineers, architects, and researchers use to model both energy consumptionfor heating, cooling, ventilation, lighting and plug and process loadsand water use in buildings. Homepage: https://energyplus.net/downloads URL: https://energyplus.net/downloads</details>|
	|[expat](https://libexpat.github.io)|tools|2.2.9, 2.2.10|<details><summary>Expand</summary>Description: Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags) Homepage: https://libexpat.github.io URL: https://libexpat.github.io</details>|
	|[fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)|bio|0.11.9, 0.12.0|<details><summary>Expand</summary>Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline. Homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</details>|
	|[fastsimcoal2](http://cmpg.unibe.ch/software/fastsimcoal2/)|bio|2.6.0.3, 2.7.0.9|<details><summary>Expand</summary>Description: While preserving all the simulation flexibility of simcoal2, fastsimcoal is now implemented under a faster continous-time sequential Markovian coalescent approximation, allowing it to efficiently generate genetic diversity for different types of markers along large genomic regions, for both present or ancient samples. It includes a parameter sampler allowing its integration into Bayesian or likelihood parameter estimation procedure. fastsimcoal can handle very complex evolutionary scenarios including an arbitrary migration matrix between samples, historical events allowing for population resize, population fusion and fission, admixture events, changes in migration matrix, or changes in population growth rates. The time of sampling can be specified independently for each sample, allowing for serial sampling in the same or in different populations. Homepage: http://cmpg.unibe.ch/software/fastsimcoal2/ URL: http://cmpg.unibe.ch/software/fastsimcoal2/</details>|
	|[filtlong](https://github.com/rrwick/Filtlong)|bio|0.2.0|<details><summary>Expand</summary>Description: Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter Homepage: https://github.com/rrwick/Filtlong URL: https://github.com/rrwick/Filtlong</details>|
	|[flexiblas](https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release)|-|3.0.4|<details><summary>Expand</summary>Description: FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it. Homepage: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release URL: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release</details>|
	|[fmriprep](https://fmriprep.org/)|-|23.0.2, 23.1.3|<details><summary>Expand</summary>Description: fMRIPrep is a NiPreps (NeuroImaging PREProcessing toolS) application (www.nipreps.org) for the preprocessing of task-based and resting-state functional MRI (fMRI). Homepage: https://fmriprep.org/ URL: https://fmriprep.org/</details>|
	|[fmt](http://fmtlib.net/)|-|5.3.0, 6.2.1, 7.0.3|<details><summary>Expand</summary>Description: fmt (formerly cppformat) is an open-source formatting library. Homepage: http://fmtlib.net/ URL: http://fmtlib.net/</details>|
	|[freesurfer](http://freesurfer.net/)|bio|5.3.0|<details><summary>Expand</summary>Description: FreeSurfer is a software package for the analysis and visualization of structural and functional neuroimaging data from cross-sectional or longitudinal studies. It is developed by the Laboratory for Computational Neuroimaging at the Athinoula A. Martinos Center for Biomedical Imaging. Homepage: http://freesurfer.net/ URL: http://freesurfer.net/</details>|
	|[g2clib](https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/)|geo|1.6.0|<details><summary>Expand</summary>Description: Library contains GRIB2 encoder/decoder ('C' version). Homepage: https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/ URL: https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/</details>|
	|[gatk](http://www.broadinstitute.org/gatk/)|bio|4.1.8.0, 4.1.8.1, 4.2.2.0, 4.2.4.0, 4.2.5.0|<details><summary>Expand</summary>Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size. Homepage: http://www.broadinstitute.org/gatk/ URL: http://www.broadinstitute.org/gatk/</details>|
	|[gblocks](http://molevol.cmima.csic.es/castresana/Gblocks.html)|-|0.91b|<details><summary>Expand</summary>Description: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis Homepage: http://molevol.cmima.csic.es/castresana/Gblocks.html URL: http://molevol.cmima.csic.es/castresana/Gblocks.html</details>|
	|[gcc](https://gcc.gnu.org/)|tools|8.4.0, 9.3.0, 10.2.0, 10.3.0, 11.3.0|<details><summary>Expand</summary>Description: The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada, as well as libraries for these languages (libstdc++, libgcj,...). Homepage: https://gcc.gnu.org/ URL: https://gcc.gnu.org/</details>|
	|[geant4-data](https://geant4.cern.ch/)|-|10.7.3, 11.1.0|<details><summary>Expand</summary>Description: Datasets for Geant4. Homepage: https://geant4.cern.ch/ URL: https://geant4.cern.ch/</details>|
	|[git-annex](http://git-annex.branchable.com/)|tools|8.20200810, 10.20221003|<details><summary>Expand</summary>Description: git-annex allows managing files with git, without checking the file contents into git. While that may seem paradoxical, it is useful when dealing with files larger than git can currently easily handle, whether due to limitations in memory, time, or disk space. Homepage: http://git-annex.branchable.com/ URL: http://git-annex.branchable.com/</details>|
	|[git-lfs](https://git-lfs.github.com)|-|2.11.0, 3.3.0|<details><summary>Expand</summary>Description: Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com Homepage: https://git-lfs.github.com URL: https://git-lfs.github.com</details>|
	|[glfw](https://www.glfw.org)|-|3.3.2|<details><summary>Expand</summary>Description: GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop Homepage: https://www.glfw.org URL: https://www.glfw.org</details>|
	|[glm](https://github.com/g-truc/glm)|vis|0.9.9.8|<details><summary>Expand</summary>Description: OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on the OpenGL Shading Language (GLSL) specifications. Homepage: https://github.com/g-truc/glm URL: https://github.com/g-truc/glm</details>|
	|[gnuplot](http://gnuplot.sourceforge.net/)|vis|5.2.8|<details><summary>Expand</summary>Description: Portable interactive, function plotting utility Homepage: http://gnuplot.sourceforge.net/ URL: http://gnuplot.sourceforge.net/</details>|
	|[go](https://www.golang.org)|tools|1.14.1, 1.18.3, 1.21.3|<details><summary>Expand</summary>Description: Go is an open source programming language that makes it easy to build simple, reliable, and efficient software. Homepage: https://www.golang.org URL: https://www.golang.org</details>|
	|[gsl](https://www.gnu.org/software/gsl/)|math|2.6|<details><summary>Expand</summary>Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting. Homepage: https://www.gnu.org/software/gsl/ URL: https://www.gnu.org/software/gsl/</details>|
	|[gsl-lite](https://github.com/gsl-lite/gsl-lite)|-|0.40.0|<details><summary>Expand</summary>Description: A single-file header-only version of ISO C++ Guidelines Support Library (GSL) for C++98, C++11, and later Homepage: https://github.com/gsl-lite/gsl-lite URL: https://github.com/gsl-lite/gsl-lite</details>|
	|[gurobi](https://www.gurobi.com)|math|9.0.3, 9.1.0, 9.1.2, 9.5.0, 9.5.2, 10.0.1, 10.0.2, 10.0.3, 11.0.0|<details><summary>Expand</summary>Description: The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms. Homepage: https://www.gurobi.com URL: https://www.gurobi.com</details>|
	|[hapgen2](https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html)|-|2.2.0|<details><summary>Expand</summary>Description: 'HAPGEN2' simulates case control datasets at SNP markers. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html</details>|
	|[haploview](https://www.broadinstitute.org/haploview/haploview)|-|4.2|<details><summary>Expand</summary>Description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses. Homepage: https://www.broadinstitute.org/haploview/haploview URL: https://www.broadinstitute.org/haploview/haploview</details>|
	|[hdf5](https://portal.hdfgroup.org/display/support)|io|1.10.6|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdf5-mpi](https://portal.hdfgroup.org/display/support)|io|1.10.6|<details><summary>Expand</summary>Description: HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data. Homepage: https://portal.hdfgroup.org/display/support URL: https://portal.hdfgroup.org/display/support</details>|
	|[hdfview](https://support.hdfgroup.org/products/java/hdfview/)|-|2.14|<details><summary>Expand</summary>Description: HDFView is a visual tool for browsing and editing HDF4 and HDF5 files. Homepage: https://support.hdfgroup.org/products/java/hdfview/ URL: https://support.hdfgroup.org/products/java/hdfview/</details>|
	|[hh-suite](https://github.com/soedinglab/hh-suite)|bio|3.3.0|<details><summary>Expand</summary>Description: HH-suite is an open-source software package for sensitive protein sequence searching. It contains programs that can search for similar protein sequences in protein sequence databases. Homepage: https://github.com/soedinglab/hh-suite URL: https://github.com/soedinglab/hh-suite</details>|
	|[hwloc](https://www.open-mpi.org/projects/hwloc/)|-|2.4.0, 2.7.1|<details><summary>Expand</summary>Description: The Portable Hardware Locality (hwloc) software package provides a portable abstraction (across OS, versions, architectures, ...) of the hierarchical topology of modern architectures, including NUMA memory nodes, sockets, shared caches, cores and simultaneous multithreading. It also gathers various system attributes such as cache and memory information as well as the locality of I/O devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily aims at helping applications with gathering information about modern computing hardware so as to exploit it accordingly and efficiently. Homepage: https://www.open-mpi.org/projects/hwloc/ URL: https://www.open-mpi.org/projects/hwloc/</details>|
	|[igblast](https://www.ncbi.nlm.nih.gov/igblast/intro.html)|bio|1.17.0, 1.18.0|<details><summary>Expand</summary>Description: IgBLAST was developed at NCBI to facilitate analysis of immunoglobulin variable domain sequences (IgBLAST has recently been extended to perform analysis for T cell receptor (TR) sequences). It uses BLAST search algorithm. Homepage: https://www.ncbi.nlm.nih.gov/igblast/intro.html URL: https://www.ncbi.nlm.nih.gov/igblast/intro.html</details>|
	|[igv](https://www.broadinstitute.org/software/igv/)|-|2.9.2|<details><summary>Expand</summary>Description: This package contains command line utilities for preprocessing, computing feature count density (coverage), sorting, and indexing data files. Homepage: https://www.broadinstitute.org/software/igv/ URL: https://www.broadinstitute.org/software/igv/</details>|
	|[imkl](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)|math|2020.1.217, 2021.2.0, 2021.4.0, 2022.1.0|<details><summary>Expand</summary>Description: Intel oneAPI Math Kernel Library Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html</details>|
	|[impute2](http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)|bio|2.3.2|<details><summary>Expand</summary>Description: IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation and haplotype phasing program based on ideas from Howie et al. 2009 Homepage: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html URL: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html</details>|
	|[impute5](https://jmarchini.org/software/#impute-5)|-|1.1.5|<details><summary>Expand</summary>Description: IMPUTE 5 is a genotype imputation method that can scale to reference panels with millions of samples. This method continues to refine the observation made in the IMPUTE2 method, that accuracy is optimized via use of a custom subset of haplotypes when imputing each individual. It achieves fast, accurate, and memory-efficient imputation by selecting haplotypes using the Positional Burrows Wheeler Transform (PBWT). By using the PBWT data structure at genotyped markers, IMPUTE 5 identifies locally best matching haplotypes and long identical by state segments. The method then uses the selected haplotypes as conditioning states within the IMPUTE model. IMPUTE5 is up to 30x faster than MINIMAC4 and up to 3x faster than BEAGLE5.1 Homepage: https://jmarchini.org/software/#impute-5 URL: https://jmarchini.org/software/#impute-5</details>|
	|[intel](https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html)|tools|2020.1.217, 2021.2.0, 2022.1.0|<details><summary>Expand</summary>Description: Intel C, C++ and Fortran compilers (classic and oneAPI) Homepage: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html URL: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html</details>|
	|[intel-opencl](https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html)|-|2021.2.0|<details><summary>Expand</summary>Description: Intel's OpenCL CPU runtime Homepage: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html URL: https://software.intel.com/content/www/us/en/develop/articles/intel-cpu-runtime-for-opencl-applications-with-sycl-support.html</details>|
	|[intelxed](https://intelxed.github.io/)|-|12.0.1|<details><summary>Expand</summary>Description: The X86 Encoder Decoder (XED), is a software library (and associated headers) for encoding and decoding X86 (IA32 and Intel64) instructions. Homepage: https://intelxed.github.io/ URL: https://intelxed.github.io/</details>|
	|[interproscan](http://www.ebi.ac.uk/interpro/)|bio|5.63-95.0, 5.64-96.0, 5.50-84.0, 5.56-89.0, 5.53-87.0, 5.55-88.0, 5.52-86.0|<details><summary>Expand</summary>Description: InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[interproscan_data](http://www.ebi.ac.uk/interpro/)|-|5.64-96.0, 5.63-95.0|<details><summary>Expand</summary>Description: This module containes only InterProScan data part. InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource. Homepage: http://www.ebi.ac.uk/interpro/ URL: http://www.ebi.ac.uk/interpro/</details>|
	|[ipp](https://software.intel.com/en-us/articles/intel-ipp/)|tools|2020.1.217|<details><summary>Expand</summary>Description: Intel Integrated Performance Primitives (Intel IPP) is an extensive library of multicore-ready, highly optimized software functions for multimedia, data processing, and communications applications. Intel IPP offers thousands of optimized functions covering frequently used fundamental algorithms. Homepage: https://software.intel.com/en-us/articles/intel-ipp/ URL: https://software.intel.com/en-us/articles/intel-ipp/</details>|
	|[ipykernel](https://ipython.org/)|-|2020a, 2020b, 2021a, 2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains ipykernel 2023b and its dependencies Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[ipython-kernel](https://ipython.org/)|-|2.7, 3.9, 3.10, 3.11|<details><summary>Expand</summary>Description: IPython kernelspec for Python 3.11 Homepage: https://ipython.org/ URL: https://ipython.org/</details>|
	|[iq-tree](http://www.iqtree.org/)|bio|2.0.7|<details><summary>Expand</summary>Description: Efficient phylogenomic software by maximum likelihood Homepage: http://www.iqtree.org/ URL: http://www.iqtree.org/</details>|
	|[itac](https://software.intel.com/en-us/intel-trace-analyzer/)|tools|2021.5.0|<details><summary>Expand</summary>Description: The Intel Trace Collector is a low-overhead tracing library that performs event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application activities gathered by the Intel Trace Collector through graphical displays. Homepage: https://software.intel.com/en-us/intel-trace-analyzer/ URL: https://software.intel.com/en-us/intel-trace-analyzer/</details>|
	|[jasper](https://www.ece.uvic.ca/~frodo/jasper/)|vis|2.0.16|<details><summary>Expand</summary>Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard. Homepage: https://www.ece.uvic.ca/~frodo/jasper/ URL: https://www.ece.uvic.ca/~frodo/jasper/</details>|
	|[java](http://openjdk.java.net)|tools|1.8.0_192, 11.0.16_8, 11.0.2, 13.0.2, 14.0.2, 17.0.2|<details><summary>Expand</summary>Description: Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers. Homepage: http://openjdk.java.net URL: http://openjdk.java.net</details>|
	|[kma](https://bitbucket.org/genomicepidemiology/kma)|-|1.3.0|<details><summary>Expand</summary>Description: KMA is a mapping method designed to map raw reads directly against redundant databases, in an ultra-fast manner using seed and extend. Homepage: https://bitbucket.org/genomicepidemiology/kma URL: https://bitbucket.org/genomicepidemiology/kma</details>|
	|[ldc](https://wiki.dlang.org/LDC)|-|0.17.6|<details><summary>Expand</summary>Description: The LLVM-based D Compiler Homepage: https://wiki.dlang.org/LDC URL: https://wiki.dlang.org/LDC</details>|
	|[libcdms](https://github.com/CDAT/libcdms/)|-|3.1.2|<details><summary>Expand</summary>Description: Climate Data Management System Library. Homepage: https://github.com/CDAT/libcdms/ URL: https://github.com/CDAT/libcdms/</details>|
	|[libcerf](https://jugit.fz-juelich.de/mlz/libcerf)|math|1.13|<details><summary>Expand</summary>Description: libcerf is a self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions. Homepage: https://jugit.fz-juelich.de/mlz/libcerf URL: https://jugit.fz-juelich.de/mlz/libcerf</details>|
	|[libdrs](https://github.com/CDAT/libdrs/)|-|3.1.2|<details><summary>Expand</summary>Description: PCMDI's old DRS format implementation Homepage: https://github.com/CDAT/libdrs/ URL: https://github.com/CDAT/libdrs/</details>|
	|[libfabric](https://ofiwg.github.io/libfabric/)|-|1.10.1|<details><summary>Expand</summary>Description: Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. Homepage: https://ofiwg.github.io/libfabric/ URL: https://ofiwg.github.io/libfabric/</details>|
	|[libffi](https://sourceware.org/libffi/)|-|3.3|<details><summary>Expand</summary>Description: The libffi library provides a portable, high level programming interface to various calling conventions. This allows a programmer to call any function specified by a call interface description at run-time. Homepage: https://sourceware.org/libffi/ URL: https://sourceware.org/libffi/</details>|
	|[libgd](https://libgd.github.io/)|vis|2.3.0|<details><summary>Expand</summary>Description: GD is an open source code library for the dynamic creation of images by programmers. Homepage: https://libgd.github.io/ URL: https://libgd.github.io/</details>|
	|[libxc](https://www.tddft.org/programs/libxc)|chem|4.3.4|<details><summary>Expand</summary>Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals. Homepage: https://www.tddft.org/programs/libxc URL: https://www.tddft.org/programs/libxc</details>|
	|[lldb](https://lldb.llvm.org/)|-|11.0.0|<details><summary>Expand</summary>Description: The debugger component of the LLVM project Homepage: https://lldb.llvm.org/ URL: https://lldb.llvm.org/</details>|
	|[llvm](https://llvm.org/)|tools|8.0.1, 9.0.1|<details><summary>Expand</summary>Description: The LLVM Core libraries provide a modern source- and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!) These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR"). The LLVM Core libraries are well documented, and it is particularly easy to invent your own language (or port an existing compiler) to use LLVM as an optimizer and code generator. Homepage: https://llvm.org/ URL: https://llvm.org/</details>|
	|[ls-opt](http://www.lstc.com/products/ls-opt)|phys|7.0.0|<details><summary>Expand</summary>Description: LS-OPT is a standalone Design Optimization and Probabilistic Analysis package with an interface to LS-DYNA. Homepage: http://www.lstc.com/products/ls-opt URL: http://www.lstc.com/products/ls-opt</details>|
	|[ls-prepost](http://lstc.com/products/ls-prepost)|-|4.8.11, 4.9.9|<details><summary>Expand</summary>Description: LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA. Homepage: http://lstc.com/products/ls-prepost URL: http://lstc.com/products/ls-prepost</details>|
	|[mariadb-connector-c](https://downloads.mariadb.org/connector-c/)|-|3.1.7|<details><summary>Expand</summary>Description: MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases. Homepage: https://downloads.mariadb.org/connector-c/ URL: https://downloads.mariadb.org/connector-c/</details>|
	|[matlab](http://www.mathworks.com/products/matlab)|tools|2020a, 2020b.4, 2020b.6, 2021a.1, 2021a.5, 2021b.3, 2022a, 2022b.2, 2023a.3|<details><summary>Expand</summary>Description: MATLAB is a high-level language and interactive environment that enables you to perform computationally intensive tasks faster than with traditional programming languages such as C, C++, and Fortran. Homepage: http://www.mathworks.com/products/matlab URL: http://www.mathworks.com/products/matlab</details>|
	|[maven](https://maven.apache.org/index.html)|tools|3.6.3|<details><summary>Expand</summary>Description: Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. Homepage: https://maven.apache.org/index.html URL: https://maven.apache.org/index.html</details>|
	|[mcr](http://www.mathworks.com/products/compiler/mcr/)|tools|R2022b, R2021b, R2020b|<details><summary>Expand</summary>Description: The MATLAB Runtime is a standalone set of shared libraries that enables the execution of compiled MATLAB applications or components on computers that do not have MATLAB installed. Homepage: http://www.mathworks.com/products/compiler/mcr/ URL: http://www.mathworks.com/products/compiler/mcr/</details>|
	|[meta-farm](https://docs.computecanada.ca/wiki/META_package_for_serial_farming)|-|1.0.2|<details><summary>Expand</summary>Description: Meta-farm is a suite of scripts designed in-house to fully automate throughput computing (running a large number of related serial/parallel/GPU jobs - so called farming). Homepage: https://docs.computecanada.ca/wiki/META_package_for_serial_farming URL: https://docs.computecanada.ca/wiki/META_package_for_serial_farming</details>|
	|[metabat](https://bitbucket.org/berkeleylab/metabat)|bio|2.12.1|<details><summary>Expand</summary>Description: MetaBAT: A robust statistical framework for reconstructing genomes from metagenomic data Homepage: https://bitbucket.org/berkeleylab/metabat URL: https://bitbucket.org/berkeleylab/metabat</details>|
	|[metageneannotator](http://metagene.nig.ac.jp/)|-|20080819|<details><summary>Expand</summary>Description: MetaGeneAnnotator is a gene-finding program for prokaryote and phage. Homepage: http://metagene.nig.ac.jp/ URL: http://metagene.nig.ac.jp/</details>|
	|[metis](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview)|math|5.1.0|<details><summary>Expand</summary>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes. Homepage: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview URL: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</details>|
	|[mii](https://github.com/codeandkey/mii)|-|1.1.1, 1.1.2|<details><summary>Expand</summary>Description: A smart search engine for module environments. Homepage: https://github.com/codeandkey/mii URL: https://github.com/codeandkey/mii</details>|
	|[minced](https://github.com/ctSkennerton/minced)|-|0.4.2|<details><summary>Expand</summary>Description: Mining CRISPRs in Environmental Datasets Homepage: https://github.com/ctSkennerton/minced URL: https://github.com/ctSkennerton/minced</details>|
	|[mixcr](http://mixcr.readthedocs.io/en/latest/quickstart.html)|bio|4.1.2|<details><summary>Expand</summary>Description: MiXCR is a universal framework that processes big immunome data from raw sequences to quantitated clonotypes. MiXCR efficiently handles paired- and single-end reads, considers sequence quality, corrects PCR errors and identifies germline hypermutations. The software supports both partial- and full-length profiling and employs all available RNA or DNA information, including sequences upstream of V and downstream of J gene segments. Homepage: http://mixcr.readthedocs.io/en/latest/quickstart.html URL: http://mixcr.readthedocs.io/en/latest/quickstart.html</details>|
	|[mpi4py](https://bitbucket.org/mpi4py/mpi4py)|tools|3.0.3, 3.1.3|<details><summary>Expand</summary>Description: MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for the Python programming language, allowing any Python program to exploit multiple processors. Homepage: https://bitbucket.org/mpi4py/mpi4py URL: https://bitbucket.org/mpi4py/mpi4py</details>|
	|[nanoflann](https://github.com/jlblancoc/nanoflann)|-|1.3.2|<details><summary>Expand</summary>Description: nanoflann is a C++11 header-only library for building KD-Trees of datasets with different topologies Homepage: https://github.com/jlblancoc/nanoflann URL: https://github.com/jlblancoc/nanoflann</details>|
	|[nccl](https://developer.nvidia.com/nccl)|tools|2.7.8, 2.8.4, 2.11.4|<details><summary>Expand</summary>Description: The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs. Homepage: https://developer.nvidia.com/nccl URL: https://developer.nvidia.com/nccl</details>|
	|[netcdf](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[netcdf-mpi](https://www.unidata.ucar.edu/software/netcdf/)|io|4.7.4|<details><summary>Expand</summary>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data. Homepage: https://www.unidata.ucar.edu/software/netcdf/ URL: https://www.unidata.ucar.edu/software/netcdf/</details>|
	|[nextflow](https://www.nextflow.io)|-|20.10.0, 21.04.3, 21.10.3, 22.04.3, 22.10.6, 22.10.8, 23.04.3|<details><summary>Expand</summary>Description: Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data Homepage: https://www.nextflow.io URL: https://www.nextflow.io</details>|
	|[nextstrain.cli](https://github.com/nextstrain/cli/tree/3.1.0)|-|3.1.0|<details><summary>Expand</summary>Description: This is the source code repository for a program called nextstrain, the Nextstrain command-line interface (CLI). It aims to provide a consistent way to run and visualize pathogen builds and access Nextstrain components like Augur and Auspice across computing environments such as Docker, Conda, and AWS Batch. Homepage: https://github.com/nextstrain/cli/tree/3.1.0 URL: https://github.com/nextstrain/cli/tree/3.1.0</details>|
	|[ninja-phylogenetics](https://github.com/TravisWheelerLab/NINJA)|-|0.97-cluster_only|<details><summary>Expand</summary>Description: Nearly Infinite Neighbor Joining Application Compute correct neighbor-joining phylogenies for inputs of more than 10,000 sequences. This is a C++/SSE port of the original Java code described in: Wheeler, T.J. 2009. Large-scale neighbor-joining with NINJA. In S.L. Salzberg and T. Warnow (Eds.), Proceedings of the 9th Workshop on Algorithms in Bioinformatics. WABI 2009, pp. 375-389. Springer, Berlin. (LNCS webpage,preprint) The Java version of NINJA was the fastest available tool computing neighbor-joining phylogenies ( 10x faster than the fastest implemenation of the canonical neighbor-joining algorithm - QuickTree ) at the time of it's release. In addition to generating phylogenies, Ninja can be used to output pairwise distances using several common sequence distance measures, and cluster sequences using a nearest-neighbor approach. Homepage: https://github.com/TravisWheelerLab/NINJA URL: https://github.com/TravisWheelerLab/NINJA</details>|
	|[nspr](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR)|-|4.25|<details><summary>Expand</summary>Description: Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR</details>|
	|[nss](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS)|-|3.51|<details><summary>Expand</summary>Description: Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications. Homepage: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS</details>|
	|[ntl](http://www.shoup.net/ntl/)|math|11.4.3|<details><summary>Expand</summary>Description: NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields. Homepage: http://www.shoup.net/ntl/ URL: http://www.shoup.net/ntl/</details>|
	|[nvhpc](https://developer.nvidia.com/hpc-sdk/)|tools|20.7, 22.1, 22.7, 23.7|<details><summary>Expand</summary>Description: C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI) Homepage: https://developer.nvidia.com/hpc-sdk/ URL: https://developer.nvidia.com/hpc-sdk/</details>|
	|[openblas](https://xianyi.github.com/OpenBLAS/)|math|0.3.17|<details><summary>Expand</summary>Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version. Homepage: https://xianyi.github.com/OpenBLAS/ URL: https://xianyi.github.com/OpenBLAS/</details>|
	|[opencoarrays](https://github.com/sourceryinstitute/opencoarrays)|-|2.9.2|<details><summary>Expand</summary>Description: OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran. Homepage: https://github.com/sourceryinstitute/opencoarrays URL: https://github.com/sourceryinstitute/opencoarrays</details>|
	|[openexr](https://www.openexr.com/)|vis|2.5.2|<details><summary>Expand</summary>Description: OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light and Magic for use in computer imaging applications Homepage: https://www.openexr.com/ URL: https://www.openexr.com/</details>|
	|[openfoam](https://www.openfoam.com/)|phys|8, v2012|<details><summary>Expand</summary>Description: OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics. Homepage: https://www.openfoam.com/ URL: https://www.openfoam.com/</details>|
	|[openimageio](https://openimageio.org/)|vis|2.1.17.0|<details><summary>Expand</summary>Description: OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities, and applications. Homepage: https://openimageio.org/ URL: https://openimageio.org/</details>|
	|[openmpi](https://www.open-mpi.org/)|mpi|4.0.3|<details><summary>Expand</summary>Description: The Open MPI Project is an open source MPI-3 implementation. Homepage: https://www.open-mpi.org/ URL: https://www.open-mpi.org/</details>|
	|[openrefine](https://github.com/OpenRefine/OpenRefine)|-|3.4.1|<details><summary>Expand</summary>Description: OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it, and augment it with data coming from the web. Homepage: https://github.com/OpenRefine/OpenRefine URL: https://github.com/OpenRefine/OpenRefine</details>|
	|[openvdb](https://github.com/AcademySoftwareFoundation/openvdb)|-|7.0.0|<details><summary>Expand</summary>Description: OpenVDB is an open source C++ library comprising a novel hierarchical data structure and a large suite of tools for the efficient storage and manipulation of sparse volumetric data discretized on three-dimensional grids. It was developed by DreamWorks Animation for use in volumetric applications typically encountered in feature film production. Homepage: https://github.com/AcademySoftwareFoundation/openvdb URL: https://github.com/AcademySoftwareFoundation/openvdb</details>|
	|[openvkl](https://github.com/OpenVKL/openvkl)|-|0.10.0|<details><summary>Expand</summary>Description: Intel Open Volume Kernel Library (Intel Open VKL) is a collection of high-performance volume computation kernels, developed at Intel. The target users of Open VKL are graphics application engineers who want to improve the performance of their volume rendering applications by leveraging Open VKL's performance-optimized kernels, which include volume traversal and sampling functionality for a variety of volumetric data formats. The kernels are optimized for the latest Intel processors with support for SSE, AVX, AVX2, and AVX-512 instructions. Homepage: https://github.com/OpenVKL/openvkl URL: https://github.com/OpenVKL/openvkl</details>|
	|[optix](https://developer.nvidia.com/optix)|-|6.5.0, 7.7.0|<details><summary>Expand</summary>Description: OptiX is NVIDIA SDK for easy ray tracing performance. It provides a simple framework for accessing the GPU's massive ray tracing power using state-of-the-art GPU algorithms. Homepage: https://developer.nvidia.com/optix URL: https://developer.nvidia.com/optix</details>|
	|[orthomcl](http://orthomcl.org/)|bio|2.0.9|<details><summary>Expand</summary>Description: OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences. Homepage: http://orthomcl.org/ URL: http://orthomcl.org/</details>|
	|[ospray](https://github.com/ospray/ospray)|-|1.8.5|<details><summary>Expand</summary>Description: Intel OSPRay is an open source, scalable, and portable ray tracing engine for high-performance, high-fidelity visualization on Intel Architecture CPUs. Homepage: https://github.com/ospray/ospray URL: https://github.com/ospray/ospray</details>|
	|[ovito](https://www.ovito.org)|-|3.3.3|<details><summary>Expand</summary>Description: OVITO is a scientific visualization and analysis software for atomistic and particle simulation data. It helps scientists gain better insights into materials phenomena and physical processes. The program is freely available for all major platforms under an open source license. It has served in a growing number of computational simulation studies as a powerful tool to analyze, understand and illustrate simulation results. Homepage: https://www.ovito.org URL: https://www.ovito.org</details>|
	|[panther](https://www.google.ca/search?q=panther)|-|14.1|<details><summary>Expand</summary>Description: Panther is the database part of InterProScan software. It contains only data. Homepage: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/ URL: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/data/</details>|
	|[paraview](https://www.paraview.org)|vis|5.8.0|<details><summary>Expand</summary>Description: ParaView is a scientific parallel visualizer. Homepage: https://www.paraview.org URL: https://www.paraview.org</details>|
	|[pcre2](https://www.pcre.org/)|-|10.34|<details><summary>Expand</summary>Description: The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. Homepage: https://www.pcre.org/ URL: https://www.pcre.org/</details>|
	|[perl](https://www.perl.org/)|tools|5.30.2|<details><summary>Expand</summary>Description: Larry Wall's Practical Extraction and Report Language Homepage: https://www.perl.org/ URL: https://www.perl.org/</details>|
	|[picard](http://samtools.github.io/hts-specs))|bio|2.23.3, 2.26.3|<details><summary>Expand</summary>Description: A set of tools (in Java) for working with next generation sequencing data in the BAM (http://samtools.github.io/hts-specs) format. Homepage: http://broadinstitute.github.io/picard/ URL: http://broadinstitute.github.io/picard/</details>|
	|[pilon](https://github.com/broadinstitute/pilon)|bio|1.23, 1.24|<details><summary>Expand</summary>Description: Pilon is an automated genome assembly improvement and variant detection tool Homepage: https://github.com/broadinstitute/pilon URL: https://github.com/broadinstitute/pilon</details>|
	|[plink](https://www.cog-genomics.org/plink/1.9/)|bio|2.00-10252019-avx2, 1.9b_6.21-x86_64|<details><summary>Expand</summary>Description: plink-1.9-x86_64: Whole-genome association analysis toolset Homepage: https://www.cog-genomics.org/plink/1.9/ URL: https://www.cog-genomics.org/plink/1.9/</details>|
	|[podman](https://podman.io)|-|4.1.1, 4.5.0|<details><summary>Expand</summary>Description: Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Homepage: https://podman.io URL: https://podman.io</details>|
	|[pplacer](http://matsen.fhcrc.org/pplacer/)|bio|1.1.alpha19|<details><summary>Expand</summary>Description: Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give useful information about uncertainty, and to offer advanced visualization and downstream analysis. Homepage: http://matsen.fhcrc.org/pplacer/ URL: http://matsen.fhcrc.org/pplacer/</details>|
	|[primme](http://www.cs.wm.edu/~andreas/software/)|-|3.2|<details><summary>Expand</summary>Description: PRIMME, pronounced as prime, is a high-performance library for computing a few eigenvalues/eigenvectors, and singular values/vectors. PRIMME is especially optimized for large, difficult problems. Real symmetric and complex Hermitian problems, standard A x = \lambda x and generalized A x = \lambda B x, are supported. Besides, standard eigenvalue problems with a normal matrix are supported. It can find largest, smallest, or interior singular/eigenvalues, and can use preconditioning to accelerate convergence. Homepage: http://www.cs.wm.edu/~andreas/software/ URL: http://www.cs.wm.edu/~andreas/software/</details>|
	|[pypy](https://pypy.org/index.html)|tools|5.8.0, 7.3.3|<details><summary>Expand</summary>Description: PyPy is a fast, compliant alternative implementation of the Python language (2.7.13 and 3.5.3). Homepage: https://pypy.org/index.html URL: https://pypy.org/index.html</details>|
	|[pytest](https://pytest.org)|-|6.1.2, 6.2.5, 7.0.1, 7.4.0|<details><summary>Expand</summary>Description: pytest: simple powerful testing with Python Homepage: https://pytest.org URL: https://pytest.org</details>|
	|[python](https://python.org/)|tools|2.7.18, 3.6.10, 3.7.7, 3.7.9, 3.8.2, 3.8.10, 3.9.6, 3.10.2, 3.11.2, 3.11.5|<details><summary>Expand</summary>Description: Python is a programming language that lets you work more quickly and integrate your systems more effectively. Homepage: https://python.org/ URL: https://python.org/</details>|
	|[python-build-bundle](http://python.org/)|-|2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains Python packages useful for building Homepage: http://python.org/ URL: http://python.org/</details>|
	|[qiime2](http://qiime2.org/)|-|2021.11, 2023.5|<details><summary>Expand</summary>Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. Homepage: http://qiime2.org/ URL: http://qiime2.org/</details>|
	|[qt](https://qt.io/)|tools|5.12.8|<details><summary>Expand</summary>Description: Qt is a comprehensive cross-platform C++ application framework. Homepage: https://qt.io/ URL: https://qt.io/</details>|
	|[quantumatk](https://www.synopsys.com/silicon/quantumatk.html)|-|2019.12|<details><summary>Expand</summary>Description: QuantumATK is a complete atomistic simulation toolkit. QuantumATK addresses key applications in the semiconductor industry and is a core component of the Synopsys Design-Technology Co-Optimization (DTCO) solution. Atomic-scale modeling tools in QuantumATK range from classical force fields for handling large and to some extent more realistic materials systems to ab initio tools that provide highly accurate results for smaller systems. Homepage: https://www.synopsys.com/silicon/quantumatk.html URL: https://www.synopsys.com/silicon/quantumatk.html</details>|
	|[quantumespresso](https://www.quantum-espresso.org)|chem|6.6|<details><summary>Expand</summary>Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). Homepage: https://www.quantum-espresso.org URL: https://www.quantum-espresso.org</details>|
	|[r](https://www.r-project.org/)|tools|4.1.0, 4.2.1|<details><summary>Expand</summary>Description: R is a free software environment for statistical computing and graphics. Homepage: https://www.r-project.org/ URL: https://www.r-project.org/</details>|
	|[racon](https://github.com/lbcb-sci/racon)|bio|1.4.13|<details><summary>Expand</summary>Description: Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. Homepage: https://github.com/lbcb-sci/racon URL: https://github.com/lbcb-sci/racon</details>|
	|[rapidjson](https://rapidjson.org)|-|1.1.0|<details><summary>Expand</summary>Description: A fast JSON parser/generator for C++ with both SAX/DOM style API Homepage: https://rapidjson.org URL: https://rapidjson.org</details>|
	|[re2c](https://re2c.org/)|-|1.3|<details><summary>Expand</summary>Description: re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons. Homepage: https://re2c.org/ URL: https://re2c.org/</details>|
	|[reframe](https://github.com/eth-cscs/reframe)|-|3.12.0|<details><summary>Expand</summary>Description: ReFrame is a framework for writing regression tests for HPC systems. Homepage: https://github.com/eth-cscs/reframe URL: https://github.com/eth-cscs/reframe</details>|
	|[rkcommon](https://github.com/ospray/rkcommon)|-|1.4.2|<details><summary>Expand</summary>Description: Intel RenderKit common library Homepage: https://github.com/ospray/rkcommon URL: https://github.com/ospray/rkcommon</details>|
	|[ruby](https://www.ruby-lang.org)|tools|2.7.1|<details><summary>Expand</summary>Description: Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write. Homepage: https://www.ruby-lang.org URL: https://www.ruby-lang.org</details>|
	|[rust](https://www.rust-lang.org)|tools|1.47.0, 1.53.0, 1.59.0, 1.65.0, 1.70.0|<details><summary>Expand</summary>Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. Homepage: https://www.rust-lang.org URL: https://www.rust-lang.org</details>|
	|[salmonbeta](https://github.com/COMBINE-lab/salmon/)|bio|0.6.0|<details><summary>Expand</summary>Description: Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data. Salmon achieves is accuracy and speed via a number of different innovations, including the use of quasi-mapping (accurate but fast-to-compute proxies for traditional read alignments), and massively-parallel stochastic collapsed variational inference. The result is a versatile tool that fits nicely into many differnt pipelines. For example, you can choose to make use of our quasi-mapping algorithm by providing Salmon with raw sequencing reads, or, if it is more convenient, you can provide Salmon with regular alignments (e.g. an unsorted BAM file produced with your favorite aligner), and it will use the same wicked-fast, state-of-the-art inference algorithm to estimate transcript-level abundances for your experiment. Homepage: https://github.com/COMBINE-lab/salmon/ URL: https://github.com/COMBINE-lab/salmon/</details>|
	|[sambamba](https://lomereiter.github.io/sambamba/)|bio|0.8.0|<details><summary>Expand</summary>Description: Sambamba is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Current functionality is an important subset of samtools functionality, including view, index, sort, markdup, and depth. Homepage: https://lomereiter.github.io/sambamba/ URL: https://lomereiter.github.io/sambamba/</details>|
	|[samstat](http://samstat.sourceforge.net/)|-|1.5.1|<details><summary>Expand</summary>Description: Displaying sequence statistics for next generation sequencing Homepage: http://samstat.sourceforge.net/ URL: http://samstat.sourceforge.net/</details>|
	|[samtools](https://www.htslib.org/)|bio|1.10|<details><summary>Expand</summary>Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format. Homepage: https://www.htslib.org/ URL: https://www.htslib.org/</details>|
	|[sbt](http://www.scala-sbt.org/)|tools|1.3.13|<details><summary>Expand</summary>Description: sbt is a build tool for Scala, Java, and more. It requires Java 1.8 or later. sbt uses a small number of concepts to support flexible and powerful build definitions. There are not that many concepts, but sbt is not exactly like other build systems and there are details you will stumble on if you haven t read the documentation. The Getting Started Guide covers the concepts you need to know to create and maintain an sbt build definition. It is highly recommended to read the Getting Started Guide. Homepage: http://www.scala-sbt.org/ URL: http://www.scala-sbt.org/</details>|
	|[scipy-stack](http://python.org/)|math|2020a, 2020b, 2021a, 2022a, 2023a, 2023b|<details><summary>Expand</summary>Description: Bundle which contains the Scientific Python stack, including Cycler, mpmath, numpy, scipy, sympy, pandas, matplotlib, ipython_genutils, traitlets, ptyprocess, pathlib2, pickleshare, pexpect, simplegeneric, ipython, ipykernel, jupyter_client, jupyter_core, pyzmq, tornado, futures and ipyparallel. Homepage: http://python.org/ URL: http://python.org/</details>|
	|[scotch](https://gforge.inria.fr/projects/scotch/)|math|6.0.9|<details><summary>Expand</summary>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning. Homepage: https://gforge.inria.fr/projects/scotch/ URL: https://gforge.inria.fr/projects/scotch/</details>|
	|[seqan-library](https://www.seqan.de/)|-|2.4.0|<details><summary>Expand</summary>Description: SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences with the focus on biological data Homepage: https://www.seqan.de/ URL: https://www.seqan.de/</details>|
	|[seqkit](https://bioinf.shenwei.me/seqkit/)|-|0.13.2, 0.15.0, 2.3.1|<details><summary>Expand</summary>Description: FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. This project describes a cross-platform ultrafast comprehensive toolkit for FASTA/Q processing. SeqKit provides executable binary files for all major operating systems, including Windows, Linux, and Mac OS X, and can be directly used without any dependencies or pre-configurations. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations. Homepage: https://bioinf.shenwei.me/seqkit/ URL: https://bioinf.shenwei.me/seqkit/</details>|
	|[shapeit](https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html)|-|2.r904|<details><summary>Expand</summary>Description: SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data. Homepage: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html URL: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</details>|
	|[shrinkwrap](https://github.com/jonathonl/shrinkwrap)|-|1.2.0|<details><summary>Expand</summary>Description: A std::streambuf wrapper for compression formats. Homepage: https://github.com/jonathonl/shrinkwrap URL: https://github.com/jonathonl/shrinkwrap</details>|
	|[signalp](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp)|bio|4.1f|<details><summary>Expand</summary>Description: SignalP predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive bacteria, Gram-negative bacteria, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp</details>|
	|[singular](https://www.singular.uni-kl.de/)|-|4.2.1|<details><summary>Expand</summary>Description: Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory. Homepage: https://www.singular.uni-kl.de/ URL: https://www.singular.uni-kl.de/</details>|
	|[slicer](https://www.slicer.org/)|-|4.11.20210226|<details><summary>Expand</summary>Description: 3D Slicer is an open source software platform for medical image informatics, image processing, and three-dimensional visualization. Built over two decades through support from the National Institutes of Health and a worldwide developer community, Slicer brings free, powerful cross-platform processing tools to physicians, researchers, and the general public. Homepage: https://www.slicer.org/ URL: https://www.slicer.org/</details>|
	|[snappy](https://github.com/google/snappy)|tools|1.1.8|<details><summary>Expand</summary>Description: Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression. Homepage: https://github.com/google/snappy URL: https://github.com/google/snappy</details>|
	|[snpeff](https://pcingola.github.io/SnpEff/)|bio|5.0|<details><summary>Expand</summary>Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes). Homepage: https://pcingola.github.io/SnpEff/ URL: https://pcingola.github.io/SnpEff/</details>|
	|[spark](http://spark.apache.org)|tools|3.0.0, 3.3.0|<details><summary>Expand</summary>Description: Spark is Hadoop MapReduce done in memory Homepage: http://spark.apache.org URL: http://spark.apache.org</details>|
	|[spectra](https://spectralib.org/index.html)|-|0.9.0|<details><summary>Expand</summary>Description: SpectrA stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library. SpectrA is implemented as a header-only C++ library, whose only dependency, Eigen, is also header-only. Hence SpectrA can be easily embedded in C++ projects that require calculating eigenvalues of large matrices. Homepage: https://spectralib.org/index.html URL: https://spectralib.org/index.html</details>|
	|[sqlite](https://www.sqlite.org/)|-|3.43.1|<details><summary>Expand</summary>Description: SQLite: SQL Database Engine in a C Library Homepage: https://www.sqlite.org/ URL: https://www.sqlite.org/</details>|
	|[starccm](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010-R8, 16.02.008-R8, 16.02.009-R8, 16.04.007-R8, 16.04.012-R8, 16.06.008-R8, 17.02.007-R8, 17.02.008-R8, 17.04.007-R8, 17.04.008-R8, 17.06.007-R8, 17.06.008-R8, 18.02.008-R8, 18.04.008-R8, 18.06.006-R8|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses double precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[starccm-mixed](http://mdx.plm.automation.siemens.com/star-ccm-plus)|phys|15.04.010, 16.02.008, 16.02.009, 16.04.007, 16.04.012, 16.06.008, 17.02.007, 17.02.008, 17.04.007, 17.04.008, 17.06.007, 17.06.008, 18.02.008, 18.04.008, 18.06.006|<details><summary>Expand</summary>Description: Predicting the real-world performance of a product requires simulation tools that span a variety of engineering disciplines. STAR-CCM+ is an all-in-one solution that delivers accurate and efficient multidisciplinary technologies in a single integrated user interface. This version uses mixed precision. Homepage: http://mdx.plm.automation.siemens.com/star-ccm-plus URL: http://mdx.plm.automation.siemens.com/star-ccm-plus</details>|
	|[supernova](https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome)|bio|2.1.1|<details><summary>Expand</summary>Description: Supernova is delivered as a single, self-contained tar file that can be unpacked anywhere on your system. It bundles all of its own required software dependencies, which are pre-compiled to run on a range of Linux distributions. Homepage: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome URL: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome</details>|
	|[swi-prolog](https://www.swi-prolog.org/)|-|9.0.3|<details><summary>Expand</summary>Description: SWI-Prolog offers a comprehensive free Prolog environment. Homepage: https://www.swi-prolog.org/ URL: https://www.swi-prolog.org/</details>|
	|[swig](http://www.swig.org/)|-|4.0.1|<details><summary>Expand</summary>Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages. Homepage: http://www.swig.org/ URL: http://www.swig.org/</details>|
	|[taxonkit](https://bioinf.shenwei.me/taxonkit/)|-|0.6.2|<details><summary>Expand</summary>Description: TaxonKit - A Cross-platform and Efficient NCBI Taxonomy Toolkit TaxonKit is a command-line toolkit for rapid manipulation of NCBI taxonomy data. It provides executable binary files for major operating systems including Windows, Linux, and Mac OS X, and can be directly used without any dependencies nor local database buiding. Homepage: https://bioinf.shenwei.me/taxonkit/ URL: https://bioinf.shenwei.me/taxonkit/</details>|
	|[tbb](https://github.com/oneapi-src/oneTBB)|tools|2020.2|<details><summary>Expand</summary>Description: Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability. Homepage: https://github.com/oneapi-src/oneTBB URL: https://github.com/oneapi-src/oneTBB</details>|
	|[tbl2asn](https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/)|-|25.8|<details><summary>Expand</summary>Description: Tbl2asn is a command-line program that automates the creation of sequence records for submission to GenBank Homepage: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/ URL: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/</details>|
	|[tensorrt](https://developer.nvidia.com/tensorrt)|-|8.6.1.6|<details><summary>Expand</summary>Description: NVIDIA TensorRT is a platform for high-performance deep learning inference Homepage: https://developer.nvidia.com/tensorrt URL: https://developer.nvidia.com/tensorrt</details>|
	|[tmhmm](http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm)|bio|2.0c|<details><summary>Expand</summary>Description: TMHMM 2.0 predicts transmembrane helices in proteins. The method is described in: Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes. Anders Krogh and Bjorn Larsson, Gunnar von Heijne, and Erik L.L. Sonnhammer. J. Mol. Biol. 305:567-580, 2001. and: A hidden Markov model for predicting transmembrane helices in protein sequences. Erik L.L. Sonnhammer, Gunnar von Heijne, and Anders Krogh. In J. Glasgow et al., eds.: Proc. of ISMB6, pages 175-182. AAAI Press, 1998. Homepage: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm URL: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?tmhmm</details>|
	|[transrate](http://hibberdlab.com/transrate/)|bio|1.0.3|<details><summary>Expand</summary>Description: Transrate is software for de-novo transcriptome assembly quality analysis. It examines your assembly in detail and compares it to experimental evidence such as the sequencing reads, reporting quality scores for contigs and assemblies. This allows you to choose between assemblers and parameters, filter out the bad contigs from an assembly, and help decide when to stop trying to improve the assembly. Homepage: http://hibberdlab.com/transrate/ URL: http://hibberdlab.com/transrate/</details>|
	|[trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)|bio|0.39|<details><summary>Expand</summary>Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.The selection of trimming steps and their associated parameters are supplied on the command line. Homepage: http://www.usadellab.org/cms/?page=trimmomatic URL: http://www.usadellab.org/cms/?page=trimmomatic</details>|
	|[ucx](http://www.openucx.org/)|-|1.8.0|<details><summary>Expand</summary>Description: Unified Communication X An open-source production grade communication framework for data centric and high-performance applications Homepage: http://www.openucx.org/ URL: http://www.openucx.org/</details>|
	|[udunits](https://www.unidata.ucar.edu/software/udunits/)|tools|2.2.26|<details><summary>Expand</summary>Description: UDUNITS supports conversion of unit specifications between formatted and binary forms, arithmetic manipulation of units, and conversion of values between compatible scales of measurement. Homepage: https://www.unidata.ucar.edu/software/udunits/ URL: https://www.unidata.ucar.edu/software/udunits/</details>|
	|[unixodbc](https://www.unixodbc.org)|-|2.3.9|<details><summary>Expand</summary>Description: unixODBC provides a uniform interface between application and database driver Homepage: https://www.unixodbc.org URL: https://www.unixodbc.org</details>|
	|[varscan](https://github.com/dkoboldt/varscan)|-|2.4.2|<details><summary>Expand</summary>Description: Variant calling and somatic mutation/CNV detection for next-generation sequencing data Homepage: https://github.com/dkoboldt/varscan URL: https://github.com/dkoboldt/varscan</details>|
	|[visit](https://wci.llnl.gov/simulation/computer-codes/visit)|vis|2.13.3|<details><summary>Expand</summary>Description: VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool. Homepage: https://wci.llnl.gov/simulation/computer-codes/visit URL: https://wci.llnl.gov/simulation/computer-codes/visit</details>|
	|[vtune](https://software.intel.com/en-us/vtune)|tools|2020.1, 2022.2|<details><summary>Expand</summary>Description: Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran, Assembly and Java. Homepage: https://software.intel.com/en-us/vtune URL: https://software.intel.com/en-us/vtune</details>|
	|[wasp](https://code.ornl.gov/neams-workbench/wasp)|-|3.1.4, 4.0.3|<details><summary>Expand</summary>Description: The Workbench Analysis Sequence Processor (WASP) was developed to streamline lexing, parsing, access, validation, and analysis of ascii text files. Homepage: https://code.ornl.gov/neams-workbench/wasp URL: https://code.ornl.gov/neams-workbench/wasp</details>|
	|[xml-libxml](https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod)|-|2.0205|<details><summary>Expand</summary>Description: Perl binding for libxml2 Homepage: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod URL: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod</details>|
	|[xmlf90](https://launchpad.net/xmlf90/)|tools|1.5.4|<details><summary>Expand</summary>Description: xmlf90 is a suite of libraries to handle XML in Fortran. It has two major components: - A XML parsing library. The parser was designed to be a useful tool in the extraction and analysis of data in the context of scientific computing, and thus the priorities were efficiency and the ability to deal with very large XML files while maintaining a small memory footprint. The most complete programming interface is based on the very successful SAX (Simple API for XML) model, although a partial DOM interface and a very experimental XPATH interface are also present. - A library (xmlf90-wxml) that facilitates the writing of well-formed XML, including such features as automatic start-tag completion, attribute pretty-printing, and element indentation. There are also helper routines to handle the output of numerical arrays. Homepage: https://launchpad.net/xmlf90/ URL: https://launchpad.net/xmlf90/</details>|
	|[xtensor](https://github.com/xtensor-stack/xtensor)|-|0.24.2|<details><summary>Expand</summary>Description: xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions. Homepage: https://github.com/xtensor-stack/xtensor URL: https://github.com/xtensor-stack/xtensor</details>|
